<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content="Downloads and returns a pre-processed dataset of GENCODE transcripts.
The dataset comes as a list object, with two elements:
metadata - a data frame with metadata regarding annotation
resources used
records - a list with two data frames of transcripts, one
for grch37, and one for grch38 (see specific column format below)


The GENCODE datasets provided were established by parsing the GENCODE
comprehensive
gene annotation GTF file with the
valr package, and
also providing cross-references to RefSeq (through the use of the
biomaRt
package), and UniProt accessions/identifiers."><title>Get human GENCODE transcript dataset — get_gencode • geneOncoX</title><!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png"><link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png"><link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png"><link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png"><link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png"><link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png"><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Get human GENCODE transcript dataset — get_gencode"><meta property="og:description" content="Downloads and returns a pre-processed dataset of GENCODE transcripts.
The dataset comes as a list object, with two elements:
metadata - a data frame with metadata regarding annotation
resources used
records - a list with two data frames of transcripts, one
for grch37, and one for grch38 (see specific column format below)


The GENCODE datasets provided were established by parsing the GENCODE
comprehensive
gene annotation GTF file with the
valr package, and
also providing cross-references to RefSeq (through the use of the
biomaRt
package), and UniProt accessions/identifiers."><meta property="og:image" content="https://sigven.github.io/geneOncoX/logo.png"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><script async src="https://www.googletagmanager.com/gtag/js?id=G-GSP3QKXJ81"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-GSP3QKXJ81');
</script></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">geneOncoX</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.8.2</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="nav-item">
  <a class="nav-link" href="../articles/geneOncoX.html">Get started</a>
</li>
<li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="../news/index.html">Changelog</a>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/sigven/geneOncoX/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.png" class="logo" alt=""><h1>Get human GENCODE transcript dataset</h1>
      <small class="dont-index">Source: <a href="https://github.com/sigven/geneOncoX/blob/HEAD/R/get_gencode.R" class="external-link"><code>R/get_gencode.R</code></a></small>
      <div class="d-none name"><code>get_gencode.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>Downloads and returns a pre-processed dataset of GENCODE transcripts.
The dataset comes as a <code>list</code> object, with two elements:</p><ul><li><p><code>metadata</code> - a data frame with metadata regarding annotation
resources used</p></li>
<li><p><code>records</code> - a list with two data frames of transcripts, one
for <code>grch37</code>, and one for <code>grch38</code> (see specific column format below)</p></li>
</ul><p>The GENCODE datasets provided were established by parsing the GENCODE
<a href="https://www.gencodegenes.org/human/" class="external-link">comprehensive
gene annotation GTF file</a> with the
<a href="https://rnabioco.github.io/valr/" class="external-link">valr</a> package, and
also providing cross-references to RefSeq (through the use of the
<a href="https://bioconductor.org/packages/biomaRt/" class="external-link">biomaRt</a>
package), and UniProt accessions/identifiers.</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">get_gencode</span><span class="op">(</span>cache_dir <span class="op">=</span> <span class="cn">NA</span>, force_download <span class="op">=</span> <span class="cn">FALSE</span>, ensembl_release <span class="op">=</span> <span class="fl">111</span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="source">Source<a class="anchor" aria-label="anchor" href="#source"></a></h2>
    <p><a href="https://www.gencodegenes.org/" class="external-link">https://www.gencodegenes.org/</a></p>
<p><a href="https://uniprot.org" class="external-link">https://uniprot.org</a></p>
<p><a href="https://www.ncbi.nlm.nih.gov/gene" class="external-link">https://www.ncbi.nlm.nih.gov/gene</a></p>
<p><a href="https://apprisws.bioinfo.cnio.es/landing_page/" class="external-link">https://apprisws.bioinfo.cnio.es/landing_page/</a></p>
    </div>
    <div class="section level2">
    <h2 id="arguments">Arguments<a class="anchor" aria-label="anchor" href="#arguments"></a></h2>
    <dl><dt>cache_dir</dt>
<dd><p>Local directory for data download</p></dd>


<dt>force_download</dt>
<dd><p>Logical indicating if local cache should be overwritten
(set to TRUE to re-download if file exists in cache)</p></dd>


<dt>ensembl_release</dt>
<dd><p>version of Ensembl to use - this will
dictate the version of GENCODE used for grch38 (e.g. 110 implies
GENCODE v44, 109 implies v43, 108 implies v42 and so on)</p></dd>

</dl></div>
    <div class="section level2">
    <h2 id="value">Value<a class="anchor" aria-label="anchor" href="#value"></a></h2>
    

<p><strong>metadata</strong> - A data frame with 3 rows and 6 columns:</p><ul><li><p><em>source</em> - gene annotation source</p></li>
<li><p><em>annotation_data</em> - type of annotations used</p></li>
<li><p><em>url</em> - URL of annotation resource</p></li>
<li><p><em>citation</em> - publication to cite for annotation source
(citation; PMID)</p></li>
<li><p><em>version</em> - version used</p></li>
<li><p><em>abbreviation</em> - abbreviation (e.g. used in column
names of records)</p></li>
</ul><p><strong>records</strong> - A list with one data frame per assembly</p><ul><li><p><em>chrom</em> - chromosome</p></li>
<li><p><em>start</em> - transcript start with 5kb padding (upstream)</p></li>
<li><p><em>end</em> - transcript end with 5kb padding (downstream)</p></li>
<li><p><em>transcript_start</em> - transcript start</p></li>
<li><p><em>transcript_end</em> - transcript end</p></li>
<li><p><em>strand</em> - strand</p></li>
<li><p><em>ensembl_gene_id</em> - Ensembl gene identifier</p></li>
<li><p><em>ensembl_transcript_id</em> - Ensembl transcript identifier</p></li>
<li><p><em>ensembl_transcript_id_full</em> - Ensembl transcript
identifier (with version)</p></li>
<li><p><em>ensembl_protein_id</em> - Ensembl protein identifier</p></li>
<li><p><em>symbol</em> - official gene symbol</p></li>
<li><p><em>hgnc_id</em> - HGNC gene identifier</p></li>
<li><p><em>entrezgene</em> - Entrez gene identifier</p></li>
<li><p><em>name</em> - genename</p></li>
<li><p><em>gene_biotype</em> -
<a href="https://www.gencodegenes.org/pages/biotypes.html" class="external-link">gene biotype</a>
(GENCODE)</p></li>
<li><p><em>transcript_biotype</em> -
<a href="https://www.gencodegenes.org/pages/biotypes.html" class="external-link">transcript biotype</a>
(GENCODE))</p></li>
<li><p><em>tag</em> -
<a href="https://www.gencodegenes.org/pages/biotypes.html" class="external-link">tag</a> (GENCODE))</p></li>
<li><p><em>refseq_protein_id</em> - RefSeq peptide identifier</p></li>
<li><p><em>refseq_transcript_id</em> - RefSeq mRNA identifier</p></li>
<li><p><em>mane_select</em> -
<a href="https://www.ncbi.nlm.nih.gov/refseq/MANE/" class="external-link">MANE</a> Select</p></li>
<li><p><em>mane_plus_clinical</em> -
<a href="https://www.ncbi.nlm.nih.gov/refseq/MANE/" class="external-link">MANE</a> Plus Clinical</p></li>
<li><p><em>principal_isoform_flag</em> - Principal
<a href="https://appris.bioinfo.cnio.es/#/help/scores" class="external-link">isoform tag</a> (APPRIS)</p></li>
<li><p><em>uniprot_acc</em> - UniProtKB protein accession</p></li>
<li><p><em>uniprot_id</em> - UniProtKB protein identifier</p></li>
<li><p><em>ensembl_version</em> - Ensembl version</p></li>
<li><p><em>gencode_version</em> - GENCODE version</p></li>
<li><p><em>uniprot_version</em> - UniProtKB version</p></li>
</ul><p>list object with GENCODE transcripts (records) and associated
metadata (metadata)</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="kw">if</span> <span class="op">(</span><span class="cn">FALSE</span><span class="op">)</span> <span class="op">{</span></span></span>
<span class="r-in"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/sigven/geneOncoX" class="external-link">geneOncoX</a></span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="va">transcripts_gencode</span> <span class="op">&lt;-</span> <span class="fu">get_gencode</span><span class="op">(</span>cache_dir <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/tempfile.html" class="external-link">tempdir</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="op">}</span></span></span>
<span class="r-in"><span></span></span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p></p><p>Developed by <a href="https://github.com/sigven" class="external-link">Sigve Nakken</a>.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer></div>

  

  

  </body></html>

