<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Getting started • geneOncoX</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="96x96" href="../favicon-96x96.png">
<link rel="icon" type="”image/svg+xml”" href="../favicon.svg">
<link rel="apple-touch-icon" sizes="180x180" href="../apple-touch-icon.png">
<link rel="icon" sizes="any" href="../favicon.ico">
<link rel="manifest" href="../site.webmanifest">
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/font-awesome-6.5.2/css/all.min.css" rel="stylesheet">
<link href="../deps/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet">
<script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Getting started">
<script async src="https://www.googletagmanager.com/gtag/js?id=G-GSP3QKXJ81"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-GSP3QKXJ81');
</script>
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-info" data-bs-theme="light" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">geneOncoX</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.3.0</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="active nav-item"><a class="nav-link" href="../articles/geneOncoX.html">Get started</a></li>
<li class="nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="nav-item"><a class="nav-link" href="../news/index.html">Changelog</a></li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><form class="form-inline" role="search">
 <input class="form-control" type="search" name="search-input" id="search-input" autocomplete="off" aria-label="Search site" placeholder="Search for" data-search-index="../search.json">
</form></li>
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/sigven/geneOncoX/" aria-label="GitHub"><span class="fa fab fa-github fa-lg"></span></a></li>
      </ul>
</div>


  </div>
</nav><div class="container template-article">



<script src="geneOncoX_files/htmlwidgets-1.6.4/htmlwidgets.js"></script><link href="geneOncoX_files/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="geneOncoX_files/datatables-binding-0.34.0/datatables.js"></script><link href="geneOncoX_files/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="geneOncoX_files/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="geneOncoX_files/dt-core-1.13.6/js/jquery.dataTables.min.js"></script><script src="geneOncoX_files/jszip-1.13.6/jszip.min.js"></script><link href="geneOncoX_files/dt-ext-buttons-1.13.6/css/buttons.dataTables.min.css" rel="stylesheet">
<script src="geneOncoX_files/dt-ext-buttons-1.13.6/js/dataTables.buttons.min.js"></script><script src="geneOncoX_files/dt-ext-buttons-1.13.6/js/buttons.html5.min.js"></script><script src="geneOncoX_files/dt-ext-buttons-1.13.6/js/buttons.colVis.min.js"></script><script src="geneOncoX_files/dt-ext-buttons-1.13.6/js/buttons.print.min.js"></script><link href="geneOncoX_files/dt-ext-responsive-1.13.6/css/responsive.dataTables.min.css" rel="stylesheet">
<script src="geneOncoX_files/dt-ext-responsive-1.13.6/js/dataTables.responsive.min.js"></script><link href="geneOncoX_files/crosstalk-1.2.2/css/crosstalk.min.css" rel="stylesheet">
<script src="geneOncoX_files/crosstalk-1.2.2/js/crosstalk.min.js"></script><div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.png" class="logo" alt=""><h1>Getting started</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/sigven/geneOncoX/blob/main/vignettes/geneOncoX.Rmd" class="external-link"><code>vignettes/geneOncoX.Rmd</code></a></small>
      <div class="d-none name"><code>geneOncoX.Rmd</code></div>
    </div>

    
    
<p><br><br></p>
<div class="section level2">
<h2 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h2>
<p><strong>IMPORTANT NOTE</strong>: <em>geneOncoX</em> requires that you
have R version 4.1 or higher installed</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="op">(</span><span class="st">"remotes"</span> <span class="op"><a href="https://rdrr.io/r/base/match.html" class="external-link">%in%</a></span> <span class="fu"><a href="https://rdrr.io/r/utils/installed.packages.html" class="external-link">installed.packages</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">{</span></span>
<span>  <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"remotes"</span><span class="op">)</span></span>
<span><span class="op">}</span></span>
<span><span class="fu">remotes</span><span class="fu">::</span><span class="fu">install_github</span><span class="op">(</span><span class="st">'sigven/geneOncoX'</span><span class="op">)</span></span></code></pre></div>
<p><br><br></p>
</div>
<div class="section level2">
<h2 id="get-basic-gene-annotations">Get basic gene annotations<a class="anchor" aria-label="anchor" href="#get-basic-gene-annotations"></a>
</h2>
<p>This shows how to retrieve basic gene and cancer relevant gene
annotations, including how to retrieve tumor suppressor genes,
proto-oncogenes, and predicted cancer driver genes.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://sigven.github.io/geneOncoX">geneOncoX</a></span><span class="op">)</span></span>
<span></span>
<span><span class="co">## load the data</span></span>
<span><span class="va">download_dir</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/tempfile.html" class="external-link">tempdir</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">gene_basic</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/get_basic.html">get_basic</a></span><span class="op">(</span>cache_dir <span class="op">=</span> <span class="va">download_dir</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Number of records</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">nrow</a></span><span class="op">(</span><span class="va">gene_basic</span><span class="op">$</span><span class="va">records</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 65407</span></span>
<span></span>
<span><span class="co">## Show metadata for underlying resources</span></span>
<span><span class="va">gene_basic</span><span class="op">$</span><span class="va">metadata</span></span>
<span><span class="co">#&gt;                             source</span></span>
<span><span class="co">#&gt; 1                       CancerMine</span></span>
<span><span class="co">#&gt; 2          Network of Cancer Genes</span></span>
<span><span class="co">#&gt; 3                          IntoGen</span></span>
<span><span class="co">#&gt; 4                             NCBI</span></span>
<span><span class="co">#&gt; 5        Bailey et al., Cell, 2018</span></span>
<span><span class="co">#&gt; 6  Sanchez-Vega et al., Cell, 2018</span></span>
<span><span class="co">#&gt; 7                            F1CDx</span></span>
<span><span class="co">#&gt; 8                           TSO500</span></span>
<span><span class="co">#&gt; 9         DNA repair gene database</span></span>
<span><span class="co">#&gt; 10                          dbNSFP</span></span>
<span><span class="co">#&gt; 11                            CPIC</span></span>
<span><span class="co">#&gt;                                                                       source_description</span></span>
<span><span class="co">#&gt; 1    Predicted tumor suppressors/oncogenes/cancer drivers from text mining of literature</span></span>
<span><span class="co">#&gt; 2                                                            Tumor suppressors/oncogenes</span></span>
<span><span class="co">#&gt; 3                                                          Predicted cancer driver genes</span></span>
<span><span class="co">#&gt; 4  Basic gene identifiers (symbol, entrez ID, HGNC ID, name, synonyms, function summary)</span></span>
<span><span class="co">#&gt; 5                     Predicted cancer driver genes / likely false positive driver genes</span></span>
<span><span class="co">#&gt; 6                                   Collection of curated signalling pathway annotations</span></span>
<span><span class="co">#&gt; 7                Collection of genes covered by Foundation One's F1CDx cancer gene panel</span></span>
<span><span class="co">#&gt; 8                     Collection of genes covered by Illumina's TSO500 cancer gene panel</span></span>
<span><span class="co">#&gt; 9                                             Collection of genes involved in DNA repair</span></span>
<span><span class="co">#&gt; 10       Gene indispensability prediction, loss-of-function intolerance predictions etc.</span></span>
<span><span class="co">#&gt; 11        Clinical Pharmacogenomics Implementation Consortium - gene/oncology drug pairs</span></span>
<span><span class="co">#&gt;                                                                   source_url</span></span>
<span><span class="co">#&gt; 1                                         http://bionlp.bcgsc.ca/cancermine/</span></span>
<span><span class="co">#&gt; 2                                           http://network-cancer-genes.org/</span></span>
<span><span class="co">#&gt; 3                                           https://www.intogen.org/download</span></span>
<span><span class="co">#&gt; 4                                         https://www.ncbi.nlm.nih.gov/gene/</span></span>
<span><span class="co">#&gt; 5                                  https://pubmed.ncbi.nlm.nih.gov/29625053/</span></span>
<span><span class="co">#&gt; 6                                  https://pubmed.ncbi.nlm.nih.gov/29625050/</span></span>
<span><span class="co">#&gt; 7                  https://www.foundationmedicine.com/test/foundationone-cdx</span></span>
<span><span class="co">#&gt; 8  https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0260089</span></span>
<span><span class="co">#&gt; 9                         https://panelapp.genomicsengland.co.uk/panels/256/</span></span>
<span><span class="co">#&gt; 10                              https://sites.google.com/site/jpopgen/dbNSFP</span></span>
<span><span class="co">#&gt; 11                                          https://cpicpgx.org/genes-drugs/</span></span>
<span><span class="co">#&gt;                                            source_citation    source_version</span></span>
<span><span class="co">#&gt; 1                Lever et al., Nat Methods, 2019; 31110280 v51 (August 2025)</span></span>
<span><span class="co">#&gt; 2               Repana et al., Genome Biol, 2019; 30606230              v7.2</span></span>
<span><span class="co">#&gt; 3  Martínez-Jiménez et al., Nat Rev Cancer, 2020; 32778778        2024.09.20</span></span>
<span><span class="co">#&gt; 4          Brown et al., Nucleic Acids Res, 2015; 25355515        2026-02-21</span></span>
<span><span class="co">#&gt; 5                      Bailey et al., Cell, 2018; 29625053              &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 6                Sanchez-Vega et al., Cell, 2018; 29625050              &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 7                                                     &lt;NA&gt;              &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 8                                                     &lt;NA&gt;              &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 9                    Woods et al., Science, 2001; 11181991              v1.1</span></span>
<span><span class="co">#&gt; 10                  Liu et al., Genome Med, 2020; 33261662              v5.3</span></span>
<span><span class="co">#&gt; 11           Caudle et al.,Curr Drug Metab, 2014; 24479687              &lt;NA&gt;</span></span>
<span><span class="co">#&gt;    source_abbreviation                         source_license</span></span>
<span><span class="co">#&gt; 1           cancermine                                CC0 1.0</span></span>
<span><span class="co">#&gt; 2                  ncg                       Free/open access</span></span>
<span><span class="co">#&gt; 3              intogen                                CC0 1.0</span></span>
<span><span class="co">#&gt; 4                 ncbi               NCBI data usage policies</span></span>
<span><span class="co">#&gt; 5           bailey2018                       Free/open access</span></span>
<span><span class="co">#&gt; 6      sanchezvega2018                       Free/open access</span></span>
<span><span class="co">#&gt; 7       foundation_one                       Free/open access</span></span>
<span><span class="co">#&gt; 8             illumina                       Free/open access</span></span>
<span><span class="co">#&gt; 9      woods_dnarepair                       Free/open access</span></span>
<span><span class="co">#&gt; 10              dbnsfp Free for non-commercial, academic use.</span></span>
<span><span class="co">#&gt; 11                cpic                                CC0 1.0</span></span>
<span><span class="co">#&gt;                                        source_license_url</span></span>
<span><span class="co">#&gt; 1      https://creativecommons.org/publicdomain/zero/1.0/</span></span>
<span><span class="co">#&gt; 2                                                    &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 3      https://creativecommons.org/publicdomain/zero/1.0/</span></span>
<span><span class="co">#&gt; 4  https://www.ncbi.nlm.nih.gov/home/about/policies/#data</span></span>
<span><span class="co">#&gt; 5                                                    &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 6                                                    &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 7                                                    &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 8                                                    &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 9                                                    &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 10                                                   &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 11                                                   &lt;NA&gt;</span></span></code></pre></div>
<p><br><br></p>
<div class="section level3">
<h3 id="get-classified-tumor-suppressor-genes">Get classified tumor suppressor genes<a class="anchor" aria-label="anchor" href="#get-classified-tumor-suppressor-genes"></a>
</h3>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## Get tumor suppressor genes - as indicated from either</span></span>
<span><span class="co">## Network of Cancer Genes (NCG)</span></span>
<span><span class="co">## - show literature support from CancerMine</span></span>
<span></span>
<span><span class="va">tsg</span> <span class="op">&lt;-</span> <span class="va">gene_basic</span><span class="op">$</span><span class="va">records</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">ncg_tsg</span> <span class="op">==</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">symbol</span>, <span class="va">entrezgene</span>, <span class="va">name</span>, <span class="va">gene_biotype</span>,</span>
<span>                <span class="va">ncg_tsg</span>, <span class="va">cancermine_cit_links_tsg</span>, </span>
<span>                <span class="va">ncbi_function_summary</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/rename.html" class="external-link">rename</a></span><span class="op">(</span>support_cancermine <span class="op">=</span> <span class="va">cancermine_cit_links_tsg</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Make as datatable</span></span>
<span><span class="va">tsg_table</span> <span class="op">&lt;-</span> <span class="fu">DT</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span></span>
<span>  <span class="va">tsg</span>, </span>
<span>  escape <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  extensions <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Buttons"</span>, <span class="st">"Responsive"</span><span class="op">)</span>, </span>
<span>  width <span class="op">=</span> <span class="st">"100%"</span>,</span>
<span>  options <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>    buttons <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"csv"</span>, <span class="st">"excel"</span><span class="op">)</span>, dom <span class="op">=</span> <span class="st">"Bfrtip"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<p><br></p>
<div id="htmlwidget-ac96cb3ee4656e2e9ec3" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-ac96cb3ee4656e2e9ec3">{"x":{"filter":"none","vertical":false,"extensions":["Buttons","Responsive"],"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315"],["ACVR1B","ACVR2A","APC","BIRC3","FAS","RHOA","RHOH","ARNT","ZFHX3","ATM","ATP1A1","ATP2B3","ATR","ATRX","B2M","BARD1","BAX","PRDM1","BLM","BMPR1A","FOXL2","BRCA1","BRCA2","BTG1","BTK","BUB1B","CARS1","CASP3","CASP8","CASP9","RUNX1","RUNX1T1","CBFA2T3","CBFB","CBL","CBLB","CCNC","CDH1","CDH10","CDH11","CDKN1A","CDKN1B","CDKN2A","CDKN2C","CDX2","CEBPA","CHD2","CLTC","KLF6","CREBBP","CUX1","CYLD","DAXX","DDB2","DDX3X","DDX10","DNM2","DNMT3A","EBF1","ELF3","ELF4","EP300","EPAS1","EPS15","ERBB4","ERCC2","ERCC3","ERCC4","ERCC5","ESR1","ETV6","EXT1","EXT2","EZH2","FANCA","FANCC","FANCD2","FANCE","FANCF","FANCG","FAT1","FBLN2","FEN1","FES","GPC5","FH","FHIT","FOXO1","FOXO3","FUS","GATA1","GATA3","GPC3","GRIN2A","MSH6","NRG1","HOXA9","HOXA11","ID3","EIF3E","IRF4","JAK1","SMAD2","SMAD3","SMAD4","MAX","MAP3K1","MEN1","MGMT","CIITA","MLF1","MLH1","FOXO4","MSH2","MUTYH","MYH9","NAB2","NBN","NF1","NF2","NFE2L2","NFKB2","NFKBIE","CNOT3","NOTCH1","NOTCH2","NTHL1","NTRK1","PAX5","PER1","PIK3R1","PML","PMS2","POLD1","POLE","POLG","PPARG","PPP2R1A","PPP6C","PRF1","PRKAR1A","PTCH1","PTEN","PTK6","PTPN6","PTPN13","PTPRB","PTPRC","PTPRD","PTPRK","RAD17","RAD21","RAD51B","RANBP2","RB1","ROBO2","RPL5","RPL10","RPL22","SDHA","SDHB","SDHC","SDHD","MAP2K4","SFPQ","SFRP4","SMARCA4","SMARCB1","SMARCD1","SMARCE1","SOX9","STAT5B","STK11","TBX3","HNF1A","TCF3","TERT","TGFBR2","NKX2-1","TNFAIP3","TP53","TPM3","TSC1","TSC2","KDM6A","VHL","WRN","WT1","XPA","XPC","YWHAE","CNBP","ZBTB16","PRDM2","CCDC6","NCOA4","KMT2D","ELL","LZTR1","CLTCL1","ZRSR2","RBM10","KDM5C","SMC1A","ARID1A","AXIN1","AXIN2","BAP1","SPOP","CUL3","IRS4","TP63","SOCS1","EED","TNFRSF14","TRIM24","FUBP1","BCL10","PHOX2B","LATS1","MAP3K13","ZMYM3","KLF4","RECQL4","QKI","APOBEC3B","NCOR1","NCOR2","ARHGEF10","KEAP1","MED12","ABI1","SH2B3","STAG1","IKZF1","NDRG1","SLC34A2","IGF2BP2","CTCF","POLQ","STAG2","PTPRT","SOX21","BAZ1A","WIF1","CHEK2","CPEB3","SPEN","SETD1B","ARHGAP26","CIC","LARP4B","CAMTA1","ARHGEF12","DICER1","LEPROTL1","SUZ12","KAT6B","PATZ1","CBLC","POT1","LATS2","PABPC1","SETD2","DROSHA","CD274","SBDS","LEF1","TRIM33","SUFU","CDK12","LRP1B","TET2","TENT5C","BCOR","RNF43","SDHAF2","RFWD3","ARHGEF10L","PBRM1","ASXL2","FBXW7","ZNF331","ETNK1","TMEM127","N4BP2","KNL1","ARID1B","MRTFA","CCNB1IP1","KMT2C","BCORL1","BCL11B","WNK2","CDC73","FAT4","TBL1XR1","PALB2","FBXO11","TET1","BRIP1","USP44","ZNRF3","TRAF7","PHF6","CREB3L1","RMI2","LRIG3","AMER1","SIRPA","ASXL1","ARID2","FLCN","BCL9L","RSPO2"],[91,92,324,330,355,387,399,405,463,472,476,492,545,546,567,580,581,639,641,657,668,672,675,694,695,701,833,836,841,842,861,862,863,865,867,868,892,999,1008,1009,1026,1027,1029,1031,1045,1050,1106,1213,1316,1387,1523,1540,1616,1643,1654,1662,1785,1788,1879,1999,2000,2033,2034,2060,2066,2068,2071,2072,2073,2099,2120,2131,2132,2146,2175,2176,2177,2178,2188,2189,2195,2199,2237,2242,2262,2271,2272,2308,2309,2521,2623,2625,2719,2903,2956,3084,3205,3207,3399,3646,3662,3716,4087,4088,4089,4149,4214,4221,4255,4261,4291,4292,4303,4436,4595,4627,4665,4683,4763,4771,4780,4791,4794,4849,4851,4853,4913,4914,5079,5187,5295,5371,5395,5424,5426,5428,5468,5518,5537,5551,5573,5727,5728,5753,5777,5783,5787,5788,5789,5796,5884,5885,5890,5903,5925,6092,6125,6134,6146,6389,6390,6391,6392,6416,6421,6424,6597,6598,6602,6605,6662,6777,6794,6926,6927,6929,7015,7048,7080,7128,7157,7170,7248,7249,7403,7428,7486,7490,7507,7508,7531,7555,7704,7799,8030,8031,8085,8178,8216,8218,8233,8241,8242,8243,8289,8312,8313,8314,8405,8452,8471,8626,8651,8726,8764,8805,8880,8915,8929,9113,9175,9203,9314,9401,9444,9582,9611,9612,9639,9817,9968,10006,10019,10274,10320,10397,10568,10644,10664,10721,10735,11122,11166,11177,11197,11200,22849,23013,23067,23092,23152,23185,23261,23365,23405,23484,23512,23522,23598,23624,25913,26524,26986,29072,29102,29126,51119,51176,51592,51684,51755,53353,54790,54855,54880,54894,54949,55159,55160,55193,55252,55294,55422,55500,55654,55728,57082,57492,57591,57820,58508,63035,64919,65268,79577,79633,79718,79728,80204,80312,83990,84101,84133,84231,84295,90993,116028,121227,139285,140885,171023,196528,201163,283149,340419],["activin A receptor type 1B","activin A receptor type 2A","APC regulator of Wnt signaling pathway","baculoviral IAP repeat containing 3","Fas cell surface death receptor","ras homolog family member A","ras homolog family member H","aryl hydrocarbon receptor nuclear translocator","zinc finger homeobox 3","ATM serine/threonine kinase","ATPase Na+/K+ transporting subunit alpha 1","ATPase plasma membrane Ca2+ transporting 3","ATR checkpoint kinase","ATRX chromatin remodeler","beta-2-microglobulin","BRCA1 associated RING domain 1","BCL2 associated X, apoptosis regulator","PR/SET domain 1","BLM RecQ like helicase","bone morphogenetic protein receptor type 1A","forkhead box L2","BRCA1 DNA repair associated","BRCA2 DNA repair associated","BTG anti-proliferation factor 1","Bruton tyrosine kinase","BUB1 mitotic checkpoint serine/threonine kinase B","cysteinyl-tRNA synthetase 1","caspase 3","caspase 8","caspase 9","RUNX family transcription factor 1","RUNX1 partner transcriptional co-repressor 1","CBFA2/RUNX1 partner transcriptional co-repressor 3","core-binding factor subunit beta","Cbl proto-oncogene","Cbl proto-oncogene B","cyclin C","cadherin 1","cadherin 10","cadherin 11","cyclin dependent kinase inhibitor 1A","cyclin dependent kinase inhibitor 1B","cyclin dependent kinase inhibitor 2A","cyclin dependent kinase inhibitor 2C","caudal type homeobox 2","CCAAT enhancer binding protein alpha","chromodomain helicase DNA binding protein 2","clathrin heavy chain","KLF transcription factor 6","CREB binding lysine acetyltransferase","cut like homeobox 1","CYLD lysine 63 deubiquitinase","death domain associated protein","damage specific DNA binding protein 2","DEAD-box helicase 3 X-linked","DEAD-box helicase 10","dynamin 2","DNA methyltransferase 3 alpha","EBF transcription factor 1","E74 like ETS transcription factor 3","E74 like ETS transcription factor 4","EP300 lysine acetyltransferase","endothelial PAS domain protein 1","epidermal growth factor receptor pathway substrate 15","erb-b2 receptor tyrosine kinase 4","ERCC excision repair 2, TFIIH core complex helicase subunit","ERCC excision repair 3, TFIIH core complex helicase subunit","ERCC excision repair 4, endonuclease catalytic subunit","ERCC excision repair 5, endonuclease","estrogen receptor 1","ETS variant transcription factor 6","exostosin glycosyltransferase 1","exostosin glycosyltransferase 2","enhancer of zeste 2 polycomb repressive complex 2 subunit","FA complementation group A","FA complementation group C","FA complementation group D2","FA complementation group E","FA complementation group F","FA complementation group G","FAT atypical cadherin 1","fibulin 2","flap structure-specific endonuclease 1","FES proto-oncogene, tyrosine kinase","glypican 5","fumarate hydratase","fragile histidine triad diadenosine triphosphatase","forkhead box O1","forkhead box O3","FUS RNA binding protein","GATA binding protein 1","GATA binding protein 3","glypican 3","glutamate ionotropic receptor NMDA type subunit 2A","mutS homolog 6","neuregulin 1","homeobox A9","homeobox A11","inhibitor of DNA binding 3","eukaryotic translation initiation factor 3 subunit E","interferon regulatory factor 4","Janus kinase 1","SMAD family member 2","SMAD family member 3","SMAD family member 4","MYC associated transcriptional regulator X","mitogen-activated protein kinase kinase kinase 1","menin 1","O-6-methylguanine-DNA methyltransferase","class II major histocompatibility complex transactivator","myeloid leukemia factor 1","mutL homolog 1","forkhead box O4","mutS homolog 2","mutY DNA glycosylase","myosin heavy chain 9","NGFI-A binding protein 2","nibrin","neurofibromin 1","NF2, moesin-ezrin-radixin like (MERLIN) tumor suppressor","NFE2 like bZIP transcription factor 2","nuclear factor kappa B subunit 2","NFKB inhibitor epsilon","CCR4-NOT transcription complex subunit 3","notch receptor 1","notch receptor 2","nth like DNA glycosylase 1","neurotrophic receptor tyrosine kinase 1","paired box 5","period circadian regulator 1","phosphoinositide-3-kinase regulatory subunit 1","PML nuclear body scaffold","PMS1 homolog 2, mismatch repair system component","DNA polymerase delta 1, catalytic subunit","DNA polymerase epsilon, catalytic subunit","DNA polymerase gamma, catalytic subunit","peroxisome proliferator activated receptor gamma","protein phosphatase 2 scaffold subunit Aalpha","protein phosphatase 6 catalytic subunit","perforin 1","protein kinase cAMP-dependent type I regulatory subunit alpha","patched 1","phosphatase and tensin homolog","protein tyrosine kinase 6","protein tyrosine phosphatase non-receptor type 6","protein tyrosine phosphatase non-receptor type 13","protein tyrosine phosphatase receptor type B","protein tyrosine phosphatase receptor type C","protein tyrosine phosphatase receptor type D","protein tyrosine phosphatase receptor type K","RAD17 checkpoint clamp loader component","RAD21 cohesin complex component","RAD51 paralog B","RAN binding protein 2","RB transcriptional corepressor 1","roundabout guidance receptor 2","ribosomal protein L5","ribosomal protein L10","ribosomal protein L22","succinate dehydrogenase complex flavoprotein subunit A","succinate dehydrogenase complex iron sulfur subunit B","succinate dehydrogenase complex subunit C","succinate dehydrogenase complex subunit D","mitogen-activated protein kinase kinase 4","splicing factor proline and glutamine rich","secreted frizzled related protein 4","SWI/SNF related BAF chromatin remodeling complex subunit ATPase 4","SWI/SNF related BAF chromatin remodeling complex subunit B1","SWI/SNF related BAF chromatin remodeling complex subunit D1","SWI/SNF related BAF chromatin remodeling complex subunit E1","SRY-box transcription factor 9","signal transducer and activator of transcription 5B","serine/threonine kinase 11","T-box transcription factor 3","HNF1 homeobox A","transcription factor 3","telomerase reverse transcriptase","transforming growth factor beta receptor 2","NK2 homeobox 1","TNF alpha induced protein 3","tumor protein p53","tropomyosin 3","TSC complex subunit 1","TSC complex subunit 2","lysine demethylase 6A","von Hippel-Lindau tumor suppressor","WRN RecQ like helicase","WT1 transcription factor","XPA, DNA damage recognition and repair factor","XPC complex subunit, DNA damage recognition and repair factor","tyrosine 3-monooxygenase/tryptophan 5-monooxygenase activation protein epsilon","CCHC-type zinc finger nucleic acid binding protein","zinc finger and BTB domain containing 16","PR/SET domain 2","coiled-coil domain containing 6","nuclear receptor coactivator 4","lysine methyltransferase 2D","elongation factor for RNA polymerase II","leucine zipper like post translational regulator 1","clathrin heavy chain like 1","zinc finger CCCH-type, RNA binding motif and serine/arginine rich 2","RNA binding motif protein 10","lysine demethylase 5C","structural maintenance of chromosomes 1A","AT-rich interaction domain 1A","axin 1","axin 2","BRCA1 associated deubiquitinase 1","speckle type BTB/POZ protein","cullin 3","insulin receptor substrate 4","tumor protein p63","suppressor of cytokine signaling 1","embryonic ectoderm development","TNF receptor superfamily member 14","tripartite motif containing 24","far upstream element binding protein 1","BCL10 immune signaling adaptor","paired like homeobox 2B","large tumor suppressor kinase 1","mitogen-activated protein kinase kinase kinase 13","zinc finger MYM-type containing 3","KLF transcription factor 4","RecQ like helicase 4","QKI, KH domain containing RNA binding","apolipoprotein B mRNA editing enzyme catalytic subunit 3B","nuclear receptor corepressor 1","nuclear receptor corepressor 2","Rho guanine nucleotide exchange factor 10","kelch like ECH associated protein 1","mediator complex subunit 12","abl interactor 1","SH2B adaptor protein 3","STAG1 cohesin complex component","IKAROS family zinc finger 1","N-myc downstream regulated 1","solute carrier family 34 member 2","insulin like growth factor 2 mRNA binding protein 2","CCCTC-binding factor","DNA polymerase theta","STAG2 cohesin complex component","protein tyrosine phosphatase receptor type T","SRY-box transcription factor 21","bromodomain adjacent to zinc finger domain 1A","Wnt inhibitory factor 1","checkpoint kinase 2","cytoplasmic polyadenylation element binding protein 3","spen family transcriptional repressor","SET domain containing 1B, histone lysine methyltransferase","Rho GTPase activating protein 26","capicua transcriptional repressor","La ribonucleoprotein 4B","calmodulin binding transcription activator 1","Rho guanine nucleotide exchange factor 12","dicer 1, ribonuclease III","leptin receptor overlapping transcript like 1","SUZ12 polycomb repressive complex 2 subunit","lysine acetyltransferase 6B","POZ/BTB and AT hook containing zinc finger 1","Cbl proto-oncogene C","protection of telomeres 1","large tumor suppressor kinase 2","poly(A) binding protein cytoplasmic 1","SET domain containing 2, histone lysine methyltransferase","drosha ribonuclease III","CD274 molecule","SBDS ribosome maturation factor","lymphoid enhancer binding factor 1","tripartite motif containing 33","SUFU negative regulator of hedgehog signaling","cyclin dependent kinase 12","LDL receptor related protein 1B","tet methylcytosine dioxygenase 2","terminal nucleotidyltransferase 5C","BCL6 corepressor","ring finger protein 43","succinate dehydrogenase complex assembly factor 2","ring finger and WD repeat domain 3","Rho guanine nucleotide exchange factor 10 like","polybromo 1","ASXL transcriptional regulator 2","F-box and WD repeat domain containing 7","zinc finger protein 331","ethanolamine kinase 1","transmembrane protein 127","NEDD4 binding protein 2","kinetochore scaffold 1","AT-rich interaction domain 1B","myocardin related transcription factor A","cyclin B1 interacting protein 1","lysine methyltransferase 2C","BCL6 corepressor like 1","BCL11 transcription factor B","WNK lysine deficient protein kinase 2","cell division cycle 73","FAT atypical cadherin 4","TBL1X/Y related 1","partner and localizer of BRCA2","F-box protein 11","tet methylcytosine dioxygenase 1","BRCA1 interacting DNA helicase 1","ubiquitin specific peptidase 44","zinc and ring finger 3","TNF receptor associated factor 7","PHD finger protein 6","cAMP responsive element binding protein 3 like 1","RecQ mediated genome instability 2","leucine rich repeats and immunoglobulin like domains 3","APC membrane recruitment protein 1","signal regulatory protein alpha","ASXL transcriptional regulator 1","AT-rich interaction domain 2","folliculin","BCL9 like","R-spondin 2"],["protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding"],[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true],["<a href='https://www.ncbi.nlm.nih.gov/pubmed/33805113' target='_blank'>Nidhi et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31480737' target='_blank'>Weng et al., J Clin Med, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34553034' target='_blank'>Li et al., Mol Ther Oncolytics, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33805113' target='_blank'>Nidhi et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15520171' target='_blank'>Deacu et al., Cancer Res, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12615714' target='_blank'>Hempen et al., Cancer Res, 2003<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34079124' target='_blank'>Flanagan et al., 2021, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32532965' target='_blank'>Chen et al., Signal Transduct Target Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32376911' target='_blank'>Klose et al., Cell Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32163279' target='_blank'>Theodoropoulos et al., J Am Chem Soc, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31337618' target='_blank'>Schatoff et al., Cancer Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28148261' target='_blank'>Chandrakesan et al., Mol Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27609830' target='_blank'>Sievers et al., Gut, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25895683' target='_blank'>Moyano et al., J Hematol Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24012984' target='_blank'>Revill et al., Gastroenterology, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23245941' target='_blank'>Rosenbluh et al., Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22155636' target='_blank'>White et al., Gastroenterology, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22009203' target='_blank'>Lao et al., Nat Rev Gastroenterol Hepatol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22005480' target='_blank'>Chen et al., Gastroenterology, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17666112' target='_blank'>Bazigou et al., Genome Biol, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9727977' target='_blank'>He et al., Science, 1998<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/28976967' target='_blank'>Gorbea et al., Nature, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24316673' target='_blank'>Gump et al., Nat Cell Biol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16489062' target='_blank'>Yeh et al., Clin Cancer Res, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11120779' target='_blank'>Müschen et al., J Exp Med, 2000<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33680947' target='_blank'>Yang et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32244355' target='_blank'>Hung et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25413277' target='_blank'>Rodrigues et al., Nat Commun, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23233736' target='_blank'>Lin et al., Cancer Res, 2013<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40596315' target='_blank'>Fu et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38871709' target='_blank'>Hu et al., 2024, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37349324' target='_blank'>Marin et al., 2023, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35347810' target='_blank'>Mangolini et al., Cell Biol Int, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34013380' target='_blank'>Li et al., Oncol Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33217982' target='_blank'>Dong et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30979864' target='_blank'>Hu et al., Oncogenesis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27167109' target='_blank'>Van et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21729327' target='_blank'>Jung et al., Mol Neurodegener, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21367855' target='_blank'>Dong et al., J Biol Chem, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20720010' target='_blank'>Dong et al., J Biol Chem, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20599712' target='_blank'>Mabuchi et al., Biochem Biophys Res Commun, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19035517' target='_blank'>Lange et al., Prostate, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16932943' target='_blank'>Sun et al., J Cancer Res Clin Oncol, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16637072' target='_blank'>Xu et al., Prostate, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38848368' target='_blank'>Lundgren et al., 2024, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36555667' target='_blank'>Pastorino et al., Int J Mol Sci, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32127357' target='_blank'>Rafiei et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31684144' target='_blank'>Polak et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28881607' target='_blank'>Shindo et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28849346' target='_blank'>Dyer et al., Breast Cancer Res Treat, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27922010' target='_blank'>Smida et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27599572' target='_blank'>Wang et al., Sci Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26676339' target='_blank'>Blackburn et al., Mol Carcinog, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26113450' target='_blank'>Liu et al., J Cancer Res Clin Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24486587' target='_blank'>Kim et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18261794' target='_blank'>Gumy-Pause et al., 2008, Leuk Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9699683' target='_blank'>Uhrhammer et al., Cancer Res, 1998<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9573030' target='_blank'>Stoppa-Lyonnet et al., Blood, 1998<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9488043' target='_blank'>Yuille et al., Oncogene, 1998<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/33409155' target='_blank'>Hou et al., 2021, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32060267' target='_blank'>Zeineldin et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30970016' target='_blank'>Oppel et al., PLoS Genet, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29212165' target='_blank'>Gleeson et al., Oncotarget, 2017<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/32722046' target='_blank'>Otsuka et al., Genes (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32708251' target='_blank'>Watters et al., Genes (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32047556' target='_blank'>Cimmino et al., J Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30760980' target='_blank'>Johnsen et al., Front Mol Neurosci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24602449' target='_blank'>Aktas et al., Mol Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23274701' target='_blank'>Domingo-Fernandez et al., Pediatr Surg Int, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15218362' target='_blank'>Reinholz et al., Breast Cancer Res Treat, 2004<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40421201' target='_blank'>Domínguez-López et al., 2025, Cell Stress<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39152150' target='_blank'>Nazari et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38053661' target='_blank'>Ding et al., 2023, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35452995' target='_blank'>Chen et al., Transl Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24350829' target='_blank'>Nishioka et al., Cancer Sci, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39367275' target='_blank'>Pudjihartono et al., 2024, Br J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39024554' target='_blank'>Lacunza et al., 2024, JCI Insight<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32732362' target='_blank'>Dong et al., Haematologica, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31337866' target='_blank'>Raimondi et al., Oncogene, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31130394' target='_blank'>Tracey et al., Trends Genet, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26530011' target='_blank'>Ma et al., Leukemia, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25382611' target='_blank'>Zhang et al., Blood Cancer J, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24434212' target='_blank'>Lohr et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22610119' target='_blank'>Barbieri et al., Nat Genet, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21722313' target='_blank'>Cubedo et al., FEBS J, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21156282' target='_blank'>Calado et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21156281' target='_blank'>Mandelbaum et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19329782' target='_blank'>D'Costa et al., Blood, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16492805' target='_blank'>Pasqualucci et al., J Exp Med, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16424392' target='_blank'>Tam et al., Blood, 2006<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/30360388' target='_blank'>Bach et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20098707' target='_blank'>Benayoun et al., PLoS One, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38072908' target='_blank'>Wang et al., 2023, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33833118' target='_blank'>Fugger et al., Science, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22172717' target='_blank'>Patel et al., 2011, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21901007' target='_blank'>Zhu et al., Nature, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21863050' target='_blank'>Lipkowitz et al., Nat Rev Cancer, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21397863' target='_blank'>Finley et al., 2011, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20847263' target='_blank'>Vander et al., Science, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18851829' target='_blank'>Wang et al., Mol Cell, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18186929' target='_blank'>Maxwell et al., Mol Cancer, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17525341' target='_blank'>Sobhian et al., Science, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16762066' target='_blank'>Debauve et al., Mol Cancer, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16116424' target='_blank'>Levran et al., Nat Genet, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14576434' target='_blank'>King et al., Science, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11431698' target='_blank'>Xu et al., Nat Genet, 2001<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10198641' target='_blank'>Xu et al., Mol Cell, 1999<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36243983' target='_blank'>Neizer-Ashun et al., Nucleic Acids Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33833118' target='_blank'>Fugger et al., Science, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29133916' target='_blank'>Han et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28904335' target='_blank'>Feng et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26883357' target='_blank'>Ying et al., Genes Dev, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26045257' target='_blank'>Sollier et al., Trends Cell Biol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23658460' target='_blank'>Lucas et al., Clin Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21576230' target='_blank'>Holthausen et al., Nucleic Acids Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20729858' target='_blank'>Thorslund et al., Nat Struct Mol Biol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19595720' target='_blank'>Antony et al., Mol Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18275380' target='_blank'>San et al., Annu Rev Biochem, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18264087' target='_blank'>Sakai et al., Nature, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16935872' target='_blank'>Heyer et al., Nucleic Acids Res, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15990877' target='_blank'>Thakurta et al., EMBO J, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14576434' target='_blank'>King et al., Science, 2003<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34389030' target='_blank'>Long et al., Infect Agent Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31452761' target='_blank'>Su et al., Oncol Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30916818' target='_blank'>Qian et al., J Cell Biochem, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30854058' target='_blank'>Yan et al., Oncol Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30350856' target='_blank'>Yuniati et al., J Cell Physiol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27979924' target='_blank'>Scheijen et al., Haematologica, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26657730' target='_blank'>Yuniati et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25117713' target='_blank'>Li et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21606506' target='_blank'>Heltemes-Harris et al., J Exp Med, 2011<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/30778371' target='_blank'>Zhu et al., Front Genet, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30177244' target='_blank'>Xu et al., EBioMedicine, 2018<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/30703581' target='_blank'>Al-Moujahed et al., Stem Cell Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27600067' target='_blank'>Song et al., Microarrays (Basel), 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26560147' target='_blank'>Clemente-Vicario et al., PLoS One, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39351539' target='_blank'>Song et al., 2024, Front Endocrinol (Lausanne)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38409266' target='_blank'>Chen et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30703581' target='_blank'>Al-Moujahed et al., Stem Cell Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27927008' target='_blank'>Lee et al., Mol Cells, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23124518' target='_blank'>Kim et al., Cell Biol Int Rep (2010), 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10802708' target='_blank'>Teitz et al., Nat Med, 2000<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36854759' target='_blank'>Guo et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33158995' target='_blank'>Khan et al., Mol Cancer Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32591643' target='_blank'>Loontiens et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31032403' target='_blank'>Hall et al., Sci Adv, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28412963' target='_blank'>Knutson et al., J Hematol Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27566955' target='_blank'>Voora et al., EBioMedicine, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26073082' target='_blank'>Li et al., Oncogene, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25884481' target='_blank'>Matouk et al., J Transl Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24823667' target='_blank'>Helman et al., Genome Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23979164' target='_blank'>Goyama et al., J Clin Invest, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23086478' target='_blank'>Giambra et al., 2012, Nat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22936064' target='_blank'>Patel et al., Mod Pathol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22751462' target='_blank'>Dulak et al., Cancer Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22366949' target='_blank'>Della et al., Nat Med, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21949550' target='_blank'>Cytron et al., 2011, Clin Epigenetics<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38555384' target='_blank'>Zhang et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34164477' target='_blank'>Chen et al., Ann Transl Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30678048' target='_blank'>Rossetti et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29358956' target='_blank'>Steinauer et al., Stem Cells Int, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27270437' target='_blank'>Parang et al., Oncogene, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21303973' target='_blank'>Barrett et al., Cancer Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16966434' target='_blank'>Kumar et al., Mol Cancer Res, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12183414' target='_blank'>Kochetkova et al., Cancer Res, 2002<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37454130' target='_blank'>Vellichirammal et al., 2023, Hum Genomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33945523' target='_blank'>Malik et al., PLoS Genet, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31061501' target='_blank'>Malik et al., Nat Commun, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40581645' target='_blank'>Tu et al., 2025, Cell Div<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37355734' target='_blank'>Diamond et al., 2023, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29104623' target='_blank'>Yu et al., Exp Ther Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27900322' target='_blank'>Clayton et al., Front Mol Biosci, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25980579' target='_blank'>Wei et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21471695' target='_blank'>Ogawa et al., Rinsho Ketsueki, 2011<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40358161' target='_blank'>Bauer et al., 2025, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26452386' target='_blank'>Broude et al., Curr Cancer Drug Targets, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26182352' target='_blank'>Clark et al., Crit Rev Biochem Mol Biol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25914884' target='_blank'>Aleem et al., Front Cell Dev Biol, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40500450' target='_blank'>Raymond et al., 2025, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40140925' target='_blank'>Lin et al., 2025, Biomark Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37258796' target='_blank'>McCarthy et al., 2023, Br J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35390277' target='_blank'>Liang et al., 2022, Mol Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33975882' target='_blank'>Hapach et al., 2021, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31827074' target='_blank'>Kim et al., Exp Mol Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31767037' target='_blank'>Carneiro et al., Cell Commun Signal, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30181244' target='_blank'>Jia et al., Cancer Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28168295' target='_blank'>Zhang et al., Nucleic Acids Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26933086' target='_blank'>Boulay et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26227500' target='_blank'>Wang et al., Med Res Rev, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22009736' target='_blank'>Kim et al., Cancer, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19671196' target='_blank'>Yamada et al., 2009, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12434297' target='_blank'>French et al., 2002, Br J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11381089' target='_blank'>Gottardi et al., J Cell Biol, 2001<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34439379' target='_blank'>Tsai et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32292512' target='_blank'>Casciello et al., Theranostics, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34295163' target='_blank'>Zheng et al., Onco Targets Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31114321' target='_blank'>Yuan et al., Cancer Manag Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27397103' target='_blank'>Piao et al., J Oral Pathol Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24433356' target='_blank'>Thériault et al., Clin Exp Ophthalmol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22139084' target='_blank'>Li et al., Oncogene, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21134280' target='_blank'>Dozmorov et al., BMC Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20421947' target='_blank'>Marchong et al., PLoS Genet, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38502859' target='_blank'>Kurlekar et al., 2024, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37283464' target='_blank'>Hu et al., 2023, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32754284' target='_blank'>Li et al., Theranostics, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32508030' target='_blank'>Shi et al., Clin Transl Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30857539' target='_blank'>Jin et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29793547' target='_blank'>Yin et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28366734' target='_blank'>Kim et al., Gastroenterology, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26590805' target='_blank'>Jagadish et al., J Exp Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26019213' target='_blank'>Wang et al., Gut, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25789975' target='_blank'>Hall et al., Cell Death Dis, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24777035' target='_blank'>Cazier et al., Nat Commun, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23159372' target='_blank'>Shankar et al., Pharmacol Ther, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19584294' target='_blank'>Shin et al., Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18413741' target='_blank'>Majid et al., Cancer Res, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18339845' target='_blank'>Xia et al., 2008, Cancer Res<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40598282' target='_blank'>Ramar et al., 2025, Cell Commun Signal<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40271196' target='_blank'>Yu et al., 2025, Genes Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37454155' target='_blank'>Pei et al., 2023, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34168918' target='_blank'>Li et al., 2021, Mol Ther Nucleic Acids<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34062779' target='_blank'>Mandal et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32899752' target='_blank'>Shin et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32086342' target='_blank'>Kohlmeyer et al., Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27669693' target='_blank'>Goto et al., Cell Mol Life Sci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25143434' target='_blank'>Zhu et al., Mol Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24252239' target='_blank'>Hatton et al., Biomark Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21422114' target='_blank'>Haferlach et al., Haematologica, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18980972' target='_blank'>Ougolkov et al., Clin Cancer Res, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15026335' target='_blank'>Chang et al., Cancer Res, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8621784' target='_blank'>Chen et al., J Clin Invest, 1996<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7882309' target='_blank'>Pietenpol et al., Cancer Res, 1995<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40628732' target='_blank'>Singh et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39753721' target='_blank'>Ganguli et al., 2025, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37598158' target='_blank'>An et al., 2023, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36596785' target='_blank'>Li et al., Signal Transduct Target Ther, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29990501' target='_blank'>Zeng et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29455666' target='_blank'>Román et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25485910' target='_blank'>Yoda et al., Nat Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20687958' target='_blank'>Xu et al., Mol Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19878870' target='_blank'>Gidekel et al., Cancer Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19149898' target='_blank'>Becker et al., Mol Cancer, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18394558' target='_blank'>Wiedemeyer et al., Cancer Cell, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17626628' target='_blank'>Rousseau et al., Mol Cancer, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17407586' target='_blank'>Boominathan et al., Mol Cancer, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12605716' target='_blank'>Schneider et al., Mol Cancer, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9462511' target='_blank'>Lissy et al., Immunity, 1998<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34298972' target='_blank'>Marini et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34065376' target='_blank'>Richter et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31645479' target='_blank'>Zhang et al., Aging (Albany NY), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30159431' target='_blank'>DiStefano et al., 2018, Noncoding RNA Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29050269' target='_blank'>Dou et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26328522' target='_blank'>Xiong et al., Mol Med Rep, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23476020' target='_blank'>van et al., Nucleic Acids Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19147556' target='_blank'>Saab et al., Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18942719' target='_blank'>van et al., Int J Cancer, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18381405' target='_blank'>Solomon et al., Cancer Res, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11263500' target='_blank'>Leuraud et al., J Neurooncol, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10994011' target='_blank'>He et al., Neurology, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10736068' target='_blank'>Santarius et al., Neuropathol Appl Neurobiol, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10515227' target='_blank'>Husemann et al., J Neuropathol Exp Neurol, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8665486' target='_blank'>Diccianni et al., Cancer Lett, 1996<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36461092' target='_blank'>Yang et al., Clin Epigenetics, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34367978' target='_blank'>Ding et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32541670' target='_blank'>Vu et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32412051' target='_blank'>Deng et al., 2020, Biosci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30867652' target='_blank'>Sun et al., Cancer Cell Int, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28819729' target='_blank'>Reggiani et al., Virchows Arch, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28627695' target='_blank'>Zheng et al., Int J Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23112155' target='_blank'>Salari et al., Proc Natl Acad Sci U S A, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22986531' target='_blank'>Saandi et al., Oncogene, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22892849' target='_blank'>Liu et al., Cancer Biol Ther, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19686845' target='_blank'>Krueger et al., Cell Signal, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19325799' target='_blank'>Chen et al., Int J Mol Sci, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17158164' target='_blank'>Calon et al., 2006, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16314840' target='_blank'>Dang et al., Oncogene, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15706431' target='_blank'>Kawai et al., Oncol Rep, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39390018' target='_blank'>Wang et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39276936' target='_blank'>Zhao et al., 2024, J Biol Chem<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38696452' target='_blank'>Bi et al., 2024, PLoS One<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38442712' target='_blank'>Chen et al., 2024, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34482361' target='_blank'>Xie et al., Signal Transduct Target Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33780265' target='_blank'>Zhang et al., Neoplasma, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31053176' target='_blank'>Weigel et al., Clin Epigenetics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30386458' target='_blank'>Krivtsova et al., World J Hepatol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29416795' target='_blank'>Pain et al., Oncotarget, 2018<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/28366734' target='_blank'>Kim et al., Gastroenterology, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22172722' target='_blank'>Amin et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21563203' target='_blank'>Tarocchi et al., Hepatology, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20660366' target='_blank'>Huh et al., J Natl Cancer Inst, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19737957' target='_blank'>Hu et al., Clin Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19088016' target='_blank'>Garcia-Closas et al., Clin Cancer Res, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18471523' target='_blank'>Yea et al., Gastroenterology, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17196295' target='_blank'>Kremer-Tal et al., J Hepatol, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16778100' target='_blank'>DiFeo et al., Clin Cancer Res, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15735005' target='_blank'>Narla et al., Cancer Res, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15172998' target='_blank'>Benzeno et al., Cancer Res, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15172991' target='_blank'>Ito et al., Cancer Res, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15057748' target='_blank'>Reeves et al., 2004, Gastroenterology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12620113' target='_blank'>Kaczynski et al., Genome Biol, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11752579' target='_blank'>Narla et al., Science, 2001<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39625049' target='_blank'>Pozonec et al., 2024, Curr Opin Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36509828' target='_blank'>Kirk et al., Exp Mol Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35411116' target='_blank'>Kim et al., 2022, Nat Biotechnol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35171684' target='_blank'>Kim et al., Sci Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33923680' target='_blank'>Dunphy et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33531375' target='_blank'>Schaff et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33220460' target='_blank'>Khan et al., 2020, Semin Cancer Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31974616' target='_blank'>Wang et al., Mol Med Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31827074' target='_blank'>Kim et al., Exp Mol Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31108964' target='_blank'>Schulze et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30274972' target='_blank'>Dominguez et al., Cancer Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30181244' target='_blank'>Jia et al., Cancer Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26760213' target='_blank'>Morrissy et al., Nature, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25423036' target='_blank'>Vyazunova et al., PLoS One, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22941188' target='_blank'>Peifer et al., Nat Genet, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40565225' target='_blank'>Ballesta-Alcaraz et al., 2025, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38849771' target='_blank'>Yue et al., 2024, BMC Nephrol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38660246' target='_blank'>Lin et al., 2024, Heliyon<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33931647' target='_blank'>Supper et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32547943' target='_blank'>Liu et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23212519' target='_blank'>McNerney et al., Blood, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40012003' target='_blank'>Zhang et al., 2025, Exp Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39739815' target='_blank'>Zheng et al., 2025, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36581942' target='_blank'>Wang et al., Mol Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33165832' target='_blank'>Park et al., Arch Pharm Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32461623' target='_blank'>Williams et al., Mod Pathol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32201366' target='_blank'>Bonacci et al., Semin Cancer Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31591386' target='_blank'>de et al., Oncogenesis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30374464' target='_blank'>Dai et al., Commun Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29880840' target='_blank'>Lei et al., Cell Death Dis, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28203236' target='_blank'>Schmid et al., 2017, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25133483' target='_blank'>Seccareccia et al., 2014, Oncogenesis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24974848' target='_blank'>Petrini et al., Nat Genet, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23300340' target='_blank'>Yang et al., 2013, Sci Signal<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22057290' target='_blank'>Ahmed et al., Nat Immunol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21868762' target='_blank'>McAllister-Lucas et al., 2011, Clin Cancer Res<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34359912' target='_blank'>Kim et al., Cells, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32203224' target='_blank'>Wu et al., Br J Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29212165' target='_blank'>Gleeson et al., Oncotarget, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/28212551' target='_blank'>Chen et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26130719' target='_blank'>Zhao et al., Nucleic Acids Res, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40216969' target='_blank'>Cao et al., 2025, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37293175' target='_blank'>Su et al., 2023, Am J Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35844798' target='_blank'>Shen et al., Int J Biol Sci, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35723050' target='_blank'>Wang et al., Bioengineered, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35194205' target='_blank'>Andrisani et al., 2022, Nat Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35121978' target='_blank'>Alkallas et al., Nat Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32553121' target='_blank'>Patmore et al., Dev Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32121322' target='_blank'>Kiruthiga et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28402257' target='_blank'>Wang et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27344963' target='_blank'>Li et al., Sci Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27007150' target='_blank'>Wu et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25382417' target='_blank'>Ojha et al., Br J Haematol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23108300' target='_blank'>Bühler et al., Dig Dis, 2012<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40171845' target='_blank'>Tang et al., 2025, Cancer Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39807333' target='_blank'>Zhang et al., 2025, Acta Pharm Sin B<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38951697' target='_blank'>Zhao et al., 2024, Br J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35449309' target='_blank'>Na et al., 2022, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34861140' target='_blank'>Xiao et al., Mol Ther, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34751409' target='_blank'>Frazzi et al., Oncol Rep, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33760811' target='_blank'>Carico et al., PLoS Genet, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32751889' target='_blank'>Zhang et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31843451' target='_blank'>Xiao et al., Mol Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31394707' target='_blank'>Lam et al., Cancer Genomics Proteomics, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31024853' target='_blank'>Pan et al., Front Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27690235' target='_blank'>Haney et al., PLoS Genet, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27677595' target='_blank'>Haney et al., Sci Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27134162' target='_blank'>Haney et al., Cell Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25130491' target='_blank'>Challen et al., Cell Stem Cell, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39236081' target='_blank'>Kafle et al., 2024, PLoS Negl Trop Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36397644' target='_blank'>Cheng et al., 2022, Kaohsiung J Med Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34484175' target='_blank'>Dawes et al., Front Immunol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33854627' target='_blank'>Armartmuntree et al., J Cancer, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34562599' target='_blank'>Na et al., Biochim Biophys Acta Mol Basis Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33293429' target='_blank'>Suzuki et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30015943' target='_blank'>Lou et al., Oncol Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27217782' target='_blank'>Zhang et al., Onco Targets Ther, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37674363' target='_blank'>Xu et al., 2023, Acta Biochim Biophys Sin (Shanghai)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21350194' target='_blank'>Sivina et al., Arterioscler Thromb Vasc Biol, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39595195' target='_blank'>Spallotta et al., 2024, Biomedicines<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35171684' target='_blank'>Kim et al., Sci Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34452526' target='_blank'>Dacus et al., Viruses, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34272401' target='_blank'>Chang et al., NPJ Genom Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33718432' target='_blank'>Tu et al., Front Mol Biosci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33220460' target='_blank'>Khan et al., 2020, Semin Cancer Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30130555' target='_blank'>Zhu et al., Toxicol Appl Pharmacol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29879950' target='_blank'>Hou et al., BMC Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29127303' target='_blank'>Du et al., Sci Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28569218' target='_blank'>Melloni et al., Breast Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28339163' target='_blank'>Scott et al., Cancer Cytopathol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27231488' target='_blank'>Wapenaar et al., Clin Epigenetics, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26036259' target='_blank'>Chen et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25805588' target='_blank'>Jiang et al., Semin Hematol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21577103' target='_blank'>Shaknovich et al., Curr Opin Hematol, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/27491085' target='_blank'>Noonan et al., Dis Model Mech, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26955619' target='_blank'>Deynoux et al., Front Oncol, 2016<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40280416' target='_blank'>Chen et al., 2025, J Biol Chem<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34987087' target='_blank'>Lucas et al., 2022, Pharmacol Rev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33446784' target='_blank'>Jung et al., Sci Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29342193' target='_blank'>Jones et al., PLoS One, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24438167' target='_blank'>Zhang et al., J Exp Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22393464' target='_blank'>Mill et al., Genes Cancer, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20404484' target='_blank'>Rudloff et al., Cell Cycle, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32377720' target='_blank'>Jian et al., Int J Mol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31340155' target='_blank'>Magen et al., Cell Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30627419' target='_blank'>Xiao et al., Cell Biosci, 2019<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40140925' target='_blank'>Lin et al., 2025, Biomark Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34322374' target='_blank'>Shen et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34050704' target='_blank'>Meng et al., Cancer Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33865377' target='_blank'>Li et al., Cancer Cell Int, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33778495' target='_blank'>Thomas et al., NAR Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32466334' target='_blank'>Nair et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32308626' target='_blank'>Huang et al., Front Pharmacol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32218824' target='_blank'>Chen et al., Oncol Lett, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31410310' target='_blank'>Liu et al., PeerJ, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30655760' target='_blank'>Zhong et al., Oncol Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29321958' target='_blank'>Yang et al., FEBS Open Bio, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26294040' target='_blank'>Weng et al., Mol Carcinog, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24939822' target='_blank'>Dai et al., Exp Biol Med (Maywood), 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23695389' target='_blank'>Hishida et al., Int J Oncol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18949413' target='_blank'>Suga et al., Oncol Rep, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36233173' target='_blank'>Nagel et al., 2022, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32326970' target='_blank'>Guo et al., J Exp Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31182087' target='_blank'>Lal et al., BMC Med Genomics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31048683' target='_blank'>Cocciardi et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27746161' target='_blank'>Tsai et al., 2016, Cancer Lett<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25807284' target='_blank'>Noetzli et al., Nat Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25435111' target='_blank'>Bhojwani et al., Pediatr Clin North Am, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9681833' target='_blank'>Maier et al., Oncogene, 1998<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40660393' target='_blank'>Zhou et al., 2025, Hereditas<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37756103' target='_blank'>Soh et al., 2023, Vet Sci<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39485844' target='_blank'>Demoen et al., 2024, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38036730' target='_blank'>Aziz et al., 2023, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37142882' target='_blank'>Xu et al., 2023, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36656933' target='_blank'>Mlynarczyk et al., 2023, Science<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36622753' target='_blank'>Lin et al., Elife, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34175897' target='_blank'>Gui et al., Cell Death Differ, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31221981' target='_blank'>Chang et al., Exp Mol Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30951661' target='_blank'>Brand et al., 2019, Cell Stem Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30890554' target='_blank'>Basheer et al., J Exp Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30341152' target='_blank'>Di et al., J Cell Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29425303' target='_blank'>Koubi et al., Nucleic Acids Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28115373' target='_blank'>Girardi et al., Blood, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24794707' target='_blank'>Chen et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22351929' target='_blank'>Oguro et al., J Exp Med, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20565746' target='_blank'>Nagel et al., Mol Cancer, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36446039' target='_blank'>Trendowski et al., JCO Precis Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32508881' target='_blank'>Miller et al., Front Genet, 2020<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40114169' target='_blank'>Vis et al., 2025, Genome Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39781458' target='_blank'>Wang et al., 2025, Int J Biol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35993362' target='_blank'>Mai et al., JCI Insight, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34439350' target='_blank'>Rasnic et al., 2021, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34185205' target='_blank'>Takaki et al., Ann Surg Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33991232' target='_blank'>Centuori et al., Curr Treat Options Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33466343' target='_blank'>Francis et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32455790' target='_blank'>Guglas et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31558781' target='_blank'>Laginestra et al., 2019, Mod Pathol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31386861' target='_blank'>Zheng et al., Dev Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31028364' target='_blank'>Laginestra et al., 2019, Mod Pathol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27328312' target='_blank'>Pileri et al., Br J Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27325650' target='_blank'>Feber et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24686850' target='_blank'>Lin et al., Nat Genet, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23354438' target='_blank'>Morris et al., Nat Genet, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33194662' target='_blank'>Li et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31663907' target='_blank'>Toma et al., Clin Transl Gastroenterol, 2019<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35560144' target='_blank'>Holland et al., PLoS Comput Biol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31038805' target='_blank'>Zhang et al., J Gastroenterol Hepatol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28463229' target='_blank'>Olvedy et al., J Clin Invest, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19051325' target='_blank'>Shaffer et al., 2008, Genes Chromosomes Cancer<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40352786' target='_blank'>Zheng et al., 2025, Front Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34079082' target='_blank'>Yang et al., Oncogene, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30352677' target='_blank'>Li et al., Trends Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29407175' target='_blank'>Yuan et al., Biochem Biophys Res Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28893348' target='_blank'>Sun et al., Oncol Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27092337' target='_blank'>Guo et al., 2016, Biochem Biophys Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25978455' target='_blank'>Takeuchi et al., PLoS One, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24682381' target='_blank'>Zhao et al., Mol Med Rep, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23962560' target='_blank'>Yang et al., Cancer Lett, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37615251' target='_blank'>Wang et al., 2023, Oncol Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34805801' target='_blank'>Yip et al., iScience, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27051561' target='_blank'>Chaturvedi et al., Scientifica (Cairo), 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23747014' target='_blank'>Sullivan et al., Mol Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17418408' target='_blank'>Pollard et al., 2007, Cancer Cell<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39762409' target='_blank'>Tao et al., 2025, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30867819' target='_blank'>Xu et al., Theranostics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26676757' target='_blank'>Murphy et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24630729' target='_blank'>McFadden et al., Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15902282' target='_blank'>Iwai et al., 2005, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15671544' target='_blank'>Rohr et al., Clin Cancer Res, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15328196' target='_blank'>Tai et al., Clin Cancer Res, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12839965' target='_blank'>Kuroki et al., Cancer Res, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12231533' target='_blank'>Yang et al., Clin Cancer Res, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11389064' target='_blank'>Krivak et al., Cancer Res, 2001<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11221865' target='_blank'>Lee et al., Cancer Res, 2001<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10618602' target='_blank'>Capuzzi et al., Cancer, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10519387' target='_blank'>Tseng et al., Cancer Res, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9823330' target='_blank'>Iwai et al., 1998, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8912845' target='_blank'>Mao et al., Cancer Res, 1996<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38658552' target='_blank'>Zhang et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36508165' target='_blank'>Mezghani et al., 2022, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33849069' target='_blank'>Su et al., Nucleic Acids Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31754475' target='_blank'>Li et al., Signal Transduct Target Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31410199' target='_blank'>Ma et al., Theranostics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28986382' target='_blank'>Yang et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28965871' target='_blank'>Shi et al., Semin Cancer Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28774834' target='_blank'>Kim et al., Semin Cancer Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28351982' target='_blank'>Inoue et al., J Exp Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22829247' target='_blank'>Di et al., Blood Cancer J, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22343918' target='_blank'>Xie et al., Blood, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20551062' target='_blank'>Wu et al., Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17485494' target='_blank'>Bois et al., J Cell Biol, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16849544' target='_blank'>Dong et al., Cancer Res, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16157701' target='_blank'>Bois et al., J Cell Biol, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38566092' target='_blank'>Dong et al., 2024, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38553562' target='_blank'>Liu et al., 2024, Commun Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38131168' target='_blank'>Uddin et al., 2023, Clin Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34202482' target='_blank'>Fontemaggi et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33795838' target='_blank'>Tsuji et al., Oncogene, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31357743' target='_blank'>Mahmud et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31303978' target='_blank'>Sanese et al., Comput Struct Biotechnol J, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30909600' target='_blank'>Grossi et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30738830' target='_blank'>Wen et al., 2019, EBioMedicine<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30064463' target='_blank'>Geng et al., J Hematol Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29721381' target='_blank'>Pyrzynska et al., Oncoimmunology, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26670622' target='_blank'>Goldgraben et al., RNA, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23828519' target='_blank'>Savkovic et al., Br J Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21926415' target='_blank'>Han et al., Hum Mol Genet, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18312651' target='_blank'>Zou et al., Breast Cancer Res, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33202812' target='_blank'>Gu et al., Genes (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18636107' target='_blank'>Yu et al., PLoS Genet, 2008<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40242955' target='_blank'>Chen et al., 2025, Int J Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33803938' target='_blank'>Moriguchi et al., Biomedicines, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33613720' target='_blank'>Zhang et al., Oncol Lett, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33115513' target='_blank'>Tang et al., Stem Cell Res Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31900418' target='_blank'>Colaprico et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31754326' target='_blank'>Li et al., Int J Biol Sci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30788428' target='_blank'>Kwon et al., Mol Ther Oncolytics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30061207' target='_blank'>Emmanuel et al., Anticancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29475976' target='_blank'>Roy et al., J Cell Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29262572' target='_blank'>Gustin et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27588951' target='_blank'>Mair et al., PLoS Genet, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27374105' target='_blank'>Xiao et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24448324' target='_blank'>Li et al., 2014, Cancer Biol Ther<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37960765' target='_blank'>Imon et al., 2023, Medicine (Baltimore)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35002303' target='_blank'>Wu et al., Int J Gen Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31956905' target='_blank'>Singh et al., 2020, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29805640' target='_blank'>Fei et al., Oncol Lett, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27259271' target='_blank'>Han et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25168166' target='_blank'>Valsechi et al., BMC Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18777135' target='_blank'>Sun et al., Breast Cancer Res Treat, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10029067' target='_blank'>Lin et al., Cancer Res, 1999<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/29966365' target='_blank'>Hajdú et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24739903' target='_blank'>Prickett et al., J Invest Dermatol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22932667' target='_blank'>Hurst et al., 2012, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22648273' target='_blank'>Prickett et al., Clin Cancer Res, 2012<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34068503' target='_blank'>Huang et al., Biomedicines, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33591377' target='_blank'>Sárközy et al., Clin Res Cardiol, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38904676' target='_blank'>Shenoy et al., 2024, Clin Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36376832' target='_blank'>Tang et al., Cancer Cell Int, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36233339' target='_blank'>Zhang et al., Int J Mol Sci, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31013831' target='_blank'>Li et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30843252' target='_blank'>Zhou et al., J Clin Lab Anal, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30575897' target='_blank'>Zhang et al., Eur Rev Med Pharmacol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28748001' target='_blank'>Park et al., Clin Epigenetics, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28527972' target='_blank'>Galani et al., Clin Neurol Neurosurg, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33514011' target='_blank'>Misawa et al., Genes (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28423531' target='_blank'>Wang et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28038461' target='_blank'>Xia et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27456940' target='_blank'>Se et al., Cancer Res Treat, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32102485' target='_blank'>de et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28258199' target='_blank'>Li et al., J Immunol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25693514' target='_blank'>Chen et al., Cancer Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23143597' target='_blank'>Love et al., Nat Genet, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/28083147' target='_blank'>Sharma et al., J Nucleic Acids, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26589636' target='_blank'>Dieudonné et al., BMC Genomics, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25964891' target='_blank'>Hashimoto et al., Front Pediatr, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34220922' target='_blank'>Wang et al., Front Genet, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21818355' target='_blank'>Pathak et al., PLoS One, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20927367' target='_blank'>Choi et al., PLoS One, 2010<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39318640' target='_blank'>Alshamrani et al., 2024, Saudi Pharm J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26439987' target='_blank'>Ramalinga et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25625110' target='_blank'>Kheirollahi et al., Adv Biomed Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15218362' target='_blank'>Reinholz et al., Breast Cancer Res Treat, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9892110' target='_blank'>Jonson et al., Genes Chromosomes Cancer, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9464509' target='_blank'>Kageyama et al., Cancer Lett, 1998<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8752209' target='_blank'>Eppert et al., Cell, 1996<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39969704' target='_blank'>Qi et al., 2025, Discov Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32127355' target='_blank'>Chen et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28639733' target='_blank'>Liu et al., Cancer Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27655675' target='_blank'>Wei et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27588495' target='_blank'>Liu et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22321641' target='_blank'>Malkoski et al., FEBS Lett, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21150326' target='_blank'>Cooley et al., Cell Cycle, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9892110' target='_blank'>Jonson et al., Genes Chromosomes Cancer, 1999<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39432061' target='_blank'>Deng et al., 2024, J Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33831375' target='_blank'>Dentro et al., Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32737864' target='_blank'>Wang et al., Protein Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31349837' target='_blank'>Zhang et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29853643' target='_blank'>Tiriac et al., Cancer Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28522603' target='_blank'>Ozawa et al., Clin Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28174172' target='_blank'>Unknown et al., Cancer Discov, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24875860' target='_blank'>Yang et al., Cancer Discov, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23913306' target='_blank'>Lan et al., Hepatology, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23764425' target='_blank'>Hagerstrand et al., Cancer Discov, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23622249' target='_blank'>Baca et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21051335' target='_blank'>Shiau et al., Nucleic Acids Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20682711' target='_blank'>Langeveld et al., Clin Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17200344' target='_blank'>Wang et al., Clin Cancer Res, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12605716' target='_blank'>Schneider et al., Mol Cancer, 2003<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/24567356' target='_blank'>Steed et al., J Cell Biol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24074785' target='_blank'>Ellis et al., Breast, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40516531' target='_blank'>Olsen et al., 2025, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36098964' target='_blank'>Zhu et al., Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35318270' target='_blank'>Bewersdorf et al., Genes Dev, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33823889' target='_blank'>Li et al., J Hematol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28859856' target='_blank'>Sundaresan et al., Gastroenterology, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28273452' target='_blank'>Dreijerink et al., Cell Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27742788' target='_blank'>Maxwell et al., Clin Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25822367' target='_blank'>Malik et al., 2015, Nat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25525250' target='_blank'>Funato et al., Science, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21726809' target='_blank'>Takahashi et al., Mol Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21252315' target='_blank'>Jiao et al., Science, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19228930' target='_blank'>Maillard et al., Blood, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16239140' target='_blank'>Yokoyama et al., Cell, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9407947' target='_blank'>Zhuang et al., Cancer Res, 1997<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9354421' target='_blank'>Zhuang et al., Cancer Res, 1997<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35723009' target='_blank'>Chen et al., Bioengineered, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34150629' target='_blank'>Ali et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34115837' target='_blank'>Kostka et al., Carcinogenesis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33591648' target='_blank'>Gomes et al., Brain Behav, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32561044' target='_blank'>Kinashi et al., Appl Radiat Isot, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27824946' target='_blank'>Ding et al., PLoS One, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27073563' target='_blank'>Hong et al., Oncol Lett, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24554053' target='_blank'>Rankeillor et al., J Neurooncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18798534' target='_blank'>Bondy et al., Cancer, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18363846' target='_blank'>Piperi et al., J Cell Mol Med, 2008<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35721189' target='_blank'>Luo et al., 2022, Front Pharmacol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35452995' target='_blank'>Chen et al., Transl Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33384663' target='_blank'>Colao et al., Front Endocrinol (Lausanne), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28878284' target='_blank'>Niki et al., Sci Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25286960' target='_blank'>Zhang et al., BMC Genomics, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22246327' target='_blank'>Koutsimpelas et al., Oncol Rep, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20959287' target='_blank'>Pardo et al., Nucleic Acids Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20423473' target='_blank'>Liu et al., 2010, BMC Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18813836' target='_blank'>Yu et al., Oncol Rep, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32764986' target='_blank'>Ren et al., Onco Targets Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31920332' target='_blank'>Chen et al., Onco Targets Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29221208' target='_blank'>Zhang et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22125836' target='_blank'>Xiang-qiang et al., Xi Bao Yu Fen Zi Mian Yi Xue Za Zhi, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33194742' target='_blank'>Mal et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28191281' target='_blank'>Nabavi et al., Genes Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15886699' target='_blank'>Wada-Hiraike et al., Br J Cancer, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10024676' target='_blank'>Yano et al., Int J Oncol, 1999<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39192336' target='_blank'>Li et al., 2024, Cell Commun Signal<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37770914' target='_blank'>Chen et al., 2023, Cancer Cell Int<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37414763' target='_blank'>Fang et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33891868' target='_blank'>Wang et al., Stem Cell Reports, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32902730' target='_blank'>Singh et al., Med Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32226520' target='_blank'>Liu et al., J Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31754475' target='_blank'>Li et al., Signal Transduct Target Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31423165' target='_blank'>Liu et al., Oncol Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31262330' target='_blank'>Yang et al., J Exp Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30854093' target='_blank'>Wang et al., J Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30739906' target='_blank'>Wang et al., 2019, Med Sci Monit<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29484121' target='_blank'>Xiong et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28388691' target='_blank'>Wang et al., PLoS One, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24436421' target='_blank'>Schramek et al., Science, 2014<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/15328517' target='_blank'>Krona et al., Br J Cancer, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12066197' target='_blank'>Varon et al., Oncol Rep, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11343781' target='_blank'>Stumm et al., Cancer Genet Cytogenet, 2001<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10928123' target='_blank'>Hama et al., Anticancer Res, 2000<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37094040' target='_blank'>Papp et al., 2023, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36113478' target='_blank'>Chryplewicz et al., Cancer Cell, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35389777' target='_blank'>Dietlein et al., Science, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26214590' target='_blank'>Krauthammer et al., Nat Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24029233' target='_blank'>McLaughlin et al., Cancer Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23476020' target='_blank'>van et al., Nucleic Acids Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22118770' target='_blank'>Tan et al., Dev Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20655465' target='_blank'>Hölzel et al., Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20505189' target='_blank'>Parkin et al., Clin Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19573811' target='_blank'>McGillicuddy et al., Cancer Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19439059' target='_blank'>Nathan et al., 2009, Genome Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18948576' target='_blank'>Yin et al., 2008, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12393498' target='_blank'>Ingram et al., Blood, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11988578' target='_blank'>Zhu et al., 2002, Science<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8499944' target='_blank'>Andersen et al., Nat Genet, 1993<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39251601' target='_blank'>Eaton et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38902237' target='_blank'>Martinez et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38879686' target='_blank'>Xu et al., 2024, NPJ Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36207139' target='_blank'>Sahebjam et al., Genes Dev, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35984495' target='_blank'>Dogan et al., Acta Neuropathol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35534562' target='_blank'>Choudhury et al., Nat Genet, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31509786' target='_blank'>Fomicheva et al., 2019, Curr Opin Cell Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31063759' target='_blank'>Mohan et al., Dev Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28603777' target='_blank'>Napolitano et al., Trends Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27912098' target='_blank'>Plouffe et al., Mol Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23221384' target='_blank'>Zhu et al., Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20179190' target='_blank'>Zhang et al., Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20156804' target='_blank'>Morales et al., Neuro Oncol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12673723' target='_blank'>Utermark et al., Cancer, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9701243' target='_blank'>Kimura et al., Nat Med, 1998<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33859939' target='_blank'>Fei et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32398863' target='_blank'>Cui et al., Cell Res, 2020<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38491013' target='_blank'>Ghashghaei et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33712580' target='_blank'>Wilkinson et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30809292' target='_blank'>Vikova et al., 2019, Theranostics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28157215' target='_blank'>Richter-Pechańska et al., 2017, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25191340' target='_blank'>Shirai et al., Front Genet, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23263491' target='_blank'>De Keersmaecker et al., 2012, Nat Genet<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39447086' target='_blank'>Shao et al., 2024, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38797752' target='_blank'>Shi et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32709844' target='_blank'>Yao et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32471470' target='_blank'>Kim et al., 2020, Genome Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31727771' target='_blank'>Oser et al., Genes Dev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29170450' target='_blank'>Natsuizaka et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27870570' target='_blank'>Ferrarotto et al., J Clin Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25850553' target='_blank'>Takebe et al., Nat Rev Clin Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25303977' target='_blank'>Pickering et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24351288' target='_blank'>Sun et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24277457' target='_blank'>Song et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23610446' target='_blank'>Cui et al., Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21948802' target='_blank'>Lobry et al., J Exp Med, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21798897' target='_blank'>Agrawal et al., Science, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20484026' target='_blank'>Hanlon et al., Cancer Res, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33029085' target='_blank'>Huo et al., 2020, Int J Med Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31727771' target='_blank'>Oser et al., Genes Dev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31480474' target='_blank'>Roszik et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31159827' target='_blank'>Lessi et al., J Transl Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28736522' target='_blank'>Hubmann et al., Front Pharmacol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27363496' target='_blank'>Wu et al., Sci Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27148486' target='_blank'>Crabtree et al., Front Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25229616' target='_blank'>So et al., PLoS One, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17593975' target='_blank'>Boulay et al., PLoS One, 2007<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39424985' target='_blank'>Gamez-Garcia et al., 2024, Nat Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37725138' target='_blank'>Gruenbacher et al., 2023, J Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36835134' target='_blank'>Qiu et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35947077' target='_blank'>Kaiser et al., J Exp Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35156727' target='_blank'>Jurado et al., EMBO J, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34484175' target='_blank'>Dawes et al., Front Immunol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34210001' target='_blank'>Kasprzyk et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31877723' target='_blank'>Niemira et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30416383' target='_blank'>Li et al., Int J Biol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30257940' target='_blank'>Jamrog et al., Proc Natl Acad Sci U S A, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28219927' target='_blank'>Smeenk et al., EMBO J, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26843424' target='_blank'>Zhao et al., J Cell Mol Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25855603' target='_blank'>Dang et al., Blood, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23323012' target='_blank'>Zhao et al., World J Gastroenterol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14592832' target='_blank'>Montesinos-Rongen et al., Blood, 2004<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34712059' target='_blank'>Liu et al., Int J Gen Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30833875' target='_blank'>Nie et al., Dose Response, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26253128' target='_blank'>Salavaty et al., Chin J Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25760074' target='_blank'>Zhanfeng et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17332281' target='_blank'>Gery et al., Clin Cancer Res, 2007<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39858051' target='_blank'>Rzepecka et al., 2025, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37831410' target='_blank'>Huang et al., 2023, Genome Biol Evol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35670774' target='_blank'>Chakraborty et al., Clin Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31938022' target='_blank'>Wang et al., Cancer Cell Int, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26676464' target='_blank'>Yan et al., 2015, Int J Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26560147' target='_blank'>Clemente-Vicario et al., PLoS One, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24229379' target='_blank'>Cizkova et al., BMC Cancer, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38627584' target='_blank'>Pai et al., 2024, Cell Death Differ<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31974171' target='_blank'>Fan et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30244973' target='_blank'>Gentric et al., Cell Metab, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28738810' target='_blank'>Fan et al., Mol Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28449010' target='_blank'>Pruikkonen et al., 2017, Br J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28330929' target='_blank'>Rabellino et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27618486' target='_blank'>Lee et al., Mol Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27545895' target='_blank'>Missiroli et al., Cell Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26627931' target='_blank'>Phillips et al., Nat Rev Mol Cell Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22723221' target='_blank'>Bernardes et al., Circ Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20965424' target='_blank'>Sengupta et al., Mol Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19521423' target='_blank'>Salomoni et al., Cell Death Differ, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18056407' target='_blank'>Stagno et al., J Cell Biol, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15626733' target='_blank'>Salomoni et al., Blood, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14630830' target='_blank'>Gurrieri et al., Blood, 2004<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/32813759' target='_blank'>Villa et al., Acta Cir Bras, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26217306' target='_blank'>Huffman et al., Front Endocrinol (Lausanne), 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16425225' target='_blank'>Teresi et al., Int J Cancer, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15853832' target='_blank'>Costa-Guda et al., Clin Endocrinol (Oxf), 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14500358' target='_blank'>Ying et al., Cancer Res, 2003<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37495516' target='_blank'>Yamamoto et al., 2023, J Vet Med Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34145991' target='_blank'>Kishimoto et al., Cancer Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33789117' target='_blank'>Cho et al., Cell Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33743547' target='_blank'>Kanazawa et al., Cancer Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25999147' target='_blank'>Boylan et al., Exp Cell Res, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/33218058' target='_blank'>Miasaki et al., Genes (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33126717' target='_blank'>Ney et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29900146' target='_blank'>Yang et al., Iran J Public Health, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27995993' target='_blank'>Wang et al., Sci Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27803029' target='_blank'>Saloustros et al., Endocr Relat Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22875002' target='_blank'>Gangoda et al., Cell Death Dis, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32311334' target='_blank'>Qi et al., Trends Biochem Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31175097' target='_blank'>Suehara et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30775456' target='_blank'>Duffy et al., Commun Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29995851' target='_blank'>Qi et al., Nature, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25306392' target='_blank'>Pricl et al., Mol Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22815529' target='_blank'>Fei et al., Cancer Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19642973' target='_blank'>Gimelli et al., Mol Cancer, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17230190' target='_blank'>Wakabayashi et al., Nature, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15888139' target='_blank'>Asplund et al., Br J Dermatol, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15521068' target='_blank'>Thievessen et al., J Cell Physiol, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14613484' target='_blank'>Rahnama et al., Biochem J, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12203113' target='_blank'>Kogerman et al., Oncogene, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10656695' target='_blank'>D'Errico et al., Oncogene, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9581815' target='_blank'>Reifenberger et al., 1998, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9192803' target='_blank'>Undén et al., 1997, Cancer Res<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37380777' target='_blank'>Shih et al., 2023, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35121955' target='_blank'>Zhang et al., Nat Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33597534' target='_blank'>Song et al., Signal Transduct Target Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32581364' target='_blank'>Terao et al., Nature, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32470392' target='_blank'>Augert et al., Cancer Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30181570' target='_blank'>Casero et al., Nat Rev Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28292441' target='_blank'>Blattner et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26974310' target='_blank'>Shojaee et al., Nat Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22464322' target='_blank'>Chen et al., Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22000016' target='_blank'>Karreth et al., Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20944599' target='_blank'>Kelly et al., Nat Biotechnol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19847259' target='_blank'>Trimboli et al., Nature, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19345329' target='_blank'>Yim et al., Cancer Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18772890' target='_blank'>Cancer et al., Nature, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18066063' target='_blank'>Saal et al., Nat Genet, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36690663' target='_blank'>Liu et al., Cell Death Dis, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30029880' target='_blank'>Wu et al., Biochem Biophys Res Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26983689' target='_blank'>Mathur et al., Mol Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26709519' target='_blank'>Zhang et al., PLoS One, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26158411' target='_blank'>Wei et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23398121' target='_blank'>Zheng et al., Eur J Clin Invest, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39732710' target='_blank'>Sun et al., 2024, Cancer Cell Int<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39590309' target='_blank'>Yu et al., 2024, Curr Issues Mol Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38534772' target='_blank'>Di Crosta et al., 2024, Curr Issues Mol Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37598158' target='_blank'>An et al., 2023, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34591414' target='_blank'>Geng et al., Cancer Biol Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34088320' target='_blank'>Varone et al., Cell Commun Signal, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31842362' target='_blank'>Owen et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31285768' target='_blank'>Huang et al., Theranostics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30816454' target='_blank'>Liu et al., Mol Med Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29776962' target='_blank'>Wen et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28594363' target='_blank'>Huang et al., Int J Mol Sci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28408843' target='_blank'>Liu et al., Onco Targets Ther, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26215037' target='_blank'>Sun et al., Radiat Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22962271' target='_blank'>Mereniuk et al., 2012, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12657462' target='_blank'>Wu et al., Gene, 2003<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38967523' target='_blank'>Peng et al., 2024, Cancer Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38766487' target='_blank'>Chen et al., 2024, PeerJ<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33322542' target='_blank'>Mcheik et al., Biomolecules, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33051595' target='_blank'>Yan et al., Oncogene, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32919955' target='_blank'>Long et al., Exp Cell Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32659965' target='_blank'>Leroux et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32536826' target='_blank'>Chen et al., Cancer Cell Int, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31938048' target='_blank'>Hamyeh et al., Theranostics, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29558404' target='_blank'>Huang et al., Cancers (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29333154' target='_blank'>Zhan et al., Gastroenterol Res Pract, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28656243' target='_blank'>Gu et al., Int J Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25543508' target='_blank'>Wang et al., Zhongguo Shi Yan Xue Ye Xue Za Zhi, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23018179' target='_blank'>Abaan et al., Exp Cell Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22245727' target='_blank'>Scrima et al., Am J Pathol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19748322' target='_blank'>Panjala et al., Auton Neurosci, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/25263441' target='_blank'>Zhao et al., Oncogene, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25242281' target='_blank'>Jeon et al., Semin Cell Dev Biol, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39454566' target='_blank'>Binks et al., 2024, Brain<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32796005' target='_blank'>Lopez et al., Genome Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30791455' target='_blank'>Tomiyama et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29558404' target='_blank'>Huang et al., Cancers (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26425837' target='_blank'>Gong et al., J Hypertens, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25113440' target='_blank'>Walia et al., Hum Mutat, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24086065' target='_blank'>Huang et al., Cold Spring Harb Perspect Med, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22305495' target='_blank'>Meehan et al., Mol Cancer, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22197930' target='_blank'>Stark et al., Nat Genet, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21387281' target='_blank'>Li et al., Cancer, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21213369' target='_blank'>Giefing et al., Genes Chromosomes Cancer, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20073072' target='_blank'>Kohno et al., Genes Chromosomes Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19478061' target='_blank'>Veeriah et al., Proc Natl Acad Sci U S A, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17420988' target='_blank'>Purdie et al., Genes Chromosomes Cancer, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16585193' target='_blank'>Stallings et al., Cancer Res, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33371188' target='_blank'>Avila et al., Mar Drugs, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30838170' target='_blank'>Xu et al., Anal Cell Pathol (Amst), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30208623' target='_blank'>Kim et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28987514' target='_blank'>Chen et al., J Psychiatr Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27925266' target='_blank'>Ciavatta et al., Med Res Rev, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24206917' target='_blank'>Parker et al., Chin J Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24002526' target='_blank'>Sun et al., Int J Oncol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23552869' target='_blank'>Sun et al., J Cancer Res Clin Oncol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23105925' target='_blank'>Kim et al., Genomics Inform, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/29089486' target='_blank'>Cheng et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19800577' target='_blank'>Bric et al., Cancer Cell, 2009<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/32457603' target='_blank'>Chen et al., Front Pharmacol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21704205' target='_blank'>Ceballos et al., Biochim Biophys Acta, 2011<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36596785' target='_blank'>Li et al., Signal Transduct Target Ther, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31031003' target='_blank'>Vasaikar et al., Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28292439' target='_blank'>Cherniack et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28256574' target='_blank'>Schneider et al., Nat Rev Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23842645' target='_blank'>Sadasivam et al., Nat Rev Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23079655' target='_blank'>Choi et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22094256' target='_blank'>Anders et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20305037' target='_blank'>Psyrri et al., 2010, Ann Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19851314' target='_blank'>Chen et al., Nat Rev Cancer, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18794899' target='_blank'>Morris et al., Nature, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18719710' target='_blank'>van et al., Nat Rev Mol Cell Biol, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15479636' target='_blank'>Korenjak et al., Cell, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12086848' target='_blank'>Johnson et al., Cancer Cell, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9462511' target='_blank'>Lissy et al., Immunity, 1998<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8653789' target='_blank'>Yamasaki et al., Cell, 1996<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35293275' target='_blank'>Ma et al., Bioengineered, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32483207' target='_blank'>Tong et al., Oncogenesis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31874600' target='_blank'>Lu et al., BMC Bioinformatics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28147343' target='_blank'>Fancello et al., Oncotarget, 2017<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/31581233' target='_blank'>Rao et al., PLoS One, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28264936' target='_blank'>Ajore et al., 2017, EMBO Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28076796' target='_blank'>Zhang et al., Cell Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27197189' target='_blank'>Rao et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24170611' target='_blank'>Le et al., Clin Chem, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39256447' target='_blank'>Rodrigues et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38948759' target='_blank'>Mandros et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35892689' target='_blank'>Bénit et al., Biomedicines, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34014604' target='_blank'>Kamai et al., Cancer Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33854214' target='_blank'>Whitworth et al., 2021, Eur J Hum Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30704450' target='_blank'>Menor et al., BMC Med Genomics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23109135' target='_blank'>Belinsky et al., Genes Chromosomes Cancer, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39319222' target='_blank'>Li et al., 2024, Urol Case Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35892689' target='_blank'>Bénit et al., Biomedicines, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34094922' target='_blank'>Fang et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33168813' target='_blank'>Seçilmiş et al., NPJ Syst Biol Appl, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31689495' target='_blank'>Ooi et al., 2019, Semin Cancer Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31579262' target='_blank'>Aghamir et al., Onco Targets Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24203292' target='_blank'>Shimizu et al., J Gastroenterol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21301796' target='_blank'>Frezza et al., J Mol Med (Berl), 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19546167' target='_blank'>Bayley et al., Endocr Relat Cancer, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15220362' target='_blank'>Morris et al., J Clin Pathol, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14500403' target='_blank'>Gimenez-Roqueplo et al., Cancer Res, 2003<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35892689' target='_blank'>Bénit et al., Biomedicines, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31689495' target='_blank'>Ooi et al., 2019, Semin Cancer Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21301796' target='_blank'>Frezza et al., J Mol Med (Berl), 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19546167' target='_blank'>Bayley et al., Endocr Relat Cancer, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35892689' target='_blank'>Bénit et al., Biomedicines, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31689495' target='_blank'>Ooi et al., 2019, Semin Cancer Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27856506' target='_blank'>Burnichon et al., J Med Genet, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21301796' target='_blank'>Frezza et al., J Mol Med (Berl), 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19546167' target='_blank'>Bayley et al., Endocr Relat Cancer, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35511434' target='_blank'>Tayoun et al., JCI Insight, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31015295' target='_blank'>Auslander et al., Proc Natl Acad Sci U S A, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28454307' target='_blank'>Ding et al., Oncol Lett, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27148394' target='_blank'>Lu et al., Algorithms Mol Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24713432' target='_blank'>Davies et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22522925' target='_blank'>Curtis et al., Nature, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21896780' target='_blank'>Ahn et al., Mol Cell Biol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21575258' target='_blank'>Davis et al., BMC Cancer, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21487811' target='_blank'>Yeasmin et al., Tumour Biol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19665249' target='_blank'>Newell et al., J Hepatol, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18688287' target='_blank'>Schutte et al., PLoS One, 2007<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35272015' target='_blank'>Y et al., 2022, Int J Biochem Cell Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33760186' target='_blank'>Wu et al., Mol Med Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32952624' target='_blank'>Liu et al., Exp Ther Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25127546' target='_blank'>Fotin-Mleczek et al., Radiat Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22363760' target='_blank'>Jacob et al., PLoS One, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39638567' target='_blank'>Cox et al., 2024, Genes Dev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38374872' target='_blank'>Gu et al., 2024, MedComm (2020)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37433987' target='_blank'>Navickas et al., 2023, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37387441' target='_blank'>Panozzi et al., 2023, Pathologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37314393' target='_blank'>Kabir et al., 2023, Chem Soc Rev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34050264' target='_blank'>Mou et al., Exp Mol Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33853662' target='_blank'>Yao et al., Genome Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33190170' target='_blank'>Zhu et al., Mol Neurobiol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32162380' target='_blank'>Hu et al., Cell Prolif, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26671993' target='_blank'>Bell et al., Clin Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25391308' target='_blank'>Schiaffino-Ortega et al., J Hematol Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21564052' target='_blank'>Bock et al., Br J Dermatol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19448667' target='_blank'>Naidu et al., 2009, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19149898' target='_blank'>Becker et al., Mol Cancer, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12566296' target='_blank'>Reisman et al., 2003, Cancer Res<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37637078' target='_blank'>Hua et al., 2023, Bioact Mater<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37186844' target='_blank'>Soeung et al., 2023, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35915143' target='_blank'>Yang et al., Leukemia, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34003336' target='_blank'>Pathak et al., Acta Neuropathol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33239431' target='_blank'>Hong et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28824165' target='_blank'>Vitte et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28249160' target='_blank'>Sen et al., Cell Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25614489' target='_blank'>Nielsen et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22719073' target='_blank'>Hertwig et al., Cancer Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19276269' target='_blank'>Jackson et al., Clin Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16099835' target='_blank'>Tsikitis et al., Proc Natl Acad Sci U S A, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15899790' target='_blank'>Modena et al., Cancer Res, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10914721' target='_blank'>Biegel et al., Clin Cancer Res, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10397258' target='_blank'>Rousseau-Merck et al., Cancer Res, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9892189' target='_blank'>Biegel et al., Cancer Res, 1999<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34470640' target='_blank'>Zuo et al., World J Surg Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33520987' target='_blank'>Sun et al., Front Cell Dev Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32308428' target='_blank'>Sun et al., Onco Targets Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30622340' target='_blank'>Domenici et al., Oncogene, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29059173' target='_blank'>Zhou et al., Oncogene, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28767682' target='_blank'>Xi et al., PLoS One, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28277188' target='_blank'>Huang et al., Oncol Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27610139' target='_blank'>Xiang et al., Stem Cells Int, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27457222' target='_blank'>Smith et al., Sci Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26036262' target='_blank'>Wang et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22246670' target='_blank'>Matheu et al., Cancer Res, 2012<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38741014' target='_blank'>Jia et al., 2024, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38351187' target='_blank'>Besse et al., 2024, Nat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38177135' target='_blank'>Yuan et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34870237' target='_blank'>Lehtiö et al., Nat Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33257855' target='_blank'>Kim et al., 2020, Nat Metab<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32571374' target='_blank'>Jiang et al., J Hematol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30415923' target='_blank'>Eichner et al., Cell Metab, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30224757' target='_blank'>Coudray et al., 2018, Nat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29455666' target='_blank'>Román et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29403037' target='_blank'>Xie et al., Nat Cell Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27300434' target='_blank'>Zanconato et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26917230' target='_blank'>Chen et al., 2016, J Thorac Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25695224' target='_blank'>Gleeson et al., 2015, J Thorac Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23352126' target='_blank'>Shackelford et al., Cancer Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21892142' target='_blank'>Mihaylova et al., 2011, Nat Cell Biol<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38909034' target='_blank'>Deng et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38081972' target='_blank'>Andreani et al., 2023, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33577921' target='_blank'>Liang et al., J Hepatol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27100732' target='_blank'>Amir et al., Br J Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26900951' target='_blank'>Willmer et al., Oncogenesis, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36566600' target='_blank'>Kim et al., Pathol Res Pract, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34071216' target='_blank'>Unfried et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31103629' target='_blank'>Lu et al., EBioMedicine, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30598639' target='_blank'>Pan et al., Cancer Cell Int, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29466992' target='_blank'>Ding et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28943628' target='_blank'>Marchetti et al., Diseases, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27604489' target='_blank'>Mehra et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27571104' target='_blank'>Staal et al., 2016, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26098869' target='_blank'>Childs et al., Nat Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25793983' target='_blank'>Luo et al., PLoS One, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25657029' target='_blank'>Kong et al., Sci Rep, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23159221' target='_blank'>Staal et al., Immunity, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40610429' target='_blank'>Yang et al., 2025, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34175897' target='_blank'>Gui et al., Cell Death Differ, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32425548' target='_blank'>Li et al., Onco Targets Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26202930' target='_blank'>Tan et al., Leukemia, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18981238' target='_blank'>Li et al., 2008, J Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17311319' target='_blank'>Barber et al., 2007, Genes Chromosomes Cancer<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40017274' target='_blank'>Arafat et al., 2025, Clin Exp Pediatr<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33807605' target='_blank'>Thilakarathna et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29121666' target='_blank'>Richtig et al., PLoS One, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26258591' target='_blank'>Li et al., PLoS One, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34003815' target='_blank'>Gao et al., 2021, Med Sci Monit<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30549280' target='_blank'>Rao et al., Hepatology, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28352351' target='_blank'>Chen et al., Exp Ther Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25315765' target='_blank'>Nadauld et al., Genome Biol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24838641' target='_blank'>Liu et al., Braz J Med Biol Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19959123' target='_blank'>Tseng et al., Heart Rhythm, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40568576' target='_blank'>Xu et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33947861' target='_blank'>Little et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33850013' target='_blank'>Kfoury et al., Proc Natl Acad Sci U S A, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33227088' target='_blank'>Amaar et al., Oncotarget, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33083012' target='_blank'>Teschendorff et al., NPJ Genom Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29267283' target='_blank'>Milewski et al., PLoS Genet, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25478793' target='_blank'>Zhao et al., PLoS One, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35939695' target='_blank'>Li et al., 2022, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34454635' target='_blank'>Lei et al., Biomark Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26538215' target='_blank'>Chen et al., Mol Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26366710' target='_blank'>Ortega-Molina et al., 2015, Nat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22730341' target='_blank'>Fulda et al., EMBO Mol Med, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22585859' target='_blank'>Bellail et al., Cancer Discov, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22343828' target='_blank'>Shembade et al., Cell Mol Immunol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22099304' target='_blank'>Skaug et al., Mol Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21119682' target='_blank'>Harhaj et al., Cell Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20705491' target='_blank'>Tavares et al., Immunity, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20383180' target='_blank'>Hymowitz et al., Nat Rev Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20185725' target='_blank'>Shembade et al., Science, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19608751' target='_blank'>Honma et al., Blood, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19380639' target='_blank'>Schmitz et al., J Exp Med, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19380636' target='_blank'>Malynn et al., J Exp Med, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37059070' target='_blank'>Halbrook et al., 2023, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30146162' target='_blank'>Minzel et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29017057' target='_blank'>Mello et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26977876' target='_blank'>Haricharan et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26639348' target='_blank'>Roberts et al., N Engl J Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22975375' target='_blank'>Navas et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22698404' target='_blank'>Jackson et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21565614' target='_blank'>Brady et al., Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19550424' target='_blank'>Hecht et al., Nat Rev Cancer, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19477430' target='_blank'>Wang et al., 2009, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18948956' target='_blank'>Zheng et al., Nature, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18183014' target='_blank'>Frew et al., 2008, Nat Biotechnol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16098465' target='_blank'>Zhu et al., Cancer Cell, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12086853' target='_blank'>Lessnick et al., Cancer Cell, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9142118' target='_blank'>Ananthaswamy et al., Nat Med, 1997<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39281855' target='_blank'>Jones et al., 2024, Res Sq<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36640300' target='_blank'>Lee et al., 2023, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31874112' target='_blank'>Zhang et al., Med Sci Monit, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30469518' target='_blank'>Balacescu et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27182479' target='_blank'>Arifi et al., Rare Cancers Ther, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25526364' target='_blank'>Shiraishi et al., PLoS One, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21798997' target='_blank'>Kucejova et al., Mol Cancer Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19966866' target='_blank'>Liang et al., Oncogene, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19401700' target='_blank'>Presneau et al., Br J Cancer, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18480009' target='_blank'>Mhawech-Fauceglia et al., Am J Clin Pathol, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18397877' target='_blank'>Pymar et al., Hum Mol Genet, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12853839' target='_blank'>Adachi et al., J Urol, 2003<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39281855' target='_blank'>Jones et al., 2024, Res Sq<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37686092' target='_blank'>Parker et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36640300' target='_blank'>Lee et al., 2023, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30888083' target='_blank'>Hino et al., Cancer Sci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28349193' target='_blank'>Klaus et al., Arch Toxicol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27182479' target='_blank'>Arifi et al., Rare Cancers Ther, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19401700' target='_blank'>Presneau et al., Br J Cancer, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11129334' target='_blank'>Parry et al., Hum Genet, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10944454' target='_blank'>Yamashita et al., Biochem Biophys Res Commun, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9186374' target='_blank'>Urakami et al., J Urol, 1997<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39256355' target='_blank'>Shi et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35705742' target='_blank'>Arnold et al., Nat Rev Endocrinol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35443743' target='_blank'>Adnan Awad et al., 2022, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34509979' target='_blank'>Revia et al., Gut, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34006303' target='_blank'>Liu et al., Mol Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34002060' target='_blank'>Zhao et al., 2021, Nat Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31044092' target='_blank'>Jiang et al., Signal Transduct Target Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30872525' target='_blank'>Chakraborty et al., Science, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30753822' target='_blank'>Wang et al., 2019, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29482581' target='_blank'>Liu et al., J Hematol Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28115373' target='_blank'>Girardi et al., Blood, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27165743' target='_blank'>Yuan et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26590715' target='_blank'>Cai et al., Mol Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25320243' target='_blank'>Van et al., Blood, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25132549' target='_blank'>Ntziachristos et al., Nature, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40251399' target='_blank'>Xie et al., 2025, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37069149' target='_blank'>Hu et al., 2023, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32273439' target='_blank'>Faubert et al., Science, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31955850' target='_blank'>Burslem et al., Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26198328' target='_blank'>Shenoy et al., J Hematol Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24656772' target='_blank'>Li et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24332042' target='_blank'>Li et al., Cancer Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23473032' target='_blank'>Gameiro et al., Cell Metab, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22101433' target='_blank'>Metallo et al., Nature, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21248752' target='_blank'>Varela et al., Nature, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20029423' target='_blank'>Collado et al., Nat Rev Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19061835' target='_blank'>Gordan et al., Cancer Cell, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18598947' target='_blank'>Turcotte et al., Cancer Cell, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10213691' target='_blank'>Kamura et al., Science, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7915601' target='_blank'>Gnarra et al., Nat Genet, 1994<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36446039' target='_blank'>Trendowski et al., JCO Precis Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30584360' target='_blank'>Zhu et al., Cancer Manag Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25538733' target='_blank'>Sharma et al., Front Genet, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37917714' target='_blank'>Gregoire et al., 2023, Science<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35590059' target='_blank'>Hetzel et al., 2022, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27613060' target='_blank'>de et al., J Hematol Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24439384' target='_blank'>Cohen et al., Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22172722' target='_blank'>Amin et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20972433' target='_blank'>De et al., Nat Med, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10490105' target='_blank'>Lee et al., Cell, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8395349' target='_blank'>Kreidberg et al., Cell, 1993<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8298644' target='_blank'>Park et al., Nat Genet, 1993<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8012395' target='_blank'>Jinno et al., Nat Genet, 1994<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7736591' target='_blank'>Larsson et al., Cell, 1995<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1655284' target='_blank'>Pelletier et al., Cell, 1991<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1654597' target='_blank'>Madden et al., Science, 1991<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1323141' target='_blank'>Drummond et al., Science, 1992<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1302008' target='_blank'>Bruening et al., Nat Genet, 1992<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/27777383' target='_blank'>Sun et al., Aging (Albany NY), 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25871391' target='_blank'>Cui et al., Oncotarget, 2015<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40552012' target='_blank'>Shi et al., 2025, Front Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39308302' target='_blank'>Liu et al., 2024, Acta Biochim Biophys Sin (Shanghai)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34489442' target='_blank'>Shi et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32517789' target='_blank'>He et al., Clin Epigenetics, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32349303' target='_blank'>Noh et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30602500' target='_blank'>Stopsack et al., 2019, Cancer Epidemiol Biomarkers Prev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29050363' target='_blank'>Jin et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27541592' target='_blank'>He et al., Neurochem Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25808865' target='_blank'>Hsieh et al., Cancer Res, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/31341405' target='_blank'>Li et al., Int J Med Sci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27757741' target='_blank'>Zhang et al., Cell Mol Neurobiol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26781093' target='_blank'>Zhang et al., BMC Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26690953' target='_blank'>Zhang et al., BMC Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25120642' target='_blank'>Dong et al., Oncol Lett, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22372344' target='_blank'>Dong et al., Clin Lab, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22363126' target='_blank'>Dong et al., World J Gastroenterol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20878080' target='_blank'>Geli et al., Int J Oncol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20159667' target='_blank'>Cheng et al., Eur J Med Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18712668' target='_blank'>Piao et al., Histol Histopathol, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15809732' target='_blank'>Geli et al., Int J Oncol, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14668725' target='_blank'>Carling et al., Surgery, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11719434' target='_blank'>Du et al., Cancer Res, 2001<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11280725' target='_blank'>Jiang et al., Cancer Res, 2001<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10508492' target='_blank'>Jiang et al., Int J Cancer, 1999<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38049396' target='_blank'>Lin et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33489900' target='_blank'>Li et al., 2021, Front Oncol<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39117659' target='_blank'>Liu et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39025450' target='_blank'>Wu et al., 2024, J Biol Chem<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37142882' target='_blank'>Xu et al., 2023, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36525973' target='_blank'>Pan et al., Cancer Cell, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33086062' target='_blank'>Maitituoheti et al., Cell Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32350066' target='_blank'>Saffie et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32243837' target='_blank'>Alam et al., Cancer Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31537689' target='_blank'>Ferrero et al., Haematologica, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31301850' target='_blank'>Aubert et al., 2019, Trends Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31128216' target='_blank'>Fagan et al., Cancer Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29950560' target='_blank'>Hillman et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28778402' target='_blank'>Heuser et al., Semin Cancer Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26366712' target='_blank'>Zhang et al., Nat Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25676421' target='_blank'>Yamamoto et al., Leukemia, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21796119' target='_blank'>Morin et al., Nature, 2011<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40542439' target='_blank'>Yu et al., 2025, Eur J Med Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38983105' target='_blank'>Ibe et al., 2024, Mol Genet Metab Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36233804' target='_blank'>Marascio et al., J Clin Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34222248' target='_blank'>Talley et al., 2021, Front Cell Dev Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34113392' target='_blank'>Zhang et al., Oncol Lett, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24362817' target='_blank'>Piotrowski et al., Nat Genet, 2014<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40155675' target='_blank'>Lv et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39144991' target='_blank'>Sun et al., 2024, Heliyon<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39090742' target='_blank'>Hwangbo et al., 2024, Virol J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38032932' target='_blank'>Lee et al., 2023, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37827547' target='_blank'>Wang et al., 2023, J Mol Cell Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34356101' target='_blank'>Mehterov et al., Genes (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34026613' target='_blank'>Lin et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33718153' target='_blank'>Cao et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30403180' target='_blank'>Han et al., Med Sci (Paris), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28091594' target='_blank'>Zhao et al., Sci Rep, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37749329' target='_blank'>Qi et al., 2023, J Cancer Res Clin Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36631623' target='_blank'>Trempenau et al., Leukemia, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36192394' target='_blank'>Xiao et al., Cell Death Dis, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35080113' target='_blank'>Qian et al., 2022, Kaohsiung J Med Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34522206' target='_blank'>Zheng et al., Theranostics, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33654410' target='_blank'>Zhang et al., Onco Targets Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32714863' target='_blank'>Zhang et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31221981' target='_blank'>Chang et al., Exp Mol Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29119601' target='_blank'>Larson et al., Genet Epidemiol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28326264' target='_blank'>Kluzek et al., J Kidney Cancer VHL, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37939712' target='_blank'>Xing et al., 2023, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37402719' target='_blank'>Fuchs et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36653445' target='_blank'>Loe et al., 2023, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36435834' target='_blank'>Li et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35410466' target='_blank'>Bordeira-Carriço et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34050264' target='_blank'>Mou et al., Exp Mol Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33579932' target='_blank'>Iftekhar et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32832616' target='_blank'>Shi et al., Sci Adv, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32494027' target='_blank'>Alzamil et al., Cell Death Differ, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30242023' target='_blank'>Chaudhary et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29136504' target='_blank'>Sun et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27922010' target='_blank'>Smida et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26523267' target='_blank'>Lee et al., Clin Mol Hepatol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23525077' target='_blank'>Dulak et al., Nat Genet, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20942669' target='_blank'>Wiegand et al., N Engl J Med, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35699406' target='_blank'>Yan et al., Cancer Biol Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34781631' target='_blank'>Fang et al., J Mov Disord, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33132633' target='_blank'>D'souza et al., World J Gastroenterol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32958699' target='_blank'>Innocenti et al., Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27613831' target='_blank'>Yang et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25236910' target='_blank'>Mazzoni et al., Cancer Lett, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23256519' target='_blank'>Saito-Diaz et al., Growth Factors, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21808024' target='_blank'>Zhang et al., Proc Natl Acad Sci U S A, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17309796' target='_blank'>Kudo et al., World J Surg Oncol, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11806834' target='_blank'>Miller et al., Genome Biol, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11739413' target='_blank'>Hsu et al., J Cell Biol, 2001<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35115535' target='_blank'>Miete et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34072517' target='_blank'>Putnová et al., Animals (Basel), 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40199870' target='_blank'>Schaeffeler et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38925123' target='_blank'>Motsinger-Reif et al., 2024, Cell Genom<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38072908' target='_blank'>Wang et al., 2023, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37137901' target='_blank'>Rüland et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35190728' target='_blank'>Dinh et al., 2022, Nat Rev Gastroenterol Hepatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34433654' target='_blank'>Schreck et al., Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33144689' target='_blank'>Jonasch et al., Nat Rev Nephrol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32541668' target='_blank'>Perkail et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32413317' target='_blank'>Stockwell et al., Trends Cell Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32029730' target='_blank'>Wang et al., 2020, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31130514' target='_blank'>Artegiani et al., Cell Stem Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30202049' target='_blank'>Zhang et al., Nat Cell Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30013160' target='_blank'>Asada et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26419610' target='_blank'>Qin et al., Nat Commun, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22683710' target='_blank'>Peña-Llopis et al., Nat Genet, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40521202' target='_blank'>Yang et al., 2025, Theranostics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36512624' target='_blank'>Zhang et al., Cancer Res, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35641493' target='_blank'>Zhou et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35512704' target='_blank'>Mo et al., 2022, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32460168' target='_blank'>Wang et al., EBioMedicine, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32235916' target='_blank'>Hu et al., Cell Mol Immunol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31901237' target='_blank'>Song et al., Mol Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31776466' target='_blank'>Jin et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30124983' target='_blank'>Hjorth-Jensen et al., Nucleic Acids Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29572225' target='_blank'>Adelaiye-Ogala et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27614073' target='_blank'>Zhu et al., Nucleic Acids Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26344095' target='_blank'>Gan et al., Mol Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25204354' target='_blank'>Zeng et al., J Exp Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25056375' target='_blank'>Khani et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24656772' target='_blank'>Li et al., Cancer Cell, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32346533' target='_blank'>Vanneste et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27664236' target='_blank'>Zhang et al., Genes Dev, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25995385' target='_blank'>Dorr et al., Mol Cancer Res, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38481282' target='_blank'>Sultanov et al., 2024, Epigenetics Chromatin<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36191828' target='_blank'>Chauhan et al., 2022, Biochim Biophys Acta Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35024725' target='_blank'>Diţescu et al., Rom J Morphol Embryol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34206538' target='_blank'>Burcher et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33672261' target='_blank'>Raue et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33156825' target='_blank'>Park et al., PLoS One, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31019894' target='_blank'>Li et al., Front Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29988031' target='_blank'>Li et al., 2018, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29441939' target='_blank'>Ding et al., Pharmazie, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26990571' target='_blank'>Nam et al., Prostate, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26878872' target='_blank'>Xu-Monette et al., Aging (Albany NY), 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25801506' target='_blank'>Zhang et al., Stem Cell Reports, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24530564' target='_blank'>Huang et al., 2014, Gynecol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16082248' target='_blank'>Hedvat et al., 2005, Appl Immunohistochem Mol Morphol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15458526' target='_blank'>Wrone et al., Dermatol Surg, 2004<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38254783' target='_blank'>Shukla et al., 2024, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33493533' target='_blank'>Santharam et al., Biochimie, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32807134' target='_blank'>Khan et al., 2020, BMC Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32664401' target='_blank'>Sartorius et al., Viruses, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32631271' target='_blank'>Lü et al., 2020, BMC Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29620202' target='_blank'>Zhang et al., Mol Med Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29085209' target='_blank'>Gui et al., World J Gastroenterol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28661477' target='_blank'>Chen et al., Cell Death Dis, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28369102' target='_blank'>Beldi-Ferchiou et al., PLoS One, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27811366' target='_blank'>Xue et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26391193' target='_blank'>Tobelaim et al., Sci Rep, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21664423' target='_blank'>Culig et al., Mol Cell Endocrinol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19223509' target='_blank'>Tanemura et al., Clin Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15074307' target='_blank'>Fujitake et al., J Gastroenterol, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14614012' target='_blank'>Okochi et al., Clin Cancer Res, 2003<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/28115373' target='_blank'>Girardi et al., Blood, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20444294' target='_blank'>Chiang et al., J Biomed Sci, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37041226' target='_blank'>Sordo-Bahamonde et al., 2023, Cancer Immunol Immunother<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33237303' target='_blank'>Razzaghi et al., J Exp Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30066919' target='_blank'>Zhu et al., Mol Med Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27693350' target='_blank'>Boice et al., Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23339845' target='_blank'>Bjordahl et al., Curr Opin Immunol, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34941167' target='_blank'>Xue et al., Medicine (Baltimore), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32731534' target='_blank'>McAvera et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27689360' target='_blank'>Chi et al., Aging (Albany NY), 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39067088' target='_blank'>Zhang et al., 2024, Transl Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31586074' target='_blank'>Caeser et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31553912' target='_blank'>Elman et al., Cell Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26280531' target='_blank'>Park et al., Nat Struct Mol Biol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24117486' target='_blank'>Baumgarten et al., Neuropathol Appl Neurobiol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23934175' target='_blank'>Odia et al., J Neurooncol, 2013<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34589154' target='_blank'>Zhou et al., Oncol Lett, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34549306' target='_blank'>Deng et al., Oncol Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32460168' target='_blank'>Wang et al., EBioMedicine, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31848340' target='_blank'>Tang et al., 2019, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30710319' target='_blank'>Xu et al., Protein Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28259899' target='_blank'>Deng et al., Mol Med Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25270913' target='_blank'>Chen et al., Int J Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23573293' target='_blank'>Yeung et al., PLoS One, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22909338' target='_blank'>Ji et al., J Exp Clin Cancer Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19799973' target='_blank'>Visser et al., Gene, 2010<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37853437' target='_blank'>Zeng et al., 2023, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37831104' target='_blank'>Gao et al., 2023, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31623606' target='_blank'>Zhang et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28366734' target='_blank'>Kim et al., Gastroenterology, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26977883' target='_blank'>Wei et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25037230' target='_blank'>Malik et al., Mol Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23919723' target='_blank'>El-Karim et al., Mol Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22677193' target='_blank'>Li et al., Gastroenterology, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22284679' target='_blank'>Gamper et al., Mol Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22155367' target='_blank'>Li et al., Gastroenterology, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20727893' target='_blank'>Wei et al., Gastroenterology, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20519630' target='_blank'>Guan et al., Blood, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19737957' target='_blank'>Hu et al., Clin Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17085651' target='_blank'>Kanai et al., Clin Cancer Res, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16954505' target='_blank'>Kharas et al., Blood, 2007<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/25538733' target='_blank'>Sharma et al., Front Genet, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23412794' target='_blank'>Sadikovic et al., Methods Mol Biol, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40052530' target='_blank'>Li et al., 2025, Protein Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39479357' target='_blank'>Guo et al., 2024, Front Neurosci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38782769' target='_blank'>Yan et al., 2024, Discov Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38769028' target='_blank'>Wang et al., 2024, Cancer Control<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38485986' target='_blank'>Duan et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36051438' target='_blank'>Aksoy et al., Front Cell Dev Biol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32047543' target='_blank'>Shi et al., J Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30400822' target='_blank'>Unknown et al., J Immunother Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30059005' target='_blank'>Li et al., Elife, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28991235' target='_blank'>Kristensen et al., Oncogene, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27841882' target='_blank'>Shingu et al., 2016, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27555542' target='_blank'>de Miguel et al., 2016, Mol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27282250' target='_blank'>Dvinge et al., Nat Rev Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25971746' target='_blank'>Wang et al., Mol Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24120142' target='_blank'>Brennan et al., Cell, 2013<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38030614' target='_blank'>Hou et al., 2023, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34022836' target='_blank'>Han et al., BMC Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32781585' target='_blank'>Valtorta et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31661545' target='_blank'>Tan et al., Biosci Rep, 2019<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/32353968' target='_blank'>Humphries et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31477830' target='_blank'>Joseph et al., Oncogene, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40565184' target='_blank'>Vobugari et al., 2025, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40083325' target='_blank'>Shen et al., 2025, Genes Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39638567' target='_blank'>Cox et al., 2024, Genes Dev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39343883' target='_blank'>Yi et al., 2024, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38862711' target='_blank'>Tabe-Bordbar et al., 2024, Commun Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38770632' target='_blank'>Li et al., 2024, Cancer Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38453898' target='_blank'>Zhou et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38413563' target='_blank'>Li et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37633973' target='_blank'>Kim et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36841242' target='_blank'>Weiss-Sadan et al., 2023, Cell Metab<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33126057' target='_blank'>Liu et al., Redox Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32560187' target='_blank'>Salgia et al., J Clin Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32365499' target='_blank'>Mohrherr et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30956168' target='_blank'>Wang et al., EBioMedicine, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30060526' target='_blank'>Testa et al., Cancers (Basel), 2018<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39354505' target='_blank'>Petersen et al., 2024, Cell Commun Signal<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32276588' target='_blank'>Faulkner et al., J Hematol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31530281' target='_blank'>Nath et al., Cell Commun Signal, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25906751' target='_blank'>Sowalsky et al., Oncotarget, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33428680' target='_blank'>Khunger et al., PLoS One, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31110180' target='_blank'>Ding et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26151821' target='_blank'>Schumacher et al., Nat Commun, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40555735' target='_blank'>Ding et al., 2025, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39862423' target='_blank'>Li et al., 2025, Cancer Biol Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38988268' target='_blank'>Dovat et al., 2024, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34042448' target='_blank'>Nishiguchi et al., J Med Chem, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33807974' target='_blank'>Klink et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33531656' target='_blank'>Ge et al., Leukemia, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33467550' target='_blank'>Dovat et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32085659' target='_blank'>Dhanyamraju et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31073152' target='_blank'>Ding et al., Leukemia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28190001' target='_blank'>Schjerven et al., 2017, J Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27818917' target='_blank'>Fan et al., Acta Pharm Sin B, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26655717' target='_blank'>Wang et al., J Biol Chem, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26219304' target='_blank'>Song et al., Blood, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26049627' target='_blank'>Bottardi et al., Trends Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24522918' target='_blank'>Iempridee et al., J Virol, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35448004' target='_blank'>Nakahara et al., Brain Sci, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34662721' target='_blank'>Hou et al., Biomed J, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33731112' target='_blank'>Lüdtke et al., Respir Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33023006' target='_blank'>Cheng et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32537867' target='_blank'>Zhang et al., Cell Prolif, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31581708' target='_blank'>Chang et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31253870' target='_blank'>Crawford et al., Oncogene, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31160382' target='_blank'>Popow et al., Mol Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30413609' target='_blank'>Jing et al., Biosci Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26713269' target='_blank'>Cho et al., Acta Pharm Sin B, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26579496' target='_blank'>Fischer et al., Front Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26011451' target='_blank'>Du et al., PLoS One, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25732125' target='_blank'>Chiang et al., Sci Rep, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22678098' target='_blank'>Lv et al., J Cancer Res Clin Oncol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20348948' target='_blank'>Redmond et al., Oncogene, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/26910912' target='_blank'>Zhang et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26156586' target='_blank'>Wang et al., J Biomed Sci, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35876041' target='_blank'>Pan et al., Clin Transl Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33952279' target='_blank'>Liu et al., J Orthop Surg Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32767321' target='_blank'>Shu et al., Eur Rev Med Pharmacol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26989293' target='_blank'>Dong et al., Dis Markers, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36037342' target='_blank'>Lebeau et al., Proc Natl Acad Sci U S A, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33673494' target='_blank'>Kang et al., Biomolecules, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32494027' target='_blank'>Alzamil et al., Cell Death Differ, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32435142' target='_blank'>Akhtar et al., Saudi Pharm J, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29212169' target='_blank'>Wu et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17363576' target='_blank'>De et al., Cancer Res, 2007<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39652422' target='_blank'>Ashkin et al., 2024, J Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39510176' target='_blank'>Chen et al., 2024, J Biol Chem<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39345539' target='_blank'>Ashkin et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37410232' target='_blank'>Park et al., 2023, Methods Mol Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37070193' target='_blank'>Mayayo-Peralta et al., 2023, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33790356' target='_blank'>Heimbruch et al., 2021, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33712344' target='_blank'>Taber et al., Urol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30112445' target='_blank'>Black et al., Bladder Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29954776' target='_blank'>Lelo et al., Clin Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28627627' target='_blank'>Wang et al., Oncol Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28430577' target='_blank'>Benedetti et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24484537' target='_blank'>Evers et al., Genome Med, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38475971' target='_blank'>Wiltshire et al., 2024, Hum Genomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29558404' target='_blank'>Huang et al., Cancers (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28625132' target='_blank'>Kim et al., Curr Pharm Des, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38538643' target='_blank'>Marallano et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38132479' target='_blank'>Bahmad et al., 2023, Curr Issues Mol Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32388501' target='_blank'>Jiang et al., Aging (Albany NY), 2020<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37306343' target='_blank'>Wang et al., 2023, Acta Biochim Pol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35422093' target='_blank'>Kim et al., 2022, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34094986' target='_blank'>Zhang et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31173859' target='_blank'>Puhl et al., Brain Res Bull, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30628650' target='_blank'>Yang et al., Mol Med Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28086904' target='_blank'>Han et al., Mol Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26042665' target='_blank'>Conti et al., PLoS Negl Trop Dis, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24755295' target='_blank'>Ji et al., Mol Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24561119' target='_blank'>Zhang et al., Cancer Lett, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21642372' target='_blank'>Lambiv et al., 2011, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21403899' target='_blank'>Jones et al., Sarcoma, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18079202' target='_blank'>Elston et al., Endocrinology, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17384664' target='_blank'>Chan et al., Lab Invest, 2007<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39858006' target='_blank'>Mukherjee et al., 2025, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37917583' target='_blank'>Tan et al., 2023, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36770705' target='_blank'>Chuang et al., Molecules, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32471470' target='_blank'>Kim et al., 2020, Genome Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32457423' target='_blank'>Baek et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29246212' target='_blank'>Yan et al., BMC Pulm Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28713588' target='_blank'>Bi et al., 2017, NPJ Genom Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28680382' target='_blank'>Jalilvand et al., Comp Clin Path, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25958056' target='_blank'>Kriege et al., J Cancer Res Clin Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18071362' target='_blank'>Hallor et al., 2007, Br J Cancer<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39805658' target='_blank'>Fernandez et al., 2025, RNA<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38494261' target='_blank'>Singh et al., 2024, J Genet Eng Biotechnol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38476610' target='_blank'>Hong et al., 2024, Biomed Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33146632' target='_blank'>Fang et al., Aging (Albany NY), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31470644' target='_blank'>Lambrecht et al., Cells, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31185986' target='_blank'>Tordjman et al., BMC Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26497556' target='_blank'>Zou et al., Oncotarget, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39627815' target='_blank'>Vandenhoeck et al., 2024, Clin Epigenetics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32641685' target='_blank'>Li et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28877752' target='_blank'>Légaré et al., Breast Cancer Res, 2017<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36849460' target='_blank'>Xu et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31004081' target='_blank'>Chen et al., Exp Mol Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25088035' target='_blank'>Aly et al., Blood Cells Mol Dis, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21074269' target='_blank'>Qian et al., Leuk Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20704716' target='_blank'>Qian et al., J Exp Clin Cancer Res, 2010<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38693111' target='_blank'>Wang et al., 2024, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31173237' target='_blank'>Yin et al., Mol Med Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29844869' target='_blank'>Franceschi et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28782243' target='_blank'>Maraia et al., 2017, Wiley Interdiscip Rev RNA<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26933087' target='_blank'>Koso et al., Cancer Res, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33316386' target='_blank'>He et al., Cell Signal, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32796005' target='_blank'>Lopez et al., Genome Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30719075' target='_blank'>Li et al., Therap Adv Gastroenterol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29996478' target='_blank'>Lamar et al., Cancers (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29137410' target='_blank'>Yang et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26039411' target='_blank'>Thompson et al., PLoS One, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21757720' target='_blank'>Van et al., J Biol Chem, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21385898' target='_blank'>Henrich et al., Cancer Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15138581' target='_blank'>Nakatani et al., Int J Oncol, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12964007' target='_blank'>Katoh et al., Int J Oncol, 2003<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32634869' target='_blank'>Lever et al., Genomics Inform, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19734946' target='_blank'>Ong et al., Oncogene, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40666073' target='_blank'>Zhang et al., 2025, Front Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37932340' target='_blank'>Lee et al., 2023, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34389035' target='_blank'>Johansson et al., Thyroid Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33176626' target='_blank'>Ramírez-Moya et al., Thyroid, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32577859' target='_blank'>Petri et al., Cancer Metastasis Rev, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30967628' target='_blank'>Ramírez-Moya et al., Oncogene, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30545127' target='_blank'>Klinge et al., Noncoding RNA, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28915552' target='_blank'>Sun et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28529604' target='_blank'>Li et al., J Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26773048' target='_blank'>Constantin et al., Genetics, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26655797' target='_blank'>Shields et al., Mol Syst Biol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25558471' target='_blank'>Adams et al., Mol Cell Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24828205' target='_blank'>Geng et al., Int J Mol Sci, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23851498' target='_blank'>Zhang et al., Oncogene, 2014<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34299796' target='_blank'>Grant et al., 2021, Int J Environ Res Public Health<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32651197' target='_blank'>Oppel et al., Dis Model Mech, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31031847' target='_blank'>Xue et al., J Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28115373' target='_blank'>Girardi et al., Blood, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25119042' target='_blank'>De et al., Nature, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40241305' target='_blank'>Lubin et al., 2025, HGG Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37958378' target='_blank'>Sinha et al., 2023, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36069525' target='_blank'>Jingyao et al., Cancer Biol Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34464167' target='_blank'>Jiang et al., Technol Cancer Res Treat, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37046851' target='_blank'>Lucà et al., 2023, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33598459' target='_blank'>Ng et al., Front Cell Dev Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31614935' target='_blank'>Srivastava et al., Biomolecules, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29584698' target='_blank'>Monaco et al., Cancers (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29186807' target='_blank'>Fedele et al., Int J Mol Sci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29137300' target='_blank'>Iesato et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25595894' target='_blank'>Chiappetta et al., Oncotarget, 2015<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38658552' target='_blank'>Zhang et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38652219' target='_blank'>Yu et al., 2024, J Cell Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34446703' target='_blank'>Deng et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32117705' target='_blank'>Onagoruwa et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31235829' target='_blank'>Das et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30896861' target='_blank'>Guo et al., Oncol Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30776747' target='_blank'>Georgiadis et al., Environ Int, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30456386' target='_blank'>Furth et al., Life Sci Alliance, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30046387' target='_blank'>Mohamed et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30010037' target='_blank'>NA et al., 2018, Gene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28000896' target='_blank'>Liu et al., Int J Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24976335' target='_blank'>Luo et al., Lung Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24360964' target='_blank'>Li et al., Cell Rep, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21245096' target='_blank'>Murakami et al., Cancer Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19799973' target='_blank'>Visser et al., Gene, 2010<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40199870' target='_blank'>Schaeffeler et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38868222' target='_blank'>Meng et al., 2024, Imeta<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37989747' target='_blank'>Rao et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36106637' target='_blank'>Kaelin et al., J Clin Invest, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34002060' target='_blank'>Zhao et al., 2021, Nat Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33831375' target='_blank'>Dentro et al., Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33144689' target='_blank'>Jonasch et al., Nat Rev Nephrol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31727677' target='_blank'>Courtney et al., Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29777171' target='_blank'>Skucha et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28122867' target='_blank'>McKinney et al., Cancer Discov, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27614073' target='_blank'>Zhu et al., Nucleic Acids Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23622243' target='_blank'>Li et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22818162' target='_blank'>Koh et al., Trends Biochem Sci, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22266761' target='_blank'>Wagner et al., Nat Rev Mol Cell Biol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22099308' target='_blank'>Kuo et al., Mol Cell, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40345854' target='_blank'>Huang et al., 2025, Genes Dev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21533017' target='_blank'>Davis-Dusenbery et al., Genes Cancer, 2010<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39188305' target='_blank'>Kundu et al., 2024, Mol Ther Nucleic Acids<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36098557' target='_blank'>Sekirnik et al., ACS Chem Biol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32908919' target='_blank'>Xu et al., Biomed Res Int, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32731534' target='_blank'>McAvera et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32723561' target='_blank'>Zhu et al., Adv Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31153371' target='_blank'>Zhang et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25639486' target='_blank'>Xue et al., Nat Commun, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21987176' target='_blank'>Fiorentino et al., Arthritis Rheum, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38282294' target='_blank'>Lee et al., 2024, Am J Med Genet A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34021128' target='_blank'>Han et al., Cell Death Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33848981' target='_blank'>Peng et al., Aging (Albany NY), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32826873' target='_blank'>Liao et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32508881' target='_blank'>Miller et al., Front Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31763177' target='_blank'>Parisi et al., Transl Sci Rare Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31127025' target='_blank'>Xu et al., Biosci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29515120' target='_blank'>Infante et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25373737' target='_blank'>Liu et al., Oncotarget, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40504161' target='_blank'>Tien et al., 2025, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39368479' target='_blank'>Tien et al., 2024, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38562774' target='_blank'>Tien et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37811895' target='_blank'>Wang et al., 2023, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33868579' target='_blank'>Hussain Qureshi et al., 2021, Genes Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32570740' target='_blank'>Liang et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31652541' target='_blank'>Bartkowiak et al., Biomolecules, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29133620' target='_blank'>Naidoo et al., Mol Cancer Ther, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27900369' target='_blank'>Biswas et al., Cold Spring Harb Mol Case Stud, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26658019' target='_blank'>Juan et al., Cell Death Differ, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26189575' target='_blank'>Bartkowiak et al., Biochim Biophys Acta, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25712099' target='_blank'>Ekumi et al., Nucleic Acids Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25429106' target='_blank'>Bartkowiak et al., J Biol Chem, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40564017' target='_blank'>Rajakaruna et al., 2025, Biomedicines<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37727135' target='_blank'>Liu et al., 2023, Mol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35841413' target='_blank'>Conconi et al., World J Urol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33805984' target='_blank'>Song et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33691643' target='_blank'>Li et al., BMC Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33653800' target='_blank'>Brown et al., J Immunother Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30992440' target='_blank'>Zhang et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30755693' target='_blank'>Lee et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30301885' target='_blank'>Gara et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30195794' target='_blank'>Xu et al., Mol Ther Nucleic Acids, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29531238' target='_blank'>Fan et al., Sci Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25945796' target='_blank'>Cotterchio et al., PLoS One, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23521319' target='_blank'>Ni et al., Cancer Sci, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20095042' target='_blank'>Lu et al., Genes Chromosomes Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15126508' target='_blank'>Cam et al., J Biol Chem, 2004<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39236169' target='_blank'>Sundaram et al., 2024, Science<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38591846' target='_blank'>Xu et al., 2024, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38536906' target='_blank'>De Coninck et al., 2024, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36847506' target='_blank'>Gunn et al., 2023, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34413196' target='_blank'>Bensberg et al., Proc Natl Acad Sci U S A, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34164626' target='_blank'>Venanzi et al., Blood Cancer Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32740973' target='_blank'>Bai et al., Hepatology, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31076479' target='_blank'>Duy et al., Cancer Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30538121' target='_blank'>Bonvin et al., Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30274972' target='_blank'>Dominguez et al., Cancer Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30206308' target='_blank'>Kong et al., Leukemia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28813659' target='_blank'>Pan et al., Immunity, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25132561' target='_blank'>Huang et al., Trends Genet, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24439369' target='_blank'>Wu et al., Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23359070' target='_blank'>Lobry et al., J Exp Med, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40427516' target='_blank'>Miluzio et al., 2025, Biomolecules<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39279976' target='_blank'>Luu et al., 2024, J Gastrointest Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38502270' target='_blank'>Ye et al., 2024, Cell Oncol (Dordr)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33454654' target='_blank'>Ren et al., Bioorg Med Chem, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33145994' target='_blank'>Liudkovska et al., Wiley Interdiscip Rev RNA, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32528962' target='_blank'>Lin et al., Front Cell Dev Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32468032' target='_blank'>Zhang et al., Mol Med Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30573978' target='_blank'>Zhuang et al., Onco Targets Ther, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28341836' target='_blank'>Zhang et al., Sci Rep, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40374855' target='_blank'>Zhang et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40113912' target='_blank'>Rompietti et al., 2025, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34484175' target='_blank'>Dawes et al., Front Immunol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32820036' target='_blank'>Kutscher et al., Genes Dev, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31150281' target='_blank'>Astolfi et al., Epigenomics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31123059' target='_blank'>Bracken et al., Genes Dev, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38902237' target='_blank'>Martinez et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38425401' target='_blank'>Cheng et al., 2024, World J Gastrointest Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37470859' target='_blank'>Takahashi et al., 2023, J Gastroenterol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35975457' target='_blank'>Zheng et al., Adv Sci (Weinh), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34973294' target='_blank'>Hosein et al., Gastroenterology, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34215350' target='_blank'>Zhu et al., Oncol Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34149925' target='_blank'>Sohn et al., J Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33188943' target='_blank'>Neumeyer et al., 2020, Cell Mol Gastroenterol Hepatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30105857' target='_blank'>Fischer et al., J Pathol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28446252' target='_blank'>Gao et al., Stem Cell Res Ther, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26184844' target='_blank'>Niu et al., Cell Physiol Biochem, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25755738' target='_blank'>Xi et al., Int J Clin Exp Pathol, 2015<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40454484' target='_blank'>Kawai et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40454477' target='_blank'>Perelli et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40199870' target='_blank'>Schaeffeler et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40023154' target='_blank'>Jiang et al., 2025, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36808431' target='_blank'>De Silva et al., 2023, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36577755' target='_blank'>Shang et al., Signal Transduct Target Ther, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36261012' target='_blank'>Brooks et al., Cell Rep, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36106637' target='_blank'>Kaelin et al., J Clin Invest, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31863007' target='_blank'>Cai et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30055950' target='_blank'>Li et al., Semin Cancer Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28329682' target='_blank'>Nargund et al., Cell Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28007623' target='_blank'>Augert et al., J Thorac Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24487277' target='_blank'>Gerlinger et al., Nat Genet, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23832661' target='_blank'>Peña-Llopis et al., Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21248752' target='_blank'>Varela et al., Nature, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33155659' target='_blank'>Hu et al., Int J Mol Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28516957' target='_blank'>Micol et al., Nat Commun, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33658012' target='_blank'>Xu et al., Mol Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32059672' target='_blank'>Lei et al., Mol Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31195063' target='_blank'>Wang et al., J Hepatol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30086763' target='_blank'>Yeh et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28716020' target='_blank'>Wang et al., Mol Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28270163' target='_blank'>Wallaert et al., 2017, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26343386' target='_blank'>Shain et al., Nat Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25753158' target='_blank'>Ji et al., Cell Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24884509' target='_blank'>Tu et al., Mol Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24838835' target='_blank'>Aydin et al., J Natl Cancer Inst, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23857984' target='_blank'>Mansour et al., J Exp Med, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23827704' target='_blank'>Levens et al., 2013, Cell Stem Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23377281' target='_blank'>Tzoneva et al., 2013, Nat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20332316' target='_blank'>Williams et al., Clin Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17646408' target='_blank'>Thompson et al., J Exp Med, 2007<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39587824' target='_blank'>Li et al., 2024, Oral Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32874785' target='_blank'>Zhao et al., PeerJ, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32346611' target='_blank'>Jing et al., Mol Ther Oncolytics, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31624740' target='_blank'>Zhan et al., World J Clin Cases, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29075358' target='_blank'>Wang et al., Clin Epigenetics, 2017<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/32575117' target='_blank'>Flores et al., J Clin Endocrinol Metab, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29547888' target='_blank'>Deng et al., Hum Mol Genet, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24899893' target='_blank'>Martins et al., Int J Endocrinol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24334765' target='_blank'>Qin et al., Hum Mol Genet, 2014<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/33949667' target='_blank'>Deogharkar et al., 2021, Hum Mol Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30867762' target='_blank'>Cui et al., Oncol Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27600067' target='_blank'>Song et al., Microarrays (Basel), 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26904939' target='_blank'>Wu et al., Cell Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26554380' target='_blank'>Bao et al., PLoS One, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25789315' target='_blank'>Prasad et al., Biomed Res Int, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24570593' target='_blank'>Lin et al., Onco Targets Ther, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21757246' target='_blank'>Lizardi et al., Trends Biotechnol, 2011<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39836524' target='_blank'>Huang et al., 2025, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38036730' target='_blank'>Aziz et al., 2023, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37261974' target='_blank'>Zhu et al., 2023, Elife<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37142882' target='_blank'>Xu et al., 2023, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36933062' target='_blank'>Simigdala et al., 2023, Cell Mol Life Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35449309' target='_blank'>Na et al., 2022, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34075047' target='_blank'>Luo et al., NPJ Breast Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33596429' target='_blank'>Chen et al., Cell Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33159839' target='_blank'>Mastoraki et al., Mol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31044091' target='_blank'>Wu et al., Signal Transduct Target Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29755131' target='_blank'>Gala et al., Oncogene, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29474905' target='_blank'>Hoshii et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26040603' target='_blank'>Sánchez-Rivera et al., Nat Rev Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24794707' target='_blank'>Chen et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23416985' target='_blank'>Yachida et al., Oncogene, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/26879601' target='_blank'>Yin et al., BMC Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24896186' target='_blank'>Wang et al., Nature, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21989985' target='_blank'>Li et al., Blood, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39359687' target='_blank'>Scordino et al., 2024, Front Mol Neurosci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38813729' target='_blank'>Meyers et al., 2024, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36376973' target='_blank'>Johansson et al., Clin Epigenetics, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35421541' target='_blank'>Permatasari et al., Exp Hematol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33632709' target='_blank'>Huyghe et al., Gut, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33093445' target='_blank'>Yang et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32591643' target='_blank'>Loontiens et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30760869' target='_blank'>McNeer et al., Leukemia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27586877' target='_blank'>Inoue et al., Sci Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26053494' target='_blank'>Melton et al., Nat Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25827435' target='_blank'>Sakamaki et al., Carcinogenesis, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25613934' target='_blank'>Takachi et al., Cancer Sci, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23663453' target='_blank'>Go et al., Cancer Sci, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23440410' target='_blank'>Yui et al., 2013, J Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20972433' target='_blank'>De et al., Nat Med, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35675801' target='_blank'>Qi et al., Cytogenet Genome Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35549643' target='_blank'>Li et al., Bioengineered, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28527972' target='_blank'>Galani et al., Clin Neurol Neurosurg, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23912455' target='_blank'>Dutruel et al., Oncogene, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21733846' target='_blank'>Rinehart et al., J Biol Chem, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38701508' target='_blank'>Wang et al., 2024, Int J Surg<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33811920' target='_blank'>Francette et al., J Mol Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28476229' target='_blank'>Simonds et al., Endocrinol Metab Clin North Am, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25113791' target='_blank'>Chiofalo et al., Cell Oncol (Dordr), 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24035866' target='_blank'>Costa-Guda et al., Mol Cell Endocrinol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23579270' target='_blank'>Yang et al., Cell Death Dis, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22297294' target='_blank'>Zhang et al., Cell Death Dis, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20178782' target='_blank'>Nguyen et al., Dev Biol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19906718' target='_blank'>Yang et al., Nucleic Acids Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19471023' target='_blank'>James et al., Sci Signal, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19136632' target='_blank'>Rozenblatt-Rosen et al., Proc Natl Acad Sci U S A, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17404568' target='_blank'>Iwata et al., 2007, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16116486' target='_blank'>Hahn et al., Oncogene, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15580289' target='_blank'>Woodard et al., 2004, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12960210' target='_blank'>Howell et al., J Med Genet, 2003<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38055646' target='_blank'>Huang et al., 2023, Hepatol Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37735184' target='_blank'>Li et al., 2023, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37466203' target='_blank'>Zhuang et al., 2023, Cancer Commun (Lond)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34116210' target='_blank'>Wang et al., Pharmacol Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33854336' target='_blank'>Wang et al., Onco Targets Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33836681' target='_blank'>Zhuang et al., BMC Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33172072' target='_blank'>Md et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32575903' target='_blank'>Urman et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29299064' target='_blank'>Čopar et al., BioData Min, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26633557' target='_blank'>Cai et al., Br J Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25679223' target='_blank'>Ito et al., PLoS One, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23685749' target='_blank'>Ho et al., Nat Genet, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21764761' target='_blank'>Berndt et al., Cancer Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20951342' target='_blank'>Pan et al., 2010, Dev Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19691111' target='_blank'>Camps et al., Genes Chromosomes Cancer, 2009<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/33139182' target='_blank'>Nepomuceno et al., Trends Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32508881' target='_blank'>Miller et al., Front Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31877165' target='_blank'>Park et al., PLoS One, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29190888' target='_blank'>Kim et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23650262' target='_blank'>Huo et al., Cancer Discov, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35892871' target='_blank'>Ancos-Pintado et al., Cancers (Basel), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34472720' target='_blank'>Yoshida et al., Cancer Rep (Hoboken), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33024076' target='_blank'>Schieber et al., Blood Cancer J, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30827673' target='_blank'>Choi et al., Cell Immunol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26433073' target='_blank'>Wu et al., Semin Cancer Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26116372' target='_blank'>Melnik et al., J Transl Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25510281' target='_blank'>Yang et al., Immunol Rev, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25203322' target='_blank'>Zheng et al., Cancer Cell, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40599235' target='_blank'>Liang et al., 2025, MedComm (2020)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39188305' target='_blank'>Kundu et al., 2024, Mol Ther Nucleic Acids<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37020036' target='_blank'>Yang et al., 2023, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33878446' target='_blank'>Sun et al., Pharmacol Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32740973' target='_blank'>Bai et al., Hepatology, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31399111' target='_blank'>Wu et al., J Exp Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29891505' target='_blank'>Good et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28249902' target='_blank'>Tie et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27346347' target='_blank'>Forloni et al., Cell Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26973719' target='_blank'>Barazeghi et al., Clin Epigenetics, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26816554' target='_blank'>Huang et al., Clin Epigenetics, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26791235' target='_blank'>Huang et al., Cancer Lett, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26194287' target='_blank'>Cimmino et al., Nat Immunol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25867473' target='_blank'>Cimmino et al., Nat Immunol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25362856' target='_blank'>Neri et al., Oncogene, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32508881' target='_blank'>Miller et al., Front Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32231423' target='_blank'>Voutsadakis et al., World J Gastroenterol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31064327' target='_blank'>Ali et al., BMC Med Genet, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29121674' target='_blank'>Frye et al., PLoS One, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28968953' target='_blank'>Oussalah et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28382101' target='_blank'>Venkateshwari et al., 2017, J Breast Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26870246' target='_blank'>Zou et al., Oncol Lett, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23276657' target='_blank'>Suhasini et al., 2013, Mutat Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21964575' target='_blank'>Rafnar et al., Nat Genet, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21345144' target='_blank'>Cantor et al., Future Oncol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19099189' target='_blank'>Wu et al., Cell Mol Life Sci, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38097536' target='_blank'>Chen et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32644293' target='_blank'>Yang et al., EMBO Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31197957' target='_blank'>Zhang et al., 2019, Kaohsiung J Med Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28938551' target='_blank'>Zou et al., Oncotarget, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33367931' target='_blank'>Sun et al., Int J Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27822070' target='_blank'>Yu et al., Onco Targets Ther, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40730747' target='_blank'>Marchi et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37845209' target='_blank'>Ye et al., 2023, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33167358' target='_blank'>Dunn et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31470906' target='_blank'>Jungwirth et al., Acta Neuropathol Commun, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38813086' target='_blank'>Fan et al., 2024, Front Cell Dev Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38812997' target='_blank'>Erdoğan et al., 2024, Turk J Med Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37393343' target='_blank'>Hou et al., 2023, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36756714' target='_blank'>Wang et al., J Cell Mol Med, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35503998' target='_blank'>Li et al., Aging (Albany NY), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32528236' target='_blank'>Zhang et al., Dose Response, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30888215' target='_blank'>Yu et al., Cancer Biother Radiopharm, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30842058' target='_blank'>Teachey et al., Lancet Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30118183' target='_blank'>Liu et al., Wiley Interdiscip Rev RNA, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27347093' target='_blank'>Li et al., 2016, Oncol Lett<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26561469' target='_blank'>Hajjari et al., Tumour Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24895337' target='_blank'>Mets et al., Haematologica, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22515449' target='_blank'>Liu et al., Future Oncol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21030981' target='_blank'>Van et al., Leukemia, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20972433' target='_blank'>De et al., Nat Med, 2010<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/33718179' target='_blank'>Peng et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24998916' target='_blank'>Lindström et al., Eur J Histochem, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19200647' target='_blank'>Cai et al., Cancer Lett, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35399726' target='_blank'>Liu et al., J Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33032635' target='_blank'>Li et al., J Exp Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31097977' target='_blank'>Panieri et al., Oxid Med Cell Longev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30631060' target='_blank'>Zhu et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29498662' target='_blank'>Perochon et al., 2018, Genes (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28367074' target='_blank'>Torquato et al., 2017, Curr Genomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26071483' target='_blank'>Sanz-Pamplona et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24931005' target='_blank'>McCubrey et al., Oncotarget, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24459086' target='_blank'>Fujita et al., Am J Med Genet A, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23778311' target='_blank'>McCubrey et al., Leukemia, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22405270' target='_blank'>Dai et al., Mol Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22285752' target='_blank'>Kim et al., Mol Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21571217' target='_blank'>Moisan et al., Dev Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19416806' target='_blank'>Rivera et al., Proc Natl Acad Sci U S A, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17510365' target='_blank'>Major et al., Science, 2007<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39361730' target='_blank'>Kemps et al., 2024, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34194626' target='_blank'>Dhar et al., Oncotarget, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32399015' target='_blank'>Imataki et al., Case Rep Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28701722' target='_blank'>Youn et al., Sci Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25000259' target='_blank'>Belani et al., Oncogenesis, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39657751' target='_blank'>Sun et al., 2024, Clin Mol Hepatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37932340' target='_blank'>Lee et al., 2023, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34272401' target='_blank'>Chang et al., NPJ Genom Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33805113' target='_blank'>Nidhi et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33786634' target='_blank'>Liu et al., Oncol Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33742126' target='_blank'>Moreno et al., Oncogene, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33262464' target='_blank'>Bala et al., Oncogene, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31480474' target='_blank'>Roszik et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30519319' target='_blank'>Wang et al., J Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29396555' target='_blank'>Li et al., Cell Death Dis, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28498550' target='_blank'>Gao et al., Cancer Sci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26169693' target='_blank'>Yu et al., Exp Cell Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25299188' target='_blank'>He et al., PLoS One, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23525077' target='_blank'>Dulak et al., Nat Genet, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22842228' target='_blank'>Krauthammer et al., Nat Genet, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37490463' target='_blank'>Namba et al., 2023, PLoS One<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34990909' target='_blank'>Haynes et al., Transl Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34805795' target='_blank'>Calvo et al., iScience, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34779410' target='_blank'>El-Houjeiri et al., 2021, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33988680' target='_blank'>Bischoff et al., J Cell Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29767721' target='_blank'>Hasumi et al., Hum Mol Genet, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28039480' target='_blank'>Nagashima et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27113757' target='_blank'>Starling et al., EMBO Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26439621' target='_blank'>Possik et al., PLoS Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25775561' target='_blank'>Hasumi et al., Proc Natl Acad Sci U S A, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23108783' target='_blank'>Schmidt et al., Fam Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22977732' target='_blank'>Nookala et al., Open Biol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22709692' target='_blank'>Baba et al., Blood, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20573232' target='_blank'>Hong et al., Mol Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19850877' target='_blank'>Hasumi et al., Proc Natl Acad Sci U S A, 2009<\/a>",null,null],["This gene encodes an activin A type IB receptor. Activins are dimeric growth and differentiation factors which belong to the transforming growth factor-beta (TGF-beta) superfamily of structurally related signaling proteins. Activins signal through a heteromeric complex of receptor serine kinases which include at least two type I and two type II receptors. This protein is a type I receptor which is essential for signaling. Mutations in this gene are associated with pituitary tumors. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Jun 2010].","This gene encodes a receptor that mediates the functions of activins, which are members of the transforming growth factor-beta (TGF-beta) superfamily involved in diverse biological processes. The encoded protein is a transmembrane serine-threonine kinase receptor which mediates signaling by forming heterodimeric complexes with various combinations of type I and type II receptors and ligands in a cell-specific manner. The encoded type II receptor is primarily involved in ligand-binding and includes an extracellular ligand-binding domain, a transmembrane domain and a cytoplasmic serine-threonine kinase domain. This gene may be associated with susceptibility to preeclampsia, a pregnancy-related disease which can result in maternal and fetal morbidity and mortality. Alternative splicing results in multiple transcript variants of this gene.","This gene encodes a tumor suppressor protein that acts as an antagonist of the Wnt signaling pathway. It is also involved in other processes including cell migration and adhesion, transcriptional activation, and apoptosis. Defects in this gene cause familial adenomatous polyposis (FAP), an autosomal dominant pre-malignant disease that usually progresses to malignancy. Mutations in the APC gene have been found to occur in most colorectal cancers, where disease-associated mutations tend to be clustered in a small region designated the mutation cluster region (MCR) and result in a truncated protein product.","This gene encodes a member of the IAP family of proteins that inhibit apoptosis by binding to tumor necrosis factor receptor-associated factors TRAF1 and TRAF2, probably by interfering with activation of ICE-like proteases. The encoded protein inhibits apoptosis induced by serum deprivation but does not affect apoptosis resulting from exposure to menadione, a potent inducer of free radicals. It contains 3 baculovirus IAP repeats and a ring finger domain. Transcript variants encoding the same isoform have been identified.","The protein encoded by this gene is a member of the TNF-receptor superfamily. This receptor contains a death domain. It has been shown to play a central role in the physiological regulation of programmed cell death, and has been implicated in the pathogenesis of various malignancies and diseases of the immune system. The interaction of this receptor with its ligand allows the formation of a death-inducing signaling complex that includes Fas-associated death domain protein (FADD), caspase 8, and caspase 10. The autoproteolytic processing of the caspases in the complex triggers a downstream caspase cascade, and leads to apoptosis. This receptor has been also shown to activate NF-kappaB, MAPK3/ERK1, and MAPK8/JNK, and is found to be involved in transducing the proliferating signals in normal diploid fibroblast and T cells. Several alternatively spliced transcript variants have been described, some of which are candidates for nonsense-mediated mRNA decay (NMD). The isoforms lacking the transmembrane domain may negatively regulate the apoptosis mediated by the full length isoform.","This gene encodes a member of the Rho family of small GTPases, which cycle between inactive GDP-bound and active GTP-bound states and function as molecular switches in signal transduction cascades. Rho proteins promote reorganization of the actin cytoskeleton and regulate cell shape, attachment, and motility. Overexpression of this gene is associated with tumor cell proliferation and metastasis. Multiple alternatively spliced variants have been identified.","The protein encoded by this gene is a member of the Ras superfamily of guanosine triphosphate (GTP)-metabolizing enzymes. The encoded protein is expressed in hematopoietic cells, where it functions as a negative regulator of cell growth and survival. This gene may be hypermutated or misexpressed in leukemias and lymphomas. Chromosomal translocations in non-Hodgkin's lymphoma occur between this locus and B-cell CLL/lymphoma 6 (BCL6) on chromosome 3, leading to the production of fusion transcripts. Alternative splicing in the 5' untranslated region results in multiple transcript variants that encode the same protein.","This gene encodes a protein containing a basic helix-loop-helix domain and two characteristic PAS domains along with a PAC domain. The encoded protein binds to ligand-bound aryl hydrocarbon receptor and aids in the movement of this complex to the nucleus, where it promotes the expression of genes involved in xenobiotic metabolism. This protein is also a co-factor for transcriptional regulation by hypoxia-inducible factor 1. Chromosomal translocation of this locus with the ETV6 (ets variant 6) gene on chromosome 12 have been described in leukemias. Alternative splicing results in multiple transcript variants.","This gene encodes a transcription factor with multiple homeodomains and zinc finger motifs, and regulates myogenic and neuronal differentiation. The encoded protein suppresses expression of the alpha-fetoprotein gene by binding to an AT-rich enhancer motif. The protein has also been shown to negatively regulate c-Myb, and transactivate the cell cycle inhibitor cyclin-dependent kinase inhibitor 1A (also known as p21CIP1). This gene is reported to function as a tumor suppressor in several cancers, and sequence variants of this gene are also associated with atrial fibrillation. Multiple transcript variants expressed from alternate promoters and encoding different isoforms have been found for this gene.","The protein encoded by this gene belongs to the PI3/PI4-kinase family. This protein is an important cell cycle checkpoint kinase that phosphorylates; thus, it functions as a regulator of a wide variety of downstream proteins, including tumor suppressor proteins p53 and BRCA1, checkpoint kinase CHK2, checkpoint proteins RAD17 and RAD9, and DNA repair protein NBS1. This protein and the closely related kinase ATR are thought to be master controllers of cell cycle checkpoint signaling pathways that are required for cell response to DNA damage and for genome stability. Mutations in this gene are associated with ataxia telangiectasia, an autosomal recessive disorder.","The protein encoded by this gene belongs to the family of P-type cation transport ATPases, and to the subfamily of Na+/K+ -ATPases. Na+/K+ -ATPase is an integral membrane protein responsible for establishing and maintaining the electrochemical gradients of Na and K ions across the plasma membrane. These gradients are essential for osmoregulation, for sodium-coupled transport of a variety of organic and inorganic molecules, and for electrical excitability of nerve and muscle. This enzyme is composed of two subunits, a large catalytic subunit (alpha) and a smaller glycoprotein subunit (beta). The catalytic subunit of Na+/K+ -ATPase is encoded by multiple genes. This gene encodes an alpha 1 subunit. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene belongs to the family of P-type primary ion transport ATPases characterized by the formation of an aspartyl phosphate intermediate during the reaction cycle. These enzymes remove bivalent calcium ions from eukaryotic cells against very large concentration gradients and play a critical role in intracellular calcium homeostasis. The mammalian plasma membrane calcium ATPase isoforms are encoded by at least four separate genes and the diversity of these enzymes is further increased by alternative splicing of transcripts. The expression of different isoforms and splice variants is regulated in a developmental, tissue- and cell type-specific manner, suggesting that these pumps are functionally adapted to the physiological needs of particular cells and tissues. This gene encodes the plasma membrane calcium ATPase isoform 3. Alternatively spliced transcript variants encoding different isoforms have been identified.","The protein encoded by this gene is a serine/threonine kinase and DNA damage sensor, activating cell cycle checkpoint signaling upon DNA stress. The encoded protein can phosphorylate and activate several proteins involved in the inhibition of DNA replication and mitosis, and can promote DNA repair, recombination, and apoptosis. This protein is also important for fragile site stability and centrosome duplication. Defects in this gene are a cause of Seckel syndrome 1.","The protein encoded by this gene contains an ATPase/helicase domain, and thus it belongs to the SWI/SNF family of chromatin remodeling proteins. This protein is found to undergo cell cycle-dependent phosphorylation, which regulates its nuclear matrix and chromatin association, and suggests its involvement in the gene regulation at interphase and chromosomal segregation in mitosis. Mutations in this gene are associated with X-linked syndromes exhibiting cognitive disabilities as well as alpha-thalassemia (ATRX) syndrome. These mutations have been shown to cause diverse changes in the pattern of DNA methylation, which may provide a link between chromatin remodeling, DNA methylation, and gene expression in developmental processes. Multiple alternatively spliced transcript variants encoding distinct isoforms have been reported.","This gene encodes a serum protein found in association with the major histocompatibility complex (MHC) class I heavy chain on the surface of nearly all nucleated cells. The protein has a predominantly beta-pleated sheet structure that can form amyloid fibrils in some pathological conditions. The encoded antimicrobial protein displays antibacterial activity in amniotic fluid. A mutation in this gene has been shown to result in hypercatabolic hypoproteinemia.[provided by RefSeq, Aug 2014].","This gene encodes a protein which interacts with the N-terminal region of BRCA1. In addition to its ability to bind BRCA1 in vivo and in vitro, it shares homology with the 2 most conserved regions of BRCA1: the N-terminal RING motif and the C-terminal BRCT domain. The RING motif is a cysteine-rich sequence found in a variety of proteins that regulate cell growth, including the products of tumor suppressor genes and dominant protooncogenes. This protein also contains 3 tandem ankyrin repeats. The BARD1/BRCA1 interaction is disrupted by tumorigenic amino acid substitutions in BRCA1, implying that the formation of a stable complex between these proteins may be an essential aspect of BRCA1 tumor suppression. This protein may be the target of oncogenic mutations in breast or ovarian cancer. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene belongs to the BCL2 protein family. BCL2 family members form hetero- or homodimers and act as anti- or pro-apoptotic regulators that are involved in a wide variety of cellular activities. This protein forms a heterodimer with BCL2, and functions as an apoptotic activator. The association and the ratio of BAX to BCL2 also determines survival or death of a cell following an apoptotic stimulus. This protein is reported to interact with, and increase the opening of, the mitochondrial voltage-dependent anion channel (VDAC), which leads to the loss in membrane potential and the release of cytochrome c. The expression of this gene is regulated by the tumor suppressor P53 and has been shown to be involved in P53-mediated apoptosis. Multiple alternatively spliced transcript variants, which encode different isoforms, have been reported for this gene.","This gene encodes a protein that acts as a repressor of beta-interferon gene expression. The protein binds specifically to the PRDI (positive regulatory domain I element) of the beta-IFN gene promoter. Transcription of this gene increases upon virus induction. Two alternatively spliced transcript variants that encode different isoforms have been reported.","The Bloom syndrome is an autosomal recessive disorder characterized by growth deficiency, microcephaly and immunodeficiency among others. It is caused by homozygous or compound heterozygous mutation in the gene encoding DNA helicase RecQ protein on chromosome 15q26. This Bloom-associated helicase unwinds a variety of DNA substrates including Holliday junction, and is involved in several pathways contributing to the maintenance of genome stability. Identification of pathogenic Bloom variants is required for heterozygote testing in at-risk families.","The bone morphogenetic protein (BMP) receptors are a family of transmembrane serine/threonine kinases that include the type I receptors BMPR1A and BMPR1B and the type II receptor BMPR2. These receptors are also closely related to the activin receptors, ACVR1 and ACVR2. The ligands of these receptors are members of the TGF-beta superfamily. TGF-betas and activins transduce their signals through the formation of heteromeric complexes with 2 different types of serine (threonine) kinase receptors: type I receptors of about 50-55 kD and type II receptors of about 70-80 kD. Type II receptors bind ligands in the absence of type I receptors, but they require their respective type I receptors for signaling, whereas type I receptors require their respective type II receptors for ligand binding.","This gene encodes a forkhead transcription factor. The protein contains a fork-head DNA-binding domain and may play a role in ovarian development and function. Expansion of a polyalanine repeat region and other mutations in this gene are a cause of blepharophimosis syndrome and premature ovarian failure 3.","This gene encodes a 190 kD nuclear phosphoprotein that plays a role in maintaining genomic stability, and it also acts as a tumor suppressor. The BRCA1 gene contains 22 exons spanning about 110 kb of DNA. The encoded protein combines with other tumor suppressors, DNA damage sensors, and signal transducers to form a large multi-subunit protein complex known as the BRCA1-associated genome surveillance complex (BASC). This gene product associates with RNA polymerase II, and through the C-terminal domain, also interacts with histone deacetylase complexes. This protein thus plays a role in transcription, DNA repair of double-stranded breaks, and recombination. Mutations in this gene are responsible for approximately 40% of inherited breast cancers and more than 80% of inherited breast and ovarian cancers. Alternative splicing plays a role in modulating the subcellular localization and physiological function of this gene. Many alternatively spliced transcript variants, some of which are disease-associated mutations, have been described for this gene, but the full-length natures of only some of these variants has been described. A related pseudogene, which is also located on chromosome 17, has been identified.","Inherited mutations in BRCA1 and this gene, BRCA2, confer increased lifetime risk of developing breast or ovarian cancer. Both BRCA1 and BRCA2 are involved in maintenance of genome stability, specifically the homologous recombination pathway for double-strand DNA repair. The largest exon in both genes is exon 11, which harbors the most important and frequent mutations in breast cancer patients. The BRCA2 gene was found on chromosome 13q12.3 in human. The BRCA2 protein contains several copies of a 70 aa motif called the BRC motif, and these motifs mediate binding to the RAD51 recombinase which functions in DNA repair. BRCA2 is considered a tumor suppressor gene, as tumors with BRCA2 mutations generally exhibit loss of heterozygosity (LOH) of the wild-type allele.","This gene is a member of an anti-proliferative gene family that regulates cell growth and differentiation. Expression of this gene is highest in the G0/G1 phases of the cell cycle and downregulated when cells progressed through G1. The encoded protein interacts with several nuclear receptors, and functions as a coactivator of cell differentiation. This locus has been shown to be involved in a t(8;12)(q24;q22) chromosomal translocation in a case of B-cell chronic lymphocytic leukemia.","The protein encoded by this gene plays a crucial role in B-cell development. Mutations in this gene cause X-linked agammaglobulinemia type 1, which is an immunodeficiency characterized by the failure to produce mature B lymphocytes, and associated with a failure of Ig heavy chain rearrangement. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a kinase involved in spindle checkpoint function. The protein has been localized to the kinetochore and plays a role in the inhibition of the anaphase-promoting complex/cyclosome (APC/C), delaying the onset of anaphase and ensuring proper chromosome segregation. Impaired spindle checkpoint function has been found in many forms of cancer.","This gene encodes a class 1 aminoacyl-tRNA synthetase, cysteinyl-tRNA synthetase. Each of the twenty aminoacyl-tRNA synthetases catalyzes the aminoacylation of a specific tRNA or tRNA isoaccepting family with the cognate amino acid. This gene is one of several located near the imprinted gene domain on chromosome 11p15.5, an important tumor-suppressor gene region. Alterations in this region have been associated with Beckwith-Wiedemann syndrome, Wilms tumor, rhabdomyosarcoma, adrenocortical carcinoma, and lung, ovarian and breast cancers. Alternative splicing of this gene results in multiple transcript variants.","The protein encoded by this gene is a cysteine-aspartic acid protease that plays a central role in the execution-phase of cell apoptosis. The encoded protein cleaves and inactivates poly(ADP-ribose) polymerase while it cleaves and activates sterol regulatory element binding proteins as well as caspases 6, 7, and 9. This protein itself is processed by caspases 8, 9, and 10. It is the predominant caspase involved in the cleavage of amyloid-beta 4A precursor protein, which is associated with neuronal death in Alzheimer's disease.","This gene encodes a member of the cysteine-aspartic acid protease (caspase) family. Sequential activation of caspases plays a central role in the execution-phase of cell apoptosis. Caspases exist as inactive proenzymes composed of a prodomain, a large protease subunit, and a small protease subunit. Activation of caspases requires proteolytic processing at conserved internal aspartic residues to generate a heterodimeric enzyme consisting of the large and small subunits. This protein is involved in the programmed cell death induced by Fas and various apoptotic stimuli. The N-terminal FADD-like death effector domain of this protein suggests that it may interact with Fas-interacting protein FADD. This protein was detected in the insoluble fraction of the affected brain region from Huntington disease patients but not in those from normal controls, which implicated the role in neurodegenerative diseases. Many alternatively spliced transcript variants encoding different isoforms have been described, although not all variants have had their full-length sequences determined.","This gene encodes a member of the cysteine-aspartic acid protease (caspase) family. Sequential activation of caspases plays a central role in the execution-phase of cell apoptosis. Caspases exist as inactive proenzymes which undergo proteolytic processing at conserved aspartic residues to produce two subunits, large and small, that dimerize to form the active enzyme. This protein can undergo autoproteolytic processing and activation by the apoptosome, a protein complex of cytochrome c and the apoptotic peptidase activating factor 1; this step is thought to be one of the earliest in the caspase activation cascade. This protein is thought to play a central role in apoptosis and to be a tumor suppressor. Alternative splicing results in multiple transcript variants.","Core binding factor (CBF) is a heterodimeric transcription factor that binds to the core element of many enhancers and promoters. The protein encoded by this gene represents the alpha subunit of CBF and is thought to be involved in the development of normal hematopoiesis. Chromosomal translocations involving this gene are well-documented and have been associated with several types of leukemia. Three transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the myeloid translocation gene family which interact with DNA-bound transcription factors and recruit a range of corepressors to facilitate transcriptional repression. The t(8;21)(q22;q22) translocation is one of the most frequent karyotypic abnormalities in acute myeloid leukemia. The translocation produces a chimeric gene made up of the 5'-region of the runt-related transcription factor 1 gene fused to the 3'-region of this gene. The chimeric protein is thought to associate with the nuclear corepressor/histone deacetylase complex to block hematopoietic differentiation. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the myeloid translocation gene family which interact with DNA-bound transcription factors and recruit a range of corepressors to facilitate transcriptional repression. The t(16;21)(q24;q22) translocation is one of the less common karyotypic abnormalities in acute myeloid leukemia. The translocation produces a chimeric gene made up of the 5'-region of the runt-related transcription factor 1 gene fused to the 3'-region of this gene. This gene is also a putative breast tumor suppressor. Alternative splicing results in transcript variants.","The protein encoded by this gene is the beta subunit of a heterodimeric core-binding transcription factor belonging to the PEBP2/CBF transcription factor family which master-regulates a host of genes specific to hematopoiesis (e.g., RUNX1) and osteogenesis (e.g., RUNX2). The beta subunit is a non-DNA binding regulatory subunit; it allosterically enhances DNA binding by alpha subunit as the complex binds to the core site of various enhancers and promoters, including murine leukemia virus, polyomavirus enhancer, T-cell receptor enhancers and GM-CSF promoters. Alternative splicing generates two mRNA variants, each encoding a distinct carboxyl terminus. In some cases, a pericentric inversion of chromosome 16 [inv(16)(p13q22)] produces a chimeric transcript consisting of the N terminus of core-binding factor beta in a fusion with the C-terminal portion of the smooth muscle myosin heavy chain 11. This chromosomal rearrangement is associated with acute myeloid leukemia of the M4Eo subtype. Two transcript variants encoding different isoforms have been found for this gene.","This gene is a proto-oncogene that encodes a RING finger E3 ubiquitin ligase. The encoded protein is one of the enzymes required for targeting substrates for degradation by the proteasome. This protein mediates the transfer of ubiquitin from ubiquitin conjugating enzymes (E2) to specific substrates. This protein also contains an N-terminal phosphotyrosine binding domain that allows it to interact with numerous tyrosine-phosphorylated substrates and target them for proteasome degradation. As such it functions as a negative regulator of many signal transduction pathways. This gene has been found to be mutated or translocated in many cancers including acute myeloid leukaemia, and expansion of CGG repeats in the 5' UTR has been associated with Jacobsen syndrome. Mutations in this gene are also the cause of Noonan syndrome-like disorder.","This gene encodes an E3 ubiquitin-protein ligase which promotes proteosome-mediated protein degradation by transferring ubiquitin from an E2 ubiquitin-conjugating enzyme to a substrate. The encoded protein is involved in the regulation of immune response by limiting T-cell receptor, B-cell receptor, and high affinity immunoglobulin epsilon receptor activation. Studies in mouse suggest that this gene is involved in antifungal host defense and that its inhibition leads to increased fungal killing. Manipulation of this gene may be beneficial in implementing immunotherapies for a variety of conditions, including cancer, autoimmune diseases, allergies, and infections.","The protein encoded by this gene is a member of the cyclin family of proteins. The encoded protein interacts with cyclin-dependent kinase 8 and induces the phophorylation of the carboxy-terminal domain of the large subunit of RNA polymerase II. The level of mRNAs for this gene peaks in the G1 phase of the cell cycle. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a classical cadherin of the cadherin superfamily. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate the mature glycoprotein. This calcium-dependent cell-cell adhesion protein is comprised of five extracellular cadherin repeats, a transmembrane region and a highly conserved cytoplasmic tail. Mutations in this gene are correlated with gastric, breast, colorectal, thyroid and ovarian cancer. Loss of function of this gene is thought to contribute to cancer progression by increasing proliferation, invasion, and/or metastasis. The ectodomain of this protein mediates bacterial adhesion to mammalian cells and the cytoplasmic domain is required for internalization. This gene is present in a gene cluster with other members of the cadherin family on chromosome 16.","This gene encodes a type II classical cadherin of the cadherin superfamily. Alternative splicing of this gene results in multiple transcript variants. At least one of these variants encodes a preproprotein that is proteolytically processed to generate the mature cadherin protein. These integral membrane proteins mediate calcium-dependent cell-cell adhesion and are composed of a large N-terminal extracellular domain, a single membrane-spanning domain, and a small, highly conserved C-terminal cytoplasmic domain. The extracellular domain consists of 5 subdomains, each containing a cadherin motif, and appears to determine the specificity of the protein's homophilic cell adhesion activity. Type II (atypical) cadherins are defined based on their lack of a histidine-alanine-valine (HAV) cell adhesion recognition sequence specific to type I cadherins. This particular cadherin is predominantly expressed in brain and is putatively involved in synaptic adhesions, axon outgrowth and guidance. Mutations in this gene may be associated with lung squamous cell carcinoma and colorectal cancer in human patients.","This gene encodes a type II classical cadherin from the cadherin superfamily, integral membrane proteins that mediate calcium-dependent cell-cell adhesion. Mature cadherin proteins are composed of a large N-terminal extracellular domain, a single membrane-spanning domain, and a small, highly conserved C-terminal cytoplasmic domain. Type II (atypical) cadherins are defined based on their lack of a HAV cell adhesion recognition sequence specific to type I cadherins. Expression of this particular cadherin in osteoblastic cell lines, and its upregulation during differentiation, suggests a specific function in bone development and maintenance.","This gene encodes a potent cyclin-dependent kinase inhibitor. The encoded protein binds to and inhibits the activity of cyclin-cyclin-dependent kinase2 or -cyclin-dependent kinase4 complexes, and thus functions as a regulator of cell cycle progression at G1. The expression of this gene is tightly controlled by the tumor suppressor protein p53, through which this protein mediates the p53-dependent cell cycle G1 phase arrest in response to a variety of stress stimuli. This protein can interact with proliferating cell nuclear antigen, a DNA polymerase accessory factor, and plays a regulatory role in S phase DNA replication and DNA damage repair. This protein was reported to be specifically cleaved by CASP3-like caspases, which thus leads to a dramatic activation of cyclin-dependent kinase2, and may be instrumental in the execution of apoptosis following caspase activation. Mice that lack this gene have the ability to regenerate damaged or missing tissue. Multiple alternatively spliced variants have been found for this gene.","This gene encodes a cyclin-dependent kinase inhibitor, which shares a limited similarity with CDK inhibitor CDKN1A/p21. The encoded protein binds to and prevents the activation of cyclin E-CDK2 or cyclin D-CDK4 complexes, and thus controls the cell cycle progression at G1. The degradation of this protein, which is triggered by its CDK dependent phosphorylation and subsequent ubiquitination by SCF complexes, is required for the cellular transition from quiescence to the proliferative state. Mutations in this gene are associated with multiple endocrine neoplasia type IV (MEN4).","This gene generates several transcript variants which differ in their first exons. At least three alternatively spliced variants encoding distinct proteins have been reported, two of which encode structurally related isoforms known to function as inhibitors of CDK4 kinase. The remaining transcript includes an alternate first exon located 20 Kb upstream of the remainder of the gene; this transcript contains an alternate open reading frame (ARF) that specifies a protein which is structurally unrelated to the products of the other variants. This ARF product functions as a stabilizer of the tumor suppressor protein p53 as it can interact with, and sequester, the E3 ubiquitin-protein ligase MDM2, a protein responsible for the degradation of p53. In spite of the structural and functional differences, the CDK inhibitor isoforms and the ARF product encoded by this gene, through the regulatory roles of CDK4 and p53 in cell cycle G1 progression, share a common functionality in cell cycle G1 control. This gene is frequently mutated or deleted in a wide variety of tumors, and is known to be an important tumor suppressor gene.","The protein encoded by this gene is a member of the INK4 family of cyclin-dependent kinase inhibitors. This protein has been shown to interact with CDK4 or CDK6, and prevent the activation of the CDK kinases, thus function as a cell growth regulator that controls cell cycle G1 progression. Ectopic expression of this gene was shown to suppress the growth of human cells in a manner that appears to correlate with the presence of a wild-type RB1 function. Studies in the knockout mice suggested the roles of this gene in regulating spermatogenesis, as well as in suppressing tumorigenesis. Two alternatively spliced transcript variants of this gene, which encode an identical protein, have been reported.","This gene is a member of the caudal-related homeobox transcription factor gene family. The encoded protein is a major regulator of intestine-specific genes involved in cell growth an differentiation. This protein also plays a role in early embryonic development of the intestinal tract. Aberrant expression of this gene is associated with intestinal inflammation and tumorigenesis.","This intronless gene encodes a transcription factor that contains a basic leucine zipper (bZIP) domain and recognizes the CCAAT motif in the promoters of target genes. The encoded protein functions in homodimers and also heterodimers with CCAAT/enhancer-binding proteins beta and gamma. Activity of this protein can modulate the expression of genes involved in cell cycle regulation as well as in body weight homeostasis. Mutation of this gene is associated with acute myeloid leukemia. The use of alternative in-frame non-AUG (GUG) and AUG start codons results in protein isoforms with different lengths. Differential translation initiation is mediated by an out-of-frame, upstream open reading frame which is located between the GUG and the first AUG start codons.","The CHD family of proteins is characterized by the presence of chromo (chromatin organization modifier) domains and SNF2-related helicase/ATPase domains. CHD genes alter gene expression possibly by modification of chromatin structure thus altering access of the transcriptional apparatus to its chromosomal DNA template. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene.","Clathrin is a major protein component of the cytoplasmic face of intracellular organelles, called coated vesicles and coated pits. These specialized organelles are involved in the intracellular trafficking of receptors and endocytosis of a variety of macromolecules.  The basic subunit of the clathrin coat is composed of three heavy chains and three light chains.","This gene encodes a member of the Kruppel-like family of transcription factors. The zinc finger protein is a transcriptional activator, and functions as a tumor suppressor. Multiple transcript variants encoding different isoforms have been found for this gene, some of which are implicated in carcinogenesis.","This gene is ubiquitously expressed and is involved in the transcriptional coactivation of many different transcription factors. First isolated as a nuclear protein that binds to cAMP-response element binding protein (CREB), this gene is now known to play critical roles in embryonic development, growth control, and homeostasis by coupling chromatin remodeling to transcription factor recognition. The protein encoded by this gene has intrinsic histone acetyltransferase activity and also acts as a scaffold to stabilize additional protein interactions with the transcription complex. This protein acetylates both histone and non-histone proteins. This protein shares regions of very high sequence similarity with protein p300 in its bromodomain, cysteine-histidine-rich regions, and histone acetyltransferase domain. Mutations in this gene cause Rubinstein-Taybi syndrome (RTS). Chromosomal translocations involving this gene have been associated with acute myeloid leukemia. Alternative splicing results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene is a member of the homeodomain family of DNA binding proteins. It may regulate gene expression, morphogenesis, and differentiation and it may also play a role in the cell cycle progession. Several alternatively spliced transcript variants encoding different isoforms have been identified.[provided by RefSeq, Feb 2011].","This gene is encodes a cytoplasmic protein with three cytoskeletal-associated protein-glycine-conserved (CAP-GLY) domains that functions as a deubiquitinating enzyme. Mutations in this gene have been associated with cylindromatosis, multiple familial trichoepithelioma, and Brooke-Spiegler syndrome. Alternate transcriptional splice variants, encoding different isoforms, have been characterized.","This gene encodes a multifunctional protein that resides in multiple locations in the nucleus and in the cytoplasm. It interacts with a wide variety of proteins, such as apoptosis antigen Fas, centromere protein C, and transcription factor erythroblastosis virus E26 oncogene homolog 1. In the nucleus, the encoded protein functions as a potent transcription repressor that binds to sumoylated transcription factors. Its repression can be relieved by the sequestration of this protein into promyelocytic leukemia nuclear bodies or nucleoli. This protein also associates with centromeres in G2 phase. In the cytoplasm, the encoded protein may function to regulate apoptosis. The subcellular localization and function of this protein are modulated by post-translational modifications, including sumoylation, phosphorylation and polyubiquitination. Alternative splicing results in multiple transcript variants.","This gene encodes a protein that is necessary for the repair of ultraviolet light-damaged DNA. This protein is the smaller subunit of a heterodimeric protein complex that participates in nucleotide excision repair, and this complex mediates the ubiquitylation of histones H3 and H4, which facilitates the cellular response to DNA damage. This subunit appears to be required for DNA binding. Mutations in this gene cause xeroderma pigmentosum complementation group E, a recessive disease that is characterized by an increased sensitivity to UV light and a high predisposition for skin cancer development, in some cases accompanied by neurological abnormalities. Two transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a member of the large DEAD-box protein family, that is defined by the presence of the conserved Asp-Glu-Ala-Asp (DEAD) motif, and has ATP-dependent RNA helicase activity. This protein has been reported to display a high level of RNA-independent ATPase activity, and unlike most DEAD-box helicases, the ATPase activity is thought to be stimulated by both RNA and DNA. This protein has multiple conserved domains and is thought to play roles in both the nucleus and cytoplasm. Nuclear roles include transcriptional regulation, mRNP assembly, pre-mRNA splicing, and mRNA export. In the cytoplasm, this protein is thought to be involved in translation, cellular signaling, and viral replication. Misregulation of this gene has been implicated in tumorigenesis. This gene has a paralog located in the nonrecombining region of the Y chromosome. Pseudogenes sharing similarity to both this gene and the DDX3Y paralog are found on chromosome 4 and the X chromosome. Alternative splicing results in multiple transcript variants.","DEAD box proteins, characterized by the conserved motif Asp-Glu-Ala-Asp (DEAD), are putative RNA helicases. They are implicated in a number of cellular processes involving alteration of RNA secondary structure such as translation initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Based on their distribution patterns, some members of this family are believed to be involved in embryogenesis, spermatogenesis, and cellular growth and division. This gene encodes a DEAD box protein, and it may be involved in ribosome assembly. Fusion of this gene and the nucleoporin gene, NUP98, by inversion 11 (p15q22) chromosome translocation is found in the patients with de novo or therapy-related myeloid malignancies.","Dynamins represent one of the subfamilies of GTP-binding proteins. These proteins share considerable sequence similarity over the N-terminal portion of the molecule, which contains the GTPase domain. Dynamins are associated with microtubules. They have been implicated in cell processes such as endocytosis and cell motility, and in alterations of the membrane that accompany certain activities such as bone resorption by osteoclasts. Dynamins bind many proteins that bind actin and other cytoskeletal proteins. Dynamins can also self-assemble, a process that stimulates GTPase activity. Five alternatively spliced transcripts encoding different proteins have been described. Additional alternatively spliced transcripts may exist, but their full-length nature has not been determined.","CpG methylation is an epigenetic modification that is important for embryonic development, imprinting, and X-chromosome inactivation. Studies in mice have demonstrated that DNA methylation is required for mammalian development. This gene encodes a DNA methyltransferase that is thought to function in de novo methylation, rather than maintenance methylation. The protein localizes to the cytoplasm and nucleus and its expression is developmentally regulated.","Predicted to enable DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in positive regulation of transcription by RNA polymerase II. Predicted to act upstream of or within positive regulation of transcription, DNA-templated. Predicted to be located in nucleus. Predicted to be part of chromatin. [provided by Alliance of Genome Resources, Apr 2022]","Enables DNA-binding transcription activator activity, RNA polymerase II-specific and sequence-specific double-stranded DNA binding activity. Involved in inflammatory response; negative regulation of transcription, DNA-templated; and positive regulation of transcription by RNA polymerase II. Located in Golgi apparatus; cytosol; and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a transcriptional activator that binds and activates the promoters of the CSF2, IL3, IL8, and PRF1 genes. The encoded protein is involved in natural killer cell development and function, innate immunity, and induction of cell cycle arrest in naive CD8+ cells. Two transcript variants encoding the same protein have been found for this gene.[provided by RefSeq, Jan 2010].","This gene encodes the adenovirus E1A-associated cellular p300 transcriptional co-activator protein. It functions as histone acetyltransferase that regulates transcription via chromatin remodeling and is important in the processes of cell proliferation and differentiation. It mediates cAMP-gene regulation by binding specifically to phosphorylated CREB protein. This gene has also been identified as a co-activator of HIF1A (hypoxia-inducible factor 1 alpha), and thus plays a role in the stimulation of hypoxia-induced genes such as VEGF. Defects in this gene are a cause of Rubinstein-Taybi syndrome and may also play a role in epithelial cancer.","This gene encodes a transcription factor involved in the induction of genes regulated by oxygen, which is induced as oxygen levels fall. The encoded protein contains a basic-helix-loop-helix domain protein dimerization domain as well as a domain found in proteins in signal transduction pathways which respond to oxygen levels. Mutations in this gene are associated with erythrocytosis familial type 4.","This gene encodes a protein that is part of the EGFR pathway. The protein is present at clatherin-coated pits and is involved in receptor-mediated endocytosis of EGF. Notably, this gene is rearranged with the HRX/ALL/MLL gene in acute myelogeneous leukemias. Alternative splicing results in multiple transcript variants encoding distinct isoforms.","This gene is a member of the Tyr protein kinase family and the epidermal growth factor receptor subfamily. It encodes a single-pass type I membrane protein with multiple cysteine rich domains, a transmembrane domain, a tyrosine kinase domain, a phosphotidylinositol-3 kinase binding site and a PDZ domain binding motif. The protein binds to and is activated by neuregulins and other factors and induces a variety of cellular responses including mitogenesis and differentiation. Multiple proteolytic events allow for the release of a cytoplasmic fragment and an extracellular fragment. Mutations in this gene have been associated with cancer. Alternatively spliced variants which encode different protein isoforms have been described; however, not all variants have been fully characterized.","The nucleotide excision repair pathway is a mechanism to repair damage to DNA. The protein encoded by this gene is involved in transcription-coupled nucleotide excision repair and is an integral member of the basal transcription factor BTF2/TFIIH complex. The gene product has ATP-dependent DNA helicase activity and belongs to the RAD3/XPD subfamily of helicases. Defects in this gene can result in three different disorders, the cancer-prone syndrome xeroderma pigmentosum complementation group D, trichothiodystrophy, and Cockayne syndrome. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","This gene encodes an ATP-dependent DNA helicase that functions in nucleotide excision repair. The encoded protein is a subunit of basal transcription factor 2 (TFIIH) and, therefore, also functions in class II transcription. Mutations in this gene are associated with Xeroderma pigmentosum B, Cockayne's syndrome, and trichothiodystrophy. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene forms a complex with ERCC1 and is involved in the 5' incision made during nucleotide excision repair. This complex is a structure specific DNA repair endonuclease that interacts with EME1. Defects in this gene are a cause of xeroderma pigmentosum complementation group F (XP-F), or xeroderma pigmentosum VI (XP6).[provided by RefSeq, Mar 2009].","This gene encodes a single-strand specific DNA endonuclease that makes the 3' incision in DNA excision repair following UV-induced damage. The protein may also function in other cellular processes, including RNA polymerase II transcription, and transcription-coupled DNA repair. Mutations in this gene cause xeroderma pigmentosum complementation group G (XP-G), which is also referred to as xeroderma pigmentosum VII (XP7), a skin disorder characterized by hypersensitivity to UV light and increased susceptibility for skin cancer development following UV exposure. Some patients also develop Cockayne syndrome, which is characterized by severe growth defects, cognitive disability, and cachexia. Read-through transcription exists between this gene and the neighboring upstream BIVM (basic, immunoglobulin-like variable motif containing) gene.","This gene encodes an estrogen receptor and ligand-activated transcription factor. The canonical protein contains an N-terminal ligand-independent transactivation domain, a central DNA binding domain, a hinge domain, and a C-terminal ligand-dependent transactivation domain. The protein localizes to the nucleus where it may form either a homodimer or a heterodimer with estrogen receptor 2. The protein encoded by this gene regulates the transcription of many estrogen-inducible genes that play a role in growth, metabolism, sexual development, gestation, and other reproductive functions and is expressed in many non-reproductive tissues. The receptor encoded by this gene plays a key role in breast cancer, endometrial cancer, and osteoporosis. This gene is reported to have dozens of transcript variants due to the use of alternate promoters and alternative splicing, however, the full-length nature of many of these variants remain uncertain.","This gene encodes an ETS family transcription factor. The product of this gene contains two functional domains: a N-terminal pointed (PNT) domain that is involved in protein-protein interactions with itself and other proteins, and a C-terminal DNA-binding domain. Gene knockout studies in mice suggest that it is required for hematopoiesis and maintenance of the developing vascular network. This gene is known to be involved in a large number of chromosomal rearrangements associated with leukemia and congenital fibrosarcoma.","This gene encodes an endoplasmic reticulum-resident type II transmembrane glycosyltransferase involved in the chain elongation step of heparan sulfate biosynthesis. Mutations in this gene cause the type I form of multiple exostoses.","This gene encodes one of two glycosyltransferases involved in the chain elongation step of heparan sulfate biosynthesis. Mutations in this gene cause the type II form of multiple exostoses. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.","This gene encodes a member of the Polycomb-group (PcG) family. PcG family members form multimeric protein complexes, which are involved in maintaining the transcriptional repressive state of genes over successive cell generations. This protein associates with the embryonic ectoderm development protein, the VAV1 oncoprotein, and the X-linked nuclear protein. This protein may play a role in the hematopoietic and central nervous systems. Multiple alternatively splcied transcript variants encoding distinct isoforms have been identified for this gene.","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group A. Alternative splicing results in multiple transcript variants encoding different isoforms. Mutations in this gene are the most common cause of Fanconi anemia.","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group C.","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group D2. This protein is monoubiquinated in response to DNA damage, resulting in its localization to nuclear foci with other proteins (BRCA1 AND BRCA2) involved in homology-directed DNA repair. Alternative splicing results in multiple transcript variants.","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group E.","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group F.","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group G.","This gene is an ortholog of the Drosophila fat gene, which encodes a tumor suppressor essential for controlling cell proliferation during Drosophila development. The gene product is a member of the cadherin superfamily, a group of integral membrane proteins characterized by the presence of cadherin-type repeats. In addition to containing 34 tandem cadherin-type repeats, the gene product has five epidermal growth factor (EGF)-like repeats and one laminin A-G domain. This gene is expressed at high levels in a number of fetal epithelia. Its product probably functions as an adhesion molecule and/or signaling receptor, and is likely to be important in developmental processes and cell communication. Transcript variants derived from alternative splicing and/or alternative promoter usage exist, but they have not been fully described.","This gene encodes an extracellular matrix protein, which belongs to the fibulin family. This protein binds various extracellular ligands and calcium. It may play a role during organ development, in particular, during the differentiation of heart, skeletal and neuronal structures. Alternatively spliced transcript variants encoding different isoforms have been identified.","The protein encoded by this gene removes 5' overhanging flaps in DNA repair and processes the 5' ends of Okazaki fragments in lagging strand DNA synthesis. Direct physical interaction between this protein and AP endonuclease 1 during long-patch base excision repair provides coordinated loading of the proteins onto the substrate, thus passing the substrate from one enzyme to another. The protein is a member of the XPG/RAD2 endonuclease family and is one of ten proteins essential for cell-free DNA replication. DNA secondary structure can inhibit flap processing at certain trinucleotide repeats in a length-dependent manner by concealing the 5' end of the flap that is necessary for both binding and cleavage by the protein encoded by this gene. Therefore, secondary structure can deter the protective function of this protein, leading to site-specific trinucleotide expansions.","This gene encodes the human cellular counterpart of a feline sarcoma retrovirus protein with transforming capabilities. The gene product has tyrosine-specific protein kinase activity and that activity is required for maintenance of cellular transformation. Its chromosomal location has linked it to a specific translocation event identified in patients with acute promyelocytic leukemia but it is also involved in normal hematopoiesis as well as growth factor and cytokine receptor signaling. Alternative splicing results in multiple variants encoding different isoforms.[provided by RefSeq, Jan 2009].","Cell surface heparan sulfate proteoglycans are composed of a membrane-associated protein core substituted with a variable number of heparan sulfate chains. Members of the glypican-related integral membrane proteoglycan family (GRIPS) contain a core protein anchored to the cytoplasmic membrane via a glycosyl phosphatidylinositol linkage.  These proteins may play a role in the control of cell division and growth regulation.","The protein encoded by this gene is an enzymatic component of the tricarboxylic acid (TCA) cycle, or Krebs cycle, and catalyzes the formation of L-malate from fumarate. It exists in both a cytosolic form and an N-terminal extended form, differing only in the translation start site used. The N-terminal extended form is targeted to the mitochondrion, where the removal of the extension generates the same form as in the cytoplasm. It is similar to some thermostable class II fumarases and functions as a homotetramer. Mutations in this gene can cause fumarase deficiency and lead to progressive encephalopathy.","The protein encoded by this gene is a P1-P3-bis(5'-adenosyl) triphosphate hydrolase involved in purine metabolism. This gene encompasses the common fragile site FRA3B on chromosome 3, where carcinogen-induced damage can lead to translocations and aberrant transcripts. In fact, aberrant transcripts from this gene have been found in about half of all esophageal, stomach, and colon carcinomas. The encoded protein is also a tumor suppressor, as loss of its activity results in replication stress and DNA damage.","This gene belongs to the forkhead family of transcription factors which are characterized by a distinct forkhead domain.  The specific function of this gene has not yet been determined; however, it may play a role in myogenic growth and differentiation. Translocation of this gene with PAX3 has been associated with alveolar rhabdomyosarcoma.","This gene belongs to the forkhead family of transcription factors which are characterized by a distinct forkhead domain. This gene likely functions as a trigger for apoptosis through expression of genes necessary for cell death. Translocation of this gene with the MLL gene is associated with secondary acute leukemia. Alternatively spliced transcript variants encoding the same protein have been observed.","This gene encodes a multifunctional protein component of the heterogeneous nuclear ribonucleoprotein (hnRNP) complex. The hnRNP complex is involved in pre-mRNA splicing and the export of fully processed mRNA to the cytoplasm. This protein belongs to the FET family of RNA-binding proteins which have been implicated in cellular processes that include regulation of gene expression, maintenance of genomic integrity and mRNA/microRNA processing. Alternative splicing results in multiple transcript variants. Defects in this gene result in amyotrophic lateral sclerosis type 6.","This gene encodes a protein which belongs to the GATA family of transcription factors. The protein plays an important role in erythroid development by regulating the switch of fetal hemoglobin to adult hemoglobin. Mutations in this gene have been associated with X-linked dyserythropoietic anemia and thrombocytopenia.","This gene encodes a protein which belongs to the GATA family of transcription factors. The protein contains two GATA-type zinc fingers and is an important regulator of T-cell development and plays an important role in endothelial cell biology. Defects in this gene are the cause of hypoparathyroidism with sensorineural deafness and renal dysplasia.","Cell surface heparan sulfate proteoglycans are composed of a membrane-associated protein core substituted with a variable number of heparan sulfate chains. Members of the glypican-related integral membrane proteoglycan family (GRIPS) contain a core protein anchored to the cytoplasmic membrane via a glycosyl phosphatidylinositol linkage. These proteins may play a role in the control of cell division and growth regulation. The protein encoded by this gene can bind to and inhibit the dipeptidyl peptidase activity of CD26, and it can induce apoptosis in certain cell types. Deletion mutations in this gene are associated with Simpson-Golabi-Behmel syndrome, also known as Simpson dysmorphia syndrome. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the glutamate-gated ion channel protein family. The encoded protein is an N-methyl-D-aspartate (NMDA) receptor subunit. NMDA receptors are both ligand-gated and voltage-dependent, and are involved in long-term potentiation, an activity-dependent increase in the efficiency of synaptic transmission thought to underlie certain kinds of memory and learning. These receptors are permeable to calcium ions, and activation results in a calcium influx into post-synaptic cells, which results in the activation of several signaling cascades. Disruption of this gene is associated with focal epilepsy and speech disorder with or without cognitive disability. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the DNA mismatch repair MutS family. In E. coli, the MutS protein helps in the recognition of mismatched nucleotides prior to their repair. A highly conserved region of approximately 150 aa, called the Walker-A adenine nucleotide binding motif, exists in MutS homologs. The encoded protein heterodimerizes with MSH2 to form a mismatch recognition complex that functions as a bidirectional molecular switch that exchanges ADP and ATP as DNA mismatches are bound and dissociated. Mutations in this gene may be associated with hereditary nonpolyposis colon cancer, colorectal cancer, and endometrial cancer. Transcripts variants encoding different isoforms have been described.","The protein encoded by this gene is a membrane glycoprotein that mediates cell-cell signaling and plays a critical role in the growth and development of multiple organ systems. An extraordinary variety of different isoforms are produced from this gene through alternative promoter usage and splicing. These isoforms are expressed in a tissue-specific manner and differ significantly in their structure, and are classified as types I, II, III, IV, V and VI. Dysregulation of this gene has been linked to diseases such as cancer, schizophrenia, and bipolar disorder (BPD).","In vertebrates, the genes encoding the class of transcription factors called homeobox genes are found in clusters named A, B, C, and D on four separate chromosomes. Expression of these proteins is spatially and temporally regulated during embryonic development. This gene is part of the A cluster on chromosome 7 and encodes a DNA-binding transcription factor which may regulate gene expression, morphogenesis, and differentiation. This gene is highly similar to the abdominal-B (Abd-B) gene of Drosophila. A specific translocation event which causes a fusion between this gene and the NUP98 gene has been associated with myeloid leukemogenesis. Read-through transcription exists between this gene and the upstream homeobox A10 (HOXA10) gene.[provided by RefSeq, Mar 2011].","In vertebrates, the genes encoding the class of transcription factors called homeobox genes are found in clusters named A, B, C, and D on four separate chromosomes. Expression of these proteins is spatially and temporally regulated during embryonic development. This gene is part of the A cluster on chromosome 7 and encodes a DNA-binding transcription factor which may regulate gene expression, morphogenesis, and differentiation. This gene is involved in the regulation of uterine development and is required for female fertility. Mutations in this gene can cause radio-ulnar synostosis with amegakaryocytic thrombocytopenia.","The protein encoded by this gene is a helix-loop-helix (HLH) protein that can form heterodimers with other HLH proteins. However, the encoded protein lacks a basic DNA-binding domain and therefore inhibits the DNA binding of any HLH protein with which it interacts.","Enables protein N-terminus binding activity. Contributes to translation initiation factor activity. Involved in positive regulation of mRNA binding activity; regulation of gene expression; and translational initiation. Located in cytosol and nucleus. Part of eukaryotic translation initiation factor 3 complex. Colocalizes with PML body. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene belongs to the IRF (interferon regulatory factor) family of transcription factors, characterized by an unique tryptophan pentad repeat DNA-binding domain. The IRFs are important in the regulation of interferons in response to infection by virus, and in the regulation of interferon-inducible genes. This family member is lymphocyte specific and negatively regulates Toll-like-receptor (TLR) signaling that is central to the activation of innate and adaptive immune systems. A chromosomal translocation involving this gene and the IgH locus, t(6;14)(p25;q32), may be a cause of multiple myeloma. Alternatively spliced transcript variants have been found for this gene.","This gene encodes a membrane protein that is a member of a class of protein-tyrosine kinases (PTK) characterized by the presence of a second phosphotransferase-related domain immediately N-terminal to the PTK domain. The encoded kinase phosphorylates STAT proteins (signal transducers and activators of transcription) and plays a key role in interferon-alpha/beta, interferon-gamma, and cytokine signal transduction. This gene plays a crucial role in effecting the expression of genes that mediate inflammation, epithelial remodeling, and metastatic cancer progression. This gene is a key component of the interleukin-6 (IL-6)/JAK1/STAT3 immune and inflammation response and is a therapeutic target for alleviating cytokine storms. The kinase activity of this gene is directly inhibited by the suppressor of cytokine signalling 1 (SOCS1) protein. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene belongs to the SMAD, a family of proteins similar to the gene products of the Drosophila gene 'mothers against decapentaplegic' (Mad) and the C. elegans gene Sma. SMAD proteins are signal transducers and transcriptional modulators that mediate multiple signaling pathways. This protein mediates the signal of the transforming growth factor (TGF)-beta, and thus regulates multiple cellular processes, such as cell proliferation, apoptosis, and differentiation. This protein is recruited to the TGF-beta receptors through its interaction with the SMAD anchor for receptor activation (SARA) protein. In response to TGF-beta signal, this protein is phosphorylated by the TGF-beta receptors. The phosphorylation induces the dissociation of this protein with SARA and the association with the family member SMAD4. The association with SMAD4 is important for the translocation of this protein into the nucleus, where it binds to target promoters and forms a transcription repressor complex with other cofactors. This protein can also be phosphorylated by activin type 1 receptor kinase, and mediates the signal from the activin. Alternatively spliced transcript variants have been observed for this gene.","The SMAD family of proteins are a group of intracellular signal transducer proteins similar to the gene products of the Drosophila gene 'mothers against decapentaplegic' (Mad) and the C. elegans gene Sma. The SMAD3 protein functions in the transforming growth factor-beta signaling pathway, and transmits signals from the cell surface to the nucleus, regulating gene activity and cell proliferation. This protein forms a complex with other SMAD proteins and binds DNA, functioning both as a transcription factor and tumor suppressor. Mutations in this gene are associated with aneurysms-osteoarthritis syndrome and Loeys-Dietz Syndrome 3.","This gene encodes a member of the Smad family of signal transduction proteins. Smad proteins are phosphorylated and activated by transmembrane serine-threonine receptor kinases in response to transforming growth factor (TGF)-beta signaling. The product of this gene forms homomeric complexes and heteromeric complexes with other activated Smad proteins, which then accumulate in the nucleus and regulate the transcription of target genes. This protein binds to DNA and recognizes an 8-bp palindromic sequence (GTCTAGAC) called the Smad-binding element (SBE). The protein acts as a tumor suppressor and inhibits epithelial cell proliferation. It may also have an inhibitory effect on tumors by reducing angiogenesis and increasing blood vessel hyperpermeability. The encoded protein is a crucial component of the bone morphogenetic protein signaling pathway. The Smad proteins are subject to complex regulation by post-translational modifications. Mutations or deletions in this gene have been shown to result in pancreatic cancer, juvenile polyposis syndrome, and hereditary hemorrhagic telangiectasia syndrome.","The protein encoded by this gene is a member of the basic helix-loop-helix leucine zipper (bHLHZ) family of transcription factors. It is able to form homodimers and heterodimers with other family members, which include Mad, Mxi1 and Myc. Myc is an oncoprotein implicated in cell proliferation, differentiation and apoptosis. The homodimers and heterodimers compete for a common DNA target site (the E box) and rearrangement among these dimer forms provides a complex system of transcriptional regulation. Mutations of this gene have been reported to be associated with hereditary pheochromocytoma. A pseudogene of this gene is located on the long arm of chromosome 7. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a serine/threonine kinase and is part of some signal transduction cascades, including the ERK and JNK kinase pathways as well as the NF-kappa-B pathway. The encoded protein is activated by autophosphorylation and requires magnesium as a cofactor in phosphorylating other proteins. This protein has E3 ligase activity conferred by a plant homeodomain (PHD) in its N-terminus and phospho-kinase activity conferred by a kinase domain in its C-terminus.","This gene encodes menin, a tumor suppressor associated with a syndrome known as multiple endocrine neoplasia type 1. Menin is a scaffold protein that functions in histone modification and epigenetic gene regulation. It is thought to regulate several pathways and processes by altering chromatin structure through the modification of histones.","Alkylating agents are potent carcinogens that can result in cell death, mutation and cancer. The protein encoded by this gene is a DNA repair protein that is involved in cellular defense against mutagenesis and toxicity from alkylating agents. The protein catalyzes transfer of methyl groups from O(6)-alkylguanine and other methylated moieties of the DNA to its own molecule, which repairs the toxic lesions. Methylation of the genes promoter has been associated with several cancer types, including colorectal cancer, lung cancer, lymphoma and glioblastoma.","This gene encodes a protein with an acidic transcriptional activation domain, 4 LRRs (leucine-rich repeats) and a GTP binding domain. The protein is located in the nucleus and acts as a positive regulator of class II major histocompatibility complex gene transcription, and is referred to as the 'master control factor' for the expression of these genes. The protein also binds GTP and uses GTP binding to facilitate its own transport into the nucleus. Once in the nucleus it does not bind DNA but rather uses an intrinsic acetyltransferase (AT) activity to act in a coactivator-like fashion. Mutations in this gene have been associated with bare lymphocyte syndrome type II (also known as hereditary MHC class II deficiency or HLA class II-deficient combined immunodeficiency), increased susceptibility to rheumatoid arthritis, multiple sclerosis, and possibly myocardial infarction. Several transcript variants encoding different isoforms have been found for this gene.","This gene encodes an oncoprotein which is thought to play a role in the phenotypic determination of hemopoetic cells. Translocations between this gene and nucleophosmin have been associated with myelodysplastic syndrome and acute myeloid leukemia. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene can heterodimerize with mismatch repair endonuclease PMS2 to form MutL alpha, part of the DNA mismatch repair system. When MutL alpha is bound by MutS beta and some accessory proteins, the PMS2 subunit of MutL alpha introduces a single-strand break near DNA mismatches, providing an entry point for exonuclease degradation. The encoded protein is also involved in DNA damage signaling and can heterodimerize with DNA mismatch repair protein MLH3 to form MutL gamma, which is involved in meiosis. This gene was identified as a locus frequently mutated in hereditary nonpolyposis colon cancer (HNPCC).","This gene encodes a member of the O class of winged helix/forkhead transcription factor family. Proteins encoded by this class are regulated by factors involved in growth and differentiation indicating they play a role in these processes. A translocation involving this gene on chromosome X and the homolog of the Drosophila trithorax gene, encoding a DNA binding protein, located on chromosome 11 is associated with leukemia. Multiple transcript variants encoding different isoforms have been found for this gene.","This locus is frequently mutated in hereditary nonpolyposis colon cancer (HNPCC). When cloned, it was discovered to be a human homolog of the E. coli mismatch repair gene mutS, consistent with the characteristic alterations in microsatellite sequences (RER+ phenotype) found in HNPCC. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a DNA glycosylase involved in oxidative DNA damage repair. The enzyme excises adenine bases from the DNA backbone at sites where adenine is inappropriately paired with guanine, cytosine, or 8-oxo-7,8-dihydroguanine, a major oxidatively damaged DNA lesion. The protein is localized to the nucleus and mitochondria. This gene product is thought to play a role in signaling apoptosis by the introduction of single-strand breaks following oxidative damage. Mutations in this gene result in heritable predisposition to colorectal cancer, termed MUTYH-associated polyposis (MAP). Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a conventional non-muscle myosin; this protein should not be confused with the unconventional myosin-9a or 9b (MYO9A or MYO9B). The encoded protein is a myosin IIA heavy chain that contains an IQ domain and a myosin head-like domain which is involved in several important functions, including cytokinesis, cell motility and maintenance of cell shape. Defects in this gene have been associated with non-syndromic sensorineural deafness autosomal dominant type 17, Epstein syndrome, Alport syndrome with macrothrombocytopenia, Sebastian syndrome, Fechtner syndrome and macrothrombocytopenia with progressive sensorineural deafness.","This gene encodes a member of the family of NGFI-A binding (NAB) proteins, which function in the nucleus to repress transcription induced by some members of the EGR (early growth response) family of transactivators. NAB proteins can homo- or hetero-multimerize with other EGR or NAB proteins through a conserved N-terminal domain, and repress transcription through two partially redundant C-terminal domains. Transcriptional repression by the encoded protein is mediated in part by interactions with the nucleosome remodeling and deactylase (NuRD) complex. Alternatively spliced transcript variants have been described, but their biological validity has not been determined.","Mutations in this gene are associated with Nijmegen breakage syndrome, an autosomal recessive chromosomal instability syndrome characterized by microcephaly, growth retardation, immunodeficiency, and cancer predisposition. The encoded protein is a member of the MRE11/RAD50 double-strand break repair complex which consists of 5 proteins. This gene product is thought to be involved in DNA double-strand break repair and DNA damage-induced checkpoint activation.","This gene product appears to function as a negative regulator of the ras signal transduction pathway. Mutations in this gene have been linked to neurofibromatosis type 1, juvenile myelomonocytic leukemia and Watson syndrome. The mRNA for this gene is subject to RNA editing (CGA>UGA->Arg1306Term) resulting in premature translation termination. Alternatively spliced transcript variants encoding different isoforms have also been described for this gene.","This gene encodes a protein that is similar to some members of the ERM (ezrin, radixin, moesin) family of proteins that link cytoskeletal components with proteins in the cell membrane. The encoded protein is involved in regulation of contact-dependent inhibition of cell proliferation and functions in cell-cell adhesion and transmembrane signaling. The encoded protein has been shown to interact with cell-surface proteins, proteins involved in cytoskeletal dynamics, and proteins involved in regulating ion transport. Disruption of this protein's function has been implicated in tumorigenesis and metastasis. Mutations in this gene are associated with neurofibromatosis type II which is characterized by nervous system and skin tumors and ocular abnormalities.","This gene encodes a transcription factor which is a member of a small family of basic leucine zipper (bZIP) proteins. The encoded transcription factor regulates genes which contain antioxidant response elements (ARE) in their promoters; many of these genes encode proteins involved in response to injury and inflammation which includes the production of free radicals. Multiple transcript variants encoding different isoforms have been characterized for this gene.","This gene encodes a subunit of the transcription factor complex nuclear factor-kappa-B (NFkB). The NFkB complex is expressed in numerous cell types and functions as a central activator of genes involved in inflammation and immune function. The protein encoded by this gene can function as both a transcriptional activator or repressor depending on its dimerization partner. The p100 full-length protein is co-translationally processed into a p52 active form. Chromosomal rearrangements and translocations of this locus have been observed in B cell lymphomas, some of which may result in the formation of fusion proteins. There is a pseudogene for this gene on chromosome 18. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene binds to components of NF-kappa-B, trapping the complex in the cytoplasm and preventing it from activating genes in the nucleus. Phosphorylation of the encoded protein targets it for destruction by the ubiquitin pathway, which activates NF-kappa-B by making it available to translocate to the nucleus.","Involved in regulation of stem cell population maintenance. Part of CCR4-NOT complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the NOTCH family of proteins. Members of this Type I transmembrane protein family share structural characteristics including an extracellular domain consisting of multiple epidermal growth factor-like (EGF) repeats, and an intracellular domain consisting of multiple different domain types. Notch signaling is an evolutionarily conserved intercellular signaling pathway that regulates interactions between physically adjacent cells through binding of Notch family receptors to their cognate ligands. The encoded preproprotein is proteolytically processed in the trans-Golgi network to generate two polypeptide chains that heterodimerize to form the mature cell-surface receptor. This receptor plays a role in the development of numerous cell and tissue types. Mutations in this gene are associated with aortic valve disease, Adams-Oliver syndrome, T-cell acute lymphoblastic leukemia, chronic lymphocytic leukemia, and head and neck squamous cell carcinoma.","This gene encodes a member of the Notch family. Members of this Type 1 transmembrane protein family share structural characteristics including an extracellular domain consisting of multiple epidermal growth factor-like (EGF) repeats, and an intracellular domain consisting of multiple, different domain types. Notch family members play a role in a variety of developmental processes by controlling cell fate decisions. The Notch signaling network is an evolutionarily conserved intercellular signaling pathway which regulates interactions between physically adjacent cells. In Drosophilia, notch interaction with its cell-bound ligands (delta, serrate) establishes an intercellular signaling pathway that plays a key role in development. Homologues of the notch-ligands have also been identified in human, but precise interactions between these ligands and the human notch homologues remain to be determined. This protein is cleaved in the trans-Golgi network, and presented on the cell surface as a heterodimer. This protein functions as a receptor for membrane bound ligands, and may play a role in vascular, renal and hepatic development. Two transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a DNA N-glycosylase of the endonuclease III family. Like a similar protein in E. coli, the encoded protein has DNA glycosylase activity on DNA substrates containing oxidized pyrimidine residues and has apurinic/apyrimidinic lyase activity.","This gene encodes a member of the neurotrophic tyrosine kinase receptor (NTKR) family. This kinase is a membrane-bound receptor that, upon neurotrophin binding, phosphorylates itself and members of the MAPK pathway. The presence of this kinase leads to cell differentiation and may play a role in specifying sensory neuron subtypes. Mutations in this gene have been associated with congenital insensitivity to pain, anhidrosis, self-mutilating behavior, cognitive disability and cancer. Alternate transcriptional splice variants of this gene have been found, but only three have been characterized to date.","This gene encodes a member of the paired box (PAX) family of transcription factors. The central feature of this gene family is a novel, highly conserved DNA-binding motif, known as the paired box. Paired box transcription factors are important regulators in early development, and alterations in the expression of their genes are thought to contribute to neoplastic transformation. This gene encodes the B-cell lineage specific activator protein that is expressed at early, but not late stages of B-cell differentiation. Its expression has also been detected in developing CNS and testis and so the encoded protein may also play a role in neural development and spermatogenesis. This gene is located at 9p13, which is involved in t(9;14)(p13;q32) translocations recurring in small lymphocytic lymphomas of the plasmacytoid subtype, and in derived large-cell lymphomas. This translocation brings the potent E-mu enhancer of the IgH gene into close proximity of the PAX5 promoter, suggesting that the deregulation of transcription of this gene contributes to the pathogenesis of these lymphomas. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene is a member of the Period family of genes and is expressed in a circadian pattern in the suprachiasmatic nucleus, the primary circadian pacemaker in the mammalian brain. Genes in this family encode components of the circadian rhythms of locomotor activity, metabolism, and behavior. This gene is upregulated by CLOCK/ARNTL heterodimers but then represses this upregulation in a feedback loop using PER/CRY heterodimers to interact with CLOCK/ARNTL. Polymorphisms in this gene may increase the risk of getting certain cancers. Alternative splicing has been observed in this gene; however, these variants have not been fully described.","Phosphatidylinositol 3-kinase phosphorylates the inositol ring of phosphatidylinositol at the 3-prime position. The enzyme comprises a 110 kD catalytic subunit and a regulatory subunit of either 85, 55, or 50 kD. This gene encodes the 85 kD regulatory subunit. Phosphatidylinositol 3-kinase plays an important role in the metabolic actions of insulin, and a mutation in this gene has been associated with insulin resistance. Alternative splicing of this gene results in four transcript variants encoding different isoforms.","The protein encoded by this gene is a member of the tripartite motif (TRIM) family. The TRIM motif includes three zinc-binding domains, a RING, a B-box type 1 and a B-box type 2, and a coiled-coil region. This phosphoprotein localizes to nuclear bodies where it functions as a transcription factor and tumor suppressor. Its expression is cell-cycle related and it regulates the p53 response to oncogenic signals. The gene is often involved in the translocation with the retinoic acid receptor alpha gene associated with acute promyelocytic leukemia (APL). Extensive alternative splicing of this gene results in several variations of the protein's central and C-terminal regions; all variants encode the same N-terminus. Alternatively spliced transcript variants encoding different isoforms have been identified.","The protein encoded by this gene is a key component of the mismatch repair system that functions to correct DNA mismatches and small insertions and deletions that can occur during DNA replication and homologous recombination. This protein forms heterodimers with the gene product of the mutL homolog 1 (MLH1) gene to form the MutL-alpha heterodimer. The MutL-alpha heterodimer possesses an endonucleolytic activity that is activated following recognition of mismatches and insertion/deletion loops by the MutS-alpha and MutS-beta heterodimers, and is necessary for removal of the mismatched DNA. There is a DQHA(X)2E(X)4E motif found at the C-terminus of the protein encoded by this gene that forms part of the active site of the nuclease. Mutations in this gene have been associated with hereditary nonpolyposis colorectal cancer (HNPCC; also known as Lynch syndrome) and Turcot syndrome.","This gene encodes the 125-kDa catalytic subunit of DNA polymerase delta. DNA polymerase delta possesses both polymerase and 3' to 5' exonuclease activity and plays a critical role in DNA replication and repair. Alternatively spliced transcript variants have been observed for this gene, and a pseudogene of this gene is located on the long arm of chromosome 6.","This gene encodes the catalytic subunit of DNA polymerase epsilon. The enzyme is involved in DNA repair and chromosomal DNA replication. Mutations in this gene have been associated with colorectal cancer 12 and facial dysmorphism, immunodeficiency, livedo, and short stature.","Mitochondrial DNA polymerase is heterotrimeric, consisting of a homodimer of accessory subunits plus a catalytic subunit. The protein encoded by this gene is the catalytic subunit of mitochondrial DNA polymerase. The encoded protein contains a polyglutamine tract near its N-terminus that may be polymorphic. Defects in this gene are a cause of progressive external ophthalmoplegia with mitochondrial DNA deletions 1 (PEOA1), sensory ataxic neuropathy dysarthria and ophthalmoparesis (SANDO), Alpers-Huttenlocher syndrome (AHS), and mitochondrial neurogastrointestinal encephalopathy syndrome (MNGIE). Two transcript variants encoding the same protein have been found for this gene.","This gene encodes a member of the peroxisome proliferator-activated receptor (PPAR) subfamily of nuclear receptors. PPARs form heterodimers with retinoid X receptors (RXRs) and these heterodimers regulate transcription of various genes. Three subtypes of PPARs are known: PPAR-alpha, PPAR-delta, and PPAR-gamma. The protein encoded by this gene is PPAR-gamma and is a regulator of adipocyte differentiation. Additionally, PPAR-gamma has been implicated in the pathology of numerous diseases including obesity, diabetes, atherosclerosis and cancer. Alternatively spliced transcript variants that encode different isoforms have been described.","This gene encodes a constant regulatory subunit of protein phosphatase 2. Protein phosphatase 2 is one of the four major Ser/Thr phosphatases, and it is implicated in the negative control of cell growth and division. It consists of a common heteromeric core enzyme, which is composed of a catalytic subunit and a constant regulatory subunit, that associates with a variety of regulatory subunits. The constant regulatory subunit A serves as a scaffolding molecule to coordinate the assembly of the catalytic subunit and a variable regulatory B subunit. This gene encodes an alpha isoform of the constant regulatory subunit A. Alternatively spliced transcript variants have been described.","This gene encodes the catalytic subunit of protein phosphatase, a component of a signaling pathway regulating cell cycle progression. Splice variants encoding different protein isoforms exist. The pseudogene of this gene is located on chromosome X.","This gene encodes a protein with structural similarities to complement component C9 that is important in immunity. This protein forms membrane pores that allow the release of granzymes and subsequent cytolysis of target cells. Whether pore formation occurs in the plasma membrane of target cells or in an endosomal membrane inside target cells is subject to debate. Mutations in this gene are associated with a variety of human disease including diabetes, multiple sclerosis, lymphomas, autoimmune lymphoproliferative syndrome (ALPS), aplastic anemia, and familial hemophagocytic lymphohistiocytosis type 2 (FHL2), a rare and lethal autosomal recessive disorder of early childhood.","cAMP is a signaling molecule important for a variety of cellular functions. cAMP exerts its effects by activating the cAMP-dependent protein kinase, which transduces the signal through phosphorylation of different target proteins. The inactive kinase holoenzyme is a tetramer composed of two regulatory and two catalytic subunits. cAMP causes the dissociation of the inactive holoenzyme into a dimer of regulatory subunits bound to four cAMP and two free monomeric catalytic subunits. Four different regulatory subunits and three catalytic subunits have been identified in humans. This gene encodes one of the regulatory subunits. This protein was found to be a tissue-specific extinguisher that down-regulates the expression of seven liver genes in hepatoma x fibroblast hybrids. Mutations in this gene cause Carney complex (CNC). This gene can fuse to the RET protooncogene by gene rearrangement and form the thyroid tumor-specific chimeric oncogene known as PTC2. A nonconventional nuclear localization sequence (NLS) has been found for this protein which suggests a role in DNA replication via the protein serving as a nuclear transport protein for the second subunit of the Replication Factor C (RFC40). Several alternatively spliced transcript variants encoding two different isoforms have been observed.","This gene encodes a member of the patched family of proteins and a component of the hedgehog signaling pathway. Hedgehog signaling is important in embryonic development and tumorigenesis. The encoded protein is the receptor for the secreted hedgehog ligands, which include sonic hedgehog, indian hedgehog and desert hedgehog. Following binding by one of the hedgehog ligands, the encoded protein is trafficked away from the primary cilium, relieving inhibition of the G-protein-coupled receptor smoothened, which results in activation of downstream signaling. Mutations of this gene have been associated with basal cell nevus syndrome and holoprosencephaly.","This gene was identified as a tumor suppressor that is mutated in a large number of cancers at high frequency. The protein encoded by this gene is a phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase. It contains a tensin like domain as well as a catalytic domain similar to that of the dual specificity protein tyrosine phosphatases. Unlike most of the protein tyrosine phosphatases, this protein preferentially dephosphorylates phosphoinositide substrates. It negatively regulates intracellular levels of phosphatidylinositol-3,4,5-trisphosphate in cells and functions as a tumor suppressor by negatively regulating AKT/PKB signaling pathway. The use of a non-canonical (CUG) upstream initiation site produces a longer isoform that initiates translation with a leucine, and is thought to be preferentially associated with the mitochondrial inner membrane. This longer isoform may help regulate energy metabolism in the mitochondria. A pseudogene of this gene is found on chromosome 9. Alternative splicing and the use of multiple translation start codons results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene is a cytoplasmic nonreceptor protein kinase which may function as an intracellular signal transducer in epithelial tissues. Overexpression of this gene in mammary epithelial cells leads to sensitization of the cells to epidermal growth factor and results in a partially transformed phenotype. Expression of this gene has been detected at low levels in some breast tumors but not in normal breast tissue. The encoded protein has been shown to undergo autophosphorylation. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. N-terminal part of this PTP contains two tandem Src homolog (SH2) domains, which act as protein phospho-tyrosine binding domains, and mediate the interaction of this PTP with its substrates. This PTP is expressed primarily in hematopoietic cells, and functions as an important regulator of multiple signaling pathways in hematopoietic cells. This PTP has been shown to interact with, and dephosphorylate a wide spectrum of phospho-proteins involved in hematopoietic cell signaling. Multiple alternatively spliced variants of this gene, which encode distinct isoforms, have been reported.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP is a large intracellular protein. It has a catalytic PTP domain at its C-terminus and two major structural domains: a region with five PDZ domains and a FERM domain that binds to plasma membrane and cytoskeletal elements. This PTP was found to interact with, and dephosphorylate, Fas receptor and IkappaBalpha through the PDZ domains. This suggests it has a role in Fas mediated programmed cell death. This PTP was also shown to interact with GTPase-activating protein, and thus may function as a regulator of Rho signaling pathways. Four alternatively spliced transcript variants, which encode distinct proteins, have been reported.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP contains an extracellular domain, a single transmembrane segment and one intracytoplasmic catalytic domain, thus belongs to receptor type PTP. The extracellular region of this PTP is composed of multiple fibronectin type_III repeats, which was shown to interact with neuronal receptor and cell adhesion molecules, such as contactin and tenascin C. This protein was also found to interact with sodium channels, and thus may regulate sodium channels by altering tyrosine phosphorylation status. The functions of the interaction partners of this protein implicate the roles of this PTP in cell adhesion, neurite growth, and neuronal differentiation. Alternate transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitosis, and oncogenic transformation. This PTP contains an extracellular domain, a single transmembrane segment and two tandem intracytoplasmic catalytic domains, and thus is classified as a receptor type PTP. This PTP has been shown to be an essential regulator of T- and B-cell antigen receptor signaling. It functions through either direct interaction with components of the antigen receptor complexes, or by activating various Src family kinases required for the antigen receptor signaling. This PTP also suppresses JAK kinases, and thus functions as a regulator of cytokine receptor signaling. Alternatively spliced transcripts variants of this gene, which encode distinct isoforms, have been reported.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP contains an extracellular region, a single transmembrane segment and two tandem intracytoplasmic catalytic domains, and thus represents a receptor-type PTP. The extracellular region of this protein is composed of three Ig-like and eight fibronectin type III-like domains. Studies of the similar genes in chicken and fly suggest the role of this PTP is in promoting neurite growth, and regulating neurons axon guidance. Multiple alternatively spliced transcript variants of this gene have been reported. A related pseudogene has been identified on chromosome 5.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP possesses an extracellular region, a single transmembrane region, and two tandem catalytic domains, and thus represents a receptor-type PTP. The extracellular region contains a meprin-A5 antigen-PTP mu (MAM) domain, an Ig-like domain and four fibronectin type III-like repeats. This PTP was shown to mediate homophilic intercellular interaction, possibly through the interaction with beta- and gamma-catenin at adherens junctions. Expression of this gene was found to be stimulated by TGF-beta 1, which may be important for the inhibition of keratinocyte proliferation.","The protein encoded by this gene is highly similar to the gene product of Schizosaccharomyces pombe rad17, a cell cycle checkpoint gene required for cell cycle arrest and DNA damage repair in response to DNA damage. This protein shares strong similarity with DNA replication factor C (RFC), and can form a complex with RFCs. This protein binds to chromatin prior to DNA damage and is phosphorylated by the checkpoint kinase ATR following damage. This protein recruits the RAD1-RAD9-HUS1 checkpoint protein complex onto chromatin after DNA damage, which may be required for its phosphorylation. The phosphorylation of this protein is required for the DNA-damage-induced cell cycle G2 arrest, and is thought to be a critical early event during checkpoint signaling in DNA-damaged cells. Multiple alternatively spliced transcript variants of this gene, which encode four distinct protein isoforms, have been reported. Two pseudogenes, located on chromosomes 7 and 13, have been identified.","The protein encoded by this gene is highly similar to the gene product of Schizosaccharomyces pombe rad21, a gene involved in the repair of DNA double-strand breaks, as well as in chromatid cohesion during mitosis. This protein is a nuclear phospho-protein, which becomes hyperphosphorylated in cell cycle M phase. The highly regulated association of this protein with mitotic chromatin specifically at the centromere region suggests its role in sister chromatid cohesion in mitotic cells.","The protein encoded by this gene is a member of the RAD51 protein family. RAD51 family members are evolutionarily conserved proteins essential for DNA repair by homologous recombination. This protein has been shown to form a stable heterodimer with the family member RAD51C, which further interacts with the other family members, such as RAD51, XRCC2, and XRCC3. Overexpression of this gene was found to cause cell cycle G1 delay and cell apoptosis, which suggested a role of this protein in sensing DNA damage. Rearrangements between this locus and high mobility group AT-hook 2 (HMGA2, GeneID 8091) have been observed in uterine leiomyomata.","RAN is a small GTP-binding protein of the RAS superfamily that is associated with the nuclear membrane and is thought to control a variety of cellular functions through its interactions with other proteins. This gene encodes a very large RAN-binding protein that immunolocalizes to the nuclear pore complex. The protein is a giant scaffold and mosaic cyclophilin-related nucleoporin implicated in the Ran-GTPase cycle. The encoded protein directly interacts with the E2 enzyme UBC9 and strongly enhances SUMO1 transfer from UBC9 to the SUMO1 target SP100. These findings place sumoylation at the cytoplasmic filaments of the nuclear pore complex and suggest that, for some substrates, modification and nuclear import are linked events. This gene is partially duplicated in a gene cluster that lies in a hot spot for recombination on chromosome 2q.","The protein encoded by this gene is a negative regulator of the cell cycle and was the first tumor suppressor gene found. The encoded protein also stabilizes constitutive heterochromatin to maintain the overall chromatin structure. The active, hypophosphorylated form of the protein binds transcription factor E2F1. Defects in this gene are a cause of childhood cancer retinoblastoma (RB), bladder cancer, and osteogenic sarcoma.","The protein encoded by this gene belongs to the ROBO family, part of the immunoglobulin superfamily of proteins that are highly conserved from fly to human. The encoded protein is a transmembrane receptor for the slit homolog 2 protein and functions in axon guidance and cell migration. Mutations in this gene are associated with vesicoureteral reflux, characterized by the backward flow of urine from the bladder into the ureters or the kidney. Alternative splicing results in multiple transcript variants.","Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of four RNA species and approximately 80 structurally distinct proteins. This gene encodes a member of the L18P family of ribosomal proteins and component of the 60S subunit. The encoded protein binds 5S rRNA to form a stable complex called the 5S ribonucleoprotein particle (RNP), which is necessary for the transport of nonribosome-associated cytoplasmic 5S rRNA to the nucleolus for assembly into ribosomes. The encoded protein may also function to inhibit tumorigenesis through the activation of downstream tumor suppressors and the downregulation of oncoprotein expression. Mutations in this gene have been identified in patients with Diamond-Blackfan Anemia (DBA). This gene is co-transcribed with the small nucleolar RNA gene U21, which is located in its fifth intron. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed throughout the genome.","This gene encodes a ribosomal protein that is a component of the 60S ribosome subunit. The related protein in chicken can bind to c-Jun and can repress c-Jun-mediated transcriptional activation. Some studies have detected an association between variation in this gene and autism spectrum disorders, though others do not detect this relationship. There are multiple pseudogenes of this gene dispersed throughout the genome. Alternative splicing results in multiple transcript variants.","Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a cytoplasmic ribosomal protein that is a component of the 60S subunit. The protein belongs to the L22E family of ribosomal proteins. Its initiating methionine residue is post-translationally removed. The protein can bind specifically to Epstein-Barr virus-encoded RNAs (EBERs) 1 and 2. The mouse protein has been shown to be capable of binding to heparin. Transcript variants utilizing alternative polyA signals exist. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. It was previously thought that this gene mapped to 3q26 and that it was fused to the acute myeloid leukemia 1 (AML1) gene located at 21q22 in some therapy-related myelodysplastic syndrome patients with 3;21 translocations; however, these fusions actually involve a ribosomal protein L22 pseudogene located at 3q26, and this gene actually maps to 1p36.3-p36.2.","This gene encodes a major catalytic subunit of succinate-ubiquinone oxidoreductase, a complex of the mitochondrial respiratory chain. The complex is composed of four nuclear-encoded subunits and is localized in the mitochondrial inner membrane. Mutations in this gene have been associated with a form of mitochondrial respiratory chain deficiency known as Leigh Syndrome. A pseudogene has been identified on chromosome 3q29. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","This tumor suppressor gene encodes the iron-sulfur protein subunit of the succinate dehydrogenase (SDH) enzyme complex which plays a critical role in mitochondria. The SDH enzyme complex is composed of four nuclear-encoded subunits. This enzyme complex converts succinate to fumarate which releases electrons as part of the citric acid cycle, and the enzyme complex additionally provides an attachment site for released electrons to be transferred to the oxidative phosphorylation pathway. The SDH enzyme complex plays a role in oxygen-related gene regulation through its conversion of succinate, which is an oxygen sensor that stabilizes the hypoxia-inducible factor 1 (HIF1) transcription factor. Sporadic and familial mutations in this gene result in paragangliomas, pheochromocytoma, and gastrointestinal stromal tumors, supporting a link between mitochondrial dysfunction and tumorigenesis. Mutations in this gene are also implicated in nuclear type 4 mitochondrial complex II deficiency.","This gene encodes one of four nuclear-encoded subunits that comprise succinate dehydrogenase, also known as mitochondrial complex II, a key enzyme complex of the tricarboxylic acid cycle and aerobic respiratory chains of mitochondria. The encoded protein is one of two integral membrane proteins that anchor other subunits of the complex, which form the catalytic core, to the inner mitochondrial membrane. There are several related pseudogenes for this gene on different chromosomes. Mutations in this gene have been associated with paragangliomas. Alternatively spliced transcript variants have been described.","This gene encodes a member of complex II of the respiratory chain, which is responsible for the oxidation of succinate. The encoded protein is one of two integral membrane proteins anchoring the complex to the matrix side of the mitochondrial inner membrane. Mutations in this gene are associated with the formation of tumors, including hereditary paraganglioma. Transmission of disease occurs almost exclusively through the paternal allele, suggesting that this locus may be maternally imprinted. There are pseudogenes for this gene on chromosomes 1, 2, 3, 7, and 18. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the mitogen-activated protein kinase (MAPK) family. Members of this family act as an integration point for multiple biochemical signals and are involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation, and development. They form a three-tiered signaling module composed of MAPKKKs, MAPKKs, and MAPKs. This protein is phosphorylated at serine and threonine residues by MAPKKKs and subsequently phosphorylates downstream MAPK targets at threonine and tyrosine residues. A similar protein in mouse has been reported to play a role in liver organogenesis. A pseudogene of this gene is located on the long arm of chromosome X. Alternative splicing results in multiple transcript variants.","Enables DNA binding activity; histone deacetylase binding activity; and protein homodimerization activity. Involved in several processes, including alternative mRNA splicing, via spliceosome; positive regulation of oxidative stress-induced intrinsic apoptotic signaling pathway; and regulation of transcription by RNA polymerase II. Acts upstream of or within double-strand break repair via homologous recombination. Located in chromatin; nuclear matrix; and paraspeckles. [provided by Alliance of Genome Resources, Apr 2022]","Secreted frizzled-related protein 4 (SFRP4) is a member of the SFRP family that contains a cysteine-rich domain homologous to the putative Wnt-binding site of Frizzled proteins. SFRPs act as soluble modulators of Wnt signaling. The expression of SFRP4 in ventricular myocardium correlates with apoptosis related gene expression.","The protein encoded by this gene is a member of the SWI/SNF family of proteins and is similar to the brahma protein of Drosophila. Members of this family have helicase and ATPase activities and are thought to regulate transcription of certain genes by altering the chromatin structure around those genes. The encoded protein is part of the large ATP-dependent chromatin remodeling complex SNF/SWI, which is required for transcriptional activation of genes normally repressed by chromatin. In addition, this protein can bind BRCA1, as well as regulate the expression of the tumorigenic protein CD44. Mutations in this gene cause rhabdoid tumor predisposition syndrome type 2. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is part of a complex that relieves repressive chromatin structures, allowing the transcriptional machinery to access its targets more effectively. The encoded nuclear protein may also bind to and enhance the DNA joining activity of HIV-1 integrase. This gene has been found to be a tumor suppressor, and mutations in it have been associated with malignant rhabdoid tumors. Alternatively spliced transcript variants have been found for this gene.","The protein encoded by this gene is a member of the SWI/SNF family of proteins, whose members display helicase and ATPase activities and which are thought to regulate transcription of certain genes by altering the chromatin structure around those genes. The encoded protein is part of the large ATP-dependent chromatin remodeling complex SNF/SWI and has sequence similarity to the yeast Swp73 protein. Two transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is part of the large ATP-dependent chromatin remodeling complex SWI/SNF, which is required for transcriptional activation of genes normally repressed by chromatin. The encoded protein, either alone or when in the SWI/SNF complex, can bind to 4-way junction DNA, which is thought to mimic the topology of DNA as it enters or exits the nucleosome. The protein contains a DNA-binding HMG domain, but disruption of this domain does not abolish the DNA-binding or nucleosome-displacement activities of the SWI/SNF complex. Unlike most of the SWI/SNF complex proteins, this protein has no yeast counterpart.","The protein encoded by this gene recognizes the sequence CCTTGAG along with other members of the HMG-box class DNA-binding proteins. It acts during chondrocyte differentiation and, with steroidogenic factor 1, regulates transcription of the anti-Muellerian hormone (AMH) gene. Deficiencies lead to the skeletal malformation syndrome campomelic dysplasia, frequently with sex reversal.","The protein encoded by this gene is a member of the STAT family of transcription factors. In response to cytokines and growth factors, STAT family members are phosphorylated by the receptor associated kinases, and then form homo- or heterodimers that translocate to the cell nucleus where they act as transcription activators. This protein mediates the signal transduction triggered by various cell ligands, such as IL2, IL4, CSF1, and different growth hormones. It has been shown to be involved in diverse biological processes, such as TCR signaling, apoptosis, adult mammary gland development, and sexual dimorphism of liver gene expression. This gene was found to fuse to retinoic acid receptor-alpha (RARA) gene in a small subset of acute promyelocytic leukemias (APLL). The dysregulation of the signaling pathways mediated by this protein may be the cause of the APLL.","The protein encoded by this gene is a serine/threonine kinase that regulates cell polarity and energy metabolism and functions as a tumor suppressor. Mutations in this gene have been associated with the autosomal dominant Peutz-Jeghers syndrome, as well as with skin, pancreatic, and testicular cancers.","This gene is a member of a phylogenetically conserved family of genes that share a common DNA-binding domain, the T-box. T-box genes encode transcription factors involved in the regulation of developmental processes. This protein is a transcriptional repressor and is thought to play a role in the anterior/posterior axis of the tetrapod forelimb. Mutations in this gene cause ulnar-mammary syndrome, affecting limb, apocrine gland, tooth, hair, and genital development. Alternative splicing of this gene results in three transcript variants encoding different isoforms; however, the full length nature of one variant has not been determined.","The protein encoded by this gene is a transcription factor required for the expression of several liver-specific genes. The encoded protein functions as a homodimer and binds to the inverted palindrome 5'-GTTAATNATTAAC-3'. Defects in this gene are a cause of maturity onset diabetes of the young type 3 (MODY3) and also can result in the appearance of hepatic adenomas. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a member of the E protein (class I) family of helix-loop-helix transcription factors. E proteins activate transcription by binding to regulatory E-box sequences on target genes as heterodimers or homodimers, and are inhibited by heterodimerization with inhibitor of DNA-binding (class IV) helix-loop-helix proteins. E proteins play a critical role in lymphopoiesis, and the encoded protein is required for B and T lymphocyte development. Deletion of this gene or diminished activity of the encoded protein may play a role in lymphoid malignancies. This gene is also involved in several chromosomal translocations that are associated with lymphoid malignancies including pre-B-cell acute lymphoblastic leukemia (t(1;19), with PBX1), childhood leukemia (t(19;19), with TFPT) and acute leukemia (t(12;19), with ZNF384). Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene, and a pseudogene of this gene is located on the short arm of chromosome 9.","Telomerase is a ribonucleoprotein polymerase that maintains telomere ends by addition of the telomere repeat TTAGGG. The enzyme consists of a protein component with reverse transcriptase activity, encoded by this gene, and an RNA component which serves as a template for the telomere repeat. Telomerase expression plays a role in cellular senescence, as it is normally repressed in postnatal somatic cells resulting in progressive shortening of telomeres. Deregulation of telomerase expression in somatic cells may be involved in oncogenesis. Studies in mouse suggest that telomerase also participates in chromosomal repair, since de novo synthesis of telomere repeats may occur at double-stranded breaks. Alternatively spliced variants encoding different isoforms of telomerase reverse transcriptase have been identified; the full-length sequence of some variants has not been determined. Alternative splicing at this locus is thought to be one mechanism of regulation of telomerase activity.","The protein encoded by this gene is a transmembrane protein that has a protein kinase domain, forms a heterodimeric complex with TGF-beta receptor type-1, and binds TGF-beta. This receptor/ligand complex phosphorylates proteins, which then enter the nucleus and regulate the transcription of genes related to cell proliferation, cell cycle arrest, wound healing, immunosuppression, and tumorigenesis. Mutations in this gene have been associated with Marfan Syndrome, Loeys-Deitz Aortic Aneurysm Syndrome, and the development of various types of tumors. Alternatively spliced transcript variants encoding different isoforms have been characterized.","This gene encodes a protein initially identified as a thyroid-specific transcription factor. The encoded protein binds to the thyroglobulin promoter and regulates the expression of thyroid-specific genes but has also been shown to regulate the expression of genes involved in morphogenesis. Mutations and deletions in this gene are associated with benign hereditary chorea, choreoathetosis, congenital hypothyroidism, and neonatal respiratory distress, and may be associated with thyroid cancer. Multiple transcript variants encoding different isoforms have been found for this gene. This gene shares the symbol/alias 'TTF1' with another gene, transcription termination factor 1, which plays a role in ribosomal gene transcription.","This gene was identified as a gene whose expression is rapidly induced by the tumor necrosis factor (TNF). The protein encoded by this gene is a zinc finger protein and ubiqitin-editing enzyme, and has been shown to inhibit NF-kappa B activation as well as TNF-mediated apoptosis. The encoded protein, which has both ubiquitin ligase and deubiquitinase activities, is involved in the cytokine-mediated immune and inflammatory responses. Several transcript variants encoding the same protein have been found for this gene.","This gene encodes a tumor suppressor protein containing transcriptional activation, DNA binding, and oligomerization domains. The encoded protein responds to diverse cellular stresses to regulate expression of target genes, thereby inducing cell cycle arrest, apoptosis, senescence, DNA repair, or changes in metabolism. Mutations in this gene are associated with a variety of human cancers, including hereditary cancers such as Li-Fraumeni syndrome. Alternative splicing of this gene and the use of alternate promoters result in multiple transcript variants and isoforms. Additional isoforms have also been shown to result from the use of alternate translation initiation codons from identical transcript variants (PMIDs: 12032546, 20937277).","This gene encodes a member of the tropomyosin family of actin-binding proteins. Tropomyosins are dimers of coiled-coil proteins that provide stability to actin filaments and regulate access of other actin-binding proteins. Mutations in this gene result in autosomal dominant nemaline myopathy and other muscle disorders. This locus is involved in translocations with other loci, including anaplastic lymphoma receptor tyrosine kinase (ALK) and neurotrophic tyrosine kinase receptor type 1 (NTRK1), which result in the formation of fusion proteins that act as oncogenes. There are numerous pseudogenes for this gene on different chromosomes. Alternative splicing results in multiple transcript variants.","This gene is a tumor suppressor gene that encodes the growth inhibitory protein hamartin. The encoded protein interacts with and stabilizes the GTPase activating protein tuberin. This hamartin-tuberin complex negatively regulates mammalian target of rapamycin complex 1 (mTORC1) signaling which is a major regulator of anabolic cell growth. This protein also functions as a co-chaperone for Hsp90 that inhibits its ATPase activity. This protein functions as a facilitator of Hsp90-mediated folding of kinase and non-kinase clients, including TSC2 and thereby preventing their ubiquitination and proteasomal degradation. Mutations in this gene have been associated with tuberous sclerosis and lymphangioleiomyomatosis.","This gene is a tumor suppressor gene that encodes the growth inhibitory protein tuberin. Tuberin interacts with hamartin to form the TSC protein complex which functions in the control of cell growth. This TSC protein complex negatively regulates mammalian target of rapamycin complex 1 (mTORC1) signaling which is a major regulator of anabolic cell growth. Mutations in this gene have been associated with tuberous sclerosis and lymphangioleiomyomatosis.","This gene is located on the X chromosome and is the corresponding locus to a Y-linked gene which encodes a tetratricopeptide repeat (TPR) protein. The encoded protein of this gene contains a JmjC-domain and catalyzes the demethylation of tri/dimethylated histone H3. Multiple alternatively spliced transcript variants have been found for this gene.","This gene encodes a component of a ubiquitination complex. The encoded protein is involved in the ubiquitination and degradation of hypoxia-inducible-factor (HIF), which is a transcription factor that plays a central role in the regulation of gene expression by oxygen. In addition to oxygen-related gene expression, this protein plays a role in many other cellular processes including cilia formation, cytokine signaling, regulation of senescence, and formation of the extracellular matrix. Variants of this gene are associated with von Hippel-Lindau syndrome, pheochromocytoma, erythrocytosis, renal cell carcinoma, and cerebellar hemangioblastoma.","This gene encodes a member of the RecQ subfamily of DNA helicase proteins. The encoded nuclear protein is important in the maintenance of genome stability and plays a role in DNA repair, replication, transcription and telomere maintenance. This protein contains a N-terminal 3' to 5' exonuclease domain, an ATP-dependent helicase domain and RQC (RecQ helicase conserved region) domain in its central region, and a C-terminal HRDC (helicase RNase D C-terminal) domain and nuclear localization signal. Defects in this gene are the cause of Werner syndrome, an autosomal recessive disorder characterized by accelerated aging and an elevated risk for certain cancers.","This gene encodes a transcription factor that contains four zinc-finger motifs at the C-terminus and a proline/glutamine-rich DNA-binding domain at the N-terminus. It has an essential role in the normal development of the urogenital system, and it is mutated in a small subset of patients with Wilms tumor. This gene exhibits complex tissue-specific and polymorphic imprinting pattern, with biallelic, and monoallelic expression from the maternal and paternal alleles in different tissues. Multiple transcript variants have been described. In several variants, there is evidence for the use of a non-AUG (CUG) translation initiation codon upstream of, and in-frame with the first AUG. Authors of PMID:7926762 also provide evidence that WT1 mRNA undergoes RNA editing in human and rat, and that this process is tissue-restricted and developmentally regulated.","This gene encodes a zinc finger protein plays a central role in nucleotide excision repair (NER), a specialized type of DNA repair. NER is responsible for repair of UV radiation-induced photoproducts and DNA adducts induced by chemical carcinogens and chemotherapeutic drugs. The encoded protein interacts with DNA and several NER proteins, acting as a scaffold to assemble the NER incision complex at sites of DNA damage. Mutations in this gene cause Xeroderma pigmentosum complementation group A (XP-A), an autosomal recessive skin disorder featuring hypersensitivity to sunlight and increased risk for skin cancer.","The protein encoded by this gene is a key component of the XPC complex, which plays an important role in the early steps of global genome nucleotide excision repair (NER). The encoded protein is important for damage sensing and DNA binding, and shows a preference for single-stranded DNA. Mutations in this gene or some other NER components can result in Xeroderma pigmentosum, a rare autosomal recessive disorder characterized by increased sensitivity to sunlight with the development of carcinomas at an early age. Alternatively spliced transcript variants have been found for this gene.","This gene product belongs to the 14-3-3 family of proteins which mediate signal transduction by binding to phosphoserine-containing proteins. This highly conserved protein family is found in both plants and mammals, and this protein is 100% identical to the mouse ortholog. It interacts with CDC25 phosphatases, RAF1 and IRS1 proteins, suggesting its role in diverse biochemical activities related to signal transduction, such as cell division and regulation of insulin sensitivity. It has also been implicated in the pathogenesis of small cell lung cancer. Two transcript variants, one protein-coding and the other non-protein-coding, have been found for this gene.","This gene encodes a nucleic-acid binding protein with seven zinc-finger domains. The protein has a preference for binding single stranded DNA and RNA. The protein functions in cap-independent translation of ornithine decarboxylase mRNA, and may also function in sterol-mediated transcriptional regulation. A CCTG expansion from <30 repeats to 75-11000 repeats in the first intron of this gene results in myotonic dystrophy type 2. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene is a member of the Krueppel C2H2-type zinc-finger protein family and encodes a zinc finger transcription factor that contains nine Kruppel-type zinc finger domains at the carboxyl terminus. This protein is located in the nucleus, is involved in cell cycle progression, and interacts with a histone deacetylase. Specific instances of aberrant gene rearrangement at this locus have been associated with acute promyelocytic leukemia (APL). Alternate transcriptional splice variants have been characterized.","This tumor suppressor gene is a member of a nuclear histone/protein methyltransferase superfamily. It encodes a zinc finger protein that can bind to retinoblastoma protein, estrogen receptor, and the TPA-responsive element (MTE) of the heme-oxygenase-1 gene. Although the functions of this protein have not been fully characterized, it may (1) play a role in transcriptional regulation during neuronal differentiation and pathogenesis of retinoblastoma, (2) act as a transcriptional activator of the heme-oxygenase-1 gene, and (3) be a specific effector of estrogen action. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a coiled-coil domain-containing protein. The encoded protein is ubiquitously expressed and may function as a tumor suppressor. A chromosomal rearrangement resulting in the expression of a fusion gene containing a portion of this gene and the intracellular kinase-encoding domain of the ret proto-oncogene is the cause of thyroid papillary carcinoma.[provided by RefSeq, Sep 2010].","This gene encodes an androgen receptor coactivator. The encoded protein interacts with the androgen receptor in a ligand-dependent manner to enhance its transcriptional activity. Chromosomal translocations between this gene and the ret tyrosine kinase gene, also located on chromosome 10, have been associated with papillary thyroid carcinoma. Alternatively spliced transcript variants have been described. Pseudogenes are present on chromosomes 4, 5, 10, and 14.","The protein encoded by this gene is a histone methyltransferase that methylates the Lys-4 position of histone H3. The encoded protein is part of a large protein complex called ASCOM, which has been shown to be a transcriptional regulator of the beta-globin and estrogen receptor genes. Mutations in this gene have been shown to be a cause of Kabuki syndrome.","Enables phosphatase binding activity. Involved in positive regulation of transcription, DNA-templated and snRNA transcription. Located in cytosol; nuclear body; and transcriptionally active chromatin. Part of transcription elongation factor complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the BTB-kelch superfamily. Initially described as a putative transcriptional regulator based on weak homology to members of the basic leucine zipper-like family, the encoded protein subsequently has been shown to localize exclusively to the Golgi network where it may help stabilize the Gogli complex. Deletion of this gene may be associated with DiGeorge syndrome.","This gene is a member of the clathrin heavy chain family and encodes a major protein of the polyhedral coat of coated pits and vesicles. Chromosomal aberrations involving this gene are associated with meningioma, DiGeorge syndrome, and velo-cardio-facial syndrome. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes an essential splicing factor. The encoded protein associates with the U2 auxiliary factor heterodimer, which is required for the recognition of a functional 3' splice site in pre-mRNA splicing, and may play a role in network interactions during spliceosome assembly.","This gene encodes a nuclear protein that belongs to a family proteins that contain an RNA-binding motif. The encoded protein associates with hnRNP proteins and may be involved in regulating alternative splicing. Defects in this gene are the cause of the X-linked recessive disorder, TARP syndrome. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Mar 2011].","This gene is a member of the SMCY homolog family and encodes a protein with one ARID domain, one JmjC domain, one JmjN domain and two PHD-type zinc fingers. The DNA-binding motifs suggest this protein is involved in the regulation of transcription and chromatin remodeling. Mutations in this gene have been associated with X-linked cognitive disability. Alternative splicing results in multiple transcript variants.","Proper cohesion of sister chromatids is a prerequisite for the correct segregation of chromosomes during cell division. The cohesin multiprotein complex is required for sister chromatid cohesion. This complex is composed partly of two structural maintenance of chromosomes (SMC) proteins, SMC3 and either SMC1B or the protein encoded by this gene. Most of the cohesin complexes dissociate from the chromosomes before mitosis, although those complexes at the kinetochore remain. Therefore, the encoded protein is thought to be an important part of functional kinetochores. In addition, this protein interacts with BRCA1 and is phosphorylated by ATM, indicating a potential role for this protein in DNA repair. This gene, which belongs to the SMC gene family, is located in an area of the X-chromosome that escapes X inactivation. Mutations in this gene result in Cornelia de Lange syndrome. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a member of the SWI/SNF family, whose members have helicase and ATPase activities and are thought to regulate transcription of certain genes by altering the chromatin structure around those genes. The encoded protein is part of the large ATP-dependent chromatin remodeling complex SNF/SWI, which is required for transcriptional activation of genes normally repressed by chromatin. It possesses at least two conserved domains that could be important for its function. First, it has a DNA-binding domain that can specifically bind an AT-rich DNA sequence known to be recognized by a SNF/SWI complex at the beta-globin locus. Second, the C-terminus of the protein can stimulate glucocorticoid receptor-dependent transcriptional activation. It is thought that the protein encoded by this gene confers specificity to the SNF/SWI complex and may recruit the complex to its targets through either protein-DNA or protein-protein interactions. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a cytoplasmic protein which contains a regulation of G-protein signaling (RGS) domain and a dishevelled and axin (DIX) domain. The encoded protein interacts with adenomatosis polyposis coli, catenin beta-1, glycogen synthase kinase 3 beta, protein phosphate 2, and itself. This protein functions as a negative regulator of the wingless-type MMTV integration site family, member 1 (WNT) signaling pathway and can induce apoptosis. The crystal structure of a portion of this protein, alone and in a complex with other proteins, has been resolved. Mutations in this gene have been associated with hepatocellular carcinoma, hepatoblastomas, ovarian endometriod adenocarcinomas, and medullablastomas. Alternative splicing results in multiple transcript variants.","The Axin-related protein, Axin2, presumably plays an important role in the regulation of the stability of beta-catenin in the Wnt signaling pathway, like its rodent homologs, mouse conductin/rat axil. In mouse, conductin organizes a multiprotein complex of APC (adenomatous polyposis of the colon), beta-catenin, glycogen synthase kinase 3-beta, and conductin, which leads to the degradation of beta-catenin. Apparently, the deregulation of beta-catenin is an important event in the genesis of a number of malignancies. The AXIN2 gene has been mapped to 17q23-q24, a region that shows frequent loss of heterozygosity in breast cancer, neuroblastoma, and other tumors. Mutations in this gene have been associated with colorectal cancer with defective mismatch repair.","This gene belongs to the ubiquitin C-terminal hydrolase subfamily of deubiquitinating enzymes that are involved in the removal of ubiquitin from proteins. The encoded enzyme binds to the breast cancer type 1 susceptibility protein (BRCA1) via the RING finger domain of the latter and acts as a tumor suppressor. In addition, the enzyme may be involved in regulation of transcription, regulation of cell cycle and growth, response to DNA damage and chromatin dynamics. Germline mutations in this gene may be associated with tumor predisposition syndrome (TPDS), which involves increased risk of cancers including malignant mesothelioma, uveal melanoma and cutaneous melanoma.","This gene encodes a protein that may modulate the transcriptional repression activities of death-associated protein 6 (DAXX), which interacts with histone deacetylase, core histones, and other histone-associated proteins. In mouse, the encoded protein binds to the putative leucine zipper domain of macroH2A1.2, a variant H2A histone that is enriched on inactivated X chromosomes. The BTB/POZ domain of this protein has been shown in other proteins to mediate transcriptional repression and to interact with components of histone deacetylase co-repressor complexes. Alternative splicing of this gene results in multiple transcript variants encoding the same protein.","This gene encodes a member of the cullin protein family. The encoded protein plays a critical role in the polyubiquitination and subsequent degradation of specific protein substrates as the core component and scaffold protein of an E3 ubiquitin ligase complex. Complexes including the encoded protein may also play a role in late endosome maturation. Mutations in this gene are a cause of type 2E pseudohypoaldosteronism. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","IRS4 encodes the insulin receptor substrate 4, a cytoplasmic protein that contains many potential tyrosine and serine/threonine phosphorylation sites. Tyrosine-phosphorylated IRS4 protein has been shown to associate with cytoplasmic signalling molecules that contain SH2 domains. The IRS4 protein is phosphorylated by the insulin receptor tyrosine kinase upon receptor stimulation..","This gene encodes a member of the p53 family of transcription factors. The functional domains of p53 family proteins include an N-terminal transactivation domain, a central DNA-binding domain and an oligomerization domain. Alternative splicing of this gene and the use of alternative promoters results in multiple transcript variants encoding different isoforms that vary in their functional properties. These isoforms function during skin development and maintenance, adult stem/progenitor cell regulation, heart development and premature aging. Some isoforms have been found to protect the germline by eliminating oocytes or testicular germ cells that have suffered DNA damage. Mutations in this gene are associated with ectodermal dysplasia, and cleft lip/palate syndrome 3 (EEC3); split-hand/foot malformation 4 (SHFM4); ankyloblepharon-ectodermal defects-cleft lip/palate; ADULT syndrome (acro-dermato-ungual-lacrimal-tooth); limb-mammary syndrome; Rap-Hodgkin syndrome (RHS); and orofacial cleft 8.","This gene encodes a member of the STAT-induced STAT inhibitor (SSI), also known as suppressor of cytokine signaling (SOCS), family. SSI family members are cytokine-inducible negative regulators of cytokine signaling. The expression of this gene can be induced by a subset of cytokines, including IL2, IL3 erythropoietin (EPO), CSF2/GM-CSF, and interferon (IFN)-gamma. The protein encoded by this gene functions downstream of cytokine receptors, and takes part in a negative feedback loop to attenuate cytokine signaling. Knockout studies in mice suggested the role of this gene as a modulator of IFN-gamma action, which is required for normal postnatal growth and survival.","This gene encodes a member of the Polycomb-group (PcG) family. PcG family members form multimeric protein complexes, which are involved in maintaining the transcriptional repressive state of genes over successive cell generations. This protein interacts with enhancer of zeste 2, the cytoplasmic tail of integrin beta7, immunodeficiency virus type 1 (HIV-1) MA protein, and histone deacetylase proteins. This protein mediates repression of gene activity through histone deacetylation, and may act as a specific regulator of integrin function. Two transcript variants encoding distinct isoforms have been identified for this gene.","This gene encodes a member of the TNF (tumor necrosis factor) receptor superfamily. The encoded protein functions in signal transduction pathways that activate inflammatory and inhibitory T-cell immune response. It binds herpes simplex virus (HSV) viral envelope glycoprotein D (gD), mediating its entry into cells. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene mediates transcriptional control by interaction with the activation function 2 (AF2) region of several nuclear receptors, including the estrogen, retinoic acid, and vitamin D3 receptors. The protein localizes to nuclear bodies and is thought to associate with chromatin and heterochromatin-associated factors. The protein is a member of the tripartite motif (TRIM) family. The TRIM motif includes three zinc-binding domains - a RING, a B-box type 1 and a B-box type 2 - and a coiled-coil region. Two alternatively spliced transcript variants encoding different isoforms have been described for this gene.","The protein encoded by this gene is a single stranded DNA-binding protein that binds to multiple DNA elements, including the far upstream element (FUSE) located upstream of c-myc. Binding to FUSE occurs on the non-coding strand, and is important to the regulation of c-myc in undifferentiated cells. This protein contains three domains, an amphipathic helix N-terminal domain, a DNA-binding central domain, and a C-terminal transactivation domain that contains three tyrosine-rich motifs. The N-terminal domain is thought to repress the activity of the C-terminal domain. This protein is also thought to bind RNA, and contains 3'-5' helicase activity with in vitro activity on both DNA-DNA and RNA-RNA duplexes. Aberrant expression of this gene has been found in malignant tissues, and this gene is important to neural system and lung development. Binding of this protein to viral RNA is thought to play a role in several viral diseases, including hepatitis C and hand, foot and mouth disease. Alternative splicing results in multiple transcript variants.","This gene was identified by its translocation in a case of mucosa-associated lymphoid tissue (MALT) lymphoma. The protein encoded by this gene contains a caspase recruitment domain (CARD), and has been shown to induce apoptosis and to activate NF-kappaB. This protein is reported to interact with other CARD domain containing proteins including CARD9, 10, 11 and 14, which are thought to function as upstream regulators in NF-kappaB signaling. This protein is found to form a complex with MALT1, a protein encoded by another gene known to be translocated in MALT lymphoma. MALT1 and this protein are thought to synergize in the activation of NF-kappaB, and the deregulation of either of them may contribute to the same pathogenetic process that leads to the malignancy. Alternative splicing results in multiple transcript variants.","The DNA-associated protein encoded by this gene is a member of the paired family of homeobox proteins localized to the nucleus. The protein functions as a transcription factor involved in the development of several major noradrenergic neuron populations and the determination of neurotransmitter phenotype. The gene product is linked to enhancement of second messenger-mediated activation of the dopamine beta-hydroylase, c-fos promoters and several enhancers, including cyclic amp-response element and serum-response element. Expansion of a 20 amino acid polyalanine tract in this protein by 5-13 aa has been associated with congenital central hypoventilation syndrome.","The protein encoded by this gene is a putative serine/threonine kinase that localizes to the mitotic apparatus and complexes with cell cycle controller CDC2 kinase in early mitosis. The protein is phosphorylated in a cell-cycle dependent manner, with late prophase phosphorylation remaining through metaphase. The N-terminal region of the protein binds CDC2 to form a complex showing reduced H1 histone kinase activity, indicating a role as a negative regulator of CDC2/cyclin A. In addition, the C-terminal kinase domain binds to its own N-terminal region, suggesting potential negative regulation through interference with complex formation via intramolecular binding. Biochemical and genetic data suggest a role as a tumor suppressor. This is supported by studies in knockout mice showing development of soft-tissue sarcomas, ovarian stromal cell tumors and a high sensitivity to carcinogenic treatments.","The protein encoded by this gene is a member of serine/threonine protein kinase family. This kinase contains a dual leucine-zipper motif, and has been shown to form dimers/oligomers through its leucine-zipper motif. This kinase can phosphorylate and activate MAPK8/JNK, MAP2K7/MKK7, which suggests a role in the JNK signaling pathway.","This gene is located on the X chromosome and is subject to X inactivation. It is highly conserved in vertebrates and most abundantly expressed in the brain. The encoded protein is a component of histone deacetylase-containing multiprotein complexes that function through modifying chromatin structure to keep genes silent. A chromosomal translocation (X;13) involving this gene is associated with X-linked cognitive disability. Several alternatively spliced transcript variants have been found for this gene.","This gene encodes a protein that belongs to the Kruppel family of transcription factors. The encoded zinc finger protein is required for normal development of the barrier function of skin. The encoded protein is thought to control the G1-to-S transition of the cell cycle following DNA damage by mediating the tumor suppressor gene p53. Mice lacking this gene have a normal appearance but lose weight rapidly, and die shortly after birth due to fluid evaporation resulting from compromised epidermal barrier function. Alternative splicing results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene is a DNA helicase that belongs to the RecQ helicase family. DNA helicases unwind double-stranded DNA into single-stranded DNAs and may modulate chromosome segregation. This gene is predominantly expressed in thymus and testis. Mutations in this gene are associated with Rothmund-Thomson, RAPADILINO and Baller-Gerold syndromes.","The protein encoded by this gene is an RNA-binding protein that regulates pre-mRNA splicing, export of mRNAs from the nucleus, protein translation, and mRNA stability. The encoded protein is involved in myelinization and oligodendrocyte differentiation and may play a role in schizophrenia. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene is a member of the cytidine deaminase gene family. It is one of seven related genes or pseudogenes found in a cluster, thought to result from gene duplication, on chromosome 22. Members of the cluster encode proteins that are structurally and functionally related to the C to U RNA-editing cytidine deaminase APOBEC1. It is thought that the proteins may be RNA editing enzymes and have roles in growth or cell cycle control. A hybrid gene results from the deletion of approximately 29.5 kb of sequence between this gene, APOBEC3B, and the adjacent gene APOBEC3A. The breakpoints of the deletion are within the two genes, so the deletion allele is predicted to have the promoter and coding region of APOBEC3A, but the 3' UTR of APOBEC3B. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a protein that mediates ligand-independent transcription repression of thyroid-hormone and retinoic-acid receptors by promoting chromatin condensation and preventing access of the transcription machinery. It is part of a complex which also includes histone deacetylases and transcriptional regulators similar to the yeast protein Sin3p. This gene is located between the Charcot-Marie-Tooth and Smith-Magenis syndrome critical regions on chromosome 17. Alternate splicing results in multiple transcript variants. Pseudogenes of this gene are found on chromosomes 17 and 20.[provided by RefSeq, Jun 2010].","This gene encodes a nuclear receptor co-repressor that mediates transcriptional silencing of certain target genes. The encoded protein is a member of a family of thyroid hormone- and retinoic acid receptor-associated co-repressors. This protein acts as part of a multisubunit complex which includes histone deacetylases to modify chromatin structure that prevents basal transcriptional activity of target genes. Aberrant expression of this gene is associated with certain cancers. Alternate splicing results in multiple transcript variants encoding different isoforms.[provided by RefSeq, Apr 2011].","This gene encodes a Rho guanine nucleotide exchange factor (GEF). Rho GEFs regulate the activity of small Rho GTPases by stimulating the exchange of guanine diphosphate (GDP) for guanine triphosphate (GTP) and may play a role in neural morphogenesis. Mutations in this gene are associated with slowed nerve conduction velocity (SNCV). Alternative splicing results in multiple transcript variants.","This gene encodes a protein containing KELCH-1 like domains, as well as a BTB/POZ domain. Kelch-like ECH-associated protein 1 interacts with NF-E2-related factor 2 in a redox-sensitive manner and the dissociation of the proteins in the cytoplasm is followed by transportation of NF-E2-related factor 2 to the nucleus. This interaction results in the expression of the catalytic subunit of gamma-glutamylcysteine synthetase. Two alternatively spliced transcript variants encoding the same isoform have been found for this gene.","The initiation of transcription is controlled in part by a large protein assembly known as the preinitiation complex. A component of this preinitiation complex is a 1.2 MDa protein aggregate called Mediator. This Mediator component binds with a CDK8 subcomplex which contains the protein encoded by this gene, mediator complex subunit 12 (MED12), along with MED13, CDK8 kinase, and cyclin C. The CDK8 subcomplex modulates Mediator-polymerase II interactions and thereby regulates transcription initiation and reinitation rates. The MED12 protein is essential for activating CDK8 kinase. Defects in this gene cause X-linked Opitz-Kaveggia syndrome, also known as FG syndrome, and Lujan-Fryns syndrome.","This gene encodes a member of the Abelson-interactor family of adaptor proteins. These proteins facilitate signal transduction as components of several multiprotein complexes, and regulate actin polymerization and cytoskeletal remodeling through interactions with Abelson tyrosine kinases. The encoded protein plays a role in macropinocytosis as a component of the WAVE2 complex, and also forms a complex with EPS8 and SOS1 that mediates signal transduction from Ras to Rac. This gene may play a role in the progression of several malignancies including melanoma, colon cancer and breast cancer, and a t(10;11) chromosomal translocation involving this gene and the MLL gene has been associated with acute myeloid leukemia. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene, and a pseudogene of this gene is located on the long arm of chromosome 14.","This gene encodes a member of the SH2B adaptor family of proteins, which are involved in a range of signaling activities by growth factor and cytokine receptors. The encoded protein is a key negative regulator of cytokine signaling and plays a critical role in hematopoiesis. Mutations in this gene have been associated with susceptibility to celiac disease type 13 and susceptibility to insulin-dependent diabetes mellitus. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","This gene is a member of the SCC3 family and is expressed in the nucleus. It encodes a component of cohesin, a multisubunit protein complex that provides sister chromatid cohesion along the length of a chromosome from DNA replication through prophase and prometaphase, after which it is dissociated in preparation for segregation during anaphase.","This gene encodes a transcription factor that belongs to the family of zinc-finger DNA-binding proteins associated with chromatin remodeling. The expression of this protein is restricted to the fetal and adult hemo-lymphopoietic system, and it functions as a regulator of lymphocyte differentiation. Several alternatively spliced transcript variants encoding different isoforms have been described for this gene. Most isoforms share a common C-terminal domain, which contains two zinc finger motifs that are required for hetero- or homo-dimerization, and for interactions with other proteins. The isoforms, however, differ in the number of N-terminal zinc finger motifs that bind DNA and in nuclear localization signal presence, resulting in members with and without DNA-binding properties. Only a few isoforms contain the requisite three or more N-terminal zinc motifs that confer high affinity binding to a specific core DNA sequence element in the promoters of target genes. The non-DNA-binding isoforms are largely found in the cytoplasm, and are thought to function as dominant-negative factors. Overexpression of some dominant-negative isoforms have been associated with B-cell malignancies, such as acute lymphoblastic leukemia (ALL).","This gene is a member of the N-myc downregulated gene family which belongs to the alpha/beta hydrolase superfamily. The protein encoded by this gene is a cytoplasmic protein involved in stress responses, hormone responses, cell growth, and differentiation. The encoded protein is necessary for p53-mediated caspase activation and apoptosis. Mutations in this gene are a cause of Charcot-Marie-Tooth disease type 4D, and expression of this gene may be a prognostic indicator for several types of cancer. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","The protein encoded by this gene is a pH-sensitive sodium-dependent phosphate transporter. Phosphate uptake is increased at lower pH. Defects in this gene are a cause of pulmonary alveolar microlithiasis. Three transcript variants encoding two different isoforms have been found for this gene.","This gene encodes a protein that binds the 5' UTR of insulin-like growth factor 2 (IGF2) mRNA and regulates its translation. It plays an important role in metabolism and variation in this gene is associated with susceptibility to diabetes. Alternative splicing and promoter usage results in multiple transcript variants. Related pseudogenes are found on several chromosomes.","This gene is a member of the BORIS + CTCF gene family and encodes a transcriptional regulator protein with 11 highly conserved zinc finger (ZF) domains. This nuclear protein is able to use different combinations of the ZF domains to bind different DNA target sequences and proteins. Depending upon the context of the site, the protein can bind a histone acetyltransferase (HAT)-containing complex and function as a transcriptional activator or bind a histone deacetylase (HDAC)-containing complex and function as a transcriptional repressor. If the protein is bound to a transcriptional insulator element, it can block communication between enhancers and upstream promoters, thereby regulating imprinted expression. Mutations in this gene have been associated with invasive breast cancers, prostate cancers, and Wilms' tumors. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","Enables catalytic activity, acting on DNA; chromatin binding activity; and identical protein binding activity. Involved in DNA repair; negative regulation of double-strand break repair via homologous recombination; and protein homooligomerization. Located in Golgi apparatus; cytosol; and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a subunit of the cohesin complex, which regulates the separation of sister chromatids during cell division. Targeted inactivation of this gene results in chromatid cohesion defects and aneuploidy, suggesting that genetic disruption of cohesin is a cause of aneuploidy in human cancer. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP possesses an extracellular region, a single transmembrane region, and two tandem intracellular catalytic domains, and thus represents a receptor-type PTP. The extracellular region contains a meprin-A5 antigen-PTP (MAM) domain, Ig-like and fibronectin type III-like repeats. The protein domain structure and the expression pattern of the mouse counterpart of this PTP suggest its roles in both signal transduction and cellular adhesion in the central nervous system. Two alternatively spliced transcript variants of this gene, which encode distinct proteins, have been reported.","SRY-related HMG-box (SOX) genes encode a family of DNA-binding proteins containing a 79-amino acid HMG (high mobility group) domain that shares at least 50% sequence identity with the DNA-binding HMG box of the SRY protein (MIM 480000). SOX proteins are divided into 6 subgroups based on sequence similarity within and outside of the HMG domain. For additional background information on SOX genes, see SOX1 (MIM 602148).","The BAZ1A gene encodes the accessory subunit of the ATP-dependent chromatin assembly factor (ACF), a member of the ISWI ('imitation switch') family of chromatin remodeling complexes (summarized by Racki et al., 2009).","The protein encoded by this gene functions to inhibit WNT proteins, which are extracellular signaling molecules that play a role in embryonic development. This protein contains a WNT inhibitory factor (WIF) domain and five epidermal growth factor (EGF)-like domains, and is thought to be involved in mesoderm segmentation. This gene functions as a tumor suppressor gene, and has been found to be epigenetically silenced in various cancers.","In response to DNA damage and replication blocks, cell cycle progression is halted through the control of critical cell cycle regulators. The protein encoded by this gene is a cell cycle checkpoint regulator and putative tumor suppressor. It contains a forkhead-associated protein interaction domain essential for activation in response to DNA damage and is rapidly phosphorylated in response to replication blocks and DNA damage. When activated, the encoded protein is known to inhibit CDC25C phosphatase, preventing entry into mitosis, and has been shown to stabilize the tumor suppressor protein p53, leading to cell cycle arrest in G1. In addition, this protein interacts with and phosphorylates BRCA1, allowing BRCA1 to restore survival after DNA damage. Mutations in this gene have been linked with Li-Fraumeni syndrome, a highly penetrant familial cancer phenotype usually associated with inherited mutations in TP53. Also, mutations in this gene are thought to confer a predisposition to sarcomas, breast cancer, and brain tumors. This nuclear protein is a member of the CDS1 subfamily of serine/threonine protein kinases. Several transcript variants encoding different isoforms have been found for this gene.","Enables mRNA 3'-UTR binding activity and translation factor activity, RNA binding. Involved in cellular response to amino acid stimulus; negative regulation of transcription by RNA polymerase II; and positive regulation of mRNA catabolic process. Located in several cellular components, including cytosol; midbody; and nucleoplasm. Part of CCR4-NOT complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a hormone inducible transcriptional repressor. Repression of transcription by this gene product can occur through interactions with other repressors, by the recruitment of proteins involved in histone deacetylation, or through sequestration of transcriptional activators. The product of this gene contains a carboxy-terminal domain that permits binding to other corepressor proteins. This domain also permits interaction with members of the NuRD complex, a nucleosome remodeling protein complex that contains deacetylase activity. In addition, this repressor contains several RNA recognition motifs that confer binding to a steroid receptor RNA coactivator; this binding can modulate the activity of both liganded and nonliganded steroid receptors.","SET1B is a component of a histone methyltransferase complex that produces trimethylated histone H3 at Lys4 (Lee et al., 2007).","Interaction of a cell with the extracellular matrix triggers integrin cell surface receptors to begin signaling cascades that regulate the organization of the actin-cytoskeleton. One of the proteins involved in these cascades is focal adhesion kinase. The protein encoded by this gene is a GTPase activating protein that binds to focal adhesion kinase and mediates the activity of the GTP binding proteins RhoA and Cdc42. Defects in this gene are a cause of juvenile myelomonocytic leukemia (JMML). Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is an ortholog of the Drosophila melanogaster capicua gene, and is a member of the high mobility group (HMG)-box superfamily of transcriptional repressors. This protein contains a conserved HMG domain that is involved in DNA binding and nuclear localization, and a conserved C-terminus. Studies suggest that the N-terminal region of this protein interacts with Atxn1 (GeneID:6310), to form a transcription repressor complex, and in vitro studies suggest that polyglutamine-expansion of ATXN1 may alter the repressor activity of this complex. Mutations in this gene have been associated with olidogdendrogliomas (PMID:21817013). In addition, translocation events resulting in gene fusions of this gene with both DUX4 (GeneID:100288687) and FOXO4 (GeneID:4303) have been associated with round cell sarcomas. There are multiple pseudogenes of this gene found on chromosomes 1, 4, 6, 7, 16, 20, and the Y chromosome. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a member of an evolutionarily conserved protein family implicated in RNA metabolism and translation. Members of this family are characterized by the presence of an La motif, which is often located adjacent to one or more RNA recognition motifs (RRM). Together, the two motifs constitute the functional region of the protein and enable its interaction with the RNA substrate. This protein family is divided into five sub-families: the genuine La proteins and four La-related protein (LARP) sub-families. The protein encoded by this gene belongs to LARP sub-family 4. It is a cytoplasmic protein that may play a stimulatory role in translation.","The protein encoded by this gene contains a CG1 DNA-binding domain, a transcription factor immunoglobulin domain, ankyrin repeats, and calmodulin-binding IQ motifs. The encoded protein is thought to be a transcription factor and may be a tumor suppressor. However, a translocation event is sometimes observed between this gene and the WWTR1 gene, with the resulting WWTR1-CAMTA1 oncoprotein leading to epithelioid hemangioendothelioma, a malignant vascular cancer.","Rho GTPases play a fundamental role in numerous cellular processes that are initiated by extracellular stimuli working through G protein-coupled receptors. The encoded protein may form a complex with G proteins and stimulate Rho-dependent signals. This protein has been observed to form a myeloid/lymphoid fusion partner in acute myeloid leukemia. Three transcript variants encoding different isoforms have been found for this gene.","This gene encodes a protein possessing an RNA helicase motif containing a DEXH box in its amino terminus and an RNA motif in the carboxy terminus. The encoded protein functions as a ribonuclease and is required by the RNA interference and small temporal RNA (stRNA) pathways to produce the active small RNA component that represses gene expression. This protein also acts as a strong antiviral agent with activity against RNA viruses, including the Zika and SARS-CoV-2 viruses. Alternative splicing results in multiple transcript variants.","Enables identical protein binding activity. Predicted to be involved in late endosome to vacuole transport via multivesicular body sorting pathway and negative regulation of growth hormone receptor signaling pathway. Predicted to be integral component of membrane. Predicted to be active in endosome. [provided by Alliance of Genome Resources, Apr 2022]","This zinc finger gene has been identified at the breakpoints of a recurrent chromosomal translocation reported in endometrial stromal sarcoma. Recombination of these breakpoints results in the fusion of this gene and JAZF1. The protein encoded by this gene contains a zinc finger domain in the C terminus of the coding region.","The protein encoded by this gene is a histone acetyltransferase and component of the MOZ/MORF protein complex. In addition to its acetyltransferase activity, the encoded protein has transcriptional activation activity in its N-terminal end and transcriptional repression activity in its C-terminal end. This protein is necessary for RUNX2-dependent transcriptional activation and could be involved in brain development. Mutations have been found in patients with genitopatellar syndrome. A translocation of this gene and the CREBBP gene results in acute myeloid leukemias. Three transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene contains an A-T hook DNA binding motif which usually binds to other DNA binding structures to play an important role in chromatin modeling and transcription regulation. Its Poz domain is thought to function as a site for protein-protein interaction and is required for transcriptional repression, and the zinc-fingers comprise the DNA binding domain. Since the encoded protein has typical features of a transcription factor, it is postulated to be a repressor of gene expression. In small round cell sarcoma, this gene is fused to EWS by a small inversion of 22q, then the hybrid is thought to be translocated (t(1;22)(p36.1;q12). The rearrangement of chromosome 22 involves intron 8 of EWS and exon 1 of this gene creating a chimeric sequence containing the transactivation domain of EWS fused to zinc finger domain of this protein. This is a distinct example of an intra-chromosomal rearrangement of chromosome 22. Four alternatively spliced transcript variants are described for this gene.","This gene encodes a member of the Cbl family of E3 ubiquitin ligases. Cbl proteins play important roles in cell signaling through the ubiquitination and subsequent downregulation of tyrosine kinases. Expression of this gene may be restricted to epithelial cells, and alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","This gene is a member of the telombin family and encodes a nuclear protein involved in telomere maintenance. Specifically, this protein functions as a member of a multi-protein complex that binds to the TTAGGG repeats of telomeres, regulating telomere length and protecting chromosome ends from illegitimate recombination, catastrophic chromosome instability, and abnormal chromosome segregation. Increased transcriptional expression of this gene is associated with stomach carcinogenesis and its progression. Alternatively spliced transcript variants have been described.","This gene encodes a serine/threonine protein kinase belonging to the LATS tumor suppressor family. The protein localizes to centrosomes during interphase, and early and late metaphase. It interacts with the centrosomal proteins aurora-A and ajuba and is required for accumulation of gamma-tubulin and spindle formation at the onset of mitosis. It also interacts with a negative regulator of p53 and may function in a positive feedback loop with p53 that responds to cytoskeleton damage. Additionally, it can function as a co-repressor of androgen-responsive gene expression.","This gene encodes a poly(A) binding protein. The protein shuttles between the nucleus and cytoplasm and binds to the 3' poly(A) tail of eukaryotic messenger RNAs via RNA-recognition motifs. The binding of this protein to poly(A) promotes ribosome recruitment and translation initiation; it is also required for poly(A) shortening which is the first step in mRNA decay. The gene is part of a small gene family including three protein-coding genes and several pseudogenes.[provided by RefSeq, Aug 2010].","Huntington's disease (HD), a neurodegenerative disorder characterized by loss of striatal neurons, is caused by an expansion of a polyglutamine tract in the HD protein huntingtin. This gene encodes a protein belonging to a class of huntingtin interacting proteins characterized by WW motifs. This protein is a histone methyltransferase that is specific for lysine-36 of histone H3, and methylation of this residue is associated with active chromatin. This protein also contains a novel transcriptional activation domain and has been found associated with hyperphosphorylated RNA polymerase II.","This gene encodes a ribonuclease (RNase) III double-stranded RNA-specific ribonuclease and subunit of the microprocessor protein complex, which catalyzes the initial processing step of microRNA (miRNA) synthesis. The encoded protein cleaves the stem loop structure from the primary microRNA (pri-miRNA) in the nucleus, yielding the precursor miRNA (pre-miRNA), which is then exported to the cytoplasm for further processing. In a human cell line lacking a functional copy of this gene, canonical miRNA synthesis is reduced. Somatic mutations in this gene have been observed in human patients with kidney cancer.","This gene encodes an immune inhibitory receptor ligand that is expressed by hematopoietic and non-hematopoietic cells, such as T cells and B cells and various types of tumor cells. The encoded protein is a type I transmembrane protein that has immunoglobulin V-like and C-like domains. Interaction of this ligand with its receptor inhibits T-cell activation and cytokine production. During infection or inflammation of normal tissue, this interaction is important for preventing autoimmunity by maintaining homeostasis of the immune response. In tumor microenvironments, this interaction provides an immune escape for tumor cells through cytotoxic T-cell inactivation. Expression of this gene in tumor cells is considered to be prognostic in many types of human malignancies, including colon cancer and renal cell carcinoma. Alternative splicing results in multiple transcript variants.","This gene encodes a highly conserved protein that plays an essential role in ribosome biogenesis. The encoded protein interacts with elongation factor-like GTPase 1 to disassociate eukaryotic initiation factor 6 from the late cytoplasmic pre-60S ribosomal subunit allowing assembly of the 80S subunit. Mutations within this gene are associated with the autosomal recessive disorder Shwachman-Bodian-Diamond syndrome. This gene has a closely linked pseudogene that is distally located.","This gene encodes a transcription factor belonging to a family of proteins that share homology with the high mobility group protein-1. The protein encoded by this gene can bind to a functionally important site in the T-cell receptor-alpha enhancer, thereby conferring maximal enhancer activity. This transcription factor is involved in the Wnt signaling pathway, and it may function in hair cell differentiation and follicle morphogenesis. Mutations in this gene have been found in somatic sebaceous tumors. This gene has also been linked to other cancers, including androgen-independent prostate cancer. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is thought to be a transcriptional corepressor. However, molecules that interact with this protein have not yet been identified. The protein is a member of the tripartite motif family. This motif includes three zinc-binding domains, a RING, a B-box type 1 and a B-box type 2, and a coiled-coil region. Three alternatively spliced transcript variants for this gene have been described, however, the full-length nature of one variant has not been determined.","The Hedgehog signaling pathway plays an important role in early human development. The pathway is a signaling cascade that plays a role in pattern formation and cellular proliferation during development. This gene encodes a negative regulator of the hedgehog signaling pathway. Defects in this gene are a cause of medulloblastoma. Alternative splicing results in multiple transcript variants.[provided by RefSeq, May 2010].","Enables RNA polymerase II CTD heptapeptide repeat kinase activity and cyclin binding activity. Involved in phosphorylation of RNA polymerase II C-terminal domain; protein autophosphorylation; and regulation of MAP kinase activity. Located in nuclear speck. Part of cyclin K-CDK12 complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the low density lipoprotein (LDL) receptor family. These receptors play a wide variety of roles in normal cell function and development due to their interactions with multiple ligands. Disruption of this gene has been reported in several types of cancer.","The protein encoded by this gene is a methylcytosine dioxygenase that catalyzes the conversion of methylcytosine to 5-hydroxymethylcytosine. The encoded protein is involved in myelopoiesis, and defects in this gene have been associated with several myeloproliferative disorders. Two variants encoding different isoforms have been found for this gene.","Enables RNA adenylyltransferase activity. Involved in mRNA stabilization. Located in cytoplasm and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene was identified as an interacting corepressor of BCL6, a POZ/zinc finger transcription repressor that is required for germinal center formation and may influence apoptosis. This protein selectively interacts with the POZ domain of BCL6, but not with eight other POZ proteins. Specific class I and II histone deacetylases (HDACs) have been shown to interact with this protein, which suggests a possible link between the two classes of HDACs. Several transcript variants encoding different isoforms have been found for this gene. A pseudogene of this gene is found on chromosome Y.[provided by RefSeq, Jun 2010].","The protein encoded by this gene is a RING-type E3 ubiquitin ligase and is predicted to contain a transmembrane domain, a protease-associated domain, an ectodomain, and a cytoplasmic RING domain. This protein is thought to negatively regulate Wnt signaling, and expression of this gene results in an increase in ubiquitination of frizzled receptors, an alteration in their subcellular distribution, resulting in reduced surface levels of these receptors. Mutations in this gene have been reported in multiple tumor cells, including colorectal and endometrial cancers. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a mitochondrial assembly factor needed for the flavination of a succinate dehydrogenase complex subunit (SDHA), which is required for activity of the succinate dehydrogenase complex. Mutations in this gene are associated with paraganglioma.","Enables MDM2/MDM4 family protein binding activity; p53 binding activity; and ubiquitin protein ligase activity. Involved in several processes, including DNA metabolic process; regulation of cell cycle phase transition; and response to ionizing radiation. Located in nucleoplasm and site of DNA damage. Colocalizes with site of double-strand break. Implicated in Fanconi anemia. [provided by Alliance of Genome Resources, Apr 2022]","This gene belongs to the RhoGEF subfamily of RhoGTPases. Members of this subfamily are activated by specific guanine nucleotide exchange factors (GEFs) and are involved in signal transduction. The encoded protein shows cytosolic distribution. Alternative splicing results in multiple transcript variants.","This locus encodes a subunit of ATP-dependent chromatin-remodeling complexes. The encoded protein has been identified as in integral component of complexes necessary for ligand-dependent transcriptional activation by nuclear hormone receptors. Mutations at this locus have been associated with primary clear cell renal cell carcinoma.","This gene encodes a member of a family of epigenetic regulators that bind various histone-modifying enzymes and are involved in the assembly of transcription factors at specific genomic loci. Naturally occurring mutations in this gene are associated with cancer in several tissue types (breast, bladder, pancreas, ovary, prostate, and blood). This gene plays an important role in neurodevelopment, cardiac function, adipogenesis, and osteoclastogenesis.","This gene encodes a member of the F-box protein family which is characterized by an approximately 40 amino acid motif, the F-box. The F-box proteins constitute one of the four subunits of ubiquitin protein ligase complex called SCFs (SKP1-cullin-F-box), which function in phosphorylation-dependent ubiquitination. The F-box proteins are divided into 3 classes: Fbws containing WD-40 domains, Fbls containing leucine-rich repeats, and Fbxs containing either different protein-protein interaction modules or no recognizable motifs. The protein encoded by this gene was previously referred to as FBX30, and belongs to the Fbws class; in addition to an F-box, this protein contains 7 tandem WD40 repeats. This protein binds directly to cyclin E and probably targets cyclin E for ubiquitin-mediated degradation. Mutations in this gene are detected in ovarian and breast cancer cell lines, implicating the gene's potential role in the pathogenesis of human cancers. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a zinc finger protein containing a KRAB (Kruppel-associated box) domain found in transcriptional repressors. This gene may be methylated and silenced in cancer cells. This gene is located within a differentially methylated region (DMR) and shows allele-specific expression in placenta. Alternative splicing and the use of alternative promoters results in multiple transcript variants encoding the same protein.","This gene encodes an ethanolamine kinase, which functions in the first committed step of the phosphatidylethanolamine synthesis pathway. This cytosolic enzyme is specific for ethanolamine and exhibits negligible kinase activity on choline. Alternative splicing results in multiple transcript variants encoding distinct isoforms.","This gene encodes a transmembrane protein with four predicted transmembrane domains. The protein is associated with a subpopulation of vesicular organelles corresponding to early endosomal structures, with the Golgi, and with lysosomes, and may participate in protein trafficking between these structures. Mutations in this gene and several other genes cause pheochromocytomas. Alternatively spliced transcript variants encoding the same protein have been identified.","This gene encodes a protein containing a polynucleotide kinase domain (PNK) near the N-terminal region, and a Small MutS Related (Smr) domain near the C-terminal region. The encoded protein can bind to both B-cell leukemia/lymphoma 3 (BCL-3) and neural precursor cell expressed, developmentally downregulated 4, (Nedd4) proteins. This protein binds and hydrolyzes ATP, may function as a 5'-polynucleotide kinase, and has the capacity to be a ubiquitylation substrate. This protein may play a role in transcription-coupled DNA repair or genetic recombination. Alternative splicing results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene is a component of the multiprotein assembly that is required for creation of kinetochore-microtubule attachments and chromosome segregation. The encoded protein functions as a scaffold for proteins that influence the spindle assembly checkpoint during the eukaryotic cell cycle and it interacts with at least five different kinetochore proteins and two checkpoint kinases. In adults, this gene is predominantly expressed in normal testes, various cancer cell lines and primary tumors from other tissues and is ubiquitously expressed in fetal tissues. This gene was originally identified as a fusion partner with the mixed-lineage leukemia (MLL) gene in t(11;15)(q23;q14). Mutations in this gene cause autosomal recessive primary microcephaly-4 (MCPH4). Alternative splicing results in multiple transcript variants encoding different isoforms. Additional splice variants have been described but their biological validity has not been confirmed.","This locus encodes an AT-rich DNA interacting domain-containing protein. The encoded protein is a component of the SWI/SNF chromatin remodeling complex and may play a role in cell-cycle activation. The protein encoded by this locus is similar to AT-rich interactive domain-containing protein 1A. These two proteins function as alternative, mutually exclusive ARID-subunits of the SWI/SNF complex. The associated complexes play opposing roles. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene interacts with the transcription factor myocardin, a key regulator of smooth muscle cell differentiation. The encoded protein is predominantly nuclear and may help transduce signals from the cytoskeleton to the nucleus. This gene is involved in a specific translocation event that creates a fusion of this gene and the RNA-binding motif protein-15 gene. This translocation has been associated with acute megakaryocytic leukemia. Alternative splicing results in multiple transcript variants.","HEI10 is a member of the E3 ubiquitin ligase family and functions in progression of the cell cycle through G(2)/M.","This gene is a member of the myeloid/lymphoid or mixed-lineage leukemia (MLL) family and encodes a nuclear protein with an AT hook DNA-binding domain, a DHHC-type zinc finger, six PHD-type zinc fingers, a SET domain, a post-SET domain and a RING-type zinc finger. This protein is a member of the ASC-2/NCOA6 complex (ASCOM), which possesses histone methylation activity and is involved in transcriptional coactivation.","The protein encoded by this gene is a transcriptional corepressor that is found tethered to promoter regions by DNA-binding proteins. The encoded protein can interact with several different class II histone deacetylases to repress transcription. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a C2H2-type zinc finger protein and is closely related to BCL11A, a gene whose translocation may be associated with B-cell malignancies. Although the specific function of this gene has not been determined, the encoded protein is known to be a transcriptional repressor, and is regulated by the NURD nucleosome remodeling and histone deacetylase complex. Four alternatively spliced transcript variants encoding distinct isoforms have been found for this gene.","The protein encoded by this gene is a cytoplasmic serine-threonine kinase that belongs to the protein kinase superfamily. The protein plays an important role in the regulation of electrolyte homeostasis, cell signaling survival, and proliferation. Alternative splicing results in multiple transcript variants.","This gene encodes a tumor suppressor that is involved in transcriptional and post-transcriptional control pathways. The protein is a component of the the PAF protein complex, which associates with the RNA polymerase II subunit POLR2A and with a histone methyltransferase complex. This protein appears to facilitate the association of 3' mRNA processing factors with actively-transcribed chromatin. Mutations in this gene have been linked to hyperparathyroidism-jaw tumor syndrome, familial isolated hyperparathyroidism, and parathyroid carcinoma.","The protein encoded by this gene is a member of the protocadherin family. This gene may play a role in regulating planar cell polarity (PCP). Studies in mice suggest that loss of PCP signaling may cause cystic kidney disease, and mutations in this gene have been associated with Van Maldergem Syndrome 2. Alternatively spliced transcript variants have been noted for this gene.","This gene is a member of the WD40 repeat-containing gene family and shares sequence similarity with transducin (beta)-like 1X-linked (TBL1X). The protein encoded by this gene is thought to be a component of both nuclear receptor corepressor (N-CoR) and histone deacetylase 3 (HDAC 3) complexes, and is required for transcriptional activation by a variety of transcription factors. Mutations in these gene have been associated with some autism spectrum disorders, and one finding suggests that haploinsufficiency of this gene may be a cause of intellectual disability with dysmorphism. Mutations in this gene as well as recurrent translocations involving this gene have also been observed in some tumors.","This gene encodes a protein that may function in tumor suppression. This protein binds to and colocalizes with the breast cancer 2 early onset protein (BRCA2) in nuclear foci and likely permits the stable intranuclear localization and accumulation of BRCA2.","This gene encodes a member of the F-box protein family which is characterized by an approximately 40 amino acid motif, the F-box. The F-box proteins constitute one of the four subunits of ubiquitin protein ligase complex called SCFs (SKP1-cullin-F-box), which function in phosphorylation-dependent ubiquitination. The F-box proteins are divided into 3 classes: Fbws containing WD-40 domains, Fbls containing leucine-rich repeats, and Fbxs containing either different protein-protein interaction modules or no recognizable motifs. The protein encoded by this gene belongs to the Fbxs class. It can function as an arginine methyltransferase that symmetrically dimethylates arginine residues, and it acts as an adaptor protein to mediate the neddylation of p53, which leads to the suppression of p53 function. This gene is known to be down-regulated in melanocytes from patients with vitiligo, a skin disorder that results in depigmentation. Polymorphisms in this gene are associated with chronic otitis media with effusion and recurrent otitis media (COME/ROM), a hearing loss disorder, and the knockout of the homologous mouse gene results in the deaf mouse mutant Jeff (Jf), a single gene model of otitis media. Alternatively spliced transcript variants encoding distinct isoforms have been identified for this gene.","DNA methylation is an epigenetic mechanism that is important for controlling gene expression. The protein encoded by this gene is a demethylase that belongs to the TET (ten-eleven translocation) family. Members of the TET protein family play a role in the DNA methylation process and gene activation.","The protein encoded by this gene is a member of the RecQ DEAH helicase family and interacts with the BRCT repeats of breast cancer, type 1 (BRCA1). The bound complex is important in the normal double-strand break repair function of breast cancer, type 1 (BRCA1). This gene may be a target of germline cancer-inducing mutations.","The protein encoded by this gene is a protease that functions as a deubiquitinating enzyme. The encoded protein is thought to help regulate the spindle assembly checkpoint by preventing early anaphase onset. This protein specifically deubiquitinates CDC20, which stabilizes the anaphase promoting complex/cyclosome.","Enables frizzled binding activity and ubiquitin-protein transferase activity. Involved in cellular protein metabolic process and negative regulation of Wnt signaling pathway. Is integral component of plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]","Tumor necrosis factor (TNF; see MIM 191160) receptor-associated factors, such as TRAF7, are signal transducers for members of the TNF receptor superfamily (see MIM 191190). TRAFs are composed of an N-terminal cysteine/histidine-rich region containing zinc RING and/or zinc finger motifs; a coiled-coil (leucine zipper) motif; and a homologous region that defines the TRAF family, the TRAF domain, which is involved in self-association and receptor binding.","This gene is a member of the plant homeodomain (PHD)-like finger (PHF) family. It encodes a protein with two PHD-type zinc finger domains, indicating a potential role in transcriptional regulation, that localizes to the nucleolus. Mutations affecting the coding region of this gene or the splicing of the transcript have been associated with Borjeson-Forssman-Lehmann syndrome (BFLS), a disorder characterized by cognitive disability, epilepsy, hypogonadism, hypometabolism, obesity, swelling of subcutaneous tissue of the face, narrow palpebral fissures, and large ears. Alternate splicing results in multiple transcript variants, encoding different isoforms.","The protein encoded by this gene is normally found in the membrane of the endoplasmic reticulum (ER). However, upon stress to the ER, the encoded protein is cleaved and the released cytoplasmic transcription factor domain translocates to the nucleus. There it activates the transcription of target genes by binding to box-B elements.","RMI2 is a component of the BLM (RECQL3; MIM 604610) complex, which plays a role in homologous recombination-dependent DNA repair and is essential for genome stability (Xu et al., 2008).","Predicted to act upstream of or within otolith morphogenesis. Located in extracellular space. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene upregulates trancriptional activation by the Wilms tumor protein and interacts with many other proteins, including CTNNB1, APC, AXIN1, and AXIN2. Defects in this gene are a cause of osteopathia striata with cranial sclerosis (OSCS).","The protein encoded by this gene is a member of the signal-regulatory-protein (SIRP) family, and also belongs to the immunoglobulin superfamily. SIRP family members are receptor-type transmembrane glycoproteins known to be involved in the negative regulation of receptor tyrosine kinase-coupled signaling processes. This protein can be phosphorylated by tyrosine kinases. The phospho-tyrosine residues of this PTP have been shown to recruit SH2 domain containing tyrosine phosphatases (PTP), and serve as substrates of PTPs. This protein was found to participate in signal transduction mediated by various growth factor receptors. CD47 has been demonstrated to be a ligand for this receptor protein. This gene and its product share very high similarity with several other members of the SIRP family. These related genes are located in close proximity to each other on chromosome 20p13. Multiple alternatively spliced transcript variants have been determined for this gene.","This gene is similar to the Drosophila additional sex combs gene, which encodes a chromatin-binding protein required for normal determination of segment identity in the developing embryo. The protein is a member of the Polycomb group of proteins, which are necessary for the maintenance of stable repression of homeotic and other loci. The protein is thought to disrupt chromatin in localized areas, enhancing transcription of certain genes while repressing the transcription of other genes. The protein encoded by this gene functions as a ligand-dependent co-activator for retinoic acid receptor in cooperation with nuclear receptor coactivator 1. Mutations in this gene are associated with myelodysplastic syndromes and chronic myelomonocytic leukemia. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the AT-rich interactive domain (ARID)-containing family of DNA-binding proteins. Members of the ARID family have roles in embryonic patterning, cell lineage gene regulation, cell cycle control, transcriptional regulation and chromatin structure modification. This protein functions as a subunit of the polybromo- and BRG1-associated factor or PBAF (SWI/SNF-B) chromatin remodeling complex which facilitates ligand-dependent transcriptional activation by nuclear receptors. Mutations in this gene are associated with hepatocellular carcinomas. A pseudogene of this gene is found on chromosome1.","This gene is located within the Smith-Magenis syndrome region on chromosome 17. Mutations in this gene are associated with Birt-Hogg-Dube syndrome, which is characterized by fibrofolliculomas, renal tumors, lung cysts, and pneumothorax. Alternative splicing of this gene results in two transcript variants encoding different isoforms.","Enables beta-catenin binding activity. Involved in several processes, including negative regulation of transforming growth factor beta receptor signaling pathway; positive regulation of epithelial to mesenchymal transition; and positive regulation of transcription by RNA polymerase II. Located in nucleolus and nucleoplasm. Part of beta-catenin-TCF complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the R-spondin family of proteins. These proteins are secreted ligands of leucine-rich repeat containing G protein-coupled receptors that enhance Wnt signaling through the inhibition of ubiquitin E3 ligases. A chromosomal translocation including this locus that results in the formation of a gene fusion has been identified in multiple human cancers. Alternative splicing results in multiple transcript variants."]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>symbol<\/th>\n      <th>entrezgene<\/th>\n      <th>name<\/th>\n      <th>gene_biotype<\/th>\n      <th>ncg_tsg<\/th>\n      <th>support_cancermine<\/th>\n      <th>ncbi_function_summary<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"buttons":["csv","excel"],"dom":"Bfrtip","columnDefs":[{"className":"dt-right","targets":2},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"symbol","targets":1},{"name":"entrezgene","targets":2},{"name":"name","targets":3},{"name":"gene_biotype","targets":4},{"name":"ncg_tsg","targets":5},{"name":"support_cancermine","targets":6},{"name":"ncbi_function_summary","targets":7}],"order":[],"autoWidth":false,"orderClasses":false,"responsive":true}},"evals":[],"jsHooks":[]}</script><p><br><br></p>
</div>
<div class="section level3">
<h3 id="get-classified-proto-oncogenes">Get classified proto-oncogenes<a class="anchor" aria-label="anchor" href="#get-classified-proto-oncogenes"></a>
</h3>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## Get proto-oncogenes - as indicated from either</span></span>
<span><span class="co">## Cancer Gene Census (CGC) or Network of Cancer Genes (NCG),</span></span>
<span><span class="co">## - show literature support from CancerMine</span></span>
<span><span class="va">oncogene</span> <span class="op">&lt;-</span> <span class="va">gene_basic</span><span class="op">$</span><span class="va">records</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">ncg_oncogene</span> <span class="op">==</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">symbol</span>, <span class="va">entrezgene</span>, </span>
<span>                <span class="va">name</span>, <span class="va">gene_biotype</span>,</span>
<span>                <span class="va">ncg_oncogene</span>,</span>
<span>                <span class="va">cancermine_cit_links_oncogene</span>, </span>
<span>                <span class="va">ncbi_function_summary</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/rename.html" class="external-link">rename</a></span><span class="op">(</span>support_cancermine <span class="op">=</span> </span>
<span>                  <span class="va">cancermine_cit_links_oncogene</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Make as datatable</span></span>
<span><span class="va">oncogene_table</span> <span class="op">&lt;-</span> <span class="fu">DT</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span></span>
<span>  <span class="va">oncogene</span>, </span>
<span>  escape <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  extensions <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Buttons"</span>, <span class="st">"Responsive"</span><span class="op">)</span>, </span>
<span>  width <span class="op">=</span> <span class="st">"100%"</span>,</span>
<span>  options <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>    buttons <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"csv"</span>, <span class="st">"excel"</span><span class="op">)</span>, </span>
<span>    dom <span class="op">=</span> <span class="st">"Bfrtip"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<p><br></p>
<div id="htmlwidget-e5c8c404fe174e4c81bd" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-e5c8c404fe174e4c81bd">{"x":{"filter":"none","vertical":false,"extensions":["Buttons","Responsive"],"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318"],["ABL1","ABL2","ACVR1","AKT1","AKT2","ALK","BIRC3","AR","ARAF","RHOA","ARHGAP5","ARNT","ATF1","ATP1A1","CCND1","BCL2","BCL3","BCL6","BCL9","TNFRSF17","BMPR1A","FOXL2","BRAF","BTK","CACNA1D","CALR","RUNX1","RUNX1T1","CBL","CCND2","CCND3","CCNE1","CD28","CD74","CD79A","CD79B","CDH17","CDK4","CDK6","CDKN1A","CHD4","CCR4","CCR7","CREB1","CREBBP","CSF1R","CSF3R","CTNNA2","CTNNB1","CTNND2","CUX1","DAXX","DDB2","DDIT3","DDX5","DDX6","DNMT1","EGFR","EIF1AX","ELF4","ELK4","EPAS1","ERBB2","ERBB3","ERBB4","ERG","ESR1","ETV1","ETV4","ETV5","MECOM","EWSR1","EZH2","FCGR2B","FES","FGFR1","FGFR3","FGFR2","FGFR4","FOXO1","FOXO3","FLI1","FLT3","FLT4","MTOR","GATA1","GATA2","GATA3","GPC3","GLI1","GNA11","GNAQ","GNAS","GRM3","H3-3A","H3-3B","HIF1A","HIP1","HLF","HMGA1","FOXA1","HNRNPA2B1","TLX1","HOXA9","HOXA11","HOXA13","HOXC11","HOXC13","HOXD11","HOXD13","HRAS","TNC","IDH1","IDH2","IKBKB","IL6ST","IL7R","IRF4","JAK1","JAK2","JAK3","JUN","KCNJ5","KDR","KIT","KRAS","AFF3","LCK","LMO1","LMO2","LPP","LYL1","MAF","MDM2","MDM4","MAP3K1","MET","MITF","KMT2A","AFDN","FOXO4","MN1","MPL","MTCP1","MYB","MYC","MYCL","MYCN","MYD88","MYOD1","NFATC2","NFE2L2","NFKB2","NOTCH1","NOTCH2","NPM1","NRAS","NTRK1","NTRK3","DDR2","NUP98","PAX3","PAX5","PBX1","PDGFB","PDGFRA","PDGFRB","PIK3CA","PIK3CB","PIM1","PIK3R2","PLAG1","PLCG1","POU2AF1","POU5F1","PPP2R1A","PRKACA","PRKAR1A","MAPK1","MAP2K1","MAP2K2","PTK6","PTPN11","RAC1","RAD21","RAF1","RAP1GDS1","RARA","KDM5A","REL","RET","TRIM27","RIT1","ROS1","MAP2K4","SET","SRSF2","SRSF3","SGK1","SH3GL1","STIL","SIX1","SKI","SMO","SOX2","SRC","SSX1","SSX2","SSX4","STAT3","STAT5B","STAT6","SYK","TAL1","TAL2","TBX3","TCF3","TCF7L2","ZEB1","TEC","TERT","TFE3","NKX2-1","TP53","TSHR","U2AF1","KDM6A","WAS","NSD2","WT1","XPO1","CXCR4","DEK","TFEB","KAT6A","NR4A3","BRD3","MLLT10","KMT2D","HMGA2","TCL1A","TAF15","TRRAP","H3C2","SPOP","IRS4","PPM1D","TP63","TRIM24","FUBP1","USP6","USP8","MAP3K13","KLF4","RECQL4","QKI","APOBEC3B","SETDB1","MAFB","AKT3","OLIG2","FSTL3","NCOA2","POLQ","SIX2","MALT1","KAT7","PSIP1","NT5C2","CIC","CRTC1","SF3B1","HEY1","BRD4","SUZ12","CBLC","ZNF521","WWTR1","SETBP1","PABPC1","SND1","FOXP1","AFF4","A1CF","TLX3","CHST11","LEF1","UBR5","BCL11A","DGCR8","FEV","NSD3","ACKR3","CYSLTR2","SALL4","BIRC6","MRTFA","BCORL1","PRDM16","CRLF2","CREB3L2","BCL11B","TBL1XR1","PREX2","TET1","PDCD1LG2","FCRL4","BCL2L12","CARD11","MAML2","RSPO3","KNSTRN","MSI2","NUTM1","BCL9L","FOXR1","P2RY8","MACC1"],[25,27,90,207,208,238,330,367,369,387,394,405,466,476,595,596,602,604,607,608,657,668,673,695,776,811,861,862,867,894,896,898,940,972,973,974,1015,1019,1021,1026,1108,1233,1236,1385,1387,1436,1441,1496,1499,1501,1523,1616,1643,1649,1655,1656,1786,1956,1964,2000,2005,2034,2064,2065,2066,2078,2099,2115,2118,2119,2122,2130,2146,2213,2242,2260,2261,2263,2264,2308,2309,2313,2322,2324,2475,2623,2624,2625,2719,2735,2767,2776,2778,2913,3020,3021,3091,3092,3131,3159,3169,3181,3195,3205,3207,3209,3227,3229,3237,3239,3265,3371,3417,3418,3551,3572,3575,3662,3716,3717,3718,3725,3762,3791,3815,3845,3899,3932,4004,4005,4026,4066,4094,4193,4194,4214,4233,4286,4297,4301,4303,4330,4352,4515,4602,4609,4610,4613,4615,4654,4773,4780,4791,4851,4853,4869,4893,4914,4916,4921,4928,5077,5079,5087,5155,5156,5159,5290,5291,5292,5296,5324,5335,5450,5460,5518,5566,5573,5594,5604,5605,5753,5781,5879,5885,5894,5910,5914,5927,5966,5979,5987,6016,6098,6416,6418,6427,6428,6446,6455,6491,6495,6497,6608,6657,6714,6756,6757,6759,6774,6777,6778,6850,6886,6887,6926,6929,6934,6935,7006,7015,7030,7080,7157,7253,7307,7403,7454,7468,7490,7514,7852,7913,7942,7994,8013,8019,8028,8085,8091,8115,8148,8295,8358,8405,8471,8493,8626,8805,8880,9098,9101,9175,9314,9401,9444,9582,9869,9935,10000,10215,10272,10499,10721,10736,10892,11143,11168,22978,23152,23373,23451,23462,23476,23512,23624,25925,25937,26040,26986,27044,27086,27125,29974,30012,50515,51176,51366,53335,54487,54738,54904,57007,57105,57167,57448,57591,63035,63976,64109,64764,64919,79718,80243,80312,80380,83417,83596,84433,84441,84870,90417,124540,256646,283149,283150,286530,346389],["ABL proto-oncogene 1, non-receptor tyrosine kinase","ABL proto-oncogene 2, non-receptor tyrosine kinase","activin A receptor type 1","AKT serine/threonine kinase 1","AKT serine/threonine kinase 2","ALK receptor tyrosine kinase","baculoviral IAP repeat containing 3","androgen receptor","A-Raf proto-oncogene, serine/threonine kinase","ras homolog family member A","Rho GTPase activating protein 5","aryl hydrocarbon receptor nuclear translocator","activating transcription factor 1","ATPase Na+/K+ transporting subunit alpha 1","cyclin D1","BCL2 apoptosis regulator","BCL3 transcription coactivator","BCL6 transcription repressor","BCL9 transcription coactivator","TNF receptor superfamily member 17","bone morphogenetic protein receptor type 1A","forkhead box L2","B-Raf proto-oncogene, serine/threonine kinase","Bruton tyrosine kinase","calcium voltage-gated channel subunit alpha1 D","calreticulin","RUNX family transcription factor 1","RUNX1 partner transcriptional co-repressor 1","Cbl proto-oncogene","cyclin D2","cyclin D3","cyclin E1","CD28 molecule","CD74 molecule","CD79a molecule","CD79b molecule","cadherin 17","cyclin dependent kinase 4","cyclin dependent kinase 6","cyclin dependent kinase inhibitor 1A","chromodomain helicase DNA binding protein 4","C-C motif chemokine receptor 4","C-C motif chemokine receptor 7","cAMP responsive element binding protein 1","CREB binding lysine acetyltransferase","colony stimulating factor 1 receptor","colony stimulating factor 3 receptor","catenin alpha 2","catenin beta 1","catenin delta 2","cut like homeobox 1","death domain associated protein","damage specific DNA binding protein 2","DNA damage inducible transcript 3","DEAD-box helicase 5","DEAD-box helicase 6","DNA methyltransferase 1","epidermal growth factor receptor","eukaryotic translation initiation factor 1A X-linked","E74 like ETS transcription factor 4","ETS transcription factor ELK4","endothelial PAS domain protein 1","erb-b2 receptor tyrosine kinase 2","erb-b2 receptor tyrosine kinase 3","erb-b2 receptor tyrosine kinase 4","ETS transcription factor ERG","estrogen receptor 1","ETS variant transcription factor 1","ETS variant transcription factor 4","ETS variant transcription factor 5","MDS1 and EVI1 complex locus","EWS RNA binding protein 1","enhancer of zeste 2 polycomb repressive complex 2 subunit","Fc gamma receptor IIb","FES proto-oncogene, tyrosine kinase","fibroblast growth factor receptor 1","fibroblast growth factor receptor 3","fibroblast growth factor receptor 2","fibroblast growth factor receptor 4","forkhead box O1","forkhead box O3","Fli-1 proto-oncogene, ETS transcription factor","fms related receptor tyrosine kinase 3","fms related receptor tyrosine kinase 4","mechanistic target of rapamycin kinase","GATA binding protein 1","GATA binding protein 2","GATA binding protein 3","glypican 3","GLI family zinc finger 1","G protein subunit alpha 11","G protein subunit alpha q","GNAS complex locus","glutamate metabotropic receptor 3","H3.3 histone A","H3.3 histone B","hypoxia inducible factor 1 subunit alpha","huntingtin interacting protein 1","HLF transcription factor, PAR bZIP family member","high mobility group AT-hook 1","forkhead box A1","heterogeneous nuclear ribonucleoprotein A2/B1","T cell leukemia homeobox 1","homeobox A9","homeobox A11","homeobox A13","homeobox C11","homeobox C13","homeobox D11","homeobox D13","HRas proto-oncogene, GTPase","tenascin C","isocitrate dehydrogenase (NADP(+)) 1","isocitrate dehydrogenase (NADP(+)) 2","inhibitor of nuclear factor kappa B kinase subunit beta","interleukin 6 cytokine family signal transducer","interleukin 7 receptor","interferon regulatory factor 4","Janus kinase 1","Janus kinase 2","Janus kinase 3","Jun proto-oncogene, AP-1 transcription factor subunit","potassium inwardly rectifying channel subfamily J member 5","kinase insert domain receptor","KIT proto-oncogene, receptor tyrosine kinase","KRAS proto-oncogene, GTPase","ALF transcription elongation factor 3","LCK proto-oncogene, Src family tyrosine kinase","LIM domain only 1","LIM domain only 2","LIM domain containing preferred translocation partner in lipoma","LYL1 basic helix-loop-helix family member","MAF bZIP transcription factor","MDM2 proto-oncogene","MDM4 regulator of p53","mitogen-activated protein kinase kinase kinase 1","MET proto-oncogene, receptor tyrosine kinase","melanocyte inducing transcription factor","lysine methyltransferase 2A","afadin, adherens junction formation factor","forkhead box O4","MN1 proto-oncogene, transcriptional regulator","MPL proto-oncogene, thrombopoietin receptor","mature T cell proliferation 1","MYB proto-oncogene, transcription factor","MYC proto-oncogene, bHLH transcription factor","MYCL proto-oncogene, bHLH transcription factor","MYCN proto-oncogene, bHLH transcription factor","MYD88 innate immune signal transduction adaptor","myogenic differentiation 1","nuclear factor of activated T cells 2","NFE2 like bZIP transcription factor 2","nuclear factor kappa B subunit 2","notch receptor 1","notch receptor 2","nucleophosmin 1","NRAS proto-oncogene, GTPase","neurotrophic receptor tyrosine kinase 1","neurotrophic receptor tyrosine kinase 3","discoidin domain receptor tyrosine kinase 2","nucleoporin 98 and 96 precursor","paired box 3","paired box 5","PBX homeobox 1","platelet derived growth factor subunit B","platelet derived growth factor receptor alpha","platelet derived growth factor receptor beta","phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha","phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit beta","Pim-1 proto-oncogene, serine/threonine kinase","phosphoinositide-3-kinase regulatory subunit 2","PLAG1 zinc finger","phospholipase C gamma 1","POU class 2 homeobox associating factor 1","POU class 5 homeobox 1","protein phosphatase 2 scaffold subunit Aalpha","protein kinase cAMP-activated catalytic subunit alpha","protein kinase cAMP-dependent type I regulatory subunit alpha","mitogen-activated protein kinase 1","mitogen-activated protein kinase kinase 1","mitogen-activated protein kinase kinase 2","protein tyrosine kinase 6","protein tyrosine phosphatase non-receptor type 11","Rac family small GTPase 1","RAD21 cohesin complex component","Raf-1 proto-oncogene, serine/threonine kinase","Rap1 GTPase-GDP dissociation stimulator 1","retinoic acid receptor alpha","lysine demethylase 5A","REL proto-oncogene, NF-kB subunit","ret proto-oncogene","tripartite motif containing 27","Ras like without CAAX 1","ROS proto-oncogene 1, receptor tyrosine kinase","mitogen-activated protein kinase kinase 4","SET nuclear proto-oncogene","serine and arginine rich splicing factor 2","serine and arginine rich splicing factor 3","serum/glucocorticoid regulated kinase 1","SH3 domain containing GRB2 like 1, endophilin A2","STIL centriolar assembly protein","SIX homeobox 1","SKI proto-oncogene","smoothened, frizzled class receptor","SRY-box transcription factor 2","SRC proto-oncogene, non-receptor tyrosine kinase","SSX family member 1","SSX family member 2","SSX family member 4","signal transducer and activator of transcription 3","signal transducer and activator of transcription 5B","signal transducer and activator of transcription 6","spleen associated tyrosine kinase","TAL bHLH transcription factor 1, erythroid differentiation factor","TAL bHLH transcription factor 2","T-box transcription factor 3","transcription factor 3","transcription factor 7 like 2","zinc finger E-box binding homeobox 1","tec protein tyrosine kinase","telomerase reverse transcriptase","transcription factor binding to IGHM enhancer 3","NK2 homeobox 1","tumor protein p53","thyroid stimulating hormone receptor","U2 small nuclear RNA auxiliary factor 1","lysine demethylase 6A","WASP actin nucleation promoting factor","nuclear receptor binding SET domain protein 2","WT1 transcription factor","exportin 1","C-X-C motif chemokine receptor 4","DEK proto-oncogene","transcription factor EB","lysine acetyltransferase 6A","nuclear receptor subfamily 4 group A member 3","bromodomain containing 3","MLLT10 histone lysine methyltransferase DOT1L cofactor","lysine methyltransferase 2D","high mobility group AT-hook 2","TCL1 family AKT coactivator A","TATA-box binding protein associated factor 15","transformation/transcription domain associated protein","H3 clustered histone 2","speckle type BTB/POZ protein","insulin receptor substrate 4","protein phosphatase, Mg2+/Mn2+ dependent 1D","tumor protein p63","tripartite motif containing 24","far upstream element binding protein 1","ubiquitin specific peptidase 6","ubiquitin specific peptidase 8","mitogen-activated protein kinase kinase kinase 13","KLF transcription factor 4","RecQ like helicase 4","QKI, KH domain containing RNA binding","apolipoprotein B mRNA editing enzyme catalytic subunit 3B","SET domain bifurcated histone lysine methyltransferase 1","MAF bZIP transcription factor B","AKT serine/threonine kinase 3","oligodendrocyte transcription factor 2","follistatin like 3","nuclear receptor coactivator 2","DNA polymerase theta","SIX homeobox 2","MALT1 paracaspase","lysine acetyltransferase 7","PC4 and SRSF1 interacting protein 1","5'-nucleotidase, cytosolic II","capicua transcriptional repressor","CREB regulated transcription coactivator 1","splicing factor 3b subunit 1","hes related family bHLH transcription factor with YRPW motif 1","bromodomain containing 4","SUZ12 polycomb repressive complex 2 subunit","Cbl proto-oncogene C","zinc finger protein 521","WW domain containing transcription regulator 1","SET binding protein 1","poly(A) binding protein cytoplasmic 1","staphylococcal nuclease and tudor domain containing 1","forkhead box P1","ALF transcription elongation factor 4","APOBEC1 complementation factor","T cell leukemia homeobox 3","carbohydrate sulfotransferase 11","lymphoid enhancer binding factor 1","ubiquitin protein ligase E3 component n-recognin 5","BCL11 transcription factor A","DGCR8 microprocessor complex subunit","FEV transcription factor, ETS family member","nuclear receptor binding SET domain protein 3","atypical chemokine receptor 3","cysteinyl leukotriene receptor 2","spalt like transcription factor 4","baculoviral IAP repeat containing 6","myocardin related transcription factor A","BCL6 corepressor like 1","PR/SET domain 16","cytokine receptor like factor 2","cAMP responsive element binding protein 3 like 2","BCL11 transcription factor B","TBL1X/Y related 1","phosphatidylinositol-3,4,5-trisphosphate dependent Rac exchange factor 2","tet methylcytosine dioxygenase 1","programmed cell death 1 ligand 2","Fc receptor like 4","BCL2 like 12","caspase recruitment domain family member 11","mastermind like transcriptional coactivator 2","R-spondin 3","kinetochore localized astrin (SPAG5) binding protein","musashi RNA binding protein 2","NUT midline carcinoma family member 1","BCL9 like","forkhead box R1","P2Y receptor family member 8","MET transcriptional regulator MACC1"],["protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding"],[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true],["<a href='https://www.ncbi.nlm.nih.gov/pubmed/40155722' target='_blank'>Kovaka et al., 2025, Nat Methods<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33828274' target='_blank'>Hendel et al., Nat Methods, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32699415' target='_blank'>Chan et al., Nature, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31474570' target='_blank'>Reina-Campos et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28344039' target='_blank'>Wang et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28319094' target='_blank'>Kesarwani et al., Nat Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28288113' target='_blank'>Wang et al., 2017, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25686603' target='_blank'>Pemovska et al., Nature, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23842646' target='_blank'>Greuber et al., 2013, Nat Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21884932' target='_blank'>Sykes et al., 2011, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21075303' target='_blank'>Restifo et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21035406' target='_blank'>Rakhra et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19269363' target='_blank'>Luo et al., Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15573099' target='_blank'>Daub et al., Nat Rev Drug Discov, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1712671' target='_blank'>Pendergast et al., Cell, 1991<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33676569' target='_blank'>Vural et al., Clin Epigenetics, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33233470' target='_blank'>Creeden et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26766902' target='_blank'>Qin et al., Drug Des Devel Ther, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24096484' target='_blank'>Fiore et al., Oncogene, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19455236' target='_blank'>Dalgin et al., Cancer Inform, 2007<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38514615' target='_blank'>Deng et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33955720' target='_blank'>Dong et al., Cancer Commun (Lond), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32727536' target='_blank'>Noorani et al., Genome Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32209098' target='_blank'>Yang et al., Mol Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26639397' target='_blank'>Juric et al., J Hepatol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26206664' target='_blank'>Scarzello et al., Gut, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26168014' target='_blank'>Hobbs et al., Nat Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24512821' target='_blank'>Marquardt et al., J Hepatol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23475878' target='_blank'>Gibney et al., Cancer Discov, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22328975' target='_blank'>Hanrahan et al., Cancer Discov, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21464222' target='_blank'>Li et al., J Exp Med, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20227039' target='_blank'>Hsieh et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19770358' target='_blank'>Herling et al., Blood, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19362540' target='_blank'>Old et al., Mol Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18981300' target='_blank'>Wang et al., Circulation, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39838379' target='_blank'>Chen et al., 2025, Biol Direct<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35582007' target='_blank'>Ulldemolins et al., Cancer Drug Resist, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32873299' target='_blank'>Liu et al., Respir Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30558539' target='_blank'>Hao et al., BMC Bioinformatics, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29667444' target='_blank'>Rafael et al., Drug Deliv, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27840954' target='_blank'>Wang et al., 2016, Mol Med Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27826244' target='_blank'>Ong et al., Front Pharmacol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25961939' target='_blank'>Moriarity et al., Nat Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25883022' target='_blank'>Li et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23603750' target='_blank'>Agarwal et al., Cell Signal, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21182834' target='_blank'>Kim et al., Exp Cell Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12605716' target='_blank'>Schneider et al., Mol Cancer, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9496907' target='_blank'>Ruggeri et al., Mol Carcinog, 1998<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8622988' target='_blank'>Cheng et al., Proc Natl Acad Sci U S A, 1996<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7657393' target='_blank'>Bellacosa et al., Int J Cancer, 1995<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40246819' target='_blank'>Zha et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38039964' target='_blank'>Bergaggio et al., 2023, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36797503' target='_blank'>Schneider et al., Nat Cancer, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30510251' target='_blank'>Menotti et al., Nat Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27573756' target='_blank'>Dagogo-Jack et al., 2016, Ann Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25759024' target='_blank'>Wilson et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25517749' target='_blank'>Bresler et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25337876' target='_blank'>Maddalo et al., Nature, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22585002' target='_blank'>Carpenter et al., Nat Rev Clin Oncol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22473102' target='_blank'>Camidge et al., Nat Rev Clin Oncol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21075303' target='_blank'>Restifo et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19412175' target='_blank'>Capasso et al., Nat Genet, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18724359' target='_blank'>Mossé et al., Nature, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18097461' target='_blank'>Chiarle et al., Nat Rev Cancer, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15895073' target='_blank'>Chiarle et al., Nat Med, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39657662' target='_blank'>Senatorov et al., 2024, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39147758' target='_blank'>Malone et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35985688' target='_blank'>Lee et al., 2022, Cancer Genomics Proteomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31406474' target='_blank'>Wang et al., Cancer Manag Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29907126' target='_blank'>Balaji et al., J Hematol Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29587428' target='_blank'>Grondona et al., Biomedicines, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28759024' target='_blank'>Wang et al., Oncogenesis, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20676140' target='_blank'>Cheng et al., Oncogene, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40553569' target='_blank'>Shigeta et al., 2025, JCI Insight<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38635890' target='_blank'>Cunningham et al., 2024, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38348349' target='_blank'>Hussain et al., 2024, Front Pharmacol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37874216' target='_blank'>Oyer et al., 2023, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36194476' target='_blank'>Sena et al., J Clin Invest, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35965537' target='_blank'>Liu et al., Front Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35495285' target='_blank'>Barrett et al., Eur Urol Open Sci, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34290239' target='_blank'>Chen et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33576154' target='_blank'>Severson et al., Mol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33547161' target='_blank'>Centenera et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32210733' target='_blank'>Deng et al., Int J Biol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27373336' target='_blank'>Li et al., Mol Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25691442' target='_blank'>Koryakina et al., Endocr Relat Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25238881' target='_blank'>Di et al., Breast Cancer Res Treat, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24574517' target='_blank'>Yu et al., Mol Cancer Res, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38217548' target='_blank'>Li et al., 2024, Aging (Albany NY)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34079392' target='_blank'>Pópulo et al., 2021, J Exp Pharmacol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33092114' target='_blank'>Skeparnias et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24569458' target='_blank'>Imielinski et al., J Clin Invest, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39376555' target='_blank'>Ren et al., 2024, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34493786' target='_blank'>Mehra et al., Commun Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34142093' target='_blank'>Lo et al., Nat Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33924428' target='_blank'>Cinier et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33037251' target='_blank'>Algaber et al., Sci Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31485674' target='_blank'>Nishizawa et al., Oncol Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29544183' target='_blank'>Li et al., Neoplasia, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29419761' target='_blank'>Diviani et al., J Cardiovasc Dev Dis, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28146427' target='_blank'>Al-Haidari et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27223089' target='_blank'>Huang et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20545614' target='_blank'>McHenry et al., Curr Drug Targets, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34490480' target='_blank'>Wei et al., Mol Med Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34285193' target='_blank'>Yang et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32547322' target='_blank'>Chen et al., Int J Med Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32483433' target='_blank'>Tian et al., Theranostics, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38806469' target='_blank'>Alafate et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31001330' target='_blank'>Wagner et al., Front Genet, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26599865' target='_blank'>Karweina et al., PLoS One, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38074516' target='_blank'>Galera et al., 2023, Case Rep Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37398210' target='_blank'>Menon et al., 2023, Res Sq<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37205599' target='_blank'>Menon et al., 2023, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31488818' target='_blank'>Komura et al., 2019, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25809881' target='_blank'>Takamura et al., J Dermatol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23410975' target='_blank'>Straessler et al., Cancer Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20362353' target='_blank'>Heyerdahl et al., Eur J Cell Biol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16766266' target='_blank'>Davis et al., Cancer Cell, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12966428' target='_blank'>Li et al., Br J Cancer, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9383826' target='_blank'>d'Amore et al., Semin Diagn Pathol, 1997<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37198482' target='_blank'>Lee et al., 2023, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34591642' target='_blank'>Swaney et al., Science, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30496486' target='_blank'>Lin et al., Nucleic Acids Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29907126' target='_blank'>Balaji et al., J Hematol Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25582697' target='_blank'>Li et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24792170' target='_blank'>Liu et al., Nucleic Acids Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22817889' target='_blank'>Hodis et al., Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22224767' target='_blank'>Shaffer et al., Annu Rev Immunol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21642962' target='_blank'>Puente et al., Nature, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21397858' target='_blank'>Sawey et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19917254' target='_blank'>Shan et al., Mol Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19143056' target='_blank'>Knudsen et al., Nat Rev Cancer, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16291983' target='_blank'>Curtin et al., N Engl J Med, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15961768' target='_blank'>Arnold et al., J Clin Oncol, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8208295' target='_blank'>Wang et al., Nature, 1994<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34002060' target='_blank'>Zhao et al., 2021, Nat Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32020081' target='_blank'>Nair et al., Nat Rev Mol Cell Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29747654' target='_blank'>Perini et al., J Hematol Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27846392' target='_blank'>Ceribelli et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10715307' target='_blank'>Miller et al., 2000, J Clin Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9209654' target='_blank'>Matolcsy et al., 1997, Ann Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8393963' target='_blank'>Pezzella et al., N Engl J Med, 1993<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7753200' target='_blank'>Shokat et al., Nature, 1995<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7715730' target='_blank'>Chittenden et al., Nature, 1995<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7612233' target='_blank'>Cory et al., Annu Rev Immunol, 1995<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/3495884' target='_blank'>Reed et al., Science, 1987<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/3287162' target='_blank'>Ngan et al., 1988, N Engl J Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2649247' target='_blank'>McDonnell et al., Cell, 1989<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2250704' target='_blank'>Strasser et al., 1990, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1406976' target='_blank'>Fanidi et al., Nature, 1992<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38767825' target='_blank'>Zhao et al., 2024, Genes Genomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38255248' target='_blank'>Turnham et al., 2024, Biomedicines<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37151134' target='_blank'>Gaire et al., 2023, FEBS Open Bio<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32710395' target='_blank'>Kazberuk et al., Pharmacol Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31727944' target='_blank'>Marcos-Villar et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30598100' target='_blank'>Zhang et al., BMC Genomics, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29587428' target='_blank'>Grondona et al., Biomedicines, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28684863' target='_blank'>Poveda et al., Exp Mol Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27822067' target='_blank'>Zhao et al., Onco Targets Ther, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27748795' target='_blank'>Wu et al., Int J Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27556362' target='_blank'>Tzaridis et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26219963' target='_blank'>Lee et al., Stem Cell Res Ther, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16732314' target='_blank'>Brocke-Heidrich et al., Oncogene, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10497212' target='_blank'>Na et al., J Biol Chem, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8196632' target='_blank'>Zhang et al., Mol Cell Biol, 1994<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/28253933' target='_blank'>Ding et al., J Hematol Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26898111' target='_blank'>Casellas et al., 2016, Nat Rev Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25848755' target='_blank'>Kaplan et al., Nat Rev Immunol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25759025' target='_blank'>Geng et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23974956' target='_blank'>Ying et al., Nat Immunol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22113614' target='_blank'>Duan et al., Nature, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21593872' target='_blank'>Duy et al., Nature, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21390126' target='_blank'>Pasqualucci et al., Nature, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20944599' target='_blank'>Kelly et al., Nat Biotechnol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20385364' target='_blank'>Cerchietti et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20084069' target='_blank'>Crotty et al., Nat Immunol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20043832' target='_blank'>Ke et al., Mol Cancer, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19763153' target='_blank'>Croce et al., Nat Rev Genet, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18066064' target='_blank'>Pasqualucci et al., Nat Genet, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16704730' target='_blank'>Liu et al., Mol Cancer, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38811552' target='_blank'>He et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37766215' target='_blank'>Akyüz et al., 2023, Viruses<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32759724' target='_blank'>Anthony et al., Genes (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27824051' target='_blank'>Gu et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25228654' target='_blank'>Zhao et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22353711' target='_blank'>de et al., Nat Commun, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20844748' target='_blank'>Gorringe et al., PLoS One, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19738061' target='_blank'>Mani et al., Cancer Res, 2009<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34002060' target='_blank'>Zhao et al., 2021, Nat Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32251397' target='_blank'>Fazio et al., Nat Rev Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31124566' target='_blank'>Kristensen et al., Ann Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30976106' target='_blank'>Zhu et al., Nat Rev Mol Cell Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30967646' target='_blank'>Bai et al., Nat Rev Clin Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29068003' target='_blank'>Rotow et al., Nat Rev Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28961848' target='_blank'>Chapman et al., Ann Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26670017' target='_blank'>Dominguez et al., Nat Rev Mol Cell Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26343581' target='_blank'>Zelenay et al., Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25843629' target='_blank'>Karreth et al., Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24957944' target='_blank'>Holderfield et al., Nat Rev Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23712190' target='_blank'>Gibney et al., Nat Rev Clin Oncol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23332765' target='_blank'>Burd et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21215707' target='_blank'>Arozarena et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20141835' target='_blank'>Heidorn et al., Cell, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40425940' target='_blank'>Xiao et al., 2025, Invest New Drugs<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39969198' target='_blank'>Kost et al., 2025, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38696711' target='_blank'>Minotti et al., 2024, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35786935' target='_blank'>Thakur et al., J Med Chem, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34315851' target='_blank'>Yeh et al., Oncogenesis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34277452' target='_blank'>Lv et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33867855' target='_blank'>Yang et al., Int J Biol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33760219' target='_blank'>Kim et al., 2021, Oncol Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32455989' target='_blank'>George et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30733667' target='_blank'>Sala et al., Front Mol Neurosci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30011241' target='_blank'>Yu et al., SLAS Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25038787' target='_blank'>Lanning et al., Nat Chem Biol, 2014<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38885318' target='_blank'>Schmidt et al., 2024, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38245510' target='_blank'>Chen et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36813992' target='_blank'>Foßelteder et al., Leukemia, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36632735' target='_blank'>Cattaneo et al., 2023, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33239297' target='_blank'>Prins et al., Sci Adv, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32382708' target='_blank'>Prins et al., Hemasphere, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32084350' target='_blank'>Kaur et al., Mol Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31915854' target='_blank'>Holmström et al., Cancer Immunol Immunother, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31266769' target='_blank'>Cimen et al., Cancer Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29296692' target='_blank'>Tefferi et al., Blood Adv, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29288169' target='_blank'>Elf et al., Blood, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26951227' target='_blank'>Elf et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26473532' target='_blank'>Guglielmelli et al., Blood Cancer J, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26182311' target='_blank'>Tefferi et al., JAMA Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25037629' target='_blank'>Tefferi et al., Blood, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39245681' target='_blank'>Bao et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35302840' target='_blank'>Hansen et al., Sci Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33483506' target='_blank'>Grinev et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32887641' target='_blank'>Yu et al., 2020, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31126316' target='_blank'>Liu et al., J Hematol Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28790107' target='_blank'>Choi et al., Blood, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28530640' target='_blank'>Morita et al., 2017, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25594381' target='_blank'>Dancy et al., Chem Rev, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24596419' target='_blank'>Pattabiraman et al., Blood, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24232254' target='_blank'>Copley et al., Exp Mol Med, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20855863' target='_blank'>Daghistani et al., Blood, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20460523' target='_blank'>Dunne et al., Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17901249' target='_blank'>Kirsammer et al., Blood, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17332359' target='_blank'>Wichmann et al., 2007, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15604243' target='_blank'>Planagumà et al., Cancer Res, 2004<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38926399' target='_blank'>Yoo et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37550642' target='_blank'>Naito et al., 2023, BMC Infect Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36980682' target='_blank'>Kanellou et al., 2023, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34298959' target='_blank'>Khamidullina et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33692956' target='_blank'>Récher et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33602907' target='_blank'>Keinan et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33483506' target='_blank'>Grinev et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32314924' target='_blank'>Wu et al., J Med Chem, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31474868' target='_blank'>Mitkevich et al., Front Pharmacol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25594381' target='_blank'>Dancy et al., Chem Rev, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20540792' target='_blank'>Taylor et al., Cell Commun Signal, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20460523' target='_blank'>Dunne et al., Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19043539' target='_blank'>Gardini et al., PLoS Genet, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17332359' target='_blank'>Wichmann et al., 2007, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11593431' target='_blank'>Fukuyama et al., Oncogene, 2001<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40126553' target='_blank'>Zeng et al., 2025, Elife<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39992860' target='_blank'>Fu et al., 2025, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32894644' target='_blank'>Foster et al., EMBO Mol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31442303' target='_blank'>Demark-Wahnefried et al., Int J Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26676746' target='_blank'>Li et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26314988' target='_blank'>Novak et al., Blood Cancer J, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25477533' target='_blank'>Liyasova et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25348515' target='_blank'>Kadera et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22684873' target='_blank'>Bojkowska et al., Hepatology, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21863050' target='_blank'>Lipkowitz et al., Nat Rev Cancer, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20501843' target='_blank'>Kales et al., 2010, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16729045' target='_blank'>Oda et al., Mol Syst Biol, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9731483' target='_blank'>Calin et al., Cancer Res, 1998<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2030914' target='_blank'>Blake et al., Oncogene, 1991<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1565471' target='_blank'>Blake et al., Oncogene, 1992<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35266852' target='_blank'>Wen et al., Bioengineered, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34335927' target='_blank'>Ning et al., J Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33727831' target='_blank'>Yue et al., Onco Targets Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33480613' target='_blank'>Qin et al., Chin Med J (Engl), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31612065' target='_blank'>Yuan et al., Oncol Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30950241' target='_blank'>Ding et al., Cancer Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27584029' target='_blank'>Woldu et al., Curr Opin Urol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25295500' target='_blank'>Hacein-Bey-Abina et al., N Engl J Med, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19577536' target='_blank'>Kobayashi et al., Biochem Biophys Res Commun, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16762081' target='_blank'>Oram et al., BMC Cancer, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37643767' target='_blank'>Schwarzer et al., 2023, Mol Cancer Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36813801' target='_blank'>He et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34545335' target='_blank'>He et al., ArXiv, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32989398' target='_blank'>Wu et al., Oncol Lett, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30950241' target='_blank'>Ding et al., Cancer Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29951542' target='_blank'>Chen et al., Biomed Res Int, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28867784' target='_blank'>Bahjat et al., Int J Mol Sci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28151717' target='_blank'>Pikman et al., 2017, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27915416' target='_blank'>Patel et al., Cell Mol Life Sci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25300797' target='_blank'>Yang et al., J Hematol Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15728238' target='_blank'>Mandal et al., J Exp Med, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40681110' target='_blank'>Cavasin et al., 2025, Tumour Virus Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40664642' target='_blank'>Singh et al., 2025, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40112818' target='_blank'>Wala et al., 2025, Cell Genom<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40083014' target='_blank'>Su et al., 2025, Clin Epigenetics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39945650' target='_blank'>House et al., 2025, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38374872' target='_blank'>Gu et al., 2024, MedComm (2020)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35259479' target='_blank'>Gupta et al., 2022, Pharmacol Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34818047' target='_blank'>Reddy et al., Sci Adv, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34622238' target='_blank'>Goehring et al., Cell Rep Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33361393' target='_blank'>Chaves-Moreira et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31324031' target='_blank'>Leskela et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30988301' target='_blank'>Ferreira et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30357595' target='_blank'>Sun et al., Protein Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24366882' target='_blank'>Karst et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23378270' target='_blank'>Sheu et al., J Pathol, 2013<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37629174' target='_blank'>Hong et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28704499' target='_blank'>Goto et al., 2017, PLoS One<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18941249' target='_blank'>Liu et al., J Immunol, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40231473' target='_blank'>Kang et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39465766' target='_blank'>Li et al., 2024, Medicine (Baltimore)<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40231473' target='_blank'>Kang et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33287695' target='_blank'>Goswami et al., BMC Genomics, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25501023' target='_blank'>Kersten et al., Blood Cancer J, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/31516924' target='_blank'>Yuan et al., Data Brief, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23298905' target='_blank'>Wang et al., Cancer Biol Ther, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20568120' target='_blank'>Zhu et al., J Cell Biochem, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20393816' target='_blank'>Zhang et al., Med Oncol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19676131' target='_blank'>Liu et al., Hepatology, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33369721' target='_blank'>Braal et al., Drugs, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33179614' target='_blank'>Sbaraglia et al., Pathologica, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32277580' target='_blank'>Cao et al., J Cell Mol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31900418' target='_blank'>Colaprico et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31199581' target='_blank'>Feng et al., J Cell Mol Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29922629' target='_blank'>Domingues et al., Immunotargets Ther, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29464001' target='_blank'>Liu et al., Theranostics, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28378523' target='_blank'>Nawaz et al., Mol Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28288139' target='_blank'>Walls et al., Oncogene, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28267263' target='_blank'>Geyer et al., Mol Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27941674' target='_blank'>Isola et al., Cancers (Basel), 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24457416' target='_blank'>Sturm et al., Nat Rev Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24113309' target='_blank'>Parham et al., Adv Anat Pathol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22094256' target='_blank'>Anders et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18790768' target='_blank'>Smalley et al., Mol Cancer Ther, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40739444' target='_blank'>Karam et al., 2025, NPJ Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39936993' target='_blank'>Harrell et al., 2025, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34714590' target='_blank'>Shen et al., 2021, Cancer Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33324155' target='_blank'>Di Cintio et al., 2020, Front Neurosci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33188661' target='_blank'>Wei et al., Aging (Albany NY), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32860670' target='_blank'>Li et al., Aging (Albany NY), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32047552' target='_blank'>Li et al., J Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29922629' target='_blank'>Domingues et al., Immunotargets Ther, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28151717' target='_blank'>Pikman et al., 2017, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26626481' target='_blank'>Shen et al., 2015, Mol Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26310374' target='_blank'>Koh et al., Transl Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25239601' target='_blank'>Nannini et al., BMC Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24423610' target='_blank'>Baba et al., 2014, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23175120' target='_blank'>Northcott et al., Nat Rev Cancer, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21810092' target='_blank'>Rodriguez-Otero et al., Br J Haematol, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40321146' target='_blank'>Sepe et al., 2025, FEBS J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32397616' target='_blank'>Pisonero-Vaquero et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30755307' target='_blank'>McClure et al., Trends Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30477117' target='_blank'>Brachtlova et al., 2018, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29669758' target='_blank'>Su et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29650632' target='_blank'>Broughton et al., Circ Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26191083' target='_blank'>Gaddis et al., Epigenetics Chromatin, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25838973' target='_blank'>Han et al., Exp Hematol Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24221458' target='_blank'>Goldenberg et al., Gut, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23361053' target='_blank'>Ghosh et al., Br J Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17616661' target='_blank'>Bae et al., 2007, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16462758' target='_blank'>De et al., Oncogene, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15709169' target='_blank'>Brzeziński et al., Clin Cancer Res, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/3308076' target='_blank'>Clair et al., Cancer Res, 1987<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2448499' target='_blank'>Fan et al., J Urol, 1988<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39097722' target='_blank'>Li et al., 2024, Cancer Cell Int<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38844966' target='_blank'>Jiang et al., 2024, Genome Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38809626' target='_blank'>Hasanlu et al., 2024, Asian Pac J Cancer Prev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32070428' target='_blank'>Shao et al., Clin Epigenetics, 2020<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/29928346' target='_blank'>Jin et al., Oncol Lett, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28451459' target='_blank'>Sonbul et al., J Pathol Clin Res, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40483692' target='_blank'>Lin et al., 2025, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40157009' target='_blank'>Hong et al., 2025, J Pharmacol Exp Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38172598' target='_blank'>Lugano et al., 2024, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37565725' target='_blank'>Li et al., 2023, J Med Virol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36595106' target='_blank'>Hu et al., Cell Regen, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33846793' target='_blank'>Wang et al., Oncol Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33125147' target='_blank'>Zhang et al., Oncol Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30858759' target='_blank'>Wang et al., Cancer Cell Int, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24055910' target='_blank'>Ha et al., Cell Signal, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23449452' target='_blank'>Gausdal et al., Cell Death Dis, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23019365' target='_blank'>Tan et al., Proc Natl Acad Sci U S A, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19117985' target='_blank'>Zhang et al., Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18676828' target='_blank'>Seo et al., Cancer Res, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16096372' target='_blank'>Kinjo et al., Cell Cycle, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15837624' target='_blank'>Shankar et al., Cancer Cell, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39740655' target='_blank'>Chen et al., 2025, J Neuropathol Exp Neurol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38889153' target='_blank'>Smolko et al., 2024, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38463444' target='_blank'>Almeida et al., 2024, Hemasphere<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36720920' target='_blank'>Zhuang et al., 2023, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31713888' target='_blank'>Tang et al., Mol Carcinog, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30262461' target='_blank'>Duque-Afonso et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29021135' target='_blank'>Lv et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28953875' target='_blank'>Lasko et al., 2017, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27782077' target='_blank'>Zhang et al., Int J Mol Sci, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24633655' target='_blank'>Carlson et al., J Cell Physiol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18537607' target='_blank'>Leone et al., Curr Med Chem, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11333923' target='_blank'>Wienzek et al., J Virol, 2001<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36206490' target='_blank'>Chen et al., Blood, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34446858' target='_blank'>Tariq et al., Br J Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30675010' target='_blank'>Nabinger et al., Leukemia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22614177' target='_blank'>Swords et al., 2012, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20436485' target='_blank'>Lamprecht et al., Nat Med, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19489726' target='_blank'>Lu et al., Annu Rev Biochem, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15289345' target='_blank'>Aharinejad et al., Cancer Res, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10554038' target='_blank'>Sapi et al., Cancer Res, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9376277' target='_blank'>Yokoyama et al., Br J Cancer, 1997<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8250012' target='_blank'>Jaeger et al., Am J Hematol, 1994<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7882368' target='_blank'>Chambers et al., Cancer Res, 1995<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/3180083' target='_blank'>Imaizumi et al., Cancer Res, 1988<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/3011859' target='_blank'>Rettenmier et al., J Clin Invest, 1986<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2433029' target='_blank'>Preisler et al., Cancer Res, 1987<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2214871' target='_blank'>Torrès et al., Leukemia, 1990<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/29449543' target='_blank'>Szuber et al., 2018, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25145552' target='_blank'>Yacoub et al., Curr Hematol Malig Rep, 2014<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38297372' target='_blank'>Lin et al., 2024, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34313788' target='_blank'>Montazeri et al., Nucleic Acids Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31395880' target='_blank'>Stewart et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31175866' target='_blank'>Fischer et al., Gastroenterology, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29303508' target='_blank'>Sohn et al., Exp Mol Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27678457' target='_blank'>Ma et al., Clin Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25388163' target='_blank'>Previs et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25016225' target='_blank'>Vilarinho et al., J Hepatol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24844798' target='_blank'>Bogorad et al., Nat Commun, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24813447' target='_blank'>Lee et al., Nucleic Acids Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22142829' target='_blank'>Shukla et al., Clin Cancer Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18006687' target='_blank'>Delmas et al., Genes Dev, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16239965' target='_blank'>Balint et al., J Clin Invest, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14563124' target='_blank'>Deveraux et al., Semin Cancer Biol, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12912959' target='_blank'>Zhang et al., Clin Cancer Res, 2003<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36348403' target='_blank'>Ghafouri-Fard et al., Cancer Cell Int, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33931647' target='_blank'>Supper et al., Nat Commun, 2021<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/32739133' target='_blank'>Beecher et al., DNA Repair (Amst), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31635251' target='_blank'>Gilson et al., Int J Mol Sci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18431487' target='_blank'>Kattan et al., PLoS One, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40071783' target='_blank'>Xiao et al., 2025, Cancer Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33659920' target='_blank'>Haddox et al., Fac Rev, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31510685' target='_blank'>Zhang et al., Bioinformatics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27270875' target='_blank'>de et al., Lab Invest, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26865464' target='_blank'>Åman et al., J Pathol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20515481' target='_blank'>Andersson et al., BMC Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20017906' target='_blank'>Bento et al., BMC Cell Biol, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18179710' target='_blank'>Fadlelmola et al., 2008, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17468515' target='_blank'>Pérez-Mancera et al., Carcinogenesis, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16651630' target='_blank'>Engström et al., Am J Pathol, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15688424' target='_blank'>Göransson et al., Int J Cancer, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40129567' target='_blank'>Zhang et al., 2025, Front Cell Dev Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39040050' target='_blank'>Wang et al., 2024, iScience<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30387548' target='_blank'>Wang et al., Cancer Sci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26212035' target='_blank'>Wang et al., Cancer Sci, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25622248' target='_blank'>Leitão et al., Int J Mol Sci, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24995518' target='_blank'>Mactier et al., Pigment Cell Melanoma Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24626184' target='_blank'>Iyer et al., Cell Cycle, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23108395' target='_blank'>Lin et al., Oncogene, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39013964' target='_blank'>Nguyen et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38834947' target='_blank'>Yao et al., 2024, Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36834609' target='_blank'>Shih et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36322827' target='_blank'>Ghashghaei et al., 2022, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34207140' target='_blank'>Zhang et al., Cells, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34158095' target='_blank'>Zhou et al., J Orthop Surg Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32796005' target='_blank'>Lopez et al., Genome Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29314290' target='_blank'>Taniguchi et al., Mol Carcinog, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40600050' target='_blank'>Zhang et al., 2025, Front Med (Lausanne)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40370595' target='_blank'>Sabir et al., 2025, Front Bioeng Biotechnol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39445003' target='_blank'>Chen et al., 2024, MedComm (2020)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37938151' target='_blank'>Chen et al., 2023, Aging (Albany NY)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34713302' target='_blank'>Blyufer et al., 2021, Int J Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34336125' target='_blank'>Lai et al., Oxid Med Cell Longev, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33614240' target='_blank'>Cui et al., Mol Ther Nucleic Acids, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33107235' target='_blank'>Byun et al., 2020, Yonsei Med J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32927122' target='_blank'>Reardon et al., J Thorac Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31817953' target='_blank'>Seliger et al., 2019, J Clin Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31105426' target='_blank'>Wong et al., Biomark Insights, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30683859' target='_blank'>Bhaskaran et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27525019' target='_blank'>Yao et al., 2016, Evid Based Complement Alternat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27355345' target='_blank'>Medina-Aguilar et al., 2016, PLoS One<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25865490' target='_blank'>Lombard et al., Mol Carcinog, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35534623' target='_blank'>Cooper et al., Nat Rev Clin Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35110731' target='_blank'>Xie et al., Nat Rev Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31406302' target='_blank'>Skoulidis et al., Nat Rev Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31150618' target='_blank'>van et al., Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29068003' target='_blank'>Rotow et al., Nat Rev Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28256574' target='_blank'>Schneider et al., Nat Rev Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27388699' target='_blank'>Vargas et al., Nat Rev Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25846554' target='_blank'>Pailler et al., Ann Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22898541' target='_blank'>Vogt et al., Nat Rev Cancer, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22854838' target='_blank'>Henderson et al., Nat Rev Cancer, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21355070' target='_blank'>Chuah et al., Ann Oncol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21175304' target='_blank'>Bredel et al., N Engl J Med, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20602996' target='_blank'>Lemmon et al., Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20574449' target='_blank'>Heyer et al., Nat Rev Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1672832' target='_blank'>Pain et al., Cell, 1991<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34177816' target='_blank'>McFadden et al., Front Endocrinol (Lausanne), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31671564' target='_blank'>Piaggio et al., 2019, Cancers (Basel)<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/29119601' target='_blank'>Larson et al., Genet Epidemiol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21660944' target='_blank'>Asai et al., J Cell Physiol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21350194' target='_blank'>Sivina et al., Arterioscler Thromb Vasc Biol, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40205485' target='_blank'>Wang et al., 2025, Biol Direct<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37786278' target='_blank'>Zhu et al., 2023, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31057610' target='_blank'>Wang et al., J Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26028036' target='_blank'>Peng et al., Oncogene, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38361033' target='_blank'>Hu et al., 2024, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33463050' target='_blank'>Xiong et al., Clin Transl Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30563765' target='_blank'>Westerlund et al., Biochem Biophys Res Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28560380' target='_blank'>Abeni et al., Int J Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27491085' target='_blank'>Noonan et al., Dis Model Mech, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26955619' target='_blank'>Deynoux et al., Front Oncol, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/31836877' target='_blank'>Goutsouliak et al., Nat Rev Clin Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25433701' target='_blank'>Sanchez-Garcia et al., Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24060865' target='_blank'>Talmadge et al., Nat Rev Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23993096' target='_blank'>Zhang et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22992590' target='_blank'>Massagué et al., 2012, Nat Rev Mol Cell Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18662538' target='_blank'>Massagué et al., Cell, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15870086' target='_blank'>Menendez et al., 2005, Ann Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15642702' target='_blank'>Menendez et al., Ann Oncol, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15520077' target='_blank'>Menendez et al., Ann Oncol, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10676567' target='_blank'>Sikic et al., Ann Oncol, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10676565' target='_blank'>Lebwohl et al., Ann Oncol, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2903446' target='_blank'>van et al., N Engl J Med, 1988<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2871941' target='_blank'>Bargmann et al., Cell, 1986<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2567634' target='_blank'>Bouchard et al., Cell, 1989<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1349853' target='_blank'>Wen et al., Cell, 1992<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37155852' target='_blank'>Kim et al., 2023, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36881002' target='_blank'>Yeo et al., 2023, Cancer Immunol Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36069825' target='_blank'>Nussinov et al., Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32704608' target='_blank'>Lee et al., JCO Precis Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32206115' target='_blank'>Liu et al., Theranostics, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32029900' target='_blank'>Chen et al., Oncogene, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30068751' target='_blank'>Wei et al., Mol Cancer Ther, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25248370' target='_blank'>Yan et al., Mol Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23028188' target='_blank'>Liang et al., 2012, Genome Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20492690' target='_blank'>Fleming et al., BMC Med, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19238171' target='_blank'>Wu et al., Cell Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19019207' target='_blank'>Jackson-Fisher et al., Breast Cancer Res, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17667926' target='_blank'>Hsieh et al., Br J Cancer, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15704104' target='_blank'>van et al., Int J Cancer, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12853564' target='_blank'>Holbro et al., Proc Natl Acad Sci U S A, 2003<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40328728' target='_blank'>Prokopov et al., 2025, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37722715' target='_blank'>Honkanen et al., 2023, Cancer Biol Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35354049' target='_blank'>Smith et al., Cell Rep, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34987087' target='_blank'>Lucas et al., 2022, Pharmacol Rev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33918692' target='_blank'>Grzywa et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32533590' target='_blank'>Hong et al., Cancer Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32335999' target='_blank'>Hedegger et al., Mol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32316671' target='_blank'>Aldaregia et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31537638' target='_blank'>Attig et al., Genome Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30068751' target='_blank'>Wei et al., Mol Cancer Ther, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30044378' target='_blank'>Donoghue et al., Cancers (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25516216' target='_blank'>Wali et al., Breast Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25048628' target='_blank'>Supernat et al., 2014, Transl Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23388702' target='_blank'>Deyle et al., Gene Ther, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19718025' target='_blank'>Prickett et al., Nat Genet, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34230470' target='_blank'>Zoma et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32661416' target='_blank'>Zhao et al., Nat Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32636365' target='_blank'>Goldman et al., 2020, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31511663' target='_blank'>Bushweller et al., Nat Rev Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29455671' target='_blank'>Werner et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29242641' target='_blank'>Senft et al., Nat Rev Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28198367' target='_blank'>Potu et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27604872' target='_blank'>Unnikrishnan et al., Nucleic Acids Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26058078' target='_blank'>Nguyen et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25653093' target='_blank'>Lunardi et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24634505' target='_blank'>Graham et al., J Natl Cancer Inst, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23549870' target='_blank'>Kron et al., Clin Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21385925' target='_blank'>Haffner et al., Clin Cancer Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20944599' target='_blank'>Kelly et al., Nat Biotechnol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20236477' target='_blank'>Newman et al., 2010, Genome Med<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40494873' target='_blank'>Buqué et al., 2025, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39558215' target='_blank'>Fang et al., 2024, Genome Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39215346' target='_blank'>Zhuang et al., 2024, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38605023' target='_blank'>Liu et al., 2024, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37696456' target='_blank'>Opitz et al., 2023, Biochem Pharmacol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34294894' target='_blank'>Hickey et al., Br J Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34215580' target='_blank'>Lopes et al., 2021, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34199844' target='_blank'>Waddell et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32923169' target='_blank'>Buque et al., Oncoimmunology, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32046982' target='_blank'>Rodriguez et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31710162' target='_blank'>Li et al., Cancer Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31362937' target='_blank'>Blanchard et al., Genome Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27005559' target='_blank'>Irish et al., Mol Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26395334' target='_blank'>Krupska et al., Hum Genomics, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23443804' target='_blank'>Nickols et al., Mol Cancer Ther, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38388484' target='_blank'>Wang et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36064595' target='_blank'>Jiang et al., Nat Rev Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34071193' target='_blank'>Wessely et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31862871' target='_blank'>Li et al., Cell Death Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31160676' target='_blank'>Oh et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30557328' target='_blank'>Gyvyte et al., PLoS One, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30224643' target='_blank'>Orlando et al., Nat Genet, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29207642' target='_blank'>Jin et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28912153' target='_blank'>Johnson et al., Oncologist, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27980689' target='_blank'>Babaian et al., Mob DNA, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25653093' target='_blank'>Lunardi et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23774214' target='_blank'>Oh et al., Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20160028' target='_blank'>Jané-Valbuena et al., Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20107508' target='_blank'>Baker et al., PLoS One, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17876339' target='_blank'>Nam et al., Prostate Cancer Prostatic Dis, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40469297' target='_blank'>Huang et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34553037' target='_blank'>Jiang et al., Mol Ther Oncolytics, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30250553' target='_blank'>Ao et al., Oncol Lett, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29996935' target='_blank'>Dumortier et al., Breast Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29371979' target='_blank'>Zeng et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27633981' target='_blank'>Panagopoulos et al., Oncol Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18977342' target='_blank'>Kurpios et al., Dev Biol, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39614096' target='_blank'>Su et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38515748' target='_blank'>Shi et al., 2024, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35022500' target='_blank'>Bluemn et al., 2022, Leukemia<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39333095' target='_blank'>Chen et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37185814' target='_blank'>Lv et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35709354' target='_blank'>Tanaka et al., Blood, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34196511' target='_blank'>Song et al., Cancer Commun (Lond), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33903593' target='_blank'>Bleu et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33010230' target='_blank'>Jofra et al., Mol Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32024999' target='_blank'>Akdemir et al., Nat Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31676734' target='_blank'>Rashkovan et al., Genes Dev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30462309' target='_blank'>Ivanochko et al., Nucleic Acids Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28191887' target='_blank'>Fenouille et al., Nat Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27397025' target='_blank'>George et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27340173' target='_blank'>Denker et al., Genes Dev, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25433809' target='_blank'>Kim et al., J Hematol Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19208846' target='_blank'>Laricchia-Robbio et al., Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15889140' target='_blank'>Yuasa et al., EMBO J, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35639772' target='_blank'>Vo et al., Nucleic Acids Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34009296' target='_blank'>Saulnier et al., Nucleic Acids Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33753930' target='_blank'>Dharia et al., Nat Genet, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33080033' target='_blank'>Shi et al., Nucleic Acids Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32407673' target='_blank'>Wimalasena et al., Mol Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31392992' target='_blank'>Selvanathan et al., Nucleic Acids Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31011208' target='_blank'>Yang et al., Nat Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28844694' target='_blank'>Boulay et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27587584' target='_blank'>Hou et al., Nucleic Acids Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25453903' target='_blank'>Riggi et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23093599' target='_blank'>Nhili et al., Nucleic Acids Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20375099' target='_blank'>Boeva et al., Nucleic Acids Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19584866' target='_blank'>Erkizan et al., Nat Med, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17541417' target='_blank'>de et al., Nat Rev Drug Discov, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15743974' target='_blank'>Alex et al., Nucleic Acids Res, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40279411' target='_blank'>Yin et al., 2025, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39915447' target='_blank'>Zheng et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37934116' target='_blank'>Saggese et al., 2023, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33795875' target='_blank'>Yi et al., Nat Cell Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31123062' target='_blank'>Yu et al., Genes Dev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30890554' target='_blank'>Basheer et al., J Exp Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30022044' target='_blank'>Anwar et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29425303' target='_blank'>Koubi et al., Nucleic Acids Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28329683' target='_blank'>Vo et al., Cell Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27881582' target='_blank'>Cardenas et al., Clin Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27312177' target='_blank'>Zhang et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26845405' target='_blank'>Kim et al., Nat Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24491801' target='_blank'>Kim et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24211739' target='_blank'>Xu et al., J Hepatol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19041150' target='_blank'>Chang et al., J Hepatol, 2009<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36969004' target='_blank'>Strøbech et al., 2023, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19997510' target='_blank'>Salemi et al., PLoS One, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/3162757' target='_blank'>Yu et al., Mol Pharmacol, 1988<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2010659' target='_blank'>Brach et al., Int J Cell Cloning, 1991<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1984516' target='_blank'>Smithgall et al., J Natl Cancer Inst, 1991<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1373879' target='_blank'>Jücker et al., Oncogene, 1992<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1310887' target='_blank'>Nishio et al., Cancer, 1992<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36482474' target='_blank'>Fu et al., J Hematol Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34093104' target='_blank'>Azizi et al., Adv Funct Mater, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34083510' target='_blank'>Peng et al., Signal Transduct Target Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32968054' target='_blank'>Chen et al., 2020, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32859279' target='_blank'>Lucas et al., Acta Neuropathol Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31815619' target='_blank'>Zhang et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30639621' target='_blank'>Quintanal-Villalonga et al., 2019, J Thorac Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27315356' target='_blank'>Lim et al., Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26940867' target='_blank'>Hnisz et al., Science, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26581390' target='_blank'>Holdman et al., Breast Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26015511' target='_blank'>Göke et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23151501' target='_blank'>Gru et al., Breast Cancer Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22684217' target='_blank'>Schildhaus et al., Mod Pathol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21160076' target='_blank'>Turner et al., Sci Transl Med, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20940404' target='_blank'>Yang et al., Cancer Res, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39738108' target='_blank'>Bannier et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39161208' target='_blank'>Noeraparast et al., 2024, Cancer Commun (Lond)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37956738' target='_blank'>Necchi et al., 2023, Ann Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35151844' target='_blank'>Li et al., Mol Ther, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34795215' target='_blank'>Liu et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32682615' target='_blank'>van Rhijn et al., 2020, Eur Urol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27812792' target='_blank'>Huse et al., Acta Neuropathol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26015511' target='_blank'>Göke et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25991007' target='_blank'>Zhou et al., 2015, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23867472' target='_blank'>Yin et al., Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23744832' target='_blank'>Herrera-Abreu et al., Cancer Discov, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23298839' target='_blank'>Babic et al., J Clin Invest, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20393505' target='_blank'>Zingone et al., Leukemia, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18939995' target='_blank'>Chen et al., 2008, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11157491' target='_blank'>Chesi et al., 2001, Blood<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37045473' target='_blank'>White et al., 2023, J Immunother Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34840108' target='_blank'>Saborowski et al., Trends Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31523191' target='_blank'>Li et al., Int J Biol Sci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30815737' target='_blank'>Sugihara et al., 2019, J Gastroenterol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30321816' target='_blank'>Luo et al., Mol Ther Nucleic Acids, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29186506' target='_blank'>Xia et al., Nucleic Acids Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29038531' target='_blank'>Berrout et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28629429' target='_blank'>Greer et al., Genome Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28628842' target='_blank'>Verlingue et al., 2017, Eur J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27640304' target='_blank'>Ghoussaini et al., Am J Hum Genet, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26482039' target='_blank'>Yoon et al., Clin Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25315765' target='_blank'>Nadauld et al., Genome Biol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24947926' target='_blank'>Yoon et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21725289' target='_blank'>Gatius et al., Mod Pathol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19088016' target='_blank'>Garcia-Closas et al., Clin Cancer Res, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39880914' target='_blank'>Azahaf et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38604131' target='_blank'>Schwarz et al., 2024, J Med Chem<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36835813' target='_blank'>Gui et al., 2023, J Clin Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35142107' target='_blank'>Ornitz et al., 2022, WIREs Mech Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34913780' target='_blank'>Xie et al., 2021, Expert Opin Investig Drugs<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33897880' target='_blank'>He et al., Theranostics, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33482911' target='_blank'>Huang et al., Genome Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32879300' target='_blank'>Xie et al., Signal Transduct Target Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32555680' target='_blank'>Qiao et al., PLoS One, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31027687' target='_blank'>Quintanal-Villalonga et al., Lung Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30403487' target='_blank'>Lu et al., J Med Chem, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27154171' target='_blank'>Choi et al., J Transl Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24503538' target='_blank'>Turkington et al., Cell Death Dis, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24332042' target='_blank'>Li et al., Cancer Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19946327' target='_blank'>Roidl et al., Oncogene, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39805813' target='_blank'>Kapplingattu et al., 2025, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35879366' target='_blank'>Dorado et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34522706' target='_blank'>Liu et al., 2021, Genes Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33455033' target='_blank'>Torrence et al., Genes Chromosomes Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33158279' target='_blank'>Tabatabaeian et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32178422' target='_blank'>Hueso et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32046161' target='_blank'>Geng et al., 2020, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31695788' target='_blank'>Lin et al., Theranostics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31557894' target='_blank'>Gehringer et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31300419' target='_blank'>Zhou et al., Blood Adv, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30459282' target='_blank'>Bharathy et al., Sci Signal, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28053699' target='_blank'>Jin et al., Cell Biosci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24884732' target='_blank'>Yang et al., J Transl Med, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22344295' target='_blank'>Lu et al., Nat Med, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20815877' target='_blank'>Chhabra et al., Mol Cancer, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40034125' target='_blank'>Lin et al., 2025, Genes Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38759605' target='_blank'>Guo et al., 2024, Transl Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37060016' target='_blank'>Wei et al., 2023, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34479611' target='_blank'>Weng et al., J Hematol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34363438' target='_blank'>Bi et al., 2021, J Cell Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33430880' target='_blank'>Wang et al., Mol Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32100957' target='_blank'>Yin et al., Thorac Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31199052' target='_blank'>Chang et al., J Cell Mol Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30483771' target='_blank'>Nie et al., Int J Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29620202' target='_blank'>Zhang et al., Mol Med Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29484124' target='_blank'>Lallemand et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29326122' target='_blank'>Thivakaran et al., Haematologica, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29050362' target='_blank'>Monteleone et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28210150' target='_blank'>Barse et al., Lung Cancer (Auckl), 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26638886' target='_blank'>Lu et al., Curr Cancer Drug Targets, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39476082' target='_blank'>Sudalagunta et al., 2024, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37468459' target='_blank'>Hughes et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37243594' target='_blank'>Kitagawa et al., 2023, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35803262' target='_blank'>Povedano et al., Cell Chem Biol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30537986' target='_blank'>Chen et al., 2018, Genome Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25662499' target='_blank'>Bertrand et al., Biomaterials, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25314065' target='_blank'>Inoue et al., Br J Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24362521' target='_blank'>Parrish et al., Oncogene, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23831569' target='_blank'>Huang et al., Oncogene, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22829238' target='_blank'>Li et al., Blood Cancer J, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18591936' target='_blank'>García-Aragoncillo et al., Oncogene, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18413612' target='_blank'>Sanchez et al., 2008, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17047056' target='_blank'>Tirado et al., Cancer Res, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15492248' target='_blank'>Rorie et al., Cancer Res, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8502483' target='_blank'>Zhang et al., Oncogene, 1993<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39592626' target='_blank'>Pogorelov et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39510293' target='_blank'>Smith et al., 2024, Pharmacol Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35288466' target='_blank'>Mehta et al., J Immunother Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32492433' target='_blank'>Wesely et al., 2020, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31050342' target='_blank'>Shi et al., Elife, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29471413' target='_blank'>Feld et al., Nucleic Acids Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28923853' target='_blank'>Bruner et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28895560' target='_blank'>Reiter et al., Leukemia, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28162770' target='_blank'>Wang et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27280396' target='_blank'>Jiang et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25493654' target='_blank'>Obata et al., Nat Commun, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23340802' target='_blank'>Volpe et al., Leukemia, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23149921' target='_blank'>Tam et al., Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21647154' target='_blank'>Andersson et al., 2011, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21453545' target='_blank'>Takahashi et al., J Hematol Oncol, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40575034' target='_blank'>Yang et al., 2025, Gastroenterol Rep (Oxf)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34733991' target='_blank'>Huang et al., Ann Transl Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33931674' target='_blank'>Wagner et al., Sci Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24924428' target='_blank'>Watari et al., 2014, PLoS One<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36631900' target='_blank'>Linke et al., Acta Neuropathol Commun, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36380016' target='_blank'>Yoon et al., Exp Mol Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34429326' target='_blank'>Zhang et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33888713' target='_blank'>Wang et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33499874' target='_blank'>Shu et al., J Exp Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33203845' target='_blank'>Attwood et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32958058' target='_blank'>Zhao et al., J Exp Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32348753' target='_blank'>Xiao et al., Cell Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32127023' target='_blank'>Park et al., Cell Commun Signal, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31085323' target='_blank'>Schmidt et al., 2019, Semin Cell Dev Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30523261' target='_blank'>Sun et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27066314' target='_blank'>Sanders et al., J Cachexia Sarcopenia Muscle, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26279297' target='_blank'>Nanos-Webb et al., Oncogene, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26187774' target='_blank'>Bauer et al., Drugs, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24717934' target='_blank'>Morran et al., Gut, 2014<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40251223' target='_blank'>Ouyang et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34072771' target='_blank'>Nagel et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31378673' target='_blank'>Menendez-Gonzalez et al., Stem Cell Reports, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29760808' target='_blank'>Chen et al., J Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27570183' target='_blank'>Herz et al., 2016, Bioessays<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39448444' target='_blank'>Roohy et al., 2024, Discov Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37203412' target='_blank'>Pu et al., 2023, Int J Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36629158' target='_blank'>Zhao et al., 2023, Asian J Androl<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36566215' target='_blank'>Lu et al., 2022, J Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34399777' target='_blank'>Qi et al., Biomed Eng Online, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33918793' target='_blank'>Yoshida et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33803938' target='_blank'>Moriguchi et al., Biomedicines, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33748145' target='_blank'>Zhao et al., Front Cell Dev Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32120995' target='_blank'>Bojcsuk et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31980032' target='_blank'>Yang et al., Genome Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31900418' target='_blank'>Colaprico et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29434343' target='_blank'>Goyal et al., Mod Pathol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29233960' target='_blank'>Winters et al., 2017, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26432018' target='_blank'>Dow et al., 2015, Trends Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18500686' target='_blank'>Thompson et al., Prostate, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40422229' target='_blank'>Piao et al., 2025, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36272230' target='_blank'>Schepers et al., 2022, J Surg Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36232466' target='_blank'>Bao et al., Int J Mol Sci, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34917553' target='_blank'>Aydin et al., 2021, J Hepatocell Carcinoma<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33735525' target='_blank'>Luo et al., Hepatology, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33420124' target='_blank'>Meng et al., Sci Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33076548' target='_blank'>Sherif et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31053802' target='_blank'>Motawi et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29805640' target='_blank'>Fei et al., Oncol Lett, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28869105' target='_blank'>Luan et al., Pathol Res Pract, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28476031' target='_blank'>Cartier et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27259271' target='_blank'>Han et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26482785' target='_blank'>Fernández-Vega et al., BMC Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20725905' target='_blank'>Lai et al., Hepatology, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40180907' target='_blank'>Xue et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38956074' target='_blank'>Peng et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38104183' target='_blank'>Rampioni Vinciguerra et al., 2023, Cell Death Differ<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36964133' target='_blank'>Ruze et al., 2023, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33127677' target='_blank'>Fang et al., Sci Adv, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31901237' target='_blank'>Song et al., Mol Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30737406' target='_blank'>Wright et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29455672' target='_blank'>Massimino et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27492255' target='_blank'>Gerling et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26603838' target='_blank'>Agarwal et al., Blood, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26168014' target='_blank'>Hobbs et al., Nat Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24975580' target='_blank'>Hirsova et al., Hepatology, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24297113' target='_blank'>Saratsis et al., Acta Neuropathol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22038837' target='_blank'>Fingas et al., Hepatology, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21076395' target='_blank'>Jagani et al., Nat Med, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40550933' target='_blank'>Pan et al., 2025, Commun Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38360887' target='_blank'>Mengoni et al., 2024, Cancer Gene Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38233537' target='_blank'>Yasuta et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31227497' target='_blank'>Anbunathan et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30204251' target='_blank'>Parish et al., Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26991400' target='_blank'>Chattopadhyay et al., Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25315378' target='_blank'>Küsters-Vandevelde et al., Pathol Oncol Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24508914' target='_blank'>O'Hayre et al., Curr Opin Cell Biol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24458518' target='_blank'>Bekers et al., Virchows Arch, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24141786' target='_blank'>Chen et al., Oncogene, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23778528' target='_blank'>Koopmans et al., Br J Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23177739' target='_blank'>Vaqué et al., Mol Cell, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40550933' target='_blank'>Pan et al., 2025, Commun Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39804140' target='_blank'>Longakit et al., 2025, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38360887' target='_blank'>Mengoni et al., 2024, Cancer Gene Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36112789' target='_blank'>Vaishnavi et al., Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34939927' target='_blank'>Urtatiz et al., Elife, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31227497' target='_blank'>Anbunathan et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30773340' target='_blank'>Feng et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30204251' target='_blank'>Parish et al., Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29376139' target='_blank'>Astone et al., NPJ Precis Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26991400' target='_blank'>Chattopadhyay et al., Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26619946' target='_blank'>Ascierto et al., J Transl Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24882515' target='_blank'>Feng et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24141786' target='_blank'>Chen et al., Oncogene, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23778528' target='_blank'>Koopmans et al., Br J Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23177739' target='_blank'>Vaqué et al., Mol Cell, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40745181' target='_blank'>Gill et al., 2025, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39219164' target='_blank'>Nitschke et al., 2024, Mol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35452280' target='_blank'>Ferguson et al., Sci Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33915081' target='_blank'>Huang et al., Cell Stem Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32696980' target='_blank'>Huang et al., J Pathol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26733942' target='_blank'>Peverelli et al., Front Endocrinol (Lausanne), 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20153804' target='_blank'>Chesnokova et al., Mol Cell Endocrinol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19106661' target='_blank'>Jamieson et al., Curr Opin Oncol, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11081196' target='_blank'>Buchfelder et al., Pituitary, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10882539' target='_blank'>Farrell et al., Front Neuroendocrinol, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7874835' target='_blank'>Vessey et al., 1994, Clin Sci (Lond)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7761094' target='_blank'>Wong et al., 1995, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7630645' target='_blank'>Challeton et al., Oncogene, 1995<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2115641' target='_blank'>Clementi et al., Oncogene, 1990<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1851251' target='_blank'>Wong et al., Nature, 1991<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39963421' target='_blank'>Ivanov et al., 2025, Comput Struct Biotechnol J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39318795' target='_blank'>Lu et al., 2024, Heliyon<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37041291' target='_blank'>Yang et al., 2023, Cell Death Differ<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31474868' target='_blank'>Mitkevich et al., Front Pharmacol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30185549' target='_blank'>Liang et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29775654' target='_blank'>Bi et al., Biochim Biophys Acta Rev Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26955619' target='_blank'>Deynoux et al., Front Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26872370' target='_blank'>Wang et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25477702' target='_blank'>Wang et al., Dis Markers, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25341034' target='_blank'>Kang et al., 2014, Cell Death Dis<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40739289' target='_blank'>Jiang et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28452374' target='_blank'>Wang et al., 2017, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24296488' target='_blank'>Jung et al., Cancer Res, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40779629' target='_blank'>Xiang et al., 2025, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33578515' target='_blank'>Liu et al., Medicine (Baltimore), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31827209' target='_blank'>Maoz et al., Sci Rep, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38774408' target='_blank'>Wang et al., 2024, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37370109' target='_blank'>Yang et al., 2023, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33755320' target='_blank'>Qiao et al., Thorac Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33505435' target='_blank'>Stewart et al., Front Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31284594' target='_blank'>Takakura et al., 2019, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30405205' target='_blank'>Unknown et al., Br J Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30365085' target='_blank'>He et al., Mol Med Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30104887' target='_blank'>Zhou et al., Onco Targets Ther, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29789601' target='_blank'>Fu et al., Cell Death Dis, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29088818' target='_blank'>Xu et al., 2017, Oncotarget<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25572132' target='_blank'>Zu et al., Int J Mol Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24837491' target='_blank'>Leung et al., PLoS One, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19903768' target='_blank'>Hillion et al., Mol Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19820691' target='_blank'>Hristov et al., Mod Pathol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17483309' target='_blank'>Tesfaye et al., Cancer Res, 2007<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40764425' target='_blank'>Tang et al., 2025, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38324694' target='_blank'>Gao et al., 2024, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38297963' target='_blank'>Muquith et al., 2024, Oncologist<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36280707' target='_blank'>Chavez et al., Nat Rev Nephrol, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35835117' target='_blank'>Orstad et al., 2022, Dev Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35703180' target='_blank'>Sahoo et al., J Clin Invest, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35121955' target='_blank'>Zhang et al., Nat Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33854885' target='_blank'>Li et al., Adv Sci (Weinh), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32929382' target='_blank'>Zhang et al., Theranostics, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32568072' target='_blank'>Okaty et al., Elife, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31913353' target='_blank'>Nagarajan et al., Nat Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31408618' target='_blank'>Shah et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29077937' target='_blank'>Lenoir et al., Nucleic Acids Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28658208' target='_blank'>Rheinbay et al., Nature, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19470727' target='_blank'>Nucera et al., Clin Cancer Res, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38383439' target='_blank'>Feichtenschlager et al., 2024, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37977260' target='_blank'>Han et al., 2023, J Adv Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36177774' target='_blank'>Wang et al., 2022, IUBMB Life<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34277403' target='_blank'>Liu et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34108450' target='_blank'>Han et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33505405' target='_blank'>Zhang et al., Front Immunol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33134163' target='_blank'>Guo et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32826868' target='_blank'>Liu et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32710725' target='_blank'>Ji et al., Aging (Albany NY), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32582536' target='_blank'>Li et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32485038' target='_blank'>Cheng et al., J Cell Mol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28077929' target='_blank'>Dai et al., Cancer Cell Int, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27530828' target='_blank'>Anczuków et al., RNA, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26030368' target='_blank'>Cho et al., ACS Chem Neurosci, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21586613' target='_blank'>Golan-Gerstl et al., Cancer Res, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40261968' target='_blank'>Alvarez-Torres et al., 2025, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34428540' target='_blank'>Poturnajova et al., Cell Signal, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33946897' target='_blank'>Inaba et al., 2021, J Clin Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33728716' target='_blank'>Okabe et al., Cancer Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33081391' target='_blank'>Bayón-Calderón et al., 2020, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32591643' target='_blank'>Loontiens et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32318567' target='_blank'>Lee et al., Front Cell Dev Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30733805' target='_blank'>Vassalli et al., Stem Cells Int, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29735923' target='_blank'>Sakata et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22366949' target='_blank'>Della et al., Nat Med, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21892159' target='_blank'>Zenatti et al., Nat Genet, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17555447' target='_blank'>Dixon et al., Br J Haematol, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16398656' target='_blank'>Owens et al., 2006, Br J Haematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15897879' target='_blank'>Riz et al., Oncogene, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11066085' target='_blank'>Watt et al., Genes Chromosomes Cancer, 2000<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39333095' target='_blank'>Chen et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38637657' target='_blank'>Lawson et al., 2024, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38016946' target='_blank'>Wright et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36192425' target='_blank'>Talarmain et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32868895' target='_blank'>Huang et al., Nat Chem Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32014012' target='_blank'>Peng et al., J Exp Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31031138' target='_blank'>Paris et al., Cell Stem Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29038232' target='_blank'>Northey et al., Cancer Discov, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28945229' target='_blank'>Serio et al., Oncogene, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28798948' target='_blank'>Brechka et al., Genes Dis, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26552488' target='_blank'>Han et al., Mol Cells, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24987060' target='_blank'>Ezponda et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24532712' target='_blank'>Lehnertz et al., Genes Dev, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23455154' target='_blank'>Martin et al., EMBO J, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12393433' target='_blank'>Mayotte et al., Blood, 2002<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33372601' target='_blank'>Meng et al., Cancer Cell Int, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31695791' target='_blank'>Wang et al., Theranostics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30106131' target='_blank'>Yang et al., Int J Mol Med, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33767982' target='_blank'>Bresesti et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32550915' target='_blank'>Lu et al., Theranostics, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31165722' target='_blank'>Liu et al., Cell Death Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28766961' target='_blank'>Dong et al., Future Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27108607' target='_blank'>Chang et al., Oncol Rep, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39875357' target='_blank'>Wang et al., 2025, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33713247' target='_blank'>Saller et al., Dig Dis Sci, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31923422' target='_blank'>Cui et al., Life Sci, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/29168599' target='_blank'>Luo et al., Cancer Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28979806' target='_blank'>Yao et al., Am J Cancer Res, 2017<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/33162825' target='_blank'>Zhang et al., Int J Biol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32065117' target='_blank'>Xu et al., 2020, Med Sci Monit<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20080498' target='_blank'>Liu et al., Vet Pathol, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40343987' target='_blank'>Maciag et al., 2025, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38828688' target='_blank'>Xu et al., 2024, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38471240' target='_blank'>Tabernero et al., 2024, ESMO Open<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36635501' target='_blank'>Tang et al., Nat Cell Biol, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34599005' target='_blank'>Barkley et al., Genome Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29958939' target='_blank'>Connor et al., J Hepatol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26842935' target='_blank'>Dos et al., Mol Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24981601' target='_blank'>Helleday et al., 2014, Nat Rev Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24637993' target='_blank'>Menke et al., J Clin Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24445538' target='_blank'>Wiesner et al., Nat Commun, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20668064' target='_blank'>Zhang et al., Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19584277' target='_blank'>Marella et al., 2009, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2643097' target='_blank'>Hurlin et al., Proc Natl Acad Sci U S A, 1989<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2247480' target='_blank'>Theodorescu et al., Proc Natl Acad Sci U S A, 1990<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1996309' target='_blank'>Reynolds et al., Proc Natl Acad Sci U S A, 1991<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39261905' target='_blank'>Hsieh et al., 2024, J Biomed Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34558654' target='_blank'>Dang et al., 2021, Int J Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31798767' target='_blank'>Mishra et al., Genes Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20042668' target='_blank'>Taraseviciute et al., Am J Pathol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19484261' target='_blank'>Koperek et al., Virchows Arch, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39793065' target='_blank'>Furth et al., 2025, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39025958' target='_blank'>Lin et al., 2024, NPJ Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37311747' target='_blank'>Liao et al., 2023, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35985343' target='_blank'>Shi et al., Cancer Cell, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34458259' target='_blank'>Zhou et al., Front Cell Dev Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33668856' target='_blank'>Khalsa et al., Cells, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31935911' target='_blank'>Li et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29805076' target='_blank'>Waitkus et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29141225' target='_blank'>Krepler et al., Cell Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28944311' target='_blank'>Stewart-Ornstein et al., Curr Opin Syst Biol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28561042' target='_blank'>Sinha et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27063596' target='_blank'>Chaturvedi et al., Leukemia, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26534967' target='_blank'>Andronesi et al., Clin Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25035396' target='_blank'>Ohba et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23369251' target='_blank'>O'Brien et al., Genome Biol, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35313945' target='_blank'>Zeng et al., J Hematol Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34997121' target='_blank'>Schulten et al., Sci Rep, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34200553' target='_blank'>Beyoğlu et al., 2021, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33916579' target='_blank'>Špačková et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25143803' target='_blank'>Lee et al., Biomol Ther (Seoul), 2014<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39346679' target='_blank'>Tao et al., 2024, Ther Adv Hematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36674902' target='_blank'>Toribio et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35581375' target='_blank'>Oliveira et al., Leukemia, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33434514' target='_blank'>Zhou et al., 2021, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31788081' target='_blank'>Wang et al., Oncol Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29720427' target='_blank'>Fan et al., Biosci Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29642462' target='_blank'>Montaño et al., Cancers (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28003999' target='_blank'>Serravalle et al., 2016, Biomed Res Int<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21892159' target='_blank'>Zenatti et al., Nat Genet, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38906156' target='_blank'>Bolomsky et al., 2024, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36077849' target='_blank'>Wong et al., Cancers (Basel), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35877199' target='_blank'>Garces et al., 2022, Am J Surg Pathol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35504924' target='_blank'>Amanda et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34711655' target='_blank'>Naxerova et al., Genes Dev, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32843547' target='_blank'>Manzano et al., mBio, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32014112' target='_blank'>Lee et al., Elife, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31050342' target='_blank'>Shi et al., Elife, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30111820' target='_blank'>Manzano et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22224767' target='_blank'>Shaffer et al., Annu Rev Immunol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19269363' target='_blank'>Luo et al., Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18987657' target='_blank'>Feldman et al., Leukemia, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17296585' target='_blank'>Naresh et al., 2007, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15959530' target='_blank'>Uranishi et al., Leukemia, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10557056' target='_blank'>Yoshida et al., 1999, Leukemia<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39703515' target='_blank'>Li et al., 2024, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35786935' target='_blank'>Thakur et al., J Med Chem, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33680428' target='_blank'>Nahon et al., 2021, Hepat Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33523587' target='_blank'>Xie et al., J Cell Mol Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33290262' target='_blank'>Shen et al., Aging (Albany NY), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30463015' target='_blank'>Wehde et al., Cell Rep, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40437171' target='_blank'>Palandri et al., 2025, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37317974' target='_blank'>Yu et al., 2023, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31697804' target='_blank'>Patel et al., Blood, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29239838' target='_blank'>Nitulescu et al., 2017, EBioMedicine<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28750624' target='_blank'>Chin et al., J Biomed Sci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28611190' target='_blank'>Lv et al., Genes Dev, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27177927' target='_blank'>Han et al., J Hematol Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26949746' target='_blank'>Lipinski et al., Trends Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26182311' target='_blank'>Tefferi et al., JAMA Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24583800' target='_blank'>Meyer et al., 2014, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24158701' target='_blank'>Lee et al., Clin Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21860432' target='_blank'>Reddy et al., Leukemia, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21156283' target='_blank'>Rui et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19647226' target='_blank'>Xiao et al., Cancer Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16493008' target='_blank'>Hoelbl et al., Blood, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37830594' target='_blank'>Velatooru et al., 2023, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36100323' target='_blank'>Long et al., J Adv Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34361750' target='_blank'>Ahmadu et al., 2021, Molecules<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37913894' target='_blank'>Hasselluhn et al., 2023, Gastroenterology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37271807' target='_blank'>Weng et al., 2023, Mil Med Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34288568' target='_blank'>Pan et al., Cancer Commun (Lond), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33058300' target='_blank'>Li et al., Hepatology, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31831742' target='_blank'>Hirata et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30961831' target='_blank'>Schuster et al., Trends Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30053321' target='_blank'>Roderfeld et al., Hepatology, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28488246' target='_blank'>Wu et al., Protein Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28327155' target='_blank'>Tu et al., Mol Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27089238' target='_blank'>Lukey et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21368833' target='_blank'>Inuzuka et al., Nature, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21245048' target='_blank'>Galardi et al., Nucleic Acids Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20214788' target='_blank'>Bitton-Worms et al., Mol Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10667594' target='_blank'>Xiao et al., Cancer Res, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1742737' target='_blank'>Kharbanda et al., Cancer Res, 1991<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40474296' target='_blank'>Ding et al., 2025, Breast Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39956818' target='_blank'>Xu et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39693033' target='_blank'>Alobaida et al., 2024, Mol Divers<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35777330' target='_blank'>Bonsang et al., Ann Diagn Pathol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34439877' target='_blank'>Ha et al., Biomolecules, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33680090' target='_blank'>El-Khayat et al., Ecancermedicalscience, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33410241' target='_blank'>Watanabe et al., Cancer Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32243127' target='_blank'>Xu et al., ACS Chem Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29854308' target='_blank'>McDonald et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26893681' target='_blank'>Kim et al., Oncol Lett, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26080815' target='_blank'>Pfister et al., Genes Dev, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23516391' target='_blank'>Shah et al., PLoS One, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20368568' target='_blank'>Lam et al., J Clin Oncol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20068150' target='_blank'>Kool et al., Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19563658' target='_blank'>Longatto-Filho et al., BMC Cancer, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39915447' target='_blank'>Zheng et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36897957' target='_blank'>Zhu et al., 2023, Sci Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31666694' target='_blank'>Flavahan et al., Nature, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31126316' target='_blank'>Liu et al., J Hematol Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29449689' target='_blank'>Frost et al., Nat Rev Endocrinol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28196207' target='_blank'>Heinrich et al., JAMA Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24356634' target='_blank'>Ganjoo et al., Ann Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23775962' target='_blank'>Hodi et al., J Clin Oncol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22617245' target='_blank'>Yasuda et al., J Thorac Oncol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20832752' target='_blank'>Mir et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20651610' target='_blank'>Petrini et al., J Thorac Oncol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19834118' target='_blank'>Paranjape et al., Gut, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16908931' target='_blank'>Curtin et al., J Clin Oncol, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14551311' target='_blank'>Yorke et al., J Clin Oncol, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9438854' target='_blank'>Hirota et al., Science, 1998<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/25323927' target='_blank'>Cox et al., Nat Rev Drug Discov, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24651013' target='_blank'>Ntziachristos et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23332765' target='_blank'>Burd et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22698401' target='_blank'>Liu et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21665147' target='_blank'>Guerra et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21514245' target='_blank'>Blasco et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21251614' target='_blank'>DuPage et al., 2011, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20832755' target='_blank'>Hatley et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20609353' target='_blank'>Puyol et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20129250' target='_blank'>Takahashi et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19878870' target='_blank'>Gidekel et al., Cancer Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17349585' target='_blank'>Guerra et al., Cancer Cell, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10436819' target='_blank'>Aspinall et al., Ann Oncol, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8616111' target='_blank'>Jacobson et al., Ann Oncol, 1995<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2199829' target='_blank'>Slebos et al., N Engl J Med, 1990<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36597889' target='_blank'>Guo et al., J Clin Lab Anal, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33558504' target='_blank'>Emad et al., NPJ Syst Biol Appl, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40579412' target='_blank'>Costea et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38321024' target='_blank'>Fang et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32605588' target='_blank'>Terkelsen et al., Breast Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23870818' target='_blank'>Skrzypski et al., 2013, Clin Lung Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8255614' target='_blank'>Danielian et al., Pathol Biol (Paris), 1993<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38653778' target='_blank'>Banerjee et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37501079' target='_blank'>Wang et al., 2023, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36632736' target='_blank'>Smith et al., Haematologica, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31819055' target='_blank'>Wang et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30867811' target='_blank'>Zhang et al., Theranostics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30791932' target='_blank'>Liang et al., J Exp Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29445148' target='_blank'>Zhang et al., Cell Death Dis, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28867147' target='_blank'>Zhu et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28718439' target='_blank'>Sengupta et al., Trends Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28260788' target='_blank'>Li et al., Leukemia, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28256574' target='_blank'>Schneider et al., Nat Rev Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26940867' target='_blank'>Hnisz et al., Science, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26539795' target='_blank'>Bosse et al., Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23302769' target='_blank'>Oram et al., Leukemia, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21124317' target='_blank'>Wang et al., Nature, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36920307' target='_blank'>Abdulla et al., 2023, J Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35121955' target='_blank'>Zhang et al., Nat Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31815635' target='_blank'>Kim et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28351982' target='_blank'>Inoue et al., J Exp Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27887606' target='_blank'>Shar et al., Mol Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26940867' target='_blank'>Hnisz et al., Science, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25295500' target='_blank'>Hacein-Bey-Abina et al., N Engl J Med, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25060579' target='_blank'>Yui et al., Nat Rev Immunol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23926305' target='_blank'>McCormack et al., Blood, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22851642' target='_blank'>Boehm et al., J Exp Med, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21076045' target='_blank'>El et al., Blood, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20093438' target='_blank'>McCormack et al., Science, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19171877' target='_blank'>Landry et al., Blood, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17675302' target='_blank'>Lufino et al., Nucleic Acids Res, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16988660' target='_blank'>Pike-Overzet et al., Nature, 2006<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39668349' target='_blank'>Heidari Horestani et al., 2024, Cell Commun Signal<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33527899' target='_blank'>Takao et al., Elife, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24385720' target='_blank'>Sayitoğlu et al., 2014, Turk J Haematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22058201' target='_blank'>Homminga et al., Haematologica, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39168636' target='_blank'>Chen et al., 2024, Genes Dev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34058615' target='_blank'>Ren et al., Redox Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32999280' target='_blank'>Xu et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29089486' target='_blank'>Cheng et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28300844' target='_blank'>Chang et al., Cell Death Dis, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28008178' target='_blank'>Nair et al., Leukemia, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26873845' target='_blank'>Perumal et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26253381' target='_blank'>Meka et al., Autoimmun Rev, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19203657' target='_blank'>Chinen et al., J Allergy Clin Immunol, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17875808' target='_blank'>Mao et al., Blood, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16424013' target='_blank'>Morito et al., Cancer Res, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15781580' target='_blank'>Turcotte et al., J Exp Med, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15613547' target='_blank'>Toki et al., Blood, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14998494' target='_blank'>Hurt et al., Cancer Cell, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9616139' target='_blank'>Chesi et al., Blood, 1998<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37182602' target='_blank'>Elkrief et al., 2023, J Thorac Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34294128' target='_blank'>Miallot et al., J Hematol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32522803' target='_blank'>Cissé et al., Sci Transl Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32346095' target='_blank'>Petroni et al., Nat Rev Immunol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31413145' target='_blank'>Vilgelm et al., Sci Transl Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30664638' target='_blank'>Wong et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30146162' target='_blank'>Minzel et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25271708' target='_blank'>Wang et al., Nat Commun, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25016295' target='_blank'>Wang et al., Gastroenterology, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22095636' target='_blank'>Embade et al., Hepatology, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21145819' target='_blank'>Tybl et al., J Hepatol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20832751' target='_blank'>Macias et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19609305' target='_blank'>Wang et al., EMBO J, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11705884' target='_blank'>Wang et al., Clin Cancer Res, 2001<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7949188' target='_blank'>Watanabe et al., Blood, 1994<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39329452' target='_blank'>Yu et al., 2024, Cancer Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37942402' target='_blank'>Barakeh et al., 2023, Case Rep Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34697572' target='_blank'>Alatawi et al., 2021, J Skin Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33429878' target='_blank'>Pellegrino et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31439820' target='_blank'>AbuHammad et al., Proc Natl Acad Sci U S A, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30886745' target='_blank'>Jiang et al., Cell Death Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30763338' target='_blank'>Murphy et al., PLoS One, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29262653' target='_blank'>Hauck et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25832654' target='_blank'>Namiki et al., Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22948473' target='_blank'>Longerich et al., Pathologe, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22820643' target='_blank'>Gembarska et al., Nat Med, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19838055' target='_blank'>Wang et al., 2009, Cell Cycle<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39073402' target='_blank'>Dong et al., 2024, Biomol Biomed<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26759750' target='_blank'>Liu et al., Springerplus, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39443938' target='_blank'>Zhu et al., 2024, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38311623' target='_blank'>Kumar et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38302461' target='_blank'>Tao et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37598158' target='_blank'>An et al., 2023, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34238332' target='_blank'>Majeed et al., J Hematol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30343272' target='_blank'>Luna et al., Gut, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29455668' target='_blank'>Zhang et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27343442' target='_blank'>Heist et al., J Thorac Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26899759' target='_blank'>Jänne et al., J Thorac Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23812322' target='_blank'>Belkhiri et al., Gut, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23704319' target='_blank'>Hur et al., Gut, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22042947' target='_blank'>Lennerz et al., J Clin Oncol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17463250' target='_blank'>Engelman et al., Science, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9140397' target='_blank'>Schmidt et al., Nat Genet, 1997<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7989714' target='_blank'>Selden et al., 1994, J Hepatol<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40495229' target='_blank'>Wu et al., 2025, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32341527' target='_blank'>Landi et al., Nat Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28619945' target='_blank'>Di et al., Science, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27124452' target='_blank'>Tirosh et al., Science, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26977879' target='_blank'>Smith et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25490445' target='_blank'>Wolf et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24022702' target='_blank'>Watson et al., Nat Rev Genet, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23416000' target='_blank'>Vazquez et al., Cancer Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22622578' target='_blank'>Berger et al., Nature, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22080950' target='_blank'>Yokoyama et al., Nature, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22012259' target='_blank'>Bertolotto et al., Nature, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20550935' target='_blank'>Levy et al., Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16862190' target='_blank'>Garraway et al., Nat Rev Cancer, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16766266' target='_blank'>Davis et al., Cancer Cell, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16001072' target='_blank'>Garraway et al., Nature, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37474833' target='_blank'>Coorens et al., 2023, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36922589' target='_blank'>Perner et al., 2023, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34002060' target='_blank'>Zhao et al., 2021, Nat Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33833453' target='_blank'>Gu et al., Nat Genet, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31535081' target='_blank'>Ni et al., Nat Metab, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31253180' target='_blank'>Brzezinka et al., J Hematol Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29290617' target='_blank'>Weng et al., Cell Stem Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28241139' target='_blank'>Erb et al., Nature, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26780094' target='_blank'>Qu et al., Nat Methods, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26766589' target='_blank'>Cheung et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24735921' target='_blank'>Matkar et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24679525' target='_blank'>Scadden et al., 2014, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23561442' target='_blank'>Shyh-Chang et al., Cell Stem Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22094252' target='_blank'>Tan et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19212323' target='_blank'>Lim et al., Nature, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38475919' target='_blank'>Du et al., 2024, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32625422' target='_blank'>EFSA et al., EFSA J, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29692408' target='_blank'>Numata et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25264566' target='_blank'>Li et al., Exp Hematol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24695851' target='_blank'>Manara et al., Blood, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38594465' target='_blank'>Shou et al., 2024, Cancer Gene Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31802891' target='_blank'>Gong et al., Onco Targets Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29484124' target='_blank'>Lallemand et al., Oncotarget, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40308577' target='_blank'>Cai et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37019972' target='_blank'>Liu et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34072771' target='_blank'>Nagel et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31105426' target='_blank'>Wong et al., Biomark Insights, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30700727' target='_blank'>Lee et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29928480' target='_blank'>Lamba et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28960191' target='_blank'>Lai et al., Blood Cancer J, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28892045' target='_blank'>Larmonie et al., Oncogene, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23626719' target='_blank'>Kandilci et al., PLoS One, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22436304' target='_blank'>Davidson et al., BMC Med Genet, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38721526' target='_blank'>Koka et al., 2024, Front Cell Dev Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38326297' target='_blank'>Sarson-Lawrence et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36632735' target='_blank'>Cattaneo et al., 2023, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34529761' target='_blank'>Kontandreopoulou et al., Blood Adv, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34068918' target='_blank'>Banerjee et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31890207' target='_blank'>Pernes et al., 2020, Clin Transl Immunology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30021591' target='_blank'>Cho et al., J Transl Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29296692' target='_blank'>Tefferi et al., Blood Adv, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26182311' target='_blank'>Tefferi et al., JAMA Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25642438' target='_blank'>Didonna et al., Ann Clin Transl Neurol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25037629' target='_blank'>Tefferi et al., Blood, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23319699' target='_blank'>Gautier et al., J Exp Med, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23183425' target='_blank'>Sugita et al., Leukemia, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16480521' target='_blank'>Ninos et al., J Transl Med, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8578519' target='_blank'>Vainchenker et al., Thromb Haemost, 1995<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37653757' target='_blank'>Yao et al., 2023, Medicine (Baltimore)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11752899' target='_blank'>Roos et al., Pathobiology, 2001<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35121955' target='_blank'>Zhang et al., Nat Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30146162' target='_blank'>Minzel et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28270163' target='_blank'>Wallaert et al., 2017, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28086098' target='_blank'>Zhang et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21907920' target='_blank'>Blobel et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18574464' target='_blank'>Ramsay et al., Nat Rev Cancer, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18252774' target='_blank'>Palumbo et al., Nucleic Acids Res, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17435759' target='_blank'>Lahortiga et al., Nat Genet, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17081304' target='_blank'>Fu et al., Mol Cancer, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10436788' target='_blank'>Wallrapp et al., Ann Oncol, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/6687762' target='_blank'>Mushinski et al., Science, 1983<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/6585957' target='_blank'>Pelicci et al., Science, 1984<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/6297778' target='_blank'>Radke et al., Cell, 1982<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/3753748' target='_blank'>Rosson et al., Nature, 1986<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2688896' target='_blank'>Ness et al., Cell, 1989<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32020081' target='_blank'>Nair et al., Nat Rev Mol Cell Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30842595' target='_blank'>Karki et al., Nat Rev Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28802037' target='_blank'>Fruman et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28643779' target='_blank'>Dang et al., Nat Rev Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25483776' target='_blank'>Meng et al., Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25342630' target='_blank'>Sabharwal et al., Nat Rev Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24481312' target='_blank'>Fruman et al., Nat Rev Drug Discov, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23791182' target='_blank'>King et al., 2013, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23582323' target='_blank'>Lovén et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23374339' target='_blank'>Alt et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22898541' target='_blank'>Vogt et al., Nat Rev Cancer, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21962511' target='_blank'>Chiarle et al., Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21215367' target='_blank'>Stephens et al., Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20371343' target='_blank'>Nussenzweig et al., Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19041750' target='_blank'>Zhan et al., Cell, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38395864' target='_blank'>Solta et al., 2024, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35748745' target='_blank'>Kelenis et al., 2022, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33514539' target='_blank'>Patel et al., Sci Adv, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31727771' target='_blank'>Oser et al., Genes Dev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30181244' target='_blank'>Jia et al., Cancer Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27620277' target='_blank'>George et al., Clin Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27373156' target='_blank'>Semenova et al., Cell Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27298335' target='_blank'>Kim et al., Genes Dev, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26729287' target='_blank'>Richart et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25490451' target='_blank'>Christensen et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24913641' target='_blank'>Bryant et al., Mol Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24297863' target='_blank'>Bandopadhayay et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21676885' target='_blank'>Xiong et al., Cancer Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2842046' target='_blank'>Johnson et al., Cancer Res, 1988<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1954386' target='_blank'>Hirvonen et al., Blood, 1991<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/27728805' target='_blank'>Dardenne et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26859456' target='_blank'>Ham et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25490451' target='_blank'>Christensen et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24599217' target='_blank'>Marshall et al., Nat Rev Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23792191' target='_blank'>Brockmann et al., Cancer Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22898541' target='_blank'>Vogt et al., Nat Rev Cancer, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22624711' target='_blank'>Swartling et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22439933' target='_blank'>Zhu et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22439928' target='_blank'>Liu et al., 2012, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22340587' target='_blank'>Hogarty et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19111875' target='_blank'>Maris et al., Cancer Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15607966' target='_blank'>Valsesia-Wittmann et al., Cancer Cell, 2004<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8532000' target='_blank'>Norris et al., N Engl J Med, 1996<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8531999' target='_blank'>Caron et al., N Engl J Med, 1996<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/4047115' target='_blank'>Seeger et al., N Engl J Med, 1985<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40718906' target='_blank'>Guérémy et al., 2025, Eur J Neurol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39744584' target='_blank'>Yu et al., 2025, J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38017534' target='_blank'>Li et al., 2023, Cell Commun Signal<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36053490' target='_blank'>Xie et al., Hematol Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33677092' target='_blank'>Tse et al., Mol Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33447829' target='_blank'>Flümann et al., Blood Cancer Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33195243' target='_blank'>Gao et al., Front Cell Dev Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32161170' target='_blank'>Seltzer et al., J Virol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31699794' target='_blank'>de et al., Haematologica, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31521180' target='_blank'>He et al., J Hematol Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26621451' target='_blank'>Kelly et al., J Exp Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25510281' target='_blank'>Yang et al., Immunol Rev, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25501023' target='_blank'>Kersten et al., Blood Cancer J, 2014<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/30772768' target='_blank'>Zhang et al., Neoplasia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30217928' target='_blank'>Wang et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29096894' target='_blank'>Qin et al., Prog Mol Biol Transl Sci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28774282' target='_blank'>Malsy et al., BMC Mol Biol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26831464' target='_blank'>Filipova et al., Sci Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22079596' target='_blank'>Baumgart et al., Gastroenterology, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21192845' target='_blank'>Chen et al., Chin J Cancer, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37838167' target='_blank'>Le Minh et al., 2023, J Biol Chem<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34479611' target='_blank'>Weng et al., J Hematol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33149122' target='_blank'>Cao et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33005474' target='_blank'>Wang et al., 2020, Liver Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32682831' target='_blank'>Schmidlin et al., 2020, Toxicol Appl Pharmacol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32629423' target='_blank'>Liu et al., iScience, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30693913' target='_blank'>Zhou et al., 2019, Med Sci Monit<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30609663' target='_blank'>Weiskirchen et al., 2019, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30561826' target='_blank'>Lee et al., 2018, Hepatology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29500367' target='_blank'>Luo et al., 2018, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28114446' target='_blank'>Campbell et al., JAMA Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27770790' target='_blank'>Wolf et al., BMC Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24123158' target='_blank'>Alvarez et al., Glia, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22239485' target='_blank'>Kachadourian et al., J Med Chem, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21192341' target='_blank'>Rushworth et al., 2010, Br J Pharmacol<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37015223' target='_blank'>Jiang et al., 2023, Cell Chem Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31727944' target='_blank'>Marcos-Villar et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12498710' target='_blank'>Häcker et al., Cancer Cell, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9523851' target='_blank'>Kurzrock et al., J Lab Clin Med, 1998<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34937946' target='_blank'>Thandapani et al., 2021, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30408569' target='_blank'>Sinicropi-Yao et al., J Thorac Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30337457' target='_blank'>Martincorena et al., Science, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30038221' target='_blank'>Kourtis et al., Nat Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28270163' target='_blank'>Wallaert et al., 2017, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27478938' target='_blank'>Ntziachristos et al., Nat Immunol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25471937' target='_blank'>Khammanivong et al., 2014, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24651013' target='_blank'>Ntziachristos et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23434461' target='_blank'>Roti et al., Cancer Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23289374' target='_blank'>Zou et al., J Hematol Oncol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23022380' target='_blank'>Yatim et al., 2012, Mol Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22960745' target='_blank'>Cancer et al., Nature, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22196886' target='_blank'>Liu et al., Mol Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21892159' target='_blank'>Zenatti et al., Nat Genet, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19536265' target='_blank'>Buonamici et al., 2009, Nature<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39149142' target='_blank'>Leng et al., 2024, J Pharm Anal<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37198207' target='_blank'>Xia et al., 2023, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35278613' target='_blank'>Yan et al., Cancer Lett, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33382997' target='_blank'>Gan et al., Exp Cell Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32699528' target='_blank'>Liu et al., Cancer Cell Int, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32457613' target='_blank'>Shi et al., Front Pharmacol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31854042' target='_blank'>Ding et al., Cancer Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30195757' target='_blank'>Luo et al., Mol Ther Nucleic Acids, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29581534' target='_blank'>Liu et al., Sci Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27572051' target='_blank'>Zou et al., Mol Med Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27148486' target='_blank'>Crabtree et al., Front Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26769750' target='_blank'>Hayashi et al., Clin Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25985737' target='_blank'>Zhu et al., Nat Commun, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20967796' target='_blank'>Aster et al., J Pathol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11986231' target='_blank'>Hubmann et al., Blood, 2002<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33619116' target='_blank'>Pawlicki et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33243988' target='_blank'>Spinler et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29587823' target='_blank'>Pan et al., J Hematol Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29455642' target='_blank'>Della et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26503946' target='_blank'>Mossé et al., Clin Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25978431' target='_blank'>Bone et al., Blood Cancer J, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25921060' target='_blank'>Wu et al., Blood, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22829165' target='_blank'>Colomba et al., Blood Cancer J, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22427348' target='_blank'>Cole et al., Clin Cancer Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22072639' target='_blank'>Bresler et al., Sci Transl Med, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20699364' target='_blank'>Lagarrigue et al., Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20043281' target='_blank'>Agrelo et al., EMBO Mol Med, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19531656' target='_blank'>Lim et al., Blood, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16835382' target='_blank'>Fawal et al., Blood, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11801565' target='_blank'>Turturro et al., Clin Cancer Res, 2002<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36947734' target='_blank'>de Braud et al., 2023, J Clin Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36577755' target='_blank'>Shang et al., Signal Transduct Target Ther, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36565724' target='_blank'>Hartmann et al., 2022, J Hepatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35561684' target='_blank'>Choi et al., Cell, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34092233' target='_blank'>Yan et al., Mol Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31124566' target='_blank'>Kristensen et al., Ann Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29804890' target='_blank'>Baumgartner et al., Cell Stem Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29275866' target='_blank'>Kunimoto et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22820643' target='_blank'>Gembarska et al., Nat Med, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18489761' target='_blank'>Field et al., Mol Cancer, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16291983' target='_blank'>Curtin et al., N Engl J Med, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/6089339' target='_blank'>Guerrero et al., Science, 1984<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/3315232' target='_blank'>Paterson et al., Cell, 1987<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/3295562' target='_blank'>Hirai et al., Nature, 1987<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2989702' target='_blank'>Bos et al., Nature, 1985<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39402859' target='_blank'>Desilets et al., 2024, Clin Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32755586' target='_blank'>Vaishnavi et al., Cell Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32127185' target='_blank'>Lu et al., J Thorac Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30741921' target='_blank'>Blondy et al., Cell Death Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29914559' target='_blank'>Farina et al., J Exp Clin Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29191878' target='_blank'>Klaeger et al., Science, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28695888' target='_blank'>Cook et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27843590' target='_blank'>Amatu et al., ESMO Open, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26565381' target='_blank'>Farago et al., J Thorac Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26216294' target='_blank'>Doebele et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25535693' target='_blank'>Morgensztern et al., J Thorac Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25527197' target='_blank'>Vaishnavi et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10646882' target='_blank'>Bounacer et al., Br J Cancer, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8441429' target='_blank'>Nakagawara et al., N Engl J Med, 1993<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/3390374' target='_blank'>Fey et al., Br J Cancer, 1988<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39235852' target='_blank'>Chen et al., 2024, Endocr Connect<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38593276' target='_blank'>Lim et al., 2024, Kaohsiung J Med Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33593392' target='_blank'>Chen et al., J Transl Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29955133' target='_blank'>Choi et al., Br J Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29623306' target='_blank'>Shukla et al., JCO Precis Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29515689' target='_blank'>Lu et al., EPMA J, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29414922' target='_blank'>Kim et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28451457' target='_blank'>Cheng et al., J Pathol Clin Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27351280' target='_blank'>Xiong et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25527197' target='_blank'>Vaishnavi et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23740772' target='_blank'>Natrajan et al., Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18068631' target='_blank'>Li et al., Cancer Cell, 2007<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17425505' target='_blank'>Teixeira et al., Crit Rev Oncog, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34109155' target='_blank'>Singh et al., 2021, Front Chem<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32117772' target='_blank'>Lafitte et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26362312' target='_blank'>Xie et al., J Exp Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26297204' target='_blank'>Lemjabbar-Alaoui et al., Biochim Biophys Acta, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24129700' target='_blank'>Potts et al., Sci Signal, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40389480' target='_blank'>Troester et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40258564' target='_blank'>Fournier et al., 2025, Exp Hematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40166939' target='_blank'>Xu et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39883527' target='_blank'>Hamamoto et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36539553' target='_blank'>Ling et al., Cell Regen, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32993115' target='_blank'>Mohanty et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32729669' target='_blank'>Yu et al., J Cell Mol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28838067' target='_blank'>Bradford et al., ILAR J, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26728554' target='_blank'>Shields et al., Genes Dev, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26285139' target='_blank'>Yoshioka et al., PLoS One, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26064935' target='_blank'>Lu et al., Biomed Res Int, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23102701' target='_blank'>Singer et al., Mol Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22058212' target='_blank'>Petit et al., Haematologica, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12393433' target='_blank'>Mayotte et al., Blood, 2002<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32407673' target='_blank'>Wimalasena et al., Mol Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32144381' target='_blank'>Brack et al., Cell Death Differ, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31823759' target='_blank'>Han et al., BMC Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31614829' target='_blank'>Huilgol et al., Genes (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31302815' target='_blank'>Chay et al., Int J Clin Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30826985' target='_blank'>Liang et al., J Mol Neurosci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30793977' target='_blank'>Pellenz et al., Hum Gene Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30459282' target='_blank'>Bharathy et al., Sci Signal, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29937714' target='_blank'>Zhu et al., Front Mol Neurosci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27458157' target='_blank'>Liu et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26977879' target='_blank'>Smith et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26252164' target='_blank'>Kubic et al., J Cell Biochem, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24188742' target='_blank'>Fang et al., J Cell Mol Med, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22710718' target='_blank'>Thuault et al., Oncogene, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10616906' target='_blank'>Kempf et al., Cell Growth Differ, 1999<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40565027' target='_blank'>Wang et al., 2025, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38780761' target='_blank'>Hosaka et al., 2024, Med Mol Morphol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37221950' target='_blank'>Wang et al., 2023, Cancer Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28076843' target='_blank'>Benzina et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28070224' target='_blank'>Benzina et al., J Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23995789' target='_blank'>Kasar et al., Oncogene, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23960188' target='_blank'>de et al., J Exp Med, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23181381' target='_blank'>Zan et al., Autoimmunity, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15155532' target='_blank'>Baumann et al., Carcinogenesis, 2004<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39377397' target='_blank'>Laub et al., 2024, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38230761' target='_blank'>Ning et al., 2024, Cell Prolif<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37324936' target='_blank'>Sun et al., 2023, Int J Biol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36632737' target='_blank'>Zhang et al., 2023, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36604432' target='_blank'>Sui et al., Signal Transduct Target Ther, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33168647' target='_blank'>Farria et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28261581' target='_blank'>Grebbin et al., Front Cell Dev Biol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27590741' target='_blank'>Jung et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24947929' target='_blank'>Shah et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21548940' target='_blank'>Ramberg et al., 2011, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19767769' target='_blank'>Shimabe et al., Oncogene, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18974129' target='_blank'>Park et al., Cancer Res, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11468159' target='_blank'>DiMartino et al., Blood, 2001<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10637514' target='_blank'>Calvo et al., 2000, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7936638' target='_blank'>Kamps et al., Oncogene, 1994<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33830670' target='_blank'>Dachy et al., J Cell Mol Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27374103' target='_blank'>Cassinelli et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26690113' target='_blank'>Bernard-Gauthier et al., Molecules, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21573601' target='_blank'>Arvelo et al., Int J Oncol, 1993<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16980946' target='_blank'>Abbott et al., Mod Pathol, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12014510' target='_blank'>Levine et al., Vet Pathol, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8662878' target='_blank'>Trejo et al., J Biol Chem, 1996<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8428339' target='_blank'>Peehl et al., Cancer, 1993<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7531297' target='_blank'>Figarella-Branger et al., Neuropathol Appl Neurobiol, 1994<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/6690975' target='_blank'>Bechet et al., N Engl J Med, 1984<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2655888' target='_blank'>Press et al., Cancer Res, 1989<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2212004' target='_blank'>Smidt et al., J Clin Invest, 1990<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2182793' target='_blank'>Kazumoto et al., J Neurosurg, 1990<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2164040' target='_blank'>Maxwell et al., J Clin Invest, 1990<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2163990' target='_blank'>Maxwell et al., Int J Cancer, 1990<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38302461' target='_blank'>Tao et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37494934' target='_blank'>Rahme et al., 2023, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36988488' target='_blank'>Goenka et al., 2023, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36485007' target='_blank'>Kumar et al., 2022, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34433651' target='_blank'>Gonzalez-Buendia et al., Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34002060' target='_blank'>Zhao et al., 2021, Nat Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33277484' target='_blank'>Pajovic et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32941624' target='_blank'>Feng et al., Nucleic Acids Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31534142' target='_blank'>Tarjan et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31032474' target='_blank'>Ryan et al., Nat Metab, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29149895' target='_blank'>Jia et al., Mol Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27021554' target='_blank'>Vanden et al., Leukemia, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20889717' target='_blank'>Ozawa et al., Genes Dev, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20621681' target='_blank'>Bardsley et al., Gastroenterology, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16645167' target='_blank'>Lierman et al., Blood, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37941480' target='_blank'>De Coninck et al., 2023, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34157012' target='_blank'>Carcamo et al., J Pediatr Hematol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31117237' target='_blank'>Salaroglio et al., Int J Mol Sci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30237393' target='_blank'>Grimm et al., Oncogenesis, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29290987' target='_blank'>Zinecker et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29207963' target='_blank'>Park et al., BMC Gastroenterol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20443129' target='_blank'>Reardon et al., J Neurooncol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19417143' target='_blank'>Abouantoun et al., Mol Cancer Ther, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15044254' target='_blank'>Cain et al., Blood, 2004<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37207338' target='_blank'>Jiménez-Santos et al., 2023, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36108220' target='_blank'>Langille et al., 2022, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34591642' target='_blank'>Swaney et al., Science, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33157015' target='_blank'>Bieging-Rolett et al., Mol Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32559461' target='_blank'>Koundouros et al., Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32341551' target='_blank'>Castel et al., Nat Rev Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32059806' target='_blank'>Ying et al., Cell Stem Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31492840' target='_blank'>Cajuso et al., 2019, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29455659' target='_blank'>Montor et al., 2018, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29222441' target='_blank'>Baker et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27604488' target='_blank'>Le et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25458846' target='_blank'>Gasser et al., Mol Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22096235' target='_blank'>Haibe-Kains et al., Nucleic Acids Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19470724' target='_blank'>Agarwal et al., Clin Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9916799' target='_blank'>Shayesteh et al., Nat Genet, 1999<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39552193' target='_blank'>Kong et al., 2024, Thorac Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19200708' target='_blank'>Jia et al., Curr Opin Cell Biol, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40378957' target='_blank'>Azad et al., 2025, J Biol Chem<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37414143' target='_blank'>Smiles et al., 2023, Mol Metab<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36410718' target='_blank'>Duke-Cohan et al., 2022, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33398037' target='_blank'>Ledet et al., Commun Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32976598' target='_blank'>Tan et al., Nucleic Acids Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29516572' target='_blank'>Zhang et al., Cancer Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29467592' target='_blank'>Xu et al., Cancer Cell Int, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28042518' target='_blank'>An et al., Exp Hematol Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26387536' target='_blank'>Blanco et al., 2015, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24739220' target='_blank'>Farooqi et al., Cancer Cell Int, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24329799' target='_blank'>Fowler et al., Immunol Rev, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23352980' target='_blank'>Kim et al., Pharmacol Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22986532' target='_blank'>Ha et al., Oncogene, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21706055' target='_blank'>Stoddart et al., Oncogene, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/3458216' target='_blank'>Nagarajan et al., Proc Natl Acad Sci U S A, 1986<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/28802037' target='_blank'>Fruman et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27191494' target='_blank'>Wu et al., Oncotarget, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38012697' target='_blank'>Lanceta et al., 2023, Mol Cytogenet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30221000' target='_blank'>Xu et al., J Anim Sci Biotechnol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28989535' target='_blank'>Zhang et al., Oncol Lett, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27013583' target='_blank'>Sun et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20687796' target='_blank'>Patz et al., Leuk Lymphoma, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15585652' target='_blank'>Landrette et al., Blood, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32210730' target='_blank'>Lu et al., Int J Biol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30623526' target='_blank'>Tang et al., Hepatol Res, 2019<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40563979' target='_blank'>Zhang et al., 2025, Biology (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39587064' target='_blank'>Pedrosa et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39154024' target='_blank'>Zhang et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38386308' target='_blank'>Laugwitz et al., 2024, Brain<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35987846' target='_blank'>Pandian et al., 2022, Funct Integr Genomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32409685' target='_blank'>Wei et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32235691' target='_blank'>Regouc et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32176716' target='_blank'>Struijk et al., PLoS One, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31399076' target='_blank'>Feng et al., 2019, BMC Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30643464' target='_blank'>Ruan et al., Cancer Manag Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28137313' target='_blank'>Nio et al., 2017, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25684227' target='_blank'>Wu et al., Stem Cell Reports, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20497257' target='_blank'>Lambrot et al., 2010, Int J Androl<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19578360' target='_blank'>West et al., Nature, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17549357' target='_blank'>De et al., Oncol Rep, 2007<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40223139' target='_blank'>Liu et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38512964' target='_blank'>Ma et al., 2024, PLoS Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38233872' target='_blank'>Zhang et al., 2024, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36692000' target='_blank'>Chan et al., Elife, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36302754' target='_blank'>Bauer et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36302174' target='_blank'>Neumayer et al., Clin Cancer Res, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33168813' target='_blank'>Seçilmiş et al., NPJ Syst Biol Appl, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32984053' target='_blank'>Kong et al., Front Oncol, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/26042218' target='_blank'>Berthon et al., Front Cell Dev Biol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20697156' target='_blank'>Molyneux et al., J Clin Invest, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15926108' target='_blank'>Libé et al., Exp Clin Endocrinol Diabetes, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40614202' target='_blank'>Urbach et al., 2025, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39578257' target='_blank'>Chen et al., 2024, Cell Mol Life Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38602457' target='_blank'>Liu et al., 2024, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31492191' target='_blank'>Guo et al., J Exp Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30228208' target='_blank'>Henry et al., 2018, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29290818' target='_blank'>Chen et al., Theranostics, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28630054' target='_blank'>Park et al., Cancer Immunol Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28619758' target='_blank'>Sahu et al., Mol Cancer Ther, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27030341' target='_blank'>Prieto et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26500057' target='_blank'>Park et al., Oncogene, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25684207' target='_blank'>Litvin et al., Mol Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25601637' target='_blank'>Kong et al., Gut, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25316819' target='_blank'>Ravi et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23435375' target='_blank'>Baquero et al., Cancer Lett, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15300592' target='_blank'>Ellenrieder et al., 2004, Gastroenterology<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37098604' target='_blank'>Guo et al., 2023, Mil Med Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36999945' target='_blank'>Ren et al., 2023, Cancer Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35709927' target='_blank'>Dhakar et al., Genomics, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35641658' target='_blank'>Xiang et al., Mod Pathol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33921303' target='_blank'>Baumgartner et al., J Pers Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32483240' target='_blank'>Williams et al., Mod Pathol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32178422' target='_blank'>Hueso et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31581557' target='_blank'>Khaliq et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30613363' target='_blank'>Costanza et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29599344' target='_blank'>Heath et al., Anticancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27765909' target='_blank'>Heavey et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27149220' target='_blank'>Bivona et al., Nat Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26583055' target='_blank'>dos Santos et al., 2015, Oxid Med Cell Longev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25273224' target='_blank'>Villaruz et al., 2014, Cancer<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40678060' target='_blank'>Wen et al., 2025, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31452773' target='_blank'>Zhang et al., 2019, Oncol Lett<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29207054' target='_blank'>Huang et al., 2017, Int J Mol Med<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38551131' target='_blank'>Takeiwa et al., 2024, RNA Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36228719' target='_blank'>Alwanian et al., J Biol Chem, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32801887' target='_blank'>Li et al., Cancer Manag Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32570949' target='_blank'>Jen et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29879184' target='_blank'>Qiu et al., PLoS One, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29167821' target='_blank'>Ito et al., 2017, NPJ Breast Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29142193' target='_blank'>Wozniak et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27830019' target='_blank'>Chen et al., Am J Transl Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25999240' target='_blank'>Goel et al., Biochim Biophys Acta, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24167725' target='_blank'>di et al., F1000Prime Rep, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20832360' target='_blank'>Ostrander et al., Curr Opin Pharmacol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16508639' target='_blank'>Heinzelmann-Schwarz et al., Br J Cancer, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39696530' target='_blank'>Pan et al., 2024, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38504984' target='_blank'>Chen et al., 2024, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37682219' target='_blank'>Sodir et al., 2023, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34739319' target='_blank'>Mirzaei et al., Sci Adv, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32848131' target='_blank'>Xiang et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29505847' target='_blank'>Liu et al., J Hepatol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29066500' target='_blank'>Zhang et al., Genes Dev, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28087641' target='_blank'>Bollu et al., Clin Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27974211' target='_blank'>Luo et al., Cell Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27803106' target='_blank'>Bai et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27275015' target='_blank'>Maeshima et al., JCI Insight, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25482303' target='_blank'>Alshaker et al., Breast Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25263441' target='_blank'>Zhao et al., Oncogene, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21575863' target='_blank'>Bard-Chapeau et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19936768' target='_blank'>Navis et al., Acta Neuropathol, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37830586' target='_blank'>Kichina et al., 2023, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36536137' target='_blank'>Liu et al., 2022, Nat Metab<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35517498' target='_blank'>Lillo et al., 2022, Front Cell Dev Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35490208' target='_blank'>Kim et al., Oncogene, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33298895' target='_blank'>Butera et al., Cell Death Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33106353' target='_blank'>Bayo et al., 2020, Gut<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32322580' target='_blank'>Maldonado et al., Front Cell Dev Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32193458' target='_blank'>Li et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31257073' target='_blank'>Lionarons et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31027363' target='_blank'>De et al., Cells, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30971746' target='_blank'>Molinie et al., Cell Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26619011' target='_blank'>Chang et al., Nat Biotechnol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26176707' target='_blank'>Vu et al., Pigment Cell Melanoma Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23728337' target='_blank'>Feng et al., Oncogene, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17167184' target='_blank'>Skotheim et al., Cell Oncol, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38378967' target='_blank'>Xu et al., 2024, Commun Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36598417' target='_blank'>Cortes-Ciriano et al., 2023, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34191246' target='_blank'>Yang et al., Biochem Genet, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28206970' target='_blank'>Sookoian et al., Oncotarget, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39155279' target='_blank'>Chen et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34174931' target='_blank'>Fujii et al., J Exp Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31801946' target='_blank'>Tan et al., 2019, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31612616' target='_blank'>Ghousein et al., Liver Int, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30639242' target='_blank'>Kovalski et al., Mol Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26379118' target='_blank'>Takeuchi et al., Int J Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24874729' target='_blank'>Galluzzi et al., Cell Death Dis, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24061634' target='_blank'>Hughes et al., Prostate Cancer Prostatic Dis, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22617245' target='_blank'>Yasuda et al., J Thorac Oncol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22043453' target='_blank'>Karreth et al., Cancer Discov, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21744342' target='_blank'>He et al., J Pathol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12414660' target='_blank'>Fedorov et al., Cancer Res, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2921777' target='_blank'>Kovacs et al., J Natl Cancer Inst, 1989<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2889852' target='_blank'>Chenevix-Trench et al., Leukemia, 1987<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2566145' target='_blank'>Trench et al., Oncogene, 1989<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36031407' target='_blank'>Zhang et al., Adv Sci (Weinh), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33602907' target='_blank'>Keinan et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29170536' target='_blank'>Anastasiadou et al., Nat Rev Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28219393' target='_blank'>Hackl et al., J Hematol Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28025581' target='_blank'>Ronchini et al., Leukemia, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27264179' target='_blank'>Gayvert et al., Cell Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26077418' target='_blank'>Unknown et al., J Extracell Vesicles, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24637324' target='_blank'>Sahin et al., J Cell Biol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21881606' target='_blank'>Richly et al., Cell Death Dis, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21331069' target='_blank'>Occhionorelli et al., 2011, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20815877' target='_blank'>Chhabra et al., Mol Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20453882' target='_blank'>Lu et al., 2010, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18026104' target='_blank'>Khetchoumian et al., 2007, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18025157' target='_blank'>van et al., Blood, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16473276' target='_blank'>Sternsdorf et al., Cancer Cell, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40654728' target='_blank'>Palla et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40545232' target='_blank'>Palla et al., 2025, J Mol Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40258564' target='_blank'>Fournier et al., 2025, Exp Hematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34002060' target='_blank'>Zhao et al., 2021, Nat Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33436536' target='_blank'>Peng et al., Cell Death Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33061428' target='_blank'>Ren et al., Onco Targets Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32257110' target='_blank'>Nandy et al., Cell Biosci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28714030' target='_blank'>Feng et al., Mol Med Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28055972' target='_blank'>Yu et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27427228' target='_blank'>Horton et al., Cell Chem Biol, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/31277480' target='_blank'>Kober-Hasslacher et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31097630' target='_blank'>Meijers et al., Haematologica, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30926791' target='_blank'>Weber et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30186235' target='_blank'>Giuliani et al., Front Endocrinol (Lausanne), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26239915' target='_blank'>Sinkovics et al., Int J Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25183205' target='_blank'>Rovida et al., Gut, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25180063' target='_blank'>Heise et al., J Exp Med, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23574119' target='_blank'>Dunleavy et al., N Engl J Med, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20575024' target='_blank'>Deambrogi et al., Am J Hematol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19503088' target='_blank'>Gregersen et al., Nat Genet, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11921291' target='_blank'>Kalaitzidis et al., Genes Chromosomes Cancer, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11748286' target='_blank'>Davis et al., J Exp Med, 2001<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9918209' target='_blank'>Wang et al., Clin Cancer Res, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8547649' target='_blank'>Houldsworth et al., Blood, 1996<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2103511' target='_blank'>Enrietto et al., Semin Cancer Biol, 1990<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34002056' target='_blank'>Cohen et al., Nat Rev Drug Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32341551' target='_blank'>Castel et al., Nat Rev Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27910877' target='_blank'>Santos et al., Nat Rev Drug Discov, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27626519' target='_blank'>Fagin et al., N Engl J Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27149220' target='_blank'>Bivona et al., Nat Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27056998' target='_blank'>Drilon et al., Ann Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26301689' target='_blank'>Hrustanovic et al., Nat Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14561794' target='_blank'>Machens et al., N Engl J Med, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12000816' target='_blank'>Neumann et al., N Engl J Med, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9739433' target='_blank'>Giuffrida et al., Ann Oncol, 1998<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8782503' target='_blank'>Eng et al., N Engl J Med, 1996<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8099202' target='_blank'>Mulligan et al., Nature, 1993<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7915822' target='_blank'>Lips et al., N Engl J Med, 1994<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7906866' target='_blank'>Hofstra et al., Nature, 1994<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2406025' target='_blank'>Grieco et al., Cell, 1990<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33772100' target='_blank'>Megremis et al., Commun Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33448640' target='_blank'>Ji et al., Mol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33264103' target='_blank'>Liu et al., Aging (Albany NY), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31921311' target='_blank'>Gao et al., Front Genet, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31719796' target='_blank'>Ma et al., Cancer Cell Int, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31423302' target='_blank'>Chen et al., Biomed Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30532557' target='_blank'>Chang et al., Onco Targets Ther, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30143645' target='_blank'>Zhang et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29767249' target='_blank'>Zhang et al., Int J Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26568293' target='_blank'>Ma et al., Lab Invest, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20422010' target='_blank'>Ohshima et al., PLoS One, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18836477' target='_blank'>Ozato et al., Nat Rev Immunol, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12787916' target='_blank'>Saenko et al., Mutat Res, 2003<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39605726' target='_blank'>Rominger et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39161959' target='_blank'>Riley et al., 2024, iScience<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38077017' target='_blank'>Riley et al., 2023, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34373451' target='_blank'>Vichas et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31247273' target='_blank'>Song et al., Cancer Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30348939' target='_blank'>Feng et al., Cell Death Dis, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28607354' target='_blank'>Mir et al., Sci Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28292439' target='_blank'>Cherniack et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24469055' target='_blank'>Berger et al., Oncogene, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12866958' target='_blank'>Li et al., Ai Zheng, 2003<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40140911' target='_blank'>Su et al., 2025, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33646820' target='_blank'>Dziadziuszko et al., J Clin Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32927123' target='_blank'>Dimou et al., J Thorac Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31118036' target='_blank'>Wu et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30510251' target='_blank'>Menotti et al., Nat Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29935306' target='_blank'>Davies et al., J Thorac Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29458371' target='_blank'>Kleczko et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29455670' target='_blank'>Cerrato et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28994423' target='_blank'>Rizvi et al., Nat Rev Clin Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28088512' target='_blank'>Lin et al., J Thorac Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27211795' target='_blank'>Leventakos et al., J Thorac Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26899759' target='_blank'>Jänne et al., J Thorac Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25667280' target='_blank'>Mazières et al., J Clin Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25384172' target='_blank'>Wijesinghe et al., J Thorac Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23242437' target='_blank'>Kobayashi et al., J Thorac Oncol, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/31819506' target='_blank'>Zhao et al., 2019, Onco Targets Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31761784' target='_blank'>Liu et al., Aging (Albany NY), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27594411' target='_blank'>Wan et al., Int J Biochem Cell Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27456846' target='_blank'>Fontelles et al., Breast Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27450453' target='_blank'>Manole et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26894955' target='_blank'>Sharma et al., Elife, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21631264' target='_blank'>Hong et al., 2011, Arch Pathol Lab Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20610622' target='_blank'>Finegan et al., Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15184866' target='_blank'>Wang et al., Oncogene, 2004<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36358647' target='_blank'>Kashani et al., Cancers (Basel), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34503164' target='_blank'>Crudele et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33327316' target='_blank'>Gao et al., Medicine (Baltimore), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32731550' target='_blank'>Cristóbal et al., J Clin Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31495599' target='_blank'>Wang et al., 2019, Dig Liver Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30871013' target='_blank'>Cristóbal et al., J Clin Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21990161' target='_blank'>Lam et al., Cytometry A, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21725597' target='_blank'>Jiang et al., Mol Med Rep, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7753797' target='_blank'>Nagata et al., Proc Natl Acad Sci U S A, 1995<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39218979' target='_blank'>Kim et al., 2024, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31775037' target='_blank'>Park et al., Cell Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26832847' target='_blank'>Barbui et al., Blood Cancer J, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25932444' target='_blank'>Kim et al., Ann Lab Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25717103' target='_blank'>McFarlane et al., J Virol, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40702000' target='_blank'>Wang et al., 2025, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40568073' target='_blank'>Yu et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36835286' target='_blank'>Liu et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36417849' target='_blank'>Urbanski et al., Cell Rep, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36282044' target='_blank'>Che et al., Mol Carcinog, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34065983' target='_blank'>Blijlevens et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33142236' target='_blank'>Chen et al., Neoplasia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32308565' target='_blank'>Wang et al., Cancer Cell Int, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31462429' target='_blank'>Song et al., Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27808105' target='_blank'>Jia et al., Sci Rep, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/18924150' target='_blank'>Yoon et al., Int J Cancer, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18794340' target='_blank'>Brune et al., J Exp Med, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15695387' target='_blank'>Zhang et al., Cancer Res, 2005<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38823260' target='_blank'>Wang et al., 2024, Transl Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36899391' target='_blank'>Xu et al., 2023, Cancer Cell Int<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33858425' target='_blank'>Ji et al., Cancer Cell Int, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30453068' target='_blank'>Wu et al., Gene, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35287543' target='_blank'>Chang et al., Bioengineered, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32826305' target='_blank'>Camolotto et al., Gut, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32220051' target='_blank'>Chen et al., J Cell Mol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30962263' target='_blank'>Nie et al., Biosci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30723235' target='_blank'>De et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28573504' target='_blank'>Zhu et al., Hum Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28367243' target='_blank'>Du et al., J Cancer, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34368858' target='_blank'>Wang et al., Int J Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29854289' target='_blank'>Frech et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29662641' target='_blank'>Fouquet et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26484093' target='_blank'>Chalk et al., Genom Data, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8471834' target='_blank'>Fumagalli et al., Melanoma Res, 1993<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38816668' target='_blank'>Xue et al., 2024, Mol Biomed<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37667236' target='_blank'>He et al., 2023, BMC Oral Health<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33955576' target='_blank'>Wang et al., Prostate, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33771969' target='_blank'>Xia et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33446260' target='_blank'>Wu et al., Genome Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30991683' target='_blank'>Peer et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30664638' target='_blank'>Wong et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29435119' target='_blank'>Parascandolo et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27519263' target='_blank'>Jiang et al., World J Surg Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26840018' target='_blank'>Sun et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25596744' target='_blank'>Chow et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25173582' target='_blank'>Du et al., Neuro Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23079662' target='_blank'>Hatley et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16339184' target='_blank'>Sicklick et al., Carcinogenesis, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12241103' target='_blank'>Katayam et al., J Neurooncol, 2002<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37062002' target='_blank'>Sivakumar et al., 2023, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32728033' target='_blank'>Zhang et al., Signal Transduct Target Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32531271' target='_blank'>Coles et al., Cancer Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31722709' target='_blank'>Jia et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30053878' target='_blank'>Zhou et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27728803' target='_blank'>Ferone et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25535693' target='_blank'>Morgensztern et al., J Thorac Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24525231' target='_blank'>Justilien et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23260487' target='_blank'>Marqués-Torrejón et al., Cell Stem Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23217424' target='_blank'>Bazzoli et al., Cell Stem Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22941189' target='_blank'>Rudin et al., Nat Genet, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21982232' target='_blank'>Arnold et al., Cell Stem Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21940738' target='_blank'>Hägerstrand et al., Neuro Oncol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21816362' target='_blank'>Barrilleaux et al., Cell Stem Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19801978' target='_blank'>Bass et al., Nat Genet, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39928721' target='_blank'>Charette et al., 2025, Chem Rev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39198451' target='_blank'>Zhao et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35590073' target='_blank'>Mund et al., Nat Biotechnol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35145327' target='_blank'>Gavert et al., Nat Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34232320' target='_blank'>Alhalabi et al., Neuro Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23993096' target='_blank'>Zhang et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21421858' target='_blank'>Chakravarty et al., Clin Cancer Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21106725' target='_blank'>Sen et al., Clin Cancer Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20087228' target='_blank'>Miller et al., J Thorac Oncol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19012396' target='_blank'>Liu et al., J Am Chem Soc, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18594017' target='_blank'>Nozawa et al., Clin Cancer Res, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12941830' target='_blank'>Myoui et al., Cancer Res, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7509341' target='_blank'>Cartwright et al., J Clin Invest, 1994<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1698545' target='_blank'>Bjelfman et al., Cancer Res, 1990<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/1380891' target='_blank'>Ottenhoff-Kalff et al., Cancer Res, 1992<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/29755264' target='_blank'>Roberts et al., Saudi J Ophthalmol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28191313' target='_blank'>de et al., Biomark Res, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33190044' target='_blank'>Li et al., Transl Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29789460' target='_blank'>Zeng et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29760724' target='_blank'>Samoilova et al., Stem Cells Int, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29755264' target='_blank'>Roberts et al., Saudi J Ophthalmol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28191313' target='_blank'>de et al., Biomark Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26947017' target='_blank'>Jones et al., Oncogene, 2016<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39169031' target='_blank'>Lv et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38498036' target='_blank'>Klein et al., 2024, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27881582' target='_blank'>Cardenas et al., Clin Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27110918' target='_blank'>Jahani-Asl et al., Nat Neurosci, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25921060' target='_blank'>Wu et al., Blood, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25879771' target='_blank'>Yin et al., J Hematol Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25734337' target='_blank'>Grabner et al., Nat Commun, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25538261' target='_blank'>Feber et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25047660' target='_blank'>Qian et al., Mol Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23741012' target='_blank'>Mangolini et al., Blood, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21997559' target='_blank'>Baumgart et al., Gut, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21385929' target='_blank'>Yang et al., Clin Cancer Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21131551' target='_blank'>Rajendran et al., 2010, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20712901' target='_blank'>Lin et al., Mol Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18167335' target='_blank'>Grivennikov et al., Cancer Cell, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/23250732' target='_blank'>Televantou et al., Pathol Oncol Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23149921' target='_blank'>Tam et al., Cancer Res, 2013<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40510356' target='_blank'>Hu et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34359636' target='_blank'>Linn et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33670716' target='_blank'>Buffard et al., Biomolecules, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33430292' target='_blank'>K et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33159047' target='_blank'>Polak et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32366932' target='_blank'>Ni et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32015510' target='_blank'>Yang et al., Br J Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31992353' target='_blank'>Elgamal et al., J Hematol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31043589' target='_blank'>Sun et al., Cell Death Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28202529' target='_blank'>Cui et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25921550' target='_blank'>Peng et al., Cell Oncol (Dordr), 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25639868' target='_blank'>Chen et al., Oncogene, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25329694' target='_blank'>Yaktapour et al., J Clin Invest, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22237022' target='_blank'>Zhang et al., Nature, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19293188' target='_blank'>Bailet et al., Cancer Res, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39753773' target='_blank'>Ahlmann-Eltze et al., 2025, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39218931' target='_blank'>Gu et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33434514' target='_blank'>Zhou et al., 2021, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32086528' target='_blank'>Li et al., 2020, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31362752' target='_blank'>Sadowski et al., Genome Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31311566' target='_blank'>Jia et al., 2019, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30951661' target='_blank'>Brand et al., 2019, Cell Stem Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28270163' target='_blank'>Wallaert et al., 2017, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28115373' target='_blank'>Girardi et al., Blood, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22897851' target='_blank'>Sanda et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21670468' target='_blank'>Tatarek et al., Blood, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21536859' target='_blank'>Thoms et al., 2011, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19171877' target='_blank'>Landry et al., Blood, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16621969' target='_blank'>Palomero et al., 2006, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15728238' target='_blank'>Mandal et al., J Exp Med, 2005<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/27876887' target='_blank'>Aravindan et al., 2016, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24816818' target='_blank'>Kobayashi et al., 2014, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22784109' target='_blank'>Ochocinska et al., J Neurochem, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40717225' target='_blank'>Serala et al., 2025, J Cell Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39972122' target='_blank'>Shu et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39632282' target='_blank'>Serala et al., 2024, J Cell Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39403898' target='_blank'>Bleloch et al., 2024, Cancer Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38965548' target='_blank'>Li et al., 2024, Cell Commun Signal<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38254762' target='_blank'>Lesovaya et al., 2024, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33577921' target='_blank'>Liang et al., J Hepatol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32098189' target='_blank'>Sims et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30979887' target='_blank'>Oh et al., Oncogenesis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29295731' target='_blank'>Feng et al., Oncol Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27265867' target='_blank'>Huber et al., Dev Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26900951' target='_blank'>Willmer et al., Oncogenesis, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26579496' target='_blank'>Fischer et al., Front Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22535523' target='_blank'>Ballim et al., Mol Biol Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19723660' target='_blank'>Jin et al., Cancer Res, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40473600' target='_blank'>Zhou et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39753773' target='_blank'>Ahlmann-Eltze et al., 2025, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38131215' target='_blank'>Zhang et al., 2023, Oncol Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33527899' target='_blank'>Takao et al., Elife, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28261581' target='_blank'>Grebbin et al., Front Cell Dev Biol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24682819' target='_blank'>Denis et al., Nucleic Acids Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24463278' target='_blank'>Ueno et al., Exp Hematol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23893300' target='_blank'>Okuyama et al., Proc Natl Acad Sci U S A, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21548940' target='_blank'>Ramberg et al., 2011, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21159644' target='_blank'>Chang et al., Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18974129' target='_blank'>Park et al., Cancer Res, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10913162' target='_blank'>Bayly et al., Mol Cell Biol, 2000<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10637514' target='_blank'>Calvo et al., 2000, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9401000' target='_blank'>Feldman et al., Oncogene, 1997<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/7936638' target='_blank'>Kamps et al., Oncogene, 1994<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34753394' target='_blank'>Xue et al., Bioengineered, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32243714' target='_blank'>Jia et al., J Cell Mol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25520196' target='_blank'>Huang et al., Nucleic Acids Res, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38715348' target='_blank'>Tang et al., 2024, Cancer Commun (Lond)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37633911' target='_blank'>Lu et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37331567' target='_blank'>Wang et al., 2023, Cell Mol Gastroenterol Hepatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35915156' target='_blank'>Tcheandjieu et al., Nat Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34965379' target='_blank'>Rodriguez et al., 2021, Mol Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33897890' target='_blank'>Zhou et al., Theranostics, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33259884' target='_blank'>Zhang et al., Pharmacol Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31263103' target='_blank'>Chen et al., Cell Death Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31078320' target='_blank'>Zhuang et al., 2019, Trends Pharmacol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30679322' target='_blank'>Goossens et al., 2019, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29930325' target='_blank'>Liu et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27929537' target='_blank'>Wang et al., 2016, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27456471' target='_blank'>Zhang et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24950909' target='_blank'>Ai et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20452118' target='_blank'>Takeyama et al., Cancer Lett, 2010<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40359938' target='_blank'>Pepe et al., 2025, Am J Hum Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38885349' target='_blank'>Shanmugam et al., 2024, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38342602' target='_blank'>Liu et al., 2024, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38183840' target='_blank'>Xiong et al., 2024, EBioMedicine<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37261974' target='_blank'>Zhu et al., 2023, Elife<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33239622' target='_blank'>Li et al., 2020, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32928281' target='_blank'>Zhao et al., J Exp Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32796005' target='_blank'>Lopez et al., Genome Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31900258' target='_blank'>Huang et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31108233' target='_blank'>Wrighton et al., 2019, Cell Mol Gastroenterol Hepatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30404026' target='_blank'>Dhanasekaran et al., Gastroenterology, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29084575' target='_blank'>Zhuang et al., J Exp Clin Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24174164' target='_blank'>Remke et al., Acta Neuropathol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23860532' target='_blank'>Zebary et al., Br J Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20487549' target='_blank'>Zhang et al., J Exp Clin Cancer Res, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37577497' target='_blank'>Achom et al., 2023, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33987007' target='_blank'>Zhang et al., PeerJ, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32882363' target='_blank'>Neha et al., Biochim Biophys Acta Gen Subj, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31722230' target='_blank'>Burns et al., Semin Cancer Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25268941' target='_blank'>Cykowski et al., Ultrastruct Pathol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23288701' target='_blank'>Kobos et al., J Pathol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17425505' target='_blank'>Teixeira et al., Crit Rev Oncog, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40139189' target='_blank'>Pulice et al., 2025, Mol Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36930718' target='_blank'>Eichner et al., 2023, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32826305' target='_blank'>Camolotto et al., Gut, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31278259' target='_blank'>Devarakonda et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29587142' target='_blank'>Tata et al., Dev Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26069186' target='_blank'>Skoulidis et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25060702' target='_blank'>Isogaya et al., Cell Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25060579' target='_blank'>Yui et al., Nat Rev Immunol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23591868' target='_blank'>Kim et al., Nat Commun, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23523371' target='_blank'>Snyder et al., Mol Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23143308' target='_blank'>Maeda et al., J Clin Invest, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21907921' target='_blank'>Taguchi et al., 2011, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21481790' target='_blank'>Homminga et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21471965' target='_blank'>Winslow et al., Nature, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19801978' target='_blank'>Bass et al., Nat Genet, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34188019' target='_blank'>Damerell et al., Signal Transduct Target Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33891890' target='_blank'>Suski et al., Cancer Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30554963' target='_blank'>Gupta et al., Cell Stem Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29907126' target='_blank'>Balaji et al., J Hematol Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29764424' target='_blank'>Liu et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29363536' target='_blank'>Lee et al., Gut, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26872698' target='_blank'>Pfirschke et al., Immunity, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26206665' target='_blank'>Wang et al., Gut, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25079552' target='_blank'>Cancer et al., Nature, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21325450' target='_blank'>Knappskog et al., Ann Oncol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20651736' target='_blank'>Trepel et al., Nat Rev Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19087308' target='_blank'>Monticone et al., Mol Cancer, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16391298' target='_blank'>Goodell et al., J Clin Oncol, 2006<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12242155' target='_blank'>Verschuren et al., Cancer Cell, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2554494' target='_blank'>Takahashi et al., Science, 1989<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34177816' target='_blank'>McFadden et al., Front Endocrinol (Lausanne), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32698392' target='_blank'>Chu et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31497532' target='_blank'>Li et al., Front Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23429735' target='_blank'>Xing et al., Nat Rev Cancer, 2013<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37692474' target='_blank'>Zhang et al., 2023, Genes Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36797244' target='_blank'>Zhao et al., Signal Transduct Target Ther, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34661759' target='_blank'>Zhou et al., J Cancer Res Clin Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34220955' target='_blank'>Punnia-Moorthy et al., Front Genet, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33003334' target='_blank'>Das et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31001470' target='_blank'>Tottone et al., Front Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30646895' target='_blank'>Xia et al., 2019, BMC Med Genomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29928692' target='_blank'>Kaneko et al., Sci Adv, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29034206' target='_blank'>Tan et al., Front Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28115373' target='_blank'>Girardi et al., Blood, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27798847' target='_blank'>Hu et al., Genes Dev, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27294188' target='_blank'>Ran et al., ScientificWorldJournal, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26944678' target='_blank'>Benyoucef et al., Genes Dev, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33036298' target='_blank'>Izdebska et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32725768' target='_blank'>Liu et al., Cancer Med, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40744921' target='_blank'>Galassi et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39645166' target='_blank'>Ghiani et al., 2024, Tumour Virus Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39251788' target='_blank'>Parolia et al., 2024, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36630129' target='_blank'>Lempiäinen et al., 2023, Biochem J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34671018' target='_blank'>Zhao et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33602783' target='_blank'>Chong et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33552680' target='_blank'>Xue et al., Mol Ther Nucleic Acids, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32826945' target='_blank'>Park et al., Commun Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30683853' target='_blank'>He et al., Cell Death Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30518758' target='_blank'>Aytes et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27872090' target='_blank'>Ding et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27109101' target='_blank'>Shah et al., Oncogene, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26169266' target='_blank'>Yang et al., Genome Biol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25280969' target='_blank'>Saloura et al., Mol Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23159737' target='_blank'>Asangani et al., 2012, Mol Cell<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40493404' target='_blank'>Ediga et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39592856' target='_blank'>Cortez Cardoso Penha et al., 2024, Mol Syst Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39412947' target='_blank'>Kawai-Kawachi et al., 2024, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37580757' target='_blank'>Yang et al., 2023, J Biomed Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37280679' target='_blank'>Zhu et al., 2023, Biomark Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32414754' target='_blank'>Rockne et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30982764' target='_blank'>Zhou et al., EBioMedicine, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30045840' target='_blank'>Akahori et al., Blood, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29455672' target='_blank'>Massimino et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28434147' target='_blank'>Li et al., Protein Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27821145' target='_blank'>Chen et al., J Exp Clin Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25743273' target='_blank'>Kang et al., Mol Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23945386' target='_blank'>Wu et al., J Thorac Oncol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23486779' target='_blank'>Dao et al., Sci Transl Med, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11300470' target='_blank'>Gaiger et al., Clin Cancer Res, 2001<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36622962' target='_blank'>Oliveira et al., Genet Mol Biol, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32565964' target='_blank'>Fan et al., Oncol Lett, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31681559' target='_blank'>Gimple et al., Front Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29434842' target='_blank'>Zhang et al., Oncol Lett, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24431073' target='_blank'>Cheng et al., Mol Cancer Ther, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37862360' target='_blank'>Tang et al., 2023, PLoS Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37286599' target='_blank'>Griffin et al., 2023, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32117744' target='_blank'>Nehrbas et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31815619' target='_blank'>Zhang et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31494331' target='_blank'>Si et al., Int J Surg, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31419987' target='_blank'>Liu et al., J Exp Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29702599' target='_blank'>Chan et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29556394' target='_blank'>Gan et al., Biomark Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28703799' target='_blank'>Liu et al., Cell Death Dis, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28685624' target='_blank'>Ma et al., Expert Rev Neurother, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28561778' target='_blank'>Yan et al., Int J Mol Sci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26150700' target='_blank'>Zhang et al., Drug Des Devel Ther, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25834393' target='_blank'>Zhou et al., Drug Des Devel Ther, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25249257' target='_blank'>Tang et al., Ann Surg Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15184868' target='_blank'>Castellone et al., Oncogene, 2004<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38443798' target='_blank'>Qin et al., 2024, Cell Mol Biol Lett<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36753024' target='_blank'>Larkin et al., 2023, Curr Oncol Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34065619' target='_blank'>Oshi et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33923958' target='_blank'>Daniel et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29287303' target='_blank'>Mor-Vaknin et al., Arthritis Rheumatol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29269732' target='_blank'>Cifdaloz et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27811057' target='_blank'>Delaunay et al., J Exp Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26125442' target='_blank'>Mitchell et al., Br J Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25278611' target='_blank'>Theurillat et al., Science, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25216995' target='_blank'>Sandén et al., Mol Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24433356' target='_blank'>Thériault et al., Clin Exp Ophthalmol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21317931' target='_blank'>Privette et al., Oncogene, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19223548' target='_blank'>Wise-Draper et al., Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15876350' target='_blank'>Wu et al., Mol Cancer, 2005<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10837023' target='_blank'>Alexiadis et al., Genes Dev, 2000<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37728021' target='_blank'>Takla et al., 2023, EMBO Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37160873' target='_blank'>Ariano et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35781277' target='_blank'>Astanina et al., FEBS Lett, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35545719' target='_blank'>Ariano et al., Angiogenesis, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35149590' target='_blank'>Fernandez et al., Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34414314' target='_blank'>Yamamoto et al., JMA J, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33860275' target='_blank'>Yun et al., Blood Cancer Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32753474' target='_blank'>Li et al., Mol Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31600176' target='_blank'>Gupta et al., Am J Surg Pathol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17425505' target='_blank'>Teixeira et al., Crit Rev Oncog, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39761381' target='_blank'>Zheng et al., 2025, J Med Chem<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39403517' target='_blank'>Liu et al., 2024, Heliyon<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38758792' target='_blank'>Liu et al., 2024, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36754959' target='_blank'>Becht et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35332266' target='_blank'>Zhao et al., Br J Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34459452' target='_blank'>Xu et al., Anticancer Drugs, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33995658' target='_blank'>Liu et al., Theranostics, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32884529' target='_blank'>Imataki et al., Case Rep Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32162442' target='_blank'>Jiang et al., Cancer Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29021135' target='_blank'>Lv et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27893709' target='_blank'>Yu et al., Oncogene, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27308345' target='_blank'>Ross et al., Mol Cell Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25220592' target='_blank'>Turner-Ivey et al., Neoplasia, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33106376' target='_blank'>Safe et al., Mol Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32125766' target='_blank'>Wang et al., J Cell Mol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31094928' target='_blank'>Haller et al., 2019, Am J Surg Pathol<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/32765078' target='_blank'>Tian et al., Cancer Manag Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22885519' target='_blank'>Novak et al., Exp Hematol, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40231263' target='_blank'>Huang et al., 2025, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38049665' target='_blank'>Leeman-Neill et al., 2023, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36924677' target='_blank'>Zhang et al., 2023, Biochem Biophys Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34201935' target='_blank'>Yang et al., Biology (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33793001' target='_blank'>Li et al., J Clin Lab Anal, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33718010' target='_blank'>Zhang et al., Transl Lung Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33569544' target='_blank'>Parsa et al., Nat Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32695399' target='_blank'>Li et al., Biomark Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32565964' target='_blank'>Fan et al., Oncol Lett, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29748621' target='_blank'>Gates et al., Oncogene, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29532228' target='_blank'>Abudureheman et al., J Cancer Res Clin Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28653505' target='_blank'>Fujimoto et al., 2017, Int J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28187001' target='_blank'>Zhikrivetskaya et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27338637' target='_blank'>Ozawa et al., Mod Pathol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22965151' target='_blank'>Younes et al., Nat Rev Clin Oncol, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40520993' target='_blank'>Jabbari et al., 2025, Genes Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33731694' target='_blank'>Wang et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32952670' target='_blank'>Shin et al., Asian J Pharm Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32719345' target='_blank'>Xu et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32650416' target='_blank'>Rinke et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32489713' target='_blank'>Catana et al., 2020, Aging Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31031083' target='_blank'>Pandolfini et al., Mol Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29391048' target='_blank'>Ma et al., J Exp Clin Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29033353' target='_blank'>Moon et al., Cell Stem Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28500786' target='_blank'>Thanasupawat et al., Mol Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27102002' target='_blank'>Kaur et al., Cancer Lett, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24825857' target='_blank'>Xu et al., Mol Cancer Ther, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23536553' target='_blank'>Li et al., Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21147603' target='_blank'>Rager et al., Environ Health Perspect, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18505920' target='_blank'>Di et al., Mol Cancer Res, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37023372' target='_blank'>Jiang et al., 2023, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36797244' target='_blank'>Zhao et al., Signal Transduct Target Ther, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27846392' target='_blank'>Ceribelli et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27213184' target='_blank'>Balatti et al., Trends Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26473382' target='_blank'>Fromm et al., Annu Rev Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26041471' target='_blank'>Prinz et al., Mol Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25886051' target='_blank'>Balatti et al., J Hematol Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25482129' target='_blank'>Motiwala et al., Leukemia, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23591791' target='_blank'>Zanesi et al., Blood, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23160471' target='_blank'>Gaudio et al., Blood, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22460735' target='_blank'>Sivina et al., Leukemia, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20863894' target='_blank'>Pekarsky et al., Semin Cancer Biol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20357824' target='_blank'>Efanov et al., Leukemia, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19770358' target='_blank'>Herling et al., Blood, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10025912' target='_blank'>Takizawa et al., Leukemia, 1999<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38226841' target='_blank'>Yang et al., 2024, Mol Carcinog<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36117111' target='_blank'>Ren et al., Adv Sci (Weinh), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32464546' target='_blank'>Ruan et al., Mol Ther Nucleic Acids, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31168049' target='_blank'>He et al., Cell Death Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30087896' target='_blank'>Ernst et al., Front Cell Dev Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26865464' target='_blank'>Åman et al., J Pathol, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32944406' target='_blank'>Wu et al., Cancer Biol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31188495' target='_blank'>Kwan et al., Hepatology, 2020<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35194188' target='_blank'>Fan et al., Cell Death Differ, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34353330' target='_blank'>Jiang et al., Mol Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34021128' target='_blank'>Han et al., Cell Death Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33712563' target='_blank'>Calder et al., Oncogenesis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32941624' target='_blank'>Feng et al., Nucleic Acids Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32512506' target='_blank'>Patel et al., EBioMedicine, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32460168' target='_blank'>Wang et al., EBioMedicine, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31901237' target='_blank'>Song et al., Mol Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31619292' target='_blank'>Saurty-Seerunghen et al., Acta Neuropathol Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31105033' target='_blank'>Gong et al., EBioMedicine, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30237511' target='_blank'>Ju et al., Cell Death Differ, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29996942' target='_blank'>Ma et al., J Exp Clin Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28810879' target='_blank'>Wang et al., J Transl Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28414305' target='_blank'>Geng et al., Oncogene, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28032859' target='_blank'>Zhi et al., Cell Death Dis, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/30026872' target='_blank'>Li et al., Theranostics, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28401183' target='_blank'>Ikink et al., Mol Cell Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27876799' target='_blank'>Ikink et al., Nat Commun, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40253363' target='_blank'>Treis et al., 2025, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38876192' target='_blank'>Khakwani et al., 2024, J Adv Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38197367' target='_blank'>Collie et al., 2024, J Med Chem<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37945900' target='_blank'>Chapman et al., 2023, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35657598' target='_blank'>Zhang et al., Mol Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35346236' target='_blank'>Chen et al., Cell Commun Signal, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32848211' target='_blank'>Wang et al., Oncogene, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32029457' target='_blank'>Cohen et al., Genes Dev, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31723810' target='_blank'>Milsom et al., Hemasphere, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31481464' target='_blank'>Clausse et al., J Biol Chem, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31439867' target='_blank'>Fons et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30413718' target='_blank'>Liu et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26714478' target='_blank'>Wu et al., Apoptosis, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24632620' target='_blank'>Buss et al., Oncogene, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20668064' target='_blank'>Zhang et al., Cancer Res, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40623191' target='_blank'>Pecorari et al., 2025, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40449847' target='_blank'>Qiao et al., 2025, Cell Mol Gastroenterol Hepatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38315837' target='_blank'>Shin et al., 2024, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33263276' target='_blank'>Riege et al., Elife, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29725063' target='_blank'>Ji et al., Cell Death Dis, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29454790' target='_blank'>Xie et al., Gastroenterology, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28041841' target='_blank'>Saladi et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26220994' target='_blank'>Alekseyenko et al., Genes Dev, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25994056' target='_blank'>Yao et al., Genome Biol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25779941' target='_blank'>Rusan et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23867503' target='_blank'>Ramsey et al., J Clin Invest, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21917857' target='_blank'>Knouf et al., Nucleic Acids Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/12086851' target='_blank'>Patturajan et al., Cancer Cell, 2002<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11336476' target='_blank'>Senoo et al., Br J Cancer, 2001<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10637430' target='_blank'>Ikawa et al., Cell Death Differ, 1999<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39160596' target='_blank'>Yao et al., 2024, Cell Biosci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34938610' target='_blank'>Lv et al., Mol Ther Nucleic Acids, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34508101' target='_blank'>Shah et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34284744' target='_blank'>Xiao et al., BMC Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33066016' target='_blank'>Eberhardt et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32293515' target='_blank'>Huang et al., J Exp Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31239270' target='_blank'>Peluffo et al., Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30698743' target='_blank'>Liu et al., Nucleic Acids Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29129908' target='_blank'>Lv et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27888625' target='_blank'>Yin et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27238081' target='_blank'>Groner et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25700357' target='_blank'>Wang et al., Oncol Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25248926' target='_blank'>Quintás-Cardama et al., Clin Lymphoma Myeloma Leuk, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21164480' target='_blank'>Tsai et al., Nature, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19843848' target='_blank'>O'Malley et al., Cancer Res, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39067088' target='_blank'>Zhang et al., 2024, Transl Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34274028' target='_blank'>Qian et al., J Hematol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32481602' target='_blank'>Kang et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28833526' target='_blank'>Montrose et al., Environ Mol Mutagen, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28076379' target='_blank'>Duan et al., PLoS One, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26925132' target='_blank'>Hong et al., Arch Med Sci, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35926707' target='_blank'>Ogura et al., J Biol Chem, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32821278' target='_blank'>Meitern et al., Evol Appl, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26198694' target='_blank'>Yamamoto et al., BMC Res Notes, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23769422' target='_blank'>Oliveira et al., Hum Pathol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14521938' target='_blank'>Bizimungu et al., Biochem Biophys Res Commun, 2003<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/8406013' target='_blank'>Onno et al., Gene, 1993<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40523964' target='_blank'>Brunner et al., 2025, BJC Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40032281' target='_blank'>Herbstein et al., 2025, J Neuroendocrinol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32825105' target='_blank'>Shiromizu et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32273758' target='_blank'>Jing et al., Cancer Manag Res, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39605563' target='_blank'>Funk et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31828126' target='_blank'>Dong et al., Biomed Res Int, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31817861' target='_blank'>Torres-Ayuso et al., Cancers (Basel), 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39587064' target='_blank'>Pedrosa et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37723138' target='_blank'>Cai et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36806560' target='_blank'>Zeng et al., 2023, Semin Cancer Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33935281' target='_blank'>Liu et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32350244' target='_blank'>Li et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32296030' target='_blank'>Yang et al., Signal Transduct Target Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31340155' target='_blank'>Magen et al., Cell Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29017567' target='_blank'>Schagdarsurengin et al., Cell Commun Signal, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28068326' target='_blank'>Riverso et al., Oncogene, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28068319' target='_blank'>Zhou et al., Oncogene, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26511107' target='_blank'>Guo et al., J Biomed Sci, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25944097' target='_blank'>Liu et al., Mol Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25341045' target='_blank'>Huang et al., Cell Death Dis, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22677193' target='_blank'>Li et al., Gastroenterology, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21242971' target='_blank'>Yu et al., Oncogene, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34477047' target='_blank'>Zhou et al., Bioengineered, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33014878' target='_blank'>Guo et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27481518' target='_blank'>Conconi et al., Tumour Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26690729' target='_blank'>Arora et al., J Pathol, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33569406' target='_blank'>Cao et al., Ann Transl Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33262952' target='_blank'>Li et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33196842' target='_blank'>Wang et al., J Mol Cell Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31281495' target='_blank'>Zhu et al., Theranostics, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/29853799' target='_blank'>Du et al., Cancer Cell Int, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23150777' target='_blank'>Shinohara et al., Sci Rep, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35474385' target='_blank'>Zakharova et al., Nucleic Acids Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33317623' target='_blank'>Charmpi et al., Genome Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33054052' target='_blank'>James et al., Haematologica, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32575858' target='_blank'>Kansara et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32143670' target='_blank'>Li et al., Breast Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31735020' target='_blank'>Park et al., Mol Cells, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31398867' target='_blank'>Cruz-Tapias et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30503969' target='_blank'>Shao et al., Mol Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29376139' target='_blank'>Astone et al., NPJ Precis Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28379616' target='_blank'>van et al., Pigment Cell Melanoma Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25832654' target='_blank'>Namiki et al., Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25611934' target='_blank'>Thompson et al., PLoS Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24657535' target='_blank'>Yen et al., Curr Opin Genet Dev, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23969693' target='_blank'>White et al., Nat Rev Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21430779' target='_blank'>Ceol et al., Nature, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40642066' target='_blank'>Akay et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39172054' target='_blank'>Li et al., 2024, Immun Inflamm Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33494394' target='_blank'>Bloedjes et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32887258' target='_blank'>Iacobas et al., Genes (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29772764' target='_blank'>González-Herrero et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28008178' target='_blank'>Nair et al., Leukemia, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27183593' target='_blank'>Miller et al., 2016, J Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26997138' target='_blank'>Panigrahi et al., Ticks Tick Borne Dis, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34658165' target='_blank'>Chen et al., Thorac Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33921698' target='_blank'>Kwa et al., Cells, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33495420' target='_blank'>Li et al., Aging (Albany NY), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32457485' target='_blank'>Zhao et al., Cancer Gene Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32127608' target='_blank'>Lee et al., Sci Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31470874' target='_blank'>Xia et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28160548' target='_blank'>Bhardwaj et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27853641' target='_blank'>Das et al., Oncoimmunology, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26233890' target='_blank'>Guo et al., J Genet Genomics, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24335962' target='_blank'>Chin et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21242979' target='_blank'>Hollander et al., Oncogene, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14680486' target='_blank'>Chau et al., Breast Cancer Res, 2004<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40744921' target='_blank'>Galassi et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35132566' target='_blank'>Sui et al., 2022, J Mol Neurosci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31050342' target='_blank'>Shi et al., Elife, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28830458' target='_blank'>Kwon et al., Mol Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27165742' target='_blank'>Lu et al., Cancer Cell, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38044354' target='_blank'>Li et al., 2023, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32280246' target='_blank'>Gao et al., Onco Targets Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17868029' target='_blank'>Forissier et al., Biol Cell, 2007<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34254334' target='_blank'>Fettke et al., 2021, Prostate<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30720096' target='_blank'>Zhao et al., Mol Med Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26146478' target='_blank'>Bishop et al., Sarcoma, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26064935' target='_blank'>Lu et al., Biomed Res Int, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24769444' target='_blank'>Qin et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21577234' target='_blank'>Antonarakis et al., Prostate Cancer Prostatic Dis, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21343398' target='_blank'>Qin et al., Cancer Res, 2011<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37623434' target='_blank'>Wimberger et al., 2023, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34926571' target='_blank'>Tan et al., Front Mol Biosci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33802402' target='_blank'>Tsui et al., Biomedicines, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33072180' target='_blank'>Charostad et al., Infect Agent Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30022982' target='_blank'>Gehring et al., Front Immunol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29881386' target='_blank'>Staal et al., Front Immunol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29765142' target='_blank'>Hyeon et al., Mod Pathol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29587428' target='_blank'>Grondona et al., Biomedicines, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28759024' target='_blank'>Wang et al., Oncogenesis, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27538487' target='_blank'>Bonsignore et al., Leukemia, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26893369' target='_blank'>Abrams et al., Dis Model Mech, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25105596' target='_blank'>Baens et al., PLoS One, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22825253' target='_blank'>Vicente-Dueñas et al., 2012, Cell Cycle<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19772915' target='_blank'>Kingeter et al., Cell Signal, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39603207' target='_blank'>Huang et al., 2024, Transl Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37542030' target='_blank'>Wang et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36724120' target='_blank'>Ma et al., 2023, J Radiat Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36630129' target='_blank'>Lempiäinen et al., 2023, Biochem J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35566260' target='_blank'>Yun et al., Molecules, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34039960' target='_blank'>Zhong et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33754016' target='_blank'>Gao et al., Theranostics, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31955177' target='_blank'>Bai et al., Med Sci Monit, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21040551' target='_blank'>Wang et al., J Exp Clin Cancer Res, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39105355' target='_blank'>Sun et al., 2024, Cancer Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37675289' target='_blank'>Zhang et al., 2023, Comput Struct Biotechnol J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27154821' target='_blank'>Zhu et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26245978' target='_blank'>Tesina et al., Nat Commun, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19667409' target='_blank'>Karaczyn et al., Ann Clin Lab Sci, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/30013198' target='_blank'>Quek et al., 2018, Nat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29383123' target='_blank'>Liu et al., Oncotarget, 2017<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39218851' target='_blank'>Asada et al., 2024, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33830080' target='_blank'>Chen et al., JCI Insight, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33544757' target='_blank'>Lee et al., PLoS One, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31652449' target='_blank'>Zhang et al., Biosci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30755304' target='_blank'>Tasoulas et al., Trends Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25953768' target='_blank'>Zhang et al., Cancer Epidemiol Biomarkers Prev, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18486532' target='_blank'>O'Neill et al., Oral Oncol, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17425505' target='_blank'>Teixeira et al., Crit Rev Oncog, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40762380' target='_blank'>Chiu et al., 2025, Cancer Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37823551' target='_blank'>Simmler et al., 2023, Elife<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35709354' target='_blank'>Tanaka et al., Blood, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33311498' target='_blank'>Tan et al., NPJ Genom Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33040078' target='_blank'>Popli et al., Cell Death Dis, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37242535' target='_blank'>Weiskittel et al., 2023, Pharmaceuticals (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34234554' target='_blank'>Chi et al., Cancer Manag Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32284759' target='_blank'>Jiang et al., J Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30106161' target='_blank'>Lu et al., Mol Med Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29212240' target='_blank'>Yu et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24014448' target='_blank'>Gerlach et al., Wiley Interdiscip Rev Dev Biol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22974708' target='_blank'>Villanueva et al., Gastroenterology, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38584203' target='_blank'>Nie et al., 2024, Mol Biomed<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37953273' target='_blank'>Talukdar et al., 2023, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36522330' target='_blank'>Ibrahim et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35105803' target='_blank'>Liu et al., Proc Natl Acad Sci U S A, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34561271' target='_blank'>McEachron et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33850013' target='_blank'>Kfoury et al., Proc Natl Acad Sci U S A, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32978368' target='_blank'>Qin et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32243828' target='_blank'>Rosencrance et al., Mol Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32156746' target='_blank'>Tischkowitz et al., Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29649019' target='_blank'>Agaimy et al., Am J Surg Pathol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29180399' target='_blank'>Kohnken et al., Blood, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28986391' target='_blank'>Bhattacharyya et al., Genome Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27846385' target='_blank'>Kleppe et al., 2016, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25594381' target='_blank'>Dancy et al., Chem Rev, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24332044' target='_blank'>Chapuy et al., 2013, Cancer Cell<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40478202' target='_blank'>Liu et al., 2025, J Cell Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39400513' target='_blank'>Hu et al., 2024, Cancer Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33535181' target='_blank'>Zhou et al., Aging (Albany NY), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29667751' target='_blank'>Wu et al., J Cell Mol Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24633887' target='_blank'>Liu et al., Tumour Biol, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35149839' target='_blank'>Hong et al., Oncogene, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34487971' target='_blank'>Kirchner et al., ESMO Open, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32549787' target='_blank'>Zhou et al., Cancer Cell Int, 2020<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38534381' target='_blank'>Di Gregorio et al., 2024, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37366601' target='_blank'>Benhammou et al., 2023, Am J Physiol Gastrointest Liver Physiol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37150829' target='_blank'>Ning et al., 2023, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36313633' target='_blank'>Yee et al., 2022, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34416231' target='_blank'>He et al., J Biol Chem, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33414893' target='_blank'>Chen et al., Oxid Med Cell Longev, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33061801' target='_blank'>Hu et al., Int J Biol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24951773' target='_blank'>Noguchi et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22470139' target='_blank'>Pan et al., Oncol Rep, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40482584' target='_blank'>Jabban et al., 2025, Leuk Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33680913' target='_blank'>Huang et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28881700' target='_blank'>Coccaro et al., 2017, Oncotarget<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28571042' target='_blank'>Chen et al., Br J Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28447248' target='_blank'>Makishima et al., Int J Hematol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26832847' target='_blank'>Barbui et al., Blood Cancer J, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26205084' target='_blank'>Vishwakarma et al., Leukemia, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25162887' target='_blank'>Spivak et al., N Engl J Med, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39851497' target='_blank'>Awan et al., 2025, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37296477' target='_blank'>Singh et al., 2023, BMC Med Genomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34988168' target='_blank'>Yang et al., Ann Transl Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30867782' target='_blank'>Wu et al., Oncol Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28874807' target='_blank'>Zhou et al., Sci Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26188282' target='_blank'>Zhang et al., Cancer Lett, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26097561' target='_blank'>Zhu et al., Int J Clin Exp Pathol, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38296623' target='_blank'>Zhang et al., 2024, Zhongguo Fei Ai Za Zhi<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37905668' target='_blank'>Wright et al., 2023, Biochem J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33918653' target='_blank'>Manna et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33305480' target='_blank'>Diao et al., Mol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32258418' target='_blank'>Chidambaranathan-Reghupaty et al., Hepatoma Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30619748' target='_blank'>Ochoa et al., Front Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24316982' target='_blank'>Bard-Chapeau et al., Nat Genet, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40672953' target='_blank'>Wen et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40169859' target='_blank'>Hong et al., 2025, Cancer Gene Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33898430' target='_blank'>Jing et al., Front Cell Dev Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32767996' target='_blank'>Wang et al., Genome Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31815635' target='_blank'>Kim et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30579865' target='_blank'>De et al., EBioMedicine, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30134017' target='_blank'>Sun et al., J Cell Mol Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29587177' target='_blank'>O'Donnell et al., Curr Opin Genet Dev, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29499945' target='_blank'>He et al., Mol Ther Nucleic Acids, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27909217' target='_blank'>van et al., Haematologica, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25650440' target='_blank'>Walker et al., Sci Signal, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25485836' target='_blank'>Rad et al., Nat Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22224767' target='_blank'>Shaffer et al., Annu Rev Immunol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20598588' target='_blank'>Rao et al., Immunity, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18765795' target='_blank'>Lenz et al., Proc Natl Acad Sci U S A, 2008<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39838044' target='_blank'>Lauwereins et al., 2025, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36947815' target='_blank'>Botten et al., 2023, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34501239' target='_blank'>Iacobucci et al., J Clin Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33946897' target='_blank'>Inaba et al., 2021, J Clin Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33728716' target='_blank'>Okabe et al., Cancer Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33081391' target='_blank'>Bayón-Calderón et al., 2020, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32318567' target='_blank'>Lee et al., Front Cell Dev Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29735923' target='_blank'>Sakata et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29296717' target='_blank'>Renou et al., 2018, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25565005' target='_blank'>Goossens et al., Nat Commun, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22366949' target='_blank'>Della et al., Nat Med, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21892159' target='_blank'>Zenatti et al., Nat Genet, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19835636' target='_blank'>Nagel et al., 2009, BMC Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17555447' target='_blank'>Dixon et al., Br J Haematol, 2007<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40164686' target='_blank'>Jiang et al., 2025, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37480049' target='_blank'>Wang et al., 2023, Biomater Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37022618' target='_blank'>Haist et al., 2023, Cancer Metastasis Rev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35514954' target='_blank'>Parriott et al., Front Immunol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34096776' target='_blank'>Qin et al., Mol Cell Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33972167' target='_blank'>Olatunde et al., Trends Immunol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32308428' target='_blank'>Sun et al., Onco Targets Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32046053' target='_blank'>Chiarini et al., 2020, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31386568' target='_blank'>Yang et al., 2019, DNA Cell Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31253868' target='_blank'>Mishra et al., Oncogene, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24488803' target='_blank'>Karlsson et al., Genes Chromosomes Cancer, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39857943' target='_blank'>Odhiambo et al., 2025, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39833846' target='_blank'>Liu et al., 2025, J Nanobiotechnology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38755129' target='_blank'>Du et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36224534' target='_blank'>Wang et al., Cell Mol Biol Lett, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33811132' target='_blank'>Gu et al., J Investig Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33122718' target='_blank'>Dhingani et al., Sci Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32934672' target='_blank'>Ding et al., Exp Ther Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29907126' target='_blank'>Balaji et al., J Hematol Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29296225' target='_blank'>Xie et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27446260' target='_blank'>Li et al., Exp Ther Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27124677' target='_blank'>Yang et al., Biol Chem, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39876014' target='_blank'>Liu et al., 2025, Stem Cell Res Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33585259' target='_blank'>Gong et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33349624' target='_blank'>Papizan et al., J Biomed Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33344235' target='_blank'>Gong et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32625084' target='_blank'>Shi et al., Front Pharmacol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31534547' target='_blank'>Zhang et al., 2019, Theranostics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31114347' target='_blank'>Zhu et al., Cancer Manag Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30926791' target='_blank'>Weber et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30776747' target='_blank'>Georgiadis et al., Environ Int, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30127402' target='_blank'>Lazarus et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28692033' target='_blank'>Ramírez-Komo et al., Oncogenesis, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28225775' target='_blank'>Zhou et al., PLoS One, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20575024' target='_blank'>Deambrogi et al., Am J Hematol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20433734' target='_blank'>Seetharam et al., BMC Genomics, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16704730' target='_blank'>Liu et al., Mol Cancer, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39445003' target='_blank'>Chen et al., 2024, MedComm (2020)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35284536' target='_blank'>Shi et al., Ann Transl Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34389030' target='_blank'>Long et al., Infect Agent Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34117091' target='_blank'>Tabatabaeian et al., Life Sci Alliance, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25823356' target='_blank'>Guo et al., Pharm Res, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38779359' target='_blank'>Li et al., 2024, Precis Clin Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38256018' target='_blank'>Nuñez et al., 2024, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34357103' target='_blank'>Kim et al., J Pers Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34354223' target='_blank'>Gómez-López et al., Commun Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33579784' target='_blank'>Unknown et al., Cancer Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31638140' target='_blank'>Gonzalez-Pecchi et al., J Mol Cell Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29884408' target='_blank'>Saloura et al., Oral Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29669750' target='_blank'>Shergalis et al., Pharmacol Rev, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28901481' target='_blank'>Liu et al., Oncol Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28484924' target='_blank'>Turner-Ivey et al., Breast Cancer Res Treat, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28102297' target='_blank'>Saloura et al., Sci Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27005559' target='_blank'>Irish et al., Mol Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25318671' target='_blank'>Jacques-Fricke et al., Mol Biol Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24875858' target='_blank'>French et al., Cancer Discov, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20940404' target='_blank'>Yang et al., Cancer Res, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40249447' target='_blank'>Wang et al., 2025, Discov Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37347559' target='_blank'>Gritsina et al., 2023, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37302847' target='_blank'>Kudo et al., 2023, J Vet Med Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30047547' target='_blank'>Yang et al., Med Sci Monit, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40550933' target='_blank'>Pan et al., 2025, Commun Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36834820' target='_blank'>Ghatak et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31227497' target='_blank'>Anbunathan et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30094568' target='_blank'>Slater et al., Cancer Metastasis Rev, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39962057' target='_blank'>Shi et al., 2025, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38173713' target='_blank'>Liu et al., 2024, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36587397' target='_blank'>Vienot et al., 2023, Mol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35613022' target='_blank'>Liu et al., N Engl J Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32513235' target='_blank'>Sun et al., J Exp Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31446059' target='_blank'>Tan et al., Gastroenterology, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31336010' target='_blank'>Chen et al., Cancer Sci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30029654' target='_blank'>Starczak et al., J Transl Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29875879' target='_blank'>Dziaman et al., Clin Epigenetics, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29436585' target='_blank'>Xia et al., Int J Mol Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28806978' target='_blank'>Marincevic-Zuniga et al., J Hematol Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26238082' target='_blank'>Cheng et al., Oncol Rep, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25736492' target='_blank'>Laronda et al., Biomaterials, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24276240' target='_blank'>Zhang et al., Oncogene, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23758232' target='_blank'>Yong et al., N Engl J Med, 2013<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39745352' target='_blank'>Garg et al., 2025, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38607068' target='_blank'>Karalis et al., 2024, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33842533' target='_blank'>Wang et al., Front Mol Biosci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33500406' target='_blank'>Fan et al., 2021, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26498848' target='_blank'>Song et al., Oncol Rep, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/29679906' target='_blank'>Xu et al., Biomed Pharmacother, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26879601' target='_blank'>Yin et al., BMC Cancer, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40439125' target='_blank'>Van Wauwe et al., 2025, Cardiovasc Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32607130' target='_blank'>Batista-Gomes et al., Mol Cytogenet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31796844' target='_blank'>Goerlitz et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28041879' target='_blank'>Goldstein et al., Stem Cell Reports, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16574097' target='_blank'>Van Campenhout et al., 2006, Dev Biol<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39295773' target='_blank'>Przybylski et al., 2024, Blood Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25644173' target='_blank'>Li et al., 2015, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24441149' target='_blank'>Abbas et al., Haematologica, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34117212' target='_blank'>Li et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33674250' target='_blank'>Luo et al., Redox Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27694893' target='_blank'>Zhou et al., Oncogene, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38609982' target='_blank'>Li et al., 2024, BMC Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36342857' target='_blank'>Lawson et al., Small GTPases, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32850962' target='_blank'>Vanni et al., Front Mol Biosci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28807941' target='_blank'>Kazanietz et al., Cancer Res, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40599235' target='_blank'>Liang et al., 2025, MedComm (2020)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39477806' target='_blank'>Liu et al., 2024, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37563110' target='_blank'>Zhang et al., 2023, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37020036' target='_blank'>Yang et al., 2023, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36989375' target='_blank'>Chen et al., 2023, Sci Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32740973' target='_blank'>Bai et al., Hepatology, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31815619' target='_blank'>Zhang et al., Mol Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30883733' target='_blank'>Chen et al., J Pathol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30622117' target='_blank'>Filipczak et al., Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29402726' target='_blank'>Wang et al., EBioMedicine, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29235481' target='_blank'>Jiang et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28228863' target='_blank'>Ramsawhook et al., Clin Epigenetics, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26791235' target='_blank'>Huang et al., Cancer Lett, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24931168' target='_blank'>Tung et al., Oncogene, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24743738' target='_blank'>Kang et al., Cell Death Dis, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38983273' target='_blank'>Sun et al., 2024, J Immunol Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38733489' target='_blank'>Nishida et al., 2024, Int J Clin Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35613340' target='_blank'>Shao et al., Hematol Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32854650' target='_blank'>Xue et al., BMC Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29596910' target='_blank'>Shibahara et al., J Thorac Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27050074' target='_blank'>Inoue et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24748806' target='_blank'>Zhang et al., Onco Targets Ther, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24078774' target='_blank'>Akbay et al., Cancer Discov, 2013<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39609384' target='_blank'>Sun et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37430086' target='_blank'>Ahn et al., 2023, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36650130' target='_blank'>Lu et al., Signal Transduct Target Ther, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33990905' target='_blank'>Abak et al., J Mol Neurosci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33692132' target='_blank'>Kumthekar et al., Sci Transl Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26686832' target='_blank'>Park et al., Adv Drug Deliv Rev, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40231473' target='_blank'>Kang et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34203062' target='_blank'>Wirsching et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33996608' target='_blank'>de et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30666272' target='_blank'>Zhang et al., Ann Appl Stat, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28993409' target='_blank'>Saba et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28086098' target='_blank'>Zhang et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25105596' target='_blank'>Baens et al., PLoS One, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23449308' target='_blank'>Young et al., Nat Rev Drug Discov, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23149938' target='_blank'>Chan et al., Mol Cell Biol, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39218851' target='_blank'>Asada et al., 2024, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36207139' target='_blank'>Sahebjam et al., Genes Dev, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33830080' target='_blank'>Chen et al., JCI Insight, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33544757' target='_blank'>Lee et al., PLoS One, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33483471' target='_blank'>Wu et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33237469' target='_blank'>Majewska et al., 2020, Virchows Arch<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31652449' target='_blank'>Zhang et al., Biosci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31355336' target='_blank'>Rodón et al., Sci Adv, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26503699' target='_blank'>Chen et al., BMC Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24121173' target='_blank'>Skálová et al., Am J Surg Pathol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19827123' target='_blank'>Anzick et al., Genes Chromosomes Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18486532' target='_blank'>O'Neill et al., Oral Oncol, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17425505' target='_blank'>Teixeira et al., Crit Rev Oncog, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38579286' target='_blank'>Hassan et al., 2024, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37692504' target='_blank'>Srivastava et al., 2023, Genes Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36106661' target='_blank'>Ter et al., J Pathol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30924770' target='_blank'>Fearnley et al., 2019, Elife<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27511199' target='_blank'>Hilkens et al., Gut, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38198023' target='_blank'>Zhang et al., 2024, Biochem Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33976733' target='_blank'>Deng et al., J Cancer, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40047406' target='_blank'>Barber et al., 2025, Elife<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37164974' target='_blank'>You et al., 2023, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37149661' target='_blank'>Li et al., 2023, J Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34237057' target='_blank'>Kharin et al., PLoS One, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34062997' target='_blank'>Bley et al., Biology (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34047477' target='_blank'>Dong et al., Clin Transl Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33723247' target='_blank'>Makhov et al., Oncogenesis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30419948' target='_blank'>Zhan et al., 2018, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30237545' target='_blank'>Xu et al., Cell Death Dis, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27449499' target='_blank'>Guo et al., Clin Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25774828' target='_blank'>Wang et al., Nat Commun, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40115133' target='_blank'>Chen et al., 2025, Blood Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38403772' target='_blank'>Matsuura et al., 2024, Intern Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38327754' target='_blank'>Herbison et al., 2024, J Immunother Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33916219' target='_blank'>Harachi et al., Metabolites, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30338611' target='_blank'>Tamura et al., Genes Chromosomes Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30257209' target='_blank'>Shiota et al., Cell Rep, 2018<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40746724' target='_blank'>Zhang et al., 2025, Front Pharmacol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33248210' target='_blank'>Vellichirammal et al., Cancer Lett, 2021<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39221900' target='_blank'>Wang et al., 2024, J Cell Physiol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36979146' target='_blank'>Lv et al., 2023, Biology (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34830078' target='_blank'>Haschemi et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33550920' target='_blank'>Lv et al., Technol Cancer Res Treat, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32613480' target='_blank'>Huang et al., Clin Exp Metastasis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31717266' target='_blank'>Zhang et al., Int J Mol Sci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30881041' target='_blank'>Peng et al., Onco Targets Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29765447' target='_blank'>Li et al., Arch Med Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29510730' target='_blank'>Zhao et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28570591' target='_blank'>Juneja et al., PLoS Biol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28348518' target='_blank'>Zhou et al., Front Mol Neurosci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28339092' target='_blank'>Duan et al., Oncol Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27547006' target='_blank'>Wu et al., World J Gastroenterol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26919111' target='_blank'>Yang et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25043301' target='_blank'>Lin et al., Oncogene, 2015<\/a>"],["This gene is a protooncogene that encodes a protein tyrosine kinase involved in a variety of cellular processes, including cell division, adhesion, differentiation, and response to stress. The activity of the protein is negatively regulated by its SH3 domain, whereby deletion of the region encoding this domain results in an oncogene. The ubiquitously expressed protein has DNA-binding activity that is regulated by CDC2-mediated phosphorylation, suggesting a cell cycle function. This gene has been found fused to a variety of translocation partner genes in various leukemias, most notably the t(9;22) translocation that results in a fusion with the 5' end of the breakpoint cluster region gene (BCR; MIM:151410). Alternative splicing of this gene results in two transcript variants, which contain alternative first exons that are spliced to the remaining common exons.","This gene encodes a member of the Abelson family of nonreceptor tyrosine protein kinases. The protein is highly similar to the c-abl oncogene 1 protein, including the tyrosine kinase, SH2 and SH3 domains, and it plays a role in cytoskeletal rearrangements through its C-terminal F-actin- and microtubule-binding sequences. This gene is expressed in both normal and tumor cells, and is involved in translocation with the ets variant 6 gene in leukemia. Multiple alternatively spliced transcript variants encoding different protein isoforms have been found for this gene.","Activins are dimeric growth and differentiation factors which belong to the transforming growth factor-beta (TGF-beta) superfamily of structurally related signaling proteins. Activins signal through a heteromeric complex of receptor serine kinases which include at least two type I ( I and IB) and two type II (II and IIB) receptors. These receptors are all transmembrane proteins, composed of a ligand-binding extracellular domain with cysteine-rich region, a transmembrane domain, and a cytoplasmic domain with predicted serine/threonine specificity. Type I receptors are essential for signaling; and type II receptors are required for binding ligands and for expression of type I receptors. Type I and II receptors form a stable complex after ligand binding, resulting in phosphorylation of type I receptors by type II receptors. This gene encodes activin A type I receptor which signals a particular transcriptional response in concert with activin type II receptors. Mutations in this gene are associated with fibrodysplasia ossificans progressive.","This gene encodes one of the three members of the human AKT serine-threonine protein kinase family which are often referred to as protein kinase B alpha, beta, and gamma. These highly similar AKT proteins all have an N-terminal pleckstrin homology domain, a serine/threonine-specific kinase domain and a C-terminal regulatory domain. These proteins are phosphorylated by phosphoinositide 3-kinase (PI3K). AKT/PI3K forms a key component of many signalling pathways that involve the binding of membrane-bound ligands such as receptor tyrosine kinases, G-protein coupled receptors, and integrin-linked kinase. These AKT proteins therefore regulate a wide variety of cellular functions including cell proliferation, survival, metabolism, and angiogenesis in both normal and malignant cells. AKT proteins are recruited to the cell membrane by phosphatidylinositol 3,4,5-trisphosphate (PIP3) after phosphorylation of phosphatidylinositol 4,5-bisphosphate (PIP2) by PI3K. Subsequent phosphorylation of both threonine residue 308 and serine residue 473 is required for full activation of the AKT1 protein encoded by this gene. Phosphorylation of additional residues also occurs, for example, in response to insulin growth factor-1 and epidermal growth factor. Protein phosphatases act as negative regulators of AKT proteins by dephosphorylating AKT or PIP3. The PI3K/AKT signalling pathway is crucial for tumor cell survival. Survival factors can suppress apoptosis in a transcription-independent manner by activating AKT1 which then phosphorylates and inactivates components of the apoptotic machinery. AKT proteins also participate in the mammalian target of rapamycin (mTOR) signalling pathway which controls the assembly of the eukaryotic translation initiation factor 4F (eIF4E) complex and this pathway, in addition to responding to extracellular signals from growth factors and cytokines, is disregulated in many cancers. Mutations in this gene are associated with multiple types of cancer and excessive tissue growth including Proteus syndrome and Cowden syndrome 6, and breast, colorectal, and ovarian cancers. Multiple alternatively spliced transcript variants have been found for this gene.","This gene is a putative oncogene encoding a protein belonging to a subfamily of serine/threonine kinases containing SH2-like (Src homology 2-like) domains, which is involved in signaling pathways. The gene serves as an oncogene in the tumorigenesis of cancer cells For example, its overexpression contributes to the malignant phenotype of a subset of human ductal pancreatic cancers. The encoded protein is a general protein kinase capable of phophorylating several known proteins, and has also been implicated in insulin signaling.","This gene encodes a receptor tyrosine kinase, which belongs to the insulin receptor superfamily. This protein comprises an extracellular domain, an hydrophobic stretch corresponding to a single pass transmembrane region, and an intracellular kinase domain. It plays an important role in the development of the brain and exerts its effects on specific neurons in the nervous system. This gene has been found to be rearranged, mutated, or amplified in a series of tumours including anaplastic large cell lymphomas, neuroblastoma, and non-small cell lung cancer. The chromosomal rearrangements are the most common genetic alterations in this gene, which result in creation of multiple fusion genes in tumourigenesis, including ALK (chromosome 2)/EML4 (chromosome 2), ALK/RANBP2 (chromosome 2), ALK/ATIC (chromosome 2), ALK/TFG (chromosome 3), ALK/NPM1 (chromosome 5), ALK/SQSTM1 (chromosome 5), ALK/KIF5B (chromosome 10), ALK/CLTC (chromosome 17), ALK/TPM4 (chromosome 19), and ALK/MSN (chromosome X).[provided by RefSeq, Jan 2011].","This gene encodes a member of the IAP family of proteins that inhibit apoptosis by binding to tumor necrosis factor receptor-associated factors TRAF1 and TRAF2, probably by interfering with activation of ICE-like proteases. The encoded protein inhibits apoptosis induced by serum deprivation but does not affect apoptosis resulting from exposure to menadione, a potent inducer of free radicals. It contains 3 baculovirus IAP repeats and a ring finger domain. Transcript variants encoding the same isoform have been identified.","The androgen receptor gene is more than 90 kb long and codes for a protein that has 3 major functional domains: the N-terminal domain, DNA-binding domain, and androgen-binding domain. The protein functions as a steroid-hormone activated transcription factor. Upon binding the hormone ligand, the receptor dissociates from accessory proteins, translocates into the nucleus, dimerizes, and then stimulates transcription of androgen responsive genes. This gene contains 2 polymorphic trinucleotide repeat segments that encode polyglutamine and polyglycine tracts in the N-terminal transactivation domain of its protein. Expansion of the polyglutamine tract from the normal 9-34 repeats to the pathogenic 38-62 repeats causes spinal bulbar muscular atrophy (SBMA, also known as Kennedy's disease). Mutations in this gene are also associated with complete androgen insensitivity (CAIS). Alternative splicing results in multiple transcript variants encoding different isoforms.","This proto-oncogene belongs to the RAF subfamily of the Ser/Thr protein kinase family, and maybe involved in cell growth and development. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jan 2012]","This gene encodes a member of the Rho family of small GTPases, which cycle between inactive GDP-bound and active GTP-bound states and function as molecular switches in signal transduction cascades. Rho proteins promote reorganization of the actin cytoskeleton and regulate cell shape, attachment, and motility. Overexpression of this gene is associated with tumor cell proliferation and metastasis. Multiple alternatively spliced variants have been identified.","Rho GTPase activating protein 5 negatively regulates RHO GTPases, a family which may mediate cytoskeleton changes by stimulating the hydrolysis of bound GTP. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a protein containing a basic helix-loop-helix domain and two characteristic PAS domains along with a PAC domain. The encoded protein binds to ligand-bound aryl hydrocarbon receptor and aids in the movement of this complex to the nucleus, where it promotes the expression of genes involved in xenobiotic metabolism. This protein is also a co-factor for transcriptional regulation by hypoxia-inducible factor 1. Chromosomal translocation of this locus with the ETV6 (ets variant 6) gene on chromosome 12 have been described in leukemias. Alternative splicing results in multiple transcript variants.","This gene encodes an activating transcription factor, which belongs to the ATF subfamily and bZIP (basic-region leucine zipper) family. It influences cellular physiologic processes by regulating the expression of downstream target genes, which are related to growth, survival, and other cellular activities. This protein is phosphorylated at serine 63 in its kinase-inducible domain by serine/threonine kinases, cAMP-dependent protein kinase A, calmodulin-dependent protein kinase I/II, mitogen- and stress-activated protein kinase and cyclin-dependent kinase 3 (cdk-3). Its phosphorylation enhances its transactivation and transcriptional activities, and enhances cell transformation. Fusion of this gene and FUS on chromosome 16 or EWSR1 on chromosome 22 induced by translocation generates chimeric proteins in angiomatoid fibrous histiocytoma and clear cell sarcoma. This gene has a pseudogene on chromosome 6.","The protein encoded by this gene belongs to the family of P-type cation transport ATPases, and to the subfamily of Na+/K+ -ATPases. Na+/K+ -ATPase is an integral membrane protein responsible for establishing and maintaining the electrochemical gradients of Na and K ions across the plasma membrane. These gradients are essential for osmoregulation, for sodium-coupled transport of a variety of organic and inorganic molecules, and for electrical excitability of nerve and muscle. This enzyme is composed of two subunits, a large catalytic subunit (alpha) and a smaller glycoprotein subunit (beta). The catalytic subunit of Na+/K+ -ATPase is encoded by multiple genes. This gene encodes an alpha 1 subunit. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance throughout the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with and functions as a regulatory subunit of CDK4 or CDK6, whose activity is required for cell cycle G1/S transition. This protein has been shown to interact with tumor suppressor protein Rb and the expression of this gene is regulated positively by Rb. Mutations, amplification and overexpression of this gene, which alters cell cycle progression, are observed frequently in a variety of human cancers.","This gene encodes an integral outer mitochondrial membrane protein that blocks the apoptotic death of some cells such as lymphocytes. Constitutive expression of BCL2, such as in the case of translocation of BCL2 to Ig heavy chain locus, is thought to be the cause of follicular lymphoma. Alternative splicing results in multiple transcript variants.","This gene is a proto-oncogene candidate. It is identified by its translocation into the immunoglobulin alpha-locus in some cases of B-cell leukemia. The protein encoded by this gene contains seven ankyrin repeats, which are most closely related to those found in I kappa B proteins. This protein functions as a transcriptional co-activator that activates through its association with NF-kappa B homodimers. The expression of this gene can be induced by NF-kappa B, which forms a part of the autoregulatory loop that controls the nuclear residence of p50 NF-kappa B.","The protein encoded by this gene is a zinc finger transcription factor and contains an N-terminal POZ domain. This protein acts as a sequence-specific repressor of transcription, and has been shown to modulate the transcription of STAT-dependent IL-4 responses of B cells. This protein can interact with a variety of POZ-containing proteins that function as transcription corepressors. This gene is found to be frequently translocated and hypermutated in diffuse large-cell lymphoma (DLCL), and may be involved in the pathogenesis of DLCL. Alternatively spliced transcript variants encoding different protein isoforms have been found for this gene.","BCL9 is associated with B-cell acute lymphoblastic leukemia. It may be a target of translocation in B-cell malignancies with abnormalities of 1q21. Its function is unknown. The overexpression of BCL9 may be of pathogenic significance in B-cell malignancies.","The protein encoded by this gene is a member of the TNF-receptor superfamily. This receptor is preferentially expressed in mature B lymphocytes, and may be important for B cell development and autoimmune response. This receptor has been shown to specifically bind to the tumor necrosis factor (ligand) superfamily, member 13b (TNFSF13B/TALL-1/BAFF), and to lead to NF-kappaB and MAPK8/JNK activation. This receptor also binds to various TRAF family members, and thus may transduce signals for cell survival and proliferation.","The bone morphogenetic protein (BMP) receptors are a family of transmembrane serine/threonine kinases that include the type I receptors BMPR1A and BMPR1B and the type II receptor BMPR2. These receptors are also closely related to the activin receptors, ACVR1 and ACVR2. The ligands of these receptors are members of the TGF-beta superfamily. TGF-betas and activins transduce their signals through the formation of heteromeric complexes with 2 different types of serine (threonine) kinase receptors: type I receptors of about 50-55 kD and type II receptors of about 70-80 kD. Type II receptors bind ligands in the absence of type I receptors, but they require their respective type I receptors for signaling, whereas type I receptors require their respective type II receptors for ligand binding.","This gene encodes a forkhead transcription factor. The protein contains a fork-head DNA-binding domain and may play a role in ovarian development and function. Expansion of a polyalanine repeat region and other mutations in this gene are a cause of blepharophimosis syndrome and premature ovarian failure 3.","This gene encodes a protein belonging to the RAF family of serine/threonine protein kinases. This protein plays a role in regulating the MAP kinase/ERK signaling pathway, which affects cell division, differentiation, and secretion. Mutations in this gene, most commonly the V600E mutation, are the most frequently identified cancer-causing mutations in melanoma, and have been identified in various other cancers as well, including non-Hodgkin lymphoma, colorectal cancer, thyroid carcinoma, non-small cell lung carcinoma, hairy cell leukemia and adenocarcinoma of lung. Mutations in this gene are also associated with cardiofaciocutaneous, Noonan, and Costello syndromes, which exhibit overlapping phenotypes. A pseudogene of this gene has been identified on the X chromosome.","The protein encoded by this gene plays a crucial role in B-cell development. Mutations in this gene cause X-linked agammaglobulinemia type 1, which is an immunodeficiency characterized by the failure to produce mature B lymphocytes, and associated with a failure of Ig heavy chain rearrangement. Alternative splicing results in multiple transcript variants encoding different isoforms.","Voltage-dependent calcium channels mediate the entry of calcium ions into excitable cells, and are also involved in a variety of calcium-dependent processes, including muscle contraction, hormone or neurotransmitter release, and gene expression. Calcium channels are multisubunit complexes composed of alpha-1, beta, alpha-2/delta, and gamma subunits. The channel activity is directed by the pore-forming alpha-1 subunit, whereas the others act as auxiliary subunits regulating this activity. The distinctive properties of the calcium channel types are related primarily to the expression of a variety of alpha-1 isoforms, namely alpha-1A, B, C, D, E, and S. This gene encodes the alpha-1D subunit. Several transcript variants encoding different isoforms have been found for this gene.","Calreticulin is a highly conserved chaperone protein which resides primarily in the endoplasmic reticulum, and is involved in a variety of cellular processes, among them, cell adhesion. Additionally, it functions in protein folding quality control and calcium homeostasis. Calreticulin is also found in the nucleus, suggesting that it may have a role in transcription regulation. Systemic lupus erythematosus is associated with increased autoantibody titers against calreticulin. Recurrent mutations in calreticulin have been linked to various neoplasms, including the myeloproliferative type.[provided by RefSeq, May 2020].","Core binding factor (CBF) is a heterodimeric transcription factor that binds to the core element of many enhancers and promoters. The protein encoded by this gene represents the alpha subunit of CBF and is thought to be involved in the development of normal hematopoiesis. Chromosomal translocations involving this gene are well-documented and have been associated with several types of leukemia. Three transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the myeloid translocation gene family which interact with DNA-bound transcription factors and recruit a range of corepressors to facilitate transcriptional repression. The t(8;21)(q22;q22) translocation is one of the most frequent karyotypic abnormalities in acute myeloid leukemia. The translocation produces a chimeric gene made up of the 5'-region of the runt-related transcription factor 1 gene fused to the 3'-region of this gene. The chimeric protein is thought to associate with the nuclear corepressor/histone deacetylase complex to block hematopoietic differentiation. Alternative splicing results in multiple transcript variants.","This gene is a proto-oncogene that encodes a RING finger E3 ubiquitin ligase. The encoded protein is one of the enzymes required for targeting substrates for degradation by the proteasome. This protein mediates the transfer of ubiquitin from ubiquitin conjugating enzymes (E2) to specific substrates. This protein also contains an N-terminal phosphotyrosine binding domain that allows it to interact with numerous tyrosine-phosphorylated substrates and target them for proteasome degradation. As such it functions as a negative regulator of many signal transduction pathways. This gene has been found to be mutated or translocated in many cancers including acute myeloid leukaemia, and expansion of CGG repeats in the 5' UTR has been associated with Jacobsen syndrome. Mutations in this gene are also the cause of Noonan syndrome-like disorder.","The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with CDK4 or CDK6 and functions as a regulatory subunit of the complex, whose activity is required for cell cycle G1/S transition. This protein has been shown to interact with and be involved in the phosphorylation of tumor suppressor protein Rb. Knockout studies of the homologous gene in mouse suggest the essential roles of this gene in ovarian granulosa and germ cell proliferation. High level expression of this gene was observed in ovarian and testicular tumors. Mutations in this gene are associated with megalencephaly-polymicrogyria-polydactyly-hydrocephalus syndrome 3 (MPPH3).","The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with and functions as a regulatory subunit of CDK4 or CDK6, whose activtiy is required for cell cycle G1/S transition. This protein has been shown to interact with and be involved in the phosphorylation of tumor suppressor protein Rb. The CDK4 activity associated with this cyclin was reported to be necessary for cell cycle progression through G2 phase into mitosis after UV radiation. Several transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with and functions as a regulatory subunit of CDK2, whose activity is required for cell cycle G1/S transition. This protein accumulates at the G1-S phase boundary and is degraded as cells progress through S phase. Overexpression of this gene has been observed in many tumors, which results in chromosome instability, and thus may contribute to tumorigenesis. This protein was found to associate with, and be involved in, the phosphorylation of NPAT protein (nuclear protein mapped to the ATM locus), which participates in cell-cycle regulated histone gene expression and plays a critical role in promoting cell-cycle progression in the absence of pRB.","The protein encoded by this gene is essential for T-cell proliferation and survival, cytokine production, and T-helper type-2 development. Several alternatively spliced transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jul 2011].","The protein encoded by this gene associates with class II major histocompatibility complex (MHC) and is an important chaperone that regulates antigen presentation for immune response. It also serves as cell surface receptor for the cytokine macrophage migration inhibitory factor (MIF) which, when bound to the encoded protein, initiates survival pathways and cell proliferation. This protein also interacts with amyloid precursor protein (APP) and suppresses the production of amyloid beta (Abeta). Multiple alternatively spliced transcript variants encoding different isoforms have been identified.","The B lymphocyte antigen receptor is a multimeric complex that includes the antigen-specific component, surface immunoglobulin (Ig). Surface Ig non-covalently associates with two other proteins, Ig-alpha and Ig-beta, which are necessary for expression and function of the B-cell antigen receptor. This gene encodes the Ig-alpha protein of the B-cell antigen component. Alternatively spliced transcript variants encoding different isoforms have been described.","The B lymphocyte antigen receptor is a multimeric complex that includes the antigen-specific component, surface immunoglobulin (Ig). Surface Ig non-covalently associates with two other proteins, Ig-alpha and Ig-beta, which are necessary for expression and function of the B-cell antigen receptor. This gene encodes the Ig-beta protein of the B-cell antigen component. Alternatively spliced transcript variants encoding different isoforms have been described.","This gene is a member of the cadherin superfamily, genes encoding calcium-dependent, membrane-associated glycoproteins. The encoded protein is cadherin-like, consisting of an extracellular region, containing 7 cadherin domains, and a transmembrane region but lacking the conserved cytoplasmic domain. The protein is a component of the gastrointestinal tract and pancreatic ducts, acting as an intestinal proton-dependent peptide transporter in the first step in oral absorption of many medically important peptide-based drugs. The protein may also play a role in the morphological organization of liver and intestine. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a member of the Ser/Thr protein kinase family. This protein is highly similar to the gene products of S. cerevisiae cdc28 and S. pombe cdc2. It is a catalytic subunit of the protein kinase complex that is important for cell cycle G1 phase progression. The activity of this kinase is restricted to the G1-S phase, which is controlled by the regulatory subunits D-type cyclins and CDK inhibitor p16(INK4a). This kinase was shown to be responsible for the phosphorylation of retinoblastoma gene product (Rb). Mutations in this gene as well as in its related proteins including D-type cyclins, p16(INK4a) and Rb were all found to be associated with tumorigenesis of a variety of cancers. Multiple polyadenylation sites of this gene have been reported.","The protein encoded by this gene is a member of the CMGC family of serine/threonine protein kinases. This kinase is a catalytic subunit of the protein kinase complex that is important for cell cycle G1 phase progression and G1/S transition. The activity of this kinase first appears in mid-G1 phase, which is controlled by the regulatory subunits including D-type cyclins and members of INK4 family of CDK inhibitors. This kinase, as well as CDK4, has been shown to phosphorylate, and thus regulate the activity of, tumor suppressor protein Rb. Altered expression of this gene has been observed in multiple human cancers. A mutation in this gene resulting in reduced cell proliferation, and impaired cell motility and polarity, and has been identified in patients with primary microcephaly.","This gene encodes a potent cyclin-dependent kinase inhibitor. The encoded protein binds to and inhibits the activity of cyclin-cyclin-dependent kinase2 or -cyclin-dependent kinase4 complexes, and thus functions as a regulator of cell cycle progression at G1. The expression of this gene is tightly controlled by the tumor suppressor protein p53, through which this protein mediates the p53-dependent cell cycle G1 phase arrest in response to a variety of stress stimuli. This protein can interact with proliferating cell nuclear antigen, a DNA polymerase accessory factor, and plays a regulatory role in S phase DNA replication and DNA damage repair. This protein was reported to be specifically cleaved by CASP3-like caspases, which thus leads to a dramatic activation of cyclin-dependent kinase2, and may be instrumental in the execution of apoptosis following caspase activation. Mice that lack this gene have the ability to regenerate damaged or missing tissue. Multiple alternatively spliced variants have been found for this gene.","The product of this gene belongs to the SNF2/RAD54 helicase family. It represents the main component of the nucleosome remodeling and deacetylase complex and plays an important role in epigenetic transcriptional repression. Patients with dermatomyositis develop antibodies against this protein. Somatic mutations in this gene are associated with serous endometrial tumors. Alternative splicing results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene belongs to the G-protein-coupled receptor family . It is a receptor for the CC chemokine - MIP-1, RANTES, TARC and MCP-1. Chemokines are a group of small polypeptide, structurally related molecules that regulate cell trafficking of various types of leukocytes. The chemokines also play fundamental roles in the development, homeostasis, and function of the immune system, and they have effects on cells of the central nervous system as well as on endothelial cells involved in angiogenesis or angiostasis.","The protein encoded by this gene is a member of the G protein-coupled receptor family. This receptor was identified as a gene induced by the Epstein-Barr virus (EBV), and is thought to be a mediator of EBV effects on B lymphocytes. This receptor is expressed in various lymphoid tissues and activates B and T lymphocytes. It has been shown to control the migration of memory T cells to inflamed tissues, as well as stimulate dendritic cell maturation. The chemokine (C-C motif) ligand 19 (CCL19/ECL) has been reported to be a specific ligand of this receptor. Signals mediated by this receptor regulate T cell homeostasis in lymph nodes, and may also function in the activation and polarization of T cells, and in chronic inflammation pathogenesis. Alternative splicing of this gene results in multiple transcript variants.","This gene encodes a transcription factor that is a member of the leucine zipper family of DNA binding proteins. This protein binds as a homodimer to the cAMP-responsive element, an octameric palindrome. The protein is phosphorylated by several protein kinases, and induces transcription of genes in response to hormonal stimulation of the cAMP pathway. Alternate splicing of this gene results in several transcript variants encoding different isoforms.","This gene is ubiquitously expressed and is involved in the transcriptional coactivation of many different transcription factors. First isolated as a nuclear protein that binds to cAMP-response element binding protein (CREB), this gene is now known to play critical roles in embryonic development, growth control, and homeostasis by coupling chromatin remodeling to transcription factor recognition. The protein encoded by this gene has intrinsic histone acetyltransferase activity and also acts as a scaffold to stabilize additional protein interactions with the transcription complex. This protein acetylates both histone and non-histone proteins. This protein shares regions of very high sequence similarity with protein p300 in its bromodomain, cysteine-histidine-rich regions, and histone acetyltransferase domain. Mutations in this gene cause Rubinstein-Taybi syndrome (RTS). Chromosomal translocations involving this gene have been associated with acute myeloid leukemia. Alternative splicing results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene is the receptor for colony stimulating factor 1, a cytokine which controls the production, differentiation, and function of macrophages. This receptor mediates most if not all of the biological effects of this cytokine. Ligand binding activates the receptor kinase through a process of oligomerization and transphosphorylation. The encoded protein is a tyrosine kinase transmembrane receptor and member of the CSF1/PDGF receptor family of tyrosine-protein kinases. Mutations in this gene have been associated with a predisposition to myeloid malignancy. The first intron of this gene contains a transcriptionally inactive ribosomal protein L7 processed pseudogene oriented in the opposite direction. Alternative splicing results in multiple transcript variants. Expression of a splice variant from an LTR promoter has been found in Hodgkin lymphoma (HL), HL cell lines and anaplastic large cell lymphoma.","The protein encoded by this gene is the receptor for colony stimulating factor 3, a cytokine that controls the production, differentiation, and function of granulocytes. The encoded protein, which is a member of the family of cytokine receptors, may also function in some cell surface adhesion or recognition processes. Alternatively spliced transcript variants have been described. Mutations in this gene are a cause of Kostmann syndrome, also known as severe congenital neutropenia.","Enables actin filament binding activity. Involved in negative regulation of Arp2/3 complex-mediated actin nucleation; regulation of neuron migration; and regulation of neuron projection development. Located in cytoplasm. Implicated in complex cortical dysplasia with other brain malformations. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is part of a complex of proteins that constitute adherens junctions (AJs). AJs are necessary for the creation and maintenance of epithelial cell layers by regulating cell growth and adhesion between cells. The encoded protein also anchors the actin cytoskeleton and may be responsible for transmitting the contact inhibition signal that causes cells to stop dividing once the epithelial sheet is complete. Finally, this protein binds to the product of the APC gene, which is mutated in adenomatous polyposis of the colon. Mutations in this gene are a cause of colorectal cancer (CRC), pilomatrixoma (PTR), medulloblastoma (MDB), and ovarian cancer. Alternative splicing results in multiple transcript variants.","This gene encodes an adhesive junction associated protein of the armadillo/beta-catenin superfamily and is implicated in brain and eye development and cancer formation. The protein encoded by this gene promotes the disruption of E-cadherin based adherens junction to favor cell spreading upon stimulation by hepatocyte growth factor. This gene is overexpressed in prostate adenocarcinomas and is associated with decreased expression of tumor suppressor E-cadherin in this tissue. This gene resides in a region of the short arm of chromosome 5 that is deleted in Cri du Chat syndrome. Alternative splicing results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene is a member of the homeodomain family of DNA binding proteins. It may regulate gene expression, morphogenesis, and differentiation and it may also play a role in the cell cycle progession. Several alternatively spliced transcript variants encoding different isoforms have been identified.[provided by RefSeq, Feb 2011].","This gene encodes a multifunctional protein that resides in multiple locations in the nucleus and in the cytoplasm. It interacts with a wide variety of proteins, such as apoptosis antigen Fas, centromere protein C, and transcription factor erythroblastosis virus E26 oncogene homolog 1. In the nucleus, the encoded protein functions as a potent transcription repressor that binds to sumoylated transcription factors. Its repression can be relieved by the sequestration of this protein into promyelocytic leukemia nuclear bodies or nucleoli. This protein also associates with centromeres in G2 phase. In the cytoplasm, the encoded protein may function to regulate apoptosis. The subcellular localization and function of this protein are modulated by post-translational modifications, including sumoylation, phosphorylation and polyubiquitination. Alternative splicing results in multiple transcript variants.","This gene encodes a protein that is necessary for the repair of ultraviolet light-damaged DNA. This protein is the smaller subunit of a heterodimeric protein complex that participates in nucleotide excision repair, and this complex mediates the ubiquitylation of histones H3 and H4, which facilitates the cellular response to DNA damage. This subunit appears to be required for DNA binding. Mutations in this gene cause xeroderma pigmentosum complementation group E, a recessive disease that is characterized by an increased sensitivity to UV light and a high predisposition for skin cancer development, in some cases accompanied by neurological abnormalities. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the CCAAT/enhancer-binding protein (C/EBP) family of transcription factors. The protein functions as a dominant-negative inhibitor by forming heterodimers with other C/EBP members, such as C/EBP and LAP (liver activator protein), and preventing their DNA binding activity. The protein is implicated in adipogenesis and erythropoiesis, is activated by endoplasmic reticulum stress, and promotes apoptosis. Fusion of this gene and FUS on chromosome 16 or EWSR1 on chromosome 22 induced by translocation generates chimeric proteins in myxoid liposarcomas or Ewing sarcoma. Multiple alternatively spliced transcript variants encoding two isoforms with different length have been identified.","This gene encodes a member of the DEAD box family of RNA helicases that are involved in a variety of cellular processes as a result of its role as an adaptor molecule, promoting interactions with a large number of other factors. This protein is involved in pathways that include the alteration of RNA structures, plays a role as a coregulator of transcription, a regulator of splicing, and in the processing of small noncoding RNAs. Members of this family contain nine conserved motifs, including the conserved Asp-Glu-Ala-Asp (DEAD) motif, important to ATP binding and hydrolysis as well as RNA binding and unwinding activities. Dysregulation of this gene may play a role in cancer development. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the DEAD box protein family. The protein is an RNA helicase found in P-bodies and stress granules, and functions in translation suppression and mRNA degradation. It is required for microRNA-induced gene silencing. Multiple alternatively spliced variants, encoding the same protein, have been identified.","This gene encodes an enzyme that transfers methyl groups to cytosine nucleotides of genomic DNA. This protein is the major enzyme responsible for maintaining methylation patterns following DNA replication and shows a preference for hemi-methylated DNA. Methylation of DNA is an important component of mammalian epigenetic gene regulation. Aberrant methylation patterns are found in human tumors and associated with developmental abnormalities. Variation in this gene has been associated with cerebellar ataxia, deafness, and narcolepsy, and neuropathy, hereditary sensory, type IE. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a transmembrane glycoprotein that is a member of the protein kinase superfamily. This protein is a receptor for members of the epidermal growth factor family. EGFR is a cell surface protein that binds to epidermal growth factor, thus inducing receptor dimerization and tyrosine autophosphorylation leading to cell proliferation. Mutations in this gene are associated with lung cancer. EGFR is a component of the cytokine storm which contributes to a severe form of Coronavirus Disease 2019 (COVID-19) resulting from infection with severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2).","This gene encodes an essential eukaryotic translation initiation factor. The protein is required for the binding of the 43S complex (a 40S subunit, eIF2/GTP/Met-tRNAi and eIF3) to the 5' end of capped RNA.","The protein encoded by this gene is a transcriptional activator that binds and activates the promoters of the CSF2, IL3, IL8, and PRF1 genes. The encoded protein is involved in natural killer cell development and function, innate immunity, and induction of cell cycle arrest in naive CD8+ cells. Two transcript variants encoding the same protein have been found for this gene.[provided by RefSeq, Jan 2010].","This gene is a member of the Ets family of transcription factors and of the ternary complex factor (TCF) subfamily. Proteins of the TCF subfamily form a ternary complex by binding to the the serum response factor and the serum reponse element in the promoter of the c-fos proto-oncogene. The protein encoded by this gene is phosphorylated by the kinases, MAPK1 and MAPK8. Several transcript variants have been described for this gene.","This gene encodes a transcription factor involved in the induction of genes regulated by oxygen, which is induced as oxygen levels fall. The encoded protein contains a basic-helix-loop-helix domain protein dimerization domain as well as a domain found in proteins in signal transduction pathways which respond to oxygen levels. Mutations in this gene are associated with erythrocytosis familial type 4.","This gene encodes a member of the epidermal growth factor (EGF) receptor family of receptor tyrosine kinases. This protein has no ligand binding domain of its own and therefore cannot bind growth factors. However, it does bind tightly to other ligand-bound EGF receptor family members to form a heterodimer, stabilizing ligand binding and enhancing kinase-mediated activation of downstream signalling pathways, such as those involving mitogen-activated protein kinase and phosphatidylinositol-3 kinase. Allelic variations at amino acid positions 654 and 655 of isoform a (positions 624 and 625 of isoform b) have been reported, with the most common allele, Ile654/Ile655, shown here. Amplification and/or overexpression of this gene has been reported in numerous cancers, including breast and ovarian tumors. Alternative splicing results in several additional transcript variants, some encoding different isoforms and others that have not been fully characterized.","This gene encodes a member of the epidermal growth factor receptor (EGFR) family of receptor tyrosine kinases. This membrane-bound protein has a neuregulin binding domain but not an active kinase domain. It therefore can bind this ligand but not convey the signal into the cell through protein phosphorylation. However, it does form heterodimers with other EGF receptor family members which do have kinase activity. Heterodimerization leads to the activation of pathways which lead to cell proliferation or differentiation. Amplification of this gene and/or overexpression of its protein have been reported in numerous cancers, including prostate, bladder, and breast tumors. Alternate transcriptional splice variants encoding different isoforms have been characterized. One isoform lacks the intermembrane region and is secreted outside the cell. This form acts to modulate the activity of the membrane-bound form. Additional splice variants have also been reported, but they have not been thoroughly characterized.","This gene is a member of the Tyr protein kinase family and the epidermal growth factor receptor subfamily. It encodes a single-pass type I membrane protein with multiple cysteine rich domains, a transmembrane domain, a tyrosine kinase domain, a phosphotidylinositol-3 kinase binding site and a PDZ domain binding motif. The protein binds to and is activated by neuregulins and other factors and induces a variety of cellular responses including mitogenesis and differentiation. Multiple proteolytic events allow for the release of a cytoplasmic fragment and an extracellular fragment. Mutations in this gene have been associated with cancer. Alternatively spliced variants which encode different protein isoforms have been described; however, not all variants have been fully characterized.","This gene encodes a member of the erythroblast transformation-specific (ETS) family of transcriptions factors. All members of this family are key regulators of embryonic development, cell proliferation, differentiation, angiogenesis, inflammation, and apoptosis. The protein encoded by this gene is mainly expressed in the nucleus. It contains an ETS DNA-binding domain and a PNT (pointed) domain which is implicated in the self-association of chimeric oncoproteins. This protein is required for platelet adhesion to the subendothelium, inducing vascular cell remodeling. It also regulates hematopoesis, and the differentiation and maturation of megakaryocytic cells. This gene is involved in chromosomal translocations, resulting in different fusion gene products, such as TMPSSR2-ERG and NDRG1-ERG in prostate cancer, EWS-ERG in Ewing's sarcoma and FUS-ERG in acute myeloid leukemia. More than two dozens of transcript variants generated from combinatorial usage of three alternative promoters and multiple alternative splicing events have been reported, but the full-length nature of many of these variants has not been determined.","This gene encodes an estrogen receptor and ligand-activated transcription factor. The canonical protein contains an N-terminal ligand-independent transactivation domain, a central DNA binding domain, a hinge domain, and a C-terminal ligand-dependent transactivation domain. The protein localizes to the nucleus where it may form either a homodimer or a heterodimer with estrogen receptor 2. The protein encoded by this gene regulates the transcription of many estrogen-inducible genes that play a role in growth, metabolism, sexual development, gestation, and other reproductive functions and is expressed in many non-reproductive tissues. The receptor encoded by this gene plays a key role in breast cancer, endometrial cancer, and osteoporosis. This gene is reported to have dozens of transcript variants due to the use of alternate promoters and alternative splicing, however, the full-length nature of many of these variants remain uncertain.","This gene encodes a member of the ETS (E twenty-six) family of transcription factors. The ETS proteins regulate many target genes that modulate biological processes like cell growth, angiogenesis, migration, proliferation and differentiation. All ETS proteins contain an ETS DNA-binding domain that binds to DNA sequences containing the consensus 5'-CGGA[AT]-3'. The protein encoded by this gene contains a conserved short acidic transactivation domain (TAD) in the N-terminal region, in addition to the ETS DNA-binding domain in the C-terminal region. This gene is involved in chromosomal translocations, which result in multiple fusion proteins including EWS-ETV1 in Ewing sarcoma and at least 10 ETV1 partners (see PMID: 19657377, Table 1) in prostate cancer. In addition to chromosomal rearrangement, this gene is overexpressed in prostate cancer, melanoma and gastrointestinal stromal tumor. Multiple alternatively spliced transcript variants encoding different isoforms have been identified.","Enables DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Involved in positive regulation of keratinocyte differentiation and positive regulation of transcription by RNA polymerase II. Located in chromosome and nucleolus. [provided by Alliance of Genome Resources, Apr 2022]","Enables DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Involved in cellular response to oxidative stress; negative regulation of transcription by RNA polymerase II; and positive regulation of transcription by RNA polymerase II. Located in nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a transcriptional regulator and oncoprotein that may be involved in hematopoiesis, apoptosis, development, and cell differentiation and proliferation. The encoded protein can interact with CTBP1, SMAD3, CREBBP, KAT2B, MAPK8, and MAPK9. This gene can undergo translocation with the AML1 gene, resulting in overexpression of this gene and the onset of leukemia. Several transcript variants encoding a few different isoforms have been found for this gene.","This gene encodes a multifunctional protein that is involved in various cellular processes, including gene expression, cell signaling, and RNA processing and transport. The protein includes an N-terminal transcriptional activation domain and a C-terminal RNA-binding domain. Chromosomal translocations between this gene and various genes encoding transcription factors result in the production of chimeric proteins that are involved in tumorigenesis. These chimeric proteins usually consist of the N-terminal transcriptional activation domain of this protein fused to the C-terminal DNA-binding domain of the transcription factor protein. Mutations in this gene, specifically a t(11;22)(q24;q12) translocation, are known to cause Ewing sarcoma as well as neuroectodermal and various other tumors. Alternative splicing of this gene results in multiple transcript variants. Related pseudogenes have been identified on chromosomes 1 and 14.","This gene encodes a member of the Polycomb-group (PcG) family. PcG family members form multimeric protein complexes, which are involved in maintaining the transcriptional repressive state of genes over successive cell generations. This protein associates with the embryonic ectoderm development protein, the VAV1 oncoprotein, and the X-linked nuclear protein. This protein may play a role in the hematopoietic and central nervous systems. Multiple alternatively splcied transcript variants encoding distinct isoforms have been identified for this gene.","The protein encoded by this gene is a low affinity receptor for the Fc region of immunoglobulin gamma complexes. The encoded protein is involved in the phagocytosis of immune complexes and in the regulation of antibody production by B-cells. Variations in this gene may increase susceptibilty to systemic lupus erythematosus (SLE). Several transcript variants encoding different isoforms have been found for this gene.","This gene encodes the human cellular counterpart of a feline sarcoma retrovirus protein with transforming capabilities. The gene product has tyrosine-specific protein kinase activity and that activity is required for maintenance of cellular transformation. Its chromosomal location has linked it to a specific translocation event identified in patients with acute promyelocytic leukemia but it is also involved in normal hematopoiesis as well as growth factor and cytokine receptor signaling. Alternative splicing results in multiple variants encoding different isoforms.[provided by RefSeq, Jan 2009].","The protein encoded by this gene is a member of the fibroblast growth factor receptor (FGFR) family, where amino acid sequence is highly conserved between members and throughout evolution. FGFR family members differ from one another in their ligand affinities and tissue distribution. A full-length representative protein consists of an extracellular region, composed of three immunoglobulin-like domains, a single hydrophobic membrane-spanning segment and a cytoplasmic tyrosine kinase domain. The extracellular portion of the protein interacts with fibroblast growth factors, setting in motion a cascade of downstream signals, ultimately influencing mitogenesis and differentiation. This particular family member binds both acidic and basic fibroblast growth factors and is involved in limb induction. Mutations in this gene have been associated with Pfeiffer syndrome, Jackson-Weiss syndrome, Antley-Bixler syndrome, osteoglophonic dysplasia, and autosomal dominant Kallmann syndrome 2. Chromosomal aberrations involving this gene are associated with stem cell myeloproliferative disorder and stem cell leukemia lymphoma syndrome. Alternatively spliced variants which encode different protein isoforms have been described; however, not all variants have been fully characterized.","This gene encodes a member of the fibroblast growth factor receptor (FGFR) family, with its amino acid sequence being highly conserved between members and among divergent species. FGFR family members differ from one another in their ligand affinities and tissue distribution. A full-length representative protein would consist of an extracellular region, composed of three immunoglobulin-like domains, a single hydrophobic membrane-spanning segment and a cytoplasmic tyrosine kinase domain. The extracellular portion of the protein interacts with fibroblast growth factors, setting in motion a cascade of downstream signals, ultimately influencing mitogenesis and differentiation. This particular family member binds acidic and basic fibroblast growth hormone and plays a role in bone development and maintenance. Mutations in this gene lead to craniosynostosis and multiple types of skeletal dysplasia.","The protein encoded by this gene is a member of the fibroblast growth factor receptor family, where amino acid sequence is highly conserved between members and throughout evolution. FGFR family members differ from one another in their ligand affinities and tissue distribution. A full-length representative protein consists of an extracellular region, composed of three immunoglobulin-like domains, a single hydrophobic membrane-spanning segment and a cytoplasmic tyrosine kinase domain. The extracellular portion of the protein interacts with fibroblast growth factors, setting in motion a cascade of downstream signals, ultimately influencing mitogenesis and differentiation. This particular family member is a high-affinity receptor for acidic, basic and/or keratinocyte growth factor, depending on the isoform. Mutations in this gene are associated with Crouzon syndrome, Pfeiffer syndrome, Craniosynostosis, Apert syndrome, Jackson-Weiss syndrome, Beare-Stevenson cutis gyrata syndrome, Saethre-Chotzen syndrome, and syndromic craniosynostosis. Multiple alternatively spliced transcript variants encoding different isoforms have been noted for this gene.","The protein encoded by this gene is a tyrosine kinase and cell surface receptor for fibroblast growth factors. The encoded protein is involved in the regulation of several pathways, including cell proliferation, cell differentiation, cell migration, lipid metabolism, bile acid biosynthesis, vitamin D metabolism, glucose uptake, and phosphate homeostasis. This protein consists of an extracellular region, composed of three immunoglobulin-like domains, a single hydrophobic membrane-spanning segment, and a cytoplasmic tyrosine kinase domain. The extracellular portion interacts with fibroblast growth factors, setting in motion a cascade of downstream signals, ultimately influencing mitogenesis and differentiation.","This gene belongs to the forkhead family of transcription factors which are characterized by a distinct forkhead domain.  The specific function of this gene has not yet been determined; however, it may play a role in myogenic growth and differentiation. Translocation of this gene with PAX3 has been associated with alveolar rhabdomyosarcoma.","This gene belongs to the forkhead family of transcription factors which are characterized by a distinct forkhead domain. This gene likely functions as a trigger for apoptosis through expression of genes necessary for cell death. Translocation of this gene with the MLL gene is associated with secondary acute leukemia. Alternatively spliced transcript variants encoding the same protein have been observed.","This gene encodes a transcription factor containing an ETS DNA-binding domain. The gene can undergo a t(11;22)(q24;q12) translocation with the Ewing sarcoma gene on chromosome 22, which results in a fusion gene that is present in the majority of Ewing sarcoma cases. An acute lymphoblastic leukemia-associated t(4;11)(q21;q23) translocation involving this gene has also been identified. Alternative splicing results in multiple transcript variants.","This gene encodes a class III receptor tyrosine kinase that regulates hematopoiesis. This receptor is activated by binding of the fms-related tyrosine kinase 3 ligand to the extracellular domain, which induces homodimer formation in the plasma membrane leading to autophosphorylation of the receptor. The activated receptor kinase subsequently phosphorylates and activates multiple cytoplasmic effector molecules in pathways involved in apoptosis, proliferation, and differentiation of hematopoietic cells in bone marrow. Mutations that result in the constitutive activation of this receptor result in acute myeloid leukemia and acute lymphoblastic leukemia.","This gene encodes a tyrosine kinase receptor for vascular endothelial growth factors C and D. The protein is thought to be involved in lymphangiogenesis and maintenance of the lymphatic endothelium. Mutations in this gene cause hereditary lymphedema type IA.","The protein encoded by this gene belongs to a family of phosphatidylinositol kinase-related kinases. These kinases mediate cellular responses to stresses such as DNA damage and nutrient deprivation. This kinase is a component of two distinct complexes, mTORC1, which controls protein synthesis, cell growth and proliferation, and mTORC2, which is a regulator of the actin cytoskeleton, and promotes cell survival and cell cycle progression. This protein acts as the target for the cell-cycle arrest and immunosuppressive effects of the FKBP12-rapamycin complex. Inhibitors of mTOR are used in organ transplants as immunosuppressants, and are being evaluated for their therapeutic potential in SARS-CoV-2 infections. Mutations in this gene are associated with Smith-Kingsmore syndrome and somatic focal cortical dysplasia type II. The ANGPTL7 gene is located in an intron of this gene.","This gene encodes a protein which belongs to the GATA family of transcription factors. The protein plays an important role in erythroid development by regulating the switch of fetal hemoglobin to adult hemoglobin. Mutations in this gene have been associated with X-linked dyserythropoietic anemia and thrombocytopenia.","This gene encodes a member of the GATA family of zinc-finger transcription factors that are named for the consensus nucleotide sequence they bind in the promoter regions of target genes. The encoded protein plays an essential role in regulating transcription of genes involved in the development and proliferation of hematopoietic and endocrine cell lineages. Alternative splicing results in multiple transcript variants.[provided by RefSeq, Mar 2009].","This gene encodes a protein which belongs to the GATA family of transcription factors. The protein contains two GATA-type zinc fingers and is an important regulator of T-cell development and plays an important role in endothelial cell biology. Defects in this gene are the cause of hypoparathyroidism with sensorineural deafness and renal dysplasia.","Cell surface heparan sulfate proteoglycans are composed of a membrane-associated protein core substituted with a variable number of heparan sulfate chains. Members of the glypican-related integral membrane proteoglycan family (GRIPS) contain a core protein anchored to the cytoplasmic membrane via a glycosyl phosphatidylinositol linkage. These proteins may play a role in the control of cell division and growth regulation. The protein encoded by this gene can bind to and inhibit the dipeptidyl peptidase activity of CD26, and it can induce apoptosis in certain cell types. Deletion mutations in this gene are associated with Simpson-Golabi-Behmel syndrome, also known as Simpson dysmorphia syndrome. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the Kruppel family of zinc finger proteins. The encoded transcription factor is activated by the sonic hedgehog signal transduction cascade and regulates stem cell proliferation. The activity and nuclear localization of this protein is negatively regulated by p53 in an inhibitory loop. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene belongs to the family of guanine nucleotide-binding proteins (G proteins), which function as modulators or transducers in various transmembrane signaling systems. G proteins are composed of 3 units: alpha, beta and gamma. This gene encodes one of the alpha subunits (subunit alpha-11). Mutations in this gene have been associated with hypocalciuric hypercalcemia type II (HHC2) and hypocalcemia dominant 2 (HYPOC2). Patients with HHC2 and HYPOC2 exhibit decreased or increased sensitivity, respectively, to changes in extracellular calcium concentrations.","This locus encodes a guanine nucleotide-binding protein. The encoded protein, an alpha subunit in the Gq class, couples a seven-transmembrane domain receptor to activation of phospolipase C-beta. Mutations at this locus have been associated with problems in platelet activation and aggregation. A related pseudogene exists on chromosome 2.[provided by RefSeq, Nov 2010].","This locus has a highly complex imprinted expression pattern. It gives rise to maternally, paternally, and biallelically expressed transcripts that are derived from four alternative promoters and 5' exons. Some transcripts contain a differentially methylated region (DMR) at their 5' exons, and this DMR is commonly found in imprinted genes and correlates with transcript expression. An antisense transcript is produced from an overlapping locus on the opposite strand. One of the transcripts produced from this locus, and the antisense transcript, are paternally expressed noncoding RNAs, and may regulate imprinting in this region. In addition, one of the transcripts contains a second overlapping ORF, which encodes a structurally unrelated protein - Alex. Alternative splicing of downstream exons is also observed, which results in different forms of the stimulatory G-protein alpha subunit, a key element of the classical signal transduction pathway linking receptor-ligand interactions with the activation of adenylyl cyclase and a variety of cellular reponses. Multiple transcript variants encoding different isoforms have been found for this gene. Mutations in this gene result in pseudohypoparathyroidism type 1a, pseudohypoparathyroidism type 1b, Albright hereditary osteodystrophy, pseudopseudohypoparathyroidism, McCune-Albright syndrome, progressive osseus heteroplasia, polyostotic fibrous dysplasia of bone, and some pituitary tumors.","L-glutamate is the major excitatory neurotransmitter in the central nervous system and activates both ionotropic and metabotropic glutamate receptors. Glutamatergic neurotransmission is involved in most aspects of normal brain function and can be perturbed in many neuropathologic conditions. The metabotropic glutamate receptors are a family of G protein-coupled receptors, that have been divided into 3 groups on the basis of sequence homology, putative signal transduction mechanisms, and pharmacologic properties. Group I includes GRM1 and GRM5 and these receptors have been shown to activate phospholipase C. Group II includes GRM2 and GRM3 while Group III includes GRM4, GRM6, GRM7 and GRM8. Group II and III receptors are linked to the inhibition of the cyclic AMP cascade but differ in their agonist selectivities.","Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Two molecules of each of the four core histones (H2A, H2B, H3, and H4) form an octamer, around which approximately 146 bp of DNA is wrapped in repeating units, called nucleosomes. The linker histone, H1, interacts with linker DNA between nucleosomes and functions in the compaction of chromatin into higher order structures. This gene contains introns and its mRNA is polyadenylated, unlike most histone genes. The protein encoded is a replication-independent member of the histone H3 family.","Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Two molecules of each of the four core histones (H2A, H2B, H3, and H4) form an octamer, around which approximately 146 bp of DNA is wrapped in repeating units, called nucleosomes. The linker histone, H1, interacts with linker DNA between nucleosomes and functions in the compaction of chromatin into higher order structures. This gene contains introns and its mRNA is polyadenylated, unlike most histone genes. The protein encoded by this gene is a replication-independent histone that is a member of the histone H3 family. Pseudogenes of this gene have been identified on the X chromosome, and on chromosomes 5, 13 and 17.","This gene encodes the alpha subunit of transcription factor hypoxia-inducible factor-1 (HIF-1), which is a heterodimer composed of an alpha and a beta subunit. HIF-1 functions as a master regulator of cellular and systemic homeostatic response to hypoxia by activating transcription of many genes, including those involved in energy metabolism, angiogenesis, apoptosis, and other genes whose protein products increase oxygen delivery or facilitate metabolic adaptation to hypoxia. HIF-1 thus plays an essential role in embryonic vascularization, tumor angiogenesis and pathophysiology of ischemic disease. Alternatively spliced transcript variants encoding different isoforms have been identified for this gene.","The product of this gene is a membrane-associated protein that functions in clathrin-mediated endocytosis and protein trafficking within the cell. The encoded protein binds to the huntingtin protein in the brain; this interaction is lost in Huntington's disease. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the proline and acidic-rich (PAR) protein family, a subset of the bZIP transcription factors. The encoded protein forms homodimers or heterodimers with other PAR family members and binds sequence-specific promoter elements to activate transcription. Chromosomal translocations fusing portions of this gene with the E2A gene cause a subset of childhood B-lineage acute lymphoid leukemias. Alternatively spliced transcript variants have been described, but their biological validity has not been determined.","This gene encodes a chromatin-associated protein involved in the regulation of gene transcription, integration of retroviruses into chromosomes, and the metastatic progression of cancer cells. The encoded protein preferentially binds to the minor groove of AT-rich regions in double-stranded DNA. Multiple transcript variants encoding different isoforms have been found for this gene. Pseudogenes of this gene have been identified on multiple chromosomes.","This gene encodes a member of the forkhead class of DNA-binding proteins. These hepatocyte nuclear factors are transcriptional activators for liver-specific transcripts such as albumin and transthyretin, and they also interact with chromatin. Similar family members in mice have roles in the regulation of metabolism and in the differentiation of the pancreas and liver.","This gene belongs to the A/B subfamily of ubiquitously expressed heterogeneous nuclear ribonucleoproteins (hnRNPs). The hnRNPs are RNA binding proteins and they complex with heterogeneous nuclear RNA (hnRNA). These proteins are associated with pre-mRNAs in the nucleus and appear to influence pre-mRNA processing and other aspects of mRNA metabolism and transport. While all of the hnRNPs are present in the nucleus, some seem to shuttle between the nucleus and the cytoplasm. The hnRNP proteins have distinct nucleic acid binding properties. The protein encoded by this gene has two repeats of quasi-RRM domains that bind to RNAs. This gene has been described to generate two alternatively spliced transcript variants which encode different isoforms.","This gene encodes a nuclear transcription factor that belongs to the NK-linked or NK-like (NKL) subfamily of homeobox genes. The encoded protein is required for normal development of the spleen during embryogenesis. This protein is also involved in specification of neuronal cell fates. Ectopic expression of this gene due to chromosomal translocations is associated with certain T-cell acute lymphoblastic leukemias. Alternate splicing results in multiple transcript variants.","In vertebrates, the genes encoding the class of transcription factors called homeobox genes are found in clusters named A, B, C, and D on four separate chromosomes. Expression of these proteins is spatially and temporally regulated during embryonic development. This gene is part of the A cluster on chromosome 7 and encodes a DNA-binding transcription factor which may regulate gene expression, morphogenesis, and differentiation. This gene is highly similar to the abdominal-B (Abd-B) gene of Drosophila. A specific translocation event which causes a fusion between this gene and the NUP98 gene has been associated with myeloid leukemogenesis. Read-through transcription exists between this gene and the upstream homeobox A10 (HOXA10) gene.[provided by RefSeq, Mar 2011].","In vertebrates, the genes encoding the class of transcription factors called homeobox genes are found in clusters named A, B, C, and D on four separate chromosomes. Expression of these proteins is spatially and temporally regulated during embryonic development. This gene is part of the A cluster on chromosome 7 and encodes a DNA-binding transcription factor which may regulate gene expression, morphogenesis, and differentiation. This gene is involved in the regulation of uterine development and is required for female fertility. Mutations in this gene can cause radio-ulnar synostosis with amegakaryocytic thrombocytopenia.","In vertebrates, the genes encoding the class of transcription factors called homeobox genes are found in clusters named A, B, C, and D on four separate chromosomes. Expression of these proteins is spatially and temporally regulated during embryonic development. This gene is part of the A cluster on chromosome 7 and encodes a DNA-binding transcription factor which may regulate gene expression, morphogenesis, and differentiation. Expansion of a polyalanine tract in the encoded protein can cause hand-foot-uterus syndrome, also known as hand-foot-genital syndrome.","This gene belongs to the homeobox family of genes. The homeobox genes encode a highly conserved family of transcription factors that play an important role in morphogenesis in all multicellular organisms. Mammals possess four similar homeobox gene clusters, HOXA, HOXB, HOXC and HOXD, which are located on different chromosomes and consist of 9 to 11 genes arranged in tandem. This gene is one of several homeobox HOXC genes located in a cluster on chromosome 12. The product of this gene binds to a promoter element of the lactase-phlorizin hydrolase. It also may play a role in early intestinal development. An alternatively spliced variant encoding a shorter isoform has been described but its full-length nature has not been determined.","This gene belongs to the homeobox family of genes. The homeobox genes encode a highly conserved family of transcription factors that play an important role in morphogenesis in all multicellular organisms. Mammals possess four similar homeobox gene clusters, HOXA, HOXB, HOXC and HOXD, which are located on different chromosomes and consist of 9 to 11 genes arranged in tandem. This gene is one of several homeobox HOXC genes located in a cluster on chromosome 12. The product of this gene may play a role in the development of hair, nail, and filiform papilla.","This gene belongs to the homeobox family of genes. The homeobox genes encode a highly conserved family of transcription factors that play an important role in morphogenesis in all multicellular organisms. Mammals possess four similar homeobox gene clusters, HOXA, HOXB, HOXC and HOXD, located on different chromosomes, consisting of 9 to 11 genes arranged in tandem. This gene is one of several homeobox HOXD genes located in a cluster on chromosome 2. Deletions that remove the entire HOXD gene cluster or the 5' end of this cluster have been associated with severe limb and genital abnormalities. The product of the mouse Hoxd11 gene plays a role in forelimb morphogenesis.","This gene belongs to the homeobox family of genes. The homeobox genes encode a highly conserved family of transcription factors that play an important role in morphogenesis in all multicellular organisms. Mammals possess four similar homeobox gene clusters, HOXA, HOXB, HOXC and HOXD, located on different chromosomes, consisting of 9 to 11 genes arranged in tandem. This gene is one of several homeobox HOXD genes located in a cluster on chromosome 2. Deletions that remove the entire HOXD gene cluster or the 5' end of this cluster have been associated with severe limb and genital abnormalities. Mutations in this particular gene cause synpolydactyly.","This gene belongs to the Ras oncogene family, whose members are related to the transforming genes of mammalian sarcoma retroviruses. The products encoded by these genes function in signal transduction pathways. These proteins can bind GTP and GDP, and they have intrinsic GTPase activity. This protein undergoes a continuous cycle of de- and re-palmitoylation, which regulates its rapid exchange between the plasma membrane and the Golgi apparatus. Mutations in this gene cause Costello syndrome, a disease characterized by increased growth at the prenatal stage, growth deficiency at the postnatal stage, predisposition to tumor formation, cognitive disability, skin and musculoskeletal abnormalities, distinctive facial appearance and cardiovascular abnormalities. Defects in this gene are implicated in a variety of cancers, including bladder cancer, follicular thyroid cancer, and oral squamous cell carcinoma. Multiple transcript variants, which encode different isoforms, have been identified for this gene.","This gene encodes an extracellular matrix protein with a spatially and temporally restricted tissue distribution. This protein is homohexameric with disulfide-linked subunits, and contains multiple EGF-like and fibronectin type-III domains. It is implicated in guidance of migrating neurons as well as axons during development, synaptic plasticity, and neuronal regeneration.","Isocitrate dehydrogenases catalyze the oxidative decarboxylation of isocitrate to 2-oxoglutarate. These enzymes belong to two distinct subclasses, one of which utilizes NAD(+) as the electron acceptor and the other NADP(+). Five isocitrate dehydrogenases have been reported: three NAD(+)-dependent isocitrate dehydrogenases, which localize to the mitochondrial matrix, and two NADP(+)-dependent isocitrate dehydrogenases, one of which is mitochondrial and the other predominantly cytosolic. Each NADP(+)-dependent isozyme is a homodimer. The protein encoded by this gene is the NADP(+)-dependent isocitrate dehydrogenase found in the cytoplasm and peroxisomes. It contains the PTS-1 peroxisomal targeting signal sequence. The presence of this enzyme in peroxisomes suggests roles in the regeneration of NADPH for intraperoxisomal reductions, such as the conversion of 2, 4-dienoyl-CoAs to 3-enoyl-CoAs, as well as in peroxisomal reactions that consume 2-oxoglutarate, namely the alpha-hydroxylation of phytanic acid. The cytoplasmic enzyme serves a significant role in cytoplasmic NADPH production. Alternatively spliced transcript variants encoding the same protein have been found for this gene.","Isocitrate dehydrogenases catalyze the oxidative decarboxylation of isocitrate to 2-oxoglutarate. These enzymes belong to two distinct subclasses, one of which utilizes NAD(+) as the electron acceptor and the other NADP(+). Five isocitrate dehydrogenases have been reported: three NAD(+)-dependent isocitrate dehydrogenases, which localize to the mitochondrial matrix, and two NADP(+)-dependent isocitrate dehydrogenases, one of which is mitochondrial and the other predominantly cytosolic. Each NADP(+)-dependent isozyme is a homodimer. The protein encoded by this gene is the NADP(+)-dependent isocitrate dehydrogenase found in the mitochondria. It plays a role in intermediary metabolism and energy production. This protein may tightly associate or interact with the pyruvate dehydrogenase complex. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene phosphorylates the inhibitor in the inhibitor/NF-kappa-B complex, causing dissociation of the inhibitor and activation of NF-kappa-B. The encoded protein itself is found in a complex of proteins. Several transcript variants, some protein-coding and some not, have been found for this gene.","The protein encoded by this gene is a signal transducer shared by many cytokines, including interleukin 6 (IL6), ciliary neurotrophic factor (CNTF), leukemia inhibitory factor (LIF), and oncostatin M (OSM). This protein functions as a part of the cytokine receptor complex. The activation of this protein is dependent upon the binding of cytokines to their receptors. vIL6, a protein related to IL6 and encoded by the Kaposi sarcoma-associated herpesvirus, can bypass the interleukin 6 receptor (IL6R) and directly activate this protein. Knockout studies in mice suggest that this gene plays a critical role in regulating myocyte apoptosis. Alternatively spliced transcript variants have been described. A related pseudogene has been identified on chromosome 17.","The protein encoded by this gene is a receptor for interleukin 7 (IL7). The function of this receptor requires the interleukin 2 receptor, gamma chain (IL2RG), which is a common gamma chain shared by the receptors of various cytokines, including interleukins 2, 4, 7, 9, and 15. This protein has been shown to play a critical role in V(D)J recombination during lymphocyte development. Defects in this gene may be associated with severe combined immunodeficiency (SCID). Alternatively spliced transcript variants have been found.","The protein encoded by this gene belongs to the IRF (interferon regulatory factor) family of transcription factors, characterized by an unique tryptophan pentad repeat DNA-binding domain. The IRFs are important in the regulation of interferons in response to infection by virus, and in the regulation of interferon-inducible genes. This family member is lymphocyte specific and negatively regulates Toll-like-receptor (TLR) signaling that is central to the activation of innate and adaptive immune systems. A chromosomal translocation involving this gene and the IgH locus, t(6;14)(p25;q32), may be a cause of multiple myeloma. Alternatively spliced transcript variants have been found for this gene.","This gene encodes a membrane protein that is a member of a class of protein-tyrosine kinases (PTK) characterized by the presence of a second phosphotransferase-related domain immediately N-terminal to the PTK domain. The encoded kinase phosphorylates STAT proteins (signal transducers and activators of transcription) and plays a key role in interferon-alpha/beta, interferon-gamma, and cytokine signal transduction. This gene plays a crucial role in effecting the expression of genes that mediate inflammation, epithelial remodeling, and metastatic cancer progression. This gene is a key component of the interleukin-6 (IL-6)/JAK1/STAT3 immune and inflammation response and is a therapeutic target for alleviating cytokine storms. The kinase activity of this gene is directly inhibited by the suppressor of cytokine signalling 1 (SOCS1) protein. Alternative splicing results in multiple transcript variants.","This gene encodes a non-receptor tyrosine kinase that plays a central role in cytokine and growth factor signalling. The primary isoform of this protein has an N-terminal FERM domain that is required for erythropoietin receptor association, an SH2 domain that binds STAT transcription factors, a pseudokinase domain and a C-terminal tyrosine kinase domain. Cytokine binding induces autophosphorylation and activation of this kinase. This kinase then recruits and phosphorylates signal transducer and activator of transcription (STAT) proteins. Growth factors like TGF-beta 1 also induce phosphorylation and activation of this kinase and translocation of downstream STAT proteins to the nucleus where they influence gene transcription. Mutations in this gene are associated with numerous inflammatory diseases and malignancies. This gene is a downstream target of the pleiotropic cytokine IL6 that is produced by B cells, T cells, dendritic cells and macrophages to produce an immune response or inflammation. Disregulation of the IL6/JAK2/STAT3 signalling pathways produces increased cellular proliferation and myeloproliferative neoplasms of hematopoietic stem cells. A nonsynonymous mutation in the pseudokinase domain of this gene disrupts the domains inhibitory effect and results in constitutive tyrosine phosphorylation activity and hypersensitivity to cytokine signalling. This gene and the IL6/JAK2/STAT3 signalling pathway is a therapeutic target for the treatment of excessive inflammatory responses to viral infections. Alternative splicing results in multiple transcript variants encoding distinct isoforms.","The protein encoded by this gene is a member of the Janus kinase (JAK) family of tyrosine kinases involved in cytokine receptor-mediated intracellular signal transduction. It is predominantly expressed in immune cells and transduces a signal in response to its activation via tyrosine phosphorylation by interleukin receptors. Mutations in this gene are associated with autosomal SCID (severe combined immunodeficiency disease).","This gene is the putative transforming gene of avian sarcoma virus 17. It encodes a protein which is highly similar to the viral protein, and which interacts directly with specific target DNA sequences to regulate gene expression. This gene is intronless and is mapped to 1p32-p31, a chromosomal region involved in both translocations and deletions in human malignancies.","This gene encodes an integral membrane protein which belongs to one of seven subfamilies of inward-rectifier potassium channel proteins called potassium channel subfamily J. The encoded protein is a subunit of the potassium channel which is homotetrameric. It is controlled by G-proteins and has a greater tendency to allow potassium to flow into a cell rather than out of a cell. Naturally occurring mutations in this gene are associated with aldosterone-producing adenomas.","Vascular endothelial growth factor (VEGF) is a major growth factor for endothelial cells. This gene encodes one of the two receptors of the VEGF. This receptor, known as kinase insert domain receptor, is a type III receptor tyrosine kinase. It functions as the main mediator of VEGF-induced endothelial proliferation, survival, migration, tubular morphogenesis and sprouting. The signalling and trafficking of this receptor are regulated by multiple factors, including Rab GTPase, P2Y purine nucleotide receptor, integrin alphaVbeta3, T-cell protein tyrosine phosphatase, etc.. Mutations of this gene are implicated in infantile capillary hemangiomas.","This gene encodes a receptor tyrosine kinase. This gene was initially identified as a homolog of the feline sarcoma viral oncogene v-kit and is often referred to as proto-oncogene c-Kit. The canonical form of this glycosylated transmembrane protein has an N-terminal extracellular region with five immunoglobulin-like domains, a transmembrane region, and an intracellular tyrosine kinase domain at the C-terminus. Upon activation by its cytokine ligand, stem cell factor (SCF), this protein phosphorylates multiple intracellular proteins that play a role in in the proliferation, differentiation, migration and apoptosis of many cell types and thereby plays an important role in hematopoiesis, stem cell maintenance, gametogenesis, melanogenesis, and in mast cell development, migration and function. This protein can be a membrane-bound or soluble protein. Mutations in this gene are associated with gastrointestinal stromal tumors, mast cell disease, acute myelogenous leukemia, and piebaldism. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene, a Kirsten ras oncogene homolog from the mammalian ras gene family, encodes a protein that is a member of the small GTPase superfamily. A single amino acid substitution is responsible for an activating mutation. The transforming protein that results is implicated in various malignancies, including lung adenocarcinoma, mucinous adenoma, ductal carcinoma of the pancreas and colorectal carcinoma. Alternative splicing leads to variants encoding two isoforms that differ in the C-terminal region.","This gene encodes a tissue-restricted nuclear transcriptional activator that is preferentially expressed in lymphoid tissue. Isolation of this protein initially defined a highly conserved LAF4/MLLT2 gene family of nuclear transcription factors that may function in lymphoid development and oncogenesis. In some ALL patients, this gene has been found fused to the gene for MLL. Multiple alternatively spliced transcript variants that encode different proteins have been found for this gene.","This gene is a member of the Src family of protein tyrosine kinases (PTKs). The encoded protein is a key signaling molecule in the selection and maturation of developing T-cells. It contains N-terminal sites for myristylation and palmitylation, a PTK domain, and SH2 and SH3 domains which are involved in mediating protein-protein interactions with phosphotyrosine-containing and proline-rich motifs, respectively. The protein localizes to the plasma membrane and pericentrosomal vesicles, and binds to cell surface receptors, including CD4 and CD8, and other signaling molecules. Multiple alternatively spliced variants encoding different isoforms have been described.","This locus encodes a transcriptional regulator that contains two cysteine-rich LIM domains but lacks a DNA-binding domain. LIM domains may play a role in protein interactions; thus the encoded protein may regulate transcription by competitively binding to specific DNA-binding transcription factors. Alterations at this locus have been associated with acute lymphoblastic T-cell leukemia. Chromosomal rearrangements have been observed between this locus and at least two loci, the delta subunit of the T-cell antigen receptor gene and the LIM domain binding 1 gene. Alternatively spliced transcript variants have been described.","LMO2 encodes a cysteine-rich, two LIM-domain protein that is required for yolk sac erythropoiesis. The LMO2 protein has a central and crucial role in hematopoietic development and is highly conserved. The LMO2 transcription start site is located approximately 25 kb downstream from the 11p13 T-cell translocation cluster (11p13 ttc), where a number T-cell acute lymphoblastic leukemia-specific translocations occur. Alternative splicing results in multiple transcript variants encoding different isoforms.[provided by RefSeq, Nov 2008].","This gene encodes a member of a subfamily of LIM domain proteins that are characterized by an N-terminal proline-rich region and three C-terminal LIM domains. The encoded protein localizes to the cell periphery in focal adhesions and may be involved in cell-cell adhesion and cell motility. This protein also shuttles through the nucleus and may function as a transcriptional co-activator. This gene is located at the junction of certain disease-related chromosomal translocations, which result in the expression of chimeric proteins that may promote tumor growth. Alternative splicing results in multiple transcript variants.","This gene represents a basic helix-loop-helix transcription factor. The encoded protein may play roles in blood vessel maturation and hematopoeisis. A translocation between this locus and the T cell receptor beta locus (GeneID 6957) on chromosome 7 has been associated with acute lymphoblastic leukemia.","The protein encoded by this gene is a DNA-binding, leucine zipper-containing transcription factor that acts as a homodimer or as a heterodimer. Depending on the binding site and binding partner, the encoded protein can be a transcriptional activator or repressor. This protein plays a role in the regulation of several cellular processes, including embryonic lens fiber cell development, increased T-cell susceptibility to apoptosis, and chondrocyte terminal differentiation. Defects in this gene are a cause of juvenile-onset pulverulent cataract as well as congenital cerulean cataract 4 (CCA4). Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a nuclear-localized E3 ubiquitin ligase. The encoded protein can promote tumor formation by targeting tumor suppressor proteins, such as p53, for proteasomal degradation. This gene is itself transcriptionally-regulated by p53. Overexpression or amplification of this locus is detected in a variety of different cancers. There is a pseudogene for this gene on chromosome 2. Alternative splicing results in a multitude of transcript variants, many of which may be expressed only in tumor cells.","This gene encodes a nuclear protein that contains a p53 binding domain at the N-terminus and a RING finger domain at the C-terminus, and shows structural similarity to p53-binding protein MDM2. Both proteins bind the p53 tumor suppressor protein and inhibit its activity, and have been shown to be overexpressed in a variety of human cancers. However, unlike MDM2 which degrades p53, this protein inhibits p53 by binding its transcriptional activation domain. This protein also interacts with MDM2 protein via the RING finger domain, and inhibits the latter's degradation. So this protein can reverse MDM2-targeted degradation of p53, while maintaining suppression of p53 transactivation and apoptotic functions. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.","The protein encoded by this gene is a serine/threonine kinase and is part of some signal transduction cascades, including the ERK and JNK kinase pathways as well as the NF-kappa-B pathway. The encoded protein is activated by autophosphorylation and requires magnesium as a cofactor in phosphorylating other proteins. This protein has E3 ligase activity conferred by a plant homeodomain (PHD) in its N-terminus and phospho-kinase activity conferred by a kinase domain in its C-terminus.","This gene encodes a member of the receptor tyrosine kinase family of proteins and the product of the proto-oncogene MET. The encoded preproprotein is proteolytically processed to generate alpha and beta subunits that are linked via disulfide bonds to form the mature receptor. Further processing of the beta subunit results in the formation of the M10 peptide, which has been shown to reduce lung fibrosis. Binding of its ligand, hepatocyte growth factor, induces dimerization and activation of the receptor, which plays a role in cellular survival, embryogenesis, and cellular migration and invasion. Mutations in this gene are associated with papillary renal cell carcinoma, hepatocellular carcinoma, and various head and neck cancers. Amplification and overexpression of this gene are also associated with multiple human cancers.","The protein encoded by this gene is a transcription factor that contains both basic helix-loop-helix and leucine zipper structural features. The encoded protein regulates melanocyte development and is responsible for pigment cell-specific transcription of the melanogenesis enzyme genes. Heterozygous mutations in the this gene cause auditory-pigmentary syndromes, such as Waardenburg syndrome type 2 and Tietz syndrome.","This gene encodes a transcriptional coactivator that plays an essential role in regulating gene expression during early development and hematopoiesis. The encoded protein contains multiple conserved functional domains. One of these domains, the SET domain, is responsible for its histone H3 lysine 4 (H3K4) methyltransferase activity which mediates chromatin modifications associated with epigenetic transcriptional activation. This protein is processed by the enzyme Taspase 1 into two fragments, MLL-C and MLL-N. These fragments reassociate and further assemble into different multiprotein complexes that regulate the transcription of specific target genes, including many of the HOX genes. Multiple chromosomal translocations involving this gene are the cause of certain acute lymphoid leukemias and acute myeloid leukemias. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Oct 2010].","This gene encodes a multi-domain protein involved in signaling and organization of cell junctions during embryogenesis. It has also been identified as the fusion partner of acute lymphoblastic leukemia (ALL-1) gene, involved in acute myeloid leukemias with t(6;11)(q27;q23) translocation. Alternatively spliced transcript variants encoding different isoforms have been described for this gene, however, not all have been fully characterized.[provided by RefSeq, May 2011].","This gene encodes a member of the O class of winged helix/forkhead transcription factor family. Proteins encoded by this class are regulated by factors involved in growth and differentiation indicating they play a role in these processes. A translocation involving this gene on chromosome X and the homolog of the Drosophila trithorax gene, encoding a DNA binding protein, located on chromosome 11 is associated with leukemia. Multiple transcript variants encoding different isoforms have been found for this gene.","Meningioma 1 (MN1) contains two sets of CAG repeats. It is disrupted by a balanced translocation (4;22) in a meningioma, and its inactivation may contribute to meningioma 32 pathogenesis.","In 1990 an oncogene, v-mpl, was identified from the murine myeloproliferative leukemia virus that was capable of immortalizing bone marrow hematopoietic cells from different lineages. In 1992 the human homologue, named, c-mpl, was cloned. Sequence data revealed that c-mpl encoded a protein that was homologous with members of the hematopoietic receptor superfamily. Presence of anti-sense oligodeoxynucleotides of c-mpl inhibited megakaryocyte colony formation. The ligand for c-mpl, thrombopoietin, was cloned in 1994. Thrombopoietin was shown to be the major regulator of megakaryocytopoiesis and platelet formation. The protein encoded by the c-mpl gene, CD110, is a 635 amino acid transmembrane domain, with two extracellular cytokine receptor domains and two intracellular cytokine receptor box motifs . TPO-R deficient mice were severely thrombocytopenic, emphasizing the important role of CD110 and thrombopoietin in megakaryocyte and platelet formation. Upon binding of thrombopoietin CD110 is dimerized and the JAK family of non-receptor tyrosine kinases, as well as the STAT family, the MAPK family, the adaptor protein Shc and the receptors themselves become tyrosine phosphorylated.","This gene was identified by involvement in some t(X;14) translocations associated with mature T-cell proliferations. This region has a complex gene structure, with a common promoter and 5' exon spliced to two different sets of 3' exons that encode two different proteins. This gene represents the upstream 13 kDa protein that is a member of the TCL1 family. This protein may be involved in leukemogenesis.","This gene encodes a protein with three HTH DNA-binding domains that functions as a transcription regulator. This protein plays an essential role in the regulation of hematopoiesis. This gene may be aberrently expressed or rearranged or undergo translocation in leukemias and lymphomas, and is considered to be an oncogene. Alternative splicing results in multiple transcript variants.","This gene is a proto-oncogene and encodes a nuclear phosphoprotein that plays a role in cell cycle progression, apoptosis and cellular transformation. The encoded protein forms a heterodimer with the related transcription factor MAX. This complex binds to the E box DNA consensus sequence and regulates the transcription of specific target genes. Amplification of this gene is frequently observed in numerous human cancers. Translocations involving this gene are associated with Burkitt lymphoma and multiple myeloma in human patients. There is evidence to show that translation initiates both from an upstream, in-frame non-AUG (CUG) and a downstream AUG start site, resulting in the production of two isoforms with distinct N-termini.","Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to act upstream of or within regulation of inner ear auditory receptor cell differentiation. Located in chromosome and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","This gene is a member of the MYC family and encodes a protein with a basic helix-loop-helix (bHLH) domain. This protein is located in the nucleus and must dimerize with another bHLH protein in order to bind DNA. Amplification of this gene is associated with a variety of tumors, most notably neuroblastomas. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene.","This gene encodes a cytosolic adapter protein that plays a central role in the innate and adaptive immune response. This protein functions as an essential signal transducer in the interleukin-1 and Toll-like receptor signaling pathways. These pathways regulate that activation of numerous proinflammatory genes. The encoded protein consists of an N-terminal death domain and a C-terminal Toll-interleukin1 receptor domain. Patients with defects in this gene have an increased susceptibility to pyogenic bacterial infections. Alternate splicing results in multiple transcript variants.","This gene encodes a nuclear protein that belongs to the basic helix-loop-helix family of transcription factors and the myogenic factors subfamily. It regulates muscle cell differentiation by inducing cell cycle arrest, a prerequisite for myogenic initiation. The protein is also involved in muscle regeneration. It activates its own transcription which may stabilize commitment to myogenesis.","This gene is a member of the nuclear factor of activated T cells (NFAT) family. The product of this gene is a DNA-binding protein with a REL-homology region (RHR) and an NFAT-homology region (NHR). This protein is present in the cytosol and only translocates to the nucleus upon T cell receptor (TCR) stimulation, where it becomes a member of the nuclear factors of activated T cells transcription complex. This complex plays a central role in inducing gene transcription during the immune response. Alternate transcriptional splice variants encoding different isoforms have been characterized.","This gene encodes a transcription factor which is a member of a small family of basic leucine zipper (bZIP) proteins. The encoded transcription factor regulates genes which contain antioxidant response elements (ARE) in their promoters; many of these genes encode proteins involved in response to injury and inflammation which includes the production of free radicals. Multiple transcript variants encoding different isoforms have been characterized for this gene.","This gene encodes a subunit of the transcription factor complex nuclear factor-kappa-B (NFkB). The NFkB complex is expressed in numerous cell types and functions as a central activator of genes involved in inflammation and immune function. The protein encoded by this gene can function as both a transcriptional activator or repressor depending on its dimerization partner. The p100 full-length protein is co-translationally processed into a p52 active form. Chromosomal rearrangements and translocations of this locus have been observed in B cell lymphomas, some of which may result in the formation of fusion proteins. There is a pseudogene for this gene on chromosome 18. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the NOTCH family of proteins. Members of this Type I transmembrane protein family share structural characteristics including an extracellular domain consisting of multiple epidermal growth factor-like (EGF) repeats, and an intracellular domain consisting of multiple different domain types. Notch signaling is an evolutionarily conserved intercellular signaling pathway that regulates interactions between physically adjacent cells through binding of Notch family receptors to their cognate ligands. The encoded preproprotein is proteolytically processed in the trans-Golgi network to generate two polypeptide chains that heterodimerize to form the mature cell-surface receptor. This receptor plays a role in the development of numerous cell and tissue types. Mutations in this gene are associated with aortic valve disease, Adams-Oliver syndrome, T-cell acute lymphoblastic leukemia, chronic lymphocytic leukemia, and head and neck squamous cell carcinoma.","This gene encodes a member of the Notch family. Members of this Type 1 transmembrane protein family share structural characteristics including an extracellular domain consisting of multiple epidermal growth factor-like (EGF) repeats, and an intracellular domain consisting of multiple, different domain types. Notch family members play a role in a variety of developmental processes by controlling cell fate decisions. The Notch signaling network is an evolutionarily conserved intercellular signaling pathway which regulates interactions between physically adjacent cells. In Drosophilia, notch interaction with its cell-bound ligands (delta, serrate) establishes an intercellular signaling pathway that plays a key role in development. Homologues of the notch-ligands have also been identified in human, but precise interactions between these ligands and the human notch homologues remain to be determined. This protein is cleaved in the trans-Golgi network, and presented on the cell surface as a heterodimer. This protein functions as a receptor for membrane bound ligands, and may play a role in vascular, renal and hepatic development. Two transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is involved in several cellular processes, including centrosome duplication, protein chaperoning, and cell proliferation. The encoded phosphoprotein shuttles between the nucleolus, nucleus, and cytoplasm, chaperoning ribosomal proteins and core histones from the nucleus to the cytoplasm. This protein is also known to sequester the tumor suppressor ARF in the nucleolus, protecting it from degradation until it is needed. Mutations in this gene are associated with acute myeloid leukemia. Dozens of pseudogenes of this gene have been identified.","This is an N-ras oncogene encoding a membrane protein that shuttles between the Golgi apparatus and the plasma membrane. This shuttling is regulated through palmitoylation and depalmitoylation by the ZDHHC9-GOLGA7 complex. The encoded protein, which has intrinsic GTPase activity, is activated by a guanine nucleotide-exchange factor and inactivated by a GTPase activating protein. Mutations in this gene have been associated with somatic rectal cancer, follicular thyroid cancer, autoimmune lymphoproliferative syndrome, Noonan syndrome, and juvenile myelomonocytic leukemia.","This gene encodes a member of the neurotrophic tyrosine kinase receptor (NTKR) family. This kinase is a membrane-bound receptor that, upon neurotrophin binding, phosphorylates itself and members of the MAPK pathway. The presence of this kinase leads to cell differentiation and may play a role in specifying sensory neuron subtypes. Mutations in this gene have been associated with congenital insensitivity to pain, anhidrosis, self-mutilating behavior, cognitive disability and cancer. Alternate transcriptional splice variants of this gene have been found, but only three have been characterized to date.","This gene encodes a member of the neurotrophic tyrosine receptor kinase (NTRK) family. This kinase is a membrane-bound receptor that, upon neurotrophin binding, phosphorylates itself and members of the MAPK pathway. Signalling through this kinase leads to cell differentiation and may play a role in the development of proprioceptive neurons that sense body position. Mutations in this gene have been associated with medulloblastomas, secretory breast carcinomas and other cancers. Several transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the discoidin domain receptor subclass of the receptor tyrosine kinase (RTKs) protein family. RTKs play a key role in the communication of cells with their microenvironment. The encoded protein is a collagen-induced receptor that activates signal transduction pathways involved in cell adhesion, proliferation, and extracellular matrix remodeling. This protein is expressed in numerous cell types and may alos be involved in wound repair and regulate tumor growth and invasiveness. Mutations in this gene are the cause of short limb-hand type spondylometaepiphyseal dysplasia.","Nuclear pore complexes (NPCs) regulate the transport of macromolecules between the nucleus and cytoplasm, and are composed of many polypeptide subunits, many of which belong to the nucleoporin family. This gene belongs to the nucleoporin gene family and encodes a 186 kDa precursor protein that undergoes autoproteolytic cleavage to generate a 98 kDa nucleoporin and 96 kDa nucleoporin. The 98 kDa nucleoporin contains a Gly-Leu-Phe-Gly (GLGF) repeat domain and participates in many cellular processes, including nuclear import, nuclear export, mitotic progression, and regulation of gene expression. The 96 kDa nucleoporin is a scaffold component of the NPC. Proteolytic cleavage is important for targeting of the proteins to the NPC. Translocations between this gene and many other partner genes have been observed in different leukemias. Rearrangements typically result in chimeras with the N-terminal GLGF domain of this gene to the C-terminus of the partner gene. Alternative splicing results in multiple transcript variants encoding different isoforms, at least two of which are proteolytically processed. Some variants lack the region that encodes the 96 kDa nucleoporin.","This gene is a member of the paired box (PAX) family of transcription factors. Members of the PAX family typically contain a paired box domain and a paired-type homeodomain. These genes play critical roles during fetal development. Mutations in paired box gene 3 are associated with Waardenburg syndrome, craniofacial-deafness-hand syndrome, and alveolar rhabdomyosarcoma. The translocation t(2;13)(q35;q14), which represents a fusion between PAX3 and the forkhead gene, is a frequent finding in alveolar rhabdomyosarcoma. Alternative splicing results in transcripts encoding isoforms with different C-termini.","This gene encodes a member of the paired box (PAX) family of transcription factors. The central feature of this gene family is a novel, highly conserved DNA-binding motif, known as the paired box. Paired box transcription factors are important regulators in early development, and alterations in the expression of their genes are thought to contribute to neoplastic transformation. This gene encodes the B-cell lineage specific activator protein that is expressed at early, but not late stages of B-cell differentiation. Its expression has also been detected in developing CNS and testis and so the encoded protein may also play a role in neural development and spermatogenesis. This gene is located at 9p13, which is involved in t(9;14)(p13;q32) translocations recurring in small lymphocytic lymphomas of the plasmacytoid subtype, and in derived large-cell lymphomas. This translocation brings the potent E-mu enhancer of the IgH gene into close proximity of the PAX5 promoter, suggesting that the deregulation of transcription of this gene contributes to the pathogenesis of these lymphomas. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a nuclear protein that belongs to the PBX homeobox family of transcriptional factors. Studies in mice suggest that this gene may be involved in the regulation of osteogenesis and required for skeletal patterning and programming. A chromosomal translocation, t(1;19) involving this gene and TCF3/E2A gene, is associated with pre-B-cell acute lymphoblastic leukemia. The resulting fusion protein, in which the DNA binding domain of E2A is replaced by the DNA binding domain of this protein, transforms cells by constitutively activating transcription of genes regulated by the PBX protein family. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the protein family comprised of both platelet-derived growth factors (PDGF) and vascular endothelial growth factors (VEGF). The encoded preproprotein is proteolytically processed to generate platelet-derived growth factor subunit B, which can homodimerize, or alternatively, heterodimerize with the related platelet-derived growth factor subunit A. These proteins bind and activate PDGF receptor tyrosine kinases, which play a role in a wide range of developmental processes. Mutations in this gene are associated with meningioma. Reciprocal translocations between chromosomes 22 and 17, at sites where this gene and that for collagen type 1, alpha 1 are located, are associated with dermatofibrosarcoma protuberans, a rare skin tumor. Alternative splicing results in multiple transcript variants.","This gene encodes a cell surface tyrosine kinase receptor for members of the platelet-derived growth factor family. These growth factors are mitogens for cells of mesenchymal origin. The identity of the growth factor bound to a receptor monomer determines whether the functional receptor is a homodimer or a heterodimer, composed of both platelet-derived growth factor receptor alpha and beta polypeptides. Studies suggest that this gene plays a role in organ development, wound healing, and tumor progression. Mutations in this gene have been associated with idiopathic hypereosinophilic syndrome, somatic and familial gastrointestinal stromal tumors, and a variety of other cancers.","The protein encoded by this gene is a cell surface tyrosine kinase receptor for members of the platelet-derived growth factor family. These growth factors are mitogens for cells of mesenchymal origin. The identity of the growth factor bound to a receptor monomer determines whether the functional receptor is a homodimer (PDGFB or PDGFD) or a heterodimer (PDGFA and PDGFB). This gene is essential for normal development of the cardiovascular system and aids in rearrangement of the actin cytoskeleton. This gene is flanked on chromosome 5 by the genes for granulocyte-macrophage colony-stimulating factor and macrophage-colony stimulating factor receptor; all three genes may be implicated in the 5-q syndrome. A translocation between chromosomes 5 and 12, that fuses this gene to that of the ETV6 gene, results in chronic myeloproliferative disorder with eosinophilia.","Phosphatidylinositol 3-kinase is composed of an 85 kDa regulatory subunit and a 110 kDa catalytic subunit. The protein encoded by this gene represents the catalytic subunit, which uses ATP to phosphorylate PtdIns, PtdIns4P and PtdIns(4,5)P2. This gene has been found to be oncogenic and has been implicated in cervical cancers. A pseudogene of this gene has been defined on chromosome 22.","This gene encodes an isoform of the catalytic subunit of phosphoinositide 3-kinase (PI3K). These kinases are important in signaling pathways involving receptors on the outer membrane of eukaryotic cells and are named for their catalytic subunit. The encoded protein is the catalytic subunit for PI3Kbeta (PI3KB). PI3KB has been shown to be part of the activation pathway in neutrophils which have bound immune complexes at sites of injury or infection. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene belongs to the Ser/Thr protein kinase family, and PIM subfamily. This gene is expressed primarily in B-lymphoid and myeloid cell lines, and is overexpressed in hematopoietic malignancies and in prostate cancer. It plays a role in signal transduction in blood cells, contributing to both cell proliferation and survival, and thus provides a selective advantage in tumorigenesis. Both the human and orthologous mouse genes have been reported to encode two isoforms (with preferential cellular localization) resulting from the use of alternative in-frame translation initiation codons, the upstream non-AUG (CUG) and downstream AUG codons (PMIDs:16186805, 1825810).[provided by RefSeq, Aug 2011].","Phosphatidylinositol 3-kinase (PI3K) is a lipid kinase that phosphorylates phosphatidylinositol and similar compounds, creating second messengers important in growth signaling pathways. PI3K functions as a heterodimer of a regulatory and a catalytic subunit. The protein encoded by this gene is a regulatory component of PI3K. Three transcript variants, one protein coding and the other two non-protein coding, have been found for this gene.","Pleomorphic adenoma gene 1 encodes a zinc finger protein with 2 putative nuclear localization signals. PLAG1, which is developmentally regulated, has been shown to be consistently rearranged in pleomorphic adenomas of the salivary glands. PLAG1 is activated by the reciprocal chromosomal translocations involving 8q12 in a subset of salivary gland pleomorphic adenomas. Three transcript variants encoding two different isoforms have been found for this gene.","The protein encoded by this gene catalyzes the formation of inositol 1,4,5-trisphosphate and diacylglycerol from phosphatidylinositol 4,5-bisphosphate. This reaction uses calcium as a cofactor and plays an important role in the intracellular transduction of receptor-mediated tyrosine kinase activators. For example, when activated by SRC, the encoded protein causes the Ras guanine nucleotide exchange factor RasGRP1 to translocate to the Golgi, where it activates Ras. Also, this protein has been shown to be a major substrate for heparin-binding growth factor 1 (acidic fibroblast growth factor)-activated tyrosine kinase. Two transcript variants encoding different isoforms have been found for this gene.","Enables transcription coactivator activity. Involved in positive regulation of transcription by RNA polymerase II. Part of RNA polymerase II transcription regulator complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a transcription factor containing a POU homeodomain that plays a key role in embryonic development and stem cell pluripotency. Aberrant expression of this gene in adult tissues is associated with tumorigenesis. This gene can participate in a translocation with the Ewing's sarcoma gene on chromosome 21, which also leads to tumor formation. Alternative splicing, as well as usage of alternative AUG and non-AUG translation initiation codons, results in multiple isoforms. One of the AUG start codons is polymorphic in human populations. Related pseudogenes have been identified on chromosomes 1, 3, 8, 10, and 12.","This gene encodes a constant regulatory subunit of protein phosphatase 2. Protein phosphatase 2 is one of the four major Ser/Thr phosphatases, and it is implicated in the negative control of cell growth and division. It consists of a common heteromeric core enzyme, which is composed of a catalytic subunit and a constant regulatory subunit, that associates with a variety of regulatory subunits. The constant regulatory subunit A serves as a scaffolding molecule to coordinate the assembly of the catalytic subunit and a variable regulatory B subunit. This gene encodes an alpha isoform of the constant regulatory subunit A. Alternatively spliced transcript variants have been described.","This gene encodes one of the catalytic subunits of protein kinase A, which exists as a tetrameric holoenzyme with two regulatory subunits and two catalytic subunits, in its inactive form. cAMP causes the dissociation of the inactive holoenzyme into a dimer of regulatory subunits bound to four cAMP and two free monomeric catalytic subunits. Four different regulatory subunits and three catalytic subunits have been identified in humans. cAMP-dependent phosphorylation of proteins by protein kinase A is important to many cellular processes, including differentiation, proliferation, and apoptosis. Constitutive activation of this gene caused either by somatic mutations, or genomic duplications of regions that include this gene, have been associated with hyperplasias and adenomas of the adrenal cortex and are linked to corticotropin-independent Cushing's syndrome. Alternative splicing results in multiple transcript variants encoding different isoforms. Tissue-specific isoforms that differ at the N-terminus have been described, and these isoforms may differ in the post-translational modifications that occur at the N-terminus of some isoforms.","cAMP is a signaling molecule important for a variety of cellular functions. cAMP exerts its effects by activating the cAMP-dependent protein kinase, which transduces the signal through phosphorylation of different target proteins. The inactive kinase holoenzyme is a tetramer composed of two regulatory and two catalytic subunits. cAMP causes the dissociation of the inactive holoenzyme into a dimer of regulatory subunits bound to four cAMP and two free monomeric catalytic subunits. Four different regulatory subunits and three catalytic subunits have been identified in humans. This gene encodes one of the regulatory subunits. This protein was found to be a tissue-specific extinguisher that down-regulates the expression of seven liver genes in hepatoma x fibroblast hybrids. Mutations in this gene cause Carney complex (CNC). This gene can fuse to the RET protooncogene by gene rearrangement and form the thyroid tumor-specific chimeric oncogene known as PTC2. A nonconventional nuclear localization sequence (NLS) has been found for this protein which suggests a role in DNA replication via the protein serving as a nuclear transport protein for the second subunit of the Replication Factor C (RFC40). Several alternatively spliced transcript variants encoding two different isoforms have been observed.","This gene encodes a member of the MAP kinase family. MAP kinases, also known as extracellular signal-regulated kinases (ERKs), act as an integration point for multiple biochemical signals, and are involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation and development. The activation of this kinase requires its phosphorylation by upstream kinases. Upon activation, this kinase translocates to the nucleus of the stimulated cells, where it phosphorylates nuclear targets. One study also suggests that this protein acts as a transcriptional repressor independent of its kinase activity. The encoded protein has been identified as a moonlighting protein based on its ability to perform mechanistically distinct functions. Two alternatively spliced transcript variants encoding the same protein, but differing in the UTRs, have been reported for this gene.","The protein encoded by this gene is a member of the dual specificity protein kinase family, which acts as a mitogen-activated protein (MAP) kinase kinase. MAP kinases, also known as extracellular signal-regulated kinases (ERKs), act as an integration point for multiple biochemical signals. This protein kinase lies upstream of MAP kinases and stimulates the enzymatic activity of MAP kinases upon wide variety of extra- and intracellular signals. As an essential component of MAP kinase signal transduction pathway, this kinase is involved in many cellular processes such as proliferation, differentiation, transcription regulation and development.","The protein encoded by this gene is a dual specificity protein kinase that belongs to the MAP kinase kinase family. This kinase is known to play a critical role in mitogen growth factor signal transduction. It phosphorylates and thus activates MAPK1/ERK2 and MAPK2/ERK3. The activation of this kinase itself is dependent on the Ser/Thr phosphorylation by MAP kinase kinase kinases. Mutations in this gene cause cardiofaciocutaneous syndrome (CFC syndrome), a disease characterized by heart defects, cognitive disability, and distinctive facial features similar to those found in Noonan syndrome. The inhibition or degradation of this kinase is also found to be involved in the pathogenesis of Yersinia and anthrax. A pseudogene, which is located on chromosome 7, has been identified for this gene.","The protein encoded by this gene is a cytoplasmic nonreceptor protein kinase which may function as an intracellular signal transducer in epithelial tissues. Overexpression of this gene in mammary epithelial cells leads to sensitization of the cells to epidermal growth factor and results in a partially transformed phenotype. Expression of this gene has been detected at low levels in some breast tumors but not in normal breast tissue. The encoded protein has been shown to undergo autophosphorylation. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP contains two tandem Src homology-2 domains, which function as phospho-tyrosine binding domains and mediate the interaction of this PTP with its substrates. This PTP is widely expressed in most tissues and plays a regulatory role in various cell signaling events that are important for a diversity of cell functions, such as mitogenic activation, metabolic control, transcription regulation, and cell migration. Mutations in this gene are a cause of Noonan syndrome as well as acute myeloid leukemia.","The protein encoded by this gene is a GTPase which belongs to the RAS superfamily of small GTP-binding proteins. Members of this superfamily appear to regulate a diverse array of cellular events, including the control of cell growth, cytoskeletal reorganization, and the activation of protein kinases. Two transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is highly similar to the gene product of Schizosaccharomyces pombe rad21, a gene involved in the repair of DNA double-strand breaks, as well as in chromatid cohesion during mitosis. This protein is a nuclear phospho-protein, which becomes hyperphosphorylated in cell cycle M phase. The highly regulated association of this protein with mitotic chromatin specifically at the centromere region suggests its role in sister chromatid cohesion in mitotic cells.","This gene is the cellular homolog of viral raf gene (v-raf). The encoded protein is a MAP kinase kinase kinase (MAP3K), which functions downstream of the Ras family of membrane associated GTPases to which it binds directly. Once activated, the cellular RAF1 protein can phosphorylate to activate the dual specificity protein kinases MEK1 and MEK2, which in turn phosphorylate to activate the serine/threonine specific protein kinases, ERK1 and ERK2. Activated ERKs are pleiotropic effectors of cell physiology and play an important role in the control of gene expression involved in the cell division cycle, apoptosis, cell differentiation and cell migration. Mutations in this gene are associated with Noonan syndrome 5 and LEOPARD syndrome 2.","The smg GDP dissociation stimulator (smgGDS) protein is a stimulatory GDP/GTP exchange protein with GTPase activity (Riess et al., 1993).","This gene represents a nuclear retinoic acid receptor. The encoded protein, retinoic acid receptor alpha, regulates transcription in a ligand-dependent manner. This gene has been implicated in regulation of development, differentiation, apoptosis, granulopoeisis, and transcription of clock genes. Translocations between this locus and several other loci have been associated with acute promyelocytic leukemia. Alternatively spliced transcript variants have been found for this locus.[provided by RefSeq, Sep 2010].","This gene encodes a member of the Jumonji, AT-rich interactive domain 1 (JARID1) histone demethylase protein family. The encoded protein plays a role in gene regulation through the histone code by specifically demethylating lysine 4 of histone H3. The encoded protein interacts with many other proteins, including retinoblastoma protein, and is implicated in the transcriptional regulation of Hox genes and cytokines. This gene may play a role in tumor progression.","This gene encodes a protein that belongs to the Rel homology domain/immunoglobulin-like fold, plexin, transcription factor (RHD/IPT) family. Members of this family regulate genes involved in apoptosis, inflammation, the immune response, and oncogenic processes. This proto-oncogene plays a role in the survival and proliferation of B lymphocytes. Mutation or amplification of this gene is associated with B-cell lymphomas, including Hodgkin's lymphoma. Single nucleotide polymorphisms in this gene are associated with susceptibility to ulcerative colitis and rheumatoid arthritis. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a transmembrane receptor and member of the tyrosine protein kinase family of proteins. Binding of ligands such as GDNF (glial cell-line derived neurotrophic factor) and other related proteins to the encoded receptor stimulates receptor dimerization and activation of downstream signaling pathways that play a role in cell differentiation, growth, migration and survival. The encoded receptor is important in development of the nervous system, and the development of organs and tissues derived from the neural crest. This proto-oncogene can undergo oncogenic activation through both cytogenetic rearrangement and activating point mutations. Mutations in this gene are associated with Hirschsprung disease and central hypoventilation syndrome and have been identified in patients with renal agenesis.","This gene encodes a member of the tripartite motif (TRIM) family. The TRIM motif includes three zinc-binding domains, a RING, a B-box type 1 and a B-box type 2, and a coiled-coil region. This protein localizes to the nuclear matrix. It interacts with the enhancer of polycomb protein and represses gene transcription. It is also thought to be involved in the differentiation of male germ cells. Fusion of the N-terminus of this protein with the truncated C-terminus of the RET gene product has been shown to result in production of the ret transforming protein.","This gene encodes a member of a subfamily of Ras-related GTPases. The encoded protein is involved in regulating p38 MAPK-dependent signaling cascades related to cellular stress. This protein also cooperates with nerve growth factor to promote neuronal development and regeneration. Alternate splicing results in multiple transcript variants.","This proto-oncogene, highly-expressed in a variety of tumor cell lines, belongs to the sevenless subfamily of tyrosine kinase insulin receptor genes. The protein encoded by this gene is a type I integral membrane protein with tyrosine kinase activity. The protein may function as a growth or differentiation factor receptor.","This gene encodes a member of the mitogen-activated protein kinase (MAPK) family. Members of this family act as an integration point for multiple biochemical signals and are involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation, and development. They form a three-tiered signaling module composed of MAPKKKs, MAPKKs, and MAPKs. This protein is phosphorylated at serine and threonine residues by MAPKKKs and subsequently phosphorylates downstream MAPK targets at threonine and tyrosine residues. A similar protein in mouse has been reported to play a role in liver organogenesis. A pseudogene of this gene is located on the long arm of chromosome X. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene inhibits acetylation of nucleosomes, especially histone H4, by histone acetylases (HAT). This inhibition is most likely accomplished by masking histone lysines from being acetylated, and the consequence is to silence HAT-dependent transcription. The encoded protein is part of a complex localized to the endoplasmic reticulum but is found in the nucleus and inhibits apoptosis following attack by cytotoxic T lymphocytes. This protein can also enhance DNA replication of the adenovirus genome. Several transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a member of the serine/arginine (SR)-rich family of pre-mRNA splicing factors, which constitute part of the spliceosome. Each of these factors contains an RNA recognition motif (RRM) for binding RNA and an RS domain for binding other proteins. The RS domain is rich in serine and arginine residues and facilitates interaction between different SR splicing factors. In addition to being critical for mRNA splicing, the SR proteins have also been shown to be involved in mRNA export from the nucleus and in translation. Two transcript variants encoding the same protein and one non-coding transcript variant have been found for this gene. In addition, a pseudogene of this gene has been found on chromosome 11.","The protein encoded by this gene is a member of the serine/arginine (SR)-rich family of pre-mRNA splicing factors, which constitute part of the spliceosome. Each of these factors contains an RNA recognition motif (RRM) for binding RNA and an RS domain for binding other proteins. The RS domain is rich in serine and arginine residues and facilitates interaction between different SR splicing factors. In addition to being critical for mRNA splicing, the SR proteins have also been shown to be involved in mRNA export from the nucleus and in translation. Two transcript variants, one protein-coding and the other non-coding, have been found for this gene.","This gene encodes a serine/threonine protein kinase that plays an important role in cellular stress response. This kinase activates certain potassium, sodium, and chloride channels, suggesting an involvement in the regulation of processes such as cell survival, neuronal excitability, and renal sodium excretion. High levels of expression of this gene may contribute to conditions such as hypertension and diabetic nephropathy. Several alternatively spliced transcript variants encoding different isoforms have been noted for this gene.","This gene encodes a member of the endophilin family of Src homology 3 domain-containing proteins. The encoded protein is involved in endocytosis and may also play a role in the cell cycle. Overexpression of this gene may play a role in leukemogenesis, and the encoded protein has been implicated in acute myeloid leukemia as a fusion partner of the myeloid-lymphoid leukemia protein. Pseudogenes of this gene are located on the long arm of chromosomes 11 and 17. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","This gene encodes a cytoplasmic protein implicated in regulation of the mitotic spindle checkpoint, a regulatory pathway that monitors chromosome segregation during cell division to ensure the proper distribution of chromosomes to daughter cells. The protein is phosphorylated in mitosis and in response to activation of the spindle checkpoint, and disappears when cells transition to G1 phase. It interacts with a mitotic regulator, and its expression is required to efficiently activate the spindle checkpoint. It is proposed to regulate Cdc2 kinase activity during spindle checkpoint arrest. Chromosomal deletions that fuse this gene and the adjacent locus commonly occur in T cell leukemias, and are thought to arise through illegitimate V-(D)-J recombination events. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a homeobox protein that is similar to the Drosophila 'sine oculis' gene product. This gene is found in a cluster of related genes on chromosome 14 and is thought to be involved in limb development. Defects in this gene are a cause of autosomal dominant deafness type 23 (DFNA23) and branchiootic syndrome type 3 (BOS3).","This gene encodes the nuclear protooncogene protein homolog of avian sarcoma viral (v-ski) oncogene. It functions as a repressor of TGF-beta signaling, and may play a role in neural tube development and muscle differentiation.","The protein encoded by this gene is a G protein-coupled receptor that interacts with the patched protein, a receptor for hedgehog proteins. The encoded protein tranduces signals to other proteins after activation by a hedgehog protein/patched protein complex.","This intronless gene encodes a member of the SRY-related HMG-box (SOX) family of transcription factors involved in the regulation of embryonic development and in the determination of cell fate. The product of this gene is required for stem-cell maintenance in the central nervous system, and also regulates gene expression in the stomach. Mutations in this gene have been associated with optic nerve hypoplasia and with syndromic microphthalmia, a severe form of structural eye malformation. This gene lies within an intron of another gene called SOX2 overlapping transcript (SOX2OT).","This gene is highly similar to the v-src gene of Rous sarcoma virus. This proto-oncogene may play a role in the regulation of embryonic development and cell growth. The protein encoded by this gene is a tyrosine-protein kinase whose activity can be inhibited by phosphorylation by c-SRC kinase. Mutations in this gene could be involved in the malignant progression of colon cancer. Two transcript variants encoding the same protein have been found for this gene.","The product of this gene belongs to the family of highly homologous synovial sarcoma X (SSX) breakpoint proteins. These proteins may function as transcriptional repressors. They are also capable of eliciting spontaneous humoral and cellular immune responses in cancer patients, and are potentially useful targets in cancer vaccine-based immunotherapy. This gene, and also the SSX2 and SSX4 family members, have been involved in t(X;18)(p11.2;q11.2) translocations that are characteristically found in all synovial sarcomas. This translocation results in the fusion of the synovial sarcoma translocation gene on chromosome 18 to one of the SSX genes on chromosome X. The encoded hybrid proteins are likely responsible for transforming activity. Alternative splicing of this gene results in multiple transcript variants. A related pseudogene has been identified on chromosome X.","The product of this gene belongs to the family of highly homologous synovial sarcoma X (SSX) breakpoint proteins. These proteins may function as transcriptional repressors. They are also capable of eliciting spontaneous humoral and cellular immune responses in cancer patients, and are potentially useful targets in cancer vaccine-based immunotherapy. This gene, and also the SSX1 and SSX4 family members, have been involved in t(X;18)(p11.2;q11.2) translocations that are characteristically found in all synovial sarcomas. This translocation results in the fusion of the synovial sarcoma translocation gene on chromosome 18 to one of the SSX genes on chromosome X. The encoded hybrid proteins are likely responsible for transforming activity. Alternative splicing of this gene results in multiple transcript variants. This gene also has an identical duplicate, GeneID: 727837, located about 45 kb downstream in the opposite orientation on chromosome X.","The product of this gene belongs to the family of highly homologous synovial sarcoma X (SSX) breakpoint proteins. These proteins may function as transcriptional repressors. They are also capable of eliciting spontaneously humoral and cellular immune responses in cancer patients, and are potentially useful targets in cancer vaccine-based immunotherapy. SSX1, SSX2 and SSX4 genes have been involved in the t(X;18) translocation characteristically found in all synovial sarcomas. This translocation results in the fusion of the synovial sarcoma translocation gene on chromosome 18 to one of the SSX genes on chromosome X. Chromosome Xp11 contains a segmental duplication resulting in two identical copies of synovial sarcoma, X breakpoint 4, SSX4 and SSX4B, in tail-to-tail orientation. This gene, SSX4, represents the more telomeric copy. Two transcript variants encoding distinct isoforms have been identified for this gene.","The protein encoded by this gene is a member of the STAT protein family. In response to cytokines and growth factors, STAT family members are phosphorylated by the receptor associated kinases, and then form homo- or heterodimers that translocate to the cell nucleus where they act as transcription activators. This protein is activated through phosphorylation in response to various cytokines and growth factors including IFNs, EGF, IL5, IL6, HGF, LIF and BMP2. This protein mediates the expression of a variety of genes in response to cell stimuli, and thus plays a key role in many cellular processes such as cell growth and apoptosis. The small GTPase Rac1 has been shown to bind and regulate the activity of this protein. PIAS3 protein is a specific inhibitor of this protein. This gene also plays a role in regulating host response to viral and bacterial infections. Mutations in this gene are associated with infantile-onset multisystem autoimmune disease and hyper-immunoglobulin E syndrome.","The protein encoded by this gene is a member of the STAT family of transcription factors. In response to cytokines and growth factors, STAT family members are phosphorylated by the receptor associated kinases, and then form homo- or heterodimers that translocate to the cell nucleus where they act as transcription activators. This protein mediates the signal transduction triggered by various cell ligands, such as IL2, IL4, CSF1, and different growth hormones. It has been shown to be involved in diverse biological processes, such as TCR signaling, apoptosis, adult mammary gland development, and sexual dimorphism of liver gene expression. This gene was found to fuse to retinoic acid receptor-alpha (RARA) gene in a small subset of acute promyelocytic leukemias (APLL). The dysregulation of the signaling pathways mediated by this protein may be the cause of the APLL.","The protein encoded by this gene is a member of the STAT family of transcription factors. In response to cytokines and growth factors, STAT family members are phosphorylated by the receptor associated kinases, and then form homo- or heterodimers that translocate to the cell nucleus where they act as transcription activators. This protein plays a central role in exerting IL4 mediated biological responses. It is found to induce the expression of BCL2L1/BCL-X(L), which is responsible for the anti-apoptotic activity of IL4. Knockout studies in mice suggested the roles of this gene in differentiation of T helper 2 (Th2) cells, expression of cell surface markers, and class switch of immunoglobulins. Alternative splicing results in multiple transcript variants.[provided by RefSeq, May 2010].","This gene encodes a member of the family of non-receptor type Tyr protein kinases. This protein is widely expressed in hematopoietic cells and is involved in coupling activated immunoreceptors to downstream signaling events that mediate diverse cellular responses, including proliferation, differentiation, and phagocytosis. It is thought to be a modulator of epithelial cell growth and a potential tumour suppressor in human breast carcinomas. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","Enables several functions, including DNA-binding transcription factor activity; E-box binding activity; and histone deacetylase binding activity. Involved in several processes, including myeloid cell differentiation; positive regulation of cellular component organization; and positive regulation of erythrocyte differentiation. Located in chromatin and nucleoplasm. Part of transcription regulator complex. Implicated in acute lymphoblastic leukemia. [provided by Alliance of Genome Resources, Apr 2022]","This intronless gene encodes a helix-loop-helix protein. Translocations between this gene on chromosome 9 and the T-cell receptor beta-chain locus on chromosome 7 have been associated with activation of the T-cell acute lymphocytic leukemia 2 gene and T-cell acute lymphoblastic leukemia.","This gene is a member of a phylogenetically conserved family of genes that share a common DNA-binding domain, the T-box. T-box genes encode transcription factors involved in the regulation of developmental processes. This protein is a transcriptional repressor and is thought to play a role in the anterior/posterior axis of the tetrapod forelimb. Mutations in this gene cause ulnar-mammary syndrome, affecting limb, apocrine gland, tooth, hair, and genital development. Alternative splicing of this gene results in three transcript variants encoding different isoforms; however, the full length nature of one variant has not been determined.","This gene encodes a member of the E protein (class I) family of helix-loop-helix transcription factors. E proteins activate transcription by binding to regulatory E-box sequences on target genes as heterodimers or homodimers, and are inhibited by heterodimerization with inhibitor of DNA-binding (class IV) helix-loop-helix proteins. E proteins play a critical role in lymphopoiesis, and the encoded protein is required for B and T lymphocyte development. Deletion of this gene or diminished activity of the encoded protein may play a role in lymphoid malignancies. This gene is also involved in several chromosomal translocations that are associated with lymphoid malignancies including pre-B-cell acute lymphoblastic leukemia (t(1;19), with PBX1), childhood leukemia (t(19;19), with TFPT) and acute leukemia (t(12;19), with ZNF384). Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene, and a pseudogene of this gene is located on the short arm of chromosome 9.","This gene encodes a high mobility group (HMG) box-containing transcription factor that plays a key role in the Wnt signaling pathway. The protein has been implicated in blood glucose homeostasis. Genetic variants of this gene are associated with increased risk of type 2 diabetes. Several transcript variants encoding multiple different isoforms have been found for this gene.[provided by RefSeq, Oct 2010].","This gene encodes a zinc finger transcription factor. The encoded protein likely plays a role in transcriptional repression of interleukin 2. Mutations in this gene have been associated with posterior polymorphous corneal dystrophy-3 and late-onset Fuchs endothelial corneal dystrophy. Alternatively spliced transcript variants encoding different isoforms have been described.[provided by RefSeq, Mar 2010].","The protein encoded by this gene belongs to the Tec family of non-receptor protein-tyrosine kinases containing a pleckstrin homology domain. Tec family kinases are involved in the intracellular signaling mechanisms of cytokine receptors, lymphocyte surface antigens, heterotrimeric G-protein coupled receptors, and integrin molecules. They are also key players in the regulation of the immune functions. Tec kinase is an integral component of T cell signaling and has a distinct role in T cell activation. This gene may be associated with myelodysplastic syndrome.","Telomerase is a ribonucleoprotein polymerase that maintains telomere ends by addition of the telomere repeat TTAGGG. The enzyme consists of a protein component with reverse transcriptase activity, encoded by this gene, and an RNA component which serves as a template for the telomere repeat. Telomerase expression plays a role in cellular senescence, as it is normally repressed in postnatal somatic cells resulting in progressive shortening of telomeres. Deregulation of telomerase expression in somatic cells may be involved in oncogenesis. Studies in mouse suggest that telomerase also participates in chromosomal repair, since de novo synthesis of telomere repeats may occur at double-stranded breaks. Alternatively spliced variants encoding different isoforms of telomerase reverse transcriptase have been identified; the full-length sequence of some variants has not been determined. Alternative splicing at this locus is thought to be one mechanism of regulation of telomerase activity.","This gene encodes a basic helix-loop-helix domain-containing transcription factor that binds MUE3-type E-box sequences in the promoter of genes. The encoded protein promotes the expression of genes downstream of transforming growth factor beta (TGF-beta) signaling. This gene may be involved in chromosomal translocations in renal cell carcinomas and other cancers, resulting in the production of fusion proteins. Translocation partners include PRCC (papillary renal cell carcinoma), NONO (non-POU domain containing, octamer-binding), and ASPSCR1 (alveolar soft part sarcoma chromosome region, candidate 1), among other genes. Alternative splicing results in multiple transcript variants.","This gene encodes a protein initially identified as a thyroid-specific transcription factor. The encoded protein binds to the thyroglobulin promoter and regulates the expression of thyroid-specific genes but has also been shown to regulate the expression of genes involved in morphogenesis. Mutations and deletions in this gene are associated with benign hereditary chorea, choreoathetosis, congenital hypothyroidism, and neonatal respiratory distress, and may be associated with thyroid cancer. Multiple transcript variants encoding different isoforms have been found for this gene. This gene shares the symbol/alias 'TTF1' with another gene, transcription termination factor 1, which plays a role in ribosomal gene transcription.","This gene encodes a tumor suppressor protein containing transcriptional activation, DNA binding, and oligomerization domains. The encoded protein responds to diverse cellular stresses to regulate expression of target genes, thereby inducing cell cycle arrest, apoptosis, senescence, DNA repair, or changes in metabolism. Mutations in this gene are associated with a variety of human cancers, including hereditary cancers such as Li-Fraumeni syndrome. Alternative splicing of this gene and the use of alternate promoters result in multiple transcript variants and isoforms. Additional isoforms have also been shown to result from the use of alternate translation initiation codons from identical transcript variants (PMIDs: 12032546, 20937277).","The protein encoded by this gene is a membrane protein and a major controller of thyroid cell metabolism. The encoded protein is a receptor for thyrothropin and thyrostimulin, and its activity is mediated by adenylate cyclase. Defects in this gene are a cause of several types of hyperthyroidism. Three transcript variants encoding different isoforms have been found for this gene.","This gene belongs to the splicing factor SR family of genes. U2 auxiliary factor, comprising a large and a small subunit, is a non-snRNP protein required for the binding of U2 snRNP to the pre-mRNA branch site. This gene encodes the small subunit which plays a critical role in both constitutive and enhancer-dependent RNA splicing by directly mediating interactions between the large subunit and proteins bound to the enhancers. Alternatively spliced transcript variants encoding different isoforms have been identified.","This gene is located on the X chromosome and is the corresponding locus to a Y-linked gene which encodes a tetratricopeptide repeat (TPR) protein. The encoded protein of this gene contains a JmjC-domain and catalyzes the demethylation of tri/dimethylated histone H3. Multiple alternatively spliced transcript variants have been found for this gene.","The Wiskott-Aldrich syndrome (WAS) family of proteins share similar domain structure, and are involved in transduction of signals from receptors on the cell surface to the actin cytoskeleton.  The presence of a number of different motifs suggests that they are regulated by a number of different stimuli, and interact with multiple proteins.  Recent studies have demonstrated that these proteins, directly or indirectly, associate with the small GTPase, Cdc42, known to regulate formation of actin filaments, and the cytoskeletal organizing complex, Arp2/3. Wiskott-Aldrich syndrome is a rare, inherited, X-linked, recessive disease characterized by immune dysregulation and microthrombocytopenia, and is caused by mutations in the WAS gene. The WAS gene product is a cytoplasmic protein, expressed exclusively in hematopoietic cells, which show signalling and cytoskeletal abnormalities in WAS patients.  A transcript variant arising as a result of alternative promoter usage, and containing a different 5' UTR sequence, has been described, however, its full-length nature is not known.","This gene encodes a protein that contains four domains present in other developmental proteins: a PWWP domain, an HMG box, a SET domain, and a PHD-type zinc finger. It is expressed ubiquitously in early development. Wolf-Hirschhorn syndrome (WHS) is a malformation syndrome associated with a hemizygous deletion of the distal short arm of chromosome 4. This gene maps to the 165 kb WHS critical region and has also been involved in the chromosomal translocation t(4;14)(p16.3;q32.3) in multiple myelomas. Alternative splicing of this gene results in multiple transcript variants encoding different isoforms. Some transcript variants are nonsense-mediated mRNA (NMD) decay candidates, hence not represented as reference sequences.","This gene encodes a transcription factor that contains four zinc-finger motifs at the C-terminus and a proline/glutamine-rich DNA-binding domain at the N-terminus. It has an essential role in the normal development of the urogenital system, and it is mutated in a small subset of patients with Wilms tumor. This gene exhibits complex tissue-specific and polymorphic imprinting pattern, with biallelic, and monoallelic expression from the maternal and paternal alleles in different tissues. Multiple transcript variants have been described. In several variants, there is evidence for the use of a non-AUG (CUG) translation initiation codon upstream of, and in-frame with the first AUG. Authors of PMID:7926762 also provide evidence that WT1 mRNA undergoes RNA editing in human and rat, and that this process is tissue-restricted and developmentally regulated.","This cell-cycle-regulated gene encodes a protein that mediates leucine-rich nuclear export signal (NES)-dependent protein transport. The protein specifically inhibits the nuclear export of Rev and U snRNAs. It is involved in the control of several cellular processes by controlling the localization of cyclin B, MPAK, and MAPKAP kinase 2. This protein also regulates NFAT and AP-1.","This gene encodes a CXC chemokine receptor specific for stromal cell-derived factor-1. The protein has 7 transmembrane regions and is located on the cell surface. It acts with the CD4 protein to support HIV entry into cells and is also highly expressed in breast cancer cells. Mutations in this gene have been associated with WHIM (warts, hypogammaglobulinemia, infections, and myelokathexis) syndrome. Alternate transcriptional splice variants, encoding different isoforms, have been characterized.","This gene encodes a protein with one SAP domain. This protein binds to cruciform and superhelical DNA and induces positive supercoils into closed circular DNA, and is also involved in splice site selection during mRNA processing. Chromosomal aberrations involving this region, increased expression of this gene, and the presence of antibodies against this protein are all associated with various diseases. Two transcript variants encoding different isoforms have been found for this gene.","Enables DNA-binding transcription factor activity; enzyme binding activity; and transcription cis-regulatory region binding activity. Involved in several processes, including cellular response to amino acid starvation; lysosome localization; and positive regulation of autophagy. Located in cytosol; lysosomal membrane; and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the MOZ, YBFR2, SAS2, TIP60 family of histone acetyltransferases. The protein is composed of a nuclear localization domain, a double C2H2 zinc finger domain that binds to acetylated histone tails, a histone acetyl-transferase domain, a glutamate/aspartate-rich region, and a serine- and methionine-rich transactivation domain. It is part of a complex that acetylates lysine-9 residues in histone 3, and in addition, it acts as a co-activator for several transcription factors. Allelic variants of this gene are associated with an autosomal dominant form of cognitive disability. Chromosomal translocations of this gene are associated with acute myeloid leukemia. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the steroid-thyroid hormone-retinoid receptor superfamily. The encoded protein may act as a transcriptional activator. The protein can efficiently bind the NGFI-B Response Element (NBRE). Three different versions of extraskeletal myxoid chondrosarcomas (EMCs) are the result of reciprocal translocations between this gene and other genes. The translocation breakpoints are associated with Nuclear Receptor Subfamily 4, Group A, Member 3 (on chromosome 9) and either Ewing Sarcome Breakpoint Region 1 (on chromosome 22), RNA Polymerase II, TATA Box-Binding Protein-Associated Factor, 68-KD (on chromosome 17), or Transcription factor 12 (on chromosome 15). Multiple transcript variants encoding different isoforms have been found for this gene.","This gene was identified based on its homology to the gene encoding the RING3 protein, a serine/threonine kinase. The gene localizes to 9q34, a region which contains several major histocompatibility complex (MHC) genes. The function of the encoded protein is not known.","This gene encodes a transcription factor and has been identified as a partner gene involved in several chromosomal rearrangements resulting in various leukemias. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a histone methyltransferase that methylates the Lys-4 position of histone H3. The encoded protein is part of a large protein complex called ASCOM, which has been shown to be a transcriptional regulator of the beta-globin and estrogen receptor genes. Mutations in this gene have been shown to be a cause of Kabuki syndrome.","This gene encodes a protein that belongs to the non-histone chromosomal high mobility group (HMG) protein family. HMG proteins function as architectural factors and are essential components of the enhancesome. This protein contains structural DNA-binding domains and may act as a transcriptional regulating factor. Identification of the deletion, amplification, and rearrangement of this gene that are associated with myxoid liposarcoma suggests a role in adipogenesis and mesenchymal differentiation. A gene knock out study of the mouse counterpart demonstrated that this gene is involved in diet-induced obesity. Alternate transcriptional splice variants, encoding different isoforms, have been characterized.","Overexpression of the TCL1 gene in humans has been implicated in the development of mature T cell leukemia, in which chromosomal rearrangements bring the TCL1 gene in close proximity to the T-cell antigen receptor (TCR)-alpha (MIM 186880) or TCR-beta (MIM 186930) regulatory elements (summarized by Virgilio et al., 1998). In normal T cells TCL1 is expressed in CD4-/CD8- cells, but not in cells at later stages of differentiation. TCL1 functions as a coactivator of the cell survival kinase AKT (MIM 164730) (Laine et al., 2000 [PubMed 10983986]).","This gene encodes a member of the TET family of RNA-binding proteins. The encoded protein plays a role in RNA polymerase II gene transcription as a component of a distinct subset of multi-subunit transcription initiation factor TFIID complexes. Translocations involving this gene play a role in acute leukemia and extraskeletal myxoid chondrosarcoma, and mutations in this gene may play a role in amyotrophic lateral sclerosis. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","This gene encodes a large multidomain protein of the phosphoinositide 3-kinase-related kinases (PIKK) family. The encoded protein is a common component of many histone acetyltransferase (HAT) complexes and plays a role in transcription and DNA repair by recruiting HAT complexes to chromatin. Deregulation of this gene may play a role in several types of cancer including glioblastoma multiforme. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. This structure consists of approximately 146 bp of DNA wrapped around a nucleosome, an octamer composed of pairs of each of the four core histones (H2A, H2B, H3, and H4). The chromatin fiber is further compacted through the interaction of a linker histone, H1, with the DNA between the nucleosomes to form higher order chromatin structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H3 family. Transcripts from this gene lack polyA tails; instead, they contain a palindromic termination element. This gene is found in the large histone gene cluster on chromosome 6p22-p21.3.","This gene encodes a protein that may modulate the transcriptional repression activities of death-associated protein 6 (DAXX), which interacts with histone deacetylase, core histones, and other histone-associated proteins. In mouse, the encoded protein binds to the putative leucine zipper domain of macroH2A1.2, a variant H2A histone that is enriched on inactivated X chromosomes. The BTB/POZ domain of this protein has been shown in other proteins to mediate transcriptional repression and to interact with components of histone deacetylase co-repressor complexes. Alternative splicing of this gene results in multiple transcript variants encoding the same protein.","IRS4 encodes the insulin receptor substrate 4, a cytoplasmic protein that contains many potential tyrosine and serine/threonine phosphorylation sites. Tyrosine-phosphorylated IRS4 protein has been shown to associate with cytoplasmic signalling molecules that contain SH2 domains. The IRS4 protein is phosphorylated by the insulin receptor tyrosine kinase upon receptor stimulation..","The protein encoded by this gene is a member of the PP2C family of Ser/Thr protein phosphatases. PP2C family members are known to be negative regulators of cell stress response pathways. The expression of this gene is induced in a p53-dependent manner in response to various environmental stresses. While being induced by tumor suppressor protein TP53/p53, this phosphatase negatively regulates the activity of p38 MAP kinase, MAPK/p38, through which it reduces the phosphorylation of p53, and in turn suppresses p53-mediated transcription and apoptosis. This phosphatase thus mediates a feedback regulation of p38-p53 signaling that contributes to growth inhibition and the suppression of stress induced apoptosis. This gene is located in a chromosomal region known to be amplified in breast cancer. The amplification of this gene has been detected in both breast cancer cell line and primary breast tumors, which suggests a role of this gene in cancer development.","This gene encodes a member of the p53 family of transcription factors. The functional domains of p53 family proteins include an N-terminal transactivation domain, a central DNA-binding domain and an oligomerization domain. Alternative splicing of this gene and the use of alternative promoters results in multiple transcript variants encoding different isoforms that vary in their functional properties. These isoforms function during skin development and maintenance, adult stem/progenitor cell regulation, heart development and premature aging. Some isoforms have been found to protect the germline by eliminating oocytes or testicular germ cells that have suffered DNA damage. Mutations in this gene are associated with ectodermal dysplasia, and cleft lip/palate syndrome 3 (EEC3); split-hand/foot malformation 4 (SHFM4); ankyloblepharon-ectodermal defects-cleft lip/palate; ADULT syndrome (acro-dermato-ungual-lacrimal-tooth); limb-mammary syndrome; Rap-Hodgkin syndrome (RHS); and orofacial cleft 8.","The protein encoded by this gene mediates transcriptional control by interaction with the activation function 2 (AF2) region of several nuclear receptors, including the estrogen, retinoic acid, and vitamin D3 receptors. The protein localizes to nuclear bodies and is thought to associate with chromatin and heterochromatin-associated factors. The protein is a member of the tripartite motif (TRIM) family. The TRIM motif includes three zinc-binding domains - a RING, a B-box type 1 and a B-box type 2 - and a coiled-coil region. Two alternatively spliced transcript variants encoding different isoforms have been described for this gene.","The protein encoded by this gene is a single stranded DNA-binding protein that binds to multiple DNA elements, including the far upstream element (FUSE) located upstream of c-myc. Binding to FUSE occurs on the non-coding strand, and is important to the regulation of c-myc in undifferentiated cells. This protein contains three domains, an amphipathic helix N-terminal domain, a DNA-binding central domain, and a C-terminal transactivation domain that contains three tyrosine-rich motifs. The N-terminal domain is thought to repress the activity of the C-terminal domain. This protein is also thought to bind RNA, and contains 3'-5' helicase activity with in vitro activity on both DNA-DNA and RNA-RNA duplexes. Aberrant expression of this gene has been found in malignant tissues, and this gene is important to neural system and lung development. Binding of this protein to viral RNA is thought to play a role in several viral diseases, including hepatitis C and hand, foot and mouth disease. Alternative splicing results in multiple transcript variants.","Enables thiol-dependent deubiquitinase. Involved in protein deubiquitination and regulation of vesicle-mediated transport. Located in plasma membrane and recycling endosome. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a protein that belongs to the ubiquitin-specific processing protease family of proteins. The encoded protein is thought to regulate the morphology of the endosome by ubiquitination of proteins on this organelle and is involved in cargo sorting and membrane trafficking at the early endosome stage. This protein is required for the cell to enter the S phase of the cell cycle and also functions as a positive regulator in the Hedgehog signaling pathway in development. Pseudogenes of this gene are present on chromosomes 2 and 6. Alternate splicing results in multiple transcript variants.","The protein encoded by this gene is a member of serine/threonine protein kinase family. This kinase contains a dual leucine-zipper motif, and has been shown to form dimers/oligomers through its leucine-zipper motif. This kinase can phosphorylate and activate MAPK8/JNK, MAP2K7/MKK7, which suggests a role in the JNK signaling pathway.","This gene encodes a protein that belongs to the Kruppel family of transcription factors. The encoded zinc finger protein is required for normal development of the barrier function of skin. The encoded protein is thought to control the G1-to-S transition of the cell cycle following DNA damage by mediating the tumor suppressor gene p53. Mice lacking this gene have a normal appearance but lose weight rapidly, and die shortly after birth due to fluid evaporation resulting from compromised epidermal barrier function. Alternative splicing results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene is a DNA helicase that belongs to the RecQ helicase family. DNA helicases unwind double-stranded DNA into single-stranded DNAs and may modulate chromosome segregation. This gene is predominantly expressed in thymus and testis. Mutations in this gene are associated with Rothmund-Thomson, RAPADILINO and Baller-Gerold syndromes.","The protein encoded by this gene is an RNA-binding protein that regulates pre-mRNA splicing, export of mRNAs from the nucleus, protein translation, and mRNA stability. The encoded protein is involved in myelinization and oligodendrocyte differentiation and may play a role in schizophrenia. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene is a member of the cytidine deaminase gene family. It is one of seven related genes or pseudogenes found in a cluster, thought to result from gene duplication, on chromosome 22. Members of the cluster encode proteins that are structurally and functionally related to the C to U RNA-editing cytidine deaminase APOBEC1. It is thought that the proteins may be RNA editing enzymes and have roles in growth or cell cycle control. A hybrid gene results from the deletion of approximately 29.5 kb of sequence between this gene, APOBEC3B, and the adjacent gene APOBEC3A. The breakpoints of the deletion are within the two genes, so the deletion allele is predicted to have the promoter and coding region of APOBEC3A, but the 3' UTR of APOBEC3B. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a histone methyltransferase which regulates histone methylation, gene silencing, and transcriptional repression. This gene has been identified as a target for treatment in Huntington Disease, given that gene silencing and transcription dysfunction likely play a role in the disease pathogenesis. Alternatively spliced transcript variants of this gene have been described.[provided by RefSeq, Jun 2011].","The protein encoded by this gene is a basic leucine zipper (bZIP) transcription factor that plays an important role in the regulation of lineage-specific hematopoiesis. The encoded nuclear protein represses ETS1-mediated transcription of erythroid-specific genes in myeloid cells. This gene contains no introns.","The protein encoded by this gene is a member of the AKT, also called PKB,  serine/threonine protein kinase family. AKT kinases are known to be regulators of cell signaling in response to insulin and growth factors. They are involved in a wide variety of biological processes including cell proliferation, differentiation, apoptosis, tumorigenesis, as well as glycogen synthesis and glucose uptake. This kinase has been shown to be stimulated by platelet-derived growth factor (PDGF), insulin, and insulin-like growth factor 1 (IGF1). Alternatively splice transcript variants encoding distinct isoforms have been described.","This gene encodes a basic helix-loop-helix transcription factor which is expressed in oligodendroglial tumors of the brain. The protein is an essential regulator of ventral neuroectodermal progenitor cell fate. The gene is involved in a chromosomal translocation t(14;21)(q11.2;q22) associated with T-cell acute lymphoblastic leukemia. Its chromosomal location is within a region of chromosome 21 which has been suggested to play a role in learning deficits associated with Down syndrome.","Follistatin-like 3 is a secreted glycoprotein of the follistatin-module-protein family. It may have a role in leukemogenesis.","The protein encoded by this gene functions as a transcriptional coactivator for nuclear hormone receptors, including steroid, thyroid, retinoid, and vitamin D receptors. The encoded protein acts as an intermediary factor for the ligand-dependent activity of these nuclear receptors, which regulate their target genes upon binding of cognate response elements. This gene has been found to be involved in translocations that result in fusions with other genes in various cancers, including the lysine acetyltransferase 6A (KAT6A) gene in acute myeloid leukemia, the ETS variant 6 (ETV6) gene in acute lymphoblastic leukemia, and the hes related family bHLH transcription factor with YRPW motif 1 (HEY1) gene in mesenchymal chondrosarcoma. Alternative splicing results in multiple transcript variants.","Enables catalytic activity, acting on DNA; chromatin binding activity; and identical protein binding activity. Involved in DNA repair; negative regulation of double-strand break repair via homologous recombination; and protein homooligomerization. Located in Golgi apparatus; cytosol; and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","This gene is a member of the vertebrate gene family which encode proteins homologous to the Drosophila 'sine oculis' homeobox protein. The encoded protein is a transcription factor which, like other members of this gene family, may be involved in limb or eye development.","This gene encodes a caspase-like protease that plays a role in BCL10-induced activation of NF-kappaB. The protein is a component of the CARMA1-BCL10-MALT1 (CBM) signalosome that triggers NF-kappaB signaling and lymphoctye activation following antigen-receptor stimulation. Mutations in this gene result in immunodeficiency 12 (IMD12). This gene has been found to be recurrently rearranged in chromosomal translocations with other genes in mucosa-associated lymphoid tissue lymphomas, including a t(11;18)(q21;q21) translocation with the baculoviral IAP repeat-containing protein 3 (also known as apoptosis inhibitor 2) locus [BIRC3(API2)-MALT1], and a t(14;18)(q32;q21) translocation with the immunoglobulin heavy chain locus (IGH-MALT1). Alternatively spliced transcript variants have been described for this gene.","The protein encoded by this gene is part of the multimeric HBO1 complex, which possesses histone H4-specific acetyltransferase activity. This activity is required for functional replication origins and is involved in transcriptional activation of some genes. In both cases, the acetylation of histone H4 helps unfold chromatin so that the DNA can be accessed and replicated or transcribed.","Enables DNA-binding transcription factor binding activity; chromatin binding activity; and transcription coactivator activity. Involved in mRNA 5'-splice site recognition and positive regulation of transcription by RNA polymerase II. Located in heterochromatin; nuclear periphery; and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a hydrolase that serves as an important role in cellular purine metabolism by acting primarily on inosine 5'-monophosphate and other purine nucleotides.","The protein encoded by this gene is an ortholog of the Drosophila melanogaster capicua gene, and is a member of the high mobility group (HMG)-box superfamily of transcriptional repressors. This protein contains a conserved HMG domain that is involved in DNA binding and nuclear localization, and a conserved C-terminus. Studies suggest that the N-terminal region of this protein interacts with Atxn1 (GeneID:6310), to form a transcription repressor complex, and in vitro studies suggest that polyglutamine-expansion of ATXN1 may alter the repressor activity of this complex. Mutations in this gene have been associated with olidogdendrogliomas (PMID:21817013). In addition, translocation events resulting in gene fusions of this gene with both DUX4 (GeneID:100288687) and FOXO4 (GeneID:4303) have been associated with round cell sarcomas. There are multiple pseudogenes of this gene found on chromosomes 1, 4, 6, 7, 16, 20, and the Y chromosome. Alternative splicing results in multiple transcript variants encoding different isoforms.","Enables cAMP response element binding protein binding activity. Involved in positive regulation of transcription by RNA polymerase II. Located in cytosol; nuclear body; and plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes subunit 1 of the splicing factor 3b protein complex. Splicing factor 3b, together with splicing factor 3a and a 12S RNA unit, forms the U2 small nuclear ribonucleoproteins complex (U2 snRNP). The splicing factor 3b/3a complex binds pre-mRNA upstream of the intron's branch site in a sequence independent manner and may anchor the U2 snRNP to the pre-mRNA. Splicing factor 3b is also a component of the minor U12-type spliceosome. The carboxy-terminal two-thirds of subunit 1 have 22 non-identical, tandem HEAT repeats that form rod-like, helical structures. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a nuclear protein belonging to the hairy and enhancer of split-related (HESR) family of basic helix-loop-helix (bHLH)-type transcriptional repressors. Expression of this gene is induced by the Notch and c-Jun signal transduction pathways. Two similar and redundant genes in mouse are required for embryonic cardiovascular development, and are also implicated in neurogenesis and somitogenesis. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is homologous to the murine protein MCAP, which associates with chromosomes during mitosis, and to the human RING3 protein, a serine/threonine kinase. Each of these proteins contains two bromodomains, a conserved sequence motif which may be involved in chromatin targeting. This gene has been implicated as the chromosome 19 target of translocation t(15;19)(q13;p13.1), which defines an upper respiratory tract carcinoma in young people. Two alternatively spliced transcript variants have been described.","This zinc finger gene has been identified at the breakpoints of a recurrent chromosomal translocation reported in endometrial stromal sarcoma. Recombination of these breakpoints results in the fusion of this gene and JAZF1. The protein encoded by this gene contains a zinc finger domain in the C terminus of the coding region.","This gene encodes a member of the Cbl family of E3 ubiquitin ligases. Cbl proteins play important roles in cell signaling through the ubiquitination and subsequent downregulation of tyrosine kinases. Expression of this gene may be restricted to epithelial cells, and alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","Enables protein domain specific binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to act upstream of or within neuron fate commitment. Located in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Enables transcription coactivator activity. Involved in several processes, including hippo signaling; positive regulation of cell differentiation; and regulation of signal transduction. Located in cytosol and nuclear body. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a protein which contains a several motifs including a ski homology region and a SET-binding region in addition to three nuclear localization signals. The encoded protein has been shown to bind the SET nuclear oncogene which is involved in DNA replication. Mutations in this gene are associated with Schinzel-Giedion midface retraction syndrome. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a poly(A) binding protein. The protein shuttles between the nucleus and cytoplasm and binds to the 3' poly(A) tail of eukaryotic messenger RNAs via RNA-recognition motifs. The binding of this protein to poly(A) promotes ribosome recruitment and translation initiation; it is also required for poly(A) shortening which is the first step in mRNA decay. The gene is part of a small gene family including three protein-coding genes and several pseudogenes.[provided by RefSeq, Aug 2010].","This gene encodes a transcriptional co-activator that interacts with the acidic domain of Epstein-Barr virus nuclear antigen 2 (EBNA 2), a transcriptional activator that is required for B-lymphocyte transformation. Other transcription factors that interact with this protein are signal transducers and activators of transcription, STATs. This protein is also thought to be essential for normal cell growth. A similar protein in mammals and other organisms is a component of the RNA-induced silencing complex (RISC).","This gene belongs to subfamily P of the forkhead box (FOX) transcription factor family. Forkhead box transcription factors play important roles in the regulation of tissue- and cell type-specific gene transcription during both development and adulthood. Forkhead box P1 protein contains both DNA-binding- and protein-protein binding-domains. This gene may act as a tumor suppressor as it is lost in several tumor types and maps to a chromosomal region (3p14.1) reported to contain a tumor suppressor gene(s). Alternative splicing results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene belongs to the AF4 family of transcription factors involved in leukemia. It is a component of the positive transcription elongation factor b (P-TEFb) complex. A chromosomal translocation involving this gene and MLL gene on chromosome 11 is found in infant acute lymphoblastic leukemia with ins(5;11)(q31;q31q23).","Mammalian apolipoprotein B mRNA undergoes site-specific C to U deamination, which is mediated by a multi-component enzyme complex containing a minimal core composed of APOBEC-1 and a complementation factor encoded by this gene. The gene product has three non-identical RNA recognition motifs and belongs to the hnRNP R family of RNA-binding proteins. It has been proposed that this complementation factor functions as an RNA-binding subunit and docks APOBEC-1 to deaminate the upstream cytidine. Studies suggest that the protein may also be involved in other RNA editing or RNA processing events. Several transcript variants encoding a few different isoforms have been found for this gene.","The protein encoded by this gene is an orphan homeobox protein that encodes a DNA-binding nuclear transcription factor. A translocation [t(5;14)(q35;q32)] involving this gene is associated with T-cell acute lymphoblastic leukemia (T-ALL) in children and young adults.","The protein encoded by this gene belongs to the sulfotransferase 2 family. It is localized to the golgi membrane, and catalyzes the transfer of sulfate to position 4 of the N-acetylgalactosamine (GalNAc) residue of chondroitin. Chondroitin sulfate constitutes the predominant proteoglycan present in cartilage, and is distributed on the surfaces of many cells and extracellular matrices. A chromosomal translocation involving this gene and IgH, t(12;14)(q23;q32), has been reported in a patient with B-cell chronic lymphocytic leukemia. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","This gene encodes a transcription factor belonging to a family of proteins that share homology with the high mobility group protein-1. The protein encoded by this gene can bind to a functionally important site in the T-cell receptor-alpha enhancer, thereby conferring maximal enhancer activity. This transcription factor is involved in the Wnt signaling pathway, and it may function in hair cell differentiation and follicle morphogenesis. Mutations in this gene have been found in somatic sebaceous tumors. This gene has also been linked to other cancers, including androgen-independent prostate cancer. Alternative splicing results in multiple transcript variants.","This gene encodes a progestin-induced protein, which belongs to the HECT (homology to E6-AP carboxyl terminus) family. The HECT family proteins function as E3 ubiquitin-protein ligases, targeting specific proteins for ubiquitin-mediated proteolysis. This gene is localized to chromosome 8q22 which is disrupted in a variety of cancers. This gene potentially has a role in regulation of cell proliferation or differentiation.","This gene encodes a C2H2 type zinc-finger protein by its similarity to the mouse Bcl11a/Evi9 protein. The corresponding mouse gene is a common site of retroviral integration in myeloid leukemia, and may function as a leukemia disease gene, in part, through its interaction with BCL6. During hematopoietic cell differentiation, this gene is down-regulated. It is possibly involved in lymphoma pathogenesis since translocations associated with B-cell malignancies also deregulates its expression. Multiple transcript variants encoding several different isoforms have been found for this gene.","This gene encodes a subunit of the microprocessor complex which mediates the biogenesis of microRNAs from the primary microRNA transcript. The encoded protein is a double-stranded RNA binding protein that functions as the non-catalytic subunit of the microprocessor complex. This protein is required for binding the double-stranded RNA substrate and facilitates cleavage of the RNA by the ribonuclease III protein, Drosha. Alternate splicing results in multiple transcript variants.","This gene belongs to the ETS transcription factor family. ETS family members have a highly conserved 85-amino acid ETS domain that binds purine-rich DNA sequences. The alanine-rich C-terminus of this gene indicates that it may act as a transcription repressor. This gene is exclusively expressed in neurons of the central serotonin (5-HT) system, a system implicated in the pathogeny of such psychiatric diseases as depression, anxiety, and eating disorders. In some types of Ewing tumors, this gene is fused to the Ewing sarcoma (EWS) gene following chromosome translocations.","This gene is related to the Wolf-Hirschhorn syndrome candidate-1 gene and encodes a protein with PWWP (proline-tryptophan-tryptophan-proline) domains. This protein methylates histone H3 at lysine residues 4 and 27, which represses gene transcription. Two alternatively spliced variants have been described.","This gene encodes a member of the G-protein coupled receptor family. Although this protein was earlier thought to be a receptor for vasoactive intestinal peptide (VIP), it is now considered to be an orphan receptor, in that its endogenous ligand has not been identified. The protein is also a coreceptor for human immunodeficiency viruses (HIV). Translocations involving this gene and HMGA2 on chromosome 12 have been observed in lipomas.","The cysteinyl leukotrienes LTC4, LTD4, and LTE4 are important mediators of human bronchial asthma. Pharmacologic studies have determined that cysteinyl leukotrienes activate at least 2 receptors, the protein encoded by this gene and CYSLTR1. This encoded receptor is a member of the superfamily of G protein-coupled receptors. It seems to play a major role in endocrine and cardiovascular systems.","This gene encodes a zinc finger transcription factor thought to play a role in the development of abducens motor neurons. Defects in this gene are a cause of Duane-radial ray syndrome (DRRS). Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a protein with a BIR (baculoviral inhibition of apoptosis protein repeat) domain and a UBCc (ubiquitin-conjugating enzyme E2, catalytic) domain. This protein inhibits apoptosis by facilitating the degradation of apoptotic proteins by ubiquitination.","The protein encoded by this gene interacts with the transcription factor myocardin, a key regulator of smooth muscle cell differentiation. The encoded protein is predominantly nuclear and may help transduce signals from the cytoskeleton to the nucleus. This gene is involved in a specific translocation event that creates a fusion of this gene and the RNA-binding motif protein-15 gene. This translocation has been associated with acute megakaryocytic leukemia. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a transcriptional corepressor that is found tethered to promoter regions by DNA-binding proteins. The encoded protein can interact with several different class II histone deacetylases to repress transcription. Two transcript variants encoding different isoforms have been found for this gene.","The reciprocal translocation t(1;3)(p36;q21) occurs in a subset of myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). This gene is located near the 1p36.3 breakpoint and has been shown to be specifically expressed in the t(1:3)(p36,q21)-positive MDS/AML. The protein encoded by this gene is a zinc finger transcription factor and contains an N-terminal PR domain. The translocation results in the overexpression of a truncated version of this protein that lacks the PR domain, which may play an important role in the pathogenesis of MDS and AML. Alternatively spliced transcript variants encoding distinct isoforms have been reported.","This gene encodes a member of the type I cytokine receptor family. The encoded protein is a receptor for thymic stromal lymphopoietin (TSLP). Together with the interleukin 7 receptor (IL7R), the encoded protein and TSLP activate STAT3, STAT5, and JAK2 pathways, which control processes such as cell proliferation and development of the hematopoietic system. Rearrangement of this gene with immunoglobulin heavy chain gene (IGH) on chromosome 14, or with P2Y purinoceptor 8 gene (P2RY8) on the same X or Y chromosomes is associated with B-progenitor acute lymphoblastic leukemia (ALL) and Down syndrome ALL. Alternatively spliced transcript variants have been found for this gene.","This gene encodes a member of the oasis bZIP transcription factor family. Members of this family can dimerize but form homodimers only. The encoded protein is a transcriptional activator. Translocations between this gene on chromosome 7 and the gene fused in sarcoma on chromosome 16 can be found in some tumors. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a C2H2-type zinc finger protein and is closely related to BCL11A, a gene whose translocation may be associated with B-cell malignancies. Although the specific function of this gene has not been determined, the encoded protein is known to be a transcriptional repressor, and is regulated by the NURD nucleosome remodeling and histone deacetylase complex. Four alternatively spliced transcript variants encoding distinct isoforms have been found for this gene.","This gene is a member of the WD40 repeat-containing gene family and shares sequence similarity with transducin (beta)-like 1X-linked (TBL1X). The protein encoded by this gene is thought to be a component of both nuclear receptor corepressor (N-CoR) and histone deacetylase 3 (HDAC 3) complexes, and is required for transcriptional activation by a variety of transcription factors. Mutations in these gene have been associated with some autism spectrum disorders, and one finding suggests that haploinsufficiency of this gene may be a cause of intellectual disability with dysmorphism. Mutations in this gene as well as recurrent translocations involving this gene have also been observed in some tumors.","The protein encoded by this gene belongs to the phosphatidylinositol 3,4,5-trisphosphate (PIP3)-dependent Rac exchanger (PREX) family, which are Dbl-type guanine-nucleotide exchange factors for Rac family small G proteins. Structural domains of this protein include the catalytic diffuse B-cell lymphoma homology and pleckstrin homology (DHPH) domain, two disheveled, EGL-10, and pleckstrin homology (DEP) domains, two PDZ domains, and a C-terminal inositol polyphosphate-4 phosphatase (IP4P) domain that is found in one of the isoforms. This protein facilitates the exchange of GDP for GTP on Rac1, allowing the GTP-bound Rac1 to activate downstream effectors. Studies also show that the pleckstrin homology domain of this protein interacts with the phosphatase and tensin homolog (PTEN) gene product to inhibit PTEN phosphatase activity, thus activating the phosphoinositide-3 kinase (PI3K) signaling pathway. Conversely, the PTEN gene product has also been shown to inhibit the GEF activity of this protein. This gene plays a role in insulin-signaling pathways, and either mutations or overexpression of this gene have been observed in some cancers.","DNA methylation is an epigenetic mechanism that is important for controlling gene expression. The protein encoded by this gene is a demethylase that belongs to the TET (ten-eleven translocation) family. Members of the TET protein family play a role in the DNA methylation process and gene activation.","Involved in negative regulation of activated T cell proliferation; negative regulation of interferon-gamma production; and negative regulation of interleukin-10 production. Predicted to be located in plasma membrane. Predicted to be active in external side of plasma membrane. Biomarker of pulmonary tuberculosis. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the immunoglobulin receptor superfamily and is one of several Fc receptor-like glycoproteins clustered on the long arm of chromosome 1. The encoded protein has four extracellular C2-type immunoglobulin domains, a transmembrane domain and a cytoplasmic domain that contains three immune-receptor tyrosine-based inhibitory motifs. This protein may play a role in the function of memory B-cells in the epithelia. Aberrations in the chromosomal region encoding this gene are associated with non-Hodgkin lymphoma and multiple myeloma.","This gene encodes a member of a family of proteins containing a Bcl-2 homology domain 2 (BH2). The encoded protein is an anti-apoptotic factor that acts as an inhibitor of caspases 3 and 7 in the cytoplasm. In the nucleus, it binds to the p53 tumor suppressor protein, preventing its association with target genes. Overexpression of this gene has been detected in a number of different cancers. There is a pseudogene for this gene on chromosome 3. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene belongs to the membrane-associated guanylate kinase (MAGUK) family, a class of proteins that functions as molecular scaffolds for the assembly of multiprotein complexes at specialized regions of the plasma membrane. This protein is also a member of the CARD protein family, which is defined by carrying a characteristic caspase-associated recruitment domain (CARD). This protein has a domain structure similar to that of CARD14 protein. The CARD domains of both proteins have been shown to specifically interact with BCL10, a protein known to function as a positive regulator of cell apoptosis and NF-kappaB activation. When expressed in cells, this protein activated NF-kappaB and induced the phosphorylation of BCL10.","The protein encoded by this gene is a member of the Mastermind-like family of proteins. All family members are proline and glutamine-rich, and contain a conserved basic domain that binds the ankyrin repeat domain of the intracellular domain of the Notch receptors (ICN1-4) in their N-terminus, and a transcriptional activation domain in their C-terminus. This protein binds to an extended groove that is formed by the interaction of CBF1, Suppressor of Hairless, LAG-1 (CSL) with ICN, and positively regulates Notch signaling. High levels of expression of this gene have been observed in several B cell-derived lymphomas. Translocations resulting in fusion proteins with both CRTC1 and CRTC3 have been implicated in the development of mucoepidermoid carcinomas, while a translocation event with CXCR4 has been linked with chronic lymphocytic leukemia (CLL). Copy number variation in the polyglutamine tract has been observed.","This gene belongs to the R-spondin family. The encoded protein plays a role in the regulation of Wnt (wingless-type MMTV integration site family)/beta-catenin and Wnt/planar cell polarity (PCP) signaling pathways, which are involved in development, cell growth and disease pathogenesis. Genome-wide association studies suggest a correlation of this gene with bone mineral density and risk of fracture. This gene may be involved in tumor development.","Enables microtubule plus-end binding activity and protein homodimerization activity. Involved in several processes, including cellular response to epidermal growth factor stimulus; mitotic sister chromatid segregation; and regulation of attachment of spindle microtubules to kinetochore. Located in several cellular components, including kinetochore; microtubule cytoskeleton; and ruffle. Implicated in actinic keratosis and skin squamous cell carcinoma. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes an RNA-binding protein that is a member of the Musashi protein family. The encoded protein is transcriptional regulator that targets genes involved in development and cell cycle regulation. Mutations in this gene are associated with poor prognosis in certain types of cancers. This gene has also been shown to be rearranged in certain cancer cells.","Predicted to be located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Enables beta-catenin binding activity. Involved in several processes, including negative regulation of transforming growth factor beta receptor signaling pathway; positive regulation of epithelial to mesenchymal transition; and positive regulation of transcription by RNA polymerase II. Located in nucleolus and nucleoplasm. Part of beta-catenin-TCF complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the forkhead box (FOX) family of transcription factors. FOX family members are monomeric, helix-turn-helix proteins with a core DNA-binding domain of approximately 110 aa. Many FOX transcription factors play roles in determining cell fates during early development. This forkhead box protein lacks the C-terminal basic region found in many other FOX family members. It is located within the 11q23.3 region which is commonly deleted in neuroblastomas.","The protein encoded by this gene belongs to the family of G-protein coupled receptors, that are preferentially activated by adenosine and uridine nucleotides. This gene is moderately expressed in undifferentiated HL60 cells, and is located on both chromosomes X and Y.","MACC1 is a key regulator of the hepatocyte growth factor (HGF; MIM 142409)-HGF receptor (HGFR, or MET; MIM 164860) pathway, which is involved in cellular growth, epithelial-mesenchymal transition, angiogenesis, cell motility, invasiveness, and metastasis. Expression of MACC1 in colon cancer (MIM 114500) specimens is an independent prognostic indicator for metastasis formation and metastasis-free survival (Stein et al., 2009)."]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>symbol<\/th>\n      <th>entrezgene<\/th>\n      <th>name<\/th>\n      <th>gene_biotype<\/th>\n      <th>ncg_oncogene<\/th>\n      <th>support_cancermine<\/th>\n      <th>ncbi_function_summary<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"buttons":["csv","excel"],"dom":"Bfrtip","columnDefs":[{"className":"dt-right","targets":2},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"symbol","targets":1},{"name":"entrezgene","targets":2},{"name":"name","targets":3},{"name":"gene_biotype","targets":4},{"name":"ncg_oncogene","targets":5},{"name":"support_cancermine","targets":6},{"name":"ncbi_function_summary","targets":7}],"order":[],"autoWidth":false,"orderClasses":false,"responsive":true}},"evals":[],"jsHooks":[]}</script><p><br><br></p>
</div>
<div class="section level3">
<h3 id="get-predicted-cancer-driver-genes">Get predicted cancer driver genes<a class="anchor" aria-label="anchor" href="#get-predicted-cancer-driver-genes"></a>
</h3>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## Get predicted cancer driver genes - as indicated from either</span></span>
<span><span class="co">## - Network of Cancer Genes (NCG) - canonical drivers</span></span>
<span><span class="co">## - IntOGen mutational driver catalogue</span></span>
<span><span class="co">## - TCGA's PanCancer driver prediction (Bailey et al., Cell, 2018)</span></span>
<span><span class="co">##</span></span>
<span><span class="co">##  Rank hits by how many sources that contribute to classification</span></span>
<span><span class="co">##</span></span>
<span><span class="va">cancer_driver</span> <span class="op">&lt;-</span> <span class="va">gene_basic</span><span class="op">$</span><span class="va">records</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">intogen_driver</span> <span class="op">==</span> <span class="cn">TRUE</span> <span class="op">|</span></span>
<span>                <span class="va">ncg_driver</span> <span class="op">==</span> <span class="cn">TRUE</span> <span class="op">|</span></span>
<span>                <span class="va">tcga_driver</span> <span class="op">==</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>driver_score <span class="op">=</span> <span class="fl">0</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="co"># dplyr::mutate(driver_score = dplyr::if_else(</span></span>
<span>  <span class="co">#   cgc_driver_tier1 == T, </span></span>
<span>  <span class="co">#   driver_score + 2,</span></span>
<span>  <span class="co">#   as.numeric(driver_score))) |&gt;</span></span>
<span>  <span class="co"># dplyr::mutate(driver_score = dplyr::if_else(</span></span>
<span>  <span class="co">#   cgc_driver_tier2 == T, </span></span>
<span>  <span class="co">#   driver_score + 1,</span></span>
<span>  <span class="co">#   as.numeric(driver_score))) |&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>driver_score <span class="op">=</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/if_else.html" class="external-link">if_else</a></span><span class="op">(</span></span>
<span>    <span class="va">intogen_driver</span> <span class="op">==</span> <span class="cn">T</span>, </span>
<span>    <span class="va">driver_score</span> <span class="op">+</span> <span class="fl">1</span>,</span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/numeric.html" class="external-link">as.numeric</a></span><span class="op">(</span><span class="va">driver_score</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>driver_score <span class="op">=</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/if_else.html" class="external-link">if_else</a></span><span class="op">(</span></span>
<span>    <span class="va">ncg_driver</span> <span class="op">==</span> <span class="cn">T</span>, </span>
<span>    <span class="va">driver_score</span> <span class="op">+</span> <span class="fl">1</span>,</span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/numeric.html" class="external-link">as.numeric</a></span><span class="op">(</span><span class="va">driver_score</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>driver_score <span class="op">=</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/if_else.html" class="external-link">if_else</a></span><span class="op">(</span></span>
<span>    <span class="va">tcga_driver</span> <span class="op">==</span> <span class="cn">T</span>, </span>
<span>    <span class="va">driver_score</span> <span class="op">+</span> <span class="fl">1</span>,</span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/numeric.html" class="external-link">as.numeric</a></span><span class="op">(</span><span class="va">driver_score</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">symbol</span>, <span class="va">entrezgene</span>, </span>
<span>                <span class="va">name</span>, <span class="va">gene_biotype</span>,</span>
<span>                <span class="va">intogen_driver</span>,</span>
<span>                <span class="va">intogen_role</span>,</span>
<span>                <span class="va">ncg_driver</span>,</span>
<span>                <span class="va">ncg_phenotype</span>,</span>
<span>                <span class="va">tcga_driver</span>,</span>
<span>                <span class="va">driver_score</span>,</span>
<span>                <span class="va">cancermine_cit_links_driver</span>, </span>
<span>                <span class="va">ncbi_function_summary</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/rename.html" class="external-link">rename</a></span><span class="op">(</span>support_cancermine <span class="op">=</span> </span>
<span>                  <span class="va">cancermine_cit_links_driver</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/desc.html" class="external-link">desc</a></span><span class="op">(</span><span class="va">driver_score</span><span class="op">)</span>, <span class="va">symbol</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Make as datatable</span></span>
<span><span class="va">driver_table</span> <span class="op">&lt;-</span> <span class="fu">DT</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span></span>
<span>  <span class="va">cancer_driver</span>, </span>
<span>  escape <span class="op">=</span> <span class="cn">FALSE</span>,</span>
<span>  extensions <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Buttons"</span>, <span class="st">"Responsive"</span><span class="op">)</span>, </span>
<span>  width <span class="op">=</span> <span class="st">"100%"</span>,</span>
<span>  options <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>    buttons <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"csv"</span>, <span class="st">"excel"</span><span class="op">)</span>, </span>
<span>    dom <span class="op">=</span> <span class="st">"Bfrtip"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<p><br></p>
<div id="htmlwidget-36aa3d2a04d42bbc2145" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-36aa3d2a04d42bbc2145">{"x":{"filter":"none","vertical":false,"extensions":["Buttons","Responsive"],"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546","547","548","549","550","551","552","553","554","555","556","557","558","559","560","561","562","563","564","565","566","567","568","569","570","571","572","573","574","575","576","577","578","579","580","581","582","583","584","585","586","587","588","589","590","591","592","593","594","595","596","597","598","599","600","601","602","603","604","605","606","607","608","609","610","611","612","613","614","615","616","617","618","619","620","621","622","623","624","625","626","627","628","629","630","631","632","633","634","635","636","637","638","639","640","641","642","643","644","645","646","647","648","649","650","651","652","653","654","655","656","657","658","659","660","661","662","663","664","665","666","667","668","669","670","671","672","673","674","675","676","677","678","679","680","681","682","683","684","685","686","687","688","689","690","691","692","693","694","695","696","697","698","699","700","701","702","703","704","705","706","707","708","709","710","711","712","713","714","715","716","717","718","719","720","721","722","723","724","725","726","727","728","729","730","731","732","733","734","735","736","737","738","739","740","741","742","743","744","745","746","747","748","749","750","751","752","753","754","755","756","757","758","759","760","761","762","763","764","765","766","767","768","769","770","771","772","773","774","775","776","777","778","779","780","781","782","783","784","785","786","787","788","789","790","791","792","793","794","795","796","797","798","799","800","801","802","803","804","805","806","807","808","809","810","811","812","813","814","815","816","817","818","819","820","821","822","823","824","825","826","827","828","829","830","831","832","833","834","835","836","837","838","839","840","841","842","843","844","845","846","847","848","849","850","851","852","853","854","855","856","857","858","859","860","861","862","863","864","865","866","867","868","869","870","871","872","873","874","875","876","877","878","879"],["ABL1","ACVR1","ACVR1B","ACVR2A","AKT1","ALK","AMER1","APC","AR","ARAF","ARID1A","ARID2","ASXL1","ASXL2","ATM","ATR","ATRX","AXIN1","B2M","BAP1","BCL2","BCOR","BRAF","BRCA1","BRCA2","CARD11","CASP8","CBFB","CCND1","CD79B","CDH1","CDK12","CDK4","CDKN1A","CDKN1B","CDKN2A","CDKN2C","CEBPA","CHD4","CHEK2","CIC","CREBBP","CTCF","CTNNB1","CUL3","CYLD","CYSLTR2","DDX3X","DICER1","DNMT3A","EGFR","EIF1AX","ELF3","EP300","EPAS1","EPHA3","ERBB2","ERBB3","ERBB4","ERCC2","ESR1","EZH2","FAT1","FBXW7","FGFR1","FGFR2","FGFR3","FLT3","FOXA1","FUBP1","GATA3","GNA11","GNAQ","GNAS","H3-3A","HRAS","IDH1","IDH2","IL6ST","IL7R","JAK1","JAK2","JAK3","KDM5C","KDM6A","KEAP1","KIT","KMT2A","KMT2C","KMT2D","KRAS","LATS1","LATS2","LZTR1","MAP2K1","MAP2K4","MAP3K1","MAPK1","MAX","MECOM","MED12","MEN1","MET","MTOR","MYC","MYCN","MYD88","MYH9","NCOR1","NF1","NF2","NFE2L2","NOTCH1","NOTCH2","NPM1","NRAS","NSD2","PAX5","PBRM1","PCBP1","PDGFRA","PHF6","PIK3CA","PIK3CB","PIK3R1","PIM1","PLCG1","PMS2","POLE","POLQ","PPM1D","PPP2R1A","PPP6C","PRKAR1A","PSIP1","PTCH1","PTEN","PTPN11","PTPRC","PTPRD","RAC1","RAD21","RAF1","RB1","RBM10","RET","RHOA","RNF43","RPL22","RPL5","RUNX1","SETBP1","SETD2","SF3B1","SMAD2","SMAD4","SMARCA4","SMARCB1","SMC1A","SOX9","SPOP","SRSF2","STAG2","STK11","TBL1XR1","TBX3","TCF7L2","TET2","TGFBR2","TNFAIP3","TP53","TSC1","TSC2","U2AF1","VHL","WT1","XPO1","ZFHX3","ZMYM3","ABL2","ACKR3","AFDN","AFF3","AFF4","AJUBA","AKT3","ANK1","ARHGAP35","ARHGAP5","ARHGEF10","ARHGEF10L","ARHGEF12","ARID1B","ARNT","ATF7IP","ATP1A1","ATP2B3","AXIN2","BARD1","BAZ1A","BCL10","BCL11A","BCL11B","BCL6","BCL9","BCL9L","BCLAF1","BCORL1","BIRC3","BIRC6","BMPR1A","BRD4","BRD7","BTG1","BTG2","BTK","CACNA1D","CAMTA1","CBL","CBLB","CCDC6","CCND2","CCND3","CCR7","CD209","CD274","CDC73","CDH10","CDH11","CDX2","CHD2","CIITA","CLTC","CLTCL1","CNBD1","CNOT3","CNOT9","COL2A1","CPEB3","CRLF2","CRTC1","CSF1R","CSF3R","CTNNA2","CTNND1","CTNND2","CUX1","CXCR4","CYP2C8","DAXX","DCAF12L2","DCC","DDB2","DDR2","DDX6","DGCR8","DHX9","DROSHA","EBF1","EIF3E","ELF4","ELK4","ELL","EPHA2","EPHA7","EPS15","ERCC3","ERG","ETV1","ETV4","ETV5","ETV6","EWSR1","FAM135B","FANCA","FAS","FAT3","FAT4","FBLN2","FBXO11","FCGR2B","FCRL4","FGFR4","FHIT","FLCN","FLNA","FLT4","FOXA2","FOXL2","FOXO1","FOXO3","FOXP1","FUS","GATA2","GLI1","GNA13","GPC5","GRIN2A","GRM3","GTF2I","H3C2","HGF","HIF1A","HIP1","HLA-A","HLA-B","HNF1A","HNRNPA2B1","HOXC13","ID3","IKBKB","IKZF1","INPPL1","IRF4","IRS4","ITGAV","KAT6A","KAT6B","KAT7","KDM5A","KDR","KLF4","KLF5","KLF6","KMT2B","KNL1","KNSTRN","LEF1","LRIG3","LRP1B","MACC1","MAF","MALT1","MAML2","MDM2","MGA","MGMT","MLH1","MPL","MSH2","MSH6","MSI2","MUC6","N4BP2","NBEA","NCOA2","NCOR2","NFATC2","NFKB2","NFKBIE","NIPBL","NKX2-1","NRG1","NSD1","NSD3","NTRK1","NTRK3","NUP98","NUTM1","P2RY8","PABPC1","PALB2","PDCD1LG2","PDGFRB","PIK3R2","PLAG1","PML","PMS1","POLD1","POT1","PRDM1","PRDM16","PRDM2","PREX2","PRKCB","PTK6","PTMA","PTPN13","PTPN6","PTPRB","PTPRK","PTPRT","QKI","RANBP2","RAP1GDS1","RARA","RASA1","RECQL4","RGPD3","RGS7","RIT1","ROBO2","ROS1","RPL10","RPS6KA3","RRAS2","RSPO2","RUNX1T1","RXRA","SALL4","SDHA","SETD1B","SETDB1","SGK1","SH2B3","SIN3A","SIRPA","SIX1","SLC34A2","SMAD3","SMARCD1","SMO","SND1","SOCS1","SOS1","SOX17","SOX21","SPEN","STAT3","STAT5B","STAT6","SUFU","SUZ12","TAF15","TAL1","TAL2","TCF12","TCL1A","TEC","TENT5C","TERT","TET1","TGIF1","THRAP3","TMSB4X","TNC","TNFRSF14","TP63","TRAF3","TRAF7","TRIM24","TRIM33","TRRAP","TSHR","UBR5","USP6","USP8","USP9X","VAV1","WAS","WNK2","WWTR1","ZBTB16","ZBTB20","ZBTB7B","ZEB1","ZFP36L1","ZMYM2","ZNF331","ZNF429","ZNF521","ZNF750","ZNRF3","ZRSR2","A1CF","ABCC4","ABI1","ACSL3","ACSL6","ACSM3","ADAMTS9","AFF1","AKAP9","AKT2","ALB","ALDH2","ALPK1","ANKRD11","APOB","APOBEC3B","ARHGAP26","ARID5B","ATF1","ATG7","ATXN3","BAX","BCL2L11","BCL2L12","BCL3","BCL7A","BCR","BLM","BMP5","BMP6","BRD3","BRIP1","BUB1B","CACNA1A","CADM2","CALR","CANT1","CARS1","CASP3","CASP9","CASZ1","CBFA2T3","CBLC","CCAR1","CCNB1IP1","CCNC","CCNE1","CCNT1","CCR4","CD28","CD58","CD70","CD74","CD79A","CDH17","CDK6","CEP89","CHD3","CHD7","CHD8","CHRDL1","CHST11","CLIP1","CMTR2","CNBP","CNTRL","COL1A1","COL5A1","COL6A3","COL6A6","CREB1","CREB3L1","CREB3L2","CREB3L3","CRNKL1","CSDE1","CSMD3","CSPG4","CTNNA1","CUL1","DACH1","DAZAP1","DDIT3","DDX10","DDX5","DEK","DIAPH2","DIS3","DMD","DNM2","DNMT1","DOT1L","DSP","DTX1","DUSP16","ECT2L","EED","EEF1A1","EEF2","EFTUD2","EGR2","EGR3","EIF4A2","ELN","ELOC","ERCC4","ERCC5","ESRRA","ETNK1","EXT1","EXT2","FAM174B","FAM47C","FANCC","FANCD2","FANCE","FANCF","FANCG","FAT2","FBN2","FEN1","FES","FEV","FGD1","FGD5","FH","FKBP9","FLG","FLI1","FMN2","FNDC3B","FOXO4","FOXQ1","FOXR1","FSTL3","GABRA6","GATA1","GMPS","GNAI2","GPAA1","GPC3","GPS2","GRIN2D","GSK3B","H1-2","H1-4","H3-3B","H3-5","H4C9","HCAR3","HDAC4","HEY1","HLA-C","HLA-DQA1","HLF","HMGA1","HMGA2","HOXA11","HOXA13","HOXA9","HOXC11","HOXD11","HOXD13","HSP90AA1","HSP90AB1","HSPG2","HUWE1","IGF2BP2","IKZF3","ING1","IRAK1","IRF1","IRF2","IRF6","ISX","JUN","KANSL1","KCNJ5","KDM3B","KEL","KIF1A","KIF5B","KLHL6","KMT5A","KRT222","LARP4B","LCK","LDB1","LEMD2","LEPROTL1","LIFR","LMO1","LMO2","LPA","LPP","LSM14A","LYL1","LYN","MACF1","MAFB","MAP2K2","MAP2K7","MAP3K13","MAP3K4","MARK2","MB21D2","MCM7","MDGA2","MDM4","MED23","MEF2B","MIB2","MITF","MLF1","MLLT1","MLLT10","MLLT3","MLLT6","MN1","MRTFA","MSH3","MSN","MTCP1","MUTYH","MYB","MYCL","MYEOV","MYH11","MYO5A","MYOD1","NAB2","NACA","NBN","NCAPH","NCKIPSD","NCOA1","NCOA4","NDRG1","NFKBIA","NIN","NKTR","NPRL2","NR4A3","NRK","NT5C2","NT5C3A","NTHL1","NTRK2","NUMA1","NUP133","NUP214","NUP93","NUTM2D","NVL","NXF1","OLIG2","PARP4","PATZ1","PAX3","PBX1","PCM1","PCMTD1","PDGFB","PDS5B","PDZD2","PEG3","PER1","PER3","PGR","PHOX2B","PI4KA","PICALM","PIK3CG","PLCB4","PLXNB2","PLXNC1","POLG","POLRMT","POU2AF1","POU2F2","POU5F1","PPARG","PPP3CA","PRF1","PRKACA","PRKCD","PRKD1","PRKD2","PRPF40B","PRUNE2","PSPC1","PTPDC1","PTPN14","PTPN3","PWWP2A","RAD17","RAD50","RAD51B","RAP1B","RASA2","RBFOX1","RBM15","RBM38","RBM39","REL","RELA","REST","RFC1","RFWD3","RHEB","RHOB","RHOH","RHPN2","RIPK1","RMI2","RNF111","RNF213","RNF6","RPS3A","RRAGC","RRN3","RSPO3","S100A7","SAMHD1","SBDS","SCAF4","SDCBP","SDHAF2","SDHB","SDHC","SDHD","SET","SF1","SFPQ","SFRP4","SH3GL1","SHTN1","SIX2","SKI","SMARCA1","SMARCA2","SMARCE1","SMC3","SNX29","SOX2","SP140","SPTA1","SPTAN1","SRC","SRGAP3","SRSF3","SSX1","SSX2","SSX4","STAG1","STIL","SUSD2","SYK","TAF1","TAF1L","TCF3","TCF4","TENT5D","TFAP4","TFE3","TFEB","TFG","TFRC","TGFBR1","TLR4","TLX1","TLX3","TMEM127","TMPRSS2","TNFRSF17","TOP1","TOP2A","TPM3","TPR","TRIM27","TRIM49C","TRIP11","TXNIP","U2AF2","UBE2A","UBXN11","UNCX","USP44","USP7","WIF1","WNT1","WRN","XPA","XPC","YWHAE","ZBTB48","ZC3H12A","ZCCHC12","ZCCHC8","ZCRB1","ZFP36L2","ZNF133","ZNF208","ZNF274","ZNF395","ZNF479","ZNF626","ZNF680","ZNF721","ZNF780A","ZNF814","ZNF90","ZNF93","ZNG1C","ZXDB"],[25,90,91,92,207,238,139285,324,367,369,8289,196528,171023,55252,472,545,546,8312,567,8314,596,54880,673,672,675,84433,841,865,595,974,999,51755,1019,1026,1027,1029,1031,1050,1108,11200,23152,1387,10664,1499,8452,1540,57105,1654,23405,1788,1956,1964,1999,2033,2034,2042,2064,2065,2066,2068,2099,2146,2195,55294,2260,2263,2261,2322,3169,8880,2625,2767,2776,2778,3020,3265,3417,3418,3572,3575,3716,3717,3718,8242,7403,9817,3815,4297,58508,8085,3845,9113,26524,8216,5604,6416,4214,5594,4149,2122,9968,4221,4233,2475,4609,4613,4615,4627,9611,4763,4771,4780,4851,4853,4869,4893,7468,5079,55193,5093,5156,84295,5290,5291,5295,5292,5335,5395,5426,10721,8493,5518,5537,5573,11168,5727,5728,5781,5788,5789,5879,5885,5894,5925,8241,5979,387,54894,6146,6125,861,26040,29072,23451,4087,4089,6597,6598,8243,6662,8405,6427,10735,6794,79718,6926,6934,54790,7048,7128,7157,7248,7249,7307,7428,7490,7514,463,9203,27,57007,4301,3899,27125,84962,10000,286,2909,394,9639,55160,23365,57492,405,55729,476,492,8313,580,11177,8915,53335,64919,604,607,283149,9774,63035,330,57448,657,23476,29117,694,7832,695,776,23261,867,868,8030,894,896,1236,30835,29126,79577,1008,1009,1045,1106,4261,1213,8218,168975,4849,9125,1280,22849,64109,23373,1436,1441,1496,1500,1501,1523,7852,1558,1616,340578,1630,1643,4921,1656,54487,1660,29102,1879,3646,2000,2005,8178,1969,2045,2060,2071,2078,2115,2118,2119,2120,2130,51059,2175,355,120114,79633,2199,80204,2213,83417,2264,2272,201163,2316,2324,3170,668,2308,2309,27086,2521,2624,2735,10672,2262,2903,2913,2969,8358,3082,3091,3092,3105,3106,6927,3181,3229,3399,3551,10320,3636,3662,8471,3685,7994,23522,11143,5927,3791,9314,688,1316,9757,57082,90417,51176,121227,53353,346389,4094,10892,84441,4193,23269,4255,4292,4352,4436,2956,124540,4588,55728,26960,10499,9612,4773,4791,4794,25836,7080,3084,64324,54904,4914,4916,4928,256646,286530,26986,79728,80380,5159,5296,5324,5371,5378,5424,25913,639,63976,7799,80243,5579,5753,5757,5783,5777,5787,5796,11122,9444,5903,5910,5914,5921,9401,653489,6000,6016,6092,6098,6134,6197,22800,340419,862,6256,57167,6389,23067,9869,6446,10019,25942,140885,6495,10568,4088,6602,6608,27044,8651,6654,64321,11166,23013,6774,6777,6778,51684,23512,8148,6886,6887,6938,8115,7006,54855,7015,80312,7050,9967,7114,3371,8764,8626,7187,84231,8805,51592,8295,7253,51366,9098,9101,8239,7409,7454,65268,25937,7704,26137,51043,6935,677,7750,55422,353088,25925,79755,84133,8233,29974,10257,10006,2181,23305,6296,56999,4299,10142,208,213,217,80216,29123,338,9582,23092,84159,466,10533,4287,581,10018,83596,602,605,613,641,653,654,8019,83990,701,773,253559,811,124583,833,836,842,54897,863,23624,55749,57820,892,898,904,1233,940,965,970,972,973,1015,1021,84902,1107,55636,57680,91851,50515,6249,55783,7555,11064,1277,1289,1293,131873,1385,90993,64764,84699,51340,7812,114788,1464,1495,8454,1602,26528,1649,1662,1655,7913,1730,22894,1756,1785,1786,84444,1832,1840,80824,345930,8726,1915,1938,9343,1959,1960,1974,2006,6921,2072,2073,2101,55500,2131,2132,400451,442444,2176,2177,2178,2188,2189,2196,2201,2237,2242,54738,2245,152273,2271,11328,2312,2313,56776,64778,4303,94234,283150,10272,2559,2623,8833,2771,8733,2719,2874,2906,2932,3006,3008,3021,440093,8294,8843,9759,23462,3107,3117,3131,3159,8091,3207,3209,3205,3227,3237,3239,3320,3326,3339,10075,10644,22806,3621,3654,3659,3660,3664,91464,3725,284058,3762,51780,3792,547,3799,89857,387893,125113,23185,3932,8861,221496,23484,3977,4004,4005,4018,4026,26065,4066,4067,23499,9935,5605,5609,9175,4216,2011,151963,4176,161357,4194,9439,100271849,142678,4286,4291,4298,8028,4300,4302,4330,57591,4437,4478,4515,4595,4602,4610,26579,4629,4644,4654,4665,4666,4683,23397,51517,8648,8031,10397,4792,51199,4820,10641,8013,203447,22978,51251,4913,4915,4926,55746,8021,9688,728130,4931,10482,10215,143,23598,5077,5087,5108,115294,5155,23047,23037,5178,5187,8863,5241,8929,5297,8301,5294,5332,23654,10154,5428,5442,5450,5452,5460,5468,5530,5551,5566,5580,5587,25865,25766,158471,55269,138639,5784,5774,114825,5884,10111,5890,5908,5922,54715,64783,55544,9584,5966,5970,5978,5981,55159,6009,388,399,85415,8737,116028,54778,57674,6049,6189,64121,54700,84870,6278,25939,51119,57466,6386,54949,6390,6391,6392,6418,7536,6421,6424,6455,57698,10736,6497,6594,6595,6605,9126,92017,6657,11262,6708,6709,6714,9901,6428,6756,6757,6759,10274,6491,56241,6850,6872,138474,6929,6925,169966,7023,7030,7942,10342,7037,7046,7099,3195,30012,55654,7113,608,7150,7153,7170,7175,5987,642612,9321,10628,11338,7319,91544,340260,84101,7874,11197,7471,7486,7507,7508,7531,3104,80149,170261,55596,85437,678,7692,7757,10782,55893,90827,199777,340252,170960,284323,730051,7643,81931,445571,158586],["ABL proto-oncogene 1, non-receptor tyrosine kinase","activin A receptor type 1","activin A receptor type 1B","activin A receptor type 2A","AKT serine/threonine kinase 1","ALK receptor tyrosine kinase","APC membrane recruitment protein 1","APC regulator of Wnt signaling pathway","androgen receptor","A-Raf proto-oncogene, serine/threonine kinase","AT-rich interaction domain 1A","AT-rich interaction domain 2","ASXL transcriptional regulator 1","ASXL transcriptional regulator 2","ATM serine/threonine kinase","ATR checkpoint kinase","ATRX chromatin remodeler","axin 1","beta-2-microglobulin","BRCA1 associated deubiquitinase 1","BCL2 apoptosis regulator","BCL6 corepressor","B-Raf proto-oncogene, serine/threonine kinase","BRCA1 DNA repair associated","BRCA2 DNA repair associated","caspase recruitment domain family member 11","caspase 8","core-binding factor subunit beta","cyclin D1","CD79b molecule","cadherin 1","cyclin dependent kinase 12","cyclin dependent kinase 4","cyclin dependent kinase inhibitor 1A","cyclin dependent kinase inhibitor 1B","cyclin dependent kinase inhibitor 2A","cyclin dependent kinase inhibitor 2C","CCAAT enhancer binding protein alpha","chromodomain helicase DNA binding protein 4","checkpoint kinase 2","capicua transcriptional repressor","CREB binding lysine acetyltransferase","CCCTC-binding factor","catenin beta 1","cullin 3","CYLD lysine 63 deubiquitinase","cysteinyl leukotriene receptor 2","DEAD-box helicase 3 X-linked","dicer 1, ribonuclease III","DNA methyltransferase 3 alpha","epidermal growth factor receptor","eukaryotic translation initiation factor 1A X-linked","E74 like ETS transcription factor 3","EP300 lysine acetyltransferase","endothelial PAS domain protein 1","EPH receptor A3","erb-b2 receptor tyrosine kinase 2","erb-b2 receptor tyrosine kinase 3","erb-b2 receptor tyrosine kinase 4","ERCC excision repair 2, TFIIH core complex helicase subunit","estrogen receptor 1","enhancer of zeste 2 polycomb repressive complex 2 subunit","FAT atypical cadherin 1","F-box and WD repeat domain containing 7","fibroblast growth factor receptor 1","fibroblast growth factor receptor 2","fibroblast growth factor receptor 3","fms related receptor tyrosine kinase 3","forkhead box A1","far upstream element binding protein 1","GATA binding protein 3","G protein subunit alpha 11","G protein subunit alpha q","GNAS complex locus","H3.3 histone A","HRas proto-oncogene, GTPase","isocitrate dehydrogenase (NADP(+)) 1","isocitrate dehydrogenase (NADP(+)) 2","interleukin 6 cytokine family signal transducer","interleukin 7 receptor","Janus kinase 1","Janus kinase 2","Janus kinase 3","lysine demethylase 5C","lysine demethylase 6A","kelch like ECH associated protein 1","KIT proto-oncogene, receptor tyrosine kinase","lysine methyltransferase 2A","lysine methyltransferase 2C","lysine methyltransferase 2D","KRAS proto-oncogene, GTPase","large tumor suppressor kinase 1","large tumor suppressor kinase 2","leucine zipper like post translational regulator 1","mitogen-activated protein kinase kinase 1","mitogen-activated protein kinase kinase 4","mitogen-activated protein kinase kinase kinase 1","mitogen-activated protein kinase 1","MYC associated transcriptional regulator X","MDS1 and EVI1 complex locus","mediator complex subunit 12","menin 1","MET proto-oncogene, receptor tyrosine kinase","mechanistic target of rapamycin kinase","MYC proto-oncogene, bHLH transcription factor","MYCN proto-oncogene, bHLH transcription factor","MYD88 innate immune signal transduction adaptor","myosin heavy chain 9","nuclear receptor corepressor 1","neurofibromin 1","NF2, moesin-ezrin-radixin like (MERLIN) tumor suppressor","NFE2 like bZIP transcription factor 2","notch receptor 1","notch receptor 2","nucleophosmin 1","NRAS proto-oncogene, GTPase","nuclear receptor binding SET domain protein 2","paired box 5","polybromo 1","poly(rC) binding protein 1","platelet derived growth factor receptor alpha","PHD finger protein 6","phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha","phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit beta","phosphoinositide-3-kinase regulatory subunit 1","Pim-1 proto-oncogene, serine/threonine kinase","phospholipase C gamma 1","PMS1 homolog 2, mismatch repair system component","DNA polymerase epsilon, catalytic subunit","DNA polymerase theta","protein phosphatase, Mg2+/Mn2+ dependent 1D","protein phosphatase 2 scaffold subunit Aalpha","protein phosphatase 6 catalytic subunit","protein kinase cAMP-dependent type I regulatory subunit alpha","PC4 and SRSF1 interacting protein 1","patched 1","phosphatase and tensin homolog","protein tyrosine phosphatase non-receptor type 11","protein tyrosine phosphatase receptor type C","protein tyrosine phosphatase receptor type D","Rac family small GTPase 1","RAD21 cohesin complex component","Raf-1 proto-oncogene, serine/threonine kinase","RB transcriptional corepressor 1","RNA binding motif protein 10","ret proto-oncogene","ras homolog family member A","ring finger protein 43","ribosomal protein L22","ribosomal protein L5","RUNX family transcription factor 1","SET binding protein 1","SET domain containing 2, histone lysine methyltransferase","splicing factor 3b subunit 1","SMAD family member 2","SMAD family member 4","SWI/SNF related BAF chromatin remodeling complex subunit ATPase 4","SWI/SNF related BAF chromatin remodeling complex subunit B1","structural maintenance of chromosomes 1A","SRY-box transcription factor 9","speckle type BTB/POZ protein","serine and arginine rich splicing factor 2","STAG2 cohesin complex component","serine/threonine kinase 11","TBL1X/Y related 1","T-box transcription factor 3","transcription factor 7 like 2","tet methylcytosine dioxygenase 2","transforming growth factor beta receptor 2","TNF alpha induced protein 3","tumor protein p53","TSC complex subunit 1","TSC complex subunit 2","U2 small nuclear RNA auxiliary factor 1","von Hippel-Lindau tumor suppressor","WT1 transcription factor","exportin 1","zinc finger homeobox 3","zinc finger MYM-type containing 3","ABL proto-oncogene 2, non-receptor tyrosine kinase","atypical chemokine receptor 3","afadin, adherens junction formation factor","ALF transcription elongation factor 3","ALF transcription elongation factor 4","ajuba LIM protein","AKT serine/threonine kinase 3","ankyrin 1","Rho GTPase activating protein 35","Rho GTPase activating protein 5","Rho guanine nucleotide exchange factor 10","Rho guanine nucleotide exchange factor 10 like","Rho guanine nucleotide exchange factor 12","AT-rich interaction domain 1B","aryl hydrocarbon receptor nuclear translocator","activating transcription factor 7 interacting protein","ATPase Na+/K+ transporting subunit alpha 1","ATPase plasma membrane Ca2+ transporting 3","axin 2","BRCA1 associated RING domain 1","bromodomain adjacent to zinc finger domain 1A","BCL10 immune signaling adaptor","BCL11 transcription factor A","BCL11 transcription factor B","BCL6 transcription repressor","BCL9 transcription coactivator","BCL9 like","BCL2 associated transcription factor 1","BCL6 corepressor like 1","baculoviral IAP repeat containing 3","baculoviral IAP repeat containing 6","bone morphogenetic protein receptor type 1A","bromodomain containing 4","bromodomain containing 7","BTG anti-proliferation factor 1","BTG anti-proliferation factor 2","Bruton tyrosine kinase","calcium voltage-gated channel subunit alpha1 D","calmodulin binding transcription activator 1","Cbl proto-oncogene","Cbl proto-oncogene B","coiled-coil domain containing 6","cyclin D2","cyclin D3","C-C motif chemokine receptor 7","CD209 molecule","CD274 molecule","cell division cycle 73","cadherin 10","cadherin 11","caudal type homeobox 2","chromodomain helicase DNA binding protein 2","class II major histocompatibility complex transactivator","clathrin heavy chain","clathrin heavy chain like 1","cyclic nucleotide binding domain containing 1","CCR4-NOT transcription complex subunit 3","CCR4-NOT transcription complex subunit 9","collagen type II alpha 1 chain","cytoplasmic polyadenylation element binding protein 3","cytokine receptor like factor 2","CREB regulated transcription coactivator 1","colony stimulating factor 1 receptor","colony stimulating factor 3 receptor","catenin alpha 2","catenin delta 1","catenin delta 2","cut like homeobox 1","C-X-C motif chemokine receptor 4","cytochrome P450 family 2 subfamily C member 8","death domain associated protein","DDB1 and CUL4 associated factor 12 like 2","DCC netrin 1 receptor","damage specific DNA binding protein 2","discoidin domain receptor tyrosine kinase 2","DEAD-box helicase 6","DGCR8 microprocessor complex subunit","DExH-box helicase 9","drosha ribonuclease III","EBF transcription factor 1","eukaryotic translation initiation factor 3 subunit E","E74 like ETS transcription factor 4","ETS transcription factor ELK4","elongation factor for RNA polymerase II","EPH receptor A2","EPH receptor A7","epidermal growth factor receptor pathway substrate 15","ERCC excision repair 3, TFIIH core complex helicase subunit","ETS transcription factor ERG","ETS variant transcription factor 1","ETS variant transcription factor 4","ETS variant transcription factor 5","ETS variant transcription factor 6","EWS RNA binding protein 1","family with sequence similarity 135 member B","FA complementation group A","Fas cell surface death receptor","FAT atypical cadherin 3","FAT atypical cadherin 4","fibulin 2","F-box protein 11","Fc gamma receptor IIb","Fc receptor like 4","fibroblast growth factor receptor 4","fragile histidine triad diadenosine triphosphatase","folliculin","filamin A","fms related receptor tyrosine kinase 4","forkhead box A2","forkhead box L2","forkhead box O1","forkhead box O3","forkhead box P1","FUS RNA binding protein","GATA binding protein 2","GLI family zinc finger 1","G protein subunit alpha 13","glypican 5","glutamate ionotropic receptor NMDA type subunit 2A","glutamate metabotropic receptor 3","general transcription factor IIi","H3 clustered histone 2","hepatocyte growth factor","hypoxia inducible factor 1 subunit alpha","huntingtin interacting protein 1","major histocompatibility complex, class I, A","major histocompatibility complex, class I, B","HNF1 homeobox A","heterogeneous nuclear ribonucleoprotein A2/B1","homeobox C13","inhibitor of DNA binding 3","inhibitor of nuclear factor kappa B kinase subunit beta","IKAROS family zinc finger 1","inositol polyphosphate phosphatase like 1","interferon regulatory factor 4","insulin receptor substrate 4","integrin subunit alpha V","lysine acetyltransferase 6A","lysine acetyltransferase 6B","lysine acetyltransferase 7","lysine demethylase 5A","kinase insert domain receptor","KLF transcription factor 4","KLF transcription factor 5","KLF transcription factor 6","lysine methyltransferase 2B","kinetochore scaffold 1","kinetochore localized astrin (SPAG5) binding protein","lymphoid enhancer binding factor 1","leucine rich repeats and immunoglobulin like domains 3","LDL receptor related protein 1B","MET transcriptional regulator MACC1","MAF bZIP transcription factor","MALT1 paracaspase","mastermind like transcriptional coactivator 2","MDM2 proto-oncogene","MAX dimerization protein MGA","O-6-methylguanine-DNA methyltransferase","mutL homolog 1","MPL proto-oncogene, thrombopoietin receptor","mutS homolog 2","mutS homolog 6","musashi RNA binding protein 2","mucin 6, oligomeric mucus/gel-forming (gene/pseudogene)","NEDD4 binding protein 2","neurobeachin","nuclear receptor coactivator 2","nuclear receptor corepressor 2","nuclear factor of activated T cells 2","nuclear factor kappa B subunit 2","NFKB inhibitor epsilon","NIPBL cohesin loading factor","NK2 homeobox 1","neuregulin 1","nuclear receptor binding SET domain protein 1","nuclear receptor binding SET domain protein 3","neurotrophic receptor tyrosine kinase 1","neurotrophic receptor tyrosine kinase 3","nucleoporin 98 and 96 precursor","NUT midline carcinoma family member 1","P2Y receptor family member 8","poly(A) binding protein cytoplasmic 1","partner and localizer of BRCA2","programmed cell death 1 ligand 2","platelet derived growth factor receptor beta","phosphoinositide-3-kinase regulatory subunit 2","PLAG1 zinc finger","PML nuclear body scaffold","PMS1 homolog 1, mismatch repair system component","DNA polymerase delta 1, catalytic subunit","protection of telomeres 1","PR/SET domain 1","PR/SET domain 16","PR/SET domain 2","phosphatidylinositol-3,4,5-trisphosphate dependent Rac exchange factor 2","protein kinase C beta","protein tyrosine kinase 6","prothymosin alpha","protein tyrosine phosphatase non-receptor type 13","protein tyrosine phosphatase non-receptor type 6","protein tyrosine phosphatase receptor type B","protein tyrosine phosphatase receptor type K","protein tyrosine phosphatase receptor type T","QKI, KH domain containing RNA binding","RAN binding protein 2","Rap1 GTPase-GDP dissociation stimulator 1","retinoic acid receptor alpha","RAS p21 protein activator 1","RecQ like helicase 4","RANBP2 like and GRIP domain containing 3","regulator of G protein signaling 7","Ras like without CAAX 1","roundabout guidance receptor 2","ROS proto-oncogene 1, receptor tyrosine kinase","ribosomal protein L10","ribosomal protein S6 kinase A3","RAS related 2","R-spondin 2","RUNX1 partner transcriptional co-repressor 1","retinoid X receptor alpha","spalt like transcription factor 4","succinate dehydrogenase complex flavoprotein subunit A","SET domain containing 1B, histone lysine methyltransferase","SET domain bifurcated histone lysine methyltransferase 1","serum/glucocorticoid regulated kinase 1","SH2B adaptor protein 3","SIN3 transcription regulator family member A","signal regulatory protein alpha","SIX homeobox 1","solute carrier family 34 member 2","SMAD family member 3","SWI/SNF related BAF chromatin remodeling complex subunit D1","smoothened, frizzled class receptor","staphylococcal nuclease and tudor domain containing 1","suppressor of cytokine signaling 1","SOS Ras/Rac guanine nucleotide exchange factor 1","SRY-box transcription factor 17","SRY-box transcription factor 21","spen family transcriptional repressor","signal transducer and activator of transcription 3","signal transducer and activator of transcription 5B","signal transducer and activator of transcription 6","SUFU negative regulator of hedgehog signaling","SUZ12 polycomb repressive complex 2 subunit","TATA-box binding protein associated factor 15","TAL bHLH transcription factor 1, erythroid differentiation factor","TAL bHLH transcription factor 2","transcription factor 12","TCL1 family AKT coactivator A","tec protein tyrosine kinase","terminal nucleotidyltransferase 5C","telomerase reverse transcriptase","tet methylcytosine dioxygenase 1","TGFB induced factor homeobox 1","thyroid hormone receptor associated protein 3","thymosin beta 4 X-linked","tenascin C","TNF receptor superfamily member 14","tumor protein p63","TNF receptor associated factor 3","TNF receptor associated factor 7","tripartite motif containing 24","tripartite motif containing 33","transformation/transcription domain associated protein","thyroid stimulating hormone receptor","ubiquitin protein ligase E3 component n-recognin 5","ubiquitin specific peptidase 6","ubiquitin specific peptidase 8","ubiquitin specific peptidase 9 X-linked","vav guanine nucleotide exchange factor 1","WASP actin nucleation promoting factor","WNK lysine deficient protein kinase 2","WW domain containing transcription regulator 1","zinc finger and BTB domain containing 16","zinc finger and BTB domain containing 20","zinc finger and BTB domain containing 7B","zinc finger E-box binding homeobox 1","ZFP36 like 1 zinc finger CCCH-type","zinc finger MYM-type containing 2","zinc finger protein 331","zinc finger protein 429","zinc finger protein 521","zinc finger protein 750","zinc and ring finger 3","zinc finger CCCH-type, RNA binding motif and serine/arginine rich 2","APOBEC1 complementation factor","ATP binding cassette subfamily C member 4 (PEL blood group)","abl interactor 1","acyl-CoA synthetase long chain family member 3","acyl-CoA synthetase long chain family member 6","acyl-CoA synthetase medium chain family member 3","ADAM metallopeptidase with thrombospondin type 1 motif 9","ALF transcription elongation factor 1","A-kinase anchoring protein 9","AKT serine/threonine kinase 2","albumin","aldehyde dehydrogenase 2 family member","alpha kinase 1","ankyrin repeat domain 11","apolipoprotein B","apolipoprotein B mRNA editing enzyme catalytic subunit 3B","Rho GTPase activating protein 26","AT-rich interaction domain 5B","activating transcription factor 1","autophagy related 7","ataxin 3","BCL2 associated X, apoptosis regulator","BCL2 like 11","BCL2 like 12","BCL3 transcription coactivator","BAF chromatin remodeling complex subunit BCL7A","BCR activator of RhoGEF and GTPase","BLM RecQ like helicase","bone morphogenetic protein 5","bone morphogenetic protein 6","bromodomain containing 3","BRCA1 interacting DNA helicase 1","BUB1 mitotic checkpoint serine/threonine kinase B","calcium voltage-gated channel subunit alpha1 A","cell adhesion molecule 2","calreticulin","calcium activated nucleotidase 1","cysteinyl-tRNA synthetase 1","caspase 3","caspase 9","castor zinc finger 1","CBFA2/RUNX1 partner transcriptional co-repressor 3","Cbl proto-oncogene C","cell division cycle and apoptosis regulator 1","cyclin B1 interacting protein 1","cyclin C","cyclin E1","cyclin T1","C-C motif chemokine receptor 4","CD28 molecule","CD58 molecule","CD70 molecule","CD74 molecule","CD79a molecule","cadherin 17","cyclin dependent kinase 6","centrosomal protein 89","chromodomain helicase DNA binding protein 3","chromodomain helicase DNA binding protein 7","chromodomain helicase DNA binding protein 8","chordin like 1","carbohydrate sulfotransferase 11","CAP-Gly domain containing linker protein 1","cap methyltransferase 2","CCHC-type zinc finger nucleic acid binding protein","centriolin","collagen type I alpha 1 chain","collagen type V alpha 1 chain","collagen type VI alpha 3 chain","collagen type VI alpha 6 chain","cAMP responsive element binding protein 1","cAMP responsive element binding protein 3 like 1","cAMP responsive element binding protein 3 like 2","cAMP responsive element binding protein 3 like 3","crooked neck pre-mRNA splicing factor 1","cold shock domain containing E1","CUB and Sushi multiple domains 3","chondroitin sulfate proteoglycan 4","catenin alpha 1","cullin 1","dachshund family transcription factor 1","DAZ associated protein 1","DNA damage inducible transcript 3","DEAD-box helicase 10","DEAD-box helicase 5","DEK proto-oncogene","diaphanous related formin 2","DIS3 exosome endoribonuclease and 3'-5' exoribonuclease","dystrophin","dynamin 2","DNA methyltransferase 1","DOT1 like histone lysine methyltransferase","desmoplakin","deltex E3 ubiquitin ligase 1","dual specificity phosphatase 16","epithelial cell transforming 2 like","embryonic ectoderm development","eukaryotic translation elongation factor 1 alpha 1","eukaryotic translation elongation factor 2","elongation factor Tu GTP binding domain containing 2","early growth response 2","early growth response 3","eukaryotic translation initiation factor 4A2","elastin","elongin C","ERCC excision repair 4, endonuclease catalytic subunit","ERCC excision repair 5, endonuclease","estrogen related receptor alpha","ethanolamine kinase 1","exostosin glycosyltransferase 1","exostosin glycosyltransferase 2","family with sequence similarity 174 member B","family with sequence similarity 47 member C","FA complementation group C","FA complementation group D2","FA complementation group E","FA complementation group F","FA complementation group G","FAT atypical cadherin 2","fibrillin 2","flap structure-specific endonuclease 1","FES proto-oncogene, tyrosine kinase","FEV transcription factor, ETS family member","FYVE, RhoGEF and PH domain containing 1","FYVE, RhoGEF and PH domain containing 5","fumarate hydratase","FKBP prolyl isomerase 9","filaggrin","Fli-1 proto-oncogene, ETS transcription factor","formin 2","fibronectin type III domain containing 3B","forkhead box O4","forkhead box Q1","forkhead box R1","follistatin like 3","gamma-aminobutyric acid type A receptor subunit alpha6","GATA binding protein 1","guanosine monophosphate synthase","G protein subunit alpha i2","glycosylphosphatidylinositol anchor attachment 1","glypican 3","G protein pathway suppressor 2","glutamate ionotropic receptor NMDA type subunit 2D","glycogen synthase kinase 3 beta","H1.2 linker histone, cluster member","H1.4 linker histone, cluster member","H3.3 histone B","H3.5 histone","H4 clustered histone 9","hydroxycarboxylic acid receptor 3","histone deacetylase 4","hes related family bHLH transcription factor with YRPW motif 1","major histocompatibility complex, class I, C","major histocompatibility complex, class II, DQ alpha 1","HLF transcription factor, PAR bZIP family member","high mobility group AT-hook 1","high mobility group AT-hook 2","homeobox A11","homeobox A13","homeobox A9","homeobox C11","homeobox D11","homeobox D13","heat shock protein 90 alpha family class A member 1","heat shock protein 90 alpha family class B member 1","heparan sulfate proteoglycan 2","HECT, UBA and WWE domain containing E3 ubiquitin protein ligase 1","insulin like growth factor 2 mRNA binding protein 2","IKAROS family zinc finger 3","inhibitor of growth family member 1","interleukin 1 receptor associated kinase 1","interferon regulatory factor 1","interferon regulatory factor 2","interferon regulatory factor 6","intestine specific homeobox","Jun proto-oncogene, AP-1 transcription factor subunit","KAT8 regulatory NSL complex subunit 1","potassium inwardly rectifying channel subfamily J member 5","lysine demethylase 3B","Kell metallo-endopeptidase (Kell blood group)","kinesin family member 1A","kinesin family member 5B","kelch like family member 6","lysine methyltransferase 5A","keratin 222","La ribonucleoprotein 4B","LCK proto-oncogene, Src family tyrosine kinase","LIM domain binding 1","LEM domain nuclear envelope protein 2","leptin receptor overlapping transcript like 1","LIF receptor subunit alpha","LIM domain only 1","LIM domain only 2","lipoprotein(a)","LIM domain containing preferred translocation partner in lipoma","LSM14A mRNA processing body assembly factor","LYL1 basic helix-loop-helix family member","LYN proto-oncogene, Src family tyrosine kinase","microtubule actin crosslinking factor 1","MAF bZIP transcription factor B","mitogen-activated protein kinase kinase 2","mitogen-activated protein kinase kinase 7","mitogen-activated protein kinase kinase kinase 13","mitogen-activated protein kinase kinase kinase 4","microtubule affinity regulating kinase 2","Mab-21 domain containing 2","minichromosome maintenance complex component 7","MAM domain containing glycosylphosphatidylinositol anchor 2","MDM4 regulator of p53","mediator complex subunit 23","myocyte enhancer factor 2B","MIB E3 ubiquitin protein ligase 2","melanocyte inducing transcription factor","myeloid leukemia factor 1","MLLT1 super elongation complex subunit","MLLT10 histone lysine methyltransferase DOT1L cofactor","MLLT3 super elongation complex subunit","MLLT6, PHD finger containing","MN1 proto-oncogene, transcriptional regulator","myocardin related transcription factor A","mutS homolog 3","moesin","mature T cell proliferation 1","mutY DNA glycosylase","MYB proto-oncogene, transcription factor","MYCL proto-oncogene, bHLH transcription factor","myeloma overexpressed","myosin heavy chain 11","myosin VA","myogenic differentiation 1","NGFI-A binding protein 2","nascent polypeptide associated complex subunit alpha","nibrin","non-SMC condensin I complex subunit H","NCK interacting protein with SH3 domain","nuclear receptor coactivator 1","nuclear receptor coactivator 4","N-myc downstream regulated 1","NFKB inhibitor alpha","ninein","natural killer cell triggering receptor","NPR2 like, GATOR1 complex subunit","nuclear receptor subfamily 4 group A member 3","Nik related kinase","5'-nucleotidase, cytosolic II","5'-nucleotidase, cytosolic IIIA","nth like DNA glycosylase 1","neurotrophic receptor tyrosine kinase 2","nuclear mitotic apparatus protein 1","nucleoporin 133","nucleoporin 214","nucleoporin 93","NUT family member 2D","nuclear VCP like","nuclear RNA export factor 1","oligodendrocyte transcription factor 2","poly(ADP-ribose) polymerase family member 4","POZ/BTB and AT hook containing zinc finger 1","paired box 3","PBX homeobox 1","pericentriolar material 1","protein-L-isoaspartate (D-aspartate) O-methyltransferase domain containing 1","platelet derived growth factor subunit B","PDS5 cohesin associated factor B","PDZ domain containing 2","paternally expressed 3","period circadian regulator 1","period circadian regulator 3","progesterone receptor","paired like homeobox 2B","phosphatidylinositol 4-kinase alpha","phosphatidylinositol binding clathrin assembly protein","phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit gamma","phospholipase C beta 4","plexin B2","plexin C1","DNA polymerase gamma, catalytic subunit","RNA polymerase mitochondrial","POU class 2 homeobox associating factor 1","POU class 2 homeobox 2","POU class 5 homeobox 1","peroxisome proliferator activated receptor gamma","protein phosphatase 3 catalytic subunit alpha","perforin 1","protein kinase cAMP-activated catalytic subunit alpha","protein kinase C delta","protein kinase D1","protein kinase D2","pre-mRNA processing factor 40B","prune homolog 2 with BCH domain","paraspeckle component 1","protein tyrosine phosphatase domain containing 1","protein tyrosine phosphatase non-receptor type 14","protein tyrosine phosphatase non-receptor type 3","PWWP domain containing 2A","RAD17 checkpoint clamp loader component","RAD50 double strand break repair protein","RAD51 paralog B","RAP1B, member of RAS oncogene family","RAS p21 protein activator 2","RNA binding fox-1 homolog 1","RNA binding motif protein 15","RNA binding motif protein 38","RNA binding motif protein 39","REL proto-oncogene, NF-kB subunit","RELA proto-oncogene, NF-kB subunit","RE1 silencing transcription factor","replication factor C subunit 1","ring finger and WD repeat domain 3","Ras homolog, mTORC1 binding","ras homolog family member B","ras homolog family member H","rhophilin Rho GTPase binding protein 2","receptor interacting serine/threonine kinase 1","RecQ mediated genome instability 2","ring finger protein 111","ring finger protein 213","ring finger protein 6","ribosomal protein S3A","Ras related GTP binding C","RNA polymerase I transcription factor RRN3","R-spondin 3","S100 calcium binding protein A7","SAM and HD domain containing deoxynucleoside triphosphate triphosphohydrolase 1","SBDS ribosome maturation factor","SR-related CTD associated factor 4","syndecan binding protein","succinate dehydrogenase complex assembly factor 2","succinate dehydrogenase complex iron sulfur subunit B","succinate dehydrogenase complex subunit C","succinate dehydrogenase complex subunit D","SET nuclear proto-oncogene","splicing factor 1","splicing factor proline and glutamine rich","secreted frizzled related protein 4","SH3 domain containing GRB2 like 1, endophilin A2","shootin 1","SIX homeobox 2","SKI proto-oncogene","SNF2 related chromatin remodeling ATPase 1","SWI/SNF related BAF chromatin remodeling complex subunit ATPase 2","SWI/SNF related BAF chromatin remodeling complex subunit E1","structural maintenance of chromosomes 3","sorting nexin 29","SRY-box transcription factor 2","SP140 nuclear body protein","spectrin alpha, erythrocytic 1","spectrin alpha, non-erythrocytic 1","SRC proto-oncogene, non-receptor tyrosine kinase","SLIT-ROBO Rho GTPase activating protein 3","serine and arginine rich splicing factor 3","SSX family member 1","SSX family member 2","SSX family member 4","STAG1 cohesin complex component","STIL centriolar assembly protein","sushi domain containing 2","spleen associated tyrosine kinase","TATA-box binding protein associated factor 1","TATA-box binding protein associated factor 1 like","transcription factor 3","transcription factor 4","terminal nucleotidyltransferase 5D","transcription factor AP-4","transcription factor binding to IGHM enhancer 3","transcription factor EB","trafficking from ER to golgi regulator","transferrin receptor","transforming growth factor beta receptor 1","toll like receptor 4","T cell leukemia homeobox 1","T cell leukemia homeobox 3","transmembrane protein 127","transmembrane serine protease 2","TNF receptor superfamily member 17","DNA topoisomerase I","DNA topoisomerase II alpha","tropomyosin 3","translocated promoter region, nuclear basket protein","tripartite motif containing 27","tripartite motif containing 49C","thyroid hormone receptor interactor 11","thioredoxin interacting protein","U2 small nuclear RNA auxiliary factor 2","ubiquitin conjugating enzyme E2 A","UBX domain protein 11","UNC homeobox","ubiquitin specific peptidase 44","ubiquitin specific peptidase 7","Wnt inhibitory factor 1","Wnt family member 1","WRN RecQ like helicase","XPA, DNA damage recognition and repair factor","XPC complex subunit, DNA damage recognition and repair factor","tyrosine 3-monooxygenase/tryptophan 5-monooxygenase activation protein epsilon","zinc finger and BTB domain containing 48","zinc finger CCCH-type containing 12A","zinc finger CCHC-type containing 12","zinc finger CCHC-type containing 8","zinc finger CCHC-type and RNA binding motif containing 1","ZFP36 like 2 zinc finger CCCH-type","zinc finger protein 133","zinc finger protein 208","zinc finger protein 274","zinc finger protein 395","zinc finger protein 479","zinc finger protein 626","zinc finger protein 680","zinc finger protein 721","zinc finger protein 780A","zinc finger protein 814","zinc finger protein 90","zinc finger protein 93","Zn regulated GTPase metalloprotein activator 1C","zinc finger X-linked duplicated B"],["protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding","protein_coding"],[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,false,true,false,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,false,true,true,true,true,true,true,false,false,true,true,true,false,false,false,false,false,true,false,false,false,false,false,true,true,false,false,true,false,false,false,false,true,false,true,false,false,false,true,false,false,true,false,false,false,true,false,false,true,false,false,false,false,false,true,false,true,false,true,false,true,true,false,true,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,false,false,false,false,false,true,false,false,false,true,true,true,true,false,false,false,false,true,true,false,true,true,false,false,false,true,false,false,false,true,false,false,false,false,false,false,true,true,false,false,false,true,true,false,false,true,false,true,true,false,false,false,false,false,false,true,true,true,false,false,false,true,false,false,false,false,true,true,true,false,true,true,false,false,false,false,false,false,false,false,false,true,true,true,false,false,true,true,true,true,false,false,false,false,false,false,true,false,false,true,true,true,false,false,false,true,false,false,true,false,false,true,false,true,false,true,false,false,false,true,false,false,true,true,true,true,false,true,true,true,false,false,true,false,true,true,false,false,false,true,false,false,false,false,true,true,true,false,false,true,false,true,true,true,false,false,true,true,true,true,false,true,false,true,false,true,true,false,true,false,true,true,true,false,true,false,false,false,true,true,false,false,true,true,false,true,false,false,true,true,false,false,false,true,false,false,false,true,false,false,true,false,false,true,true,true,false,true,true,false,true,true,true,false,true,false,true,true,true,true,true,true,false,true,true,false,false,false,false,false,true,true,false,false,true,true,true,true,true,false,false,true,false,false,true,false,false,false,false,false,false,false,false,false,true,false,false,false,true,false,false,true,false,true,false,false,false,true,false,false,false,false,false,false,true,false,false,true,false,true,false,true,false,false,true,true,true,false,false,false,false,true,false,true,true,false,true,false,true,true,false,true,true,true,false,false,true,false,true,false,false,false,false,true,false,false,true,true,false,false,true,true,true,false,true,true,true,true,true,true,true,false,true],["Loss_of_Function","Activating","Loss_of_Function","Loss_of_Function&Activating","Activating","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Activating&Ambiguous","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function&Activating","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function&Ambiguous&Activating","Activating&Loss_of_Function","Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating","Loss_of_Function&Activating","Activating&Loss_of_Function&Ambiguous","Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Ambiguous&Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating","Activating&Loss_of_Function","Loss_of_Function&Activating&Ambiguous","Activating&Loss_of_Function","Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Ambiguous&Loss_of_Function","Loss_of_Function&Activating","Activating","Activating&Loss_of_Function","Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Ambiguous&Loss_of_Function","Activating","Loss_of_Function&Ambiguous&Activating","Loss_of_Function&Activating","Activating","Activating","Activating","Activating&Loss_of_Function","Ambiguous&Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function&Ambiguous","Loss_of_Function&Activating&Ambiguous","Loss_of_Function&Activating","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Activating","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function&Activating&Ambiguous","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Ambiguous&Activating","Activating&Loss_of_Function","Activating","Ambiguous","Activating","Loss_of_Function&Activating","Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Activating&Loss_of_Function&Ambiguous","Activating&Loss_of_Function","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating","Loss_of_Function&Activating","Activating","Loss_of_Function&Ambiguous&Activating","Loss_of_Function&Activating","Loss_of_Function&Activating&Ambiguous","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function&Activating&Ambiguous","Loss_of_Function&Activating","Activating","Loss_of_Function","Loss_of_Function&Activating","Activating","Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function","Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Ambiguous","Loss_of_Function&Activating&Ambiguous","Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating","Loss_of_Function","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function&Activating","Activating&Loss_of_Function&Ambiguous","Loss_of_Function","Loss_of_Function","Activating","Activating&Loss_of_Function",null,"Activating","Loss_of_Function","Loss_of_Function&Activating","Activating","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function","Ambiguous&Activating","Loss_of_Function&Activating&Ambiguous","Loss_of_Function","Activating","Loss_of_Function","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating","Loss_of_Function&Activating","Activating","Activating&Loss_of_Function","Activating","Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Activating","Loss_of_Function&Activating&Ambiguous","Activating&Loss_of_Function","Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function",null,"Activating","Activating","Loss_of_Function","Activating","Loss_of_Function","Activating","Loss_of_Function","Loss_of_Function&Activating","Activating&Ambiguous&Loss_of_Function",null,"Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating","Loss_of_Function","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating","Loss_of_Function","Activating","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function&Activating","Activating","Loss_of_Function","Activating","Activating&Loss_of_Function","Activating","Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating&Ambiguous","Loss_of_Function&Activating&Ambiguous","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function","Loss_of_Function","Activating","Loss_of_Function",null,"Activating&Loss_of_Function","Loss_of_Function","Activating","Activating&Loss_of_Function","Activating","Loss_of_Function","Ambiguous","Activating&Loss_of_Function&Ambiguous","Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Ambiguous&Activating","Activating&Loss_of_Function","Activating","Activating&Loss_of_Function","Activating","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function&Activating","Activating","Loss_of_Function","Loss_of_Function","Activating","Loss_of_Function&Activating","Loss_of_Function","Activating","Activating&Loss_of_Function","Loss_of_Function","Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Activating","Activating&Loss_of_Function","Activating&Loss_of_Function&Ambiguous","Activating&Loss_of_Function","Loss_of_Function","Loss_of_Function","Loss_of_Function","Activating","Loss_of_Function","Loss_of_Function","Activating&Loss_of_Function&Ambiguous","Loss_of_Function","Activating","Activating","Ambiguous&Loss_of_Function","Activating","Loss_of_Function",null,null,"Ambiguous",null,null,"Loss_of_Function","Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating&Ambiguous","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function","Activating","Loss_of_Function&Activating","Activating","Loss_of_Function","Loss_of_Function&Activating","Activating","Loss_of_Function","Activating","Activating",null,"Activating","Loss_of_Function",null,"Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function","Ambiguous","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating","Activating","Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Loss_of_Function&Ambiguous","Activating&Loss_of_Function",null,"Loss_of_Function","Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function&Activating",null,"Loss_of_Function&Activating","Loss_of_Function&Activating","Activating","Loss_of_Function","Activating","Activating","Loss_of_Function&Activating","Activating","Activating","Activating&Ambiguous","Loss_of_Function&Activating","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating","Loss_of_Function","Activating&Loss_of_Function","Activating","Activating&Loss_of_Function","Loss_of_Function","Loss_of_Function","Activating&Loss_of_Function","Activating","Activating&Loss_of_Function","Activating","Activating","Loss_of_Function","Loss_of_Function&Ambiguous&Activating","Activating","Activating","Activating","Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Ambiguous","Activating","Loss_of_Function&Ambiguous","Loss_of_Function","Activating","Activating&Loss_of_Function","Activating","Loss_of_Function&Activating","Loss_of_Function","Loss_of_Function","Loss_of_Function&Activating","Activating&Loss_of_Function","Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Activating","Activating","Loss_of_Function","Activating&Loss_of_Function&Ambiguous","Activating&Loss_of_Function","Activating&Loss_of_Function","Activating&Ambiguous","Activating","Loss_of_Function&Activating","Loss_of_Function","Activating","Activating&Loss_of_Function","Activating","Activating&Loss_of_Function","Loss_of_Function","Loss_of_Function","Activating&Loss_of_Function","Loss_of_Function&Activating","Activating","Activating","Loss_of_Function&Ambiguous&Activating","Activating&Loss_of_Function","Loss_of_Function","Loss_of_Function","Loss_of_Function",null,"Activating",null,"Activating","Activating","Activating","Loss_of_Function","Loss_of_Function","Ambiguous&Activating&Loss_of_Function",null,null,"Activating","Activating","Loss_of_Function",null,null,null,null,null,"Loss_of_Function",null,null,null,null,null,"Activating","Loss_of_Function&Activating",null,null,"Activating",null,null,null,null,"Activating",null,"Activating",null,null,null,"Loss_of_Function",null,null,"Loss_of_Function",null,null,null,"Ambiguous",null,null,"Loss_of_Function",null,null,null,null,null,"Activating",null,"Loss_of_Function",null,"Loss_of_Function",null,"Loss_of_Function","Loss_of_Function",null,"Activating&Loss_of_Function","Loss_of_Function&Activating",null,"Loss_of_Function","Loss_of_Function",null,null,null,null,null,null,"Activating&Loss_of_Function&Ambiguous","Activating","Loss_of_Function&Activating",null,null,null,null,null,null,null,null,"Activating",null,null,null,"Loss_of_Function","Loss_of_Function","Loss_of_Function","Loss_of_Function",null,null,null,null,"Activating","Activating",null,"Loss_of_Function","Loss_of_Function",null,null,null,"Activating",null,null,null,"Activating",null,null,null,null,null,null,"Loss_of_Function","Loss_of_Function",null,null,null,"Activating","Activating",null,null,"Activating",null,"Loss_of_Function","Loss_of_Function",null,null,null,null,null,null,"Activating","Activating","Activating",null,null,null,"Loss_of_Function",null,null,null,null,"Activating","Activating","Activating",null,"Loss_of_Function","Activating",null,null,null,null,null,null,null,null,null,"Loss_of_Function","Loss_of_Function","Activating",null,null,"Activating","Loss_of_Function","Loss_of_Function","Loss_of_Function",null,null,null,null,null,null,"Loss_of_Function",null,null,"Activating","Activating&Loss_of_Function","Activating",null,null,null,"Loss_of_Function",null,null,"Ambiguous&Activating",null,null,"Loss_of_Function",null,"Activating",null,"Activating&Loss_of_Function",null,null,null,"Loss_of_Function",null,null,"Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function","Activating",null,"Loss_of_Function","Loss_of_Function&Activating","Loss_of_Function",null,null,"Activating",null,"Ambiguous","Activating&Loss_of_Function",null,null,null,"Activating&Loss_of_Function&Ambiguous",null,null,null,null,"Activating","Loss_of_Function&Activating","Loss_of_Function&Ambiguous&Activating",null,null,"Ambiguous&Activating",null,"Activating","Loss_of_Function","Ambiguous",null,null,"Loss_of_Function","Activating&Loss_of_Function","Activating","Loss_of_Function",null,"Loss_of_Function",null,"Activating",null,"Activating","Loss_of_Function",null,"Activating",null,"Activating","Activating","Loss_of_Function",null,"Activating",null,null,null,"Loss_of_Function","Loss_of_Function",null,null,"Loss_of_Function","Loss_of_Function",null,"Activating",null,null,"Activating","Loss_of_Function",null,null,null,"Loss_of_Function",null,null,null,"Loss_of_Function&Activating",null,null,"Loss_of_Function",null,null,"Loss_of_Function","Loss_of_Function&Activating","Activating",null,"Loss_of_Function","Activating",null,"Loss_of_Function","Loss_of_Function","Loss_of_Function",null,"Activating",null,"Loss_of_Function","Loss_of_Function","Loss_of_Function","Loss_of_Function","Loss_of_Function","Activating",null,"Loss_of_Function","Loss_of_Function",null,null,null,null,null,"Activating&Loss_of_Function","Loss_of_Function",null,null,"Loss_of_Function&Activating","Loss_of_Function","Activating","Activating","Activating",null,null,"Loss_of_Function",null,null,"Loss_of_Function",null,null,null,null,null,null,null,null,null,"Activating",null,null,null,"Activating",null,null,"Loss_of_Function&Activating",null,"Loss_of_Function",null,null,null,"Activating",null,null,null,null,null,null,"Activating",null,null,"Loss_of_Function",null,"Loss_of_Function",null,"Loss_of_Function&Activating",null,null,"Activating","Loss_of_Function&Activating","Loss_of_Function",null,null,null,null,"Activating",null,"Loss_of_Function&Activating","Activating",null,"Activating&Loss_of_Function",null,"Activating","Loss_of_Function",null,"Activating","Loss_of_Function","Activating",null,null,"Loss_of_Function",null,"Loss_of_Function",null,null,null,null,"Activating",null,null,"Activating","Activating",null,null,"Activating","Activating","Activating",null,"Activating","Activating","Activating","Activating","Activating","Activating","Activating",null,"Loss_of_Function"],[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,false,true,true,true,true,true,true,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,true,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,true,true,false,true,true,true,false,true,false,true,true,false,false,true,true,true,true,true,true,false,true,true,true,true,true,true,true,true,true,false,true,false,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,true,false,true,true,true,true,true,true,true,false,true,true,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,true,true,false,false,true,true,false,true,true,true,true,true,true,false,true,true,true,true,true,true,true,true,false,false,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,false,false,false,true,true,true,false,true,true,true,true,true,true,true,true,false,true,true,true,true,true,false,false,true,false,false,true,true,true,false,true,true,true,false,true,false,false,false,false,false,false,true,false,false,false,false,false,true,true,false,true,false,false,true,false,true,true,false,false,true,true,false,true,true,true,false,false,true,false,true,true,true,false,true,true,false,true,true,true,false,true,true,false,false,true,true,true,true,false,false,false,false,false,true,false,false,true,false,false,false,false,false,true,true,true,false,true,false,false,false,false,false,false,false,true,true,true,true,false,false,false,true,true,false,false,false,false,true,true,false,false,false,false,false,false,false,false,true,true,false,true,true,true,false,true,true,true,true,true,true,false,false,true,true,true,false,false,true,true,false,true,false,false,true,false,true,true,false,true,false,false,false,true,false,false,false,false,false,true,false,false,false,false,true,false,false,true,true,true,true,true,true,true,true,true,false,false,false,false,true,false,false,false,false,false,false,true,true,false,true,false,false,false,false,false,false,false,true,true,false,false,true,false,true,true,false,true,false,true,false,false,true,true,false,true,false,false,false,false,false,true,false,false,false,true,true,false,true,false,false,true,true,false,false,true,true,true,true,false,false,false,true,true,false,true,false,false,false,true,true,false,false,false,false,true,false,true,false,true,false,false,false,false,false,false,false,false,true,false,true,true,true,false,false,true,false,false,false,true,false,false,true,false,false,false,false,false,false,true,false,true,false,true,true,false,true,true,false,false,false,true,false,false,false,false,false,false,true,false,true,false,false,false,false,false,false,true,false,false,false,true,false,false,true,false,false,true,false,false,false,false,false,false,true,true,false,true,false,false,true,true,true,true,true,false,true,true,true,false,true,true,false,false,true,false,false,true,false,false,false,true,false,true,true,true,true,true,true,false,true,false,false,true,false,false,false,true,true,false,false,false,false,true,true,true,false,true,false,false,true,false,true,false,false,false,false,false,false,false,true,false,true,false,true,true,true,true,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false],["Pan-cancer adult, Pan-cancer paediatric","Astrocytoma, Diffuse intrinsic pontine glioma, Paediatric high grade glioma, Pan-cancer adult, Pan-cancer paediatric","Ampullary adenocarcinoma, Breast cancer, Cholangiocarcinoma, Colorectal adenocarcinoma, Esophageal adenocarcinoma, Pan-cancer adult, Pancreatic cancer (All histologies)","Ampullary adenocarcinoma, Cholangiocarcinoma, Colorectal adenocarcinoma, Esophageal adenocarcinoma, Gastric adenocarcinoma, Hepatocellular carcinoma, Pan-cancer adult, Pan-Gynecological And Breast, Pancreatic cancer (All histologies)","Angiosarcoma, Breast cancer, Chronic myeloid leukemia, Endometrial cancer, Meningioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gynecological And Breast, Prostate cancer, Triple negative breast cancer","Melanoma, Neuroblastoma, Pan-cancer adult, Pan-cancer paediatric, Thymic carcinoma, Triple negative breast cancer","Colorectal adenocarcinoma, Gastric adenocarcinoma, Pan-cancer adult, Pan-cancer paediatric","Adrenocortical adenoma, Ampullary adenocarcinoma, Breast cancer, Cholangiocarcinoma, Colorectal adenocarcinoma, Duodenal adenocarcinoma, Esophageal adenocarcinoma, Gastric adenocarcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Pan-cancer adult, Pan-Gastric, Prostate cancer, Triple negative breast cancer","Gastric cancer, Pan-cancer adult, Prostate cancer","Cholangiocarcinoma, Pan-cancer adult","Acute promyelocytic leukemia, Adrenocortical adenoma, Ampullary adenocarcinoma, Biliary tract cancer, Bladder cancer, Breast cancer, Burkitt Lymphoma, Cervical cancer (All histologies), Cholangiocarcinoma, Clear cell endometrial cancer, Clear cell renal cancer, Colorectal adenocarcinoma, Cutaneous T-Cell lymphoma, Desmoplastic melanoma, Diffuse gastric adenocarcinoma, Endometrial cancer, Endometrioid ovarian cancer, Esophageal (Squamous and adenocarcinoma), Esophageal adenocarcinoma, Gastric adenocarcinoma, Hepatocellular carcinoma, Low grade glioma, Lung adenocarcinoma, Lung cancer (All histologies), Mucinous gastric cancer, Nasopharyngeal carcinoma, Natural killer/T cell lymphoma, Neuroblastoma, Ovarian clear-Cell carcinoma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pan-Glioma, Pan-Gynecological And Breast, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma, Prostate cancer, Renal cancer (All histologies), Skin basal cell carcinoma, Triple negative breast cancer, Uterine carcinosarcoma","Ampullary adenocarcinoma, Biliary tract cancer, Breast cancer, Cholangiocarcinoma, Desmoplastic melanoma, Duodenal adenocarcinoma, Esophageal adenocarcinoma, Gastric adenocarcinoma, Hepatocellular carcinoma, Lung cancer (All histologies), Melanoma, Multiple myeloma, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Pancreatic ductal adenocarcinoma, Papillary renal cell carcinoma","Acute myeloid leukemia, Angiosarcoma, Bladder cancer, Breast cancer, Cholangiocarcinoma, Chronic lymphocytic leukemia, Chronic myeloid leukemia, Chronic myelomonocytic leukemia, Colorectal adenocarcinoma, Juvenile myelomonocytic leukemia, Myelodysplasia, Myeloproliferative neoplasm, Pan-cancer adult, Pan-cancer paediatric, Penile squamous cancer","Bladder cancer, Chronic myelomonocytic leukemia, Pan-cancer adult, Pan-cancer paediatric","Ampullary adenocarcinoma, Bladder cancer, Breast cancer, Cholangiocarcinoma, Chronic lymphocytic leukemia, Clear cell renal cancer, Colorectal adenocarcinoma, Gastric adenocarcinoma, Hepatocellular carcinoma, Hurthle cell carcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Nasopharyngeal carcinoma, Non-Small cell lung cancer, Osteosarcoma, Pan-cancer adult, Pan-Gastric, Pan-Gynecological And Breast, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma, Peripheral T-Cell lymphoma, Prostate cancer, Triple negative breast cancer","Breast cancer, Endometrial cancer, Pan-cancer adult","Adrenocortical carcinoma, Astrocytoma, Breast cancer, Glioblastoma, Glioma, Low grade glioma, Lung adenocarcinoma, Mucosal melanoma, Neuroblastoma, Osteosarcoma, Paediatric high grade glioma, Paediatric low grade glioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Pancreatic neuroendocrine tumours, Soft tissue sarcoma","Ampullary adenocarcinoma, Breast cancer, Esophageal adenocarcinoma, Hepatocellular carcinoma, Cholangiocarcinoma, Pan-cancer adult, Pancreatic ductal adenocarcinoma","Diffuse large B-Cell lymphoma, DLBCL, Follicular lymphoma, Esophageal adenocarcinoma, Melanoma, Non-Hodgkin lymphoma, Pan-cancer adult, Pan-Gastric, Pan-Gynecological And Breast, Peripheral T-Cell lymphoma, T-Cell leukemia/Lymphoma","Biliary tract cancer, Breast cancer, Cholangiocarcinoma, Clear cell renal cancer, Hepatocellular carcinoma, Malignant Pleural Mesothelioma, Melanoma, Mucosal melanoma, Nasopharyngeal carcinoma, Osteosarcoma, Pan-cancer adult, Papillary renal cell carcinoma, Renal cancer (All histologies), Thymic carcinoma, Uveal melanoma","Diffuse large B-Cell lymphoma, DLBCL, Follicular lymphoma, Non-Hodgkin lymphoma, Pan-cancer adult","Breast cancer, Breast fibroepithelial tumours, Chronic lymphocytic leukemia, Chronic myelomonocytic leukemia, Endometrial cancer, Gastric adenocarcinoma, Medulloblastoma, Mucinous gastric cancer, Myelodysplasia, Paediatric high grade glioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pan-Glioma, Retinoblastoma","Acute lymphoblastic leukemia, Ameloblastoma, Ampullary adenocarcinoma, Anaplastic thyroid carcinoma, Astrocytoma, Breast cancer, Cholangiocarcinoma, Chronic lymphocytic leukemia, Colorectal adenocarcinoma, Craniopharyngioma, Cutaneous T-Cell lymphoma, Desmoplastic melanoma, Esophageal adenocarcinoma, Glioblastoma, Hurthle cell carcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Melanoma, Mucosal melanoma, Multiple myeloma, Non-Small cell lung cancer, Ovarian cancer, Paediatric high grade glioma, Paediatric low grade glioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pan-Glioma, Papillary thyroid cancer, Prostate cancer","Breast cancer, Lung adenocarcinoma, Mucosal melanoma, Ovarian cancer, Ovarian serous carcinoma, Pan-cancer adult, Pan-Gynecological And Breast, Penile squamous cancer, Prostate cancer","Breast cancer, Cholangiocarcinoma, Gastric adenocarcinoma, Mucinous gastric cancer, Osteosarcoma, Ovarian cancer, Ovarian serous carcinoma, Pan-cancer adult, Pancreatic ductal adenocarcinoma, Prostate cancer, Triple negative breast cancer","Chronic lymphocytic leukemia, Clear cell renal cancer, Cutaneous T-Cell lymphoma, Diffuse large B-Cell lymphoma, Follicular lymphoma, Non-Hodgkin lymphoma, Pan-cancer adult, Pan-cancer paediatric, T-Cell leukemia/Lymphoma","Bladder cancer, Breast cancer, Cervical cancer (All histologies), Colorectal adenocarcinoma, Oral squamous cell carcinoma, Pan-cancer adult, Pan-Gynecological And Breast, Skin basal cell carcinoma, Squamous head and neck cancer","Breast cancer, Cervical cancer (All histologies), Pan-cancer adult, Pan-cancer paediatric","Biliary tract cancer, Breast cancer, Cholangiocarcinoma, Endometrial cancer, Hepatocellular carcinoma, Multiple myeloma, Nasopharyngeal carcinoma, Paediatric high grade glioma, Pan-cancer adult, Pan-Gynecological And Breast, Parathyroid carcinoma, Prostate cancer","Burkitt Lymphoma, Cutaneous DLBCL, Diffuse large B-Cell lymphoma, Follicular lymphoma, Non-Hodgkin lymphoma, Pan-cancer adult","Breast cancer, Colorectal adenocarcinoma, Diffuse gastric adenocarcinoma, Esophageal adenocarcinoma, Gastric adenocarcinoma, Gastric cancer, Mucinous gastric cancer, Pan-cancer adult, Pan-Gastric, Pan-Gynecological And Breast, Soft tissue sarcoma","Ovarian cancer, Pan-cancer adult, Prostate cancer","Leiomysarcoma, Liposarcoma, Paediatric high grade glioma, Pan-cancer adult","Bladder cancer, Cholangiocarcinoma, Hepatocellular carcinoma, Hurthle cell carcinoma, Pan-cancer adult, Renal cancer (All histologies)","Anaplastic thyroid carcinoma, Breast cancer, Cholangiocarcinoma, Esophageal adenocarcinoma, Multiple myeloma, Neuroendocrine tumour, Pan-cancer adult, Pan-cancer paediatric, Pan-Gynecological And Breast, Prostate cancer","Acute lymphoblastic leukemia, Acute lymphocytic leukemia, Ampullary adenocarcinoma, Angiosarcoma, Biliary tract cancer, Bladder cancer, Breast cancer, Cholangiocarcinoma, Cutaneous T-Cell lymphoma, Desmoplastic melanoma, Esophageal (Squamous and adenocarcinoma), Esophageal adenocarcinoma, Esophageal squamous carcinoma, Ewing sarcoma, Hepatocellular carcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Lung squamous cell carcinoma, Malignant peripheral nerve sheath tumour, Melanoma, Mucosal melanoma, Nasopharyngeal carcinoma, Non-Small cell lung cancer, Paediatric high grade glioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma, Prostate cancer, Skin basal cell carcinoma, Squamous head and neck cancer, T-Cell leukemia/Lymphoma, Thymic carcinoma, Triple negative breast cancer, Uveal melanoma","Anaplastic thyroid carcinoma, Pan-cancer adult","Acute myeloid leukemia, Acute promyelocytic leukemia, Pan-cancer adult, Pan-cancer paediatric","Colorectal adenocarcinoma, Endometrial cancer, Esophageal adenocarcinoma, Gastric adenocarcinoma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gynecological And Breast, Serous endometrial cancer, Uterine carcinosarcoma","Angiosarcoma, Astrocytoma, Breast cancer, Chronic lymphocytic leukemia, Pan-cancer adult, Papillary thyroid cancer","Breast cancer, Gastric adenocarcinoma, Low grade glioma, Oligodendroglioma, Paediatric low grade glioma, Pan-cancer adult, Pan-Glioma","Acute lymphoblastic leukemia, Angiosarcoma, Bladder cancer, Breast cancer, Burkitt Lymphoma, Diffuse large B-Cell lymphoma, Esophageal squamous carcinoma, Follicular lymphoma, Hepatocellular carcinoma, Non-Hodgkin lymphoma, Pan-cancer adult, Pan-cancer paediatric, Retinoblastoma, Small cell lung cancer, Triple negative breast cancer","Acute myeloid leukemia, Breast cancer, Burkitt Lymphoma, Colorectal adenocarcinoma, Endometrial cancer, Gastric adenocarcinoma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gynecological And Breast","Adrenocortical adenoma, Adrenocortical carcinoma, Ampullary adenocarcinoma, Anaplastic thyroid carcinoma, Breast cancer, Cholangiocarcinoma, Colorectal adenocarcinoma, Craniopharyngioma, Duodenal adenocarcinoma, Endometrial cancer, Endometrioid ovarian cancer, Esophageal adenocarcinoma, Gastric adenocarcinoma, Hepatocellular carcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Medulloblastoma, Melanoma, Mucosal melanoma, Non-Small cell lung cancer, Ovarian cancer, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pan-Gynecological And Breast, Pancreatic neoplastic cysts, Prostate cancer, Soft tissue sarcoma, Triple negative breast cancer","Esophageal squamous carcinoma, Lung cancer (All histologies), Pan-cancer adult, Papillary renal cell carcinoma","Cylindroma, Spiradenocarcinoma, Spiradenoma, Multiple myeloma, Nasopharyngeal carcinoma, Pan-cancer adult, Thymic carcinoma","Melanoma, Pan-cancer adult, Uveal melanoma","Burkitt Lymphoma, Chronic lymphocytic leukemia, Malignant Pleural Mesothelioma, Medulloblastoma, Melanoma, Natural killer/T cell lymphoma, Pan-cancer adult, Pan-cancer paediatric, Squamous head and neck cancer","Endometrial cancer, Pan-cancer adult, Triple negative breast cancer","Acute monocytic leukemia, Acute myeloid leukemia, Acute promyelocytic leukemia, Angioimmunoblastic T-Cell lymphoma, Breast cancer, Chronic myelomonocytic leukemia, Cutaneous T-Cell lymphoma, Cylindroma, Spiradenocarcinoma, Spiradenoma, Juvenile myelomonocytic leukemia, Myelodysplasia, Myeloproliferative neoplasm, Pan-cancer adult, Pan-Glioma, Papillary thyroid cancer, Peripheral T-Cell lymphoma, T-Cell leukemia/Lymphoma, Triple negative breast cancer","Biliary tract cancer, Breast cancer, Breast fibroepithelial tumours, Gallbladder carcinoma, Gastric adenocarcinoma, Glioblastoma, Low grade glioma, Lung adenocarcinoma, Lung cancer (All histologies), Non-Small cell lung cancer, Paediatric high grade glioma, Pan-cancer adult, Pan-Glioma","Anaplastic thyroid carcinoma, Hurthle cell carcinoma, Melanoma, Pan-cancer adult, Pan-Glioma, Papillary thyroid cancer, Uveal melanoma","Ampullary adenocarcinoma, Biliary tract cancer, Bladder cancer, Cervical cancer (All histologies), Cholangiocarcinoma, Colorectal adenocarcinoma, Esophageal adenocarcinoma, Gastric adenocarcinoma, Pan-cancer adult, Pan-Gastric","Acute lymphoblastic leukemia, Bladder cancer, Breast cancer, Cervical cancer (All histologies), Diffuse large B-Cell lymphoma, Esophageal squamous carcinoma, Follicular lymphoma, Multiple myeloma, Nasopharyngeal carcinoma, Natural killer/T cell lymphoma, Paediatric low grade glioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gynecological And Breast, Pancreatic ductal adenocarcinoma, Small cell lung cancer, T-Cell leukemia/Lymphoma, Triple negative breast cancer","Pan-cancer adult, Pheochromocytoma, Paraganglioma","Undefined tumor type(s)","Ampullary adenocarcinoma, Anaplastic thyroid carcinoma, Bladder cancer, Breast cancer, Cholangiocarcinoma, Desmoplastic melanoma, Endometrial cancer, Esophageal (Squamous and adenocarcinoma), Esophageal adenocarcinoma, Gallbladder carcinoma, Gastric adenocarcinoma, Glioblastoma, Lung cancer (All histologies), Mucosal melanoma, Nasopharyngeal carcinoma, Pan-cancer adult, Pan-Gastric, Skin basal cell carcinoma","Ampullary adenocarcinoma, Bladder cancer, Breast cancer, Cervical cancer (All histologies), Colorectal adenocarcinoma, Diffuse gastric adenocarcinoma, Endometrial cancer, Gallbladder carcinoma, Gastric adenocarcinoma, Gastric cancer, Nasopharyngeal carcinoma, Pan-cancer adult, Triple negative breast cancer","Breast fibroepithelial tumours, Gallbladder carcinoma, Gastric adenocarcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Melanoma, Pan-cancer adult, Soft tissue sarcoma, Triple negative breast cancer","Bladder cancer, Non-Clear cell renal cancer, Pan-cancer adult","Breast cancer, Endometrial cancer, Pan-cancer adult","Acute lymphoblastic leukemia, Acute myeloid leukemia, Chronic myeloid leukemia, Chronic myelomonocytic leukemia, Desmoplastic melanoma, Diffuse large B-Cell lymphoma, Ewing sarcoma, Follicular lymphoma, Gastric adenocarcinoma, Juvenile myelomonocytic leukemia, Myelodysplasia, Myeloproliferative neoplasm, Non-Hodgkin lymphoma, Pan-cancer adult, Pan-cancer paediatric, Uveal melanoma","Bladder cancer, Colorectal adenocarcinoma, Diffuse large B-Cell lymphoma, Esophageal squamous carcinoma, Gastric adenocarcinoma, Lung cancer (All histologies), Mucinous gastric cancer, Nasopharyngeal carcinoma, Oral squamous cell carcinoma, Pan-cancer adult, Papillary renal cell carcinoma, Penile squamous cancer, Squamous head and neck cancer","Acute lymphoblastic leukemia, Ampullary adenocarcinoma, Bladder cancer, Breast cancer, Cervical cancer (All histologies), Cholangiocarcinoma, Chronic lymphocytic leukemia, Colorectal adenocarcinoma, Desmoplastic melanoma, Diffuse gastric adenocarcinoma, Endometrial cancer, Esophageal (Squamous and adenocarcinoma), Esophageal adenocarcinoma, Esophageal squamous carcinoma, Gastric cancer, Lung adenocarcinoma, Lung cancer (All histologies), Lung squamous cell carcinoma, Nasopharyngeal carcinoma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pan-Gynecological And Breast, Pancreatic cancer (All histologies), Serous endometrial cancer, Skin basal cell carcinoma, Soft tissue sarcoma, Squamous head and neck cancer, Triple negative breast cancer, Uterine carcinosarcoma","Astrocytoma, Biliary tract cancer, Glioma, Neuroblastoma, Paediatric low grade glioma, Pan-cancer adult, Pan-cancer paediatric, Triple negative breast cancer","Ameloblastoma, Biliary tract cancer, Breast cancer, Cholangiocarcinoma, Chronic lymphocytic leukemia, Endometrial cancer, Nasopharyngeal carcinoma, Paediatric low grade glioma, Pan-cancer adult","Biliary tract cancer, Bladder cancer, Lung adenocarcinoma, Multiple myeloma, Paediatric low grade glioma, Pan-cancer adult, Skin basal cell carcinoma","Acute lymphoblastic leukemia, Acute lymphocytic leukemia, Acute myeloid leukemia, Acute promyelocytic leukemia, Lung cancer (All histologies), Pan-cancer adult, Pan-cancer paediatric","Bladder cancer, Breast cancer, Pan-cancer adult, Prostate cancer","Chronic lymphocytic leukemia, Low grade glioma, Multiple myeloma, Oligodendroglioma, Pan-cancer adult, Pan-Glioma","Acute lymphoblastic leukemia, Breast cancer, Gastric adenocarcinoma, Hepatocellular carcinoma, Male breast cancer, Pan-cancer adult, Pan-cancer paediatric, Pan-Gynecological And Breast, T-Cell leukemia/Lymphoma, Triple negative breast cancer","Melanoma, Pan-cancer adult, Uveal melanoma","Melanoma, Mucosal melanoma, Pan-cancer adult, Uveal melanoma","Adrenocortical adenoma, Ampullary adenocarcinoma, Biliary tract cancer, Breast cancer, Cholangiocarcinoma, Colorectal adenocarcinoma, Gastric adenocarcinoma, Gastric cancer, Lung adenocarcinoma, Mucinous gastric cancer, Mucosal melanoma, Pan-cancer adult, Pancreatic ductal adenocarcinoma, Pancreatic neoplastic cysts, Prostate cancer","Astrocytoma, Bone giant cell tumour, Chondroblastoma, Diffuse intrinsic pontine glioma, Glioblastoma, Glioma, Paediatric low grade glioma, Pan-cancer adult, Pan-cancer paediatric","Anaplastic thyroid carcinoma, Angiosarcoma, Bladder cancer, Breast cancer, Lung cancer (All histologies), Melanoma, Neuroblastoma, Oral squamous cell carcinoma, Pan-cancer adult, Papillary thyroid cancer, Pheochromocytoma, Paraganglioma, Prostate cancer, Skin basal cell carcinoma, Squamous head and neck cancer, Thymic carcinoma","Acute myeloid leukemia, Acute promyelocytic leukemia, Biliary tract cancer, Cholangiocarcinoma, Chondrosarcoma, Desmoplastic melanoma, Glioblastoma, Glioma, Low grade glioma, Melanoma, Myeloproliferative neoplasm, Oligodendroglioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Prostate cancer","Acute myeloid leukemia, Acute promyelocytic leukemia, Angioimmunoblastic T-Cell lymphoma, Cholangiocarcinoma, Chondrosarcoma, Chronic myelomonocytic leukemia, Low grade glioma, Myeloproliferative neoplasm, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Peripheral T-Cell lymphoma, T-Cell leukemia/Lymphoma","Hepatocellular carcinoma, Pan-cancer adult","Acute lymphoblastic leukemia, Colorectal adenocarcinoma, Nasopharyngeal carcinoma, Pan-cancer adult, Pan-cancer paediatric","Acute lymphoblastic leukemia, Endometrial cancer, Esophageal adenocarcinoma, Pan-cancer adult, Pan-cancer paediatric","Chronic myelomonocytic leukemia, Myeloproliferative neoplasm, Non-Small cell lung cancer, Pan-cancer adult, Pan-cancer paediatric","Acute lymphoblastic leukemia, Juvenile myelomonocytic leukemia, Pan-cancer adult, Pan-cancer paediatric, T-Cell leukemia/Lymphoma","Clear cell renal cancer, Pan-cancer adult, Renal cancer (All histologies)","Acute lymphoblastic leukemia, Ampullary adenocarcinoma, Angiosarcoma, Bladder cancer, Breast cancer, Clear cell renal cancer, Colorectal adenocarcinoma, Esophageal adenocarcinoma, Esophageal squamous carcinoma, Gastric adenocarcinoma, Hurthle cell carcinoma, Lung cancer (All histologies), Medulloblastoma, Multiple myeloma, Pan-cancer adult, Pan-cancer paediatric, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma, Papillary renal cell carcinoma, Prostate cancer, Soft tissue sarcoma, T-Cell leukemia/Lymphoma, Triple negative breast cancer","Hepatocellular carcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Lung squamous cell carcinoma, Non-Small cell lung cancer, Pan-cancer adult, Papillary renal cell carcinoma","Acute myeloid leukemia, Acute promyelocytic leukemia, Intracranial germ cell, Melanoma, Mucosal melanoma, Pan-cancer adult, Pan-cancer paediatric, Soft tissue sarcoma, Testicular germ cell cancer, Triple negative breast cancer","Bladder cancer, Breast cancer, Hepatocellular carcinoma, Nasopharyngeal carcinoma, Pan-cancer adult, Pan-cancer paediatric, Pancreatic ductal adenocarcinoma, Prostate cancer, Triple negative breast cancer","Acute myeloid leukemia, Acute promyelocytic leukemia, Bladder cancer, Breast cancer, Cholangiocarcinoma, Clear cell renal cancer, Esophageal squamous carcinoma, Gastric adenocarcinoma, Gastric cancer, Hepatocellular carcinoma, Lung cancer (All histologies), Medulloblastoma, Mucinous gastric cancer, Mucosal melanoma, Nasopharyngeal carcinoma, Pan-cancer adult, Pan-cancer paediatric, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma, Prostate cancer, Renal cancer (All histologies), Triple negative breast cancer","Anaplastic thyroid carcinoma, Bladder cancer, Breast cancer, Breast fibroepithelial tumours, Burkitt Lymphoma, Clear cell renal cancer, Diffuse large B-Cell lymphoma, Endometrioid ovarian cancer, Esophageal (Squamous and adenocarcinoma), Esophageal squamous carcinoma, Follicular lymphoma, Gastric adenocarcinoma, Gastric cancer, Granulosa Tumours Of The Ovary, Hurthle cell carcinoma, Lung cancer (All histologies), Lung squamous cell carcinoma, Medulloblastoma, Mucinous gastric cancer, Nasopharyngeal carcinoma, Natural killer/T cell lymphoma, Non-Hodgkin lymphoma, Pan-cancer adult, Pan-cancer paediatric, Pancreatic ductal adenocarcinoma, Prostate cancer, Squamous head and neck cancer, Triple negative breast cancer","Acute lymphoblastic leukemia, Acute lymphocytic leukemia, Acute myeloid leukemia, Ameloblastoma, Ampullary adenocarcinoma, Anaplastic thyroid carcinoma, Angiosarcoma, Astrocytoma, Biliary tract cancer, Bladder cancer, Breast cancer, Cervical cancer (All histologies), Cholangiocarcinoma, Chronic lymphocytic leukemia, Chronic myelomonocytic leukemia, Colorectal adenocarcinoma, Diffuse gastric adenocarcinoma, Duodenal adenocarcinoma, Endometrial cancer, Esophageal adenocarcinoma, Gallbladder carcinoma, Gastric adenocarcinoma, Hepatocellular carcinoma, Hurthle cell carcinoma, Intracranial germ cell, Juvenile myelomonocytic leukemia, Lung adenocarcinoma, Lung cancer (All histologies), Lung squamous cell carcinoma, Melanoma, Mucosal melanoma, Multiple myeloma, Myelodysplasia, Nasopharyngeal carcinoma, Neuroblastoma, Non-Small cell lung cancer, Ovarian cancer, Ovarian clear-Cell carcinoma, Paediatric low grade glioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pan-Glioma, Pan-Gynecological And Breast, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma, Pancreatic neoplastic cysts, Papillary thyroid cancer, Skin basal cell carcinoma, Squamous head and neck cancer, Testicular germ cell cancer, Thymic carcinoma, Triple negative breast cancer, Uterine carcinosarcoma","Pan-cancer adult, Pan-Gynecological And Breast, Skin basal cell carcinoma","Malignant Pleural Mesothelioma, Pan-cancer adult, Skin basal cell carcinoma","Glioblastoma, Hepatocellular carcinoma, Pan-cancer adult, Pan-Glioma","Chronic lymphocytic leukemia, Colorectal adenocarcinoma, Desmoplastic melanoma, Diffuse gastric adenocarcinoma, Lung cancer (All histologies), Melanoma, Mucosal melanoma, Pan-cancer adult","Ampullary adenocarcinoma, Breast cancer, Cholangiocarcinoma, Pan-cancer adult, Pan-Gynecological And Breast, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma","Anaplastic thyroid carcinoma, Breast cancer, Breast fibroepithelial tumours, Esophageal adenocarcinoma, Gastric adenocarcinoma, Male breast cancer, Pan-cancer adult, Pan-Gastric, Pan-Gynecological And Breast","Cervical cancer (All histologies), Chronic lymphocytic leukemia, Pan-cancer adult","Endometrial cancer, Multiple myeloma, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma","Endometrial cancer, Pan-cancer adult","Adrenocortical carcinoma, Breast fibroadenoma, Breast fibroepithelial tumours, Chronic lymphocytic leukemia, Leiomyoma, Nasopharyngeal carcinoma, Pan-cancer adult, Pan-cancer paediatric, Prostate cancer, Uveal melanoma","Adrenocortical carcinoma, Breast cancer, Hepatocellular carcinoma, Hurthle cell carcinoma, Mucosal melanoma, Pan-cancer adult, Pancreatic neuroendocrine tumours","Angiosarcoma, Lung adenocarcinoma, Lung cancer (All histologies), Non-Clear cell renal cancer, Paediatric high grade glioma, Pan-cancer adult, Papillary renal cell carcinoma, Renal cancer (All histologies), Triple negative breast cancer","Anaplastic thyroid carcinoma, Burkitt Lymphoma, Clear cell renal cancer, Hurthle cell carcinoma, Intracranial germ cell, Mucinous gastric cancer, Pan-cancer adult, Parathyroid carcinoma, Renal cancer (All histologies), Squamous head and neck cancer","Biliary tract cancer, Breast cancer, Burkitt Lymphoma, Cutaneous DLBCL, Diffuse large B-Cell lymphoma, Paediatric high grade glioma, Pan-cancer adult, Pan-cancer paediatric, Prostate cancer, Triple negative breast cancer","Endometrial cancer, Neuroblastoma, Paediatric high grade glioma, Pan-cancer adult, Pan-cancer paediatric, Skin basal cell carcinoma","Chronic lymphocytic leukemia, Cutaneous DLBCL, Diffuse large B-Cell lymphoma, Esophageal adenocarcinoma, Follicular lymphoma, Non-Hodgkin lymphoma, Pan-cancer adult","Mucinous gastric cancer, Pan-cancer adult","Anaplastic thyroid carcinoma, Bladder cancer, Breast cancer, Cholangiocarcinoma, Hepatocellular carcinoma, Pan-cancer adult, Pan-cancer paediatric, Prostate cancer","Acute lymphocytic leukemia, Anaplastic thyroid carcinoma, Angiosarcoma, Astrocytoma, Bladder cancer, Breast cancer, Breast fibroepithelial tumours, Cholangiocarcinoma, Chronic myelomonocytic leukemia, Desmoplastic melanoma, Gastric adenocarcinoma, Gastric cancer, Glioblastoma, Hurthle cell carcinoma, Intracranial germ cell, Juvenile myelomonocytic leukemia, Low grade glioma, Lung adenocarcinoma, Lung cancer (All histologies), Malignant peripheral nerve sheath tumour, Melanoma, Mucosal melanoma, Neuroblastoma, Ovarian cancer, Ovarian serous carcinoma, Paediatric high grade glioma, Paediatric low grade glioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Pan-Gynecological And Breast, Pheochromocytoma, Paraganglioma, Soft tissue sarcoma, Triple negative breast cancer","Anaplastic thyroid carcinoma, Breast cancer, Cholangiocarcinoma, Clear cell renal cancer, Malignant Pleural Mesothelioma, Meningioma, Mucosal melanoma, Non-Clear cell renal cancer, Pan-cancer adult, Pancreatic cancer (All histologies), Papillary renal cell carcinoma, Renal cancer (All histologies)","Bladder cancer, Cervical cancer (All histologies), Clear cell renal cancer, Endometrial cancer, Esophageal (Squamous and adenocarcinoma), Esophageal squamous carcinoma, Hepatocellular carcinoma, Cholangiocarcinoma, Hurthle cell carcinoma, Lung cancer (All histologies), Lung squamous cell carcinoma, Pan-cancer adult, Pancreatic ductal adenocarcinoma, Papillary renal cell carcinoma, Penile squamous cancer, Renal cancer (All histologies), Squamous head and neck cancer","Acute lymphoblastic leukemia, Bladder cancer, Breast cancer, Burkitt Lymphoma, Chronic lymphocytic leukemia, Diffuse large B-Cell lymphoma, Esophageal (Squamous and adenocarcinoma), Esophageal adenocarcinoma, Esophageal squamous carcinoma, Hepatocellular carcinoma, Low grade glioma, Lung cancer (All histologies), Lung squamous cell carcinoma, Nasopharyngeal carcinoma, Ocular adnexal sebaceous carcinoma, Oligodendroglioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Skin basal cell carcinoma, Small cell lung cancer, Soft tissue sarcoma, Squamous head and neck cancer, T-Cell leukemia/Lymphoma, Triple negative breast cancer","Anaplastic thyroid carcinoma, Bladder cancer, Breast cancer, Esophageal (Squamous and adenocarcinoma), Esophageal squamous carcinoma, Gastric adenocarcinoma, Hepatocellular carcinoma, Mucinous gastric cancer, Nasopharyngeal carcinoma, Pan-cancer adult, Pan-cancer paediatric, Skin basal cell carcinoma, Splenic marginal zone lymphoma, Triple negative breast cancer","Acute myeloid leukemia, Acute promyelocytic leukemia, Pan-cancer adult, Pan-cancer paediatric","Acute lymphoblastic leukemia, Acute lymphocytic leukemia, Acute myeloid leukemia, Acute promyelocytic leukemia, Anaplastic thyroid carcinoma, Angiosarcoma, Biliary tract cancer, Bladder cancer, Breast cancer, Cholangiocarcinoma, Chronic lymphocytic leukemia, Chronic myelomonocytic leukemia, Colorectal adenocarcinoma, Desmoplastic melanoma, Endometrial cancer, Hepatocellular carcinoma, Hurthle cell carcinoma, Intracranial germ cell, Juvenile myelomonocytic leukemia, Lung adenocarcinoma, Lung cancer (All histologies), Melanoma, Mucosal melanoma, Multiple myeloma, Myelodysplasia, Nasopharyngeal carcinoma, Natural killer/T cell lymphoma, Neuroblastoma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pan-Glioma, Papillary thyroid cancer, Skin basal cell carcinoma, Testicular germ cell cancer, Thymic carcinoma","Acute lymphoblastic leukemia, Pan-cancer adult, Pan-cancer paediatric","Acute lymphoblastic leukemia, Acute lymphocytic leukemia, DLBCL, Follicular lymphoma, Lung cancer (All histologies), Pan-cancer adult, Pan-cancer paediatric","Ampullary adenocarcinoma, Breast cancer, Cholangiocarcinoma, Clear cell renal cancer, Esophageal adenocarcinoma, Gastric adenocarcinoma, Pan-cancer adult, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma, Papillary renal cell carcinoma, Renal cancer (All histologies), Thymic carcinoma, Triple negative breast cancer, Uveal melanoma","Undefined tumor type(s)","Astrocytoma, Breast cancer, Glioblastoma, Glioma, Lung adenocarcinoma, Paediatric high grade glioma, Pan-cancer adult, Pan-cancer paediatric","Acute lymphoblastic leukemia, Acute myeloid leukemia, Breast cancer, Chronic myelomonocytic leukemia, Pan-cancer adult, Pan-cancer paediatric","Acute lymphocytic leukemia, Ampullary adenocarcinoma, Anaplastic thyroid carcinoma, Astrocytoma, Biliary tract cancer, Bladder cancer, Breast cancer, Breast fibroepithelial tumours, Cervical cancer (All histologies), Cholangiocarcinoma, Clear cell endometrial cancer, Clear cell renal cancer, Colorectal adenocarcinoma, Desmoplastic melanoma, Diffuse gastric adenocarcinoma, Diffuse intrinsic pontine glioma, Endometrial cancer, Endometrioid ovarian cancer, Esophageal adenocarcinoma, Esophageal squamous carcinoma, Gastric adenocarcinoma, Gastric cancer, Glioblastoma, Hepatocellular carcinoma, Leiomysarcoma, Liposarcoma, Low grade glioma, Lung adenocarcinoma, Lung cancer (All histologies), Lung squamous cell carcinoma, Male breast cancer, Medulloblastoma, Nasopharyngeal carcinoma, Non-Small cell lung cancer, Oligodendroglioma, Oral squamous cell carcinoma, Ovarian clear-Cell carcinoma, Paediatric high grade glioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pan-Glioma, Pan-Gynecological And Breast, Pancreatic ductal adenocarcinoma, Pancreatic neuroendocrine tumours, Parathyroid carcinoma, Prostate cancer, Renal cancer (All histologies), Serous endometrial cancer, Skin basal cell carcinoma, Soft tissue sarcoma, Squamous head and neck cancer, Triple negative breast cancer, Uterine carcinosarcoma, Uveal melanoma","Pan-cancer adult, Prostate cancer, Triple negative breast cancer","Acute lymphocytic leukemia, Ampullary adenocarcinoma, Astrocytoma, Breast cancer, Burkitt Lymphoma, Cholangiocarcinoma, Clear cell endometrial cancer, Diffuse intrinsic pontine glioma, Endometrial cancer, Endometrioid ovarian cancer, Esophageal adenocarcinoma, Glioblastoma, Low grade glioma, Oligodendroglioma, Paediatric high grade glioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Pan-Gynecological And Breast, Prostate cancer, Squamous head and neck cancer, Triple negative breast cancer, Uterine carcinosarcoma","Chronic lymphocytic leukemia, Cutaneous DLBCL, Diffuse large B-Cell lymphoma, Follicular lymphoma, Multiple myeloma, Pan-cancer adult","Angiosarcoma, Cutaneous T-Cell lymphoma, Low grade glioma, Nasopharyngeal carcinoma, Pan-cancer adult, Pan-Glioma, T-Cell leukemia/Lymphoma","Bladder cancer, Breast cancer, Pan-cancer adult","Endometrial cancer, Gastric cancer, Mucosal melanoma, Pan-cancer adult","Esophageal adenocarcinoma, Pan-cancer adult","Diffuse intrinsic pontine glioma, Esophageal adenocarcinoma, Glioma, Paediatric high grade glioma, Pan-cancer adult, Papillary thyroid cancer","Clear cell endometrial cancer, Endometrial cancer, Lung adenocarcinoma, Ovarian clear-Cell carcinoma, Pan-cancer adult, Pan-Gynecological And Breast, Serous endometrial cancer, Thymic carcinoma, Uterine carcinosarcoma","Desmoplastic melanoma, Melanoma, Pan-cancer adult, Skin basal cell carcinoma","Adrenocortical adenoma, Adrenocortical carcinoma, Breast cancer, Pan-cancer adult","Bladder cancer, Pan-cancer adult","Anaplastic thyroid carcinoma, Chondrosarcoma, Chronic myelomonocytic leukemia, Esophageal squamous carcinoma, Lung adenocarcinoma, Medulloblastoma, Pan-cancer adult, Pan-cancer paediatric, Skin basal cell carcinoma","Acute lymphoblastic leukemia, Ampullary adenocarcinoma, Astrocytoma, Bladder cancer, Breast cancer, Burkitt Lymphoma, Cervical cancer (All histologies), Cholangiocarcinoma, Chromophobe renal cell carcinoma, Clear cell renal cancer, Colorectal adenocarcinoma, Diffuse intrinsic pontine glioma, Endometrial cancer, Endometrioid ovarian cancer, Esophageal adenocarcinoma, Esophageal squamous carcinoma, Gastric adenocarcinoma, Glioblastoma, Hepatocellular carcinoma, Hurthle cell carcinoma, Leiomysarcoma, Liposarcoma, Low grade glioma, Lung adenocarcinoma, Lung cancer (All histologies), Lung squamous cell carcinoma, Medulloblastoma, Melanoma, Non-Clear cell renal cancer, Osteosarcoma, Ovarian serous carcinoma, Paediatric low grade glioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pan-Glioma, Pan-Gynecological And Breast, Pancreatic neuroendocrine tumours, Prostate cancer, Renal cancer (All histologies), Soft tissue sarcoma, Squamous head and neck cancer, Triple negative breast cancer, Uterine carcinosarcoma, Uveal melanoma","Acute lymphoblastic leukemia, Acute lymphocytic leukemia, Acute myeloid leukemia, Acute promyelocytic leukemia, Chronic lymphocytic leukemia, Desmoplastic melanoma, Glioblastoma, Juvenile myelomonocytic leukemia, Low grade glioma, Multiple myeloma, Neuroblastoma, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Triple negative breast cancer","Gastric adenocarcinoma, Pan-cancer adult, T-Cell leukemia/Lymphoma","Breast cancer, Lung adenocarcinoma, Mucosal melanoma, Pan-cancer adult, Pan-cancer paediatric, Pancreatic ductal adenocarcinoma, Triple negative breast cancer","Desmoplastic melanoma, Melanoma, Pan-cancer adult, Skin basal cell carcinoma","Acute myeloid leukemia, Pan-cancer adult","Lung cancer (All histologies), Paediatric low grade glioma, Pan-cancer adult, Prostate cancer","Acute lymphocytic leukemia, Bladder cancer, Breast cancer, Breast fibroepithelial tumours, Cholangiocarcinoma, Colorectal adenocarcinoma, Desmoplastic melanoma, Esophageal squamous carcinoma, Glioblastoma, Hepatocellular carcinoma, Leiomysarcoma, Liposarcoma, Lung adenocarcinoma, Lung cancer (All histologies), Lung squamous cell carcinoma, Melanoma, Multiple myeloma, Ocular adnexal sebaceous carcinoma, Osteosarcoma, Ovarian cancer, Ovarian serous carcinoma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pan-Glioma, Pan-Gynecological And Breast, Prostate cancer, Retinoblastoma, Skin basal cell carcinoma, Small cell lung cancer, Soft tissue sarcoma, Triple negative breast cancer, Uterine carcinosarcoma","Bladder cancer, Colorectal adenocarcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Pan-cancer adult, Pan-Gastric, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma","Anaplastic thyroid carcinoma, Breast cancer, Burkitt Lymphoma, Osteosarcoma, Pan-cancer adult, Pheochromocytoma, Paraganglioma, Thymic carcinoma, Triple negative breast cancer","Angioimmunoblastic T-Cell lymphoma, Bladder cancer, Breast cancer, Burkitt Lymphoma, Cutaneous T-Cell lymphoma, Diffuse gastric adenocarcinoma, Gastric adenocarcinoma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Peripheral T-Cell lymphoma, T-Cell leukemia/Lymphoma","Ampullary adenocarcinoma, Cholangiocarcinoma, Colorectal adenocarcinoma, Endometrial cancer, Esophageal adenocarcinoma, Gastric adenocarcinoma, Gastric cancer, Pan-cancer adult, Pan-Gastric, Pan-Gynecological And Breast, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma, Pancreatic neoplastic cysts, Prostate cancer","Adrenocortical carcinoma, Biliary tract cancer, Endometrial cancer, Esophageal adenocarcinoma, Gastric adenocarcinoma, Hepatocellular carcinoma, Cholangiocarcinoma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gynecological And Breast","Acute lymphoblastic leukemia, Glioblastoma, Multiple myeloma, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Pan-Gynecological And Breast","Acute lymphoblastic leukemia, Acute myeloid leukemia, Acute promyelocytic leukemia, Breast cancer, Breast fibroepithelial tumours, Chronic myeloid leukemia, Chronic myelomonocytic leukemia, Juvenile myelomonocytic leukemia, Myelodysplasia, Pan-cancer adult, Pan-cancer paediatric, Pan-Gynecological And Breast, Triple negative breast cancer","Chronic myeloid leukemia, Chronic myelomonocytic leukemia, Juvenile myelomonocytic leukemia, Mucinous gastric cancer, Pan-cancer adult","Acute lymphoblastic leukemia, Anaplastic thyroid carcinoma, Angiosarcoma, Astrocytoma, Breast cancer, Breast fibroepithelial tumours, Clear cell renal cancer, Gastric adenocarcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Malignant Pleural Mesothelioma, Multiple myeloma, Non-Clear cell renal cancer, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Pancreatic cancer (All histologies), Papillary renal cell carcinoma, Renal cancer (All histologies), T-Cell leukemia/Lymphoma","Anaplastic thyroid carcinoma, Bladder cancer, Breast cancer, Cholangiocarcinoma, Chronic lymphocytic leukemia, Chronic myelomonocytic leukemia, Hepatocellular carcinoma, Medulloblastoma, Melanoma, Mucosal melanoma, Myelodysplasia, Pan-cancer adult, Pan-cancer paediatric, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma, Uveal melanoma","Colorectal adenocarcinoma, Gastric adenocarcinoma, Pan-cancer adult, Pan-Gastric","Ampullary adenocarcinoma, Biliary tract cancer, Breast cancer, Cholangiocarcinoma, Colorectal adenocarcinoma, Esophageal (Squamous and adenocarcinoma), Esophageal adenocarcinoma, Gastric adenocarcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Pan-cancer adult, Pan-Gastric, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma","Breast cancer, Burkitt Lymphoma, Esophageal adenocarcinoma, Gastric adenocarcinoma, Hepatocellular carcinoma, Low grade glioma, Lung adenocarcinoma, Lung cancer (All histologies), Medulloblastoma, Ovarian small-Cell carcinoma, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma","Anaplastic thyroid carcinoma, Melanoma, Pan-cancer adult, Pan-cancer paediatric, Papillary renal cell carcinoma, Renal cancer (All histologies), Rhabdoid tumour","Acute myeloid leukemia, Acute promyelocytic leukemia, Bladder cancer, Pan-cancer adult","Ampullary adenocarcinoma, Colorectal adenocarcinoma, Pan-cancer adult, Pan-Gastric","Clear cell endometrial cancer, Endometrial cancer, Pan-cancer adult, Pan-Gynecological And Breast, Prostate cancer, Serous endometrial cancer, Triple negative breast cancer, Uterine carcinosarcoma","Chronic myelomonocytic leukemia, Myelodysplasia, Pan-cancer adult","Acute lymphoblastic leukemia, Acute myeloid leukemia, Acute promyelocytic leukemia, Bladder cancer, Breast cancer, Ewing sarcoma, Glioblastoma, Pan-cancer adult, Pan-cancer paediatric, Pan-Glioma, Papillary renal cell carcinoma, Renal cancer (All histologies)","Breast cancer, Cervical cancer (All histologies), Cholangiocarcinoma, Esophageal adenocarcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Non-Small cell lung cancer, Pan-cancer adult","Acute lymphoblastic leukemia, Breast cancer, Cutaneous DLBCL, Diffuse large B-Cell lymphoma, Hepatocellular carcinoma, Cholangiocarcinoma, Pan-cancer adult, Pan-cancer paediatric, T-Cell leukemia/Lymphoma","Breast cancer, Pan-cancer adult, Pan-Gynecological And Breast","Colorectal adenocarcinoma, Pan-cancer adult, Pan-Gastric","Acute myeloid leukemia, Acute promyelocytic leukemia, Angioimmunoblastic T-Cell lymphoma, Breast cancer, Chronic myeloid leukemia, Chronic myelomonocytic leukemia, Multiple myeloma, Myelodysplasia, Myeloproliferative neoplasm, Nasopharyngeal carcinoma, Pan-cancer adult, Pan-cancer paediatric, Peripheral T-Cell lymphoma, T-Cell leukemia/Lymphoma","Ampullary adenocarcinoma, Cervical cancer (All histologies), Cholangiocarcinoma, Colorectal adenocarcinoma, Esophageal (Squamous and adenocarcinoma), Esophageal adenocarcinoma, Esophageal squamous carcinoma, Gastric adenocarcinoma, Pan-cancer adult, Pan-Gastric, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma, Squamous head and neck cancer, Triple negative breast cancer","Diffuse large B-Cell lymphoma, Follicular lymphoma, Pan-cancer adult, T-Cell leukemia/Lymphoma, Triple negative breast cancer","Acute lymphocytic leukemia, Acute myeloid leukemia, Acute promyelocytic leukemia, Adrenocortical carcinoma, Ampullary adenocarcinoma, Anaplastic thyroid carcinoma, Angiosarcoma, Astrocytoma, Biliary tract cancer, Bladder cancer, Breast cancer, Breast fibroepithelial tumours, Burkitt Lymphoma, Cholangiocarcinoma, Chondrosarcoma, Chromophobe renal cell carcinoma, Chronic lymphocytic leukemia, Clear cell endometrial cancer, Clear cell renal cancer, Colorectal adenocarcinoma, Cutaneous T-Cell lymphoma, Cylindroma, Spiradenocarcinoma, Spiradenoma, Desmoplastic melanoma, Diffuse gastric adenocarcinoma, Diffuse intrinsic pontine glioma, Diffuse large B-Cell lymphoma, Duodenal adenocarcinoma, Endometrial cancer, Endometrioid ovarian cancer, Esophageal (Squamous and adenocarcinoma), Esophageal adenocarcinoma, Esophageal squamous carcinoma, Ewing sarcoma, Follicular lymphoma, Gallbladder carcinoma, Gastric adenocarcinoma, Gastric cancer, Glioblastoma, Glioma, Hepatocellular carcinoma, Hurthle cell carcinoma, Intracranial germ cell, Leiomysarcoma, Liposarcoma, Low grade glioma, Lung adenocarcinoma, Lung cancer (All histologies), Lung squamous cell carcinoma, Male breast cancer, Malignant Pleural Mesothelioma, Malignant peripheral nerve sheath tumour, Medulloblastoma, Melanoma, Mucinous gastric cancer, Mucosal melanoma, Multiple myeloma, Myelodysplasia, Nasopharyngeal carcinoma, Natural killer/T cell lymphoma, Non-Clear cell renal cancer, Non-Hodgkin lymphoma, Non-Small cell lung cancer, Ocular adnexal sebaceous carcinoma, Oral squamous cell carcinoma, Osteosarcoma, Ovarian cancer, Ovarian serous carcinoma, Paediatric high grade glioma, Paediatric low grade glioma, Pan-cancer adult, Pan-cancer paediatric, Pan-Gastric, Pan-Glioma, Pan-Gynecological And Breast, Pancreatic cancer (All histologies), Pancreatic ductal adenocarcinoma, Papillary renal cell carcinoma, Papillary thyroid cancer, Penile squamous cancer, Prostate cancer, Renal cancer (All histologies), Serous endometrial cancer, Skin basal cell carcinoma, Small cell lung cancer, Soft tissue sarcoma, Squamous head and neck cancer, T-Cell leukemia/Lymphoma, Thymic carcinoma, Triple negative breast cancer, Uterine carcinosarcoma","Bladder cancer, Clear cell renal cancer, Hepatocellular carcinoma, Pan-cancer adult, Renal angiomyolipoma, Squamous head and neck cancer, Triple negative breast cancer","Clear cell renal cancer, Hepatocellular carcinoma, Pan-cancer adult, Pancreatic neuroendocrine tumours, Renal angiomyolipoma, Squamous head and neck cancer","Acute myeloid leukemia, Acute promyelocytic leukemia, Chronic myelomonocytic leukemia, Endometrial cancer, Lung adenocarcinoma, Lung cancer (All histologies), Myelodysplasia, Pan-cancer adult, Pan-cancer paediatric, Pancreatic cancer (All histologies), Uterine carcinosarcoma","Clear cell renal cancer, Pan-cancer adult, Pancreatic neoplastic cysts, Renal cancer (All histologies)","Acute lymphoblastic leukemia, Acute myeloid leukemia, Acute promyelocytic leukemia, Pan-cancer adult, Pan-cancer paediatric, Triple negative breast cancer","Chronic lymphocytic leukemia, Pan-cancer adult","Bladder cancer, Endometrial cancer, Esophageal adenocarcinoma, Lung squamous cell carcinoma, Pan-cancer adult, Prostate cancer","Breast cancer, Chronic lymphocytic leukemia, Medulloblastoma, Pan-cancer adult, Pan-cancer paediatric, Prostate cancer","Undefined tumor type(s)","Undefined tumor type(s)","Breast cancer","Undefined tumor type(s)","Hepatocellular carcinoma, Cholangiocarcinoma",null,"Biliary tract cancer, Triple negative breast cancer","Undefined tumor type(s)",null,"Undefined tumor type(s)","Undefined tumor type(s)","Neuroblastoma","Lung cancer (All histologies)","Acute promyelocytic leukemia, Breast cancer, Cholangiocarcinoma, Esophageal adenocarcinoma, Gastric cancer, Hepatocellular carcinoma, Lung adenocarcinoma, Neuroblastoma, Pan-cancer adult","Undefined tumor type(s)",null,"Undefined tumor type(s)","Undefined tumor type(s)","Colorectal adenocarcinoma, Gastric adenocarcinoma, Pan-cancer adult","Undefined tumor type(s)","Pan-cancer paediatric","Follicular lymphoma, Non-Hodgkin lymphoma","DLBCL, Follicular lymphoma","Acute lymphoblastic leukemia, Mucinous gastric cancer, Pan-cancer paediatric","Burkitt Lymphoma, DLBCL, Follicular lymphoma, Hepatocellular carcinoma, Cholangiocarcinoma, Non-Hodgkin lymphoma, Pan-cancer adult","Pan-Gastric","Colorectal adenocarcinoma, Gastric adenocarcinoma","Undefined tumor type(s)","Gastric adenocarcinoma, Intracranial germ cell, Paediatric high grade glioma, Pan-cancer paediatric, Pancreatic cancer (All histologies)","Chronic lymphocytic leukemia, DLBCL, Follicular lymphoma, Pan-cancer adult","Gastric cancer","Undefined tumor type(s)","Undefined tumor type(s)",null,"Acute lymphoblastic leukemia, Diffuse large B-Cell lymphoma, Multiple myeloma, Non-Hodgkin lymphoma, Pan-cancer paediatric",null,"Follicular lymphoma","Undefined tumor type(s)","Diffuse large B-Cell lymphoma, Pan-cancer adult","Chronic myelomonocytic leukemia, Desmoplastic melanoma, Intracranial germ cell, Juvenile myelomonocytic leukemia, Lung adenocarcinoma, Pan-cancer paediatric, Triple negative breast cancer","Breast cancer, T-Cell leukemia/Lymphoma","Pan-cancer adult","Paediatric high grade glioma, Pan-cancer paediatric","Acute monocytic leukemia, Biliary tract cancer, Bladder cancer, Breast cancer, Burkitt Lymphoma, Diffuse large B-Cell lymphoma, Non-Hodgkin lymphoma, Paediatric high grade glioma, Pan-cancer paediatric","T-Cell leukemia/Lymphoma","Undefined tumor type(s)","Undefined tumor type(s)","Gastric adenocarcinoma, Parathyroid carcinoma","Lung squamous cell carcinoma, Pan-cancer adult, Pancreatic ductal adenocarcinoma, Small cell lung cancer, Squamous head and neck cancer","Lung squamous cell carcinoma","Nasopharyngeal carcinoma","Chronic lymphocytic leukemia, Paediatric low grade glioma","Breast cancer, Diffuse large B-Cell lymphoma, Natural killer/T cell lymphoma, Non-Hodgkin lymphoma, Pan-cancer adult","Undefined tumor type(s)","Oral squamous cell carcinoma","Undefined tumor type(s)","Acute lymphoblastic leukemia, Breast cancer, Chronic lymphocytic leukemia, Pan-cancer adult, Pan-cancer paediatric",null,"Undefined tumor type(s)","Undefined tumor type(s)","Pan-cancer paediatric","Non-Clear cell renal cancer","Undefined tumor type(s)","Pan-cancer paediatric","Gastric adenocarcinoma, Gastric cancer, Pancreatic ductal adenocarcinoma","Undefined tumor type(s)","Pan-cancer adult","Breast cancer, Chronic myelomonocytic leukemia, Pan-cancer adult","Burkitt Lymphoma, DLBCL, Follicular lymphoma, Pan-cancer adult","Undefined tumor type(s)","Acute myeloid leukemia, Adrenocortical carcinoma, Anaplastic thyroid carcinoma, Glioblastoma, Hurthle cell carcinoma, Pan-cancer adult, Pancreatic neuroendocrine tumours, Triple negative breast cancer","Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Biliary tract cancer, Gastric adenocarcinoma, Triple negative breast cancer","Undefined tumor type(s)","Pan-cancer paediatric",null,"Gastric adenocarcinoma, Pan-cancer paediatric","Follicular lymphoma, Pan-cancer paediatric","Undefined tumor type(s)","Chronic lymphocytic leukemia, Pan-cancer adult","Undefined tumor type(s)","Nasopharyngeal carcinoma",null,"Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Nasopharyngeal carcinoma, Pan-cancer paediatric, Prostate cancer, Triple negative breast cancer","Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Acute lymphoblastic leukemia, Acute promyelocytic leukemia, Diffuse large B-Cell lymphoma, Pan-cancer adult, Pan-cancer paediatric, Triple negative breast cancer","Chronic lymphocytic leukemia","Undefined tumor type(s)","Acute promyelocytic leukemia, Bladder cancer, Gastric adenocarcinoma, Osteosarcoma","Cutaneous T-Cell lymphoma, Non-Hodgkin lymphoma, T-Cell leukemia/Lymphoma, Triple negative breast cancer","Undefined tumor type(s)","Bladder cancer, Chronic myelomonocytic leukemia, Colorectal adenocarcinoma, Diffuse large B-Cell lymphoma, Gastric adenocarcinoma, Lung squamous cell carcinoma, Melanoma","Pancreatic cancer (All histologies)","Burkitt Lymphoma, Nasopharyngeal carcinoma, Pan-cancer adult, Pan-cancer paediatric","Undefined tumor type(s)","Undefined tumor type(s)","Glioblastoma, Lung adenocarcinoma","Undefined tumor type(s)","Papillary renal cell carcinoma","Undefined tumor type(s)","Soft tissue sarcoma",null,"Granulosa Tumours Of The Ovary","Burkitt Lymphoma, Non-Hodgkin lymphoma","Breast cancer","Breast cancer","Undefined tumor type(s)","Acute monocytic leukemia, Acute myeloid leukemia, Juvenile myelomonocytic leukemia, Pan-cancer paediatric, Triple negative breast cancer","Undefined tumor type(s)",null,"Bladder cancer","Gastric adenocarcinoma, Melanoma, Triple negative breast cancer","Glioblastoma, Melanoma",null,"Astrocytoma, Bladder cancer, Breast cancer, Diffuse intrinsic pontine glioma, Diffuse large B-Cell lymphoma, Esophageal adenocarcinoma, Pan-cancer adult, Pan-cancer paediatric",null,"Undefined tumor type(s)","Undefined tumor type(s)",null,null,"Hepatocellular carcinoma","Hepatocellular carcinoma, T-Cell leukemia/Lymphoma","Pan-cancer adult","Burkitt Lymphoma, Pan-cancer adult, Pan-cancer paediatric","Biliary tract cancer, Pan-cancer adult, T-Cell leukemia/Lymphoma","Acute lymphoblastic leukemia, Acute lymphocytic leukemia, Acute promyelocytic leukemia, DLBCL, Follicular lymphoma, Pan-cancer paediatric",null,"Chronic lymphocytic leukemia, Gastric cancer, Multiple myeloma, Non-Hodgkin lymphoma, Pan-cancer adult, T-Cell leukemia/Lymphoma","Breast cancer","Undefined tumor type(s)","Esophageal adenocarcinoma","Undefined tumor type(s)","Undefined tumor type(s)","Pan-cancer adult","Cholangiocarcinoma, Lung adenocarcinoma, Lung cancer (All histologies), Melanoma, Pan-cancer adult","Meningioma, Pan-cancer adult",null,"Triple negative breast cancer",null,"Undefined tumor type(s)","Melanoma, Skin basal cell carcinoma","Pan-cancer paediatric","Undefined tumor type(s)","Chronic lymphocytic leukemia, Clear cell renal cancer, Colorectal adenocarcinoma, Diffuse large B-Cell lymphoma, Esophageal squamous carcinoma, Gastric adenocarcinoma, Gastric cancer, Lung adenocarcinoma, Melanoma, Multiple myeloma, Non-Small cell lung cancer","Papillary thyroid cancer","Undefined tumor type(s)","Cutaneous T-Cell lymphoma","Chronic myelomonocytic leukemia, Paediatric low grade glioma","Biliary tract cancer, Breast cancer",null,"Pan-cancer adult","Anaplastic thyroid carcinoma, Breast cancer, Gastric adenocarcinoma, Nasopharyngeal carcinoma, Pan-cancer adult, Prostate cancer","Chronic myelomonocytic leukemia, Hepatocellular carcinoma, Myeloproliferative neoplasm","Breast cancer, Pan-cancer adult, Prostate cancer, Triple negative breast cancer","Anaplastic thyroid carcinoma, Chronic myeloid leukemia, Colorectal adenocarcinoma, Glioblastoma, Hurthle cell carcinoma, Nasopharyngeal carcinoma, Pan-cancer adult","Undefined tumor type(s)",null,"Pan-cancer adult","Undefined tumor type(s)","Triple negative breast cancer","Gastric adenocarcinoma, Prostate cancer","Lung adenocarcinoma","Cutaneous T-Cell lymphoma, Multiple myeloma, Pan-cancer adult","Desmoplastic melanoma",null,"Pan-cancer paediatric","Undefined tumor type(s)",null,"Undefined tumor type(s)","Lung adenocarcinoma, Paediatric high grade glioma, Soft tissue sarcoma, Triple negative breast cancer","Breast cancer, Lung adenocarcinoma, Paediatric high grade glioma","Pan-cancer paediatric","Undefined tumor type(s)","Diffuse large B-Cell lymphoma, Lung cancer (All histologies), Pan-cancer adult, Pan-cancer paediatric","Diffuse large B-Cell lymphoma, Esophageal squamous carcinoma, Meningioma, Multiple myeloma, Pan-cancer adult, Pancreatic ductal adenocarcinoma","Breast cancer, Gastric adenocarcinoma, Pancreatic ductal adenocarcinoma","Undefined tumor type(s)","Soft tissue sarcoma","Pan-cancer adult, Uveal melanoma","Undefined tumor type(s)","Acute promyelocytic leukemia","Undefined tumor type(s)","Breast cancer","Chronic lymphocytic leukemia, T-Cell leukemia/Lymphoma","Breast cancer, Diffuse large B-Cell lymphoma, Multiple myeloma, Pan-cancer adult","Undefined tumor type(s)","Bladder cancer, Glioblastoma, Pan-cancer adult","Breast cancer, Gastric adenocarcinoma, Melanoma, Pan-cancer adult, Pancreatic ductal adenocarcinoma","Undefined tumor type(s)","Undefined tumor type(s)",null,"Gastric adenocarcinoma","Undefined tumor type(s)","Angiosarcoma","Melanoma, Pan-cancer adult","Acute myeloid leukemia, Angiosarcoma, Pan-cancer adult","Cholangiocarcinoma, Glioblastoma, Paediatric low grade glioma, Pan-cancer adult, Pan-Glioma","Undefined tumor type(s)","Undefined tumor type(s)","Breast fibroepithelial tumours, Pan-cancer adult",null,"Ampullary adenocarcinoma, Osteosarcoma","Undefined tumor type(s)","Undefined tumor type(s)","Lung adenocarcinoma, Lung cancer (All histologies), Pan-cancer adult","Cholangiocarcinoma, Lung adenocarcinoma, Pan-cancer adult","Breast fibroepithelial tumours, Chronic lymphocytic leukemia, Lung cancer (All histologies), Pan-cancer paediatric","Acute lymphoblastic leukemia, Multiple myeloma, Pan-cancer paediatric",null,null,"Prostate cancer","Lung cancer (All histologies), Small cell lung cancer",null,"Gastric adenocarcinoma","Undefined tumor type(s)","Gastric cancer, Parathyroid carcinoma","Hepatocellular carcinoma, Cholangiocarcinoma, Malignant Pleural Mesothelioma, Pan-cancer adult","Diffuse large B-Cell lymphoma, DLBCL, Follicular lymphoma, Endometrial cancer, Non-Hodgkin lymphoma, Pan-cancer adult","Acute lymphoblastic leukemia, Chronic myelomonocytic leukemia, Gastric adenocarcinoma, Juvenile myelomonocytic leukemia, Pan-cancer paediatric",null,"Pan-cancer paediatric","Pan-cancer paediatric","Undefined tumor type(s)","Breast cancer, Pan-Gastric, Pancreatic cancer (All histologies)","Breast cancer, Cholangiocarcinoma","Ameloblastoma, Meningioma, Pan-cancer adult, Pan-cancer paediatric, Skin basal cell carcinoma","Undefined tumor type(s)","Diffuse large B-Cell lymphoma, Follicular lymphoma, Mediastinal B-Cell lymphoma, Pan-cancer adult",null,null,"Undefined tumor type(s)","Breast cancer, Pan-cancer adult","Adrenocortical adenoma, Diffuse large B-Cell lymphoma, Follicular lymphoma, Natural killer/T cell lymphoma, T-Cell leukemia/Lymphoma","Cutaneous T-Cell lymphoma, Natural killer/T cell lymphoma, Pan-cancer paediatric","Diffuse large B-Cell lymphoma, Follicular lymphoma, Pan-cancer adult","Pan-cancer paediatric, Skin basal cell carcinoma","Acute lymphoblastic leukemia, Malignant peripheral nerve sheath tumour, Pan-cancer paediatric","Undefined tumor type(s)","Pan-cancer paediatric","Pan-cancer paediatric",null,"Burkitt Lymphoma, Multiple myeloma, Pan-cancer adult","Cutaneous T-Cell lymphoma","Multiple myeloma, Pan-cancer adult","Bladder cancer, Cholangiocarcinoma, Desmoplastic melanoma, Hepatocellular carcinoma, Hurthle cell carcinoma, Melanoma, Mucosal melanoma, Pan-cancer adult, Pan-cancer paediatric","Acute lymphoblastic leukemia, Acute myeloid leukemia, Nasopharyngeal carcinoma",null,null,null,"Melanoma","Diffuse large B-Cell lymphoma, Follicular lymphoma, Non-Hodgkin lymphoma, Pan-cancer adult","Melanoma, Oral squamous cell carcinoma, Pan-cancer adult, Pan-Glioma, Squamous head and neck cancer, Triple negative breast cancer",null,"Anaplastic thyroid carcinoma, Meningioma","Pan-cancer adult","Undefined tumor type(s)","Bladder cancer, Clear cell renal cancer","Anaplastic thyroid carcinoma, Hurthle cell carcinoma","Gastric adenocarcinoma","Undefined tumor type(s)","Undefined tumor type(s)",null,"Undefined tumor type(s)","Undefined tumor type(s)","Burkitt Lymphoma, Chronic myelomonocytic leukemia","Undefined tumor type(s)","Prostate cancer",null,null,"Cutaneous T-Cell lymphoma, Esophageal squamous carcinoma, Parathyroid carcinoma, T-Cell leukemia/Lymphoma",null,null,"Undefined tumor type(s)","Undefined tumor type(s)","Diffuse large B-Cell lymphoma, Lung squamous cell carcinoma, Small cell lung cancer",null,"Adrenocortical carcinoma, Pan-cancer adult, Prostate cancer","Chronic myelomonocytic leukemia, Juvenile myelomonocytic leukemia, Myelodysplasia, Pan-cancer adult","Undefined tumor type(s)",null,"Undefined tumor type(s)",null,null,null,null,null,null,"Breast cancer, Pan-cancer adult, Squamous head and neck cancer",null,null,null,null,null,"Undefined tumor type(s)","Pan-cancer adult",null,"Undefined tumor type(s)",null,null,"Pan-cancer adult",null,"Undefined tumor type(s)","Cholangiocarcinoma",null,null,"Undefined tumor type(s)","Undefined tumor type(s)",null,"Lung adenocarcinoma","Breast cancer","Breast cancer",null,null,"Diffuse large B-Cell lymphoma, Myeloproliferative neoplasm",null,"Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)",null,"Chronic myelomonocytic leukemia","Ovarian cancer",null,"Pan-cancer adult","Undefined tumor type(s)","Biliary tract cancer, Breast cancer, Pan-cancer adult, Triple negative breast cancer",null,"T-Cell leukemia/Lymphoma","Angioimmunoblastic T-Cell lymphoma, Cutaneous T-Cell lymphoma, T-Cell leukemia/Lymphoma",null,null,"Undefined tumor type(s)","Follicular lymphoma","Undefined tumor type(s)","Breast cancer, Paediatric high grade glioma",null,null,null,null,null,"Undefined tumor type(s)",null,null,"Undefined tumor type(s)",null,null,null,null,null,"Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)",null,"Undefined tumor type(s)",null,null,null,null,null,null,null,"Pan-cancer adult","Acute lymphocytic leukemia","Pan-cancer adult, Pan-Glioma","Undefined tumor type(s)",null,null,null,"Acute lymphoblastic leukemia, Pan-cancer paediatric","Acute myeloid leukemia, Burkitt Lymphoma, Squamous head and neck cancer",null,null,null,null,"Undefined tumor type(s)","Acute lymphoblastic leukemia, Malignant peripheral nerve sheath tumour, Pan-cancer adult, Pan-cancer paediatric",null,null,null,null,null,null,null,null,"Bladder cancer, Breast cancer","Undefined tumor type(s)",null,"Chronic myelomonocytic leukemia","Undefined tumor type(s)","Breast cancer",null,"Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)",null,null,"Nasopharyngeal carcinoma","Undefined tumor type(s)","Undefined tumor type(s)",null,null,"Leiomyoma, Triple negative breast cancer","Undefined tumor type(s)",null,"DLBCL, Follicular lymphoma",null,null,"Undefined tumor type(s)",null,"Undefined tumor type(s)","Undefined tumor type(s)",null,"Pan-cancer paediatric",null,null,null,"Undefined tumor type(s)",null,null,null,null,null,"Chondroblastoma",null,null,null,null,"Undefined tumor type(s)",null,null,"Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Pan-cancer adult","Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)",null,null,null,null,"Undefined tumor type(s)",null,null,null,null,null,null,"Undefined tumor type(s)","Undefined tumor type(s)",null,"Undefined tumor type(s)",null,null,null,null,null,null,null,"Pan-cancer adult","Undefined tumor type(s)",null,null,"Bladder cancer, Breast cancer, Pan-cancer adult",null,"Undefined tumor type(s)","Pan-cancer paediatric",null,"DLBCL, Follicular lymphoma, Pan-cancer adult",null,"Undefined tumor type(s)",null,null,"Undefined tumor type(s)","Gastric cancer, Melanoma",null,"Breast cancer, Pan-cancer adult",null,null,null,null,null,"Pan-cancer adult",null,null,null,"Triple negative breast cancer","Undefined tumor type(s)",null,"Pan-cancer paediatric",null,null,"Undefined tumor type(s)","Undefined tumor type(s)",null,null,"Undefined tumor type(s)","Osteosarcoma","Breast cancer, Paediatric low grade glioma, Pan-cancer adult, Pan-cancer paediatric","Undefined tumor type(s)",null,null,null,"Rhabdomyosarcoma","Undefined tumor type(s)",null,"Undefined tumor type(s)",null,null,null,"Undefined tumor type(s)","Undefined tumor type(s)",null,null,null,null,"Undefined tumor type(s)",null,"Undefined tumor type(s)",null,"Undefined tumor type(s)",null,null,null,null,null,null,null,null,"Undefined tumor type(s)",null,"Nasopharyngeal carcinoma","Undefined tumor type(s)","Undefined tumor type(s)",null,null,"Undefined tumor type(s)",null,null,null,"Undefined tumor type(s)",null,null,"Undefined tumor type(s)",null,null,null,null,null,null,"Undefined tumor type(s)",null,"Follicular lymphoma, Pan-cancer adult",null,"Undefined tumor type(s)","Undefined tumor type(s)",null,"Undefined tumor type(s)","Adrenocortical adenoma",null,null,null,"Undefined tumor type(s)",null,null,null,null,null,null,"Undefined tumor type(s)",null,"Ovarian serous carcinoma",null,null,null,null,null,null,"Undefined tumor type(s)",null,null,null,"Undefined tumor type(s)",null,null,"DLBCL, Follicular lymphoma, Pan-cancer adult",null,null,"Undefined tumor type(s)",null,null,null,null,null,null,"Undefined tumor type(s)","Undefined tumor type(s)",null,"Undefined tumor type(s)",null,null,"Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Pan-cancer adult","Undefined tumor type(s)",null,"Acute myeloid leukemia, Pan-cancer adult","Undefined tumor type(s)","Undefined tumor type(s)",null,"Pan-cancer paediatric","Undefined tumor type(s)",null,null,"Undefined tumor type(s)",null,null,"Undefined tumor type(s)",null,null,null,"Undefined tumor type(s)",null,"Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)","Pan-cancer adult","Undefined tumor type(s)",null,"Diffuse large B-Cell lymphoma, Melanoma, Soft tissue sarcoma",null,null,"Burkitt Lymphoma, Pan-cancer adult, Pan-cancer paediatric",null,null,null,"Undefined tumor type(s)","Undefined tumor type(s)",null,null,null,null,"Pan-cancer paediatric","Pan-cancer paediatric","Undefined tumor type(s)",null,"Undefined tumor type(s)",null,null,"Undefined tumor type(s)",null,"Undefined tumor type(s)",null,null,null,null,null,null,null,"Undefined tumor type(s)",null,"Lung squamous cell carcinoma",null,"Osteosarcoma","Undefined tumor type(s)","Undefined tumor type(s)","Undefined tumor type(s)",null,null,null,null,null,null,null,null,null,null,"Undefined tumor type(s)",null,null,null,null,null,null,null,null,null],[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,false,false,false,false,true,false,false,true,false,false,false,false,false,false,true,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,true,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,true,false,false,false,false,false,false,false,true,false,false,false,true,false,true,false,false,true,true,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,true,false,true,false,false,false,false,false,false,false,false,false,false,true,true,true,false,true,true,false,true,false,false,false,false,false,false,false,true,false,false,true,false,false,false,false,false,false,false,false,false,false,true,false,false,true,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,true,true,false,false,false,true,false,false,false,true,true,false,false,true,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,true,true,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,true,true,true,false,false,false,true,false,false,false,false,false,false,false,false,true,false,false,false,false,false,true,true,false,true,true,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,true,false,false,true,false,false,true,false,true,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,true,false,true,false,false,false,false,false,false,false,true,false,false,false,false,false,true,false,true,false,false,false,true,true,true,false,false,false,false,true,false,true,false,false,false,false,false,false,false,false,true,true,false,false,true,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,true,false,false,false,false,false,true,true,false,true,true,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,true,true,false,false,true,false,false,true,true,false,false,false,true,false,false,false,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,true,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,true,false,false,false,true,true,true,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,true,true,false,false,false,false,true,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,true,false,false,false,false,false,false,true,false,false,true,false,false,false,true,true,false,false,false,false,false,false,false,false,false,false,true,false,false,false,true,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,true,false,false,false,false,false,false,false,false,false,true,true,false,false,true,true,false,false,false,false,false,false,false,false,false,false,false,true,false],[3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],["<a href='https://www.ncbi.nlm.nih.gov/pubmed/36854750' target='_blank'>Qiu et al., 2023, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33335095' target='_blank'>Carter et al., Signal Transduct Target Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32699415' target='_blank'>Chan et al., Nature, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31474570' target='_blank'>Reina-Campos et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31447321' target='_blank'>Gouw et al., Cell Metab, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30745579' target='_blank'>Pinho et al., Nat Rev Mol Cell Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30107177' target='_blank'>Vanden et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29551267' target='_blank'>Xiao et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26073130' target='_blank'>Shojaee et al., 2015, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25686603' target='_blank'>Pemovska et al., Nature, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25117713' target='_blank'>Li et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24525236' target='_blank'>Puissant et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24481312' target='_blank'>Fruman et al., Nat Rev Drug Discov, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21593872' target='_blank'>Duy et al., Nature, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17206155' target='_blank'>Sergina et al., Nature, 2007<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36861402' target='_blank'>Tefferi et al., 2023, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33627422' target='_blank'>Serresi et al., Sci Adv, 2021<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40108474' target='_blank'>Yuan et al., 2025, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39833612' target='_blank'>Snaebjornsson et al., 2025, Nat Metab<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39775867' target='_blank'>Landry et al., 2025, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35489428' target='_blank'>Zhang et al., 2022, Gastroenterology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32749453' target='_blank'>Varga et al., J Exp Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31686003' target='_blank'>Hoxhaj et al., Nat Rev Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29661000' target='_blank'>Mettlen et al., Annu Rev Biochem, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28283069' target='_blank'>Saxton et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27938406' target='_blank'>Huang et al., Mol Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27088857' target='_blank'>Lien et al., Nat Cell Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26206664' target='_blank'>Scarzello et al., Gut, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25656847' target='_blank'>Papadakis et al., Cell Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24463355' target='_blank'>Bernhart et al., Neuro Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24291004' target='_blank'>Piovan et al., 2013, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23902739' target='_blank'>Conley-LaComb et al., Mol Cancer, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38965396' target='_blank'>Sen et al., 2024, Nat Rev Clin Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38039964' target='_blank'>Bergaggio et al., 2023, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35534623' target='_blank'>Cooper et al., Nat Rev Clin Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32822576' target='_blank'>Maynard et al., 2020, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32649874' target='_blank'>Gillette et al., Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31614113' target='_blank'>Berko et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31125062' target='_blank'>Mazieres et al., Ann Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30108370' target='_blank'>Recondo et al., Nat Rev Clin Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26144315' target='_blank'>Zou et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25759024' target='_blank'>Wilson et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25517749' target='_blank'>Bresler et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23064464' target='_blank'>Laimer et al., Nat Med, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22789543' target='_blank'>Berry et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22585002' target='_blank'>Carpenter et al., Nat Rev Clin Oncol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21383740' target='_blank'>Wellstein et al., Nat Med, 2011<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39755622' target='_blank'>Tu et al., 2025, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39375704' target='_blank'>Gabel et al., 2024, Genome Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38523155' target='_blank'>Hu et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37580540' target='_blank'>Vande Voorde et al., 2023, Nat Metab<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36446897' target='_blank'>Brunner et al., 2022, Nat Rev Endocrinol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32376773' target='_blank'>Leibold et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31685988' target='_blank'>Schmidt et al., Nat Cell Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31685958' target='_blank'>Jakubek et al., 2019, Nat Biotechnol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29610476' target='_blank'>Rogers et al., Nat Genet, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28425994' target='_blank'>Maddocks et al., Nature, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27698416' target='_blank'>Blokzijl et al., Nature, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25594431' target='_blank'>Rothlin et al., Annu Rev Immunol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25337879' target='_blank'>Sánchez-Rivera et al., Nature, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24609463' target='_blank'>Fancy et al., Nat Neurosci, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20858269' target='_blank'>Huang et al., Mol Cancer, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37847647' target='_blank'>Ma et al., 2023, J Natl Cancer Inst<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37271632' target='_blank'>Linder et al., 2023, Eur Urol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33431496' target='_blank'>Welti et al., Cancer Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33328331' target='_blank'>Dong et al., 2020, Sci Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32941624' target='_blank'>Feng et al., Nucleic Acids Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32057540' target='_blank'>Alyamani et al., Ann Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31796514' target='_blank'>Cao et al., Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31363002' target='_blank'>Beltran et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31218365' target='_blank'>Mateo et al., Ann Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31198906' target='_blank'>Bader et al., Nat Metab, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30755441' target='_blank'>Robinson et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29909987' target='_blank'>Takeda et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29694822' target='_blank'>Abate-Shen et al., N Engl J Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19359544' target='_blank'>Tran et al., 2009, Science<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18645193' target='_blank'>Attard et al., J Clin Oncol, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38217548' target='_blank'>Li et al., 2024, Aging (Albany NY)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35302851' target='_blank'>Mooz et al., Sci Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32923479' target='_blank'>Lin et al., Biomed Res Int, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24569458' target='_blank'>Imielinski et al., J Clin Invest, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38600891' target='_blank'>Xiao et al., 2024, CNS Neurosci Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38143235' target='_blank'>Liu et al., 2023, Cancer Commun (Lond)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38102228' target='_blank'>Viergever et al., 2023, Br J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37402719' target='_blank'>Fuchs et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36937541' target='_blank'>Teboul et al., 2023, NAR Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36044839' target='_blank'>Guo et al., Cell Rep, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35187519' target='_blank'>Hirt et al., Cell Genom, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33542970' target='_blank'>Caeser et al., JCO Precis Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30829920' target='_blank'>Xie et al., 2019, Clin Transl Gastroenterol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30467048' target='_blank'>Karachaliou et al., J Thorac Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29784854' target='_blank'>Saito et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29590609' target='_blank'>Fukumoto et al., Cell Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28737768' target='_blank'>Bitler et al., Nat Cell Biol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27941798' target='_blank'>Mathur et al., Nat Genet, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26170901' target='_blank'>MacNeil et al., 2015, Genome Med<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39123171' target='_blank'>Kassem et al., 2024, BMC Med Genomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36301018' target='_blank'>Zhang et al., Hematol Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33742126' target='_blank'>Moreno et al., Oncogene, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32382660' target='_blank'>Alvarez et al., Health Sci Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25790038' target='_blank'>Kovac et al., Nat Commun, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40772034' target='_blank'>Wang et al., 2025, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39199016' target='_blank'>Shin et al., 2024, Ann Clin Transl Neurol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39078402' target='_blank'>Maese et al., 2024, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38359087' target='_blank'>Köhnke et al., 2024, Blood Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37917239' target='_blank'>Ge et al., 2023, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37053013' target='_blank'>Lin et al., 2023, JCI Insight<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36005565' target='_blank'>Gleitz et al., 2022, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35133065' target='_blank'>Liu et al., Cancer Sci, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34341073' target='_blank'>Zhao et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31160638' target='_blank'>Tamura et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30181385' target='_blank'>Schwartz et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29743720' target='_blank'>Guo et al., Leukemia, 2018<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38903501' target='_blank'>Bertilaccio et al., 2024, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38841800' target='_blank'>Davies et al., 2024, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37524790' target='_blank'>Bland et al., 2023, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36862025' target='_blank'>Angela De Stefano et al., 2023, Endocr Relat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33809641' target='_blank'>Mosquera et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31570692' target='_blank'>Mosquera et al., Blood Cancer J, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31543463' target='_blank'>Guièze et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30400006' target='_blank'>Xu et al., Endocr Relat Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30038380' target='_blank'>Leeksma et al., Leukemia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29615613' target='_blank'>McGrail et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28679620' target='_blank'>Wang et al., Genome Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27580852' target='_blank'>Frazzi et al., J Transl Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23894700' target='_blank'>Tang et al., Oncoimmunology, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20086003' target='_blank'>Ouillette et al., Clin Cancer Res, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/11756185' target='_blank'>Stankovic et al., Blood, 2002<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39807103' target='_blank'>Kubo-Kaneda et al., 2025, Oncol Lett<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37936324' target='_blank'>Derby et al., 2023, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37491836' target='_blank'>Xu et al., 2023, Mol Oncol<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35354845' target='_blank'>Jiang et al., Sci Rep, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33934450' target='_blank'>Ono et al., Cancer Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31428092' target='_blank'>Wang et al., Front Immunol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30664638' target='_blank'>Wong et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28056863' target='_blank'>Tonini et al., BMC Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27209355' target='_blank'>Comino-Méndez et al., Cancer Genet, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26618343' target='_blank'>Bai et al., Nat Genet, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24468746' target='_blank'>Qadeer et al., J Invest Dermatol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24214964' target='_blank'>Cheng et al., Nucleic Acids Res, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39123171' target='_blank'>Kassem et al., 2024, BMC Med Genomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37369946' target='_blank'>Janin et al., 2023, Cancer Metastasis Rev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35921500' target='_blank'>Liang et al., Hepatology, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30737831' target='_blank'>Qiao et al., Hepatology, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28100566' target='_blank'>Picco et al., 2017, EMBO Mol Med<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40251399' target='_blank'>Xie et al., 2025, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40131247' target='_blank'>Elias et al., 2025, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39592856' target='_blank'>Cortez Cardoso Penha et al., 2024, Mol Syst Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39344744' target='_blank'>Repo et al., 2024, Pigment Cell Melanoma Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38781609' target='_blank'>Li et al., 2024, Int Immunopharmacol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35758895' target='_blank'>Rappold et al., 2022, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31862487' target='_blank'>Hirsch et al., J Hepatol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30914057' target='_blank'>Ladanyi et al., 2019, Genome Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30858363' target='_blank'>Syafruddin et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30013160' target='_blank'>Asada et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29651023' target='_blank'>Hou et al., Exp Mol Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26081045' target='_blank'>Mori et al., Cancer Sci, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25790038' target='_blank'>Kovac et al., Nat Commun, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24880939' target='_blank'>Gorantla et al., Hematol Oncol Clin North Am, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23303902' target='_blank'>Pedersen et al., Cancer Discov, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40240348' target='_blank'>Cui et al., 2025, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39235528' target='_blank'>Portelinha et al., 2024, J Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38811841' target='_blank'>Lazar et al., 2024, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37289191' target='_blank'>Ramkumar et al., 2023, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36656933' target='_blank'>Mlynarczyk et al., 2023, Science<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36346827' target='_blank'>Flümann et al., Blood Cancer Discov, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33632214' target='_blank'>Kosiol et al., Mol Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33569544' target='_blank'>Parsa et al., Nat Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33447829' target='_blank'>Flümann et al., Blood Cancer Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32350066' target='_blank'>Saffie et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31543463' target='_blank'>Guièze et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31519498' target='_blank'>Meyer et al., Immunity, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29335468' target='_blank'>Teater et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25895919' target='_blank'>Hata et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20885445' target='_blank'>Oh et al., Cell Death Differ, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40565027' target='_blank'>Wang et al., 2025, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40113912' target='_blank'>Rompietti et al., 2025, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38722224' target='_blank'>Saoud et al., 2024, Genes Chromosomes Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33926021' target='_blank'>Eckardt et al., 2021, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32490123' target='_blank'>Lei et al., Pract Lab Med, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/31836838' target='_blank'>Fane et al., Nat Rev Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28751651' target='_blank'>Levy et al., Nat Rev Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28273028' target='_blank'>Hochhaus et al., N Engl J Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28256574' target='_blank'>Schneider et al., Nat Rev Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28111070' target='_blank'>Spitzer et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26967284' target='_blank'>Lemmon et al., Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24957944' target='_blank'>Holderfield et al., Nat Rev Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24481312' target='_blank'>Fruman et al., Nat Rev Drug Discov, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24315093' target='_blank'>Guo et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23993097' target='_blank'>Chang et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23791173' target='_blank'>Olenchock et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23429735' target='_blank'>Xing et al., Nat Rev Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23332765' target='_blank'>Burd et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22965151' target='_blank'>Younes et al., Nat Rev Clin Oncol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22000016' target='_blank'>Karreth et al., Cell, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36108220' target='_blank'>Langille et al., 2022, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35641483' target='_blank'>Wang et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33686070' target='_blank'>Bach et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33262145' target='_blank'>Shen et al., Genes Dev, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31604914' target='_blank'>Kanakkanthara et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31267556' target='_blank'>Tharmapalan et al., EMBO J, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30914433' target='_blank'>Kraya et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29241686' target='_blank'>Rao et al., Trends Cell Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28649985' target='_blank'>Zhang et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27815389' target='_blank'>Li et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27664235' target='_blank'>Amaravadi et al., Genes Dev, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27241552' target='_blank'>Sigl et al., Cell Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25412762' target='_blank'>Jiang et al., Nat Commun, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24315093' target='_blank'>Guo et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/16585167' target='_blank'>Smolen et al., Cancer Res, 2006<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40104509' target='_blank'>Luo et al., 2025, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38091511' target='_blank'>Liu et al., 2023, Gigascience<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33669657' target='_blank'>McNevin et al., Diagnostics (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32980867' target='_blank'>Chen et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32751922' target='_blank'>Czarnecka et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32210163' target='_blank'>Testa et al., Med Sci (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31433991' target='_blank'>Gruber et al., Cell Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30914433' target='_blank'>Kraya et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30250803' target='_blank'>Hou et al., Adv Sci (Weinh), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28514723' target='_blank'>Thibodeau et al., Cold Spring Harb Mol Case Stud, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28067867' target='_blank'>Taylor et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27797936' target='_blank'>Mace et al., Gut, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27555886' target='_blank'>Iyevleva et al., Hered Cancer Clin Pract, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24315093' target='_blank'>Guo et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20735817' target='_blank'>Holstege et al., BMC Cancer, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34175492' target='_blank'>Riller et al., Biomed J, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33542970' target='_blank'>Caeser et al., JCO Precis Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29259013' target='_blank'>Ekambaram et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28993409' target='_blank'>Saba et al., Cancer Res, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/31797588' target='_blank'>Tao et al., Pac Symp Biocomput, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31217499' target='_blank'>Guo et al., Sci Rep, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39177514' target='_blank'>Lints et al., 2024, Biol Open<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37992684' target='_blank'>Popescu et al., 2023, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37904054' target='_blank'>Cerella et al., 2023, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37339484' target='_blank'>Abel et al., 2023, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36831308' target='_blank'>Khan et al., 2023, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36711871' target='_blank'>Abel et al., medRxiv, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35756660' target='_blank'>Kurtz et al., Front Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33080932' target='_blank'>Lamba et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32414754' target='_blank'>Rockne et al., Cancer Res, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39034322' target='_blank'>Wang et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37100955' target='_blank'>Vitale et al., 2023, Cell Death Differ<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35513054' target='_blank'>Caksa et al., 2022, Pharmacol Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32584970' target='_blank'>Nadeu et al., Blood, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31618628' target='_blank'>Botton et al., Cell Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30899002' target='_blank'>Qie et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30718512' target='_blank'>Xue et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30665445' target='_blank'>Zhang et al., J Exp Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27013210' target='_blank'>Smith et al., Blood, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26658964' target='_blank'>Sherr et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24282282' target='_blank'>Ju et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23318439' target='_blank'>Vaites et al., Oncogene, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20951943' target='_blank'>Aggarwal et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19147556' target='_blank'>Saab et al., Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18797463' target='_blank'>Le et al., Br J Cancer, 2008<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40578157' target='_blank'>NA et al., 2025, Comput Biol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40495692' target='_blank'>Li et al., 2025, Cancer Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37973922' target='_blank'>Fitzpatrick et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33204396' target='_blank'>Vargas et al., Oxid Med Cell Longev, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33080522' target='_blank'>Wu et al., Transl Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32886101' target='_blank'>Ozawa et al., J Cell Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31835751' target='_blank'>Feng et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31771969' target='_blank'>Zhang et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31174492' target='_blank'>Dominguez et al., BMC Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29688752' target='_blank'>Devlin et al., Am J Respir Cell Mol Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28574612' target='_blank'>Bakshi et al., Mol Carcinog, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28275688' target='_blank'>Katona et al., Cell Mol Gastroenterol Hepatol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25174397' target='_blank'>Truesdell et al., Oncogene, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20609236' target='_blank'>Chao et al., Mol Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18697830' target='_blank'>Spiczka et al., J Cell Sci, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40389480' target='_blank'>Troester et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39859211' target='_blank'>Serrano et al., 2025, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39368479' target='_blank'>Tien et al., 2024, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38562774' target='_blank'>Tien et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38380359' target='_blank'>Wang et al., 2024, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34686673' target='_blank'>Liang et al., Cell Death Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32483448' target='_blank'>Liu et al., Theranostics, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31468695' target='_blank'>Choi et al., EMBO Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29909985' target='_blank'>Viswanathan et al., Cell, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40013319' target='_blank'>Wang et al., 2025, Cancer Commun (Lond)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39702187' target='_blank'>Su et al., 2024, Hered Cancer Clin Pract<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33369721' target='_blank'>Braal et al., Drugs, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33078086' target='_blank'>Zhou et al., 2020, Mol Ther Nucleic Acids<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25832654' target='_blank'>Namiki et al., Cancer Res, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36937541' target='_blank'>Teboul et al., 2023, NAR Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34788144' target='_blank'>Zhang et al., Genet Test Mol Biomarkers, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34580063' target='_blank'>Fletcher-Sananikone et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25902869' target='_blank'>Ong et al., Breast Cancer Res, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33401758' target='_blank'>Mohr et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23470457' target='_blank'>Lo et al., Cell Cycle, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22929890' target='_blank'>Salim et al., Br J Cancer, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20404527' target='_blank'>Bhatia et al., Cell Cycle, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19648963' target='_blank'>Kelly-Spratt et al., Oncogene, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40106536' target='_blank'>Dummer et al., 2025, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36869047' target='_blank'>Mainwaring et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36165893' target='_blank'>Lander et al., 2022, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36129461' target='_blank'>Dias Costa et al., 2022, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35187519' target='_blank'>Hirt et al., Cell Genom, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32398706' target='_blank'>Ho et al., Nat Rev Clin Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31320640' target='_blank'>Newell et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30889378' target='_blank'>Cooper et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30686769' target='_blank'>Connor et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30401703' target='_blank'>Abdul-Aziz et al., Blood, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29990501' target='_blank'>Zeng et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29337182' target='_blank'>Seino et al., 2018, Cell Stem Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28232723' target='_blank'>Ju et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27811314' target='_blank'>Hessmann et al., Gut, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24717934' target='_blank'>Morran et al., Gut, 2014<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37996430' target='_blank'>Negri et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37267435' target='_blank'>Schulz et al., 2023, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33623142' target='_blank'>Heyes et al., Leukemia, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32591645' target='_blank'>Jensen et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32381577' target='_blank'>D'Altri et al., Haematologica, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30877232' target='_blank'>Volpe et al., Life Sci Alliance, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29175379' target='_blank'>Swords et al., Leuk Res, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40480417' target='_blank'>Shi et al., 2025, Cancer Lett<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33981601' target='_blank'>Novillo et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29406849' target='_blank'>Blazer et al., Rambam Maimonides Med J, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26700815' target='_blank'>Flavahan et al., Nature, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23104009' target='_blank'>Le et al., Nat Genet, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36061833' target='_blank'>Zhang et al., JTO Clin Res Rep, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35020107' target='_blank'>Iyevleva et al., Breast Cancer Res Treat, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34513715' target='_blank'>Kolijn et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30695561' target='_blank'>Bessonov et al., Vopr Onkol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27896994' target='_blank'>Gallion et al., Pac Symp Biocomput, 2017<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/33577785' target='_blank'>Wang et al., Cancer Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30181244' target='_blank'>Jia et al., Cancer Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24111892' target='_blank'>Dasgupta et al., Annu Rev Med, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39883527' target='_blank'>Hamamoto et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38950902' target='_blank'>Chen et al., 2024, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34078608' target='_blank'>Wang et al., 2021, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33859938' target='_blank'>Puvvula et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32690881' target='_blank'>Fang et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32688344' target='_blank'>Lai et al., Aging (Albany NY), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32435142' target='_blank'>Akhtar et al., Saudi Pharm J, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32374727' target='_blank'>Rinaldi et al., PLoS One, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29415465' target='_blank'>Eitsuka et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28862757' target='_blank'>Zhang et al., J Pathol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28819414' target='_blank'>Lee et al., 2017, J Cancer<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39800372' target='_blank'>Cao et al., 2025, J Immunother Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39242496' target='_blank'>Chan et al., 2024, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38103030' target='_blank'>LaPelusa et al., 2023, Oncologist<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36894036' target='_blank'>Saito et al., 2023, Gastroenterology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36280791' target='_blank'>Villar et al., 2022, Nat Chem Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34488905' target='_blank'>Wang et al., 2021, Biomark Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34387871' target='_blank'>Hlady et al., 2021, Hepatology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32649874' target='_blank'>Gillette et al., Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32409304' target='_blank'>Ji et al., Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29628508' target='_blank'>Labgaa et al., Oncogene, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29101368' target='_blank'>Letouzé et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27390343' target='_blank'>Ganesh et al., Mol Cancer Ther, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26170901' target='_blank'>MacNeil et al., 2015, Genome Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25457204' target='_blank'>Delgado et al., J Hepatol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22811581' target='_blank'>Lachenmayer et al., Clin Cancer Res, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32284825' target='_blank'>Dhondt et al., J Extracell Vesicles, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25995385' target='_blank'>Dorr et al., Mol Cancer Res, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/31101826' target='_blank'>Rashid et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31093340' target='_blank'>Verhoeft et al., Cancers Head Neck, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26413810' target='_blank'>Chaudhary et al., Oncotarget, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36834820' target='_blank'>Ghatak et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33824477' target='_blank'>Dhomen et al., Br J Cancer, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38057330' target='_blank'>Radford et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36050548' target='_blank'>Brady et al., 2022, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33490290' target='_blank'>Xu et al., J Immunol Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32963611' target='_blank'>Huh et al., Oncol Lett, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31204176' target='_blank'>Huang et al., Cell Stem Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28679620' target='_blank'>Wang et al., Genome Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26541825' target='_blank'>Bol et al., Mol Cancer, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40240142' target='_blank'>Fraire et al., 2025, Genes Dev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39921216' target='_blank'>Yang et al., 2025, Endocr Rev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39121796' target='_blank'>Alirezaie et al., 2024, Lung Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38771248' target='_blank'>Benton et al., 2024, Mol Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38712047' target='_blank'>Fraire et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35066877' target='_blank'>Hanna et al., J Pathol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31367007' target='_blank'>Sutaria et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30956758' target='_blank'>Poma et al., Oncotarget, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30877433' target='_blank'>Lee et al., Acta Neuropathol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30704144' target='_blank'>Djavaheri-Mergny et al., Cells, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30649606' target='_blank'>Lee et al., Acta Neuropathol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28916654' target='_blank'>Hanna et al., Cancer Res, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37344104' target='_blank'>Ivanovic et al., 2023, Genome Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36797244' target='_blank'>Zhao et al., Signal Transduct Target Ther, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36457063' target='_blank'>Bäumer et al., J Hematol Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35943291' target='_blank'>Gamlen et al., 2022, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35449309' target='_blank'>Na et al., 2022, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33087320' target='_blank'>Venugopal et al., 2020, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31955845' target='_blank'>Catania et al., Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31101826' target='_blank'>Rashid et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30262806' target='_blank'>Rusch et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30181385' target='_blank'>Schwartz et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30049810' target='_blank'>Sanders et al., 2018, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28959968' target='_blank'>Smith et al., Nature, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27841873' target='_blank'>Guryanova et al., Nat Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27300438' target='_blank'>Yang et al., 2016, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25693834' target='_blank'>Yang et al., Nat Rev Cancer, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35534623' target='_blank'>Cooper et al., Nat Rev Clin Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32203277' target='_blank'>Carneiro et al., Nat Rev Clin Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30850748' target='_blank'>Kung et al., Nat Rev Drug Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30532012' target='_blank'>Altorki et al., Nat Rev Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30108370' target='_blank'>Recondo et al., Nat Rev Clin Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29625053' target='_blank'>Bailey et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29217837' target='_blank'>Dugger et al., 2017, Nat Rev Drug Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27388699' target='_blank'>Vargas et al., Nat Rev Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27079802' target='_blank'>Topalian et al., Nat Rev Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25679766' target='_blank'>Chen et al., Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25056707' target='_blank'>Chen et al., Nat Rev Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22579283' target='_blank'>Lee et al., Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20966921' target='_blank'>Pao et al., Nat Rev Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20574449' target='_blank'>Heyer et al., Nat Rev Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20371346' target='_blank'>Sharma et al., Cell, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39225726' target='_blank'>Abi-Raad et al., 2024, Virchows Arch<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34475951' target='_blank'>Patel et al., Int J Endocrinol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30912334' target='_blank'>Song et al., Endocrinol Metab (Seoul), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30089490' target='_blank'>Pstrąg et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28117295' target='_blank'>Alsina et al., Mol Imaging Radionucl Ther, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27089234' target='_blank'>Johnson et al., Pigment Cell Melanoma Res, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37422534' target='_blank'>Ooki et al., 2023, Cancer Metastasis Rev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36999792' target='_blank'>Zhao et al., 2023, Cancer Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36840413' target='_blank'>Horie et al., 2023, Cancer Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29738475' target='_blank'>Li et al., Molecules, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28389139' target='_blank'>Rizvi et al., J Hepatol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26806338' target='_blank'>Yachida et al., Cancer Cell, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40319015' target='_blank'>Umphred-Wilson et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38994683' target='_blank'>Waddell et al., 2024, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38566191' target='_blank'>Hou et al., 2024, Cell Commun Signal<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38469149' target='_blank'>Waddell et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33577785' target='_blank'>Wang et al., Cancer Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28465297' target='_blank'>Kaymaz et al., 2017, Mol Cancer Res<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40121376' target='_blank'>Hong et al., 2025, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38746132' target='_blank'>Zhao et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35676472' target='_blank'>Patel et al., Nature, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30135421' target='_blank'>Tarade et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29875134' target='_blank'>Rodrigues et al., Cancer Discov, 2018<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35304604' target='_blank'>Goel et al., Nat Rev Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31836877' target='_blank'>Goutsouliak et al., Nat Rev Clin Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29422598' target='_blank'>Hunter et al., Nat Rev Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26598547' target='_blank'>Mazières et al., Ann Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25899785' target='_blank'>Kris et al., Ann Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25633797' target='_blank'>Asghar et al., Nat Rev Drug Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24581498' target='_blank'>Valiente et al., Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24060865' target='_blank'>Talmadge et al., Nat Rev Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22579283' target='_blank'>Lee et al., Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22439924' target='_blank'>Magee et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22094256' target='_blank'>Anders et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21741601' target='_blank'>Ni et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20129248' target='_blank'>Jeselsohn et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19644473' target='_blank'>Liu et al., Nat Rev Drug Discov, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18662538' target='_blank'>Massagué et al., Cell, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40353993' target='_blank'>Lim et al., 2025, Cell Oncol (Dordr)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39797621' target='_blank'>Li et al., 2025, Cancer Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36727489' target='_blank'>Cheng et al., Bioinformatics, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35302851' target='_blank'>Mooz et al., Sci Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32820012' target='_blank'>Del et al., ESMO Open, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30355677' target='_blank'>Hill et al., Mol Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26434585' target='_blank'>Schneider et al., Oncogene, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25849870' target='_blank'>Ren et al., Breast Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25836719' target='_blank'>Cohen et al., Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25609069' target='_blank'>Campbell et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23999432' target='_blank'>Morrison et al., J Clin Invest, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23378579' target='_blank'>Asrani et al., Mol Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22105361' target='_blank'>Lindzen et al., Oncogene, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21741601' target='_blank'>Ni et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21725367' target='_blank'>Smirnova et al., Oncogene, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/29371993' target='_blank'>Mishra et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29342193' target='_blank'>Jones et al., PLoS One, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24258993' target='_blank'>Lau et al., Methods Mol Biol, 2014<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40494627' target='_blank'>Zekri et al., 2025, Genome Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40057527' target='_blank'>Scandino et al., 2025, NPJ Breast Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38321573' target='_blank'>Lawrence-Paul et al., 2024, Genome Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37071495' target='_blank'>Gou et al., 2023, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35881485' target='_blank'>Udden et al., JCI Insight, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35575456' target='_blank'>Hosfield et al., 2022, Elife<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34294894' target='_blank'>Hickey et al., Br J Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32816914' target='_blank'>Conway et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31092827' target='_blank'>Liu et al., 2019, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30089255' target='_blank'>Lei et al., Cell Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29662238' target='_blank'>Katzenellenbogen et al., Nat Rev Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28786813' target='_blank'>Raj et al., Elife, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28273452' target='_blank'>Dreijerink et al., Cell Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27571262' target='_blank'>Bailey et al., Nat Genet, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26836308' target='_blank'>Fanning et al., Elife, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40595652' target='_blank'>Zhang et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36670102' target='_blank'>Chen et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34347390' target='_blank'>Wu et al., Cancer Commun (Lond), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33512434' target='_blank'>Wen et al., Blood, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32610068' target='_blank'>Cao et al., 2020, Trends Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31481404' target='_blank'>Li et al., Cancer Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31263101' target='_blank'>Smith et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31174843' target='_blank'>Knudsen et al., Trends Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30587549' target='_blank'>Bhatia et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30487290' target='_blank'>Serresi et al., J Exp Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30008323' target='_blank'>Zingg et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27587466' target='_blank'>Frankel et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27312177' target='_blank'>Zhang et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23636686' target='_blank'>Yang et al., Protein Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20154697' target='_blank'>Min et al., Nat Med, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40713501' target='_blank'>Arora et al., 2025, Cell Mol Biol Lett<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40667321' target='_blank'>Glasmacher et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40379735' target='_blank'>Yanping et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40242237' target='_blank'>Wang et al., 2025, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39738653' target='_blank'>Lu et al., 2025, Nat Cell Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36462630' target='_blank'>Li et al., 2022, Genomics Proteomics Bioinformatics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33970999' target='_blank'>Silva et al., 2021, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33672117' target='_blank'>Zengin et al., J Pers Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33120354' target='_blank'>Santana et al., Eur J Endocrinol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29599409' target='_blank'>Xu et al., Clin Cancer Res, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40375984' target='_blank'>Xian et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39851497' target='_blank'>Awan et al., 2025, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37587857' target='_blank'>Mayo et al., 2023, EMBO Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37468606' target='_blank'>Hall et al., 2023, Commun Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36322817' target='_blank'>Yang et al., Blood Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31772025' target='_blank'>Cuevas et al., Proc Natl Acad Sci U S A, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31690218' target='_blank'>Brandstadter et al., 2019, Open Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30885698' target='_blank'>Mahamdallie et al., Lancet Child Adolesc Health, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30642944' target='_blank'>Ruiz et al., J Exp Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27602765' target='_blank'>Spinella et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26432751' target='_blank'>Heo et al., Semin Cancer Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26178611' target='_blank'>Cho et al., Pigment Cell Melanoma Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24831540' target='_blank'>Fang et al., Stem Cells, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22675565' target='_blank'>Kalender et al., PLoS One, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37549632' target='_blank'>Fettke et al., 2023, EBioMedicine<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34046693' target='_blank'>Lötsch et al., Acta Neuropathol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32973083' target='_blank'>Kim et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30413411' target='_blank'>Hu et al., Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29720732' target='_blank'>Hu et al., Leukemia, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29458371' target='_blank'>Kleczko et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29455658' target='_blank'>Butti et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28775339' target='_blank'>Khan et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26359452' target='_blank'>Singleton et al., Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26015511' target='_blank'>Göke et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24771645' target='_blank'>Wynes et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24305876' target='_blank'>Carstens et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22781593' target='_blank'>Ren et al., Leukemia, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21367659' target='_blank'>Greulich et al., Trends Mol Med, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19458078' target='_blank'>Tomlinson et al., Cancer Res, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40223139' target='_blank'>Liu et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40014401' target='_blank'>Chaturantabut et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38751556' target='_blank'>Nishida et al., 2024, Liver Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38346946' target='_blank'>Spahn et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35798968' target='_blank'>Sahoo et al., Mod Pathol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35420673' target='_blank'>Wu et al., Cancer Discov, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33709535' target='_blank'>Kendre et al., Hepatology, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33268819' target='_blank'>Krook et al., Br J Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29784854' target='_blank'>Saito et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29038531' target='_blank'>Berrout et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26359452' target='_blank'>Singleton et al., Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26294741' target='_blank'>Kwak et al., Mol Cancer Ther, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26170901' target='_blank'>MacNeil et al., 2015, Genome Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25035393' target='_blank'>Tchaicha et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19501013' target='_blank'>Kono et al., Drug Resist Updat, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38638430' target='_blank'>Ren et al., 2024, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38609519' target='_blank'>Lee et al., 2024, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37768887' target='_blank'>Ouyang et al., 2023, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37479885' target='_blank'>Wang et al., 2023, NPJ Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37377403' target='_blank'>Facchinetti et al., 2023, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35420673' target='_blank'>Wu et al., Cancer Discov, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34046693' target='_blank'>Lötsch et al., Acta Neuropathol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31434712' target='_blank'>Jimenez-Pascual et al., Cancer Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29784854' target='_blank'>Saito et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28746860' target='_blank'>Rogawski et al., Transl Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27998968' target='_blank'>Chandrani et al., Ann Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24784839' target='_blank'>Acquaviva et al., Mol Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23744832' target='_blank'>Herrera-Abreu et al., Cancer Discov, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23019225' target='_blank'>Du et al., Cancer Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22045636' target='_blank'>Foldynova-Trantirkova et al., Hum Mutat, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38649537' target='_blank'>Yan et al., 2024, EMBO J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38471061' target='_blank'>Levis et al., 2024, J Clin Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37191437' target='_blank'>Kan et al., 2023, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36206490' target='_blank'>Chen et al., Blood, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33436370' target='_blank'>Bjelosevic et al., Cancer Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33335095' target='_blank'>Carter et al., Signal Transduct Target Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33076970' target='_blank'>Brinton et al., J Hematol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31286998' target='_blank'>Solovey et al., 2019, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30146162' target='_blank'>Minzel et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28967922' target='_blank'>Weisberg et al., Nat Chem Biol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25477533' target='_blank'>Liyasova et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24584101' target='_blank'>Ciceri et al., Nat Chem Biol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22829080' target='_blank'>Hart et al., Blood Cancer J, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22323452' target='_blank'>Greenblatt et al., Blood, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19162530' target='_blank'>Chu et al., Drug Resist Updat, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40032852' target='_blank'>Yang et al., 2025, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39763649' target='_blank'>Hai et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39113611' target='_blank'>German et al., 2024, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33657416' target='_blank'>Grossmann et al., Cell Stem Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33417085' target='_blank'>Kumar et al., Breast Cancer Res Treat, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32946061' target='_blank'>Teng et al., Protein Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31709755' target='_blank'>Heo et al., EMBO Mol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31635067' target='_blank'>Xia et al., Cells, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31395975' target='_blank'>Stone et al., Nat Rev Urol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31324883' target='_blank'>Gao et al., Cell Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30272002' target='_blank'>Annala et al., Commun Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28658208' target='_blank'>Rheinbay et al., Nature, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28515055' target='_blank'>Huang et al., Cancer Discov, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25652398' target='_blank'>Orr et al., Hum Mol Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22096659' target='_blank'>Prensner et al., Cancer Discov, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40580249' target='_blank'>Liu et al., 2025, Med Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31553912' target='_blank'>Elman et al., Cell Rep, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39941924' target='_blank'>Liu et al., 2025, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39868536' target='_blank'>Wang et al., 2025, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33208147' target='_blank'>Nachmanson et al., BMC Med Genomics, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33024807' target='_blank'>Lin et al., Gynecol Oncol Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32729618' target='_blank'>Zhang et al., 2020, Biosci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31910858' target='_blank'>Beesley et al., Genome Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31810000' target='_blank'>Du et al., iScience, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31780652' target='_blank'>Hausser et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31519704' target='_blank'>Belver et al., 2019, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30093726' target='_blank'>Liu et al., Sci Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28108658' target='_blank'>Sun et al., Nucleic Acids Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27588951' target='_blank'>Mair et al., PLoS Genet, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27267060' target='_blank'>Jiang et al., Cell Signal, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25479686' target='_blank'>Izzo et al., Breast Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20189993' target='_blank'>Yan et al., J Biol Chem, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38233483' target='_blank'>Elbatsh et al., 2024, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37966164' target='_blank'>Proteau et al., 2023, EMBO Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37858338' target='_blank'>Arang et al., 2023, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37830586' target='_blank'>Kichina et al., 2023, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33824477' target='_blank'>Dhomen et al., Br J Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32933997' target='_blank'>Truong et al., Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31186280' target='_blank'>Lehmann et al., Mol Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29991680' target='_blank'>Birkeland et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29490280' target='_blank'>Moore et al., Cell Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27864876' target='_blank'>Turner et al., Pigment Cell Melanoma Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27089234' target='_blank'>Johnson et al., Pigment Cell Melanoma Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26712692' target='_blank'>Rothermel et al., Clin Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24705312' target='_blank'>Yeh et al., Am J Surg Pathol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24345920' target='_blank'>Hutchinson et al., Clin Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23303902' target='_blank'>Pedersen et al., Cancer Discov, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38233483' target='_blank'>Elbatsh et al., 2024, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35512098' target='_blank'>Phelps et al., Proc Natl Acad Sci U S A, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33824477' target='_blank'>Dhomen et al., Br J Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32209601' target='_blank'>Klemen et al., J Immunother Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31831627' target='_blank'>Koo et al., Genes Dev, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31186280' target='_blank'>Lehmann et al., Mol Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29570931' target='_blank'>Perez et al., Pigment Cell Melanoma Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29490280' target='_blank'>Moore et al., Cell Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27348266' target='_blank'>Patel et al., Oncogenesis, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27089234' target='_blank'>Johnson et al., Pigment Cell Melanoma Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26712692' target='_blank'>Rothermel et al., Clin Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24705312' target='_blank'>Yeh et al., Am J Surg Pathol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24423917' target='_blank'>Dono et al., Br J Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24345920' target='_blank'>Hutchinson et al., Clin Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23303902' target='_blank'>Pedersen et al., Cancer Discov, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40050874' target='_blank'>Raut et al., 2025, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40018643' target='_blank'>Ayala et al., 2025, Front Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37470859' target='_blank'>Takahashi et al., 2023, J Gastroenterol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36859393' target='_blank'>Sawai et al., 2023, J Med Case Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36535512' target='_blank'>Raut et al., 2022, Biochim Biophys Acta Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36151521' target='_blank'>Yu et al., BMC Genomics, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33915081' target='_blank'>Huang et al., Cell Stem Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33342770' target='_blank'>Li et al., J Biomed Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31807183' target='_blank'>Ma et al., Oncol Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31624069' target='_blank'>Tokita et al., Cold Spring Harb Mol Case Stud, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23752180' target='_blank'>Garcia-Murillas et al., Oncogene, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37929934' target='_blank'>Chi et al., 2023, Elife<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31653819' target='_blank'>Fomchenko et al., J Neurosurg Pediatr, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25732611' target='_blank'>Makova et al., Nat Rev Genet, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40615399' target='_blank'>Varela-Vázquez et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38763973' target='_blank'>Zhou et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36646686' target='_blank'>Li et al., Signal Transduct Target Ther, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36045302' target='_blank'>Schmitt et al., Nat Rev Clin Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34903602' target='_blank'>Xia et al., Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32341551' target='_blank'>Castel et al., Nat Rev Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32284825' target='_blank'>Dhondt et al., J Extracell Vesicles, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29760048' target='_blank'>Untch et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29321164' target='_blank'>Fujimura et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28666118' target='_blank'>Simanshu et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26818109' target='_blank'>Jolly et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26282165' target='_blank'>Lau et al., Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26183406' target='_blank'>McNeal et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25843002' target='_blank'>von et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24740997' target='_blank'>Elhammali et al., Cancer Discov, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38662454' target='_blank'>Song et al., 2024, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35985343' target='_blank'>Shi et al., Cancer Cell, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33024108' target='_blank'>Kim et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31780652' target='_blank'>Hausser et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31160393' target='_blank'>Gimple et al., Genes Dev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30635340' target='_blank'>Iorgulescu et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30482943' target='_blank'>Lawson et al., Nat Cell Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30181385' target='_blank'>Schwartz et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29805076' target='_blank'>Waitkus et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29343685' target='_blank'>Zhu et al., 2018, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27693047' target='_blank'>Bardella et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27397612' target='_blank'>Löffler et al., J Hepatol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26771115' target='_blank'>Pavlova et al., Cell Metab, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26436839' target='_blank'>Okoye-Okafor et al., Nat Chem Biol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24065765' target='_blank'>Chen et al., Genes Dev, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39655332' target='_blank'>Fang et al., 2024, PeerJ<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38353122' target='_blank'>Furnari et al., 2024, Dis Model Mech<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37847741' target='_blank'>Nelde et al., 2023, Blood Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33807183' target='_blank'>DeSouza et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33500247' target='_blank'>Barnabas et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33409153' target='_blank'>Barbato et al., Front Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33054125' target='_blank'>Madan et al., Haematologica, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32938585' target='_blank'>Li et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32333643' target='_blank'>Rodriguez et al., Cancer Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30297697' target='_blank'>Beier et al., 2018, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29805076' target='_blank'>Waitkus et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27913435' target='_blank'>Chiang et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24714777' target='_blank'>Wakimoto et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24065765' target='_blank'>Chen et al., Genes Dev, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23532369' target='_blank'>Cohen et al., Curr Neurol Neurosci Rep, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38600086' target='_blank'>Dmello et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38381540' target='_blank'>Petroni et al., 2024, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33986188' target='_blank'>Filppu et al., JCI Insight, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26177470' target='_blank'>Kaliyaperumal et al., Redox Biol, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40408005' target='_blank'>Annadurai et al., 2025, Discov Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38328861' target='_blank'>Barata et al., 2024, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35581375' target='_blank'>Oliveira et al., Leukemia, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34193976' target='_blank'>Thomas et al., Leukemia, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24678068' target='_blank'>Rozovski et al., 2014, Cancer Med<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40050874' target='_blank'>Raut et al., 2025, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33342770' target='_blank'>Li et al., J Biomed Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31947841' target='_blank'>Wang et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27602765' target='_blank'>Spinella et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27029346' target='_blank'>Leroy et al., Biochem J, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40205062' target='_blank'>Kamizela et al., 2025, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37992313' target='_blank'>Codilupi et al., 2023, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37926720' target='_blank'>Vieri et al., 2023, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36928379' target='_blank'>Bennett et al., 2023, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36100613' target='_blank'>Rai et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35271702' target='_blank'>Gotlib et al., Blood, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35082276' target='_blank'>Pandey et al., Blood Cancer J, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33046688' target='_blank'>Wang et al., 2020, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31750881' target='_blank'>Yue et al., Blood, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31676777' target='_blank'>Caiado et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31511238' target='_blank'>Rao et al., Blood, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28464892' target='_blank'>Li et al., J Hematol Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25849990' target='_blank'>Tapper et al., Nat Commun, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24747640' target='_blank'>Lane et al., Nat Genet, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18394555' target='_blank'>Geron et al., Cancer Cell, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40505232' target='_blank'>Zhang et al., 2025, Int Immunopharmacol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37393343' target='_blank'>Hou et al., 2023, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36982575' target='_blank'>Tseng et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36376832' target='_blank'>Tang et al., Cancer Cell Int, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35394814' target='_blank'>Nixon et al., Sci Immunol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34465864' target='_blank'>Yuan et al., Leukemia, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32625422' target='_blank'>EFSA et al., EFSA J, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30401535' target='_blank'>Mazhar et al., Biochim Biophys Acta Mol Cell Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29046866' target='_blank'>Martinez et al., Clin Cancer Drugs, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27602765' target='_blank'>Spinella et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26412458' target='_blank'>Hauer et al., Aging (Albany NY), 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25193870' target='_blank'>Degryse et al., 2014, Blood<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35454801' target='_blank'>Lemster et al., Cancers (Basel), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34336858' target='_blank'>Rodríguez-Hernández et al., Front Cell Dev Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25790038' target='_blank'>Kovac et al., Nat Commun, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37934116' target='_blank'>Saggese et al., 2023, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36306422' target='_blank'>Yang et al., Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36055401' target='_blank'>Shea et al., 2022, J Invest Dermatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34661759' target='_blank'>Zhou et al., J Cancer Res Clin Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34465286' target='_blank'>Chen et al., 2021, BMC Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31044091' target='_blank'>Wu et al., Signal Transduct Target Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29068423' target='_blank'>Shukla et al., Proteomes, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27602765' target='_blank'>Spinella et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27294188' target='_blank'>Ran et al., ScientificWorldJournal, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27208808' target='_blank'>Andricovich et al., Exp Hematol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25320243' target='_blank'>Van et al., Blood, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25225064' target='_blank'>Nickerson et al., Clin Cancer Res, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40267200' target='_blank'>Brumage et al., 2025, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39034402' target='_blank'>Lee et al., 2024, Genome Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38601319' target='_blank'>Wei et al., 2024, Oncoimmunology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38390340' target='_blank'>Tang et al., 2024, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38184997' target='_blank'>Deen et al., 2024, Redox Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37889752' target='_blank'>Zavitsanou et al., 2023, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36689560' target='_blank'>Osman et al., 2023, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36602799' target='_blank'>Lebow et al., JAMA Netw Open, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32866624' target='_blank'>Marinelli et al., 2020, Ann Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32576270' target='_blank'>Gong et al., Cell Commun Signal, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32234331' target='_blank'>Kopacz et al., Free Radic Biol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31398338' target='_blank'>Sanghvi et al., Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28967920' target='_blank'>Romero et al., Nat Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28617433' target='_blank'>Wang et al., Cell Death Dis, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27663899' target='_blank'>Jeong et al., Cancer Discov, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40447230' target='_blank'>Blay et al., 2025, Ann Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34873347' target='_blank'>DeAngelo et al., Nat Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34873345' target='_blank'>Gotlib et al., Nat Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34343033' target='_blank'>Kang et al., 2021, J Clin Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34117457' target='_blank'>Patton et al., Nat Rev Drug Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33632214' target='_blank'>Kosiol et al., Mol Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32251397' target='_blank'>Fazio et al., Nat Rev Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30385465' target='_blank'>Ablain et al., Science, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29132397' target='_blank'>Kanojia et al., J Hematol Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28426120' target='_blank'>Wagner et al., Ann Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25937435' target='_blank'>Marquardt et al., J Hepatol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22614970' target='_blank'>George et al., J Clin Oncol, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20818844' target='_blank'>Flaherty et al., N Engl J Med, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20385359' target='_blank'>Liu et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18591451' target='_blank'>Chen et al., Circulation, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/31511663' target='_blank'>Bushweller et al., Nat Rev Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30745579' target='_blank'>Pinho et al., Nat Rev Mol Cell Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30146162' target='_blank'>Minzel et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28399410' target='_blank'>Mohr et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28065413' target='_blank'>Liang et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27908736' target='_blank'>Bajaj et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27889185' target='_blank'>Xu et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27641501' target='_blank'>Sykes et al., Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27344946' target='_blank'>Stavropoulou et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27070701' target='_blank'>Brien et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26766589' target='_blank'>Cheung et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25873173' target='_blank'>Shih et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25464900' target='_blank'>Deshpande et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25242043' target='_blank'>Schneider et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21741597' target='_blank'>Bernt et al., Cancer Cell, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39841459' target='_blank'>Alikarami et al., 2025, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38806682' target='_blank'>Mills et al., 2024, NPJ Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35449309' target='_blank'>Na et al., 2022, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33355042' target='_blank'>Zheng et al., Technol Cancer Res Treat, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28453651' target='_blank'>Mateo et al., Nucleic Acids Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27556701' target='_blank'>Ueda et al., Oncotarget, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40185723' target='_blank'>Ogden et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39117659' target='_blank'>Liu et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37697729' target='_blank'>Song et al., 2023, Cancer Res Treat<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37142882' target='_blank'>Xu et al., 2023, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36640300' target='_blank'>Lee et al., 2023, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36525973' target='_blank'>Pan et al., Cancer Cell, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32243837' target='_blank'>Alam et al., Cancer Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30588243' target='_blank'>Robbins et al., J Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27729836' target='_blank'>Choi et al., Genomics Inform, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27729834' target='_blank'>Chung et al., Genomics Inform, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32649885' target='_blank'>Hayes et al., Cancer Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32531271' target='_blank'>Coles et al., Cancer Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30846598' target='_blank'>Hoxhaj et al., Science, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30661928' target='_blank'>Ericksen et al., Cell Metab, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30305738' target='_blank'>Song et al., Nature, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26073941' target='_blank'>Cubillos-Ruiz et al., Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24726384' target='_blank'>Xie et al., Cell Metab, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24525237' target='_blank'>Grabocka et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24505617' target='_blank'>Radu et al., Nat Rev Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23332765' target='_blank'>Burd et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22698401' target='_blank'>Liu et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22189424' target='_blank'>Ahearn et al., Nat Rev Mol Cell Biol, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21514245' target='_blank'>Blasco et al., Cancer Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19477428' target='_blank'>Singh et al., Cancer Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19029981' target='_blank'>Engelman et al., Nat Med, 2008<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37890861' target='_blank'>Ivanisevic et al., 2023, Cancer Commun (Lond)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35904492' target='_blank'>Chen et al., Cancer Discov, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30664951' target='_blank'>Jacquinet et al., Eur J Med Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28622513' target='_blank'>Cancer et al., Cell, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40171226' target='_blank'>Angeli et al., 2025, Front Cell Dev Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38622747' target='_blank'>Fan et al., 2024, Genome Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32483240' target='_blank'>Williams et al., Mod Pathol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32296058' target='_blank'>Mitra et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26582713' target='_blank'>Gannon et al., Mol Cancer Res, 2016<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40593119' target='_blank'>Cha et al., 2025, Commun Med (Lond)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30962452' target='_blank'>Johansson et al., Nat Commun, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39825392' target='_blank'>Wang et al., 2025, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39638567' target='_blank'>Cox et al., 2024, Genes Dev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38843329' target='_blank'>Klomp et al., 2024, Science<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38012363' target='_blank'>Maltas et al., 2023, Nat Ecol Evol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35614034' target='_blank'>Zheng et al., Cell Death Dis, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32699415' target='_blank'>Chan et al., Nature, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32160544' target='_blank'>Day et al., Cell Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31804622' target='_blank'>Rajan et al., Oncogene, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30833752' target='_blank'>Bryant et al., Nat Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30737375' target='_blank'>Bunda et al., 2019, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28951457' target='_blank'>Yaeger et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28130226' target='_blank'>Du et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24844911' target='_blank'>Savino et al., Cancer Immunol Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24568222' target='_blank'>Luan et al., Mol Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19584278' target='_blank'>Yue et al., Cancer Res, 2009<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40664642' target='_blank'>Singh et al., 2025, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38748792' target='_blank'>Pastoors et al., 2024, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37810550' target='_blank'>Lux et al., 2023, Hemasphere<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37735473' target='_blank'>Kugler et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36095844' target='_blank'>Schmoellerl et al., Blood, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34363719' target='_blank'>Mizuno et al., Cancer Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33903593' target='_blank'>Bleu et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33753715' target='_blank'>Birdwell et al., Blood Cancer J, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33670178' target='_blank'>Pasquer et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33082307' target='_blank'>Paredes et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32806592' target='_blank'>Liu et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31676734' target='_blank'>Rashkovan et al., Genes Dev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28191887' target='_blank'>Fenouille et al., Nat Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26553406' target='_blank'>Babu et al., Nucleus, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24735919' target='_blank'>Koche et al., Cancer Cell, 2014<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40589555' target='_blank'>Guo et al., 2025, Mol Clin Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36922593' target='_blank'>Issa et al., 2023, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34439335' target='_blank'>Sharma et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33542482' target='_blank'>Libbrecht et al., Leukemia, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33205067' target='_blank'>Smith et al., Cell Rep Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28485270' target='_blank'>Ehrlich et al., Gene Expr, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28273452' target='_blank'>Dreijerink et al., Cell Rep, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39215042' target='_blank'>Zhao et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37500626' target='_blank'>Zhou et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36256480' target='_blank'>Dai et al., J Clin Invest, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35921500' target='_blank'>Liang et al., Hepatology, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33526511' target='_blank'>Huang et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31383647' target='_blank'>Xu et al., Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30954949' target='_blank'>Che et al., Gut, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30737831' target='_blank'>Qiao et al., Hepatology, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29303510' target='_blank'>Xu et al., Exp Mol Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29303506' target='_blank'>Nam et al., Exp Mol Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26771115' target='_blank'>Pavlova et al., Cell Metab, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26022942' target='_blank'>Steuer et al., Mol Aspects Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24085787' target='_blank'>Avan et al., Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23079662' target='_blank'>Hatley et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19676131' target='_blank'>Liu et al., Hepatology, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40069402' target='_blank'>Middleton et al., 2025, NPJ Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39455853' target='_blank'>Shao et al., 2024, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38297372' target='_blank'>Lin et al., 2024, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38124228' target='_blank'>Xiong et al., 2023, Mil Med Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37804830' target='_blank'>Mossmann et al., 2023, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36451866' target='_blank'>Luo et al., Theranostics, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34429326' target='_blank'>Zhang et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32782783' target='_blank'>Li et al., Cell Biosci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32335998' target='_blank'>Zhou et al., Mol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32123314' target='_blank'>Wolf et al., Oncogene, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30796032' target='_blank'>Zhang et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30532012' target='_blank'>Altorki et al., Nat Rev Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29562234' target='_blank'>Hindupur et al., Nature, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29238073' target='_blank'>Darido et al., Cell Death Differ, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18701474' target='_blank'>Nakamura et al., Cancer Res, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38965396' target='_blank'>Sen et al., 2024, Nat Rev Clin Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34862480' target='_blank'>Stine et al., Nat Rev Drug Discov, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33214692' target='_blank'>Chen et al., Nat Rev Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30837715' target='_blank'>Sermer et al., Nat Rev Clin Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30181570' target='_blank'>Casero et al., Nat Rev Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28127048' target='_blank'>Otto et al., Nat Rev Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27492215' target='_blank'>Altman et al., Nat Rev Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27112207' target='_blank'>Truitt et al., Nat Rev Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24599217' target='_blank'>Marshall et al., Nat Rev Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23303139' target='_blank'>Wade et al., Nat Rev Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23175120' target='_blank'>Northcott et al., Nat Rev Cancer, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22863009' target='_blank'>Rounbehler et al., Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22424228' target='_blank'>Mendell et al., Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19524505' target='_blank'>Kota et al., Cell, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19269363' target='_blank'>Luo et al., Cell, 2009<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33450197' target='_blank'>Chang et al., Cancer Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31564637' target='_blank'>Burr et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31287989' target='_blank'>Balanis et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28127048' target='_blank'>Otto et al., Nat Rev Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27941793' target='_blank'>Lu et al., Nat Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27728805' target='_blank'>Dardenne et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27618649' target='_blank'>Tavana et al., Nat Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27050099' target='_blank'>Lee et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26766587' target='_blank'>Vo et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26766585' target='_blank'>Mathsyaraja et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25533335' target='_blank'>Hill et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25175806' target='_blank'>Gustafson et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22789543' target='_blank'>Berry et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22624711' target='_blank'>Swartling et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22340587' target='_blank'>Hogarty et al., Cancer Cell, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39571170' target='_blank'>Tolley et al., 2024, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39550582' target='_blank'>Lainšček et al., 2024, Cell Commun Signal<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38454120' target='_blank'>Lantermans et al., 2024, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36346827' target='_blank'>Flümann et al., Blood Cancer Discov, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35044826' target='_blank'>Rodriguez et al., Sci Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34799397' target='_blank'>Nouri et al., J Immunother Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34279092' target='_blank'>Fu et al., J Med Chem, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34132782' target='_blank'>Yang et al., Blood, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32848129' target='_blank'>Shuai et al., Blood Cancer J, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32325488' target='_blank'>Jo et al., Blood, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32005797' target='_blank'>Munshi et al., Blood Cancer J, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31935288' target='_blank'>Liu et al., Blood Adv, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31197259' target='_blank'>Giménez et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27057457' target='_blank'>Bald et al., Oncoimmunology, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24322981' target='_blank'>Morales et al., Cancer Res, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40578157' target='_blank'>NA et al., 2025, Comput Biol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29903892' target='_blank'>Fomchenko et al., Cold Spring Harb Mol Case Stud, 2018<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39137777' target='_blank'>Kloosterman et al., 2024, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35797217' target='_blank'>Li et al., 2022, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34416167' target='_blank'>Wang et al., Cancer Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33857425' target='_blank'>Gangoso et al., Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32727536' target='_blank'>Noorani et al., Genome Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32665005' target='_blank'>Unruh et al., J Hematol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32620753' target='_blank'>Oh et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32532995' target='_blank'>Pattwell et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31257073' target='_blank'>Lionarons et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29127119' target='_blank'>Hayashi et al., Clin Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28193624' target='_blank'>Ascierto et al., Clin Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27426982' target='_blank'>Plaisier et al., Cell Syst, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23685747' target='_blank'>Rahrmann et al., Nat Genet, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21737130' target='_blank'>Liu et al., Cell, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20627072' target='_blank'>Egeblad et al., Dev Cell, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39251601' target='_blank'>Eaton et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38695575' target='_blank'>Wang et al., 2024, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36416208' target='_blank'>Chow et al., 2022, J Med Chem<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35534562' target='_blank'>Choudhury et al., Nat Genet, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34359639' target='_blank'>Boetto et al., 2021, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34261524' target='_blank'>Meiller et al., Genome Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34209798' target='_blank'>Behling et al., 2021, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33074854' target='_blank'>Schieffer et al., Am J Surg Pathol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32751922' target='_blank'>Czarnecka et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32742192' target='_blank'>Kim et al., Cancer Cell Int, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31652973' target='_blank'>Lee et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31341276' target='_blank'>Wu et al., Nature, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29764516' target='_blank'>Du et al., Cancer Commun (Lond), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28126595' target='_blank'>Bassiri et al., EBioMedicine, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22749019' target='_blank'>Efeyan et al., Trends Mol Med, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40421600' target='_blank'>Friehmann et al., 2025, RNA Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40270498' target='_blank'>Nandi et al., 2025, Front Med (Lausanne)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38628662' target='_blank'>Liu et al., 2024, Glob Med Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36602799' target='_blank'>Lebow et al., JAMA Netw Open, 2023<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36725930' target='_blank'>Bartman et al., 2023, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34937946' target='_blank'>Thandapani et al., 2021, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31526760' target='_blank'>Jackstadt et al., 2019, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30038221' target='_blank'>Kourtis et al., Nat Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29136506' target='_blank'>Robles-Valero et al., 2017, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27941793' target='_blank'>Lu et al., Nat Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27070701' target='_blank'>Brien et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26390244' target='_blank'>Herranz et al., 2015, Nat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26058075' target='_blank'>Pitt et al., 2015, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24651013' target='_blank'>Ntziachristos et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24291004' target='_blank'>Piovan et al., 2013, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24172333' target='_blank'>Ling et al., Nat Rev Drug Discov, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23791182' target='_blank'>King et al., 2013, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22897852' target='_blank'>Maraver et al., 2012, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19907488' target='_blank'>Moellering et al., 2009, Nature<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40667321' target='_blank'>Glasmacher et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33750924' target='_blank'>Pappas et al., Commun Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33382997' target='_blank'>Gan et al., Exp Cell Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28177878' target='_blank'>Tang et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25985737' target='_blank'>Zhu et al., Nat Commun, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39883527' target='_blank'>Hamamoto et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39626264' target='_blank'>Franco et al., 2024, J Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39475509' target='_blank'>Heikamp et al., 2024, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37992275' target='_blank'>Gibson et al., 2023, J Am Chem Soc<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37847741' target='_blank'>Nelde et al., 2023, Blood Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37344104' target='_blank'>Ivanovic et al., 2023, Genome Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36922589' target='_blank'>Perner et al., 2023, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35180428' target='_blank'>Luo et al., Mol Cell, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33243988' target='_blank'>Spinler et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32573700' target='_blank'>Prokoph et al., Blood, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30510251' target='_blank'>Menotti et al., Nat Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30131584' target='_blank'>Prutsch et al., Leukemia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27698101' target='_blank'>Hourigan et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26997274' target='_blank'>Nichol et al., Cell Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23064464' target='_blank'>Laimer et al., Nat Med, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40108474' target='_blank'>Yuan et al., 2025, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37221195' target='_blank'>Yin et al., 2023, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35561684' target='_blank'>Choi et al., Cell, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34416167' target='_blank'>Wang et al., Cancer Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32528145' target='_blank'>Moore et al., Nat Rev Drug Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31257073' target='_blank'>Lionarons et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31061481' target='_blank'>Newman et al., Nat Biotechnol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29551267' target='_blank'>Xiao et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29320480' target='_blank'>Sulli et al., 2018, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29275866' target='_blank'>Kunimoto et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29245011' target='_blank'>Hosono et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26974310' target='_blank'>Shojaee et al., Nat Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26343581' target='_blank'>Zelenay et al., Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24022702' target='_blank'>Watson et al., Nat Rev Genet, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22817889' target='_blank'>Hodis et al., Cell, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38062230' target='_blank'>Li et al., 2023, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37016431' target='_blank'>Narang et al., 2023, Genome Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33355042' target='_blank'>Zheng et al., Technol Cancer Res Treat, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31582846' target='_blank'>Husmann et al., Nat Struct Mol Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30518758' target='_blank'>Aytes et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27368234' target='_blank'>Bobby et al., Biomol NMR Assign, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40394211' target='_blank'>Auer et al., 2025, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39779852' target='_blank'>Fu et al., 2025, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39005360' target='_blank'>Fu et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36387144' target='_blank'>Jia et al., Front Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36287107' target='_blank'>Lai et al., 2022, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34336858' target='_blank'>Rodríguez-Hernández et al., Front Cell Dev Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33255367' target='_blank'>Ratti et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31170889' target='_blank'>Janke et al., Vet Pathol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30643249' target='_blank'>Gu et al., Nat Genet, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30520556' target='_blank'>Spina et al., Mol Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30216339' target='_blank'>Hart et al., 2018, PLoS Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29029550' target='_blank'>Zhang et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24939936' target='_blank'>Liu et al., Genes Dev, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40131247' target='_blank'>Elias et al., 2025, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39874221' target='_blank'>Shaikh et al., 2025, Epigenetics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39250899' target='_blank'>Kumar et al., 2024, Oncology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38228729' target='_blank'>Singla et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35932824' target='_blank'>Kapur et al., 2022, Hum Pathol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35654752' target='_blank'>Acosta et al., 2022, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32206160' target='_blank'>Elias et al., 2019, Kidney Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31807183' target='_blank'>Ma et al., Oncol Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30660076' target='_blank'>Bihr et al., Neoplasia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29651023' target='_blank'>Hou et al., Exp Mol Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28473526' target='_blank'>Gu et al., Cancer Discov, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28329682' target='_blank'>Nargund et al., Cell Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27383411' target='_blank'>Ji et al., Sci Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22249190' target='_blank'>Larkin et al., Nat Rev Urol, 2012<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40447230' target='_blank'>Blay et al., 2025, Ann Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38167404' target='_blank'>Teuber et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37011011' target='_blank'>McNicholas et al., 2023, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36988488' target='_blank'>Goenka et al., 2023, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36971786' target='_blank'>Rossi et al., 2023, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36720864' target='_blank'>Wang et al., Nat Commun, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34551970' target='_blank'>Carvalho et al., Cancer Discov, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34407970' target='_blank'>Singh et al., Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33277484' target='_blank'>Pajovic et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32972961' target='_blank'>Grunewald et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32049048' target='_blank'>Amin et al., Cancer Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31910904' target='_blank'>Yoshida et al., J Hematol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25865047' target='_blank'>Hayashi et al., Gastroenterology, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25373456' target='_blank'>Lourenço et al., Mol Cancer, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23041331' target='_blank'>Pelczar et al., Gastroenterology, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36982575' target='_blank'>Tseng et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34465864' target='_blank'>Yuan et al., Leukemia, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33251223' target='_blank'>Loontiens et al., Front Cell Dev Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32647798' target='_blank'>McEwan et al., 2020, Hemasphere<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30567843' target='_blank'>Wendorff et al., Cancer Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29780592' target='_blank'>Ng et al., Biomark Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28637487' target='_blank'>Philpott et al., Hum Genomics, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27602765' target='_blank'>Spinella et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22675565' target='_blank'>Kalender et al., PLoS One, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36115839' target='_blank'>Franz et al., Signal Transduct Target Ther, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35618834' target='_blank'>Kansler et al., Nat Immunol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33071215' target='_blank'>Binkley et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32059806' target='_blank'>Ying et al., Cell Stem Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32033971' target='_blank'>Unknown et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31996845' target='_blank'>Yu et al., Nature, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30975663' target='_blank'>Tateishi et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30823937' target='_blank'>Wang et al., Cancer Commun (Lond), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29581176' target='_blank'>Pearson et al., Cancer Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27827358' target='_blank'>Gellert et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27426982' target='_blank'>Plaisier et al., Cell Syst, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27060149' target='_blank'>Lai et al., Nucleic Acids Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26334752' target='_blank'>Eng et al., J Thorac Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25079324' target='_blank'>Wang et al., Nature, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24214964' target='_blank'>Cheng et al., Nucleic Acids Res, 2014<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38489912' target='_blank'>Rausio et al., 2024, Neoplasia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35441778' target='_blank'>Modlin et al., 2022, Am J Med Genet A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31867792' target='_blank'>Febres-Aldana et al., Pathol Int, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28453651' target='_blank'>Mateo et al., Nucleic Acids Res, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38097734' target='_blank'>Chauhan et al., 2023, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36907773' target='_blank'>Julson et al., 2023, J Pediatr Surg<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34484175' target='_blank'>Dawes et al., Front Immunol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32753387' target='_blank'>Lim et al., 2020, Mol Cancer Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32146726' target='_blank'>Singh et al., Mol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25238262' target='_blank'>Saurabh et al., Oncotarget, 2014<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37709843' target='_blank'>Burocziova et al., 2023, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37378944' target='_blank'>Biswas et al., 2023, Asian Pac J Cancer Prev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35105861' target='_blank'>Khadka et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29046514' target='_blank'>Jeong et al., J Pathol Transl Med, 2018<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/32474700' target='_blank'>Li et al., Protein Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28186096' target='_blank'>Lazova et al., Mod Pathol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26868000' target='_blank'>Ogoh et al., 2016, Mech Dev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25759478' target='_blank'>Wengrod et al., Sci Signal, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24336958' target='_blank'>Gold et al., 2013, Mol Cancer Res<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40301543' target='_blank'>Collinson et al., 2025, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39091814' target='_blank'>Keahi et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39078737' target='_blank'>Casey et al., 2024, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38736183' target='_blank'>Barateiro et al., 2024, Neuropathology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38370799' target='_blank'>Casey et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35397477' target='_blank'>Fowler et al., Cancer Discov, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31204176' target='_blank'>Huang et al., Cell Stem Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30487124' target='_blank'>Rodriguez-Blanco et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30319965' target='_blank'>Geyer et al., Front Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27811357' target='_blank'>Xavier et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24994715' target='_blank'>Li et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23079662' target='_blank'>Hatley et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21150899' target='_blank'>Gibson et al., 2010, Nature<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34862480' target='_blank'>Stine et al., Nat Rev Drug Discov, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33857425' target='_blank'>Gangoso et al., Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33568791' target='_blank'>Ghiraldini et al., Nat Rev Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32555376' target='_blank'>Dey et al., Nat Rev Drug Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32152485' target='_blank'>Quintanal-Villalonga et al., Nat Rev Clin Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31185211' target='_blank'>Chen et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29438700' target='_blank'>Yu et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29326431' target='_blank'>Spranger et al., Nat Rev Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28802037' target='_blank'>Fruman et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28283069' target='_blank'>Saxton et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28111070' target='_blank'>Spitzer et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25533673' target='_blank'>Thorpe et al., Nat Rev Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24630729' target='_blank'>McFadden et al., Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23706742' target='_blank'>Fritsch et al., Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23201680' target='_blank'>Qin et al., Nature, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39403923' target='_blank'>Bohlen et al., 2024, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38152053' target='_blank'>Wintering et al., 2023, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36566878' target='_blank'>Polubothu et al., 2022, J Invest Dermatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36528691' target='_blank'>Yan et al., Sci Rep, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36349709' target='_blank'>Eboreime et al., 2022, Hum Mutat<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35267643' target='_blank'>Fiñana et al., Cancers (Basel), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34300250' target='_blank'>Mayerhofer et al., J Clin Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33139265' target='_blank'>Schönung et al., Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31940719' target='_blank'>Na et al., Mol Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30355677' target='_blank'>Hill et al., Mol Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30262568' target='_blank'>Giménez et al., Haematologica, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28329685' target='_blank'>Zhang et al., Cell Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28306669' target='_blank'>Pandey et al., Curr Opin Hematol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27899802' target='_blank'>Vesely et al., Leukemia, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26617336' target='_blank'>Bunda et al., Nat Commun, 2015<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38712822' target='_blank'>Nikolaou et al., 2024, Elife<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38243078' target='_blank'>Cannon et al., 2024, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38109551' target='_blank'>Mishra et al., 2023, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36458417' target='_blank'>Unknown et al., Mol Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36054930' target='_blank'>Stejerean-Todoran et al., 2022, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33302361' target='_blank'>Boueid et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32466585' target='_blank'>Diazzi et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32460002' target='_blank'>Cannon et al., Trends Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31257073' target='_blank'>Lionarons et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31027363' target='_blank'>De et al., Cells, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30312528' target='_blank'>Calapre et al., Mol Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30261690' target='_blank'>Hudson et al., Cancers (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28807941' target='_blank'>Kazanietz et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28186096' target='_blank'>Lazova et al., Mod Pathol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23322732' target='_blank'>Guo et al., Mol Cancer Res, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39190349' target='_blank'>Su et al., 2024, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38895459' target='_blank'>Tharp et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34689165' target='_blank'>Shang et al., Cell Death Dis, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37996408' target='_blank'>Montenegro et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31618628' target='_blank'>Botton et al., Cell Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23242278' target='_blank'>Horbinski et al., J Neuropathol Exp Neurol, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40350470' target='_blank'>Shinozaki et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40240142' target='_blank'>Fraire et al., 2025, Genes Dev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37470668' target='_blank'>Davies et al., 2023, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36928314' target='_blank'>Wang et al., 2023, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36653597' target='_blank'>Koerner et al., 2023, Cell Death Differ<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36635503' target='_blank'>Sparbier et al., Nat Cell Biol, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36509828' target='_blank'>Kirk et al., Exp Mol Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35179959' target='_blank'>Miao et al., Sci Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34315877' target='_blank'>Norrie et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33199493' target='_blank'>DeLucia et al., Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32265224' target='_blank'>Zhao et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31174843' target='_blank'>Knudsen et al., Trends Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29848757' target='_blank'>Tsui et al., EMBO Mol Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22759861' target='_blank'>Dees et al., Genome Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21457491' target='_blank'>Schulz et al., Genome Med, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39026820' target='_blank'>Krishnamoorthy et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36853175' target='_blank'>Bao et al., 2023, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34367963' target='_blank'>Liu et al., Front Oncol, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40517166' target='_blank'>Huang et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37204852' target='_blank'>Gild et al., 2023, Endocr Rev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34002056' target='_blank'>Cohen et al., Nat Rev Drug Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33771190' target='_blank'>Drusbosky et al., J Hematol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33109256' target='_blank'>Huang et al., J Hematol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33071215' target='_blank'>Binkley et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32866654' target='_blank'>Tan et al., J Thorac Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32649874' target='_blank'>Gillette et al., Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31819229' target='_blank'>Tiedje et al., Nat Rev Endocrinol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29657135' target='_blank'>Subbiah et al., Cancer Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27626519' target='_blank'>Fagin et al., N Engl J Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25069464' target='_blank'>Raman et al., Nat Rev Endocrinol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24419419' target='_blank'>Heist et al., J Thorac Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23584291' target='_blank'>Nebhan et al., J Thorac Oncol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22327624' target='_blank'>Kohno et al., Nat Med, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40746849' target='_blank'>Ma et al., 2025, Biochem Biophys Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39966435' target='_blank'>El Masri et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39910105' target='_blank'>Alem et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36458417' target='_blank'>Unknown et al., Mol Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35833297' target='_blank'>Zhang et al., Haematologica, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32349449' target='_blank'>Witjes et al., Int J Mol Sci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31771969' target='_blank'>Zhang et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31485674' target='_blank'>Nishizawa et al., Oncol Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31062076' target='_blank'>Wirthschaft et al., Acta Neuropathol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30232004' target='_blank'>Abraham et al., 2018, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29769144' target='_blank'>Zuo et al., Breast Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27104658' target='_blank'>Porter et al., Small GTPases, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26864623' target='_blank'>Konen et al., Mol Biol Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23322732' target='_blank'>Guo et al., Mol Cancer Res, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40557758' target='_blank'>Leoni et al., 2025, J Pathol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40070825' target='_blank'>Wang et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37470859' target='_blank'>Takahashi et al., 2023, J Gastroenterol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36732592' target='_blank'>Ishino et al., 2023, Br J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33202731' target='_blank'>Kleeman et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32420140' target='_blank'>Zhang et al., Transl Androl Urol, 2020<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40479062' target='_blank'>Ismail et al., 2025, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39450904' target='_blank'>Chen et al., 2024, Protein Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38972954' target='_blank'>Mulet-Lazaro et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37992684' target='_blank'>Popescu et al., 2023, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37957175' target='_blank'>Xu et al., 2023, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37319434' target='_blank'>Fei et al., 2023, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35981544' target='_blank'>Groves et al., 2022, Cell Syst<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35190728' target='_blank'>Dinh et al., 2022, Nat Rev Gastroenterol Hepatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33512434' target='_blank'>Wen et al., Blood, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32344427' target='_blank'>Schmoellerl et al., Blood, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30300583' target='_blank'>Martinez-Soria et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26666262' target='_blank'>Hatlen et al., J Exp Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26594843' target='_blank'>Esposito et al., 2015, Nat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25873173' target='_blank'>Shih et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24855208' target='_blank'>Sasca et al., Blood, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39078402' target='_blank'>Maese et al., 2024, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29217778' target='_blank'>Pastor et al., Haematologica, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27863435' target='_blank'>Nguyen et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25299648' target='_blank'>Olde et al., PLoS One, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40131247' target='_blank'>Elias et al., 2025, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39874221' target='_blank'>Shaikh et al., 2025, Epigenetics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37528416' target='_blank'>Javaid et al., 2023, BMC Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35758895' target='_blank'>Rappold et al., 2022, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34157286' target='_blank'>Kearns et al., J Biol Chem, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33080522' target='_blank'>Wu et al., Transl Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33042282' target='_blank'>Qin et al., 2020, Theranostics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30943919' target='_blank'>Campbell et al., BMC Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30406665' target='_blank'>Liu et al., J Proteome Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29860315' target='_blank'>Bao et al., Nucleic Acids Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29724721' target='_blank'>Sakthikumar et al., 2018, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29724720' target='_blank'>Chiang et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25790038' target='_blank'>Kovac et al., Nat Commun, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24634093' target='_blank'>Wu et al., Cancer Med, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24316982' target='_blank'>Bard-Chapeau et al., Nat Genet, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40070829' target='_blank'>Liang et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38017105' target='_blank'>Playa-Albinyana et al., 2023, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37823551' target='_blank'>Simmler et al., 2023, Elife<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37582363' target='_blank'>Cortés-López et al., 2023, Cell Stem Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35927489' target='_blank'>Knisbacher et al., 2022, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35672297' target='_blank'>Royo et al., 2022, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33230298' target='_blank'>Conway et al., Nat Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31092926' target='_blank'>Gaiti et al., 2019, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30664638' target='_blank'>Wong et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30038380' target='_blank'>Leeksma et al., Leukemia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29875770' target='_blank'>Díaz-Muñoz et al., Front Immunol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29670635' target='_blank'>Darwiche et al., Front Immunol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28679620' target='_blank'>Wang et al., Genome Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28622513' target='_blank'>Cancer et al., Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27890934' target='_blank'>Young et al., Leukemia, 2017<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39937911' target='_blank'>Pal et al., 2025, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39841099' target='_blank'>Lloyd et al., 2025, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39448564' target='_blank'>Ghosh et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35570408' target='_blank'>Du et al., Adv Sci (Weinh), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35121522' target='_blank'>Kruger et al., ESMO Open, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32933947' target='_blank'>Grant et al., Gut, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32385075' target='_blank'>Zhao et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32376602' target='_blank'>Huang et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32154941' target='_blank'>Kalisz et al., EMBO J, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30686769' target='_blank'>Connor et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29653981' target='_blank'>You et al., Cancer Immunol Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26701088' target='_blank'>Wang et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26676757' target='_blank'>Murphy et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24717934' target='_blank'>Morran et al., Gut, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21289624' target='_blank'>Ding et al., Nature, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40451925' target='_blank'>Kim et al., 2025, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40407709' target='_blank'>Raupach et al., 2025, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40009666' target='_blank'>Kim et al., 2025, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39630912' target='_blank'>Brodeur et al., 2024, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39080761' target='_blank'>Redin et al., 2024, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38601319' target='_blank'>Wei et al., 2024, Oncoimmunology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37345679' target='_blank'>Mazibrada et al., 2023, Am J Surg Pathol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36922568' target='_blank'>Zeng et al., 2023, NPJ Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36265117' target='_blank'>Navitski et al., JCO Precis Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35618573' target='_blank'>Torborg et al., 2022, Trends Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34422119' target='_blank'>Leary et al., Ther Adv Med Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31137686' target='_blank'>Yang et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30718512' target='_blank'>Xue et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30646939' target='_blank'>Moufarrij et al., Clin Epigenetics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25886974' target='_blank'>Moes-Sosnowska et al., Orphanet J Rare Dis, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39971779' target='_blank'>Trejo-Villegas et al., 2025, Cancer Gene Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39960392' target='_blank'>Alimova et al., 2025, Mol Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39237495' target='_blank'>Krämer et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36088476' target='_blank'>Dermawan et al., 2022, Mod Pathol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35650187' target='_blank'>Moe et al., Oncogenesis, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33469207' target='_blank'>Bhat et al., Nat Rev Drug Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33332735' target='_blank'>Chasse et al., EMBO Mol Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32725435' target='_blank'>Lechner et al., Curr Oncol Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32558936' target='_blank'>Xue et al., J Pathol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31806885' target='_blank'>Sampson et al., Nat Rev Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31504799' target='_blank'>Meel et al., Neuro Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30898143' target='_blank'>Alfert et al., Epigenetics Chromatin, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30397315' target='_blank'>Michel et al., Nat Cell Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29879107' target='_blank'>Nebbioso et al., PLoS Genet, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21076395' target='_blank'>Jagani et al., Nat Med, 2010<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40594780' target='_blank'>Huang et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40145543' target='_blank'>Wu et al., 2025, Cancer Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38018030' target='_blank'>Yu et al., 2023, Cancer Biol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37258742' target='_blank'>Zhong et al., 2023, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36696357' target='_blank'>Anstine et al., 2023, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36002248' target='_blank'>Fan et al., Gut, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34624089' target='_blank'>Shen et al., Blood, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33762742' target='_blank'>Backx et al., Cell Death Differ, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33632299' target='_blank'>Tellez et al., Clin Epigenetics, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33117804' target='_blank'>Li et al., Front Cell Dev Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29484136' target='_blank'>Francis et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29348895' target='_blank'>Ruan et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28507152' target='_blank'>Jeselsohn et al., Proc Natl Acad Sci U S A, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27625374' target='_blank'>Suryo et al., EMBO J, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27043282' target='_blank'>Ma et al., 2016, J Clin Invest<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35438056' target='_blank'>Gao et al., Autophagy, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34610962' target='_blank'>Burleson et al., Mol Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34353330' target='_blank'>Jiang et al., Mol Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33568750' target='_blank'>Armstrong et al., Prostate Cancer Prostatic Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32460168' target='_blank'>Wang et al., EBioMedicine, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32297747' target='_blank'>Dong et al., J Med Chem, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29996942' target='_blank'>Ma et al., J Exp Clin Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29229583' target='_blank'>Gillard et al., Eur Urol Focus, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29202479' target='_blank'>Shoag et al., J Clin Invest, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29105795' target='_blank'>Davidsson et al., Prostate, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28515055' target='_blank'>Huang et al., Cancer Discov, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28448495' target='_blank'>Jin et al., PLoS Genet, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28292441' target='_blank'>Blattner et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27238081' target='_blank'>Groner et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23759327' target='_blank'>Barbieri et al., Eur Urol, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35690645' target='_blank'>Tashakori et al., Mod Pathol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29875770' target='_blank'>Díaz-Muñoz et al., Front Immunol, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/24822266' target='_blank'>Gomes et al., 2014, J Oral Pathol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24777035' target='_blank'>Cazier et al., Nat Commun, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36871040' target='_blank'>Bai et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36462630' target='_blank'>Li et al., 2022, Genomics Proteomics Bioinformatics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33323404' target='_blank'>Sitthideatphaiboon et al., Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31350328' target='_blank'>Hollstein et al., Cancer Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31089135' target='_blank'>Liu et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30692209' target='_blank'>Bhatt et al., Genes Dev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30297358' target='_blank'>Kitajima et al., Cancer Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28598439' target='_blank'>Zhang et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28387316' target='_blank'>Zhang et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28099846' target='_blank'>Nagaraj et al., Cell Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26413869' target='_blank'>Peña et al., J Clin Invest, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25517383' target='_blank'>Weinberg et al., Nat Chem Biol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24322983' target='_blank'>Cheng et al., Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20541700' target='_blank'>Carretero et al., Cancer Cell, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18245476' target='_blank'>Contreras et al., Cancer Res, 2008<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39403898' target='_blank'>Bleloch et al., 2024, Cancer Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39264104' target='_blank'>Ncube et al., 2024, Mol Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37902422' target='_blank'>Tang et al., 2023, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31669645' target='_blank'>Khan et al., Gene, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30373182' target='_blank'>Gallo et al., High Throughput, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29344954' target='_blank'>McCart et al., J Pathol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26579496' target='_blank'>Fischer et al., Front Oncol, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40111422' target='_blank'>Zhang et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39358506' target='_blank'>Zou et al., 2024, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38019014' target='_blank'>Yu et al., 2023, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37816954' target='_blank'>Xie et al., 2023, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36073548' target='_blank'>Ramdas et al., JCI Insight, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34937926' target='_blank'>Shukla et al., Nat Immunol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32938585' target='_blank'>Li et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32591645' target='_blank'>Jensen et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32579941' target='_blank'>Cai et al., Cell Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32503976' target='_blank'>Zhao et al., Leukemia, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32160545' target='_blank'>Viaud et al., Cell Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31578317' target='_blank'>Das et al., Blood Cancer J, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28566492' target='_blank'>Durham et al., Blood, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26949746' target='_blank'>Lipinski et al., Trends Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25263596' target='_blank'>Hon et al., Mol Cell, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40277917' target='_blank'>Johnson et al., 2025, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39829747' target='_blank'>Johnson et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36908331' target='_blank'>Barrett et al., 2018, J Immunol Regen Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34408796' target='_blank'>Li et al., Ther Adv Med Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31807183' target='_blank'>Ma et al., Oncol Lett, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30004628' target='_blank'>Breunig et al., Mol Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29282223' target='_blank'>Sakai et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28376875' target='_blank'>Fricke et al., Cell Commun Signal, 2017<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34244683' target='_blank'>Kerk et al., 2021, Nat Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32398706' target='_blank'>Ho et al., Nat Rev Clin Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32234521' target='_blank'>Ruscetti et al., Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31983610' target='_blank'>Cheung et al., Cancer Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31821786' target='_blank'>Yin et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30975481' target='_blank'>Blasco et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30686769' target='_blank'>Connor et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30595505' target='_blank'>Larson et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30580964' target='_blank'>Moon et al., Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29533787' target='_blank'>Andricovich et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25533335' target='_blank'>Hill et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24725405' target='_blank'>Weissmueller et al., Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23344544' target='_blank'>Su et al., Nat Rev Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22169972' target='_blank'>Keith et al., Nat Rev Cancer, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21737130' target='_blank'>Liu et al., Cell, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36451866' target='_blank'>Luo et al., Theranostics, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32738450' target='_blank'>Luo et al., J Hepatol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32206160' target='_blank'>Elias et al., 2019, Kidney Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27211490' target='_blank'>Umemura et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25505789' target='_blank'>Delaney et al., Front Cell Dev Biol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21075312' target='_blank'>Gan et al., Cancer Cell, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40706452' target='_blank'>Anwaier et al., 2025, Neoplasia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28512249' target='_blank'>Lam et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27753446' target='_blank'>Lam et al., J Pathol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25505789' target='_blank'>Delaney et al., Front Cell Dev Biol, 2014<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40043000' target='_blank'>Pal et al., 2025, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38564827' target='_blank'>Zhang et al., 2024, EBioMedicine<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37914932' target='_blank'>Terekhanova et al., 2023, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37069149' target='_blank'>Hu et al., 2023, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36973268' target='_blank'>Wu et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36563681' target='_blank'>Li et al., 2022, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36539415' target='_blank'>Sciacovelli et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35115713' target='_blank'>Xie et al., 2022, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33758423' target='_blank'>Xie et al., 2021, Nat Metab<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30886153' target='_blank'>Huang et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28329682' target='_blank'>Nargund et al., Cell Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28099114' target='_blank'>Reznik et al., Elife, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24607405' target='_blank'>Oskarsson et al., Cell Stem Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24487277' target='_blank'>Gerlinger et al., Nat Genet, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19401348' target='_blank'>Luu et al., 2009, Clin Cancer Res<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40166161' target='_blank'>Wang et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39412947' target='_blank'>Kawai-Kawachi et al., 2024, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39135775' target='_blank'>Brown et al., 2024, Front Cell Dev Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36506091' target='_blank'>Magrath et al., Front Cell Dev Biol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35443736' target='_blank'>Hartono et al., Oncogenesis, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33718380' target='_blank'>Custers et al., Front Cell Dev Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33525546' target='_blank'>Mello et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32856872' target='_blank'>Abd et al., Asian Pac J Cancer Prev, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32703985' target='_blank'>Hingorani et al., Sci Rep, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39246801' target='_blank'>Don et al., 2024, Case Rep Hematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36763002' target='_blank'>Vujovic et al., 2023, Blood Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35672297' target='_blank'>Royo et al., 2022, Blood Cancer J<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40134607' target='_blank'>Liu et al., 2025, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30952632' target='_blank'>Li et al., Cancer Res, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/25464900' target='_blank'>Deshpande et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25264566' target='_blank'>Li et al., Exp Hematol, 2014<\/a>",null,null,null,null,null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39281922' target='_blank'>Natarajan et al., 2024, ACS Omega<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37024492' target='_blank'>Sharifnia et al., 2023, Nat Commun<\/a>",null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40231473' target='_blank'>Kang et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39894894' target='_blank'>Coughlin et al., 2025, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27135977' target='_blank'>Juilland et al., Curr Opin Hematol, 2016<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40247105' target='_blank'>Buckley et al., 2025, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36947815' target='_blank'>Botten et al., 2023, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34440292' target='_blank'>Bardelli et al., Genes (Basel), 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39702508' target='_blank'>Huang et al., 2024, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35653645' target='_blank'>Huckvale et al., J Med Chem, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34050029' target='_blank'>Fangazio et al., Proc Natl Acad Sci U S A, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33362210' target='_blank'>Delgado et al., PLoS Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33278036' target='_blank'>Wiggins et al., Immunol Rev, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32699415' target='_blank'>Chan et al., Nature, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32547565' target='_blank'>Stratigopoulou et al., Front Immunol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31915197' target='_blank'>Mondello et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26385350' target='_blank'>Gu et al., Blood, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25676421' target='_blank'>Yamamoto et al., Leukemia, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23913470' target='_blank'>Amengual et al., Blood, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23523425' target='_blank'>Junco et al., Structure, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23299888' target='_blank'>Romero-Camarero et al., Nat Commun, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22897848' target='_blank'>Sander et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20876806' target='_blank'>Gruber et al., Cancer Res, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36835467' target='_blank'>Zhao et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32352029' target='_blank'>Elsarraj et al., NPJ Breast Cancer, 2020<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38636894' target='_blank'>Zhang et al., 2024, Cancer Lett<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38340178' target='_blank'>Yu et al., 2024, Cell Mol Life Sci<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37623434' target='_blank'>Wimberger et al., 2023, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31570692' target='_blank'>Mosquera et al., Blood Cancer J, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30520556' target='_blank'>Spina et al., Mol Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29463802' target='_blank'>Herling et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28903739' target='_blank'>Georgiadis et al., BMC Genomics, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26888114' target='_blank'>Wang et al., Sci Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26847028' target='_blank'>Rose-Zerilli et al., Leukemia, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21331077' target='_blank'>Darding et al., Cell Death Differ, 2011<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40319015' target='_blank'>Umphred-Wilson et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40103284' target='_blank'>Mondal et al., 2025, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38594748' target='_blank'>Fonseca et al., 2024, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37747726' target='_blank'>Huang et al., 2023, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37644036' target='_blank'>Rasool et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37312634' target='_blank'>Peñuelas-Haro et al., 2023, EMBO J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37055414' target='_blank'>Fiskus et al., 2023, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35417004' target='_blank'>French et al., Clin Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34750516' target='_blank'>Wu et al., Oncogene, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33452461' target='_blank'>Eagen et al., Oncogene, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32371576' target='_blank'>Morrison-Smith et al., Mol Cancer Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28933601' target='_blank'>Sakamaki et al., Autophagy, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28525743' target='_blank'>Sakamaki et al., Mol Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26631615' target='_blank'>Henssen et al., Clin Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22722403' target='_blank'>Belkina et al., Nat Rev Cancer, 2012<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40240348' target='_blank'>Cui et al., 2025, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36656933' target='_blank'>Mlynarczyk et al., 2023, Science<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36375119' target='_blank'>Delage et al., Blood, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35880434' target='_blank'>Kim et al., BMB Rep, 2022<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35880434' target='_blank'>Kim et al., BMB Rep, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35300128' target='_blank'>Zhang et al., Int J Gen Med, 2022<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39266715' target='_blank'>Ju et al., 2024, NPJ Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38968154' target='_blank'>Aslan et al., 2024, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38559060' target='_blank'>Black et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36287107' target='_blank'>Lai et al., 2022, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34157287' target='_blank'>Jackson et al., J Biol Chem, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33579791' target='_blank'>Afaghani et al., Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32705581' target='_blank'>Pikatan et al., Cell Oncol (Dordr), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30011241' target='_blank'>Yu et al., SLAS Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27297551' target='_blank'>Unknown et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26715645' target='_blank'>Gunderson et al., Cancer Discov, 2016<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39403923' target='_blank'>Bohlen et al., 2024, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35267643' target='_blank'>Fiñana et al., Cancers (Basel), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31943762' target='_blank'>Weisberg et al., J Cell Mol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31657576' target='_blank'>Pearson et al., 2019, J Proteome Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28611190' target='_blank'>Lv et al., Genes Dev, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28166483' target='_blank'>Gallo et al., Cell Cycle, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28082680' target='_blank'>Nadeau et al., J Biol Chem, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26980750' target='_blank'>Bresolin et al., 2016, Oncotarget<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26028666' target='_blank'>Katzav et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25534201' target='_blank'>Taylor et al., Exp Hematol, 2015<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34406363' target='_blank'>Pieters et al., J Exp Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33565425' target='_blank'>von et al., Asian J Androl, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33020271' target='_blank'>Fish et al., Proc Natl Acad Sci U S A, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31186416' target='_blank'>Kampen et al., 2019, Nat Commun<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34685420' target='_blank'>Poltavets et al., Life (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30940644' target='_blank'>Cristiani et al., Cancer Immunol Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26936935' target='_blank'>Weitzenfeld et al., J Leukoc Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24586197' target='_blank'>Huser et al., PLoS Genet, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20605642' target='_blank'>Lalor et al., 2010, J Neuroimmunol<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40135473' target='_blank'>Zhong et al., 2025, Int J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40102299' target='_blank'>Zhang et al., 2025, Cancer Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39991958' target='_blank'>Rocha et al., 2025, Oncoimmunology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39817951' target='_blank'>Wang et al., 2025, Drug Resist Updat<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38828669' target='_blank'>Wang et al., 2024, CNS Neurosci Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36894036' target='_blank'>Saito et al., 2023, Gastroenterology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35985510' target='_blank'>Hu et al., Cancer Lett, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34026331' target='_blank'>Gutting et al., Oncoimmunology, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33753247' target='_blank'>Di et al., Crit Rev Oncol Hematol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33403009' target='_blank'>Li et al., Ther Adv Med Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32923114' target='_blank'>Liu et al., Oncoimmunology, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32830120' target='_blank'>Xu et al., ESMO Open, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32425930' target='_blank'>Xie et al., Front Immunol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28451792' target='_blank'>Chen et al., Cancer Immunol Immunother, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25901287' target='_blank'>Romano et al., J Immunother Cancer, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38328479' target='_blank'>Costa-Guda et al., 2024, J Endocr Soc<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33778063' target='_blank'>Mao et al., Int J Genomics, 2021<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/33659038' target='_blank'>Lewis et al., Oncotarget, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32541670' target='_blank'>Vu et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31558674' target='_blank'>Rawat et al., Haematologica, 2020<\/a>",null,null,null,null,null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39705539' target='_blank'>Luo et al., 2024, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38641704' target='_blank'>Yang et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36638621' target='_blank'>Sun et al., Bioorg Med Chem, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34071627' target='_blank'>Song et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30675010' target='_blank'>Nabinger et al., Leukemia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30661740' target='_blank'>Grimm et al., Bioorg Med Chem, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29152664' target='_blank'>Zhang et al., Mol Med Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18175263' target='_blank'>Cheng et al., IDrugs, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38531054' target='_blank'>Kesarwani et al., 2024, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35482940' target='_blank'>Carratt et al., Blood, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35200567' target='_blank'>Adam et al., Curr Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26254266' target='_blank'>Mehta et al., J Immunol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23656643' target='_blank'>Maxson et al., N Engl J Med, 2013<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37098512' target='_blank'>Gu et al., 2023, BMC Genomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34070180' target='_blank'>Griesmann et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26481584' target='_blank'>DeNicola et al., Genome Biol, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40461894' target='_blank'>Zhang et al., 2025, Ann Hematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37490147' target='_blank'>Ildiz et al., 2023, Clin Exp Metastasis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34363012' target='_blank'>Lewis et al., Leukemia, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33669329' target='_blank'>Kaiser et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33530455' target='_blank'>López-Gil et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32802894' target='_blank'>Manukyan et al., J Immunol Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29665227' target='_blank'>Białopiotrowicz et al., J Cell Mol Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27185877' target='_blank'>Lopez-Haber et al., Mol Cell Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26980947' target='_blank'>Sainz et al., Mediators Inflamm, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26212080' target='_blank'>Kwon et al., Cell Stem Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24770893' target='_blank'>Hattermann et al., Cell Tissue Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24452472' target='_blank'>Clift et al., Mol Pharmacol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23868972' target='_blank'>Struckhoff et al., J Cell Sci, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22052462' target='_blank'>Sengupta et al., Cancer Res, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20102637' target='_blank'>Sun et al., Mol Cancer, 2010<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/26206333' target='_blank'>Xu et al., Mol Cancer Ther, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22752582' target='_blank'>Gold et al., Top Curr Chem, 2013<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40240142' target='_blank'>Fraire et al., 2025, Genes Dev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38712047' target='_blank'>Fraire et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38647377' target='_blank'>Tiburcio et al., 2024, Mol Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36946612' target='_blank'>Bortoletto et al., 2023, Cancer Res<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40060438' target='_blank'>Koutouan et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34204178' target='_blank'>Udompholkul et al., Molecules, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33626345' target='_blank'>Wang et al., Cell Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30987999' target='_blank'>Emmons et al., Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29599409' target='_blank'>Xu et al., Clin Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27808240' target='_blank'>Pulido-Tamayo et al., Sci Rep, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27533087' target='_blank'>Efazat et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21205836' target='_blank'>Taddei et al., Mol Cancer Res, 2011<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38991192' target='_blank'>Zerella et al., 2024, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37973913' target='_blank'>Liao et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37735473' target='_blank'>Kugler et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34329586' target='_blank'>Seong et al., Cancer Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33279858' target='_blank'>Jiang et al., EBioMedicine, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31511663' target='_blank'>Bushweller et al., Nat Rev Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30291252' target='_blank'>Shin et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30181359' target='_blank'>Wang et al., Genes Dev, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28198367' target='_blank'>Potu et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26844274' target='_blank'>Petrovics et al., EBioMedicine, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26701088' target='_blank'>Wang et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26344095' target='_blank'>Gan et al., Mol Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24759320' target='_blank'>Asangani et al., Nature, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23704282' target='_blank'>Schoenborn et al., 2013, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23148372' target='_blank'>Alumkal et al., Cancer Discov, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38388484' target='_blank'>Wang et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36064595' target='_blank'>Jiang et al., Nat Rev Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34092790' target='_blank'>Wang et al., Nat Methods, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30181359' target='_blank'>Wang et al., Genes Dev, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25866208' target='_blank'>Cooper et al., J Biol Chem, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40606603' target='_blank'>Li et al., 2025, Front Pharmacol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32018225' target='_blank'>Hao et al., Aging (Albany NY), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28827401' target='_blank'>Simón-Carrasco et al., 2017, Genes Dev<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38511262' target='_blank'>Kumar et al., 2024, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37832550' target='_blank'>Derks et al., 2023, Cell Stem Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37503586' target='_blank'>Renzi et al., 2023, Curr Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37286599' target='_blank'>Griffin et al., 2023, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36658219' target='_blank'>Gao et al., 2023, Nat Cell Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35190728' target='_blank'>Dinh et al., 2022, Nat Rev Gastroenterol Hepatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34429303' target='_blank'>Keddy et al., Cold Spring Harb Mol Case Stud, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34211850' target='_blank'>Wang et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33747936' target='_blank'>Li et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31645351' target='_blank'>Kung et al., Cold Spring Harb Mol Case Stud, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31395741' target='_blank'>Hurtz et al., Genes Dev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30381299' target='_blank'>Polak et al., Haematologica, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29623306' target='_blank'>Shukla et al., JCO Precis Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28739882' target='_blank'>Liu et al., J Immunol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23969693' target='_blank'>White et al., Nat Rev Cancer, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39353441' target='_blank'>Chang et al., 2024, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39190349' target='_blank'>Su et al., 2024, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39024100' target='_blank'>Suvarna et al., 2024, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37963210' target='_blank'>Graca Marques et al., 2023, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34060252' target='_blank'>Su et al., Adv Sci (Weinh), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34009296' target='_blank'>Saulnier et al., Nucleic Acids Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33766983' target='_blank'>Su et al., Genes Dev, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33147457' target='_blank'>Zhou et al., Cell Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32948771' target='_blank'>Nabet et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31577941' target='_blank'>Srivastava et al., Cell Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31431459' target='_blank'>Yeung et al., Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31392992' target='_blank'>Selvanathan et al., Nucleic Acids Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30045945' target='_blank'>Stolte et al., J Exp Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26859456' target='_blank'>Ham et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25453903' target='_blank'>Riggi et al., Cancer Cell, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38881420' target='_blank'>Zhang et al., 2024, Am J Physiol Cell Physiol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35979619' target='_blank'>Zhang et al., Clin Transl Med, 2022<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/25613377' target='_blank'>Teodorczyk et al., Cell Death Differ, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19628709' target='_blank'>Klapproth et al., Blood, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10749152' target='_blank'>Shinohara et al., Cancer Res, 2000<\/a>",null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39974881' target='_blank'>Crawford et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39088700' target='_blank'>Shin et al., 2024, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38627607' target='_blank'>Braun et al., 2024, Br J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36787089' target='_blank'>Harding et al., Target Oncol, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36097178' target='_blank'>Fiorito et al., Br J Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34344433' target='_blank'>Chew et al., Breast Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33182650' target='_blank'>Alijaj et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31575540' target='_blank'>Hatlen et al., Cancer Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31201553' target='_blank'>Schumacher et al., Handb Exp Pharmacol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30634399' target='_blank'>Lang et al., Cells, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28943626' target='_blank'>Repana et al., Diseases, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24279986' target='_blank'>Luo et al., Cancer Metab, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23944363' target='_blank'>Heinzle et al., Curr Pharm Des, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38234243' target='_blank'>Simón-Carrasco et al., 2024, Cell Cycle<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17374991' target='_blank'>Bianchi et al., Cell Cycle, 2007<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40277780' target='_blank'>Hassan et al., 2025, Curr Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36701047' target='_blank'>Hofstedter et al., Endocr Pathol, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36357396' target='_blank'>Asrani et al., Nat Commun, 2022<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40419484' target='_blank'>Kawasaki et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38895460' target='_blank'>Subramanian et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36332622' target='_blank'>Han et al., Cancer Cell, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35835117' target='_blank'>Orstad et al., 2022, Dev Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31324031' target='_blank'>Leskela et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30894157' target='_blank'>Lai et al., BMC Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30091462' target='_blank'>Neff et al., 2018, Int J Cancer<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39776254' target='_blank'>Amarilla-Quintana et al., 2025, Mol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36409821' target='_blank'>Llano et al., Cancer Res, 2023<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40211872' target='_blank'>Wang et al., 2025, Clin Mol Hepatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38992040' target='_blank'>Murray et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38508144' target='_blank'>Abu-Zaid et al., 2024, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37094128' target='_blank'>Mota et al., 2023, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36656711' target='_blank'>Watson et al., 2023, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35857643' target='_blank'>Singh et al., Sci Transl Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34386729' target='_blank'>Sunkel et al., iScience, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33080033' target='_blank'>Shi et al., Nucleic Acids Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32943730' target='_blank'>Chen et al., Oncogene, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32416589' target='_blank'>Gryder et al., iScience, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32251397' target='_blank'>Fazio et al., Nat Rev Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31869746' target='_blank'>Timme et al., Transl Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31031007' target='_blank'>Yan et al., Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29362363' target='_blank'>Li et al., Oncogenesis, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28446439' target='_blank'>Gryder et al., Cancer Discov, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38374116' target='_blank'>Rao et al., 2024, Genome Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22718346' target='_blank'>Marlow et al., J Cell Sci, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40169859' target='_blank'>Hong et al., 2025, Cancer Gene Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36408691' target='_blank'>Xie et al., FEBS Open Bio, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33716296' target='_blank'>Li et al., Oncogene, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33006432' target='_blank'>Zhang et al., Cancer Med, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37388925' target='_blank'>Jevtic et al., 2023, Hemasphere<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37015324' target='_blank'>Obeidin et al., 2023, J Pathol Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31220736' target='_blank'>Yu et al., Neoplasia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31143874' target='_blank'>Chen et al., EClinicalMedicine, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30898787' target='_blank'>Trautmann et al., 2019, EMBO Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27822137' target='_blank'>Kerr et al., 2016, Sarcoma<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23533276' target='_blank'>Tang et al., Proc Natl Acad Sci U S A, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39976744' target='_blank'>Vendemini et al., 2025, J Clin Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39624268' target='_blank'>Xu et al., 2024, Front Cell Infect Microbiol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36475518' target='_blank'>Rein et al., 2022, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34387894' target='_blank'>Homan et al., Hum Mutat, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32493985' target='_blank'>Fu et al., Sci Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31554341' target='_blank'>Ordoñez et al., 2019, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29780592' target='_blank'>Ng et al., Biomark Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28174417' target='_blank'>Dang et al., Leukemia, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24448395' target='_blank'>Chiang et al., Oncotarget, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40248085' target='_blank'>Ashizawa et al., 2025, Front Med (Lausanne)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40115023' target='_blank'>Wang et al., 2025, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39821893' target='_blank'>Yakubov et al., 2025, J Neurooncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39483334' target='_blank'>Qi et al., 2024, Front Cell Dev Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38493151' target='_blank'>Lei et al., 2024, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34206026' target='_blank'>Uribe et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33203881' target='_blank'>Pietrobono et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32615027' target='_blank'>Grund-Gröschke et al., Mol Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31878932' target='_blank'>Grund-Gröschke et al., Cell Commun Signal, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28457748' target='_blank'>Schneider et al., 2017, Cell Stem Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26029667' target='_blank'>Cage et al., Front Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25492239' target='_blank'>Tilley et al., Mol Carcinog, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25111486' target='_blank'>Wang et al., 2014, Epigenomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24223768' target='_blank'>Fertig et al., PLoS One, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22294553' target='_blank'>Eberl et al., EMBO Mol Med, 2012<\/a>",null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39216686' target='_blank'>Gowda et al., 2024, J Adv Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38485737' target='_blank'>Kumar et al., 2024, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38216787' target='_blank'>Chuang et al., 2024, EMBO Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36534523' target='_blank'>Chen et al., Clin Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34902246' target='_blank'>Damalanka et al., J Med Chem, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34425853' target='_blank'>Drusbosky et al., J Hematol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33049974' target='_blank'>O'Neill et al., Antioxidants (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30094412' target='_blank'>Oba et al., 2018, JCO Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28888686' target='_blank'>Datta et al., Cell Signal, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25873032' target='_blank'>Zoratti et al., Nat Commun, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24167653' target='_blank'>Xie et al., Genes Cancer, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23212109' target='_blank'>Hansen et al., Clin Pharmacol Ther, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22689056' target='_blank'>Whale et al., Oncogene, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21744257' target='_blank'>Ogunwobi et al., Clin Exp Metastasis, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20807782' target='_blank'>Coon et al., Mol Cancer Ther, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39806421' target='_blank'>Ma et al., 2025, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37382594' target='_blank'>Zhang et al., 2023, Aging (Albany NY)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33658640' target='_blank'>Gampala et al., Br J Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29758199' target='_blank'>Ma et al., 2018, Biochem Biophys Res Commun<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40731412' target='_blank'>Crawford et al., 2025, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39974881' target='_blank'>Crawford et al., 2025, bioRxiv<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39810713' target='_blank'>Sun et al., 2025, Kaohsiung J Med Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36451863' target='_blank'>Liu et al., Theranostics, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32485038' target='_blank'>Cheng et al., J Cell Mol Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27530828' target='_blank'>Anczuków et al., RNA, 2016<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37555980' target='_blank'>Quintanilla-Martinez et al., 2023, Virchows Arch<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34462424' target='_blank'>Xu et al., Cell Death Discov, 2021<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36991428' target='_blank'>Huh et al., 2023, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31333801' target='_blank'>Han et al., J Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30862934' target='_blank'>Schroeder et al., 2019, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28190000' target='_blank'>Witkowski et al., J Exp Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26321221' target='_blank'>Churchman et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25655195' target='_blank'>Witkowski et al., 2015, Leukemia<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39029465' target='_blank'>McRae et al., 2024, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35504924' target='_blank'>Amanda et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29588546' target='_blank'>Schleussner et al., Leukemia, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29348807' target='_blank'>Michaelides et al., ACS Med Chem Lett, 2018<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38889153' target='_blank'>Smolko et al., 2024, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29179522' target='_blank'>Liu et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27033163' target='_blank'>Haladyna et al., Exp Hematol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26766589' target='_blank'>Cheung et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26075180' target='_blank'>Sun et al., Front Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23149921' target='_blank'>Tam et al., Cancer Res, 2013<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40557064' target='_blank'>Lin et al., 2025, ACS Med Chem Lett<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37803464' target='_blank'>Peroni et al., 2023, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35532219' target='_blank'>Sanchez et al., 2022, Bioessays<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33479542' target='_blank'>Terlecki-Zaniewicz et al., Nat Struct Mol Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32344427' target='_blank'>Schmoellerl et al., Blood, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31698461' target='_blank'>Cardin et al., Blood Adv, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40365104' target='_blank'>Topcu et al., 2025, Turk J Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36542714' target='_blank'>Kossinna et al., Sci Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31706255' target='_blank'>Zhong et al., Aging (Albany NY), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23516391' target='_blank'>Shah et al., PLoS One, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40040508' target='_blank'>Lin et al., 2025, Expert Rev Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39995670' target='_blank'>Ju et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39775867' target='_blank'>Landry et al., 2025, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37470859' target='_blank'>Takahashi et al., 2023, J Gastroenterol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33028669' target='_blank'>Fujikura et al., 2020, Gut<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31281496' target='_blank'>Mai et al., Theranostics, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30279357' target='_blank'>Testa et al., Med Sci (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29606053' target='_blank'>Bi et al., Neurosurg Focus, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28195122' target='_blank'>Harmancı et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26338995' target='_blank'>Sun et al., Clin Cancer Res, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38433576' target='_blank'>Zhang et al., 2024, Acta Biochim Biophys Sin (Shanghai)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38087142' target='_blank'>Wang et al., 2023, Med Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35522942' target='_blank'>Wang et al., Clin Transl Med, 2022<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38352401' target='_blank'>Izadmehr et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33816511' target='_blank'>Hu et al., Front Cell Dev Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31065692' target='_blank'>Yang et al., J Mol Cell Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28881705' target='_blank'>Xiping et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23345610' target='_blank'>Hatami et al., Sci Transl Med, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33889830' target='_blank'>Bubie et al., iScience, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30521023' target='_blank'>Huang et al., 2018, Gigascience<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39113657' target='_blank'>Gonzalez-Gil et al., 2024, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38972954' target='_blank'>Mulet-Lazaro et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32943730' target='_blank'>Chen et al., Oncogene, 2020<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39580452' target='_blank'>Schöpe et al., 2024, Biomark Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39221900' target='_blank'>Wang et al., 2024, J Cell Physiol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36333284' target='_blank'>Zhang et al., Cell Death Dis, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35597866' target='_blank'>Schmid et al., Br J Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33540543' target='_blank'>Mercogliano et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26686083' target='_blank'>Schmid et al., Oncogene, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35995507' target='_blank'>Merchand-Reyes et al., 2022, J Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32943730' target='_blank'>Chen et al., Oncogene, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31795494' target='_blank'>Diefenbach et al., Cancers (Basel), 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38863711' target='_blank'>Moud et al., 2024, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38109544' target='_blank'>Juilland et al., 2023, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37623434' target='_blank'>Wimberger et al., 2023, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34214002' target='_blank'>Hamp et al., Expert Opin Ther Pat, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32252279' target='_blank'>Hammouda et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30737474' target='_blank'>Dixit et al., 2019, Cell Death Differ<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29880602' target='_blank'>García-Ramírez et al., EMBO J, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29772764' target='_blank'>González-Herrero et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27135977' target='_blank'>Juilland et al., Curr Opin Hematol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26498146' target='_blank'>Brown et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25982276' target='_blank'>Pan et al., Oncogene, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25556945' target='_blank'>Eitelhuber et al., Chem Biol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21331077' target='_blank'>Darding et al., Cell Death Differ, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18931663' target='_blank'>Gyrd-Hansen et al., Nat Cell Biol, 2008<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39218851' target='_blank'>Asada et al., 2024, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38788713' target='_blank'>Thirimanne et al., 2024, Cell Genom<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25703106' target='_blank'>Abe et al., 2015, Int J Surg Case Rep<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39702187' target='_blank'>Su et al., 2024, Hered Cancer Clin Pract<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39651730' target='_blank'>Köster et al., 2024, Cytopathology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36982575' target='_blank'>Tseng et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35179215' target='_blank'>Alimova et al., Int J Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34188019' target='_blank'>Damerell et al., Signal Transduct Target Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29439719' target='_blank'>Xie et al., J Exp Clin Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28892044' target='_blank'>Comiskey et al., Oncogene, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28714986' target='_blank'>Spies et al., Nat Methods, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26909605' target='_blank'>Swetzig et al., Oncotarget, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33672117' target='_blank'>Zengin et al., J Pers Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30150207' target='_blank'>Hu et al., Blood, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37919784' target='_blank'>McCord et al., 2023, Acta Neuropathol Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34424912' target='_blank'>Saini et al., PLoS Comput Biol, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/30275527' target='_blank'>Patel et al., Leukemia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29264386' target='_blank'>Cai et al., JAAD Case Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20967208' target='_blank'>Mrkonjic et al., PLoS One, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40062529' target='_blank'>Loscocco et al., 2025, Am J Hematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39279726' target='_blank'>Sun et al., 2024, Eur J Haematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38035089' target='_blank'>De Luca et al., 2023, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37396034' target='_blank'>Zhang et al., 2023, Front Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36709354' target='_blank'>Staehle et al., 2023, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35637336' target='_blank'>Sousos et al., 2022, Nat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34845187' target='_blank'>Varghese et al., Blood Cancer J, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33144355' target='_blank'>Michail et al., J Clin Pathol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33094091' target='_blank'>Horton et al., Curr Stem Cell Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32781570' target='_blank'>Skov et al., 2020, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32770086' target='_blank'>Chowdhury et al., Leukemia, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31890207' target='_blank'>Pernes et al., 2020, Clin Transl Immunology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30262727' target='_blank'>Mi et al., Cancers (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29306106' target='_blank'>Gángó et al., Leuk Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26919114' target='_blank'>Whetton et al., Oncotarget, 2016<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38234720' target='_blank'>Spinler et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36167829' target='_blank'>Erazo et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34458856' target='_blank'>Prieto et al., Nat Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28143872' target='_blank'>Kudinov et al., Clin Cancer Res, 2017<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40467924' target='_blank'>Zhang et al., 2025, Commun Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38889153' target='_blank'>Smolko et al., 2024, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36656711' target='_blank'>Watson et al., 2023, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34837064' target='_blank'>Safaric et al., Lab Invest, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29179522' target='_blank'>Liu et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26766589' target='_blank'>Cheung et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26075180' target='_blank'>Sun et al., Front Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23149921' target='_blank'>Tam et al., Cancer Res, 2013<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34234374' target='_blank'>Catherinet et al., 2021, PLoS One<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32397368' target='_blank'>Wang et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25072963' target='_blank'>Qin et al., Biochim Biophys Acta, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37087499' target='_blank'>Sim et al., 2023, Commun Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24740322' target='_blank'>Liu et al., Clin Cancer Res, 2014<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40691407' target='_blank'>Lu et al., 2025, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39383000' target='_blank'>Yang et al., 2024, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36932191' target='_blank'>Heylen et al., 2023, Br J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33439550' target='_blank'>Tong et al., EMBO Mol Med, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37903909' target='_blank'>Jo et al., 2023, Br J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35932909' target='_blank'>Moody et al., 2022, Peptides<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35135829' target='_blank'>Schram et al., Cancer Discov, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34077268' target='_blank'>Drilon et al., J Clin Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33839363' target='_blank'>Odintsov et al., J Thorac Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32206189' target='_blank'>Dell'Aquila et al., Oncotarget, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31406302' target='_blank'>Skoulidis et al., Nat Rev Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29228724' target='_blank'>Conway et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26992917' target='_blank'>Dong et al., Neoplasia, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26486077' target='_blank'>Wang et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26134228' target='_blank'>Jung et al., 2015, J Thorac Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25759024' target='_blank'>Wilson et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20421926' target='_blank'>Larriba et al., PLoS One, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40334081' target='_blank'>Mohanty et al., 2025, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35073946' target='_blank'>Jevtic et al., J Exp Clin Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34095772' target='_blank'>Richardson et al., Hemasphere, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33626351' target='_blank'>Farhangdoost et al., Cell Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33557090' target='_blank'>Ernst et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32993115' target='_blank'>Mohanty et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32815876' target='_blank'>Behnert et al., J Pediatr Hematol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32344427' target='_blank'>Schmoellerl et al., Blood, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28754676' target='_blank'>Su et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26118503' target='_blank'>Chen et al., Exp Hematol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24927133' target='_blank'>Drake et al., Assay Drug Dev Technol, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40586874' target='_blank'>Chava et al., 2025, Cell Mol Life Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38779359' target='_blank'>Li et al., 2024, Precis Clin Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38256018' target='_blank'>Nuñez et al., 2024, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36360250' target='_blank'>Murali et al., Genes (Basel), 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34083510' target='_blank'>Peng et al., Signal Transduct Target Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34002060' target='_blank'>Zhao et al., 2021, Nat Rev Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33536620' target='_blank'>Yuan et al., Nature, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28901481' target='_blank'>Liu et al., Oncol Rep, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28484924' target='_blank'>Turner-Ivey et al., Breast Cancer Res Treat, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37958889' target='_blank'>Park et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37743366' target='_blank'>Miyazaki et al., 2023, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33643817' target='_blank'>Jiang et al., Acta Pharm Sin B, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32755586' target='_blank'>Vaishnavi et al., Cell Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32497496' target='_blank'>Li et al., Mol Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32315394' target='_blank'>Joshi et al., Blood, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31406302' target='_blank'>Skoulidis et al., Nat Rev Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31134065' target='_blank'>Kleczko et al., Front Immunol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30220707' target='_blank'>Ziegler et al., Br J Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29654229' target='_blank'>Oldrini et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29305654' target='_blank'>Brodeur et al., Cell Tissue Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29191878' target='_blank'>Klaeger et al., Science, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28695888' target='_blank'>Cook et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26881293' target='_blank'>Passiglia et al., Expert Opin Investig Drugs, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25331179' target='_blank'>Brodeur et al., Nat Rev Clin Oncol, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40061872' target='_blank'>Mohammadi et al., 2025, Bioinform Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39554798' target='_blank'>Chen et al., 2024, MedComm (2020)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39365700' target='_blank'>Schmid et al., 2024, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38558981' target='_blank'>Schmid et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37743366' target='_blank'>Miyazaki et al., 2023, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34429303' target='_blank'>Keddy et al., Cold Spring Harb Mol Case Stud, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34389035' target='_blank'>Johansson et al., Thyroid Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34211850' target='_blank'>Wang et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31645351' target='_blank'>Kung et al., Cold Spring Harb Mol Case Stud, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29623306' target='_blank'>Shukla et al., JCO Precis Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26884591' target='_blank'>Drilon et al., Ann Oncol, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40389480' target='_blank'>Troester et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40166939' target='_blank'>Xu et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39933697' target='_blank'>Li et al., 2025, Nucleic Acids Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39883527' target='_blank'>Hamamoto et al., 2025, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37803464' target='_blank'>Peroni et al., 2023, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37147424' target='_blank'>Matsukawa et al., 2023, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35073946' target='_blank'>Jevtic et al., J Exp Clin Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34582559' target='_blank'>Heikamp et al., Blood, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34095772' target='_blank'>Richardson et al., Hemasphere, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33479542' target='_blank'>Terlecki-Zaniewicz et al., Nat Struct Mol Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32344427' target='_blank'>Schmoellerl et al., Blood, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31698461' target='_blank'>Cardin et al., Blood Adv, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28202522' target='_blank'>Valerio et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27889185' target='_blank'>Xu et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25464900' target='_blank'>Deshpande et al., Cancer Cell, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40791552' target='_blank'>Luo et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37082775' target='_blank'>Charlab et al., 2023, Clin Transl Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36941503' target='_blank'>Juhlin et al., 2023, Head Neck Pathol<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/31119201' target='_blank'>Santana-Codina et al., Annu Rev Cancer Biol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27037419' target='_blank'>White et al., Cold Spring Harb Perspect Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24492513' target='_blank'>Lozy et al., Autophagy, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38548747' target='_blank'>Liu et al., 2024, Int J Oral Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34011561' target='_blank'>Miao et al., Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25901287' target='_blank'>Romano et al., J Immunother Cancer, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40628732' target='_blank'>Singh et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38071343' target='_blank'>Miyazaki et al., 2023, NPJ Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37985498' target='_blank'>Schoelinck et al., 2023, Virchows Arch<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37941480' target='_blank'>De Coninck et al., 2023, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35659339' target='_blank'>Lv et al., 2022, Dev Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35239349' target='_blank'>Chen et al., J Med Chem, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33478100' target='_blank'>Aldaz et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32060308' target='_blank'>Ivanov et al., Sci Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29777182' target='_blank'>Laug et al., Nat Rev Neurosci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28289245' target='_blank'>Venkatesan et al., Cold Spring Harb Perspect Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28138037' target='_blank'>Almiron et al., Mol Cancer Ther, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27011036' target='_blank'>Boikos et al., 2016, JAMA Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26316776' target='_blank'>Spitaleri et al., Onco Targets Ther, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23580781' target='_blank'>Smith et al., Clin Cancer Res, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22614970' target='_blank'>George et al., J Clin Oncol, 2012<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40479062' target='_blank'>Ismail et al., 2025, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39913208' target='_blank'>Pitzen et al., 2025, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39694694' target='_blank'>Bourgeois et al., 2024, Cold Spring Harb Perspect Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38066546' target='_blank'>Voon et al., 2023, Genome Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36450825' target='_blank'>Missiroli et al., Cell Death Differ, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34529761' target='_blank'>Kontandreopoulou et al., Blood Adv, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33531470' target='_blank'>Bernasocchi et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30472087' target='_blank'>Li et al., EBioMedicine, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29795382' target='_blank'>Lehmann-Che et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28174417' target='_blank'>Dang et al., Leukemia, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26126967' target='_blank'>Sive et al., Leukemia, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25683711' target='_blank'>Prunier et al., Cell Rep, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25466492' target='_blank'>Lunardi et al., Trends Mol Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22333881' target='_blank'>Leiva et al., Leukemia, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10611345' target='_blank'>Pollock et al., Proc Natl Acad Sci U S A, 1999<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39246801' target='_blank'>Don et al., 2024, Case Rep Hematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31937561' target='_blank'>Shen et al., J Med Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29523635' target='_blank'>Müller et al., 2018, G3 (Bethesda)<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33746966' target='_blank'>Salem et al., Front Immunol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28790031' target='_blank'>Chiou et al., Cancer Discov, 2017<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39636745' target='_blank'>Ford et al., 2024, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30796242' target='_blank'>Yang et al., 2019, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29068423' target='_blank'>Shukla et al., Proteomes, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25829446' target='_blank'>Mense et al., Sci Signal, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/25594057' target='_blank'>Hampton et al., Oncoscience, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23928995' target='_blank'>Regan et al., Cancer Res, 2013<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40777043' target='_blank'>Zhang et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31792216' target='_blank'>Boone et al., Nat Commun, 2019<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/33673070' target='_blank'>Funakoshi et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29693319' target='_blank'>Urbanski et al., Wiley Interdiscip Rev RNA, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27973981' target='_blank'>Jain et al., Cell Cycle, 2017<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39913208' target='_blank'>Pitzen et al., 2025, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39694694' target='_blank'>Bourgeois et al., 2024, Cold Spring Harb Perspect Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37655965' target='_blank'>Bercier et al., 2023, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37506247' target='_blank'>Wang et al., 2023, Aging (Albany NY)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35882408' target='_blank'>Guo et al., Biomed Environ Sci, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34529761' target='_blank'>Kontandreopoulou et al., Blood Adv, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33531470' target='_blank'>Bernasocchi et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30472087' target='_blank'>Li et al., EBioMedicine, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29795382' target='_blank'>Lehmann-Che et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26126967' target='_blank'>Sive et al., Leukemia, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23509325' target='_blank'>Ablain et al., J Exp Med, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22333881' target='_blank'>Leiva et al., Leukemia, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/10611345' target='_blank'>Pollock et al., Proc Natl Acad Sci U S A, 1999<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/9096882' target='_blank'>Costa et al., Biol Reprod, 1997<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38291083' target='_blank'>Vescio et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35425962' target='_blank'>Yousefi et al., Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33125148' target='_blank'>Zhang et al., Oncol Rep, 2020<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39605726' target='_blank'>Rominger et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35904492' target='_blank'>Chen et al., Cancer Discov, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35419481' target='_blank'>Riley et al., Mol Cell Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34854279' target='_blank'>Bertrums et al., 2021, Haematologica<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/29068423' target='_blank'>Shukla et al., Proteomes, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26608094' target='_blank'>Xu et al., 2015, Nat Commun<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40517166' target='_blank'>Huang et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38104183' target='_blank'>Rampioni Vinciguerra et al., 2023, Cell Death Differ<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37743366' target='_blank'>Miyazaki et al., 2023, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37012551' target='_blank'>Goldsmith et al., 2023, Nat Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33667368' target='_blank'>Hahn et al., Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33071215' target='_blank'>Binkley et al., Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32649874' target='_blank'>Gillette et al., Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32589943' target='_blank'>Kudo et al., 2020, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29602801' target='_blank'>Khanna et al., Clin Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29127121' target='_blank'>Di et al., Clin Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28088512' target='_blank'>Lin et al., J Thorac Oncol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25667280' target='_blank'>Mazières et al., J Clin Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24771645' target='_blank'>Wynes et al., Clin Cancer Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23665120' target='_blank'>Myant et al., Cell Stem Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22617245' target='_blank'>Yasuda et al., J Thorac Oncol, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/30850735' target='_blank'>Kampen et al., 2019, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30650663' target='_blank'>Penzo et al., 2019, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29930300' target='_blank'>Kampen et al., 2018, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28715419' target='_blank'>Patchett et al., PLoS Genet, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35176205' target='_blank'>Wright et al., 2022, FEBS J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27829215' target='_blank'>Sulzmaier et al., Oncotarget, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38136362' target='_blank'>Hortal et al., 2023, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22918885' target='_blank'>Patmore et al., Cancer Res, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/31701073' target='_blank'>Conboy et al., Hepatol Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29505958' target='_blank'>Szemes et al., Neoplasia, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40479062' target='_blank'>Ismail et al., 2025, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39450904' target='_blank'>Chen et al., 2024, Protein Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38992040' target='_blank'>Murray et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37992684' target='_blank'>Popescu et al., 2023, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37957175' target='_blank'>Xu et al., 2023, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37807875' target='_blank'>Magliulo et al., 2023, EMBO Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37319434' target='_blank'>Fei et al., 2023, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32344427' target='_blank'>Schmoellerl et al., Blood, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31130394' target='_blank'>Tracey et al., Trends Genet, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30472087' target='_blank'>Li et al., EBioMedicine, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30300583' target='_blank'>Martinez-Soria et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28778402' target='_blank'>Heuser et al., Semin Cancer Biol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26666262' target='_blank'>Hatlen et al., J Exp Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25873173' target='_blank'>Shih et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24332853' target='_blank'>Vu et al., Cell Rep, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/29738475' target='_blank'>Li et al., Molecules, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29143738' target='_blank'>Halstead et al., Elife, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/32513235' target='_blank'>Sun et al., J Exp Clin Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32489324' target='_blank'>Shao et al., Cancer Cell Int, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26407074' target='_blank'>Liu et al., PLoS One, 2015<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37883365' target='_blank'>Perez et al., 2023, PLoS Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33373647' target='_blank'>Saha et al., Biochim Biophys Acta Rev Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33355042' target='_blank'>Zheng et al., Technol Cancer Res Treat, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33064882' target='_blank'>Fazio et al., Pigment Cell Melanoma Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31131878' target='_blank'>Orouji et al., Int J Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29926931' target='_blank'>Xiao et al., J Pathol, 2018<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37053475' target='_blank'>Smolnig et al., 2023, J Proteome Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33466757' target='_blank'>Tang et al., Int J Mol Sci, 2021<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39753728' target='_blank'>Zhou et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37378633' target='_blank'>Yu et al., 2023, Hepatol Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33724679' target='_blank'>Gautron et al., EMBO Mol Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33669690' target='_blank'>Du et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31798766' target='_blank'>Singha et al., Genes Cancer, 2019<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38094249' target='_blank'>Malawsky et al., 2023, iScience<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34205978' target='_blank'>Papatsirou et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33446260' target='_blank'>Wu et al., Genome Biol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33107052' target='_blank'>Wang et al., J Cell Physiol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31279575' target='_blank'>Han et al., Dev Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31027259' target='_blank'>Riobo-Del et al., Cells, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30884815' target='_blank'>Fabbri et al., Int J Mol Sci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30319965' target='_blank'>Geyer et al., Front Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29606053' target='_blank'>Bi et al., Neurosurg Focus, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29103956' target='_blank'>Wu et al., Dev Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28777795' target='_blank'>Han et al., PLoS Biol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27539549' target='_blank'>Habib et al., Cancer Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24569090' target='_blank'>Tech et al., Cancer Lett, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21874045' target='_blank'>Fernandez-L et al., Oncogene, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20530674' target='_blank'>Heine et al., Cancer Res, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37905668' target='_blank'>Wright et al., 2023, Biochem J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33768972' target='_blank'>Wright et al., Epigenomes, 2021<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37730692' target='_blank'>Baltanás et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27792127' target='_blank'>Bii et al., Cancers (Basel), 2016<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38044445' target='_blank'>Xiong et al., 2023, Mil Med Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37463901' target='_blank'>Jiang et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36630607' target='_blank'>Brandstoetter et al., Blood, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32679107' target='_blank'>Siersbæk et al., Cancer Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32296034' target='_blank'>Luszczak et al., Signal Transduct Target Ther, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31889184' target='_blank'>Lauria et al., Nucleic Acids Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31678994' target='_blank'>Fan et al., Neuro Oncol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30446007' target='_blank'>Reilley et al., J Immunother Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30209389' target='_blank'>Pan et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28191885' target='_blank'>Chuang et al., Nat Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27881582' target='_blank'>Cardenas et al., Clin Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26215634' target='_blank'>Tactacan et al., Mol Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23486688' target='_blank'>Priester et al., Neuro Oncol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23472872' target='_blank'>Liu et al., Cell Stem Cell, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22190379' target='_blank'>Nevzorova et al., Hepatology, 2012<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38498036' target='_blank'>Klein et al., 2024, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36240739' target='_blank'>Philips et al., Cell, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33478100' target='_blank'>Aldaz et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30621209' target='_blank'>Berendsen et al., Cancers (Basel), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28369050' target='_blank'>Katerndahl et al., Nat Immunol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25302990' target='_blank'>Gressot et al., Int J Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24573384' target='_blank'>Kontro et al., Leukemia, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40196499' target='_blank'>Mondaza-Hernandez et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38659891' target='_blank'>Hill et al., 2024, bioRxiv<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/31023785' target='_blank'>Korfhage et al., Mol Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27920939' target='_blank'>Suresh et al., Surg Neurol Int, 2016<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40579412' target='_blank'>Costea et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40247105' target='_blank'>Buckley et al., 2025, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37692474' target='_blank'>Zhang et al., 2023, Genes Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35514954' target='_blank'>Parriott et al., Front Immunol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33003334' target='_blank'>Das et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32943730' target='_blank'>Chen et al., Oncogene, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32732223' target='_blank'>Tricarico et al., 2020, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32392348' target='_blank'>Erbe et al., Nucleic Acids Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31886826' target='_blank'>Kang et al., 2019, J Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31073164' target='_blank'>Liu et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30759065' target='_blank'>Hung et al., 2019, Elife<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30038221' target='_blank'>Kourtis et al., Nat Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26944678' target='_blank'>Benyoucef et al., Genes Dev, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25522333' target='_blank'>Goossens et al., PLoS Genet, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23079656' target='_blank'>Sawai et al., 2012, Cancer Cell<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39900937' target='_blank'>Pfeuffer et al., 2025, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39570282' target='_blank'>Saleem et al., 2024, J Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39551787' target='_blank'>Tigu et al., 2024, Cell Death Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35939046' target='_blank'>Ogran et al., Elife, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33596640' target='_blank'>Erkeland et al., Haematologica, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33301037' target='_blank'>Dearden et al., 2020, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33075129' target='_blank'>Alankus et al., J Exp Med, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32667245' target='_blank'>Barthel et al., Leuk Lymphoma, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30262843' target='_blank'>Zaborsky et al., Leukemia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26979393' target='_blank'>Cubillos-Ruiz et al., Clin Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26041471' target='_blank'>Prinz et al., Mol Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20863894' target='_blank'>Pekarsky et al., Semin Cancer Biol, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20357824' target='_blank'>Efanov et al., Leukemia, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17846228' target='_blank'>Despouy et al., Blood, 2007<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/28931820' target='_blank'>Mroczek et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27060136' target='_blank'>Kuchta et al., Nucleic Acids Res, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40253695' target='_blank'>Udutha et al., 2025, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39760332' target='_blank'>Hollander et al., 2025, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39743888' target='_blank'>Kim et al., 2025, Clin Mol Hepatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39557834' target='_blank'>Elliott et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39034322' target='_blank'>Wang et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38874196' target='_blank'>Karin et al., 2024, Mol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38255300' target='_blank'>Tatsuta et al., 2024, Biomedicines<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37142597' target='_blank'>Thirant et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37081420' target='_blank'>Chen et al., 2023, BMC Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34387871' target='_blank'>Hlady et al., 2021, Hepatology<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33572923' target='_blank'>Alunni-Fabbroni et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32903507' target='_blank'>Rittiner et al., Front Mol Neurosci, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32825005' target='_blank'>Eckburg et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30327349' target='_blank'>Kodama et al., Proc Natl Acad Sci U S A, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29597259' target='_blank'>Pezzuto et al., Int J Mol Sci, 2018<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40116537' target='_blank'>Hawking et al., 2025, Cancer Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31266538' target='_blank'>Poole et al., Epigenetics Chromatin, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28160624' target='_blank'>Ramezani-Rad et al., Curr Opin Immunol, 2017<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38168254' target='_blank'>Palmbos et al., 2024, Res Sq<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37492374' target='_blank'>Oyelakin et al., 2023, NAR Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32913988' target='_blank'>Bagge et al., 2018, JCO Precis Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31144617' target='_blank'>Hegde et al., Elife, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26206333' target='_blank'>Xu et al., Mol Cancer Ther, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39775867' target='_blank'>Landry et al., 2025, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39707575' target='_blank'>Vo et al., 2024, Acta Neuropathol Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38695575' target='_blank'>Wang et al., 2024, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37043537' target='_blank'>Mishra-Gorur et al., 2023, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36840836' target='_blank'>Wang et al., 2023, J Neurooncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32461543' target='_blank'>Lyu et al., 2020, Am J Case Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30279357' target='_blank'>Testa et al., Med Sci (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29606053' target='_blank'>Bi et al., Neurosurg Focus, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28195122' target='_blank'>Harmancı et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28177878' target='_blank'>Tang et al., Oncotarget, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40502048' target='_blank'>Hinz et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38828688' target='_blank'>Xu et al., 2024, Adv Sci (Weinh)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34508101' target='_blank'>Shah et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27238081' target='_blank'>Groner et al., Cancer Cell, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39389957' target='_blank'>Xia et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38104650' target='_blank'>Culig et al., 2023, Am J Pathol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35785414' target='_blank'>Chen et al., EMBO Rep, 2022<\/a>",null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39367146' target='_blank'>Cui et al., 2024, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35752351' target='_blank'>Shalom et al., Cell Signal, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29136506' target='_blank'>Robles-Valero et al., 2017, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24332539' target='_blank'>Razidlo et al., 2013, Curr Biol<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35151501' target='_blank'>Zhou et al., J Hepatol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31349001' target='_blank'>Zhou et al., J Hepatol, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38316783' target='_blank'>Company et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33679721' target='_blank'>Liang et al., Front Immunol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33110073' target='_blank'>Yee et al., Nat Commun, 2020<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36216898' target='_blank'>Wang et al., Acta Pharmacol Sin, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32608199' target='_blank'>Pham et al., Yonsei Med J, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25931453' target='_blank'>Coltella et al., Clin Cancer Res, 2015<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35440541' target='_blank'>Mohammadi et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35246504' target='_blank'>Jiang et al., 2022, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34462429' target='_blank'>Luo et al., Cell Death Dis, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33882319' target='_blank'>Bota-Rabassedas et al., Cell Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33675037' target='_blank'>Ruh et al., J Pathol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32796736' target='_blank'>Bruneel et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31263103' target='_blank'>Chen et al., Cell Death Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31063755' target='_blank'>Lu et al., Dev Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30778199' target='_blank'>Li et al., Cell Death Differ, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30679322' target='_blank'>Goossens et al., 2019, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27694892' target='_blank'>Peng et al., Oncogene, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27596438' target='_blank'>Richard et al., EMBO Mol Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27041571' target='_blank'>Zhang et al., Oncogene, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25247240' target='_blank'>Misso et al., Mol Ther Nucleic Acids, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22850877' target='_blank'>Ahn et al., J Clin Invest, 2012<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/29367757' target='_blank'>Hu et al., Oncogene, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27005999' target='_blank'>Ren et al., Int J Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22781593' target='_blank'>Ren et al., Leukemia, 2013<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35993362' target='_blank'>Mai et al., JCI Insight, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32341351' target='_blank'>Bi et al., Cell Death Dis, 2020<\/a>",null,null,null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40195782' target='_blank'>Crump et al., 2025, Bioessays<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38245503' target='_blank'>Zhang et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38048400' target='_blank'>Lin et al., 2023, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37850412' target='_blank'>Zhu et al., 2023, Clin Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37626123' target='_blank'>Crump et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36335100' target='_blank'>Okuda et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34706236' target='_blank'>Symeonidou et al., Cell Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34373735' target='_blank'>Wu et al., Theranostics, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34321607' target='_blank'>Tran et al., Leukemia, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34162911' target='_blank'>Jaiswal et al., Sci Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33069783' target='_blank'>Symeonidou et al., Exp Hematol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31681706' target='_blank'>Mercher et al., Front Pediatr, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31381950' target='_blank'>Malouf et al., Exp Hematol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30266823' target='_blank'>Chu et al., Blood Adv, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26590715' target='_blank'>Cai et al., Mol Cell, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/32873299' target='_blank'>Liu et al., Respir Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31062368' target='_blank'>Xu et al., Hepatology, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30962452' target='_blank'>Johansson et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28923911' target='_blank'>Sulima et al., Cancer Discov, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26853850' target='_blank'>Fahl et al., 2016, Crit Rev Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25304263' target='_blank'>Hu et al., Cancer Res, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39762645' target='_blank'>Zhang et al., 2025, EMBO J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35926060' target='_blank'>Kiyose et al., PLoS Genet, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32017470' target='_blank'>Kong et al., Cancer Med, 2020<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39292106' target='_blank'>Carson et al., 2024, Clin Exp Dermatol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31101826' target='_blank'>Rashid et al., Nat Commun, 2019<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40466366' target='_blank'>Nishi et al., 2025, Biochem Biophys Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35998012' target='_blank'>Torrens et al., Clin Cancer Res, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35926060' target='_blank'>Kiyose et al., PLoS Genet, 2022<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37398210' target='_blank'>Menon et al., 2023, Res Sq<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37205599' target='_blank'>Menon et al., 2023, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35426856' target='_blank'>Daley et al., J Pediatr Hematol Oncol, 2022<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/35727403' target='_blank'>Yamamoto et al., J Gastroenterol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31225458' target='_blank'>Derle et al., 2019, Cell Stress<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30470740' target='_blank'>Lee et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30302772' target='_blank'>Folkerts et al., Med Res Rev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29643976' target='_blank'>Marinković et al., Oxid Med Cell Longev, 2018<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/27045317' target='_blank'>Findlay et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25855960' target='_blank'>Gai et al., Cell Death Dis, 2015<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40597443' target='_blank'>Bullock et al., 2025, Breast Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29973405' target='_blank'>Wu et al., Sci Transl Med, 2018<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34380642' target='_blank'>Linley et al., Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34103328' target='_blank'>Tirado-Gonzalez et al., Cancer Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33878293' target='_blank'>Sadras et al., Mol Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33736651' target='_blank'>Cortes et al., J Hematol Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33447829' target='_blank'>Flümann et al., Blood Cancer Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32974613' target='_blank'>Silvestri et al., Blood Cancer Discov, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32699415' target='_blank'>Chan et al., Nature, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31628323' target='_blank'>Menzl et al., Nat Commun, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30898264' target='_blank'>Chatterjee et al., Trends Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28862699' target='_blank'>Matsushita et al., Blood Cancer J, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28190000' target='_blank'>Witkowski et al., J Exp Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27213184' target='_blank'>Balatti et al., Trends Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26073130' target='_blank'>Shojaee et al., 2015, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25759017' target='_blank'>Trimarchi et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21593872' target='_blank'>Duy et al., Nature, 2011<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/26306989' target='_blank'>Yang et al., Curr Top Med Chem, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23403638' target='_blank'>Cheng et al., Clin Cancer Res, 2013<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40355863' target='_blank'>Naseer et al., 2025, Cell Commun Signal<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40145277' target='_blank'>Du et al., 2025, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40062529' target='_blank'>Loscocco et al., 2025, Am J Hematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38885318' target='_blank'>Schmidt et al., 2024, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38253566' target='_blank'>Barosi et al., 2024, Blood Cancer J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38035089' target='_blank'>De Luca et al., 2023, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36788093' target='_blank'>Al-Ghamdi et al., 2023, Mod Pathol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36698617' target='_blank'>Kamphuis et al., Hemasphere, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35902731' target='_blank'>Schönfeld et al., 2022, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34678208' target='_blank'>Olschok et al., Stem Cell Reports, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34320791' target='_blank'>Hage-Sleiman et al., Haematologica, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33202418' target='_blank'>Jia et al., Blood, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32770086' target='_blank'>Chowdhury et al., Leukemia, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29217781' target='_blank'>Ianotto et al., Haematologica, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28464892' target='_blank'>Li et al., J Hematol Oncol, 2017<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39384814' target='_blank'>Garfinkle et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38903535' target='_blank'>Fagnan et al., 2024, Hemasphere<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37716355' target='_blank'>Neault et al., 2023, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37584296' target='_blank'>Jevtic et al., 2023, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35138899' target='_blank'>Benbarche et al., Sci Adv, 2022<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39990340' target='_blank'>Sine et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38825334' target='_blank'>Guo et al., 2024, Aging (Albany NY)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37620309' target='_blank'>Otto et al., 2023, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34830835' target='_blank'>Sonntag et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34177954' target='_blank'>Li et al., Front Immunol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33889830' target='_blank'>Bubie et al., iScience, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32827753' target='_blank'>Mutlu et al., Biochim Biophys Acta Gene Regul Mech, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31657115' target='_blank'>Guo et al., Cancer Med, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31313989' target='_blank'>Guo et al., Aging (Albany NY), 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30150405' target='_blank'>Sonntag et al., Proc Natl Acad Sci U S A, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25557169' target='_blank'>Taylor-Harding et al., Oncotarget, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/33576080' target='_blank'>Yoshida et al., Cancer Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30591657' target='_blank'>Marcuzzi et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28415693' target='_blank'>Klein et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25512583' target='_blank'>Shannon et al., J Exp Med, 2014<\/a>",null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40588529' target='_blank'>Park et al., 2025, Exp Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39113657' target='_blank'>Gonzalez-Gil et al., 2024, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33477674' target='_blank'>Finch et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26707936' target='_blank'>Jena et al., 2015, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25914884' target='_blank'>Aleem et al., Front Cell Dev Biol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25397735' target='_blank'>Brenner et al., Molecules, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24953629' target='_blank'>Bellail et al., Nat Commun, 2014<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38896052' target='_blank'>Sigafoos et al., 2024, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34725129' target='_blank'>Alendar et al., Genes Dev, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30850678' target='_blank'>Machado et al., Sci Rep, 2019<\/a>",null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/31250523' target='_blank'>Brown et al., Pediatr Blood Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27785808' target='_blank'>Cowell et al., Br J Haematol, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39888730' target='_blank'>Shah et al., 2025, Biomarkers<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37978903' target='_blank'>Liu et al., 2023, Cancer Biol Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26216294' target='_blank'>Doebele et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/15794712' target='_blank'>Jones et al., Expert Opin Drug Saf, 2005<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38091012' target='_blank'>Peglion et al., 2023, J Cell Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37565725' target='_blank'>Li et al., 2023, J Med Virol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30232221' target='_blank'>Srinivasan et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26840025' target='_blank'>Mitton et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26743006' target='_blank'>Sunkel et al., Nucleic Acids Res, 2016<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38600987' target='_blank'>Ciocia et al., 2024, NAR Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31526761' target='_blank'>Wurth et al., 2019, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27908735' target='_blank'>Wurth et al., Cancer Cell, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40534701' target='_blank'>Li et al., 2025, Front Med (Lausanne)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37697729' target='_blank'>Song et al., 2023, Cancer Res Treat<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37015324' target='_blank'>Obeidin et al., 2023, J Pathol Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36494342' target='_blank'>Berastegui et al., 2022, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31220736' target='_blank'>Yu et al., Neoplasia, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31143874' target='_blank'>Chen et al., EClinicalMedicine, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30420787' target='_blank'>Bharath et al., J R Stat Soc Ser C Appl Stat, 2018<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39692896' target='_blank'>Zhang et al., 2024, Discov Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35941390' target='_blank'>Chiriches et al., Ann Hematol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32708944' target='_blank'>Pease et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32434859' target='_blank'>Xu et al., Mol Cancer Res, 2020<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39741267' target='_blank'>Yao et al., 2025, J Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38914477' target='_blank'>Honer et al., 2024, Genes Dev<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35307730' target='_blank'>Fu et al., Signal Transduct Target Ther, 2022<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/34210975' target='_blank'>Yang et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29527516' target='_blank'>Sun et al., Front Oncol, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25006183' target='_blank'>Annala et al., EMBO Mol Med, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24319589' target='_blank'>Altman et al., Int J Hematol Oncol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23895657' target='_blank'>Ramakrishnan et al., Epigenomics, 2013<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40672300' target='_blank'>Li et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31023785' target='_blank'>Korfhage et al., Mol Cancer Res, 2019<\/a>",null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/31281685' target='_blank'>Malagola et al., Case Rep Hematol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20959825' target='_blank'>Devy et al., Br J Cancer, 2010<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39302603' target='_blank'>Kratz et al., 2024, Target Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27655244' target='_blank'>Yao et al., BMC Gastroenterol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27593106' target='_blank'>Raggi et al., J Hepatol, 2017<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40678055' target='_blank'>Du et al., 2025, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33240807' target='_blank'>Peng et al., Front Oncol, 2020<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38461240' target='_blank'>Wang et al., 2024, BMC Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37205599' target='_blank'>Menon et al., 2023, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37193069' target='_blank'>Hu et al., 2023, J Inflamm Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35803262' target='_blank'>Povedano et al., Cell Chem Biol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33664993' target='_blank'>Chen et al., Mol Ther Nucleic Acids, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33143299' target='_blank'>Dinhof et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30741932' target='_blank'>Liu et al., Cell Death Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30237855' target='_blank'>Parrish et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30171262' target='_blank'>Lau et al., Oncogene, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28174417' target='_blank'>Dang et al., Leukemia, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27735950' target='_blank'>Javaheri et al., Cell Death Dis, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26258070' target='_blank'>Cidre-Aranaz et al., Front Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26156017' target='_blank'>Song et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24909161' target='_blank'>Li et al., Oncogene, 2015<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/30360388' target='_blank'>Bach et al., Int J Mol Sci, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25502837' target='_blank'>Meng et al., Cancer Res, 2015<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40249915' target='_blank'>Li et al., 2025, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37895004' target='_blank'>Peroni et al., 2023, Int J Mol Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35587378' target='_blank'>Arkoun et al., J Clin Invest, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32764607' target='_blank'>Galat et al., 2020, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32094462' target='_blank'>Lukes et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31925603' target='_blank'>Conneely et al., Cancer Metastasis Rev, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23733339' target='_blank'>Nikolaev et al., Blood, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39690246' target='_blank'>Guo et al., 2024, Cell Death Differ<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29599919' target='_blank'>O'Connor et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25909885' target='_blank'>Bianchi-Smiraglia et al., 2015, Cell Death Differ<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35114133' target='_blank'>Shi et al., Lancet Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31996845' target='_blank'>Yu et al., Nature, 2020<\/a>",null,null,null,null,null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/31485330' target='_blank'>Neeve et al., Clin Transl Immunology, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26485003' target='_blank'>Porta-Pardo et al., PLoS Comput Biol, 2015<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36978140' target='_blank'>Li et al., 2023, Biomark Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33505435' target='_blank'>Stewart et al., Front Genet, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27855356' target='_blank'>Andreozzi et al., Neoplasia, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27561949' target='_blank'>Kim et al., Exp Mol Med, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27001612' target='_blank'>Hillion et al., Gynecol Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22276142' target='_blank'>Belton et al., PLoS One, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19903768' target='_blank'>Hillion et al., Mol Cancer Res, 2009<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19820691' target='_blank'>Hristov et al., Mod Pathol, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38802807' target='_blank'>Najibi et al., 2024, World J Surg Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33374674' target='_blank'>Fatalska et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29109785' target='_blank'>Chang et al., Theranostics, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28288630' target='_blank'>Wang et al., Mol Cancer, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26165842' target='_blank'>Yang et al., Oncogene, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25368430' target='_blank'>Alam et al., Mol Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25014774' target='_blank'>Efanov et al., Blood Cancer J, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24305048' target='_blank'>Kumar et al., Nature, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23536553' target='_blank'>Li et al., Cancer Res, 2013<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40528227' target='_blank'>Cao et al., 2025, J Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40240608' target='_blank'>Hosseini et al., 2025, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38637657' target='_blank'>Lawson et al., 2024, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33468999' target='_blank'>Wang et al., Signal Transduct Target Ther, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33450187' target='_blank'>Wang et al., Cell Stem Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31786140' target='_blank'>Luo et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30270123' target='_blank'>Sun et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28399410' target='_blank'>Mohr et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28202494' target='_blank'>Guitart et al., J Exp Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28049638' target='_blank'>Shafat et al., Blood, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27889185' target='_blank'>Xu et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27641501' target='_blank'>Sykes et al., Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26666754' target='_blank'>Bhatia et al., J Exp Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22305570' target='_blank'>Gibbs et al., Cell Stem Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21741597' target='_blank'>Bernt et al., Cancer Cell, 2011<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39561012' target='_blank'>Kay et al., 2024, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34070799' target='_blank'>Mayoral-Varo et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32998330' target='_blank'>Nguyen et al., Biomedicines, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29928471' target='_blank'>Man et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28894287' target='_blank'>Volpe et al., Sci Rep, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/33738259' target='_blank'>Yin et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26219697' target='_blank'>Butler et al., Mol Cancer Res, 2015<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36696464' target='_blank'>Bonfiglio et al., 2023, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36469010' target='_blank'>Penter et al., 2022, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36287107' target='_blank'>Lai et al., 2022, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33689703' target='_blank'>Lazarian et al., Cancer Cell, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30520556' target='_blank'>Spina et al., Mol Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29386193' target='_blank'>Ozer et al., Cancer Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28584254' target='_blank'>Burns et al., Leukemia, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26466571' target='_blank'>Landau et al., Nature, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34743084' target='_blank'>Bennett et al., Curr Opin Hematol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30988621' target='_blank'>Yang et al., Onco Targets Ther, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26585835' target='_blank'>Wee et al., 2015, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26503059' target='_blank'>Wee et al., 2015, Nat Commun<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40458800' target='_blank'>Hu et al., 2025, Front Pharmacol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31020719' target='_blank'>Dickinson et al., Mol Carcinog, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28946669' target='_blank'>Lee et al., Int J Mol Sci, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25691060' target='_blank'>Park et al., Oncotarget, 2015<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38657045' target='_blank'>Bakiri et al., 2024, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31378787' target='_blank'>Kappelmann-Fenzl et al., Cell Death Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28107482' target='_blank'>Beheshti et al., PLoS One, 2017<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40502048' target='_blank'>Hinz et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38690167' target='_blank'>Dong et al., 2024, Front Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22797671' target='_blank'>Yokota et al., Oncol Rep, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22661970' target='_blank'>Menicali et al., Front Endocrinol (Lausanne), 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22327624' target='_blank'>Kohno et al., Nat Med, 2012<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39814552' target='_blank'>Samarakoon et al., 2025, Life Sci Alliance<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37363966' target='_blank'>Saygin et al., 2023, Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36711753' target='_blank'>Harr et al., Res Sq, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34151288' target='_blank'>Gocho et al., 2021, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33999861' target='_blank'>Liu et al., Aging (Albany NY), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32918044' target='_blank'>Junco et al., 2020, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20876849' target='_blank'>Salmond et al., Blood, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18069898' target='_blank'>Garrett-Engele et al., PLoS Genet, 2007<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36920307' target='_blank'>Abdulla et al., 2023, J Exp Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33483615' target='_blank'>Abdulla et al., 2021, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32422006' target='_blank'>Ahmed et al., 2019, Crit Rev Eukaryot Gene Expr<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32127638' target='_blank'>Horak et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31380372' target='_blank'>Raboso-Gallego et al., 2019, Front Cell Dev Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30962539' target='_blank'>Pais et al., 2019, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30679322' target='_blank'>Goossens et al., 2019, Haematologica<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30166543' target='_blank'>Tremblay et al., 2018, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29880602' target='_blank'>García-Ramírez et al., EMBO J, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28716817' target='_blank'>Efimenko et al., 2017, Mol Cancer Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26598604' target='_blank'>Layer et al., Mol Cell Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25060579' target='_blank'>Yui et al., Nat Rev Immunol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23926305' target='_blank'>McCormack et al., Blood, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23220237' target='_blank'>Joshi et al., Exp Hematol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20676125' target='_blank'>Oram et al., 2010, Oncogene<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36879212' target='_blank'>Zhao et al., 2023, BMC Genomics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34462276' target='_blank'>Liu et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31668663' target='_blank'>Juin et al., Dev Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27259233' target='_blank'>Mitra et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27166196' target='_blank'>Ha et al., 2016, Oncotarget<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25313567' target='_blank'>Muinonen-Martin et al., PLoS Biol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20234370' target='_blank'>Panupinthu et al., Br J Cancer, 2010<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/19440550' target='_blank'>Murph et al., PLoS One, 2009<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40433554' target='_blank'>Delafoy et al., 2025, Hemasphere<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40247105' target='_blank'>Buckley et al., 2025, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32181817' target='_blank'>Li et al., 2020, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32127638' target='_blank'>Horak et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20705338' target='_blank'>Lukov et al., Leuk Res, 2011<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40703447' target='_blank'>Shi et al., 2025, iScience<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39425789' target='_blank'>Li et al., 2024, Ann Hematol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37343247' target='_blank'>Glasheen et al., 2023, Mol Cancer Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33916908' target='_blank'>Kreß et al., Int J Mol Sci, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32634121' target='_blank'>Franklin et al., JCI Insight, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31579105' target='_blank'>Park et al., Ther Adv Med Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31527715' target='_blank'>Parameswaran et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31247859' target='_blank'>Hunsaker et al., Inorg Chem, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29666306' target='_blank'>Hudson et al., Sci Signal, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29437873' target='_blank'>Gampa et al., Drug Metab Dispos, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28263969' target='_blank'>Yang et al., 2017, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27330805' target='_blank'>Sridharan et al., J Immunother Cancer, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26452217' target='_blank'>Kong et al., Oncotarget, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25955301' target='_blank'>Andersen et al., Int J Oncol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22730329' target='_blank'>Yadav et al., J Biol Chem, 2012<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35780183' target='_blank'>Zeng et al., Oncogene, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33807608' target='_blank'>Ahn et al., Int J Mol Sci, 2021<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/31000522' target='_blank'>Hüllein et al., Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27925213' target='_blank'>Xiong et al., J Pathol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26909605' target='_blank'>Swetzig et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25994018' target='_blank'>Ng et al., Genome Biol, 2015<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/33789714' target='_blank'>Sánchez-Del-Campo et al., J Exp Clin Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33276788' target='_blank'>Ballotti et al., Mol Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30898264' target='_blank'>Chatterjee et al., Trends Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30072397' target='_blank'>Huang et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29763622' target='_blank'>Konieczkowski et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29567766' target='_blank'>Ohanna et al., Genes Dev, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29454361' target='_blank'>Sinnberg et al., Mol Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28606996' target='_blank'>Smith et al., EMBO Mol Med, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27857118' target='_blank'>Pérez-Guijarro et al., 2016, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27185926' target='_blank'>Dar et al., Proc Natl Acad Sci U S A, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25848750' target='_blank'>Mann et al., Nat Genet, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25805798' target='_blank'>Goldberg et al., Clin Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25502142' target='_blank'>Müller et al., Nat Commun, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24443471' target='_blank'>Webster et al., Genome Res, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24185007' target='_blank'>Johannessen et al., Nature, 2013<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40624356' target='_blank'>Doggett et al., 2025, EMBO Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39009564' target='_blank'>Song et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38806478' target='_blank'>Edtmayer et al., 2024, Cell Death Dis<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38655899' target='_blank'>Liu et al., 2024, Cancer Discov<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35730653' target='_blank'>Cantilena et al., Clin Transl Med, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35442741' target='_blank'>Yuan et al., Sci Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32179751' target='_blank'>Basilico et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31405949' target='_blank'>Okeyo-Owuor et al., Blood Adv, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30302772' target='_blank'>Folkerts et al., Med Res Rev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29590011' target='_blank'>Stanton et al., Science, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27625305' target='_blank'>Morgado-Palacin et al., Sci Signal, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27607576' target='_blank'>Liu et al., Cell Death Dis, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27373927' target='_blank'>Velasco-Hernandez et al., Stem Cell Reports, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26728554' target='_blank'>Shields et al., Genes Dev, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25079327' target='_blank'>Santos et al., Nature, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/38798450' target='_blank'>Ge et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37147424' target='_blank'>Matsukawa et al., 2023, Leukemia<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35070954' target='_blank'>Fertal et al., Front Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34440253' target='_blank'>Moser et al., Biomedicines, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34226546' target='_blank'>Klein et al., Nat Commun, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33690798' target='_blank'>Chen et al., Blood, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31022428' target='_blank'>Barbosa et al., Exp Hematol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23487024' target='_blank'>Conway et al., Blood, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23138183' target='_blank'>Chen et al., Leukemia, 2013<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/36179686' target='_blank'>Chan et al., 2022, Cancer Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31821784' target='_blank'>Krivtsov et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31511663' target='_blank'>Bushweller et al., Nat Rev Cancer, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31408619' target='_blank'>Fong et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30745579' target='_blank'>Pinho et al., Nat Rev Mol Cell Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30423295' target='_blank'>Celik et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30146162' target='_blank'>Minzel et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29894694' target='_blank'>Greenblatt et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28609655' target='_blank'>Chen et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27344946' target='_blank'>Stavropoulou et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27070701' target='_blank'>Brien et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25822366' target='_blank'>Chen et al., Nat Med, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25242043' target='_blank'>Schneider et al., Cancer Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25215489' target='_blank'>Wang et al., Cell, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21741597' target='_blank'>Bernt et al., Cancer Cell, 2011<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37439113' target='_blank'>Meyer et al., 2023, Autophagy<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37058562' target='_blank'>Weinhäuser et al., 2023, Sci Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33542482' target='_blank'>Libbrecht et al., Leukemia, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31332244' target='_blank'>Pieters et al., Sci Rep, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28892045' target='_blank'>Larmonie et al., Oncogene, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28375985' target='_blank'>Yang et al., Curr Opin Hematol, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28174417' target='_blank'>Dang et al., Leukemia, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40435410' target='_blank'>Mayday et al., 2025, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40060447' target='_blank'>Mayday et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37325571' target='_blank'>Li et al., 2023, Front Cell Dev Biol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36577757' target='_blank'>Konopa et al., Oncogenesis, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35766837' target='_blank'>Guo et al., 2022, Compr Physiol<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39499902' target='_blank'>Booth et al., 2024, JCI Insight<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38835346' target='_blank'>Biersack et al., 2024, Cancer Drug Resist<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38463444' target='_blank'>Almeida et al., 2024, Hemasphere<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35020834' target='_blank'>Pieters et al., Blood Adv, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30146162' target='_blank'>Minzel et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29739933' target='_blank'>Geyer et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29500329' target='_blank'>Natrajan et al., Cancer Discov, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28671688' target='_blank'>Liu et al., 2017, Nat Genet<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27973981' target='_blank'>Jain et al., Cell Cycle, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27707899' target='_blank'>Uttarkar et al., Mol Cancer Ther, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27213588' target='_blank'>Mikse et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26969893' target='_blank'>Rajan et al., J Pathol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26851182' target='_blank'>Gonda et al., Cancer Discov, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26657649' target='_blank'>Srivastava et al., Br J Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25671128' target='_blank'>Cross et al., Clin Transl Immunology, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39858036' target='_blank'>Das et al., 2025, Cancers (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36643868' target='_blank'>Valentine et al., 2023, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34344693' target='_blank'>Ciampricotti et al., Cancer Discov, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32470392' target='_blank'>Augert et al., Cancer Cell, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31164374' target='_blank'>Chalishazar et al., Clin Cancer Res, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29697361' target='_blank'>Kim et al., BMC Med Genomics, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28089889' target='_blank'>Mollaoglu et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26829312' target='_blank'>Bunn et al., J Thorac Oncol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24401838' target='_blank'>Huijbers et al., EMBO Mol Med, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/2668739' target='_blank'>Birrer et al., Mol Cell Biol, 1989<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39962891' target='_blank'>Zhang et al., 2025, Technol Cancer Res Treat<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37992684' target='_blank'>Popescu et al., 2023, Cell Rep Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37267435' target='_blank'>Schulz et al., 2023, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35756660' target='_blank'>Kurtz et al., Front Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33080932' target='_blank'>Lamba et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29958106' target='_blank'>Pulikkan et al., Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27211267' target='_blank'>Mitton et al., Leukemia, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25678665' target='_blank'>Illendula et al., Science, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18796164' target='_blank'>Alhopuro et al., BMC Cancer, 2008<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38344872' target='_blank'>Mendiburu-Eliçabe et al., 2024, Clin Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37886490' target='_blank'>Mendiburu-Eliçabe et al., 2023, Res Sq<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39366066' target='_blank'>Liu et al., 2024, Drug Resist Updat<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38049396' target='_blank'>Lin et al., 2023, Cell Death Dis<\/a>",null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/27756303' target='_blank'>Hnízda et al., BMC Biol, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27211266' target='_blank'>Bashford-Rogers et al., Leukemia, 2016<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39711373' target='_blank'>Zhu et al., 2024, Ann Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37743366' target='_blank'>Miyazaki et al., 2023, Nat Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33768871' target='_blank'>Britton et al., Clin Case Rep, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33572160' target='_blank'>Samaržija et al., Biomolecules, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29581842' target='_blank'>Gomez et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29059151' target='_blank'>Abbaci et al., Oncogene, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27402815' target='_blank'>Ni et al., Neuro Oncol, 2017<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/32934780' target='_blank'>Mendes et al., Oncotarget, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30107177' target='_blank'>Vanden et al., Cancer Cell, 2018<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39609428' target='_blank'>Myers et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37507199' target='_blank'>Mikheeva et al., 2023, Mol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37274223' target='_blank'>Szu et al., 2023, Front Neurosci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32383980' target='_blank'>Gevaert et al., JCO Clin Cancer Inform, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27165742' target='_blank'>Lu et al., Cancer Cell, 2016<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40428399' target='_blank'>Moore et al., 2025, Genes (Basel)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37094128' target='_blank'>Mota et al., 2023, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36696581' target='_blank'>Shackleford et al., 2023, Mol Cancer Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33526511' target='_blank'>Huang et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32943730' target='_blank'>Chen et al., Oncogene, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32251397' target='_blank'>Fazio et al., Nat Rev Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31031007' target='_blank'>Yan et al., Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27864345' target='_blank'>Lacey et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25531327' target='_blank'>Kubic et al., Oncogene, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39747430' target='_blank'>DeGolier et al., 2025, Nat Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38196657' target='_blank'>DeGolier et al., 2024, Res Sq<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34678207' target='_blank'>Muggeo et al., Stem Cell Reports, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33542097' target='_blank'>Lee et al., Proc Natl Acad Sci U S A, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30824532' target='_blank'>Selleri et al., Genes Dev, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28824851' target='_blank'>Perna et al., 2017, Transl Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27590521' target='_blank'>Lindqvist et al., 2016, Oncotarget<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27460500' target='_blank'>Jacoby et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26237075' target='_blank'>Gabriel et al., Epigenetics, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26215677' target='_blank'>Magnani et al., 2015, Oncotarget<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26098938' target='_blank'>Woodcroft et al., PLoS One, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25759017' target='_blank'>Trimarchi et al., Cancer Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24847444' target='_blank'>Jacoby et al., Front Oncol, 2014<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/37966120' target='_blank'>Miller et al., 2023, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37739938' target='_blank'>Henikoff et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37733448' target='_blank'>Chen et al., 2023, J Clin Invest<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37012245' target='_blank'>Chen et al., 2023, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36805975' target='_blank'>Unknown et al., Neuro Oncol, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36594465' target='_blank'>Friedmann-Morvinski et al., J Clin Invest, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35738865' target='_blank'>Weishaupt et al., Neuro Oncol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34562087' target='_blank'>Rao et al., Neuro Oncol, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33020472' target='_blank'>De et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28695888' target='_blank'>Cook et al., Nat Commun, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28235764' target='_blank'>Chen et al., Cancer Res, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27088132' target='_blank'>Hambardzumyan et al., Trends Cancer, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26282165' target='_blank'>Lau et al., Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26216294' target='_blank'>Doebele et al., Cancer Discov, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24962027' target='_blank'>Jeon et al., Cancer Res, 2014<\/a>",null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38994678' target='_blank'>McDaniel et al., 2024, Cancer Res Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24562770' target='_blank'>Wilson-Edell et al., Breast Cancer Res Treat, 2014<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/30200332' target='_blank'>Barr et al., Children (Basel), 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24599217' target='_blank'>Marshall et al., Nat Rev Cancer, 2014<\/a>",null,null,null,null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34079088' target='_blank'>Nathansen et al., Oncogene, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28454439' target='_blank'>Sun et al., Oncol Lett, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27765847' target='_blank'>Lee et al., Mol Cancer Ther, 2016<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38888469' target='_blank'>Shirani et al., 2024, Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37980543' target='_blank'>Neumayer et al., 2023, Mol Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29697361' target='_blank'>Kim et al., BMC Med Genomics, 2018<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40435410' target='_blank'>Mayday et al., 2025, Blood<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40060447' target='_blank'>Mayday et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39488447' target='_blank'>Wang et al., 2024, Clin Breast Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39406308' target='_blank'>Shi et al., 2024, Life Sci<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37325571' target='_blank'>Li et al., 2023, Front Cell Dev Biol<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39975057' target='_blank'>Bakırdöğen et al., 2025, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25450861' target='_blank'>Frasor et al., Mol Cell Endocrinol, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18089804' target='_blank'>Belguise et al., Cancer Res, 2007<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39901723' target='_blank'>de Sousa et al., 2025, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30553016' target='_blank'>Biswas et al., Biochim Biophys Acta Mol Basis Dis, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28041912' target='_blank'>Jeong et al., Mol Cell, 2017<\/a>",null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/35422482' target='_blank'>Martens et al., Cell Death Dis, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32752206' target='_blank'>Sprooten et al., Cells, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32269263' target='_blank'>Laurien et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26959742' target='_blank'>Liu et al., Oncotarget, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25724678' target='_blank'>Liu et al., Cancer Res, 2015<\/a>",null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40777043' target='_blank'>Zhang et al., 2025, Front Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37998393' target='_blank'>Mehdawi et al., 2023, Cells<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37932819' target='_blank'>Wu et al., 2023, Clin Epigenetics<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37406727' target='_blank'>Ter Steege et al., 2023, Cancer Lett<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36106661' target='_blank'>Ter et al., J Pathol, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31904401' target='_blank'>Oldani et al., Toxicol In Vitro, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27511199' target='_blank'>Hilkens et al., Gut, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26257057' target='_blank'>Madan et al., Oncogene, 2016<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38710701' target='_blank'>Acton et al., 2024, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32576829' target='_blank'>Morris et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30520556' target='_blank'>Spina et al., Mol Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28584254' target='_blank'>Burns et al., Leukemia, 2018<\/a>",null,null,null,null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36052485' target='_blank'>Li et al., 2022, Mol Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34016745' target='_blank'>Li et al., 2021, Mol Cancer Res<\/a>",null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39779467' target='_blank'>Lei et al., 2025, J Cell Mol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33167358' target='_blank'>Dunn et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28885261' target='_blank'>Wojcik et al., Adv Anat Pathol, 2017<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40016470' target='_blank'>Powell et al., 2025, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28105457' target='_blank'>Marquez-Vilendrer et al., Oncoscience, 2016<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39577862' target='_blank'>Sahm et al., 2024, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38883782' target='_blank'>Floros et al., 2024, Res Sq<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36633435' target='_blank'>Chen et al., 2023, Nucleus<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40327401' target='_blank'>Sengottuvel et al., 2025, JCI Insight<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/40301302' target='_blank'>Korleski et al., 2025, Signal Transduct Target Ther<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33558335' target='_blank'>Davenport et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33203881' target='_blank'>Pietrobono et al., Nat Commun, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32055024' target='_blank'>Luiken et al., Oncogene, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31968252' target='_blank'>Liu et al., Cell Rep, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31378680' target='_blank'>Yin et al., Cancer Cell, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31201181' target='_blank'>Gimple et al., Cancer Discov, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29849070' target='_blank'>Sánchez-Danés et al., Nat Rev Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28504716' target='_blank'>Kim et al., Oncogene, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27728803' target='_blank'>Ferone et al., Cancer Cell, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26898758' target='_blank'>Lopez-Bertoni et al., Oncogene, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26206333' target='_blank'>Xu et al., Mol Cancer Ther, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25823023' target='_blank'>Cheung et al., Oncogene, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24954133' target='_blank'>Vanner et al., Cancer Cell, 2014<\/a>",null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/38755258' target='_blank'>Moye et al., 2024, EMBO J<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38507464' target='_blank'>Álvarez-Vázquez et al., 2024, Neuro Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36795481' target='_blank'>Nandi et al., J Clin Invest, 2023<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34006524' target='_blank'>Deneka et al., Cancer Res, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33020459' target='_blank'>Torrisi et al., Cancers (Basel), 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31805962' target='_blank'>Lang et al., J Hematol Oncol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30622697' target='_blank'>Picco et al., Cell Biosci, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30002479' target='_blank'>Hamidi et al., Nat Rev Cancer, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29422647' target='_blank'>Wang et al., Nat Commun, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26899176' target='_blank'>Sancho-Martinez et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25772245' target='_blank'>Spring et al., Oncogene, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21135112' target='_blank'>Cai et al., Cancer Res, 2011<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/18679417' target='_blank'>Chang et al., 2008, Oncogene<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/17700528' target='_blank'>Srinivasan et al., Oncogene, 2008<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/14977859' target='_blank'>Boyd et al., Clin Cancer Res, 2004<\/a>",null,null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/36287107' target='_blank'>Lai et al., 2022, Blood Adv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35868661' target='_blank'>Aguirre-Ducler et al., J Immunother Cancer, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34359655' target='_blank'>Barabino et al., Cancers (Basel), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32414848' target='_blank'>Loftus et al., Haematologica, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32015510' target='_blank'>Yang et al., Br J Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32015509' target='_blank'>Cremer et al., Br J Cancer, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32005797' target='_blank'>Munshi et al., Blood Cancer J, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28881711' target='_blank'>Weisberg et al., Oncotarget, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24564859' target='_blank'>Udyavar et al., BMC Syst Biol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23398911' target='_blank'>Hatton et al., Am J Transplant, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21908615' target='_blank'>Hatton et al., J Biol Chem, 2011<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39747430' target='_blank'>DeGolier et al., 2025, Nat Immunol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38196657' target='_blank'>DeGolier et al., 2024, Res Sq<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37555980' target='_blank'>Quintanilla-Martinez et al., 2023, Virchows Arch<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36336953' target='_blank'>Tyryshkin et al., Leuk Lymphoma, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35981544' target='_blank'>Groves et al., 2022, Cell Syst<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33612479' target='_blank'>Huang et al., Aging (Albany NY), 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32409685' target='_blank'>Wei et al., Cell Death Dis, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28824851' target='_blank'>Perna et al., 2017, Transl Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27590521' target='_blank'>Lindqvist et al., 2016, Oncotarget<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27460500' target='_blank'>Jacoby et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25759017' target='_blank'>Trimarchi et al., Cancer Cell, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39697983' target='_blank'>Willemsen et al., 2024, Immunooncol Technol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29784854' target='_blank'>Saito et al., Cancer Res, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28089356' target='_blank'>Imielinski et al., 2017, Cell<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27428425' target='_blank'>Sinnberg et al., EBioMedicine, 2016<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/40189703' target='_blank'>Custode et al., 2025, Cell Death Differ<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39915638' target='_blank'>Li et al., 2025, Nat Metab<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39502403' target='_blank'>Mondaza-Hernandez et al., 2024, Ther Adv Med Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39484513' target='_blank'>Li et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39149323' target='_blank'>Li et al., 2024, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39059513' target='_blank'>Colino-Lage et al., 2024, Free Radic Biol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37577497' target='_blank'>Achom et al., 2023, bioRxiv<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37131332' target='_blank'>Malik et al., 2023, Int J Surg Pathol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37015324' target='_blank'>Obeidin et al., 2023, J Pathol Transl Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36357396' target='_blank'>Asrani et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34561551' target='_blank'>Chiang et al., 2021, Mod Pathol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33459596' target='_blank'>Glykofridis et al., Elife, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31650096' target='_blank'>Perera et al., Annu Rev Cancer Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30920708' target='_blank'>Suurmeijer et al., Genes Chromosomes Cancer, 2019<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/39059513' target='_blank'>Colino-Lage et al., 2024, Free Radic Biol Med<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/38724668' target='_blank'>Xiao et al., 2024, Nat Biotechnol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36357396' target='_blank'>Asrani et al., Nat Commun, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33292395' target='_blank'>Xu et al., Chin Med, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27668431' target='_blank'>Calcagnì et al., Elife, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21804531' target='_blank'>Peña-Llopis et al., 2011, EMBO J<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/25301630' target='_blank'>de et al., Science, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/20500843' target='_blank'>Pasche et al., J Exp Clin Cancer Res, 2010<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40458800' target='_blank'>Hu et al., 2025, Front Pharmacol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39915884' target='_blank'>He et al., 2025, J Exp Clin Cancer Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39844891' target='_blank'>Mohamed et al., 2025, Noncoding RNA Res<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36162136' target='_blank'>Kwan et al., 2022, Cancer Prev Res (Phila)<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34528710' target='_blank'>Rogava et al., Int J Cancer, 2021<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/33360689' target='_blank'>Jandova et al., Redox Biol, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31068809' target='_blank'>Ding et al., Front Pharmacol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30034633' target='_blank'>Nguyen et al., Oncotarget, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/29602239' target='_blank'>Zheng et al., J Cell Mol Med, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26063617' target='_blank'>Haricharan et al., Proc Natl Acad Sci U S A, 2015<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/40579412' target='_blank'>Costea et al., 2025, Nat Commun<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36653435' target='_blank'>Liebig et al., 2023, Sci Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36650499' target='_blank'>Smith et al., 2023, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32591643' target='_blank'>Loontiens et al., Leukemia, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30107177' target='_blank'>Vanden et al., Cancer Cell, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26108691' target='_blank'>Durinck et al., Leukemia, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/23695450' target='_blank'>Haydu et al., Curr Opin Hematol, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/21326611' target='_blank'>Rakowski et al., PLoS One, 2011<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pubmed/37060567' target='_blank'>Smits et al., 2023, Cell Rep<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/36650499' target='_blank'>Smith et al., 2023, Mol Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30575306' target='_blank'>Haider et al., Cancer Med, 2019<\/a>",null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39663641' target='_blank'>McGrath et al., 2024, Int J Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39190349' target='_blank'>Su et al., 2024, Proc Natl Acad Sci U S A<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/35452553' target='_blank'>Raina et al., Mol Carcinog, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/34543404' target='_blank'>Clarke et al., Endocrinology, 2022<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31702856' target='_blank'>Wheeler et al., Angew Chem Int Ed Engl, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31639029' target='_blank'>Haas et al., Genome Biol, 2019<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30865299' target='_blank'>Kessler et al., 2019, Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30150711' target='_blank'>Xiao et al., Sci Rep, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/28344039' target='_blank'>Wang et al., Cancer Cell, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25266362' target='_blank'>He et al., Mol Oncol, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24759320' target='_blank'>Asangani et al., Nature, 2014<\/a>",null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/34405474' target='_blank'>Atkin et al., 2021, Genes Chromosomes Cancer<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/32201520' target='_blank'>Kou et al., J Cancer, 2020<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,"<a href='https://www.ncbi.nlm.nih.gov/pubmed/39880930' target='_blank'>Venkateswaran et al., 2025, Nat Protoc<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/39633046' target='_blank'>Bornes et al., 2024, Nature<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37946084' target='_blank'>Kreuzaler et al., 2023, Nat Metab<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/37749625' target='_blank'>Tan et al., 2023, Exp Hematol Oncol<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/31744816' target='_blank'>Matthew-Onabanjo et al., Cancer Res, 2020<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/30274786' target='_blank'>Ganesh et al., Mol Ther, 2018<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27782124' target='_blank'>Jardé et al., Nat Commun, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27617575' target='_blank'>Zhan et al., Oncogene, 2017<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/27197172' target='_blank'>Yeo et al., Cancer Res, 2016<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26581390' target='_blank'>Holdman et al., Breast Cancer Res, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/26091037' target='_blank'>Dow et al., Cell, 2015<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/25066122' target='_blank'>Chapman et al., Cell Rep, 2014<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/24277854' target='_blank'>Liu et al., Proc Natl Acad Sci U S A, 2013<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22439924' target='_blank'>Magee et al., Cancer Cell, 2012<\/a>, <a href='https://www.ncbi.nlm.nih.gov/pubmed/22086849' target='_blank'>Judd et al., Cancer Res, 2012<\/a>",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],["This gene is a protooncogene that encodes a protein tyrosine kinase involved in a variety of cellular processes, including cell division, adhesion, differentiation, and response to stress. The activity of the protein is negatively regulated by its SH3 domain, whereby deletion of the region encoding this domain results in an oncogene. The ubiquitously expressed protein has DNA-binding activity that is regulated by CDC2-mediated phosphorylation, suggesting a cell cycle function. This gene has been found fused to a variety of translocation partner genes in various leukemias, most notably the t(9;22) translocation that results in a fusion with the 5' end of the breakpoint cluster region gene (BCR; MIM:151410). Alternative splicing of this gene results in two transcript variants, which contain alternative first exons that are spliced to the remaining common exons.","Activins are dimeric growth and differentiation factors which belong to the transforming growth factor-beta (TGF-beta) superfamily of structurally related signaling proteins. Activins signal through a heteromeric complex of receptor serine kinases which include at least two type I ( I and IB) and two type II (II and IIB) receptors. These receptors are all transmembrane proteins, composed of a ligand-binding extracellular domain with cysteine-rich region, a transmembrane domain, and a cytoplasmic domain with predicted serine/threonine specificity. Type I receptors are essential for signaling; and type II receptors are required for binding ligands and for expression of type I receptors. Type I and II receptors form a stable complex after ligand binding, resulting in phosphorylation of type I receptors by type II receptors. This gene encodes activin A type I receptor which signals a particular transcriptional response in concert with activin type II receptors. Mutations in this gene are associated with fibrodysplasia ossificans progressive.","This gene encodes an activin A type IB receptor. Activins are dimeric growth and differentiation factors which belong to the transforming growth factor-beta (TGF-beta) superfamily of structurally related signaling proteins. Activins signal through a heteromeric complex of receptor serine kinases which include at least two type I and two type II receptors. This protein is a type I receptor which is essential for signaling. Mutations in this gene are associated with pituitary tumors. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Jun 2010].","This gene encodes a receptor that mediates the functions of activins, which are members of the transforming growth factor-beta (TGF-beta) superfamily involved in diverse biological processes. The encoded protein is a transmembrane serine-threonine kinase receptor which mediates signaling by forming heterodimeric complexes with various combinations of type I and type II receptors and ligands in a cell-specific manner. The encoded type II receptor is primarily involved in ligand-binding and includes an extracellular ligand-binding domain, a transmembrane domain and a cytoplasmic serine-threonine kinase domain. This gene may be associated with susceptibility to preeclampsia, a pregnancy-related disease which can result in maternal and fetal morbidity and mortality. Alternative splicing results in multiple transcript variants of this gene.","This gene encodes one of the three members of the human AKT serine-threonine protein kinase family which are often referred to as protein kinase B alpha, beta, and gamma. These highly similar AKT proteins all have an N-terminal pleckstrin homology domain, a serine/threonine-specific kinase domain and a C-terminal regulatory domain. These proteins are phosphorylated by phosphoinositide 3-kinase (PI3K). AKT/PI3K forms a key component of many signalling pathways that involve the binding of membrane-bound ligands such as receptor tyrosine kinases, G-protein coupled receptors, and integrin-linked kinase. These AKT proteins therefore regulate a wide variety of cellular functions including cell proliferation, survival, metabolism, and angiogenesis in both normal and malignant cells. AKT proteins are recruited to the cell membrane by phosphatidylinositol 3,4,5-trisphosphate (PIP3) after phosphorylation of phosphatidylinositol 4,5-bisphosphate (PIP2) by PI3K. Subsequent phosphorylation of both threonine residue 308 and serine residue 473 is required for full activation of the AKT1 protein encoded by this gene. Phosphorylation of additional residues also occurs, for example, in response to insulin growth factor-1 and epidermal growth factor. Protein phosphatases act as negative regulators of AKT proteins by dephosphorylating AKT or PIP3. The PI3K/AKT signalling pathway is crucial for tumor cell survival. Survival factors can suppress apoptosis in a transcription-independent manner by activating AKT1 which then phosphorylates and inactivates components of the apoptotic machinery. AKT proteins also participate in the mammalian target of rapamycin (mTOR) signalling pathway which controls the assembly of the eukaryotic translation initiation factor 4F (eIF4E) complex and this pathway, in addition to responding to extracellular signals from growth factors and cytokines, is disregulated in many cancers. Mutations in this gene are associated with multiple types of cancer and excessive tissue growth including Proteus syndrome and Cowden syndrome 6, and breast, colorectal, and ovarian cancers. Multiple alternatively spliced transcript variants have been found for this gene.","This gene encodes a receptor tyrosine kinase, which belongs to the insulin receptor superfamily. This protein comprises an extracellular domain, an hydrophobic stretch corresponding to a single pass transmembrane region, and an intracellular kinase domain. It plays an important role in the development of the brain and exerts its effects on specific neurons in the nervous system. This gene has been found to be rearranged, mutated, or amplified in a series of tumours including anaplastic large cell lymphomas, neuroblastoma, and non-small cell lung cancer. The chromosomal rearrangements are the most common genetic alterations in this gene, which result in creation of multiple fusion genes in tumourigenesis, including ALK (chromosome 2)/EML4 (chromosome 2), ALK/RANBP2 (chromosome 2), ALK/ATIC (chromosome 2), ALK/TFG (chromosome 3), ALK/NPM1 (chromosome 5), ALK/SQSTM1 (chromosome 5), ALK/KIF5B (chromosome 10), ALK/CLTC (chromosome 17), ALK/TPM4 (chromosome 19), and ALK/MSN (chromosome X).[provided by RefSeq, Jan 2011].","The protein encoded by this gene upregulates trancriptional activation by the Wilms tumor protein and interacts with many other proteins, including CTNNB1, APC, AXIN1, and AXIN2. Defects in this gene are a cause of osteopathia striata with cranial sclerosis (OSCS).","This gene encodes a tumor suppressor protein that acts as an antagonist of the Wnt signaling pathway. It is also involved in other processes including cell migration and adhesion, transcriptional activation, and apoptosis. Defects in this gene cause familial adenomatous polyposis (FAP), an autosomal dominant pre-malignant disease that usually progresses to malignancy. Mutations in the APC gene have been found to occur in most colorectal cancers, where disease-associated mutations tend to be clustered in a small region designated the mutation cluster region (MCR) and result in a truncated protein product.","The androgen receptor gene is more than 90 kb long and codes for a protein that has 3 major functional domains: the N-terminal domain, DNA-binding domain, and androgen-binding domain. The protein functions as a steroid-hormone activated transcription factor. Upon binding the hormone ligand, the receptor dissociates from accessory proteins, translocates into the nucleus, dimerizes, and then stimulates transcription of androgen responsive genes. This gene contains 2 polymorphic trinucleotide repeat segments that encode polyglutamine and polyglycine tracts in the N-terminal transactivation domain of its protein. Expansion of the polyglutamine tract from the normal 9-34 repeats to the pathogenic 38-62 repeats causes spinal bulbar muscular atrophy (SBMA, also known as Kennedy's disease). Mutations in this gene are also associated with complete androgen insensitivity (CAIS). Alternative splicing results in multiple transcript variants encoding different isoforms.","This proto-oncogene belongs to the RAF subfamily of the Ser/Thr protein kinase family, and maybe involved in cell growth and development. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jan 2012]","This gene encodes a member of the SWI/SNF family, whose members have helicase and ATPase activities and are thought to regulate transcription of certain genes by altering the chromatin structure around those genes. The encoded protein is part of the large ATP-dependent chromatin remodeling complex SNF/SWI, which is required for transcriptional activation of genes normally repressed by chromatin. It possesses at least two conserved domains that could be important for its function. First, it has a DNA-binding domain that can specifically bind an AT-rich DNA sequence known to be recognized by a SNF/SWI complex at the beta-globin locus. Second, the C-terminus of the protein can stimulate glucocorticoid receptor-dependent transcriptional activation. It is thought that the protein encoded by this gene confers specificity to the SNF/SWI complex and may recruit the complex to its targets through either protein-DNA or protein-protein interactions. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the AT-rich interactive domain (ARID)-containing family of DNA-binding proteins. Members of the ARID family have roles in embryonic patterning, cell lineage gene regulation, cell cycle control, transcriptional regulation and chromatin structure modification. This protein functions as a subunit of the polybromo- and BRG1-associated factor or PBAF (SWI/SNF-B) chromatin remodeling complex which facilitates ligand-dependent transcriptional activation by nuclear receptors. Mutations in this gene are associated with hepatocellular carcinomas. A pseudogene of this gene is found on chromosome1.","This gene is similar to the Drosophila additional sex combs gene, which encodes a chromatin-binding protein required for normal determination of segment identity in the developing embryo. The protein is a member of the Polycomb group of proteins, which are necessary for the maintenance of stable repression of homeotic and other loci. The protein is thought to disrupt chromatin in localized areas, enhancing transcription of certain genes while repressing the transcription of other genes. The protein encoded by this gene functions as a ligand-dependent co-activator for retinoic acid receptor in cooperation with nuclear receptor coactivator 1. Mutations in this gene are associated with myelodysplastic syndromes and chronic myelomonocytic leukemia. Alternative splicing results in multiple transcript variants.","This gene encodes a member of a family of epigenetic regulators that bind various histone-modifying enzymes and are involved in the assembly of transcription factors at specific genomic loci. Naturally occurring mutations in this gene are associated with cancer in several tissue types (breast, bladder, pancreas, ovary, prostate, and blood). This gene plays an important role in neurodevelopment, cardiac function, adipogenesis, and osteoclastogenesis.","The protein encoded by this gene belongs to the PI3/PI4-kinase family. This protein is an important cell cycle checkpoint kinase that phosphorylates; thus, it functions as a regulator of a wide variety of downstream proteins, including tumor suppressor proteins p53 and BRCA1, checkpoint kinase CHK2, checkpoint proteins RAD17 and RAD9, and DNA repair protein NBS1. This protein and the closely related kinase ATR are thought to be master controllers of cell cycle checkpoint signaling pathways that are required for cell response to DNA damage and for genome stability. Mutations in this gene are associated with ataxia telangiectasia, an autosomal recessive disorder.","The protein encoded by this gene is a serine/threonine kinase and DNA damage sensor, activating cell cycle checkpoint signaling upon DNA stress. The encoded protein can phosphorylate and activate several proteins involved in the inhibition of DNA replication and mitosis, and can promote DNA repair, recombination, and apoptosis. This protein is also important for fragile site stability and centrosome duplication. Defects in this gene are a cause of Seckel syndrome 1.","The protein encoded by this gene contains an ATPase/helicase domain, and thus it belongs to the SWI/SNF family of chromatin remodeling proteins. This protein is found to undergo cell cycle-dependent phosphorylation, which regulates its nuclear matrix and chromatin association, and suggests its involvement in the gene regulation at interphase and chromosomal segregation in mitosis. Mutations in this gene are associated with X-linked syndromes exhibiting cognitive disabilities as well as alpha-thalassemia (ATRX) syndrome. These mutations have been shown to cause diverse changes in the pattern of DNA methylation, which may provide a link between chromatin remodeling, DNA methylation, and gene expression in developmental processes. Multiple alternatively spliced transcript variants encoding distinct isoforms have been reported.","This gene encodes a cytoplasmic protein which contains a regulation of G-protein signaling (RGS) domain and a dishevelled and axin (DIX) domain. The encoded protein interacts with adenomatosis polyposis coli, catenin beta-1, glycogen synthase kinase 3 beta, protein phosphate 2, and itself. This protein functions as a negative regulator of the wingless-type MMTV integration site family, member 1 (WNT) signaling pathway and can induce apoptosis. The crystal structure of a portion of this protein, alone and in a complex with other proteins, has been resolved. Mutations in this gene have been associated with hepatocellular carcinoma, hepatoblastomas, ovarian endometriod adenocarcinomas, and medullablastomas. Alternative splicing results in multiple transcript variants.","This gene encodes a serum protein found in association with the major histocompatibility complex (MHC) class I heavy chain on the surface of nearly all nucleated cells. The protein has a predominantly beta-pleated sheet structure that can form amyloid fibrils in some pathological conditions. The encoded antimicrobial protein displays antibacterial activity in amniotic fluid. A mutation in this gene has been shown to result in hypercatabolic hypoproteinemia.[provided by RefSeq, Aug 2014].","This gene belongs to the ubiquitin C-terminal hydrolase subfamily of deubiquitinating enzymes that are involved in the removal of ubiquitin from proteins. The encoded enzyme binds to the breast cancer type 1 susceptibility protein (BRCA1) via the RING finger domain of the latter and acts as a tumor suppressor. In addition, the enzyme may be involved in regulation of transcription, regulation of cell cycle and growth, response to DNA damage and chromatin dynamics. Germline mutations in this gene may be associated with tumor predisposition syndrome (TPDS), which involves increased risk of cancers including malignant mesothelioma, uveal melanoma and cutaneous melanoma.","This gene encodes an integral outer mitochondrial membrane protein that blocks the apoptotic death of some cells such as lymphocytes. Constitutive expression of BCL2, such as in the case of translocation of BCL2 to Ig heavy chain locus, is thought to be the cause of follicular lymphoma. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene was identified as an interacting corepressor of BCL6, a POZ/zinc finger transcription repressor that is required for germinal center formation and may influence apoptosis. This protein selectively interacts with the POZ domain of BCL6, but not with eight other POZ proteins. Specific class I and II histone deacetylases (HDACs) have been shown to interact with this protein, which suggests a possible link between the two classes of HDACs. Several transcript variants encoding different isoforms have been found for this gene. A pseudogene of this gene is found on chromosome Y.[provided by RefSeq, Jun 2010].","This gene encodes a protein belonging to the RAF family of serine/threonine protein kinases. This protein plays a role in regulating the MAP kinase/ERK signaling pathway, which affects cell division, differentiation, and secretion. Mutations in this gene, most commonly the V600E mutation, are the most frequently identified cancer-causing mutations in melanoma, and have been identified in various other cancers as well, including non-Hodgkin lymphoma, colorectal cancer, thyroid carcinoma, non-small cell lung carcinoma, hairy cell leukemia and adenocarcinoma of lung. Mutations in this gene are also associated with cardiofaciocutaneous, Noonan, and Costello syndromes, which exhibit overlapping phenotypes. A pseudogene of this gene has been identified on the X chromosome.","This gene encodes a 190 kD nuclear phosphoprotein that plays a role in maintaining genomic stability, and it also acts as a tumor suppressor. The BRCA1 gene contains 22 exons spanning about 110 kb of DNA. The encoded protein combines with other tumor suppressors, DNA damage sensors, and signal transducers to form a large multi-subunit protein complex known as the BRCA1-associated genome surveillance complex (BASC). This gene product associates with RNA polymerase II, and through the C-terminal domain, also interacts with histone deacetylase complexes. This protein thus plays a role in transcription, DNA repair of double-stranded breaks, and recombination. Mutations in this gene are responsible for approximately 40% of inherited breast cancers and more than 80% of inherited breast and ovarian cancers. Alternative splicing plays a role in modulating the subcellular localization and physiological function of this gene. Many alternatively spliced transcript variants, some of which are disease-associated mutations, have been described for this gene, but the full-length natures of only some of these variants has been described. A related pseudogene, which is also located on chromosome 17, has been identified.","Inherited mutations in BRCA1 and this gene, BRCA2, confer increased lifetime risk of developing breast or ovarian cancer. Both BRCA1 and BRCA2 are involved in maintenance of genome stability, specifically the homologous recombination pathway for double-strand DNA repair. The largest exon in both genes is exon 11, which harbors the most important and frequent mutations in breast cancer patients. The BRCA2 gene was found on chromosome 13q12.3 in human. The BRCA2 protein contains several copies of a 70 aa motif called the BRC motif, and these motifs mediate binding to the RAD51 recombinase which functions in DNA repair. BRCA2 is considered a tumor suppressor gene, as tumors with BRCA2 mutations generally exhibit loss of heterozygosity (LOH) of the wild-type allele.","The protein encoded by this gene belongs to the membrane-associated guanylate kinase (MAGUK) family, a class of proteins that functions as molecular scaffolds for the assembly of multiprotein complexes at specialized regions of the plasma membrane. This protein is also a member of the CARD protein family, which is defined by carrying a characteristic caspase-associated recruitment domain (CARD). This protein has a domain structure similar to that of CARD14 protein. The CARD domains of both proteins have been shown to specifically interact with BCL10, a protein known to function as a positive regulator of cell apoptosis and NF-kappaB activation. When expressed in cells, this protein activated NF-kappaB and induced the phosphorylation of BCL10.","This gene encodes a member of the cysteine-aspartic acid protease (caspase) family. Sequential activation of caspases plays a central role in the execution-phase of cell apoptosis. Caspases exist as inactive proenzymes composed of a prodomain, a large protease subunit, and a small protease subunit. Activation of caspases requires proteolytic processing at conserved internal aspartic residues to generate a heterodimeric enzyme consisting of the large and small subunits. This protein is involved in the programmed cell death induced by Fas and various apoptotic stimuli. The N-terminal FADD-like death effector domain of this protein suggests that it may interact with Fas-interacting protein FADD. This protein was detected in the insoluble fraction of the affected brain region from Huntington disease patients but not in those from normal controls, which implicated the role in neurodegenerative diseases. Many alternatively spliced transcript variants encoding different isoforms have been described, although not all variants have had their full-length sequences determined.","The protein encoded by this gene is the beta subunit of a heterodimeric core-binding transcription factor belonging to the PEBP2/CBF transcription factor family which master-regulates a host of genes specific to hematopoiesis (e.g., RUNX1) and osteogenesis (e.g., RUNX2). The beta subunit is a non-DNA binding regulatory subunit; it allosterically enhances DNA binding by alpha subunit as the complex binds to the core site of various enhancers and promoters, including murine leukemia virus, polyomavirus enhancer, T-cell receptor enhancers and GM-CSF promoters. Alternative splicing generates two mRNA variants, each encoding a distinct carboxyl terminus. In some cases, a pericentric inversion of chromosome 16 [inv(16)(p13q22)] produces a chimeric transcript consisting of the N terminus of core-binding factor beta in a fusion with the C-terminal portion of the smooth muscle myosin heavy chain 11. This chromosomal rearrangement is associated with acute myeloid leukemia of the M4Eo subtype. Two transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance throughout the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with and functions as a regulatory subunit of CDK4 or CDK6, whose activity is required for cell cycle G1/S transition. This protein has been shown to interact with tumor suppressor protein Rb and the expression of this gene is regulated positively by Rb. Mutations, amplification and overexpression of this gene, which alters cell cycle progression, are observed frequently in a variety of human cancers.","The B lymphocyte antigen receptor is a multimeric complex that includes the antigen-specific component, surface immunoglobulin (Ig). Surface Ig non-covalently associates with two other proteins, Ig-alpha and Ig-beta, which are necessary for expression and function of the B-cell antigen receptor. This gene encodes the Ig-beta protein of the B-cell antigen component. Alternatively spliced transcript variants encoding different isoforms have been described.","This gene encodes a classical cadherin of the cadherin superfamily. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate the mature glycoprotein. This calcium-dependent cell-cell adhesion protein is comprised of five extracellular cadherin repeats, a transmembrane region and a highly conserved cytoplasmic tail. Mutations in this gene are correlated with gastric, breast, colorectal, thyroid and ovarian cancer. Loss of function of this gene is thought to contribute to cancer progression by increasing proliferation, invasion, and/or metastasis. The ectodomain of this protein mediates bacterial adhesion to mammalian cells and the cytoplasmic domain is required for internalization. This gene is present in a gene cluster with other members of the cadherin family on chromosome 16.","Enables RNA polymerase II CTD heptapeptide repeat kinase activity and cyclin binding activity. Involved in phosphorylation of RNA polymerase II C-terminal domain; protein autophosphorylation; and regulation of MAP kinase activity. Located in nuclear speck. Part of cyclin K-CDK12 complex. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a member of the Ser/Thr protein kinase family. This protein is highly similar to the gene products of S. cerevisiae cdc28 and S. pombe cdc2. It is a catalytic subunit of the protein kinase complex that is important for cell cycle G1 phase progression. The activity of this kinase is restricted to the G1-S phase, which is controlled by the regulatory subunits D-type cyclins and CDK inhibitor p16(INK4a). This kinase was shown to be responsible for the phosphorylation of retinoblastoma gene product (Rb). Mutations in this gene as well as in its related proteins including D-type cyclins, p16(INK4a) and Rb were all found to be associated with tumorigenesis of a variety of cancers. Multiple polyadenylation sites of this gene have been reported.","This gene encodes a potent cyclin-dependent kinase inhibitor. The encoded protein binds to and inhibits the activity of cyclin-cyclin-dependent kinase2 or -cyclin-dependent kinase4 complexes, and thus functions as a regulator of cell cycle progression at G1. The expression of this gene is tightly controlled by the tumor suppressor protein p53, through which this protein mediates the p53-dependent cell cycle G1 phase arrest in response to a variety of stress stimuli. This protein can interact with proliferating cell nuclear antigen, a DNA polymerase accessory factor, and plays a regulatory role in S phase DNA replication and DNA damage repair. This protein was reported to be specifically cleaved by CASP3-like caspases, which thus leads to a dramatic activation of cyclin-dependent kinase2, and may be instrumental in the execution of apoptosis following caspase activation. Mice that lack this gene have the ability to regenerate damaged or missing tissue. Multiple alternatively spliced variants have been found for this gene.","This gene encodes a cyclin-dependent kinase inhibitor, which shares a limited similarity with CDK inhibitor CDKN1A/p21. The encoded protein binds to and prevents the activation of cyclin E-CDK2 or cyclin D-CDK4 complexes, and thus controls the cell cycle progression at G1. The degradation of this protein, which is triggered by its CDK dependent phosphorylation and subsequent ubiquitination by SCF complexes, is required for the cellular transition from quiescence to the proliferative state. Mutations in this gene are associated with multiple endocrine neoplasia type IV (MEN4).","This gene generates several transcript variants which differ in their first exons. At least three alternatively spliced variants encoding distinct proteins have been reported, two of which encode structurally related isoforms known to function as inhibitors of CDK4 kinase. The remaining transcript includes an alternate first exon located 20 Kb upstream of the remainder of the gene; this transcript contains an alternate open reading frame (ARF) that specifies a protein which is structurally unrelated to the products of the other variants. This ARF product functions as a stabilizer of the tumor suppressor protein p53 as it can interact with, and sequester, the E3 ubiquitin-protein ligase MDM2, a protein responsible for the degradation of p53. In spite of the structural and functional differences, the CDK inhibitor isoforms and the ARF product encoded by this gene, through the regulatory roles of CDK4 and p53 in cell cycle G1 progression, share a common functionality in cell cycle G1 control. This gene is frequently mutated or deleted in a wide variety of tumors, and is known to be an important tumor suppressor gene.","The protein encoded by this gene is a member of the INK4 family of cyclin-dependent kinase inhibitors. This protein has been shown to interact with CDK4 or CDK6, and prevent the activation of the CDK kinases, thus function as a cell growth regulator that controls cell cycle G1 progression. Ectopic expression of this gene was shown to suppress the growth of human cells in a manner that appears to correlate with the presence of a wild-type RB1 function. Studies in the knockout mice suggested the roles of this gene in regulating spermatogenesis, as well as in suppressing tumorigenesis. Two alternatively spliced transcript variants of this gene, which encode an identical protein, have been reported.","This intronless gene encodes a transcription factor that contains a basic leucine zipper (bZIP) domain and recognizes the CCAAT motif in the promoters of target genes. The encoded protein functions in homodimers and also heterodimers with CCAAT/enhancer-binding proteins beta and gamma. Activity of this protein can modulate the expression of genes involved in cell cycle regulation as well as in body weight homeostasis. Mutation of this gene is associated with acute myeloid leukemia. The use of alternative in-frame non-AUG (GUG) and AUG start codons results in protein isoforms with different lengths. Differential translation initiation is mediated by an out-of-frame, upstream open reading frame which is located between the GUG and the first AUG start codons.","The product of this gene belongs to the SNF2/RAD54 helicase family. It represents the main component of the nucleosome remodeling and deacetylase complex and plays an important role in epigenetic transcriptional repression. Patients with dermatomyositis develop antibodies against this protein. Somatic mutations in this gene are associated with serous endometrial tumors. Alternative splicing results in multiple transcript variants encoding different isoforms.","In response to DNA damage and replication blocks, cell cycle progression is halted through the control of critical cell cycle regulators. The protein encoded by this gene is a cell cycle checkpoint regulator and putative tumor suppressor. It contains a forkhead-associated protein interaction domain essential for activation in response to DNA damage and is rapidly phosphorylated in response to replication blocks and DNA damage. When activated, the encoded protein is known to inhibit CDC25C phosphatase, preventing entry into mitosis, and has been shown to stabilize the tumor suppressor protein p53, leading to cell cycle arrest in G1. In addition, this protein interacts with and phosphorylates BRCA1, allowing BRCA1 to restore survival after DNA damage. Mutations in this gene have been linked with Li-Fraumeni syndrome, a highly penetrant familial cancer phenotype usually associated with inherited mutations in TP53. Also, mutations in this gene are thought to confer a predisposition to sarcomas, breast cancer, and brain tumors. This nuclear protein is a member of the CDS1 subfamily of serine/threonine protein kinases. Several transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is an ortholog of the Drosophila melanogaster capicua gene, and is a member of the high mobility group (HMG)-box superfamily of transcriptional repressors. This protein contains a conserved HMG domain that is involved in DNA binding and nuclear localization, and a conserved C-terminus. Studies suggest that the N-terminal region of this protein interacts with Atxn1 (GeneID:6310), to form a transcription repressor complex, and in vitro studies suggest that polyglutamine-expansion of ATXN1 may alter the repressor activity of this complex. Mutations in this gene have been associated with olidogdendrogliomas (PMID:21817013). In addition, translocation events resulting in gene fusions of this gene with both DUX4 (GeneID:100288687) and FOXO4 (GeneID:4303) have been associated with round cell sarcomas. There are multiple pseudogenes of this gene found on chromosomes 1, 4, 6, 7, 16, 20, and the Y chromosome. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene is ubiquitously expressed and is involved in the transcriptional coactivation of many different transcription factors. First isolated as a nuclear protein that binds to cAMP-response element binding protein (CREB), this gene is now known to play critical roles in embryonic development, growth control, and homeostasis by coupling chromatin remodeling to transcription factor recognition. The protein encoded by this gene has intrinsic histone acetyltransferase activity and also acts as a scaffold to stabilize additional protein interactions with the transcription complex. This protein acetylates both histone and non-histone proteins. This protein shares regions of very high sequence similarity with protein p300 in its bromodomain, cysteine-histidine-rich regions, and histone acetyltransferase domain. Mutations in this gene cause Rubinstein-Taybi syndrome (RTS). Chromosomal translocations involving this gene have been associated with acute myeloid leukemia. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene is a member of the BORIS + CTCF gene family and encodes a transcriptional regulator protein with 11 highly conserved zinc finger (ZF) domains. This nuclear protein is able to use different combinations of the ZF domains to bind different DNA target sequences and proteins. Depending upon the context of the site, the protein can bind a histone acetyltransferase (HAT)-containing complex and function as a transcriptional activator or bind a histone deacetylase (HDAC)-containing complex and function as a transcriptional repressor. If the protein is bound to a transcriptional insulator element, it can block communication between enhancers and upstream promoters, thereby regulating imprinted expression. Mutations in this gene have been associated with invasive breast cancers, prostate cancers, and Wilms' tumors. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is part of a complex of proteins that constitute adherens junctions (AJs). AJs are necessary for the creation and maintenance of epithelial cell layers by regulating cell growth and adhesion between cells. The encoded protein also anchors the actin cytoskeleton and may be responsible for transmitting the contact inhibition signal that causes cells to stop dividing once the epithelial sheet is complete. Finally, this protein binds to the product of the APC gene, which is mutated in adenomatous polyposis of the colon. Mutations in this gene are a cause of colorectal cancer (CRC), pilomatrixoma (PTR), medulloblastoma (MDB), and ovarian cancer. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the cullin protein family. The encoded protein plays a critical role in the polyubiquitination and subsequent degradation of specific protein substrates as the core component and scaffold protein of an E3 ubiquitin ligase complex. Complexes including the encoded protein may also play a role in late endosome maturation. Mutations in this gene are a cause of type 2E pseudohypoaldosteronism. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","This gene is encodes a cytoplasmic protein with three cytoskeletal-associated protein-glycine-conserved (CAP-GLY) domains that functions as a deubiquitinating enzyme. Mutations in this gene have been associated with cylindromatosis, multiple familial trichoepithelioma, and Brooke-Spiegler syndrome. Alternate transcriptional splice variants, encoding different isoforms, have been characterized.","The cysteinyl leukotrienes LTC4, LTD4, and LTE4 are important mediators of human bronchial asthma. Pharmacologic studies have determined that cysteinyl leukotrienes activate at least 2 receptors, the protein encoded by this gene and CYSLTR1. This encoded receptor is a member of the superfamily of G protein-coupled receptors. It seems to play a major role in endocrine and cardiovascular systems.","The protein encoded by this gene is a member of the large DEAD-box protein family, that is defined by the presence of the conserved Asp-Glu-Ala-Asp (DEAD) motif, and has ATP-dependent RNA helicase activity. This protein has been reported to display a high level of RNA-independent ATPase activity, and unlike most DEAD-box helicases, the ATPase activity is thought to be stimulated by both RNA and DNA. This protein has multiple conserved domains and is thought to play roles in both the nucleus and cytoplasm. Nuclear roles include transcriptional regulation, mRNP assembly, pre-mRNA splicing, and mRNA export. In the cytoplasm, this protein is thought to be involved in translation, cellular signaling, and viral replication. Misregulation of this gene has been implicated in tumorigenesis. This gene has a paralog located in the nonrecombining region of the Y chromosome. Pseudogenes sharing similarity to both this gene and the DDX3Y paralog are found on chromosome 4 and the X chromosome. Alternative splicing results in multiple transcript variants.","This gene encodes a protein possessing an RNA helicase motif containing a DEXH box in its amino terminus and an RNA motif in the carboxy terminus. The encoded protein functions as a ribonuclease and is required by the RNA interference and small temporal RNA (stRNA) pathways to produce the active small RNA component that represses gene expression. This protein also acts as a strong antiviral agent with activity against RNA viruses, including the Zika and SARS-CoV-2 viruses. Alternative splicing results in multiple transcript variants.","CpG methylation is an epigenetic modification that is important for embryonic development, imprinting, and X-chromosome inactivation. Studies in mice have demonstrated that DNA methylation is required for mammalian development. This gene encodes a DNA methyltransferase that is thought to function in de novo methylation, rather than maintenance methylation. The protein localizes to the cytoplasm and nucleus and its expression is developmentally regulated.","The protein encoded by this gene is a transmembrane glycoprotein that is a member of the protein kinase superfamily. This protein is a receptor for members of the epidermal growth factor family. EGFR is a cell surface protein that binds to epidermal growth factor, thus inducing receptor dimerization and tyrosine autophosphorylation leading to cell proliferation. Mutations in this gene are associated with lung cancer. EGFR is a component of the cytokine storm which contributes to a severe form of Coronavirus Disease 2019 (COVID-19) resulting from infection with severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2).","This gene encodes an essential eukaryotic translation initiation factor. The protein is required for the binding of the 43S complex (a 40S subunit, eIF2/GTP/Met-tRNAi and eIF3) to the 5' end of capped RNA.","Enables DNA-binding transcription activator activity, RNA polymerase II-specific and sequence-specific double-stranded DNA binding activity. Involved in inflammatory response; negative regulation of transcription, DNA-templated; and positive regulation of transcription by RNA polymerase II. Located in Golgi apparatus; cytosol; and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes the adenovirus E1A-associated cellular p300 transcriptional co-activator protein. It functions as histone acetyltransferase that regulates transcription via chromatin remodeling and is important in the processes of cell proliferation and differentiation. It mediates cAMP-gene regulation by binding specifically to phosphorylated CREB protein. This gene has also been identified as a co-activator of HIF1A (hypoxia-inducible factor 1 alpha), and thus plays a role in the stimulation of hypoxia-induced genes such as VEGF. Defects in this gene are a cause of Rubinstein-Taybi syndrome and may also play a role in epithelial cancer.","This gene encodes a transcription factor involved in the induction of genes regulated by oxygen, which is induced as oxygen levels fall. The encoded protein contains a basic-helix-loop-helix domain protein dimerization domain as well as a domain found in proteins in signal transduction pathways which respond to oxygen levels. Mutations in this gene are associated with erythrocytosis familial type 4.","This gene belongs to the ephrin receptor subfamily of the protein-tyrosine kinase family. EPH and EPH-related receptors have been implicated in mediating developmental events, particularly in the nervous system. Receptors in the EPH subfamily typically have a single kinase domain and an extracellular region containing a Cys-rich domain and 2 fibronectin type III repeats. The ephrin receptors are divided into 2 groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. This gene encodes a protein that binds ephrin-A ligands. Two alternatively spliced transcript variants have been described for this gene.","This gene encodes a member of the epidermal growth factor (EGF) receptor family of receptor tyrosine kinases. This protein has no ligand binding domain of its own and therefore cannot bind growth factors. However, it does bind tightly to other ligand-bound EGF receptor family members to form a heterodimer, stabilizing ligand binding and enhancing kinase-mediated activation of downstream signalling pathways, such as those involving mitogen-activated protein kinase and phosphatidylinositol-3 kinase. Allelic variations at amino acid positions 654 and 655 of isoform a (positions 624 and 625 of isoform b) have been reported, with the most common allele, Ile654/Ile655, shown here. Amplification and/or overexpression of this gene has been reported in numerous cancers, including breast and ovarian tumors. Alternative splicing results in several additional transcript variants, some encoding different isoforms and others that have not been fully characterized.","This gene encodes a member of the epidermal growth factor receptor (EGFR) family of receptor tyrosine kinases. This membrane-bound protein has a neuregulin binding domain but not an active kinase domain. It therefore can bind this ligand but not convey the signal into the cell through protein phosphorylation. However, it does form heterodimers with other EGF receptor family members which do have kinase activity. Heterodimerization leads to the activation of pathways which lead to cell proliferation or differentiation. Amplification of this gene and/or overexpression of its protein have been reported in numerous cancers, including prostate, bladder, and breast tumors. Alternate transcriptional splice variants encoding different isoforms have been characterized. One isoform lacks the intermembrane region and is secreted outside the cell. This form acts to modulate the activity of the membrane-bound form. Additional splice variants have also been reported, but they have not been thoroughly characterized.","This gene is a member of the Tyr protein kinase family and the epidermal growth factor receptor subfamily. It encodes a single-pass type I membrane protein with multiple cysteine rich domains, a transmembrane domain, a tyrosine kinase domain, a phosphotidylinositol-3 kinase binding site and a PDZ domain binding motif. The protein binds to and is activated by neuregulins and other factors and induces a variety of cellular responses including mitogenesis and differentiation. Multiple proteolytic events allow for the release of a cytoplasmic fragment and an extracellular fragment. Mutations in this gene have been associated with cancer. Alternatively spliced variants which encode different protein isoforms have been described; however, not all variants have been fully characterized.","The nucleotide excision repair pathway is a mechanism to repair damage to DNA. The protein encoded by this gene is involved in transcription-coupled nucleotide excision repair and is an integral member of the basal transcription factor BTF2/TFIIH complex. The gene product has ATP-dependent DNA helicase activity and belongs to the RAD3/XPD subfamily of helicases. Defects in this gene can result in three different disorders, the cancer-prone syndrome xeroderma pigmentosum complementation group D, trichothiodystrophy, and Cockayne syndrome. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","This gene encodes an estrogen receptor and ligand-activated transcription factor. The canonical protein contains an N-terminal ligand-independent transactivation domain, a central DNA binding domain, a hinge domain, and a C-terminal ligand-dependent transactivation domain. The protein localizes to the nucleus where it may form either a homodimer or a heterodimer with estrogen receptor 2. The protein encoded by this gene regulates the transcription of many estrogen-inducible genes that play a role in growth, metabolism, sexual development, gestation, and other reproductive functions and is expressed in many non-reproductive tissues. The receptor encoded by this gene plays a key role in breast cancer, endometrial cancer, and osteoporosis. This gene is reported to have dozens of transcript variants due to the use of alternate promoters and alternative splicing, however, the full-length nature of many of these variants remain uncertain.","This gene encodes a member of the Polycomb-group (PcG) family. PcG family members form multimeric protein complexes, which are involved in maintaining the transcriptional repressive state of genes over successive cell generations. This protein associates with the embryonic ectoderm development protein, the VAV1 oncoprotein, and the X-linked nuclear protein. This protein may play a role in the hematopoietic and central nervous systems. Multiple alternatively splcied transcript variants encoding distinct isoforms have been identified for this gene.","This gene is an ortholog of the Drosophila fat gene, which encodes a tumor suppressor essential for controlling cell proliferation during Drosophila development. The gene product is a member of the cadherin superfamily, a group of integral membrane proteins characterized by the presence of cadherin-type repeats. In addition to containing 34 tandem cadherin-type repeats, the gene product has five epidermal growth factor (EGF)-like repeats and one laminin A-G domain. This gene is expressed at high levels in a number of fetal epithelia. Its product probably functions as an adhesion molecule and/or signaling receptor, and is likely to be important in developmental processes and cell communication. Transcript variants derived from alternative splicing and/or alternative promoter usage exist, but they have not been fully described.","This gene encodes a member of the F-box protein family which is characterized by an approximately 40 amino acid motif, the F-box. The F-box proteins constitute one of the four subunits of ubiquitin protein ligase complex called SCFs (SKP1-cullin-F-box), which function in phosphorylation-dependent ubiquitination. The F-box proteins are divided into 3 classes: Fbws containing WD-40 domains, Fbls containing leucine-rich repeats, and Fbxs containing either different protein-protein interaction modules or no recognizable motifs. The protein encoded by this gene was previously referred to as FBX30, and belongs to the Fbws class; in addition to an F-box, this protein contains 7 tandem WD40 repeats. This protein binds directly to cyclin E and probably targets cyclin E for ubiquitin-mediated degradation. Mutations in this gene are detected in ovarian and breast cancer cell lines, implicating the gene's potential role in the pathogenesis of human cancers. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a member of the fibroblast growth factor receptor (FGFR) family, where amino acid sequence is highly conserved between members and throughout evolution. FGFR family members differ from one another in their ligand affinities and tissue distribution. A full-length representative protein consists of an extracellular region, composed of three immunoglobulin-like domains, a single hydrophobic membrane-spanning segment and a cytoplasmic tyrosine kinase domain. The extracellular portion of the protein interacts with fibroblast growth factors, setting in motion a cascade of downstream signals, ultimately influencing mitogenesis and differentiation. This particular family member binds both acidic and basic fibroblast growth factors and is involved in limb induction. Mutations in this gene have been associated with Pfeiffer syndrome, Jackson-Weiss syndrome, Antley-Bixler syndrome, osteoglophonic dysplasia, and autosomal dominant Kallmann syndrome 2. Chromosomal aberrations involving this gene are associated with stem cell myeloproliferative disorder and stem cell leukemia lymphoma syndrome. Alternatively spliced variants which encode different protein isoforms have been described; however, not all variants have been fully characterized.","The protein encoded by this gene is a member of the fibroblast growth factor receptor family, where amino acid sequence is highly conserved between members and throughout evolution. FGFR family members differ from one another in their ligand affinities and tissue distribution. A full-length representative protein consists of an extracellular region, composed of three immunoglobulin-like domains, a single hydrophobic membrane-spanning segment and a cytoplasmic tyrosine kinase domain. The extracellular portion of the protein interacts with fibroblast growth factors, setting in motion a cascade of downstream signals, ultimately influencing mitogenesis and differentiation. This particular family member is a high-affinity receptor for acidic, basic and/or keratinocyte growth factor, depending on the isoform. Mutations in this gene are associated with Crouzon syndrome, Pfeiffer syndrome, Craniosynostosis, Apert syndrome, Jackson-Weiss syndrome, Beare-Stevenson cutis gyrata syndrome, Saethre-Chotzen syndrome, and syndromic craniosynostosis. Multiple alternatively spliced transcript variants encoding different isoforms have been noted for this gene.","This gene encodes a member of the fibroblast growth factor receptor (FGFR) family, with its amino acid sequence being highly conserved between members and among divergent species. FGFR family members differ from one another in their ligand affinities and tissue distribution. A full-length representative protein would consist of an extracellular region, composed of three immunoglobulin-like domains, a single hydrophobic membrane-spanning segment and a cytoplasmic tyrosine kinase domain. The extracellular portion of the protein interacts with fibroblast growth factors, setting in motion a cascade of downstream signals, ultimately influencing mitogenesis and differentiation. This particular family member binds acidic and basic fibroblast growth hormone and plays a role in bone development and maintenance. Mutations in this gene lead to craniosynostosis and multiple types of skeletal dysplasia.","This gene encodes a class III receptor tyrosine kinase that regulates hematopoiesis. This receptor is activated by binding of the fms-related tyrosine kinase 3 ligand to the extracellular domain, which induces homodimer formation in the plasma membrane leading to autophosphorylation of the receptor. The activated receptor kinase subsequently phosphorylates and activates multiple cytoplasmic effector molecules in pathways involved in apoptosis, proliferation, and differentiation of hematopoietic cells in bone marrow. Mutations that result in the constitutive activation of this receptor result in acute myeloid leukemia and acute lymphoblastic leukemia.","This gene encodes a member of the forkhead class of DNA-binding proteins. These hepatocyte nuclear factors are transcriptional activators for liver-specific transcripts such as albumin and transthyretin, and they also interact with chromatin. Similar family members in mice have roles in the regulation of metabolism and in the differentiation of the pancreas and liver.","The protein encoded by this gene is a single stranded DNA-binding protein that binds to multiple DNA elements, including the far upstream element (FUSE) located upstream of c-myc. Binding to FUSE occurs on the non-coding strand, and is important to the regulation of c-myc in undifferentiated cells. This protein contains three domains, an amphipathic helix N-terminal domain, a DNA-binding central domain, and a C-terminal transactivation domain that contains three tyrosine-rich motifs. The N-terminal domain is thought to repress the activity of the C-terminal domain. This protein is also thought to bind RNA, and contains 3'-5' helicase activity with in vitro activity on both DNA-DNA and RNA-RNA duplexes. Aberrant expression of this gene has been found in malignant tissues, and this gene is important to neural system and lung development. Binding of this protein to viral RNA is thought to play a role in several viral diseases, including hepatitis C and hand, foot and mouth disease. Alternative splicing results in multiple transcript variants.","This gene encodes a protein which belongs to the GATA family of transcription factors. The protein contains two GATA-type zinc fingers and is an important regulator of T-cell development and plays an important role in endothelial cell biology. Defects in this gene are the cause of hypoparathyroidism with sensorineural deafness and renal dysplasia.","The protein encoded by this gene belongs to the family of guanine nucleotide-binding proteins (G proteins), which function as modulators or transducers in various transmembrane signaling systems. G proteins are composed of 3 units: alpha, beta and gamma. This gene encodes one of the alpha subunits (subunit alpha-11). Mutations in this gene have been associated with hypocalciuric hypercalcemia type II (HHC2) and hypocalcemia dominant 2 (HYPOC2). Patients with HHC2 and HYPOC2 exhibit decreased or increased sensitivity, respectively, to changes in extracellular calcium concentrations.","This locus encodes a guanine nucleotide-binding protein. The encoded protein, an alpha subunit in the Gq class, couples a seven-transmembrane domain receptor to activation of phospolipase C-beta. Mutations at this locus have been associated with problems in platelet activation and aggregation. A related pseudogene exists on chromosome 2.[provided by RefSeq, Nov 2010].","This locus has a highly complex imprinted expression pattern. It gives rise to maternally, paternally, and biallelically expressed transcripts that are derived from four alternative promoters and 5' exons. Some transcripts contain a differentially methylated region (DMR) at their 5' exons, and this DMR is commonly found in imprinted genes and correlates with transcript expression. An antisense transcript is produced from an overlapping locus on the opposite strand. One of the transcripts produced from this locus, and the antisense transcript, are paternally expressed noncoding RNAs, and may regulate imprinting in this region. In addition, one of the transcripts contains a second overlapping ORF, which encodes a structurally unrelated protein - Alex. Alternative splicing of downstream exons is also observed, which results in different forms of the stimulatory G-protein alpha subunit, a key element of the classical signal transduction pathway linking receptor-ligand interactions with the activation of adenylyl cyclase and a variety of cellular reponses. Multiple transcript variants encoding different isoforms have been found for this gene. Mutations in this gene result in pseudohypoparathyroidism type 1a, pseudohypoparathyroidism type 1b, Albright hereditary osteodystrophy, pseudopseudohypoparathyroidism, McCune-Albright syndrome, progressive osseus heteroplasia, polyostotic fibrous dysplasia of bone, and some pituitary tumors.","Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Two molecules of each of the four core histones (H2A, H2B, H3, and H4) form an octamer, around which approximately 146 bp of DNA is wrapped in repeating units, called nucleosomes. The linker histone, H1, interacts with linker DNA between nucleosomes and functions in the compaction of chromatin into higher order structures. This gene contains introns and its mRNA is polyadenylated, unlike most histone genes. The protein encoded is a replication-independent member of the histone H3 family.","This gene belongs to the Ras oncogene family, whose members are related to the transforming genes of mammalian sarcoma retroviruses. The products encoded by these genes function in signal transduction pathways. These proteins can bind GTP and GDP, and they have intrinsic GTPase activity. This protein undergoes a continuous cycle of de- and re-palmitoylation, which regulates its rapid exchange between the plasma membrane and the Golgi apparatus. Mutations in this gene cause Costello syndrome, a disease characterized by increased growth at the prenatal stage, growth deficiency at the postnatal stage, predisposition to tumor formation, cognitive disability, skin and musculoskeletal abnormalities, distinctive facial appearance and cardiovascular abnormalities. Defects in this gene are implicated in a variety of cancers, including bladder cancer, follicular thyroid cancer, and oral squamous cell carcinoma. Multiple transcript variants, which encode different isoforms, have been identified for this gene.","Isocitrate dehydrogenases catalyze the oxidative decarboxylation of isocitrate to 2-oxoglutarate. These enzymes belong to two distinct subclasses, one of which utilizes NAD(+) as the electron acceptor and the other NADP(+). Five isocitrate dehydrogenases have been reported: three NAD(+)-dependent isocitrate dehydrogenases, which localize to the mitochondrial matrix, and two NADP(+)-dependent isocitrate dehydrogenases, one of which is mitochondrial and the other predominantly cytosolic. Each NADP(+)-dependent isozyme is a homodimer. The protein encoded by this gene is the NADP(+)-dependent isocitrate dehydrogenase found in the cytoplasm and peroxisomes. It contains the PTS-1 peroxisomal targeting signal sequence. The presence of this enzyme in peroxisomes suggests roles in the regeneration of NADPH for intraperoxisomal reductions, such as the conversion of 2, 4-dienoyl-CoAs to 3-enoyl-CoAs, as well as in peroxisomal reactions that consume 2-oxoglutarate, namely the alpha-hydroxylation of phytanic acid. The cytoplasmic enzyme serves a significant role in cytoplasmic NADPH production. Alternatively spliced transcript variants encoding the same protein have been found for this gene.","Isocitrate dehydrogenases catalyze the oxidative decarboxylation of isocitrate to 2-oxoglutarate. These enzymes belong to two distinct subclasses, one of which utilizes NAD(+) as the electron acceptor and the other NADP(+). Five isocitrate dehydrogenases have been reported: three NAD(+)-dependent isocitrate dehydrogenases, which localize to the mitochondrial matrix, and two NADP(+)-dependent isocitrate dehydrogenases, one of which is mitochondrial and the other predominantly cytosolic. Each NADP(+)-dependent isozyme is a homodimer. The protein encoded by this gene is the NADP(+)-dependent isocitrate dehydrogenase found in the mitochondria. It plays a role in intermediary metabolism and energy production. This protein may tightly associate or interact with the pyruvate dehydrogenase complex. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a signal transducer shared by many cytokines, including interleukin 6 (IL6), ciliary neurotrophic factor (CNTF), leukemia inhibitory factor (LIF), and oncostatin M (OSM). This protein functions as a part of the cytokine receptor complex. The activation of this protein is dependent upon the binding of cytokines to their receptors. vIL6, a protein related to IL6 and encoded by the Kaposi sarcoma-associated herpesvirus, can bypass the interleukin 6 receptor (IL6R) and directly activate this protein. Knockout studies in mice suggest that this gene plays a critical role in regulating myocyte apoptosis. Alternatively spliced transcript variants have been described. A related pseudogene has been identified on chromosome 17.","The protein encoded by this gene is a receptor for interleukin 7 (IL7). The function of this receptor requires the interleukin 2 receptor, gamma chain (IL2RG), which is a common gamma chain shared by the receptors of various cytokines, including interleukins 2, 4, 7, 9, and 15. This protein has been shown to play a critical role in V(D)J recombination during lymphocyte development. Defects in this gene may be associated with severe combined immunodeficiency (SCID). Alternatively spliced transcript variants have been found.","This gene encodes a membrane protein that is a member of a class of protein-tyrosine kinases (PTK) characterized by the presence of a second phosphotransferase-related domain immediately N-terminal to the PTK domain. The encoded kinase phosphorylates STAT proteins (signal transducers and activators of transcription) and plays a key role in interferon-alpha/beta, interferon-gamma, and cytokine signal transduction. This gene plays a crucial role in effecting the expression of genes that mediate inflammation, epithelial remodeling, and metastatic cancer progression. This gene is a key component of the interleukin-6 (IL-6)/JAK1/STAT3 immune and inflammation response and is a therapeutic target for alleviating cytokine storms. The kinase activity of this gene is directly inhibited by the suppressor of cytokine signalling 1 (SOCS1) protein. Alternative splicing results in multiple transcript variants.","This gene encodes a non-receptor tyrosine kinase that plays a central role in cytokine and growth factor signalling. The primary isoform of this protein has an N-terminal FERM domain that is required for erythropoietin receptor association, an SH2 domain that binds STAT transcription factors, a pseudokinase domain and a C-terminal tyrosine kinase domain. Cytokine binding induces autophosphorylation and activation of this kinase. This kinase then recruits and phosphorylates signal transducer and activator of transcription (STAT) proteins. Growth factors like TGF-beta 1 also induce phosphorylation and activation of this kinase and translocation of downstream STAT proteins to the nucleus where they influence gene transcription. Mutations in this gene are associated with numerous inflammatory diseases and malignancies. This gene is a downstream target of the pleiotropic cytokine IL6 that is produced by B cells, T cells, dendritic cells and macrophages to produce an immune response or inflammation. Disregulation of the IL6/JAK2/STAT3 signalling pathways produces increased cellular proliferation and myeloproliferative neoplasms of hematopoietic stem cells. A nonsynonymous mutation in the pseudokinase domain of this gene disrupts the domains inhibitory effect and results in constitutive tyrosine phosphorylation activity and hypersensitivity to cytokine signalling. This gene and the IL6/JAK2/STAT3 signalling pathway is a therapeutic target for the treatment of excessive inflammatory responses to viral infections. Alternative splicing results in multiple transcript variants encoding distinct isoforms.","The protein encoded by this gene is a member of the Janus kinase (JAK) family of tyrosine kinases involved in cytokine receptor-mediated intracellular signal transduction. It is predominantly expressed in immune cells and transduces a signal in response to its activation via tyrosine phosphorylation by interleukin receptors. Mutations in this gene are associated with autosomal SCID (severe combined immunodeficiency disease).","This gene is a member of the SMCY homolog family and encodes a protein with one ARID domain, one JmjC domain, one JmjN domain and two PHD-type zinc fingers. The DNA-binding motifs suggest this protein is involved in the regulation of transcription and chromatin remodeling. Mutations in this gene have been associated with X-linked cognitive disability. Alternative splicing results in multiple transcript variants.","This gene is located on the X chromosome and is the corresponding locus to a Y-linked gene which encodes a tetratricopeptide repeat (TPR) protein. The encoded protein of this gene contains a JmjC-domain and catalyzes the demethylation of tri/dimethylated histone H3. Multiple alternatively spliced transcript variants have been found for this gene.","This gene encodes a protein containing KELCH-1 like domains, as well as a BTB/POZ domain. Kelch-like ECH-associated protein 1 interacts with NF-E2-related factor 2 in a redox-sensitive manner and the dissociation of the proteins in the cytoplasm is followed by transportation of NF-E2-related factor 2 to the nucleus. This interaction results in the expression of the catalytic subunit of gamma-glutamylcysteine synthetase. Two alternatively spliced transcript variants encoding the same isoform have been found for this gene.","This gene encodes a receptor tyrosine kinase. This gene was initially identified as a homolog of the feline sarcoma viral oncogene v-kit and is often referred to as proto-oncogene c-Kit. The canonical form of this glycosylated transmembrane protein has an N-terminal extracellular region with five immunoglobulin-like domains, a transmembrane region, and an intracellular tyrosine kinase domain at the C-terminus. Upon activation by its cytokine ligand, stem cell factor (SCF), this protein phosphorylates multiple intracellular proteins that play a role in in the proliferation, differentiation, migration and apoptosis of many cell types and thereby plays an important role in hematopoiesis, stem cell maintenance, gametogenesis, melanogenesis, and in mast cell development, migration and function. This protein can be a membrane-bound or soluble protein. Mutations in this gene are associated with gastrointestinal stromal tumors, mast cell disease, acute myelogenous leukemia, and piebaldism. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a transcriptional coactivator that plays an essential role in regulating gene expression during early development and hematopoiesis. The encoded protein contains multiple conserved functional domains. One of these domains, the SET domain, is responsible for its histone H3 lysine 4 (H3K4) methyltransferase activity which mediates chromatin modifications associated with epigenetic transcriptional activation. This protein is processed by the enzyme Taspase 1 into two fragments, MLL-C and MLL-N. These fragments reassociate and further assemble into different multiprotein complexes that regulate the transcription of specific target genes, including many of the HOX genes. Multiple chromosomal translocations involving this gene are the cause of certain acute lymphoid leukemias and acute myeloid leukemias. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Oct 2010].","This gene is a member of the myeloid/lymphoid or mixed-lineage leukemia (MLL) family and encodes a nuclear protein with an AT hook DNA-binding domain, a DHHC-type zinc finger, six PHD-type zinc fingers, a SET domain, a post-SET domain and a RING-type zinc finger. This protein is a member of the ASC-2/NCOA6 complex (ASCOM), which possesses histone methylation activity and is involved in transcriptional coactivation.","The protein encoded by this gene is a histone methyltransferase that methylates the Lys-4 position of histone H3. The encoded protein is part of a large protein complex called ASCOM, which has been shown to be a transcriptional regulator of the beta-globin and estrogen receptor genes. Mutations in this gene have been shown to be a cause of Kabuki syndrome.","This gene, a Kirsten ras oncogene homolog from the mammalian ras gene family, encodes a protein that is a member of the small GTPase superfamily. A single amino acid substitution is responsible for an activating mutation. The transforming protein that results is implicated in various malignancies, including lung adenocarcinoma, mucinous adenoma, ductal carcinoma of the pancreas and colorectal carcinoma. Alternative splicing leads to variants encoding two isoforms that differ in the C-terminal region.","The protein encoded by this gene is a putative serine/threonine kinase that localizes to the mitotic apparatus and complexes with cell cycle controller CDC2 kinase in early mitosis. The protein is phosphorylated in a cell-cycle dependent manner, with late prophase phosphorylation remaining through metaphase. The N-terminal region of the protein binds CDC2 to form a complex showing reduced H1 histone kinase activity, indicating a role as a negative regulator of CDC2/cyclin A. In addition, the C-terminal kinase domain binds to its own N-terminal region, suggesting potential negative regulation through interference with complex formation via intramolecular binding. Biochemical and genetic data suggest a role as a tumor suppressor. This is supported by studies in knockout mice showing development of soft-tissue sarcomas, ovarian stromal cell tumors and a high sensitivity to carcinogenic treatments.","This gene encodes a serine/threonine protein kinase belonging to the LATS tumor suppressor family. The protein localizes to centrosomes during interphase, and early and late metaphase. It interacts with the centrosomal proteins aurora-A and ajuba and is required for accumulation of gamma-tubulin and spindle formation at the onset of mitosis. It also interacts with a negative regulator of p53 and may function in a positive feedback loop with p53 that responds to cytoskeleton damage. Additionally, it can function as a co-repressor of androgen-responsive gene expression.","This gene encodes a member of the BTB-kelch superfamily. Initially described as a putative transcriptional regulator based on weak homology to members of the basic leucine zipper-like family, the encoded protein subsequently has been shown to localize exclusively to the Golgi network where it may help stabilize the Gogli complex. Deletion of this gene may be associated with DiGeorge syndrome.","The protein encoded by this gene is a member of the dual specificity protein kinase family, which acts as a mitogen-activated protein (MAP) kinase kinase. MAP kinases, also known as extracellular signal-regulated kinases (ERKs), act as an integration point for multiple biochemical signals. This protein kinase lies upstream of MAP kinases and stimulates the enzymatic activity of MAP kinases upon wide variety of extra- and intracellular signals. As an essential component of MAP kinase signal transduction pathway, this kinase is involved in many cellular processes such as proliferation, differentiation, transcription regulation and development.","This gene encodes a member of the mitogen-activated protein kinase (MAPK) family. Members of this family act as an integration point for multiple biochemical signals and are involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation, and development. They form a three-tiered signaling module composed of MAPKKKs, MAPKKs, and MAPKs. This protein is phosphorylated at serine and threonine residues by MAPKKKs and subsequently phosphorylates downstream MAPK targets at threonine and tyrosine residues. A similar protein in mouse has been reported to play a role in liver organogenesis. A pseudogene of this gene is located on the long arm of chromosome X. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a serine/threonine kinase and is part of some signal transduction cascades, including the ERK and JNK kinase pathways as well as the NF-kappa-B pathway. The encoded protein is activated by autophosphorylation and requires magnesium as a cofactor in phosphorylating other proteins. This protein has E3 ligase activity conferred by a plant homeodomain (PHD) in its N-terminus and phospho-kinase activity conferred by a kinase domain in its C-terminus.","This gene encodes a member of the MAP kinase family. MAP kinases, also known as extracellular signal-regulated kinases (ERKs), act as an integration point for multiple biochemical signals, and are involved in a wide variety of cellular processes such as proliferation, differentiation, transcription regulation and development. The activation of this kinase requires its phosphorylation by upstream kinases. Upon activation, this kinase translocates to the nucleus of the stimulated cells, where it phosphorylates nuclear targets. One study also suggests that this protein acts as a transcriptional repressor independent of its kinase activity. The encoded protein has been identified as a moonlighting protein based on its ability to perform mechanistically distinct functions. Two alternatively spliced transcript variants encoding the same protein, but differing in the UTRs, have been reported for this gene.","The protein encoded by this gene is a member of the basic helix-loop-helix leucine zipper (bHLHZ) family of transcription factors. It is able to form homodimers and heterodimers with other family members, which include Mad, Mxi1 and Myc. Myc is an oncoprotein implicated in cell proliferation, differentiation and apoptosis. The homodimers and heterodimers compete for a common DNA target site (the E box) and rearrangement among these dimer forms provides a complex system of transcriptional regulation. Mutations of this gene have been reported to be associated with hereditary pheochromocytoma. A pseudogene of this gene is located on the long arm of chromosome 7. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a transcriptional regulator and oncoprotein that may be involved in hematopoiesis, apoptosis, development, and cell differentiation and proliferation. The encoded protein can interact with CTBP1, SMAD3, CREBBP, KAT2B, MAPK8, and MAPK9. This gene can undergo translocation with the AML1 gene, resulting in overexpression of this gene and the onset of leukemia. Several transcript variants encoding a few different isoforms have been found for this gene.","The initiation of transcription is controlled in part by a large protein assembly known as the preinitiation complex. A component of this preinitiation complex is a 1.2 MDa protein aggregate called Mediator. This Mediator component binds with a CDK8 subcomplex which contains the protein encoded by this gene, mediator complex subunit 12 (MED12), along with MED13, CDK8 kinase, and cyclin C. The CDK8 subcomplex modulates Mediator-polymerase II interactions and thereby regulates transcription initiation and reinitation rates. The MED12 protein is essential for activating CDK8 kinase. Defects in this gene cause X-linked Opitz-Kaveggia syndrome, also known as FG syndrome, and Lujan-Fryns syndrome.","This gene encodes menin, a tumor suppressor associated with a syndrome known as multiple endocrine neoplasia type 1. Menin is a scaffold protein that functions in histone modification and epigenetic gene regulation. It is thought to regulate several pathways and processes by altering chromatin structure through the modification of histones.","This gene encodes a member of the receptor tyrosine kinase family of proteins and the product of the proto-oncogene MET. The encoded preproprotein is proteolytically processed to generate alpha and beta subunits that are linked via disulfide bonds to form the mature receptor. Further processing of the beta subunit results in the formation of the M10 peptide, which has been shown to reduce lung fibrosis. Binding of its ligand, hepatocyte growth factor, induces dimerization and activation of the receptor, which plays a role in cellular survival, embryogenesis, and cellular migration and invasion. Mutations in this gene are associated with papillary renal cell carcinoma, hepatocellular carcinoma, and various head and neck cancers. Amplification and overexpression of this gene are also associated with multiple human cancers.","The protein encoded by this gene belongs to a family of phosphatidylinositol kinase-related kinases. These kinases mediate cellular responses to stresses such as DNA damage and nutrient deprivation. This kinase is a component of two distinct complexes, mTORC1, which controls protein synthesis, cell growth and proliferation, and mTORC2, which is a regulator of the actin cytoskeleton, and promotes cell survival and cell cycle progression. This protein acts as the target for the cell-cycle arrest and immunosuppressive effects of the FKBP12-rapamycin complex. Inhibitors of mTOR are used in organ transplants as immunosuppressants, and are being evaluated for their therapeutic potential in SARS-CoV-2 infections. Mutations in this gene are associated with Smith-Kingsmore syndrome and somatic focal cortical dysplasia type II. The ANGPTL7 gene is located in an intron of this gene.","This gene is a proto-oncogene and encodes a nuclear phosphoprotein that plays a role in cell cycle progression, apoptosis and cellular transformation. The encoded protein forms a heterodimer with the related transcription factor MAX. This complex binds to the E box DNA consensus sequence and regulates the transcription of specific target genes. Amplification of this gene is frequently observed in numerous human cancers. Translocations involving this gene are associated with Burkitt lymphoma and multiple myeloma in human patients. There is evidence to show that translation initiates both from an upstream, in-frame non-AUG (CUG) and a downstream AUG start site, resulting in the production of two isoforms with distinct N-termini.","This gene is a member of the MYC family and encodes a protein with a basic helix-loop-helix (bHLH) domain. This protein is located in the nucleus and must dimerize with another bHLH protein in order to bind DNA. Amplification of this gene is associated with a variety of tumors, most notably neuroblastomas. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene.","This gene encodes a cytosolic adapter protein that plays a central role in the innate and adaptive immune response. This protein functions as an essential signal transducer in the interleukin-1 and Toll-like receptor signaling pathways. These pathways regulate that activation of numerous proinflammatory genes. The encoded protein consists of an N-terminal death domain and a C-terminal Toll-interleukin1 receptor domain. Patients with defects in this gene have an increased susceptibility to pyogenic bacterial infections. Alternate splicing results in multiple transcript variants.","This gene encodes a conventional non-muscle myosin; this protein should not be confused with the unconventional myosin-9a or 9b (MYO9A or MYO9B). The encoded protein is a myosin IIA heavy chain that contains an IQ domain and a myosin head-like domain which is involved in several important functions, including cytokinesis, cell motility and maintenance of cell shape. Defects in this gene have been associated with non-syndromic sensorineural deafness autosomal dominant type 17, Epstein syndrome, Alport syndrome with macrothrombocytopenia, Sebastian syndrome, Fechtner syndrome and macrothrombocytopenia with progressive sensorineural deafness.","This gene encodes a protein that mediates ligand-independent transcription repression of thyroid-hormone and retinoic-acid receptors by promoting chromatin condensation and preventing access of the transcription machinery. It is part of a complex which also includes histone deacetylases and transcriptional regulators similar to the yeast protein Sin3p. This gene is located between the Charcot-Marie-Tooth and Smith-Magenis syndrome critical regions on chromosome 17. Alternate splicing results in multiple transcript variants. Pseudogenes of this gene are found on chromosomes 17 and 20.[provided by RefSeq, Jun 2010].","This gene product appears to function as a negative regulator of the ras signal transduction pathway. Mutations in this gene have been linked to neurofibromatosis type 1, juvenile myelomonocytic leukemia and Watson syndrome. The mRNA for this gene is subject to RNA editing (CGA>UGA->Arg1306Term) resulting in premature translation termination. Alternatively spliced transcript variants encoding different isoforms have also been described for this gene.","This gene encodes a protein that is similar to some members of the ERM (ezrin, radixin, moesin) family of proteins that link cytoskeletal components with proteins in the cell membrane. The encoded protein is involved in regulation of contact-dependent inhibition of cell proliferation and functions in cell-cell adhesion and transmembrane signaling. The encoded protein has been shown to interact with cell-surface proteins, proteins involved in cytoskeletal dynamics, and proteins involved in regulating ion transport. Disruption of this protein's function has been implicated in tumorigenesis and metastasis. Mutations in this gene are associated with neurofibromatosis type II which is characterized by nervous system and skin tumors and ocular abnormalities.","This gene encodes a transcription factor which is a member of a small family of basic leucine zipper (bZIP) proteins. The encoded transcription factor regulates genes which contain antioxidant response elements (ARE) in their promoters; many of these genes encode proteins involved in response to injury and inflammation which includes the production of free radicals. Multiple transcript variants encoding different isoforms have been characterized for this gene.","This gene encodes a member of the NOTCH family of proteins. Members of this Type I transmembrane protein family share structural characteristics including an extracellular domain consisting of multiple epidermal growth factor-like (EGF) repeats, and an intracellular domain consisting of multiple different domain types. Notch signaling is an evolutionarily conserved intercellular signaling pathway that regulates interactions between physically adjacent cells through binding of Notch family receptors to their cognate ligands. The encoded preproprotein is proteolytically processed in the trans-Golgi network to generate two polypeptide chains that heterodimerize to form the mature cell-surface receptor. This receptor plays a role in the development of numerous cell and tissue types. Mutations in this gene are associated with aortic valve disease, Adams-Oliver syndrome, T-cell acute lymphoblastic leukemia, chronic lymphocytic leukemia, and head and neck squamous cell carcinoma.","This gene encodes a member of the Notch family. Members of this Type 1 transmembrane protein family share structural characteristics including an extracellular domain consisting of multiple epidermal growth factor-like (EGF) repeats, and an intracellular domain consisting of multiple, different domain types. Notch family members play a role in a variety of developmental processes by controlling cell fate decisions. The Notch signaling network is an evolutionarily conserved intercellular signaling pathway which regulates interactions between physically adjacent cells. In Drosophilia, notch interaction with its cell-bound ligands (delta, serrate) establishes an intercellular signaling pathway that plays a key role in development. Homologues of the notch-ligands have also been identified in human, but precise interactions between these ligands and the human notch homologues remain to be determined. This protein is cleaved in the trans-Golgi network, and presented on the cell surface as a heterodimer. This protein functions as a receptor for membrane bound ligands, and may play a role in vascular, renal and hepatic development. Two transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is involved in several cellular processes, including centrosome duplication, protein chaperoning, and cell proliferation. The encoded phosphoprotein shuttles between the nucleolus, nucleus, and cytoplasm, chaperoning ribosomal proteins and core histones from the nucleus to the cytoplasm. This protein is also known to sequester the tumor suppressor ARF in the nucleolus, protecting it from degradation until it is needed. Mutations in this gene are associated with acute myeloid leukemia. Dozens of pseudogenes of this gene have been identified.","This is an N-ras oncogene encoding a membrane protein that shuttles between the Golgi apparatus and the plasma membrane. This shuttling is regulated through palmitoylation and depalmitoylation by the ZDHHC9-GOLGA7 complex. The encoded protein, which has intrinsic GTPase activity, is activated by a guanine nucleotide-exchange factor and inactivated by a GTPase activating protein. Mutations in this gene have been associated with somatic rectal cancer, follicular thyroid cancer, autoimmune lymphoproliferative syndrome, Noonan syndrome, and juvenile myelomonocytic leukemia.","This gene encodes a protein that contains four domains present in other developmental proteins: a PWWP domain, an HMG box, a SET domain, and a PHD-type zinc finger. It is expressed ubiquitously in early development. Wolf-Hirschhorn syndrome (WHS) is a malformation syndrome associated with a hemizygous deletion of the distal short arm of chromosome 4. This gene maps to the 165 kb WHS critical region and has also been involved in the chromosomal translocation t(4;14)(p16.3;q32.3) in multiple myelomas. Alternative splicing of this gene results in multiple transcript variants encoding different isoforms. Some transcript variants are nonsense-mediated mRNA (NMD) decay candidates, hence not represented as reference sequences.","This gene encodes a member of the paired box (PAX) family of transcription factors. The central feature of this gene family is a novel, highly conserved DNA-binding motif, known as the paired box. Paired box transcription factors are important regulators in early development, and alterations in the expression of their genes are thought to contribute to neoplastic transformation. This gene encodes the B-cell lineage specific activator protein that is expressed at early, but not late stages of B-cell differentiation. Its expression has also been detected in developing CNS and testis and so the encoded protein may also play a role in neural development and spermatogenesis. This gene is located at 9p13, which is involved in t(9;14)(p13;q32) translocations recurring in small lymphocytic lymphomas of the plasmacytoid subtype, and in derived large-cell lymphomas. This translocation brings the potent E-mu enhancer of the IgH gene into close proximity of the PAX5 promoter, suggesting that the deregulation of transcription of this gene contributes to the pathogenesis of these lymphomas. Alternative splicing results in multiple transcript variants encoding different isoforms.","This locus encodes a subunit of ATP-dependent chromatin-remodeling complexes. The encoded protein has been identified as in integral component of complexes necessary for ligand-dependent transcriptional activation by nuclear hormone receptors. Mutations at this locus have been associated with primary clear cell renal cell carcinoma.","This intronless gene is thought to have been generated by retrotransposition of a fully processed PCBP-2 mRNA. This gene and PCBP-2 have paralogues (PCBP3 and PCBP4) which are thought to have arisen as a result of duplication events of entire genes. The protein encoded by this gene appears to be multifunctional. It along with PCBP-2 and hnRNPK corresponds to the major cellular poly(rC)-binding protein. It contains three K-homologous (KH) domains which may be involved in RNA binding. This encoded protein together with PCBP-2 also functions as translational coactivators of poliovirus RNA via a sequence-specific interaction with stem-loop IV of the IRES and promote poliovirus RNA replication by binding to its 5'-terminal cloverleaf structure. It has also been implicated in translational control of the 15-lipoxygenase mRNA, human Papillomavirus type 16 L2 mRNA, and hepatitis A virus RNA. The encoded protein is also suggested to play a part in formation of a sequence-specific alpha-globin mRNP complex which is associated with alpha-globin mRNA stability.","This gene encodes a cell surface tyrosine kinase receptor for members of the platelet-derived growth factor family. These growth factors are mitogens for cells of mesenchymal origin. The identity of the growth factor bound to a receptor monomer determines whether the functional receptor is a homodimer or a heterodimer, composed of both platelet-derived growth factor receptor alpha and beta polypeptides. Studies suggest that this gene plays a role in organ development, wound healing, and tumor progression. Mutations in this gene have been associated with idiopathic hypereosinophilic syndrome, somatic and familial gastrointestinal stromal tumors, and a variety of other cancers.","This gene is a member of the plant homeodomain (PHD)-like finger (PHF) family. It encodes a protein with two PHD-type zinc finger domains, indicating a potential role in transcriptional regulation, that localizes to the nucleolus. Mutations affecting the coding region of this gene or the splicing of the transcript have been associated with Borjeson-Forssman-Lehmann syndrome (BFLS), a disorder characterized by cognitive disability, epilepsy, hypogonadism, hypometabolism, obesity, swelling of subcutaneous tissue of the face, narrow palpebral fissures, and large ears. Alternate splicing results in multiple transcript variants, encoding different isoforms.","Phosphatidylinositol 3-kinase is composed of an 85 kDa regulatory subunit and a 110 kDa catalytic subunit. The protein encoded by this gene represents the catalytic subunit, which uses ATP to phosphorylate PtdIns, PtdIns4P and PtdIns(4,5)P2. This gene has been found to be oncogenic and has been implicated in cervical cancers. A pseudogene of this gene has been defined on chromosome 22.","This gene encodes an isoform of the catalytic subunit of phosphoinositide 3-kinase (PI3K). These kinases are important in signaling pathways involving receptors on the outer membrane of eukaryotic cells and are named for their catalytic subunit. The encoded protein is the catalytic subunit for PI3Kbeta (PI3KB). PI3KB has been shown to be part of the activation pathway in neutrophils which have bound immune complexes at sites of injury or infection. Alternative splicing results in multiple transcript variants.","Phosphatidylinositol 3-kinase phosphorylates the inositol ring of phosphatidylinositol at the 3-prime position. The enzyme comprises a 110 kD catalytic subunit and a regulatory subunit of either 85, 55, or 50 kD. This gene encodes the 85 kD regulatory subunit. Phosphatidylinositol 3-kinase plays an important role in the metabolic actions of insulin, and a mutation in this gene has been associated with insulin resistance. Alternative splicing of this gene results in four transcript variants encoding different isoforms.","The protein encoded by this gene belongs to the Ser/Thr protein kinase family, and PIM subfamily. This gene is expressed primarily in B-lymphoid and myeloid cell lines, and is overexpressed in hematopoietic malignancies and in prostate cancer. It plays a role in signal transduction in blood cells, contributing to both cell proliferation and survival, and thus provides a selective advantage in tumorigenesis. Both the human and orthologous mouse genes have been reported to encode two isoforms (with preferential cellular localization) resulting from the use of alternative in-frame translation initiation codons, the upstream non-AUG (CUG) and downstream AUG codons (PMIDs:16186805, 1825810).[provided by RefSeq, Aug 2011].","The protein encoded by this gene catalyzes the formation of inositol 1,4,5-trisphosphate and diacylglycerol from phosphatidylinositol 4,5-bisphosphate. This reaction uses calcium as a cofactor and plays an important role in the intracellular transduction of receptor-mediated tyrosine kinase activators. For example, when activated by SRC, the encoded protein causes the Ras guanine nucleotide exchange factor RasGRP1 to translocate to the Golgi, where it activates Ras. Also, this protein has been shown to be a major substrate for heparin-binding growth factor 1 (acidic fibroblast growth factor)-activated tyrosine kinase. Two transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a key component of the mismatch repair system that functions to correct DNA mismatches and small insertions and deletions that can occur during DNA replication and homologous recombination. This protein forms heterodimers with the gene product of the mutL homolog 1 (MLH1) gene to form the MutL-alpha heterodimer. The MutL-alpha heterodimer possesses an endonucleolytic activity that is activated following recognition of mismatches and insertion/deletion loops by the MutS-alpha and MutS-beta heterodimers, and is necessary for removal of the mismatched DNA. There is a DQHA(X)2E(X)4E motif found at the C-terminus of the protein encoded by this gene that forms part of the active site of the nuclease. Mutations in this gene have been associated with hereditary nonpolyposis colorectal cancer (HNPCC; also known as Lynch syndrome) and Turcot syndrome.","This gene encodes the catalytic subunit of DNA polymerase epsilon. The enzyme is involved in DNA repair and chromosomal DNA replication. Mutations in this gene have been associated with colorectal cancer 12 and facial dysmorphism, immunodeficiency, livedo, and short stature.","Enables catalytic activity, acting on DNA; chromatin binding activity; and identical protein binding activity. Involved in DNA repair; negative regulation of double-strand break repair via homologous recombination; and protein homooligomerization. Located in Golgi apparatus; cytosol; and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a member of the PP2C family of Ser/Thr protein phosphatases. PP2C family members are known to be negative regulators of cell stress response pathways. The expression of this gene is induced in a p53-dependent manner in response to various environmental stresses. While being induced by tumor suppressor protein TP53/p53, this phosphatase negatively regulates the activity of p38 MAP kinase, MAPK/p38, through which it reduces the phosphorylation of p53, and in turn suppresses p53-mediated transcription and apoptosis. This phosphatase thus mediates a feedback regulation of p38-p53 signaling that contributes to growth inhibition and the suppression of stress induced apoptosis. This gene is located in a chromosomal region known to be amplified in breast cancer. The amplification of this gene has been detected in both breast cancer cell line and primary breast tumors, which suggests a role of this gene in cancer development.","This gene encodes a constant regulatory subunit of protein phosphatase 2. Protein phosphatase 2 is one of the four major Ser/Thr phosphatases, and it is implicated in the negative control of cell growth and division. It consists of a common heteromeric core enzyme, which is composed of a catalytic subunit and a constant regulatory subunit, that associates with a variety of regulatory subunits. The constant regulatory subunit A serves as a scaffolding molecule to coordinate the assembly of the catalytic subunit and a variable regulatory B subunit. This gene encodes an alpha isoform of the constant regulatory subunit A. Alternatively spliced transcript variants have been described.","This gene encodes the catalytic subunit of protein phosphatase, a component of a signaling pathway regulating cell cycle progression. Splice variants encoding different protein isoforms exist. The pseudogene of this gene is located on chromosome X.","cAMP is a signaling molecule important for a variety of cellular functions. cAMP exerts its effects by activating the cAMP-dependent protein kinase, which transduces the signal through phosphorylation of different target proteins. The inactive kinase holoenzyme is a tetramer composed of two regulatory and two catalytic subunits. cAMP causes the dissociation of the inactive holoenzyme into a dimer of regulatory subunits bound to four cAMP and two free monomeric catalytic subunits. Four different regulatory subunits and three catalytic subunits have been identified in humans. This gene encodes one of the regulatory subunits. This protein was found to be a tissue-specific extinguisher that down-regulates the expression of seven liver genes in hepatoma x fibroblast hybrids. Mutations in this gene cause Carney complex (CNC). This gene can fuse to the RET protooncogene by gene rearrangement and form the thyroid tumor-specific chimeric oncogene known as PTC2. A nonconventional nuclear localization sequence (NLS) has been found for this protein which suggests a role in DNA replication via the protein serving as a nuclear transport protein for the second subunit of the Replication Factor C (RFC40). Several alternatively spliced transcript variants encoding two different isoforms have been observed.","Enables DNA-binding transcription factor binding activity; chromatin binding activity; and transcription coactivator activity. Involved in mRNA 5'-splice site recognition and positive regulation of transcription by RNA polymerase II. Located in heterochromatin; nuclear periphery; and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the patched family of proteins and a component of the hedgehog signaling pathway. Hedgehog signaling is important in embryonic development and tumorigenesis. The encoded protein is the receptor for the secreted hedgehog ligands, which include sonic hedgehog, indian hedgehog and desert hedgehog. Following binding by one of the hedgehog ligands, the encoded protein is trafficked away from the primary cilium, relieving inhibition of the G-protein-coupled receptor smoothened, which results in activation of downstream signaling. Mutations of this gene have been associated with basal cell nevus syndrome and holoprosencephaly.","This gene was identified as a tumor suppressor that is mutated in a large number of cancers at high frequency. The protein encoded by this gene is a phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase. It contains a tensin like domain as well as a catalytic domain similar to that of the dual specificity protein tyrosine phosphatases. Unlike most of the protein tyrosine phosphatases, this protein preferentially dephosphorylates phosphoinositide substrates. It negatively regulates intracellular levels of phosphatidylinositol-3,4,5-trisphosphate in cells and functions as a tumor suppressor by negatively regulating AKT/PKB signaling pathway. The use of a non-canonical (CUG) upstream initiation site produces a longer isoform that initiates translation with a leucine, and is thought to be preferentially associated with the mitochondrial inner membrane. This longer isoform may help regulate energy metabolism in the mitochondria. A pseudogene of this gene is found on chromosome 9. Alternative splicing and the use of multiple translation start codons results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP contains two tandem Src homology-2 domains, which function as phospho-tyrosine binding domains and mediate the interaction of this PTP with its substrates. This PTP is widely expressed in most tissues and plays a regulatory role in various cell signaling events that are important for a diversity of cell functions, such as mitogenic activation, metabolic control, transcription regulation, and cell migration. Mutations in this gene are a cause of Noonan syndrome as well as acute myeloid leukemia.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitosis, and oncogenic transformation. This PTP contains an extracellular domain, a single transmembrane segment and two tandem intracytoplasmic catalytic domains, and thus is classified as a receptor type PTP. This PTP has been shown to be an essential regulator of T- and B-cell antigen receptor signaling. It functions through either direct interaction with components of the antigen receptor complexes, or by activating various Src family kinases required for the antigen receptor signaling. This PTP also suppresses JAK kinases, and thus functions as a regulator of cytokine receptor signaling. Alternatively spliced transcripts variants of this gene, which encode distinct isoforms, have been reported.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP contains an extracellular region, a single transmembrane segment and two tandem intracytoplasmic catalytic domains, and thus represents a receptor-type PTP. The extracellular region of this protein is composed of three Ig-like and eight fibronectin type III-like domains. Studies of the similar genes in chicken and fly suggest the role of this PTP is in promoting neurite growth, and regulating neurons axon guidance. Multiple alternatively spliced transcript variants of this gene have been reported. A related pseudogene has been identified on chromosome 5.","The protein encoded by this gene is a GTPase which belongs to the RAS superfamily of small GTP-binding proteins. Members of this superfamily appear to regulate a diverse array of cellular events, including the control of cell growth, cytoskeletal reorganization, and the activation of protein kinases. Two transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is highly similar to the gene product of Schizosaccharomyces pombe rad21, a gene involved in the repair of DNA double-strand breaks, as well as in chromatid cohesion during mitosis. This protein is a nuclear phospho-protein, which becomes hyperphosphorylated in cell cycle M phase. The highly regulated association of this protein with mitotic chromatin specifically at the centromere region suggests its role in sister chromatid cohesion in mitotic cells.","This gene is the cellular homolog of viral raf gene (v-raf). The encoded protein is a MAP kinase kinase kinase (MAP3K), which functions downstream of the Ras family of membrane associated GTPases to which it binds directly. Once activated, the cellular RAF1 protein can phosphorylate to activate the dual specificity protein kinases MEK1 and MEK2, which in turn phosphorylate to activate the serine/threonine specific protein kinases, ERK1 and ERK2. Activated ERKs are pleiotropic effectors of cell physiology and play an important role in the control of gene expression involved in the cell division cycle, apoptosis, cell differentiation and cell migration. Mutations in this gene are associated with Noonan syndrome 5 and LEOPARD syndrome 2.","The protein encoded by this gene is a negative regulator of the cell cycle and was the first tumor suppressor gene found. The encoded protein also stabilizes constitutive heterochromatin to maintain the overall chromatin structure. The active, hypophosphorylated form of the protein binds transcription factor E2F1. Defects in this gene are a cause of childhood cancer retinoblastoma (RB), bladder cancer, and osteogenic sarcoma.","This gene encodes a nuclear protein that belongs to a family proteins that contain an RNA-binding motif. The encoded protein associates with hnRNP proteins and may be involved in regulating alternative splicing. Defects in this gene are the cause of the X-linked recessive disorder, TARP syndrome. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Mar 2011].","This gene encodes a transmembrane receptor and member of the tyrosine protein kinase family of proteins. Binding of ligands such as GDNF (glial cell-line derived neurotrophic factor) and other related proteins to the encoded receptor stimulates receptor dimerization and activation of downstream signaling pathways that play a role in cell differentiation, growth, migration and survival. The encoded receptor is important in development of the nervous system, and the development of organs and tissues derived from the neural crest. This proto-oncogene can undergo oncogenic activation through both cytogenetic rearrangement and activating point mutations. Mutations in this gene are associated with Hirschsprung disease and central hypoventilation syndrome and have been identified in patients with renal agenesis.","This gene encodes a member of the Rho family of small GTPases, which cycle between inactive GDP-bound and active GTP-bound states and function as molecular switches in signal transduction cascades. Rho proteins promote reorganization of the actin cytoskeleton and regulate cell shape, attachment, and motility. Overexpression of this gene is associated with tumor cell proliferation and metastasis. Multiple alternatively spliced variants have been identified.","The protein encoded by this gene is a RING-type E3 ubiquitin ligase and is predicted to contain a transmembrane domain, a protease-associated domain, an ectodomain, and a cytoplasmic RING domain. This protein is thought to negatively regulate Wnt signaling, and expression of this gene results in an increase in ubiquitination of frizzled receptors, an alteration in their subcellular distribution, resulting in reduced surface levels of these receptors. Mutations in this gene have been reported in multiple tumor cells, including colorectal and endometrial cancers. Alternative splicing results in multiple transcript variants encoding different isoforms.","Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a cytoplasmic ribosomal protein that is a component of the 60S subunit. The protein belongs to the L22E family of ribosomal proteins. Its initiating methionine residue is post-translationally removed. The protein can bind specifically to Epstein-Barr virus-encoded RNAs (EBERs) 1 and 2. The mouse protein has been shown to be capable of binding to heparin. Transcript variants utilizing alternative polyA signals exist. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. It was previously thought that this gene mapped to 3q26 and that it was fused to the acute myeloid leukemia 1 (AML1) gene located at 21q22 in some therapy-related myelodysplastic syndrome patients with 3;21 translocations; however, these fusions actually involve a ribosomal protein L22 pseudogene located at 3q26, and this gene actually maps to 1p36.3-p36.2.","Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of four RNA species and approximately 80 structurally distinct proteins. This gene encodes a member of the L18P family of ribosomal proteins and component of the 60S subunit. The encoded protein binds 5S rRNA to form a stable complex called the 5S ribonucleoprotein particle (RNP), which is necessary for the transport of nonribosome-associated cytoplasmic 5S rRNA to the nucleolus for assembly into ribosomes. The encoded protein may also function to inhibit tumorigenesis through the activation of downstream tumor suppressors and the downregulation of oncoprotein expression. Mutations in this gene have been identified in patients with Diamond-Blackfan Anemia (DBA). This gene is co-transcribed with the small nucleolar RNA gene U21, which is located in its fifth intron. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed throughout the genome.","Core binding factor (CBF) is a heterodimeric transcription factor that binds to the core element of many enhancers and promoters. The protein encoded by this gene represents the alpha subunit of CBF and is thought to be involved in the development of normal hematopoiesis. Chromosomal translocations involving this gene are well-documented and have been associated with several types of leukemia. Three transcript variants encoding different isoforms have been found for this gene.","This gene encodes a protein which contains a several motifs including a ski homology region and a SET-binding region in addition to three nuclear localization signals. The encoded protein has been shown to bind the SET nuclear oncogene which is involved in DNA replication. Mutations in this gene are associated with Schinzel-Giedion midface retraction syndrome. Multiple transcript variants encoding different isoforms have been found for this gene.","Huntington's disease (HD), a neurodegenerative disorder characterized by loss of striatal neurons, is caused by an expansion of a polyglutamine tract in the HD protein huntingtin. This gene encodes a protein belonging to a class of huntingtin interacting proteins characterized by WW motifs. This protein is a histone methyltransferase that is specific for lysine-36 of histone H3, and methylation of this residue is associated with active chromatin. This protein also contains a novel transcriptional activation domain and has been found associated with hyperphosphorylated RNA polymerase II.","This gene encodes subunit 1 of the splicing factor 3b protein complex. Splicing factor 3b, together with splicing factor 3a and a 12S RNA unit, forms the U2 small nuclear ribonucleoproteins complex (U2 snRNP). The splicing factor 3b/3a complex binds pre-mRNA upstream of the intron's branch site in a sequence independent manner and may anchor the U2 snRNP to the pre-mRNA. Splicing factor 3b is also a component of the minor U12-type spliceosome. The carboxy-terminal two-thirds of subunit 1 have 22 non-identical, tandem HEAT repeats that form rod-like, helical structures. Alternative splicing results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene belongs to the SMAD, a family of proteins similar to the gene products of the Drosophila gene 'mothers against decapentaplegic' (Mad) and the C. elegans gene Sma. SMAD proteins are signal transducers and transcriptional modulators that mediate multiple signaling pathways. This protein mediates the signal of the transforming growth factor (TGF)-beta, and thus regulates multiple cellular processes, such as cell proliferation, apoptosis, and differentiation. This protein is recruited to the TGF-beta receptors through its interaction with the SMAD anchor for receptor activation (SARA) protein. In response to TGF-beta signal, this protein is phosphorylated by the TGF-beta receptors. The phosphorylation induces the dissociation of this protein with SARA and the association with the family member SMAD4. The association with SMAD4 is important for the translocation of this protein into the nucleus, where it binds to target promoters and forms a transcription repressor complex with other cofactors. This protein can also be phosphorylated by activin type 1 receptor kinase, and mediates the signal from the activin. Alternatively spliced transcript variants have been observed for this gene.","This gene encodes a member of the Smad family of signal transduction proteins. Smad proteins are phosphorylated and activated by transmembrane serine-threonine receptor kinases in response to transforming growth factor (TGF)-beta signaling. The product of this gene forms homomeric complexes and heteromeric complexes with other activated Smad proteins, which then accumulate in the nucleus and regulate the transcription of target genes. This protein binds to DNA and recognizes an 8-bp palindromic sequence (GTCTAGAC) called the Smad-binding element (SBE). The protein acts as a tumor suppressor and inhibits epithelial cell proliferation. It may also have an inhibitory effect on tumors by reducing angiogenesis and increasing blood vessel hyperpermeability. The encoded protein is a crucial component of the bone morphogenetic protein signaling pathway. The Smad proteins are subject to complex regulation by post-translational modifications. Mutations or deletions in this gene have been shown to result in pancreatic cancer, juvenile polyposis syndrome, and hereditary hemorrhagic telangiectasia syndrome.","The protein encoded by this gene is a member of the SWI/SNF family of proteins and is similar to the brahma protein of Drosophila. Members of this family have helicase and ATPase activities and are thought to regulate transcription of certain genes by altering the chromatin structure around those genes. The encoded protein is part of the large ATP-dependent chromatin remodeling complex SNF/SWI, which is required for transcriptional activation of genes normally repressed by chromatin. In addition, this protein can bind BRCA1, as well as regulate the expression of the tumorigenic protein CD44. Mutations in this gene cause rhabdoid tumor predisposition syndrome type 2. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is part of a complex that relieves repressive chromatin structures, allowing the transcriptional machinery to access its targets more effectively. The encoded nuclear protein may also bind to and enhance the DNA joining activity of HIV-1 integrase. This gene has been found to be a tumor suppressor, and mutations in it have been associated with malignant rhabdoid tumors. Alternatively spliced transcript variants have been found for this gene.","Proper cohesion of sister chromatids is a prerequisite for the correct segregation of chromosomes during cell division. The cohesin multiprotein complex is required for sister chromatid cohesion. This complex is composed partly of two structural maintenance of chromosomes (SMC) proteins, SMC3 and either SMC1B or the protein encoded by this gene. Most of the cohesin complexes dissociate from the chromosomes before mitosis, although those complexes at the kinetochore remain. Therefore, the encoded protein is thought to be an important part of functional kinetochores. In addition, this protein interacts with BRCA1 and is phosphorylated by ATM, indicating a potential role for this protein in DNA repair. This gene, which belongs to the SMC gene family, is located in an area of the X-chromosome that escapes X inactivation. Mutations in this gene result in Cornelia de Lange syndrome. Alternative splicing results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene recognizes the sequence CCTTGAG along with other members of the HMG-box class DNA-binding proteins. It acts during chondrocyte differentiation and, with steroidogenic factor 1, regulates transcription of the anti-Muellerian hormone (AMH) gene. Deficiencies lead to the skeletal malformation syndrome campomelic dysplasia, frequently with sex reversal.","This gene encodes a protein that may modulate the transcriptional repression activities of death-associated protein 6 (DAXX), which interacts with histone deacetylase, core histones, and other histone-associated proteins. In mouse, the encoded protein binds to the putative leucine zipper domain of macroH2A1.2, a variant H2A histone that is enriched on inactivated X chromosomes. The BTB/POZ domain of this protein has been shown in other proteins to mediate transcriptional repression and to interact with components of histone deacetylase co-repressor complexes. Alternative splicing of this gene results in multiple transcript variants encoding the same protein.","The protein encoded by this gene is a member of the serine/arginine (SR)-rich family of pre-mRNA splicing factors, which constitute part of the spliceosome. Each of these factors contains an RNA recognition motif (RRM) for binding RNA and an RS domain for binding other proteins. The RS domain is rich in serine and arginine residues and facilitates interaction between different SR splicing factors. In addition to being critical for mRNA splicing, the SR proteins have also been shown to be involved in mRNA export from the nucleus and in translation. Two transcript variants encoding the same protein and one non-coding transcript variant have been found for this gene. In addition, a pseudogene of this gene has been found on chromosome 11.","The protein encoded by this gene is a subunit of the cohesin complex, which regulates the separation of sister chromatids during cell division. Targeted inactivation of this gene results in chromatid cohesion defects and aneuploidy, suggesting that genetic disruption of cohesin is a cause of aneuploidy in human cancer. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a serine/threonine kinase that regulates cell polarity and energy metabolism and functions as a tumor suppressor. Mutations in this gene have been associated with the autosomal dominant Peutz-Jeghers syndrome, as well as with skin, pancreatic, and testicular cancers.","This gene is a member of the WD40 repeat-containing gene family and shares sequence similarity with transducin (beta)-like 1X-linked (TBL1X). The protein encoded by this gene is thought to be a component of both nuclear receptor corepressor (N-CoR) and histone deacetylase 3 (HDAC 3) complexes, and is required for transcriptional activation by a variety of transcription factors. Mutations in these gene have been associated with some autism spectrum disorders, and one finding suggests that haploinsufficiency of this gene may be a cause of intellectual disability with dysmorphism. Mutations in this gene as well as recurrent translocations involving this gene have also been observed in some tumors.","This gene is a member of a phylogenetically conserved family of genes that share a common DNA-binding domain, the T-box. T-box genes encode transcription factors involved in the regulation of developmental processes. This protein is a transcriptional repressor and is thought to play a role in the anterior/posterior axis of the tetrapod forelimb. Mutations in this gene cause ulnar-mammary syndrome, affecting limb, apocrine gland, tooth, hair, and genital development. Alternative splicing of this gene results in three transcript variants encoding different isoforms; however, the full length nature of one variant has not been determined.","This gene encodes a high mobility group (HMG) box-containing transcription factor that plays a key role in the Wnt signaling pathway. The protein has been implicated in blood glucose homeostasis. Genetic variants of this gene are associated with increased risk of type 2 diabetes. Several transcript variants encoding multiple different isoforms have been found for this gene.[provided by RefSeq, Oct 2010].","The protein encoded by this gene is a methylcytosine dioxygenase that catalyzes the conversion of methylcytosine to 5-hydroxymethylcytosine. The encoded protein is involved in myelopoiesis, and defects in this gene have been associated with several myeloproliferative disorders. Two variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a transmembrane protein that has a protein kinase domain, forms a heterodimeric complex with TGF-beta receptor type-1, and binds TGF-beta. This receptor/ligand complex phosphorylates proteins, which then enter the nucleus and regulate the transcription of genes related to cell proliferation, cell cycle arrest, wound healing, immunosuppression, and tumorigenesis. Mutations in this gene have been associated with Marfan Syndrome, Loeys-Deitz Aortic Aneurysm Syndrome, and the development of various types of tumors. Alternatively spliced transcript variants encoding different isoforms have been characterized.","This gene was identified as a gene whose expression is rapidly induced by the tumor necrosis factor (TNF). The protein encoded by this gene is a zinc finger protein and ubiqitin-editing enzyme, and has been shown to inhibit NF-kappa B activation as well as TNF-mediated apoptosis. The encoded protein, which has both ubiquitin ligase and deubiquitinase activities, is involved in the cytokine-mediated immune and inflammatory responses. Several transcript variants encoding the same protein have been found for this gene.","This gene encodes a tumor suppressor protein containing transcriptional activation, DNA binding, and oligomerization domains. The encoded protein responds to diverse cellular stresses to regulate expression of target genes, thereby inducing cell cycle arrest, apoptosis, senescence, DNA repair, or changes in metabolism. Mutations in this gene are associated with a variety of human cancers, including hereditary cancers such as Li-Fraumeni syndrome. Alternative splicing of this gene and the use of alternate promoters result in multiple transcript variants and isoforms. Additional isoforms have also been shown to result from the use of alternate translation initiation codons from identical transcript variants (PMIDs: 12032546, 20937277).","This gene is a tumor suppressor gene that encodes the growth inhibitory protein hamartin. The encoded protein interacts with and stabilizes the GTPase activating protein tuberin. This hamartin-tuberin complex negatively regulates mammalian target of rapamycin complex 1 (mTORC1) signaling which is a major regulator of anabolic cell growth. This protein also functions as a co-chaperone for Hsp90 that inhibits its ATPase activity. This protein functions as a facilitator of Hsp90-mediated folding of kinase and non-kinase clients, including TSC2 and thereby preventing their ubiquitination and proteasomal degradation. Mutations in this gene have been associated with tuberous sclerosis and lymphangioleiomyomatosis.","This gene is a tumor suppressor gene that encodes the growth inhibitory protein tuberin. Tuberin interacts with hamartin to form the TSC protein complex which functions in the control of cell growth. This TSC protein complex negatively regulates mammalian target of rapamycin complex 1 (mTORC1) signaling which is a major regulator of anabolic cell growth. Mutations in this gene have been associated with tuberous sclerosis and lymphangioleiomyomatosis.","This gene belongs to the splicing factor SR family of genes. U2 auxiliary factor, comprising a large and a small subunit, is a non-snRNP protein required for the binding of U2 snRNP to the pre-mRNA branch site. This gene encodes the small subunit which plays a critical role in both constitutive and enhancer-dependent RNA splicing by directly mediating interactions between the large subunit and proteins bound to the enhancers. Alternatively spliced transcript variants encoding different isoforms have been identified.","This gene encodes a component of a ubiquitination complex. The encoded protein is involved in the ubiquitination and degradation of hypoxia-inducible-factor (HIF), which is a transcription factor that plays a central role in the regulation of gene expression by oxygen. In addition to oxygen-related gene expression, this protein plays a role in many other cellular processes including cilia formation, cytokine signaling, regulation of senescence, and formation of the extracellular matrix. Variants of this gene are associated with von Hippel-Lindau syndrome, pheochromocytoma, erythrocytosis, renal cell carcinoma, and cerebellar hemangioblastoma.","This gene encodes a transcription factor that contains four zinc-finger motifs at the C-terminus and a proline/glutamine-rich DNA-binding domain at the N-terminus. It has an essential role in the normal development of the urogenital system, and it is mutated in a small subset of patients with Wilms tumor. This gene exhibits complex tissue-specific and polymorphic imprinting pattern, with biallelic, and monoallelic expression from the maternal and paternal alleles in different tissues. Multiple transcript variants have been described. In several variants, there is evidence for the use of a non-AUG (CUG) translation initiation codon upstream of, and in-frame with the first AUG. Authors of PMID:7926762 also provide evidence that WT1 mRNA undergoes RNA editing in human and rat, and that this process is tissue-restricted and developmentally regulated.","This cell-cycle-regulated gene encodes a protein that mediates leucine-rich nuclear export signal (NES)-dependent protein transport. The protein specifically inhibits the nuclear export of Rev and U snRNAs. It is involved in the control of several cellular processes by controlling the localization of cyclin B, MPAK, and MAPKAP kinase 2. This protein also regulates NFAT and AP-1.","This gene encodes a transcription factor with multiple homeodomains and zinc finger motifs, and regulates myogenic and neuronal differentiation. The encoded protein suppresses expression of the alpha-fetoprotein gene by binding to an AT-rich enhancer motif. The protein has also been shown to negatively regulate c-Myb, and transactivate the cell cycle inhibitor cyclin-dependent kinase inhibitor 1A (also known as p21CIP1). This gene is reported to function as a tumor suppressor in several cancers, and sequence variants of this gene are also associated with atrial fibrillation. Multiple transcript variants expressed from alternate promoters and encoding different isoforms have been found for this gene.","This gene is located on the X chromosome and is subject to X inactivation. It is highly conserved in vertebrates and most abundantly expressed in the brain. The encoded protein is a component of histone deacetylase-containing multiprotein complexes that function through modifying chromatin structure to keep genes silent. A chromosomal translocation (X;13) involving this gene is associated with X-linked cognitive disability. Several alternatively spliced transcript variants have been found for this gene.","This gene encodes a member of the Abelson family of nonreceptor tyrosine protein kinases. The protein is highly similar to the c-abl oncogene 1 protein, including the tyrosine kinase, SH2 and SH3 domains, and it plays a role in cytoskeletal rearrangements through its C-terminal F-actin- and microtubule-binding sequences. This gene is expressed in both normal and tumor cells, and is involved in translocation with the ets variant 6 gene in leukemia. Multiple alternatively spliced transcript variants encoding different protein isoforms have been found for this gene.","This gene encodes a member of the G-protein coupled receptor family. Although this protein was earlier thought to be a receptor for vasoactive intestinal peptide (VIP), it is now considered to be an orphan receptor, in that its endogenous ligand has not been identified. The protein is also a coreceptor for human immunodeficiency viruses (HIV). Translocations involving this gene and HMGA2 on chromosome 12 have been observed in lipomas.","This gene encodes a multi-domain protein involved in signaling and organization of cell junctions during embryogenesis. It has also been identified as the fusion partner of acute lymphoblastic leukemia (ALL-1) gene, involved in acute myeloid leukemias with t(6;11)(q27;q23) translocation. Alternatively spliced transcript variants encoding different isoforms have been described for this gene, however, not all have been fully characterized.[provided by RefSeq, May 2011].","This gene encodes a tissue-restricted nuclear transcriptional activator that is preferentially expressed in lymphoid tissue. Isolation of this protein initially defined a highly conserved LAF4/MLLT2 gene family of nuclear transcription factors that may function in lymphoid development and oncogenesis. In some ALL patients, this gene has been found fused to the gene for MLL. Multiple alternatively spliced transcript variants that encode different proteins have been found for this gene.","The protein encoded by this gene belongs to the AF4 family of transcription factors involved in leukemia. It is a component of the positive transcription elongation factor b (P-TEFb) complex. A chromosomal translocation involving this gene and MLL gene on chromosome 11 is found in infant acute lymphoblastic leukemia with ins(5;11)(q31;q31q23).","Enables alpha-catenin binding activity and transcription corepressor activity. Involved in several processes, including negative regulation of hippo signaling; positive regulation of gene silencing by miRNA; and regulation of cellular response to hypoxia. Acts upstream of or within gene silencing by miRNA and positive regulation of protein-containing complex assembly. Located in several cellular components, including Golgi apparatus; P-body; and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a member of the AKT, also called PKB,  serine/threonine protein kinase family. AKT kinases are known to be regulators of cell signaling in response to insulin and growth factors. They are involved in a wide variety of biological processes including cell proliferation, differentiation, apoptosis, tumorigenesis, as well as glycogen synthesis and glucose uptake. This kinase has been shown to be stimulated by platelet-derived growth factor (PDGF), insulin, and insulin-like growth factor 1 (IGF1). Alternatively splice transcript variants encoding distinct isoforms have been described.","Ankyrins are a family of proteins that link the integral membrane proteins to the underlying spectrin-actin cytoskeleton and play key roles in activities such as cell motility, activation, proliferation, contact and the maintenance of specialized membrane domains. Multiple isoforms of ankyrin with different affinities for various target proteins are expressed in a tissue-specific, developmentally regulated manner. Most ankyrins are typically composed of three structural domains: an amino-terminal domain containing multiple ankyrin repeats; a central region with a highly conserved spectrin binding domain; and a carboxy-terminal regulatory domain which is the least conserved and subject to variation. Ankyrin 1, the prototype of this family, was first discovered in the erythrocytes, but since has also been found in brain and muscles. Mutations in erythrocytic ankyrin 1 have been associated in approximately half of all patients with hereditary spherocytosis. Complex patterns of alternative splicing in the regulatory domain, giving rise to different isoforms of ankyrin 1 have been described. Truncated muscle-specific isoforms of ankyrin 1 resulting from usage of an alternate promoter have also been identified.","The human glucocorticoid receptor DNA binding factor, which associates with the promoter region of the glucocorticoid receptor gene (hGR gene), is a repressor of glucocorticoid receptor transcription. The amino acid sequence deduced from the cDNA sequences show the presence of three sequence motifs characteristic of a zinc finger and one motif suggestive of a leucine zipper in which 1 cysteine is found instead of all leucines. The GRLF1 enhances the homologous down-regulation of wild-type hGR gene expression. Biochemical analysis suggests that GRLF1 interaction is sequence specific and that transcriptional efficacy of GRLF1 is regulated through its interaction with specific sequence motif. The level of expression is regulated by glucocorticoids.","Rho GTPase activating protein 5 negatively regulates RHO GTPases, a family which may mediate cytoskeleton changes by stimulating the hydrolysis of bound GTP. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a Rho guanine nucleotide exchange factor (GEF). Rho GEFs regulate the activity of small Rho GTPases by stimulating the exchange of guanine diphosphate (GDP) for guanine triphosphate (GTP) and may play a role in neural morphogenesis. Mutations in this gene are associated with slowed nerve conduction velocity (SNCV). Alternative splicing results in multiple transcript variants.","This gene belongs to the RhoGEF subfamily of RhoGTPases. Members of this subfamily are activated by specific guanine nucleotide exchange factors (GEFs) and are involved in signal transduction. The encoded protein shows cytosolic distribution. Alternative splicing results in multiple transcript variants.","Rho GTPases play a fundamental role in numerous cellular processes that are initiated by extracellular stimuli working through G protein-coupled receptors. The encoded protein may form a complex with G proteins and stimulate Rho-dependent signals. This protein has been observed to form a myeloid/lymphoid fusion partner in acute myeloid leukemia. Three transcript variants encoding different isoforms have been found for this gene.","This locus encodes an AT-rich DNA interacting domain-containing protein. The encoded protein is a component of the SWI/SNF chromatin remodeling complex and may play a role in cell-cycle activation. The protein encoded by this locus is similar to AT-rich interactive domain-containing protein 1A. These two proteins function as alternative, mutually exclusive ARID-subunits of the SWI/SNF complex. The associated complexes play opposing roles. Alternative splicing results in multiple transcript variants.","This gene encodes a protein containing a basic helix-loop-helix domain and two characteristic PAS domains along with a PAC domain. The encoded protein binds to ligand-bound aryl hydrocarbon receptor and aids in the movement of this complex to the nucleus, where it promotes the expression of genes involved in xenobiotic metabolism. This protein is also a co-factor for transcriptional regulation by hypoxia-inducible factor 1. Chromosomal translocation of this locus with the ETV6 (ets variant 6) gene on chromosome 12 have been described in leukemias. Alternative splicing results in multiple transcript variants.","ATF7IP is a multifunctional nuclear protein that associates with heterochromatin. It can act as a transcriptional coactivator or corepressor depending upon its binding partners (summary by Liu et al., 2009).","The protein encoded by this gene belongs to the family of P-type cation transport ATPases, and to the subfamily of Na+/K+ -ATPases. Na+/K+ -ATPase is an integral membrane protein responsible for establishing and maintaining the electrochemical gradients of Na and K ions across the plasma membrane. These gradients are essential for osmoregulation, for sodium-coupled transport of a variety of organic and inorganic molecules, and for electrical excitability of nerve and muscle. This enzyme is composed of two subunits, a large catalytic subunit (alpha) and a smaller glycoprotein subunit (beta). The catalytic subunit of Na+/K+ -ATPase is encoded by multiple genes. This gene encodes an alpha 1 subunit. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene belongs to the family of P-type primary ion transport ATPases characterized by the formation of an aspartyl phosphate intermediate during the reaction cycle. These enzymes remove bivalent calcium ions from eukaryotic cells against very large concentration gradients and play a critical role in intracellular calcium homeostasis. The mammalian plasma membrane calcium ATPase isoforms are encoded by at least four separate genes and the diversity of these enzymes is further increased by alternative splicing of transcripts. The expression of different isoforms and splice variants is regulated in a developmental, tissue- and cell type-specific manner, suggesting that these pumps are functionally adapted to the physiological needs of particular cells and tissues. This gene encodes the plasma membrane calcium ATPase isoform 3. Alternatively spliced transcript variants encoding different isoforms have been identified.","The Axin-related protein, Axin2, presumably plays an important role in the regulation of the stability of beta-catenin in the Wnt signaling pathway, like its rodent homologs, mouse conductin/rat axil. In mouse, conductin organizes a multiprotein complex of APC (adenomatous polyposis of the colon), beta-catenin, glycogen synthase kinase 3-beta, and conductin, which leads to the degradation of beta-catenin. Apparently, the deregulation of beta-catenin is an important event in the genesis of a number of malignancies. The AXIN2 gene has been mapped to 17q23-q24, a region that shows frequent loss of heterozygosity in breast cancer, neuroblastoma, and other tumors. Mutations in this gene have been associated with colorectal cancer with defective mismatch repair.","This gene encodes a protein which interacts with the N-terminal region of BRCA1. In addition to its ability to bind BRCA1 in vivo and in vitro, it shares homology with the 2 most conserved regions of BRCA1: the N-terminal RING motif and the C-terminal BRCT domain. The RING motif is a cysteine-rich sequence found in a variety of proteins that regulate cell growth, including the products of tumor suppressor genes and dominant protooncogenes. This protein also contains 3 tandem ankyrin repeats. The BARD1/BRCA1 interaction is disrupted by tumorigenic amino acid substitutions in BRCA1, implying that the formation of a stable complex between these proteins may be an essential aspect of BRCA1 tumor suppression. This protein may be the target of oncogenic mutations in breast or ovarian cancer. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene.","The BAZ1A gene encodes the accessory subunit of the ATP-dependent chromatin assembly factor (ACF), a member of the ISWI ('imitation switch') family of chromatin remodeling complexes (summarized by Racki et al., 2009).","This gene was identified by its translocation in a case of mucosa-associated lymphoid tissue (MALT) lymphoma. The protein encoded by this gene contains a caspase recruitment domain (CARD), and has been shown to induce apoptosis and to activate NF-kappaB. This protein is reported to interact with other CARD domain containing proteins including CARD9, 10, 11 and 14, which are thought to function as upstream regulators in NF-kappaB signaling. This protein is found to form a complex with MALT1, a protein encoded by another gene known to be translocated in MALT lymphoma. MALT1 and this protein are thought to synergize in the activation of NF-kappaB, and the deregulation of either of them may contribute to the same pathogenetic process that leads to the malignancy. Alternative splicing results in multiple transcript variants.","This gene encodes a C2H2 type zinc-finger protein by its similarity to the mouse Bcl11a/Evi9 protein. The corresponding mouse gene is a common site of retroviral integration in myeloid leukemia, and may function as a leukemia disease gene, in part, through its interaction with BCL6. During hematopoietic cell differentiation, this gene is down-regulated. It is possibly involved in lymphoma pathogenesis since translocations associated with B-cell malignancies also deregulates its expression. Multiple transcript variants encoding several different isoforms have been found for this gene.","This gene encodes a C2H2-type zinc finger protein and is closely related to BCL11A, a gene whose translocation may be associated with B-cell malignancies. Although the specific function of this gene has not been determined, the encoded protein is known to be a transcriptional repressor, and is regulated by the NURD nucleosome remodeling and histone deacetylase complex. Four alternatively spliced transcript variants encoding distinct isoforms have been found for this gene.","The protein encoded by this gene is a zinc finger transcription factor and contains an N-terminal POZ domain. This protein acts as a sequence-specific repressor of transcription, and has been shown to modulate the transcription of STAT-dependent IL-4 responses of B cells. This protein can interact with a variety of POZ-containing proteins that function as transcription corepressors. This gene is found to be frequently translocated and hypermutated in diffuse large-cell lymphoma (DLCL), and may be involved in the pathogenesis of DLCL. Alternatively spliced transcript variants encoding different protein isoforms have been found for this gene.","BCL9 is associated with B-cell acute lymphoblastic leukemia. It may be a target of translocation in B-cell malignancies with abnormalities of 1q21. Its function is unknown. The overexpression of BCL9 may be of pathogenic significance in B-cell malignancies.","Enables beta-catenin binding activity. Involved in several processes, including negative regulation of transforming growth factor beta receptor signaling pathway; positive regulation of epithelial to mesenchymal transition; and positive regulation of transcription by RNA polymerase II. Located in nucleolus and nucleoplasm. Part of beta-catenin-TCF complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a transcriptional repressor that interacts with several members of the BCL2 family of proteins. Overexpression of this protein induces apoptosis, which can be suppressed by co-expression of BCL2 proteins. The protein localizes to dot-like structures throughout the nucleus, and redistributes to a zone near the nuclear envelope in cells undergoing apoptosis. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a transcriptional corepressor that is found tethered to promoter regions by DNA-binding proteins. The encoded protein can interact with several different class II histone deacetylases to repress transcription. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the IAP family of proteins that inhibit apoptosis by binding to tumor necrosis factor receptor-associated factors TRAF1 and TRAF2, probably by interfering with activation of ICE-like proteases. The encoded protein inhibits apoptosis induced by serum deprivation but does not affect apoptosis resulting from exposure to menadione, a potent inducer of free radicals. It contains 3 baculovirus IAP repeats and a ring finger domain. Transcript variants encoding the same isoform have been identified.","This gene encodes a protein with a BIR (baculoviral inhibition of apoptosis protein repeat) domain and a UBCc (ubiquitin-conjugating enzyme E2, catalytic) domain. This protein inhibits apoptosis by facilitating the degradation of apoptotic proteins by ubiquitination.","The bone morphogenetic protein (BMP) receptors are a family of transmembrane serine/threonine kinases that include the type I receptors BMPR1A and BMPR1B and the type II receptor BMPR2. These receptors are also closely related to the activin receptors, ACVR1 and ACVR2. The ligands of these receptors are members of the TGF-beta superfamily. TGF-betas and activins transduce their signals through the formation of heteromeric complexes with 2 different types of serine (threonine) kinase receptors: type I receptors of about 50-55 kD and type II receptors of about 70-80 kD. Type II receptors bind ligands in the absence of type I receptors, but they require their respective type I receptors for signaling, whereas type I receptors require their respective type II receptors for ligand binding.","The protein encoded by this gene is homologous to the murine protein MCAP, which associates with chromosomes during mitosis, and to the human RING3 protein, a serine/threonine kinase. Each of these proteins contains two bromodomains, a conserved sequence motif which may be involved in chromatin targeting. This gene has been implicated as the chromosome 19 target of translocation t(15;19)(q13;p13.1), which defines an upper respiratory tract carcinoma in young people. Two alternatively spliced transcript variants have been described.","This gene encodes a protein which is a member of the bromodomain-containing protein family. The product of this gene has been identified as a component of one form of the SWI/SNF chromatin remodeling complex, and as a protein which interacts with p53 and is required for p53-dependent oncogene-induced senescence which prevents tumor growth. Pseudogenes have been described on chromosomes 2, 3, 6, 13 and 14. Alternative splicing results in multiple transcript variants.","This gene is a member of an anti-proliferative gene family that regulates cell growth and differentiation. Expression of this gene is highest in the G0/G1 phases of the cell cycle and downregulated when cells progressed through G1. The encoded protein interacts with several nuclear receptors, and functions as a coactivator of cell differentiation. This locus has been shown to be involved in a t(8;12)(q24;q22) chromosomal translocation in a case of B-cell chronic lymphocytic leukemia.","The protein encoded by this gene is a member of the BTG/Tob family. This family has structurally related proteins that appear to have antiproliferative properties. This encoded protein is involved in the regulation of the G1/S transition of the cell cycle.","The protein encoded by this gene plays a crucial role in B-cell development. Mutations in this gene cause X-linked agammaglobulinemia type 1, which is an immunodeficiency characterized by the failure to produce mature B lymphocytes, and associated with a failure of Ig heavy chain rearrangement. Alternative splicing results in multiple transcript variants encoding different isoforms.","Voltage-dependent calcium channels mediate the entry of calcium ions into excitable cells, and are also involved in a variety of calcium-dependent processes, including muscle contraction, hormone or neurotransmitter release, and gene expression. Calcium channels are multisubunit complexes composed of alpha-1, beta, alpha-2/delta, and gamma subunits. The channel activity is directed by the pore-forming alpha-1 subunit, whereas the others act as auxiliary subunits regulating this activity. The distinctive properties of the calcium channel types are related primarily to the expression of a variety of alpha-1 isoforms, namely alpha-1A, B, C, D, E, and S. This gene encodes the alpha-1D subunit. Several transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene contains a CG1 DNA-binding domain, a transcription factor immunoglobulin domain, ankyrin repeats, and calmodulin-binding IQ motifs. The encoded protein is thought to be a transcription factor and may be a tumor suppressor. However, a translocation event is sometimes observed between this gene and the WWTR1 gene, with the resulting WWTR1-CAMTA1 oncoprotein leading to epithelioid hemangioendothelioma, a malignant vascular cancer.","This gene is a proto-oncogene that encodes a RING finger E3 ubiquitin ligase. The encoded protein is one of the enzymes required for targeting substrates for degradation by the proteasome. This protein mediates the transfer of ubiquitin from ubiquitin conjugating enzymes (E2) to specific substrates. This protein also contains an N-terminal phosphotyrosine binding domain that allows it to interact with numerous tyrosine-phosphorylated substrates and target them for proteasome degradation. As such it functions as a negative regulator of many signal transduction pathways. This gene has been found to be mutated or translocated in many cancers including acute myeloid leukaemia, and expansion of CGG repeats in the 5' UTR has been associated with Jacobsen syndrome. Mutations in this gene are also the cause of Noonan syndrome-like disorder.","This gene encodes an E3 ubiquitin-protein ligase which promotes proteosome-mediated protein degradation by transferring ubiquitin from an E2 ubiquitin-conjugating enzyme to a substrate. The encoded protein is involved in the regulation of immune response by limiting T-cell receptor, B-cell receptor, and high affinity immunoglobulin epsilon receptor activation. Studies in mouse suggest that this gene is involved in antifungal host defense and that its inhibition leads to increased fungal killing. Manipulation of this gene may be beneficial in implementing immunotherapies for a variety of conditions, including cancer, autoimmune diseases, allergies, and infections.","This gene encodes a coiled-coil domain-containing protein. The encoded protein is ubiquitously expressed and may function as a tumor suppressor. A chromosomal rearrangement resulting in the expression of a fusion gene containing a portion of this gene and the intracellular kinase-encoding domain of the ret proto-oncogene is the cause of thyroid papillary carcinoma.[provided by RefSeq, Sep 2010].","The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with CDK4 or CDK6 and functions as a regulatory subunit of the complex, whose activity is required for cell cycle G1/S transition. This protein has been shown to interact with and be involved in the phosphorylation of tumor suppressor protein Rb. Knockout studies of the homologous gene in mouse suggest the essential roles of this gene in ovarian granulosa and germ cell proliferation. High level expression of this gene was observed in ovarian and testicular tumors. Mutations in this gene are associated with megalencephaly-polymicrogyria-polydactyly-hydrocephalus syndrome 3 (MPPH3).","The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with and functions as a regulatory subunit of CDK4 or CDK6, whose activtiy is required for cell cycle G1/S transition. This protein has been shown to interact with and be involved in the phosphorylation of tumor suppressor protein Rb. The CDK4 activity associated with this cyclin was reported to be necessary for cell cycle progression through G2 phase into mitosis after UV radiation. Several transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a member of the G protein-coupled receptor family. This receptor was identified as a gene induced by the Epstein-Barr virus (EBV), and is thought to be a mediator of EBV effects on B lymphocytes. This receptor is expressed in various lymphoid tissues and activates B and T lymphocytes. It has been shown to control the migration of memory T cells to inflamed tissues, as well as stimulate dendritic cell maturation. The chemokine (C-C motif) ligand 19 (CCL19/ECL) has been reported to be a specific ligand of this receptor. Signals mediated by this receptor regulate T cell homeostasis in lymph nodes, and may also function in the activation and polarization of T cells, and in chronic inflammation pathogenesis. Alternative splicing of this gene results in multiple transcript variants.","This gene encodes a C-type lectin that functions in cell adhesion and pathogen recognition. This receptor recognizes a wide range of evolutionarily divergent pathogens with a large impact on public health, including leprosy and tuberculosis mycobacteria, the Ebola, hepatitis C, HIV-1 and Dengue viruses, and the SARS-CoV acute respiratory syndrome coronavirus. The protein is organized into four distinct domains: a C-terminal carbohydrate recognition domain, a flexible tandem-repeat neck domain, a transmembrane region and an N-terminal cytoplasmic domain involved in internalization. This gene is closely related in terms of both sequence and function to a neighboring gene, CLEC4M (Gene ID: 10332), also known as L-SIGN. The two genes differ in viral recognition and expression patterns, with this gene showing high expression on the surface of dendritic cells. Polymorphisms in the neck region are associated with protection from HIV-1 infection, while single nucleotide polymorphisms in the promoter of this gene are associated with differing resistance and susceptibility to and severity of infectious disease, including rs4804803, which is associated with SARS severity.","This gene encodes an immune inhibitory receptor ligand that is expressed by hematopoietic and non-hematopoietic cells, such as T cells and B cells and various types of tumor cells. The encoded protein is a type I transmembrane protein that has immunoglobulin V-like and C-like domains. Interaction of this ligand with its receptor inhibits T-cell activation and cytokine production. During infection or inflammation of normal tissue, this interaction is important for preventing autoimmunity by maintaining homeostasis of the immune response. In tumor microenvironments, this interaction provides an immune escape for tumor cells through cytotoxic T-cell inactivation. Expression of this gene in tumor cells is considered to be prognostic in many types of human malignancies, including colon cancer and renal cell carcinoma. Alternative splicing results in multiple transcript variants.","This gene encodes a tumor suppressor that is involved in transcriptional and post-transcriptional control pathways. The protein is a component of the the PAF protein complex, which associates with the RNA polymerase II subunit POLR2A and with a histone methyltransferase complex. This protein appears to facilitate the association of 3' mRNA processing factors with actively-transcribed chromatin. Mutations in this gene have been linked to hyperparathyroidism-jaw tumor syndrome, familial isolated hyperparathyroidism, and parathyroid carcinoma.","This gene encodes a type II classical cadherin of the cadherin superfamily. Alternative splicing of this gene results in multiple transcript variants. At least one of these variants encodes a preproprotein that is proteolytically processed to generate the mature cadherin protein. These integral membrane proteins mediate calcium-dependent cell-cell adhesion and are composed of a large N-terminal extracellular domain, a single membrane-spanning domain, and a small, highly conserved C-terminal cytoplasmic domain. The extracellular domain consists of 5 subdomains, each containing a cadherin motif, and appears to determine the specificity of the protein's homophilic cell adhesion activity. Type II (atypical) cadherins are defined based on their lack of a histidine-alanine-valine (HAV) cell adhesion recognition sequence specific to type I cadherins. This particular cadherin is predominantly expressed in brain and is putatively involved in synaptic adhesions, axon outgrowth and guidance. Mutations in this gene may be associated with lung squamous cell carcinoma and colorectal cancer in human patients.","This gene encodes a type II classical cadherin from the cadherin superfamily, integral membrane proteins that mediate calcium-dependent cell-cell adhesion. Mature cadherin proteins are composed of a large N-terminal extracellular domain, a single membrane-spanning domain, and a small, highly conserved C-terminal cytoplasmic domain. Type II (atypical) cadherins are defined based on their lack of a HAV cell adhesion recognition sequence specific to type I cadherins. Expression of this particular cadherin in osteoblastic cell lines, and its upregulation during differentiation, suggests a specific function in bone development and maintenance.","This gene is a member of the caudal-related homeobox transcription factor gene family. The encoded protein is a major regulator of intestine-specific genes involved in cell growth an differentiation. This protein also plays a role in early embryonic development of the intestinal tract. Aberrant expression of this gene is associated with intestinal inflammation and tumorigenesis.","The CHD family of proteins is characterized by the presence of chromo (chromatin organization modifier) domains and SNF2-related helicase/ATPase domains. CHD genes alter gene expression possibly by modification of chromatin structure thus altering access of the transcriptional apparatus to its chromosomal DNA template. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene.","This gene encodes a protein with an acidic transcriptional activation domain, 4 LRRs (leucine-rich repeats) and a GTP binding domain. The protein is located in the nucleus and acts as a positive regulator of class II major histocompatibility complex gene transcription, and is referred to as the 'master control factor' for the expression of these genes. The protein also binds GTP and uses GTP binding to facilitate its own transport into the nucleus. Once in the nucleus it does not bind DNA but rather uses an intrinsic acetyltransferase (AT) activity to act in a coactivator-like fashion. Mutations in this gene have been associated with bare lymphocyte syndrome type II (also known as hereditary MHC class II deficiency or HLA class II-deficient combined immunodeficiency), increased susceptibility to rheumatoid arthritis, multiple sclerosis, and possibly myocardial infarction. Several transcript variants encoding different isoforms have been found for this gene.","Clathrin is a major protein component of the cytoplasmic face of intracellular organelles, called coated vesicles and coated pits. These specialized organelles are involved in the intracellular trafficking of receptors and endocytosis of a variety of macromolecules.  The basic subunit of the clathrin coat is composed of three heavy chains and three light chains.","This gene is a member of the clathrin heavy chain family and encodes a major protein of the polyhedral coat of coated pits and vesicles. Chromosomal aberrations involving this gene are associated with meningioma, DiGeorge syndrome, and velo-cardio-facial syndrome. Multiple transcript variants encoding different isoforms have been found for this gene.",null,"Involved in regulation of stem cell population maintenance. Part of CCR4-NOT complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the highly conserved RCD1 protein family. The encoded protein is a transcriptional cofactor and a core protein of the CCR4-NOT complex. It may be involved in signal transduction as well as retinoic acid-regulated cell differentiation and development. Alternatively spliced transcript variants have been described for this gene.","This gene encodes the alpha-1 chain of type II collagen, a fibrillar collagen found in cartilage and the vitreous humor of the eye. Mutations in this gene are associated with achondrogenesis, chondrodysplasia, early onset familial osteoarthritis, SED congenita, Langer-Saldino achondrogenesis, Kniest dysplasia, Stickler syndrome type I, and spondyloepimetaphyseal dysplasia Strudwick type. In addition, defects in processing chondrocalcin, a calcium binding protein that is the C-propeptide of this collagen molecule, are also associated with chondrodysplasia. There are two transcripts identified for this gene.","Enables mRNA 3'-UTR binding activity and translation factor activity, RNA binding. Involved in cellular response to amino acid stimulus; negative regulation of transcription by RNA polymerase II; and positive regulation of mRNA catabolic process. Located in several cellular components, including cytosol; midbody; and nucleoplasm. Part of CCR4-NOT complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the type I cytokine receptor family. The encoded protein is a receptor for thymic stromal lymphopoietin (TSLP). Together with the interleukin 7 receptor (IL7R), the encoded protein and TSLP activate STAT3, STAT5, and JAK2 pathways, which control processes such as cell proliferation and development of the hematopoietic system. Rearrangement of this gene with immunoglobulin heavy chain gene (IGH) on chromosome 14, or with P2Y purinoceptor 8 gene (P2RY8) on the same X or Y chromosomes is associated with B-progenitor acute lymphoblastic leukemia (ALL) and Down syndrome ALL. Alternatively spliced transcript variants have been found for this gene.","Enables cAMP response element binding protein binding activity. Involved in positive regulation of transcription by RNA polymerase II. Located in cytosol; nuclear body; and plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is the receptor for colony stimulating factor 1, a cytokine which controls the production, differentiation, and function of macrophages. This receptor mediates most if not all of the biological effects of this cytokine. Ligand binding activates the receptor kinase through a process of oligomerization and transphosphorylation. The encoded protein is a tyrosine kinase transmembrane receptor and member of the CSF1/PDGF receptor family of tyrosine-protein kinases. Mutations in this gene have been associated with a predisposition to myeloid malignancy. The first intron of this gene contains a transcriptionally inactive ribosomal protein L7 processed pseudogene oriented in the opposite direction. Alternative splicing results in multiple transcript variants. Expression of a splice variant from an LTR promoter has been found in Hodgkin lymphoma (HL), HL cell lines and anaplastic large cell lymphoma.","The protein encoded by this gene is the receptor for colony stimulating factor 3, a cytokine that controls the production, differentiation, and function of granulocytes. The encoded protein, which is a member of the family of cytokine receptors, may also function in some cell surface adhesion or recognition processes. Alternatively spliced transcript variants have been described. Mutations in this gene are a cause of Kostmann syndrome, also known as severe congenital neutropenia.","Enables actin filament binding activity. Involved in negative regulation of Arp2/3 complex-mediated actin nucleation; regulation of neuron migration; and regulation of neuron projection development. Located in cytoplasm. Implicated in complex cortical dysplasia with other brain malformations. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the Armadillo protein family, which function in adhesion between cells and signal transduction. Multiple translation initiation codons and alternative splicing result in many different isoforms being translated. Not all of the full-length natures of the described transcript variants have been determined. Read-through transcription also exists between this gene and the neighboring upstream thioredoxin-related transmembrane protein 2 (TMX2) gene.","This gene encodes an adhesive junction associated protein of the armadillo/beta-catenin superfamily and is implicated in brain and eye development and cancer formation. The protein encoded by this gene promotes the disruption of E-cadherin based adherens junction to favor cell spreading upon stimulation by hepatocyte growth factor. This gene is overexpressed in prostate adenocarcinomas and is associated with decreased expression of tumor suppressor E-cadherin in this tissue. This gene resides in a region of the short arm of chromosome 5 that is deleted in Cri du Chat syndrome. Alternative splicing results in multiple transcript variants encoding different isoforms.","The protein encoded by this gene is a member of the homeodomain family of DNA binding proteins. It may regulate gene expression, morphogenesis, and differentiation and it may also play a role in the cell cycle progession. Several alternatively spliced transcript variants encoding different isoforms have been identified.[provided by RefSeq, Feb 2011].","This gene encodes a CXC chemokine receptor specific for stromal cell-derived factor-1. The protein has 7 transmembrane regions and is located on the cell surface. It acts with the CD4 protein to support HIV entry into cells and is also highly expressed in breast cancer cells. Mutations in this gene have been associated with WHIM (warts, hypogammaglobulinemia, infections, and myelokathexis) syndrome. Alternate transcriptional splice variants, encoding different isoforms, have been characterized.","This gene encodes a member of the cytochrome P450 superfamily of enzymes. The cytochrome P450 proteins are monooxygenases which catalyze many reactions involved in drug metabolism and synthesis of cholesterol, steroids and other lipids. This protein localizes to the endoplasmic reticulum and its expression is induced by phenobarbital. The enzyme is known to metabolize many xenobiotics, including the anticonvulsive drug mephenytoin, benzo(a)pyrene, 7-ethyoxycoumarin, and the anti-cancer drug taxol. This gene is located within a cluster of cytochrome P450 genes on chromosome 10q24. Several transcript variants encoding a few different isoforms have been found for this gene.","This gene encodes a multifunctional protein that resides in multiple locations in the nucleus and in the cytoplasm. It interacts with a wide variety of proteins, such as apoptosis antigen Fas, centromere protein C, and transcription factor erythroblastosis virus E26 oncogene homolog 1. In the nucleus, the encoded protein functions as a potent transcription repressor that binds to sumoylated transcription factors. Its repression can be relieved by the sequestration of this protein into promyelocytic leukemia nuclear bodies or nucleoli. This protein also associates with centromeres in G2 phase. In the cytoplasm, the encoded protein may function to regulate apoptosis. The subcellular localization and function of this protein are modulated by post-translational modifications, including sumoylation, phosphorylation and polyubiquitination. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the WD repeat protein family. WD repeats are minimally conserved regions of approximately 40 amino acids typically bracketed by Gly-His and Trp-Asp (GH-WD), which may facilitate formation of heterotrimeric or multi-protein complexes. Members of this family are involved in a variety of cellular processes, including cell cycle progression, signal transduction, apoptosis, and gene regulation. This gene appears to represent an intronless retrocopy of a related multi-exon gene located on chromosome 9. However, the CDS of this intronless gene remains intact, it is conserved in other mammalian species, it is known to be transcribed, and it is therefore thought to encode a functional protein.","This gene encodes a netrin 1 receptor. The transmembrane protein is a member of the immunoglobulin superfamily of cell adhesion molecules, and mediates axon guidance of neuronal growth cones towards sources of netrin 1 ligand. The cytoplasmic tail interacts with the tyrosine kinases Src and focal adhesion kinase (FAK, also known as PTK2) to mediate axon attraction. The protein partially localizes to lipid rafts, and induces apoptosis in the absence of ligand. The protein functions as a tumor suppressor, and is frequently mutated or downregulated in colorectal cancer and esophageal carcinoma.","This gene encodes a protein that is necessary for the repair of ultraviolet light-damaged DNA. This protein is the smaller subunit of a heterodimeric protein complex that participates in nucleotide excision repair, and this complex mediates the ubiquitylation of histones H3 and H4, which facilitates the cellular response to DNA damage. This subunit appears to be required for DNA binding. Mutations in this gene cause xeroderma pigmentosum complementation group E, a recessive disease that is characterized by an increased sensitivity to UV light and a high predisposition for skin cancer development, in some cases accompanied by neurological abnormalities. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the discoidin domain receptor subclass of the receptor tyrosine kinase (RTKs) protein family. RTKs play a key role in the communication of cells with their microenvironment. The encoded protein is a collagen-induced receptor that activates signal transduction pathways involved in cell adhesion, proliferation, and extracellular matrix remodeling. This protein is expressed in numerous cell types and may alos be involved in wound repair and regulate tumor growth and invasiveness. Mutations in this gene are the cause of short limb-hand type spondylometaepiphyseal dysplasia.","This gene encodes a member of the DEAD box protein family. The protein is an RNA helicase found in P-bodies and stress granules, and functions in translation suppression and mRNA degradation. It is required for microRNA-induced gene silencing. Multiple alternatively spliced variants, encoding the same protein, have been identified.","This gene encodes a subunit of the microprocessor complex which mediates the biogenesis of microRNAs from the primary microRNA transcript. The encoded protein is a double-stranded RNA binding protein that functions as the non-catalytic subunit of the microprocessor complex. This protein is required for binding the double-stranded RNA substrate and facilitates cleavage of the RNA by the ribonuclease III protein, Drosha. Alternate splicing results in multiple transcript variants.","This gene encodes a member of the DEAH-containing family of RNA helicases. The encoded protein is an enzyme that catalyzes the ATP-dependent unwinding of double-stranded RNA and DNA-RNA complexes. This protein localizes to both the nucleus and the cytoplasm and functions as a transcriptional regulator. This protein may also be involved in the expression and nuclear export of retroviral RNAs. Alternate splicing results in multiple transcript variants. Pseudogenes of this gene are found on chromosomes 11 and 13.[provided by RefSeq, Feb 2010].","This gene encodes a ribonuclease (RNase) III double-stranded RNA-specific ribonuclease and subunit of the microprocessor protein complex, which catalyzes the initial processing step of microRNA (miRNA) synthesis. The encoded protein cleaves the stem loop structure from the primary microRNA (pri-miRNA) in the nucleus, yielding the precursor miRNA (pre-miRNA), which is then exported to the cytoplasm for further processing. In a human cell line lacking a functional copy of this gene, canonical miRNA synthesis is reduced. Somatic mutations in this gene have been observed in human patients with kidney cancer.","Predicted to enable DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in positive regulation of transcription by RNA polymerase II. Predicted to act upstream of or within positive regulation of transcription, DNA-templated. Predicted to be located in nucleus. Predicted to be part of chromatin. [provided by Alliance of Genome Resources, Apr 2022]","Enables protein N-terminus binding activity. Contributes to translation initiation factor activity. Involved in positive regulation of mRNA binding activity; regulation of gene expression; and translational initiation. Located in cytosol and nucleus. Part of eukaryotic translation initiation factor 3 complex. Colocalizes with PML body. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a transcriptional activator that binds and activates the promoters of the CSF2, IL3, IL8, and PRF1 genes. The encoded protein is involved in natural killer cell development and function, innate immunity, and induction of cell cycle arrest in naive CD8+ cells. Two transcript variants encoding the same protein have been found for this gene.[provided by RefSeq, Jan 2010].","This gene is a member of the Ets family of transcription factors and of the ternary complex factor (TCF) subfamily. Proteins of the TCF subfamily form a ternary complex by binding to the the serum response factor and the serum reponse element in the promoter of the c-fos proto-oncogene. The protein encoded by this gene is phosphorylated by the kinases, MAPK1 and MAPK8. Several transcript variants have been described for this gene.","Enables phosphatase binding activity. Involved in positive regulation of transcription, DNA-templated and snRNA transcription. Located in cytosol; nuclear body; and transcriptionally active chromatin. Part of transcription elongation factor complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene belongs to the ephrin receptor subfamily of the protein-tyrosine kinase family. EPH and EPH-related receptors have been implicated in mediating developmental events, particularly in the nervous system. Receptors in the EPH subfamily typically have a single kinase domain and an extracellular region containing a Cys-rich domain and 2 fibronectin type III repeats. The ephrin receptors are divided into 2 groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. This gene encodes a protein that binds ephrin-A ligands. Mutations in this gene are the cause of certain genetically-related cataract disorders.[provided by RefSeq, May 2010].","This gene belongs to the ephrin receptor subfamily of the protein-tyrosine kinase family. EPH and EPH-related receptors have been implicated in mediating developmental events, particularly in the nervous system. Receptors in the EPH subfamily typically have a single kinase domain and an extracellular region containing a Cys-rich domain and 2 fibronectin type III repeats. The ephrin receptors are divided into 2 groups based on the similarity of their extracellular domain sequences and their affinities for binding ephrin-A and ephrin-B ligands. Increased expression of this gene is associated with multiple forms of carcinoma. Alternative splicing results in multiple transcript variants.","This gene encodes a protein that is part of the EGFR pathway. The protein is present at clatherin-coated pits and is involved in receptor-mediated endocytosis of EGF. Notably, this gene is rearranged with the HRX/ALL/MLL gene in acute myelogeneous leukemias. Alternative splicing results in multiple transcript variants encoding distinct isoforms.","This gene encodes an ATP-dependent DNA helicase that functions in nucleotide excision repair. The encoded protein is a subunit of basal transcription factor 2 (TFIIH) and, therefore, also functions in class II transcription. Mutations in this gene are associated with Xeroderma pigmentosum B, Cockayne's syndrome, and trichothiodystrophy. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the erythroblast transformation-specific (ETS) family of transcriptions factors. All members of this family are key regulators of embryonic development, cell proliferation, differentiation, angiogenesis, inflammation, and apoptosis. The protein encoded by this gene is mainly expressed in the nucleus. It contains an ETS DNA-binding domain and a PNT (pointed) domain which is implicated in the self-association of chimeric oncoproteins. This protein is required for platelet adhesion to the subendothelium, inducing vascular cell remodeling. It also regulates hematopoesis, and the differentiation and maturation of megakaryocytic cells. This gene is involved in chromosomal translocations, resulting in different fusion gene products, such as TMPSSR2-ERG and NDRG1-ERG in prostate cancer, EWS-ERG in Ewing's sarcoma and FUS-ERG in acute myeloid leukemia. More than two dozens of transcript variants generated from combinatorial usage of three alternative promoters and multiple alternative splicing events have been reported, but the full-length nature of many of these variants has not been determined.","This gene encodes a member of the ETS (E twenty-six) family of transcription factors. The ETS proteins regulate many target genes that modulate biological processes like cell growth, angiogenesis, migration, proliferation and differentiation. All ETS proteins contain an ETS DNA-binding domain that binds to DNA sequences containing the consensus 5'-CGGA[AT]-3'. The protein encoded by this gene contains a conserved short acidic transactivation domain (TAD) in the N-terminal region, in addition to the ETS DNA-binding domain in the C-terminal region. This gene is involved in chromosomal translocations, which result in multiple fusion proteins including EWS-ETV1 in Ewing sarcoma and at least 10 ETV1 partners (see PMID: 19657377, Table 1) in prostate cancer. In addition to chromosomal rearrangement, this gene is overexpressed in prostate cancer, melanoma and gastrointestinal stromal tumor. Multiple alternatively spliced transcript variants encoding different isoforms have been identified.","Enables DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Involved in positive regulation of keratinocyte differentiation and positive regulation of transcription by RNA polymerase II. Located in chromosome and nucleolus. [provided by Alliance of Genome Resources, Apr 2022]","Enables DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Involved in cellular response to oxidative stress; negative regulation of transcription by RNA polymerase II; and positive regulation of transcription by RNA polymerase II. Located in nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes an ETS family transcription factor. The product of this gene contains two functional domains: a N-terminal pointed (PNT) domain that is involved in protein-protein interactions with itself and other proteins, and a C-terminal DNA-binding domain. Gene knockout studies in mice suggest that it is required for hematopoiesis and maintenance of the developing vascular network. This gene is known to be involved in a large number of chromosomal rearrangements associated with leukemia and congenital fibrosarcoma.","This gene encodes a multifunctional protein that is involved in various cellular processes, including gene expression, cell signaling, and RNA processing and transport. The protein includes an N-terminal transcriptional activation domain and a C-terminal RNA-binding domain. Chromosomal translocations between this gene and various genes encoding transcription factors result in the production of chimeric proteins that are involved in tumorigenesis. These chimeric proteins usually consist of the N-terminal transcriptional activation domain of this protein fused to the C-terminal DNA-binding domain of the transcription factor protein. Mutations in this gene, specifically a t(11;22)(q24;q12) translocation, are known to cause Ewing sarcoma as well as neuroectodermal and various other tumors. Alternative splicing of this gene results in multiple transcript variants. Related pseudogenes have been identified on chromosomes 1 and 14.","Predicted to be involved in cellular lipid metabolic process. [provided by Alliance of Genome Resources, Apr 2022]","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group A. Alternative splicing results in multiple transcript variants encoding different isoforms. Mutations in this gene are the most common cause of Fanconi anemia.","The protein encoded by this gene is a member of the TNF-receptor superfamily. This receptor contains a death domain. It has been shown to play a central role in the physiological regulation of programmed cell death, and has been implicated in the pathogenesis of various malignancies and diseases of the immune system. The interaction of this receptor with its ligand allows the formation of a death-inducing signaling complex that includes Fas-associated death domain protein (FADD), caspase 8, and caspase 10. The autoproteolytic processing of the caspases in the complex triggers a downstream caspase cascade, and leads to apoptosis. This receptor has been also shown to activate NF-kappaB, MAPK3/ERK1, and MAPK8/JNK, and is found to be involved in transducing the proliferating signals in normal diploid fibroblast and T cells. Several alternatively spliced transcript variants have been described, some of which are candidates for nonsense-mediated mRNA decay (NMD). The isoforms lacking the transmembrane domain may negatively regulate the apoptosis mediated by the full length isoform.","Predicted to enable calcium ion binding activity. Predicted to be involved in cell-cell adhesion. Predicted to act upstream of or within several processes, including negative regulation of dendrite development; neuron migration; and retina layer formation. Predicted to be located in dendrite and plasma membrane. Predicted to be integral component of membrane. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a member of the protocadherin family. This gene may play a role in regulating planar cell polarity (PCP). Studies in mice suggest that loss of PCP signaling may cause cystic kidney disease, and mutations in this gene have been associated with Van Maldergem Syndrome 2. Alternatively spliced transcript variants have been noted for this gene.","This gene encodes an extracellular matrix protein, which belongs to the fibulin family. This protein binds various extracellular ligands and calcium. It may play a role during organ development, in particular, during the differentiation of heart, skeletal and neuronal structures. Alternatively spliced transcript variants encoding different isoforms have been identified.","This gene encodes a member of the F-box protein family which is characterized by an approximately 40 amino acid motif, the F-box. The F-box proteins constitute one of the four subunits of ubiquitin protein ligase complex called SCFs (SKP1-cullin-F-box), which function in phosphorylation-dependent ubiquitination. The F-box proteins are divided into 3 classes: Fbws containing WD-40 domains, Fbls containing leucine-rich repeats, and Fbxs containing either different protein-protein interaction modules or no recognizable motifs. The protein encoded by this gene belongs to the Fbxs class. It can function as an arginine methyltransferase that symmetrically dimethylates arginine residues, and it acts as an adaptor protein to mediate the neddylation of p53, which leads to the suppression of p53 function. This gene is known to be down-regulated in melanocytes from patients with vitiligo, a skin disorder that results in depigmentation. Polymorphisms in this gene are associated with chronic otitis media with effusion and recurrent otitis media (COME/ROM), a hearing loss disorder, and the knockout of the homologous mouse gene results in the deaf mouse mutant Jeff (Jf), a single gene model of otitis media. Alternatively spliced transcript variants encoding distinct isoforms have been identified for this gene.","The protein encoded by this gene is a low affinity receptor for the Fc region of immunoglobulin gamma complexes. The encoded protein is involved in the phagocytosis of immune complexes and in the regulation of antibody production by B-cells. Variations in this gene may increase susceptibilty to systemic lupus erythematosus (SLE). Several transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the immunoglobulin receptor superfamily and is one of several Fc receptor-like glycoproteins clustered on the long arm of chromosome 1. The encoded protein has four extracellular C2-type immunoglobulin domains, a transmembrane domain and a cytoplasmic domain that contains three immune-receptor tyrosine-based inhibitory motifs. This protein may play a role in the function of memory B-cells in the epithelia. Aberrations in the chromosomal region encoding this gene are associated with non-Hodgkin lymphoma and multiple myeloma.","The protein encoded by this gene is a tyrosine kinase and cell surface receptor for fibroblast growth factors. The encoded protein is involved in the regulation of several pathways, including cell proliferation, cell differentiation, cell migration, lipid metabolism, bile acid biosynthesis, vitamin D metabolism, glucose uptake, and phosphate homeostasis. This protein consists of an extracellular region, composed of three immunoglobulin-like domains, a single hydrophobic membrane-spanning segment, and a cytoplasmic tyrosine kinase domain. The extracellular portion interacts with fibroblast growth factors, setting in motion a cascade of downstream signals, ultimately influencing mitogenesis and differentiation.","The protein encoded by this gene is a P1-P3-bis(5'-adenosyl) triphosphate hydrolase involved in purine metabolism. This gene encompasses the common fragile site FRA3B on chromosome 3, where carcinogen-induced damage can lead to translocations and aberrant transcripts. In fact, aberrant transcripts from this gene have been found in about half of all esophageal, stomach, and colon carcinomas. The encoded protein is also a tumor suppressor, as loss of its activity results in replication stress and DNA damage.","This gene is located within the Smith-Magenis syndrome region on chromosome 17. Mutations in this gene are associated with Birt-Hogg-Dube syndrome, which is characterized by fibrofolliculomas, renal tumors, lung cysts, and pneumothorax. Alternative splicing of this gene results in two transcript variants encoding different isoforms.","The protein encoded by this gene is an actin-binding protein that crosslinks actin filaments and links actin filaments to membrane glycoproteins. The encoded protein is involved in remodeling the cytoskeleton to effect changes in cell shape and migration. This protein interacts with integrins, transmembrane receptor complexes, and second messengers. Defects in this gene are a cause of several syndromes, including periventricular nodular heterotopias (PVNH1, PVNH4), otopalatodigital syndromes (OPD1, OPD2), frontometaphyseal dysplasia (FMD), Melnick-Needles syndrome (MNS), and X-linked congenital idiopathic intestinal pseudoobstruction (CIIPX). Two transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Mar 2009].","This gene encodes a tyrosine kinase receptor for vascular endothelial growth factors C and D. The protein is thought to be involved in lymphangiogenesis and maintenance of the lymphatic endothelium. Mutations in this gene cause hereditary lymphedema type IA.","This gene encodes a member of the forkhead class of DNA-binding proteins. These hepatocyte nuclear factors are transcriptional activators for liver-specific genes such as albumin and transthyretin, and they also interact with chromatin. Similar family members in mice have roles in the regulation of metabolism and in the differentiation of the pancreas and liver. This gene has been linked to sporadic cases of maturity-onset diabetes of the young. Transcript variants encoding different isoforms have been identified for this gene.","This gene encodes a forkhead transcription factor. The protein contains a fork-head DNA-binding domain and may play a role in ovarian development and function. Expansion of a polyalanine repeat region and other mutations in this gene are a cause of blepharophimosis syndrome and premature ovarian failure 3.","This gene belongs to the forkhead family of transcription factors which are characterized by a distinct forkhead domain.  The specific function of this gene has not yet been determined; however, it may play a role in myogenic growth and differentiation. Translocation of this gene with PAX3 has been associated with alveolar rhabdomyosarcoma.","This gene belongs to the forkhead family of transcription factors which are characterized by a distinct forkhead domain. This gene likely functions as a trigger for apoptosis through expression of genes necessary for cell death. Translocation of this gene with the MLL gene is associated with secondary acute leukemia. Alternatively spliced transcript variants encoding the same protein have been observed.","This gene belongs to subfamily P of the forkhead box (FOX) transcription factor family. Forkhead box transcription factors play important roles in the regulation of tissue- and cell type-specific gene transcription during both development and adulthood. Forkhead box P1 protein contains both DNA-binding- and protein-protein binding-domains. This gene may act as a tumor suppressor as it is lost in several tumor types and maps to a chromosomal region (3p14.1) reported to contain a tumor suppressor gene(s). Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a multifunctional protein component of the heterogeneous nuclear ribonucleoprotein (hnRNP) complex. The hnRNP complex is involved in pre-mRNA splicing and the export of fully processed mRNA to the cytoplasm. This protein belongs to the FET family of RNA-binding proteins which have been implicated in cellular processes that include regulation of gene expression, maintenance of genomic integrity and mRNA/microRNA processing. Alternative splicing results in multiple transcript variants. Defects in this gene result in amyotrophic lateral sclerosis type 6.","This gene encodes a member of the GATA family of zinc-finger transcription factors that are named for the consensus nucleotide sequence they bind in the promoter regions of target genes. The encoded protein plays an essential role in regulating transcription of genes involved in the development and proliferation of hematopoietic and endocrine cell lineages. Alternative splicing results in multiple transcript variants.[provided by RefSeq, Mar 2009].","This gene encodes a member of the Kruppel family of zinc finger proteins. The encoded transcription factor is activated by the sonic hedgehog signal transduction cascade and regulates stem cell proliferation. The activity and nuclear localization of this protein is negatively regulated by p53 in an inhibitory loop. Multiple transcript variants encoding different isoforms have been found for this gene.","Predicted to enable D5 dopamine receptor binding activity; G-protein beta/gamma-subunit complex binding activity; and GTPase activity. Predicted to be involved in several processes, including Rho protein signal transduction; activation of phospholipase D activity; and multicellular organism aging. Predicted to act upstream of or within several processes, including branching involved in blood vessel morphogenesis; negative regulation of vascular associated smooth muscle cell migration; and negative regulation of vascular associated smooth muscle cell proliferation. Located in cytosol and nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Cell surface heparan sulfate proteoglycans are composed of a membrane-associated protein core substituted with a variable number of heparan sulfate chains. Members of the glypican-related integral membrane proteoglycan family (GRIPS) contain a core protein anchored to the cytoplasmic membrane via a glycosyl phosphatidylinositol linkage.  These proteins may play a role in the control of cell division and growth regulation.","This gene encodes a member of the glutamate-gated ion channel protein family. The encoded protein is an N-methyl-D-aspartate (NMDA) receptor subunit. NMDA receptors are both ligand-gated and voltage-dependent, and are involved in long-term potentiation, an activity-dependent increase in the efficiency of synaptic transmission thought to underlie certain kinds of memory and learning. These receptors are permeable to calcium ions, and activation results in a calcium influx into post-synaptic cells, which results in the activation of several signaling cascades. Disruption of this gene is associated with focal epilepsy and speech disorder with or without cognitive disability. Alternative splicing results in multiple transcript variants.","L-glutamate is the major excitatory neurotransmitter in the central nervous system and activates both ionotropic and metabotropic glutamate receptors. Glutamatergic neurotransmission is involved in most aspects of normal brain function and can be perturbed in many neuropathologic conditions. The metabotropic glutamate receptors are a family of G protein-coupled receptors, that have been divided into 3 groups on the basis of sequence homology, putative signal transduction mechanisms, and pharmacologic properties. Group I includes GRM1 and GRM5 and these receptors have been shown to activate phospholipase C. Group II includes GRM2 and GRM3 while Group III includes GRM4, GRM6, GRM7 and GRM8. Group II and III receptors are linked to the inhibition of the cyclic AMP cascade but differ in their agonist selectivities.","This gene encodes a phosphoprotein containing six characteristic repeat motifs. The encoded protein binds to the initiator element (Inr) and E-box element in promoters and functions as a regulator of transcription. This locus, along with several other neighboring genes, is deleted in Williams-Beuren syndrome. There are many closely related genes and pseudogenes for this gene on chromosome 7. This gene also has pseudogenes on chromosomes 9, 13, and 21. Alternatively spliced transcript variants encoding multiple isoforms have been observed.","Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. This structure consists of approximately 146 bp of DNA wrapped around a nucleosome, an octamer composed of pairs of each of the four core histones (H2A, H2B, H3, and H4). The chromatin fiber is further compacted through the interaction of a linker histone, H1, with the DNA between the nucleosomes to form higher order chromatin structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H3 family. Transcripts from this gene lack polyA tails; instead, they contain a palindromic termination element. This gene is found in the large histone gene cluster on chromosome 6p22-p21.3.","This gene encodes a protein that binds to the hepatocyte growth factor receptor to regulate cell growth, cell motility and morphogenesis in numerous cell and tissue types. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate alpha and beta chains, which form the mature heterodimer. This protein is secreted by mesenchymal cells and acts as a multi-functional cytokine on cells of mainly epithelial origin. This protein also plays a role in angiogenesis, tumorogenesis, and tissue regeneration. Although the encoded protein is a member of the peptidase S1 family of serine proteases, it lacks peptidase activity. Mutations in this gene are associated with nonsyndromic hearing loss.","This gene encodes the alpha subunit of transcription factor hypoxia-inducible factor-1 (HIF-1), which is a heterodimer composed of an alpha and a beta subunit. HIF-1 functions as a master regulator of cellular and systemic homeostatic response to hypoxia by activating transcription of many genes, including those involved in energy metabolism, angiogenesis, apoptosis, and other genes whose protein products increase oxygen delivery or facilitate metabolic adaptation to hypoxia. HIF-1 thus plays an essential role in embryonic vascularization, tumor angiogenesis and pathophysiology of ischemic disease. Alternatively spliced transcript variants encoding different isoforms have been identified for this gene.","The product of this gene is a membrane-associated protein that functions in clathrin-mediated endocytosis and protein trafficking within the cell. The encoded protein binds to the huntingtin protein in the brain; this interaction is lost in Huntington's disease. Alternative splicing results in multiple transcript variants.","HLA-A belongs to the HLA class I heavy chain paralogues. This class I molecule is a heterodimer consisting of a heavy chain and a light chain (beta-2 microglobulin). The heavy chain is anchored in the membrane. Class I molecules play a central role in the immune system by presenting peptides derived from the endoplasmic reticulum lumen so that they can be recognized by cytotoxic T cells. They are expressed in nearly all cells. The heavy chain is approximately 45 kDa and its gene contains 8 exons. Exon 1 encodes the leader peptide, exons 2 and 3 encode the alpha1 and alpha2 domains, which both bind the peptide, exon 4 encodes the alpha3 domain, exon 5 encodes the transmembrane region, and exons 6 and 7 encode the cytoplasmic tail. Polymorphisms within exon 2 and exon 3 are responsible for the peptide binding specificity of each class one molecule. Typing for these polymorphisms is routinely done for bone marrow and kidney transplantation. More than 6000 HLA-A alleles have been described. The HLA system plays an important role in the occurrence and outcome of infectious diseases, including those caused by the malaria parasite, the human immunodeficiency virus (HIV), and the severe acute respiratory syndrome coronavirus (SARS-CoV). The structural spike and the nucleocapsid proteins of the novel coronavirus SARS-CoV-2, which causes coronavirus disease 2019 (COVID-19), are reported to contain multiple Class I epitopes with predicted HLA restrictions. Individual HLA genetic variation may help explain different immune responses to a virus across a population.[provided by RefSeq, Aug 2020].","HLA-B belongs to the HLA class I heavy chain paralogues. This class I molecule is a heterodimer consisting of a heavy chain and a light chain (beta-2 microglobulin). The heavy chain is anchored in the membrane. Class I molecules play a central role in the immune system by presenting peptides derived from the endoplasmic reticulum lumen. They are expressed in nearly all cells. The heavy chain is approximately 45 kDa and its gene contains 8 exons. Exon 1 encodes the leader peptide, exon 2 and 3 encode the alpha1 and alpha2 domains, which both bind the peptide, exon 4 encodes the alpha3 domain, exon 5 encodes the transmembrane region and exons 6 and 7 encode the cytoplasmic tail. Polymorphisms within exon 2 and exon 3 are responsible for the peptide binding specificity of each class one molecule. Typing for these polymorphisms is routinely done for bone marrow and kidney transplantation. Hundreds of HLA-B alleles have been described.","The protein encoded by this gene is a transcription factor required for the expression of several liver-specific genes. The encoded protein functions as a homodimer and binds to the inverted palindrome 5'-GTTAATNATTAAC-3'. Defects in this gene are a cause of maturity onset diabetes of the young type 3 (MODY3) and also can result in the appearance of hepatic adenomas. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene belongs to the A/B subfamily of ubiquitously expressed heterogeneous nuclear ribonucleoproteins (hnRNPs). The hnRNPs are RNA binding proteins and they complex with heterogeneous nuclear RNA (hnRNA). These proteins are associated with pre-mRNAs in the nucleus and appear to influence pre-mRNA processing and other aspects of mRNA metabolism and transport. While all of the hnRNPs are present in the nucleus, some seem to shuttle between the nucleus and the cytoplasm. The hnRNP proteins have distinct nucleic acid binding properties. The protein encoded by this gene has two repeats of quasi-RRM domains that bind to RNAs. This gene has been described to generate two alternatively spliced transcript variants which encode different isoforms.","This gene belongs to the homeobox family of genes. The homeobox genes encode a highly conserved family of transcription factors that play an important role in morphogenesis in all multicellular organisms. Mammals possess four similar homeobox gene clusters, HOXA, HOXB, HOXC and HOXD, which are located on different chromosomes and consist of 9 to 11 genes arranged in tandem. This gene is one of several homeobox HOXC genes located in a cluster on chromosome 12. The product of this gene may play a role in the development of hair, nail, and filiform papilla.","The protein encoded by this gene is a helix-loop-helix (HLH) protein that can form heterodimers with other HLH proteins. However, the encoded protein lacks a basic DNA-binding domain and therefore inhibits the DNA binding of any HLH protein with which it interacts.","The protein encoded by this gene phosphorylates the inhibitor in the inhibitor/NF-kappa-B complex, causing dissociation of the inhibitor and activation of NF-kappa-B. The encoded protein itself is found in a complex of proteins. Several transcript variants, some protein-coding and some not, have been found for this gene.","This gene encodes a transcription factor that belongs to the family of zinc-finger DNA-binding proteins associated with chromatin remodeling. The expression of this protein is restricted to the fetal and adult hemo-lymphopoietic system, and it functions as a regulator of lymphocyte differentiation. Several alternatively spliced transcript variants encoding different isoforms have been described for this gene. Most isoforms share a common C-terminal domain, which contains two zinc finger motifs that are required for hetero- or homo-dimerization, and for interactions with other proteins. The isoforms, however, differ in the number of N-terminal zinc finger motifs that bind DNA and in nuclear localization signal presence, resulting in members with and without DNA-binding properties. Only a few isoforms contain the requisite three or more N-terminal zinc motifs that confer high affinity binding to a specific core DNA sequence element in the promoters of target genes. The non-DNA-binding isoforms are largely found in the cytoplasm, and are thought to function as dominant-negative factors. Overexpression of some dominant-negative isoforms have been associated with B-cell malignancies, such as acute lymphoblastic leukemia (ALL).","The protein encoded by this gene is an SH2-containing 5'-inositol phosphatase that is involved in the regulation of insulin function. The encoded protein also plays a role in the regulation of epidermal growth factor receptor turnover and actin remodelling. Additionally, this gene supports metastatic growth in breast cancer and is a valuable biomarker for breast cancer.","The protein encoded by this gene belongs to the IRF (interferon regulatory factor) family of transcription factors, characterized by an unique tryptophan pentad repeat DNA-binding domain. The IRFs are important in the regulation of interferons in response to infection by virus, and in the regulation of interferon-inducible genes. This family member is lymphocyte specific and negatively regulates Toll-like-receptor (TLR) signaling that is central to the activation of innate and adaptive immune systems. A chromosomal translocation involving this gene and the IgH locus, t(6;14)(p25;q32), may be a cause of multiple myeloma. Alternatively spliced transcript variants have been found for this gene.","IRS4 encodes the insulin receptor substrate 4, a cytoplasmic protein that contains many potential tyrosine and serine/threonine phosphorylation sites. Tyrosine-phosphorylated IRS4 protein has been shown to associate with cytoplasmic signalling molecules that contain SH2 domains. The IRS4 protein is phosphorylated by the insulin receptor tyrosine kinase upon receptor stimulation..","The product of this gene belongs to the integrin alpha chain family. Integrins are heterodimeric integral membrane proteins composed of an alpha subunit and a beta subunit that function in cell surface adhesion and signaling. The encoded preproprotein is proteolytically processed to generate light and heavy chains that comprise the alpha V subunit. This subunit associates with beta 1, beta 3, beta 5, beta 6 and beta 8 subunits. The heterodimer consisting of alpha V and beta 3 subunits is also known as the vitronectin receptor. This integrin may regulate angiogenesis and cancer progression. Alternative splicing results in multiple transcript variants. Note that the integrin alpha 5 and integrin alpha V subunits are encoded by distinct genes.","This gene encodes a member of the MOZ, YBFR2, SAS2, TIP60 family of histone acetyltransferases. The protein is composed of a nuclear localization domain, a double C2H2 zinc finger domain that binds to acetylated histone tails, a histone acetyl-transferase domain, a glutamate/aspartate-rich region, and a serine- and methionine-rich transactivation domain. It is part of a complex that acetylates lysine-9 residues in histone 3, and in addition, it acts as a co-activator for several transcription factors. Allelic variants of this gene are associated with an autosomal dominant form of cognitive disability. Chromosomal translocations of this gene are associated with acute myeloid leukemia. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a histone acetyltransferase and component of the MOZ/MORF protein complex. In addition to its acetyltransferase activity, the encoded protein has transcriptional activation activity in its N-terminal end and transcriptional repression activity in its C-terminal end. This protein is necessary for RUNX2-dependent transcriptional activation and could be involved in brain development. Mutations have been found in patients with genitopatellar syndrome. A translocation of this gene and the CREBBP gene results in acute myeloid leukemias. Three transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is part of the multimeric HBO1 complex, which possesses histone H4-specific acetyltransferase activity. This activity is required for functional replication origins and is involved in transcriptional activation of some genes. In both cases, the acetylation of histone H4 helps unfold chromatin so that the DNA can be accessed and replicated or transcribed.","This gene encodes a member of the Jumonji, AT-rich interactive domain 1 (JARID1) histone demethylase protein family. The encoded protein plays a role in gene regulation through the histone code by specifically demethylating lysine 4 of histone H3. The encoded protein interacts with many other proteins, including retinoblastoma protein, and is implicated in the transcriptional regulation of Hox genes and cytokines. This gene may play a role in tumor progression.","Vascular endothelial growth factor (VEGF) is a major growth factor for endothelial cells. This gene encodes one of the two receptors of the VEGF. This receptor, known as kinase insert domain receptor, is a type III receptor tyrosine kinase. It functions as the main mediator of VEGF-induced endothelial proliferation, survival, migration, tubular morphogenesis and sprouting. The signalling and trafficking of this receptor are regulated by multiple factors, including Rab GTPase, P2Y purine nucleotide receptor, integrin alphaVbeta3, T-cell protein tyrosine phosphatase, etc.. Mutations of this gene are implicated in infantile capillary hemangiomas.","This gene encodes a protein that belongs to the Kruppel family of transcription factors. The encoded zinc finger protein is required for normal development of the barrier function of skin. The encoded protein is thought to control the G1-to-S transition of the cell cycle following DNA damage by mediating the tumor suppressor gene p53. Mice lacking this gene have a normal appearance but lose weight rapidly, and die shortly after birth due to fluid evaporation resulting from compromised epidermal barrier function. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a member of the Kruppel-like factor subfamily of zinc finger proteins. The encoded protein is a transcriptional activator that binds directly to a specific recognition motif in the promoters of target genes. This protein acts downstream of multiple different signaling pathways and is regulated by post-translational modification. It may participate in both promoting and suppressing cell proliferation. Expression of this gene may be changed in a variety of different cancers and in cardiovascular disease. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the Kruppel-like family of transcription factors. The zinc finger protein is a transcriptional activator, and functions as a tumor suppressor. Multiple transcript variants encoding different isoforms have been found for this gene, some of which are implicated in carcinogenesis.","This gene encodes a protein which contains multiple domains including a CXXC zinc finger, three PHD zinc fingers, two FY-rich domains, and a SET (suppressor of variegation, enhancer of zeste, and trithorax) domain. The SET domain is a conserved C-terminal domain that characterizes proteins of the MLL (mixed-lineage leukemia) family. This gene is ubiquitously expressed in adult tissues. It is also amplified in solid tumor cell lines, and may be involved in human cancer. Two alternatively spliced transcript variants encoding distinct isoforms have been reported for this gene, however, the full length nature of the shorter transcript is not known.","The protein encoded by this gene is a component of the multiprotein assembly that is required for creation of kinetochore-microtubule attachments and chromosome segregation. The encoded protein functions as a scaffold for proteins that influence the spindle assembly checkpoint during the eukaryotic cell cycle and it interacts with at least five different kinetochore proteins and two checkpoint kinases. In adults, this gene is predominantly expressed in normal testes, various cancer cell lines and primary tumors from other tissues and is ubiquitously expressed in fetal tissues. This gene was originally identified as a fusion partner with the mixed-lineage leukemia (MLL) gene in t(11;15)(q23;q14). Mutations in this gene cause autosomal recessive primary microcephaly-4 (MCPH4). Alternative splicing results in multiple transcript variants encoding different isoforms. Additional splice variants have been described but their biological validity has not been confirmed.","Enables microtubule plus-end binding activity and protein homodimerization activity. Involved in several processes, including cellular response to epidermal growth factor stimulus; mitotic sister chromatid segregation; and regulation of attachment of spindle microtubules to kinetochore. Located in several cellular components, including kinetochore; microtubule cytoskeleton; and ruffle. Implicated in actinic keratosis and skin squamous cell carcinoma. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a transcription factor belonging to a family of proteins that share homology with the high mobility group protein-1. The protein encoded by this gene can bind to a functionally important site in the T-cell receptor-alpha enhancer, thereby conferring maximal enhancer activity. This transcription factor is involved in the Wnt signaling pathway, and it may function in hair cell differentiation and follicle morphogenesis. Mutations in this gene have been found in somatic sebaceous tumors. This gene has also been linked to other cancers, including androgen-independent prostate cancer. Alternative splicing results in multiple transcript variants.","Predicted to act upstream of or within otolith morphogenesis. Located in extracellular space. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the low density lipoprotein (LDL) receptor family. These receptors play a wide variety of roles in normal cell function and development due to their interactions with multiple ligands. Disruption of this gene has been reported in several types of cancer.","MACC1 is a key regulator of the hepatocyte growth factor (HGF; MIM 142409)-HGF receptor (HGFR, or MET; MIM 164860) pathway, which is involved in cellular growth, epithelial-mesenchymal transition, angiogenesis, cell motility, invasiveness, and metastasis. Expression of MACC1 in colon cancer (MIM 114500) specimens is an independent prognostic indicator for metastasis formation and metastasis-free survival (Stein et al., 2009).","The protein encoded by this gene is a DNA-binding, leucine zipper-containing transcription factor that acts as a homodimer or as a heterodimer. Depending on the binding site and binding partner, the encoded protein can be a transcriptional activator or repressor. This protein plays a role in the regulation of several cellular processes, including embryonic lens fiber cell development, increased T-cell susceptibility to apoptosis, and chondrocyte terminal differentiation. Defects in this gene are a cause of juvenile-onset pulverulent cataract as well as congenital cerulean cataract 4 (CCA4). Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a caspase-like protease that plays a role in BCL10-induced activation of NF-kappaB. The protein is a component of the CARMA1-BCL10-MALT1 (CBM) signalosome that triggers NF-kappaB signaling and lymphoctye activation following antigen-receptor stimulation. Mutations in this gene result in immunodeficiency 12 (IMD12). This gene has been found to be recurrently rearranged in chromosomal translocations with other genes in mucosa-associated lymphoid tissue lymphomas, including a t(11;18)(q21;q21) translocation with the baculoviral IAP repeat-containing protein 3 (also known as apoptosis inhibitor 2) locus [BIRC3(API2)-MALT1], and a t(14;18)(q32;q21) translocation with the immunoglobulin heavy chain locus (IGH-MALT1). Alternatively spliced transcript variants have been described for this gene.","The protein encoded by this gene is a member of the Mastermind-like family of proteins. All family members are proline and glutamine-rich, and contain a conserved basic domain that binds the ankyrin repeat domain of the intracellular domain of the Notch receptors (ICN1-4) in their N-terminus, and a transcriptional activation domain in their C-terminus. This protein binds to an extended groove that is formed by the interaction of CBF1, Suppressor of Hairless, LAG-1 (CSL) with ICN, and positively regulates Notch signaling. High levels of expression of this gene have been observed in several B cell-derived lymphomas. Translocations resulting in fusion proteins with both CRTC1 and CRTC3 have been implicated in the development of mucoepidermoid carcinomas, while a translocation event with CXCR4 has been linked with chronic lymphocytic leukemia (CLL). Copy number variation in the polyglutamine tract has been observed.","This gene encodes a nuclear-localized E3 ubiquitin ligase. The encoded protein can promote tumor formation by targeting tumor suppressor proteins, such as p53, for proteasomal degradation. This gene is itself transcriptionally-regulated by p53. Overexpression or amplification of this locus is detected in a variety of different cancers. There is a pseudogene for this gene on chromosome 2. Alternative splicing results in a multitude of transcript variants, many of which may be expressed only in tumor cells.","Predicted to enable DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in cell fate specification and positive regulation of transcription by RNA polymerase II. Predicted to act upstream of or within cellular response to leukemia inhibitory factor. Part of MLL1 complex. [provided by Alliance of Genome Resources, Apr 2022]","Alkylating agents are potent carcinogens that can result in cell death, mutation and cancer. The protein encoded by this gene is a DNA repair protein that is involved in cellular defense against mutagenesis and toxicity from alkylating agents. The protein catalyzes transfer of methyl groups from O(6)-alkylguanine and other methylated moieties of the DNA to its own molecule, which repairs the toxic lesions. Methylation of the genes promoter has been associated with several cancer types, including colorectal cancer, lung cancer, lymphoma and glioblastoma.","The protein encoded by this gene can heterodimerize with mismatch repair endonuclease PMS2 to form MutL alpha, part of the DNA mismatch repair system. When MutL alpha is bound by MutS beta and some accessory proteins, the PMS2 subunit of MutL alpha introduces a single-strand break near DNA mismatches, providing an entry point for exonuclease degradation. The encoded protein is also involved in DNA damage signaling and can heterodimerize with DNA mismatch repair protein MLH3 to form MutL gamma, which is involved in meiosis. This gene was identified as a locus frequently mutated in hereditary nonpolyposis colon cancer (HNPCC).","In 1990 an oncogene, v-mpl, was identified from the murine myeloproliferative leukemia virus that was capable of immortalizing bone marrow hematopoietic cells from different lineages. In 1992 the human homologue, named, c-mpl, was cloned. Sequence data revealed that c-mpl encoded a protein that was homologous with members of the hematopoietic receptor superfamily. Presence of anti-sense oligodeoxynucleotides of c-mpl inhibited megakaryocyte colony formation. The ligand for c-mpl, thrombopoietin, was cloned in 1994. Thrombopoietin was shown to be the major regulator of megakaryocytopoiesis and platelet formation. The protein encoded by the c-mpl gene, CD110, is a 635 amino acid transmembrane domain, with two extracellular cytokine receptor domains and two intracellular cytokine receptor box motifs . TPO-R deficient mice were severely thrombocytopenic, emphasizing the important role of CD110 and thrombopoietin in megakaryocyte and platelet formation. Upon binding of thrombopoietin CD110 is dimerized and the JAK family of non-receptor tyrosine kinases, as well as the STAT family, the MAPK family, the adaptor protein Shc and the receptors themselves become tyrosine phosphorylated.","This locus is frequently mutated in hereditary nonpolyposis colon cancer (HNPCC). When cloned, it was discovered to be a human homolog of the E. coli mismatch repair gene mutS, consistent with the characteristic alterations in microsatellite sequences (RER+ phenotype) found in HNPCC. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the DNA mismatch repair MutS family. In E. coli, the MutS protein helps in the recognition of mismatched nucleotides prior to their repair. A highly conserved region of approximately 150 aa, called the Walker-A adenine nucleotide binding motif, exists in MutS homologs. The encoded protein heterodimerizes with MSH2 to form a mismatch recognition complex that functions as a bidirectional molecular switch that exchanges ADP and ATP as DNA mismatches are bound and dissociated. Mutations in this gene may be associated with hereditary nonpolyposis colon cancer, colorectal cancer, and endometrial cancer. Transcripts variants encoding different isoforms have been described.","This gene encodes an RNA-binding protein that is a member of the Musashi protein family. The encoded protein is transcriptional regulator that targets genes involved in development and cell cycle regulation. Mutations in this gene are associated with poor prognosis in certain types of cancers. This gene has also been shown to be rearranged in certain cancer cells.","This gene encodes a member of the mucin protein family. Mucins are high molecular weight glycoproteins produced by many epithelial tissues. The protein encoded by this gene is secreted and forms an insoluble mucous barrier that protects the gut lumen.","This gene encodes a protein containing a polynucleotide kinase domain (PNK) near the N-terminal region, and a Small MutS Related (Smr) domain near the C-terminal region. The encoded protein can bind to both B-cell leukemia/lymphoma 3 (BCL-3) and neural precursor cell expressed, developmentally downregulated 4, (Nedd4) proteins. This protein binds and hydrolyzes ATP, may function as a 5'-polynucleotide kinase, and has the capacity to be a ubiquitylation substrate. This protein may play a role in transcription-coupled DNA repair or genetic recombination. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a member of a large, diverse group of A-kinase anchor proteins that target the activity of protein kinase A to specific subcellular sites by binding to its type II regulatory subunits. Brain-specific expression and coat protein-like membrane recruitment of a highly similar protein in mouse suggest an involvement in neuronal post-Golgi membrane traffic. Mutations in this gene may be associated with a form of autism. This gene and its expression are frequently disrupted in patients with multiple myeloma. Alternative splicing results in multiple transcript variants encoding distinct isoforms. Additional transcript variants may exist, but their full-length nature has not been determined.[provided by RefSeq, Feb 2011].","The protein encoded by this gene functions as a transcriptional coactivator for nuclear hormone receptors, including steroid, thyroid, retinoid, and vitamin D receptors. The encoded protein acts as an intermediary factor for the ligand-dependent activity of these nuclear receptors, which regulate their target genes upon binding of cognate response elements. This gene has been found to be involved in translocations that result in fusions with other genes in various cancers, including the lysine acetyltransferase 6A (KAT6A) gene in acute myeloid leukemia, the ETS variant 6 (ETV6) gene in acute lymphoblastic leukemia, and the hes related family bHLH transcription factor with YRPW motif 1 (HEY1) gene in mesenchymal chondrosarcoma. Alternative splicing results in multiple transcript variants.","This gene encodes a nuclear receptor co-repressor that mediates transcriptional silencing of certain target genes. The encoded protein is a member of a family of thyroid hormone- and retinoic acid receptor-associated co-repressors. This protein acts as part of a multisubunit complex which includes histone deacetylases to modify chromatin structure that prevents basal transcriptional activity of target genes. Aberrant expression of this gene is associated with certain cancers. Alternate splicing results in multiple transcript variants encoding different isoforms.[provided by RefSeq, Apr 2011].","This gene is a member of the nuclear factor of activated T cells (NFAT) family. The product of this gene is a DNA-binding protein with a REL-homology region (RHR) and an NFAT-homology region (NHR). This protein is present in the cytosol and only translocates to the nucleus upon T cell receptor (TCR) stimulation, where it becomes a member of the nuclear factors of activated T cells transcription complex. This complex plays a central role in inducing gene transcription during the immune response. Alternate transcriptional splice variants encoding different isoforms have been characterized.","This gene encodes a subunit of the transcription factor complex nuclear factor-kappa-B (NFkB). The NFkB complex is expressed in numerous cell types and functions as a central activator of genes involved in inflammation and immune function. The protein encoded by this gene can function as both a transcriptional activator or repressor depending on its dimerization partner. The p100 full-length protein is co-translationally processed into a p52 active form. Chromosomal rearrangements and translocations of this locus have been observed in B cell lymphomas, some of which may result in the formation of fusion proteins. There is a pseudogene for this gene on chromosome 18. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene binds to components of NF-kappa-B, trapping the complex in the cytoplasm and preventing it from activating genes in the nucleus. Phosphorylation of the encoded protein targets it for destruction by the ubiquitin pathway, which activates NF-kappa-B by making it available to translocate to the nucleus.","This gene encodes the homolog of the Drosophila melanogaster Nipped-B gene product and fungal Scc2-type sister chromatid cohesion proteins. The Drosophila protein facilitates enhancer-promoter communication of remote enhancers and plays a role in developmental regulation. It is also homologous to a family of chromosomal adherins with broad roles in sister chromatid cohesion, chromosome condensation, and DNA repair. The human protein has a bipartite nuclear targeting sequence and a putative HEAT repeat. Condensins, cohesins and other complexes with chromosome-related functions also contain HEAT repeats. Mutations in this gene result in Cornelia de Lange syndrome, a disorder characterized by dysmorphic facial features, growth delay, limb reduction defects, and cognitive disability. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a protein initially identified as a thyroid-specific transcription factor. The encoded protein binds to the thyroglobulin promoter and regulates the expression of thyroid-specific genes but has also been shown to regulate the expression of genes involved in morphogenesis. Mutations and deletions in this gene are associated with benign hereditary chorea, choreoathetosis, congenital hypothyroidism, and neonatal respiratory distress, and may be associated with thyroid cancer. Multiple transcript variants encoding different isoforms have been found for this gene. This gene shares the symbol/alias 'TTF1' with another gene, transcription termination factor 1, which plays a role in ribosomal gene transcription.","The protein encoded by this gene is a membrane glycoprotein that mediates cell-cell signaling and plays a critical role in the growth and development of multiple organ systems. An extraordinary variety of different isoforms are produced from this gene through alternative promoter usage and splicing. These isoforms are expressed in a tissue-specific manner and differ significantly in their structure, and are classified as types I, II, III, IV, V and VI. Dysregulation of this gene has been linked to diseases such as cancer, schizophrenia, and bipolar disorder (BPD).","This gene encodes a protein containing a SET domain, 2 LXXLL motifs, 3 nuclear translocation signals (NLSs), 4 plant homeodomain (PHD) finger regions, and a proline-rich region. The encoded protein enhances androgen receptor (AR) transactivation, and this enhancement can be increased further in the presence of other androgen receptor associated coregulators. This protein may act as a nucleus-localized, basic transcriptional factor and also as a bifunctional transcriptional regulator. Mutations of this gene have been associated with Sotos syndrome and Weaver syndrome. One version of childhood acute myeloid leukemia is the result of a cryptic translocation with the breakpoints occurring within nuclear receptor-binding Su-var, enhancer of zeste, and trithorax domain protein 1 on chromosome 5 and nucleoporin, 98-kd on chromosome 11. Multiple transcript variants encoding distinct isoforms have been identified for this gene.","This gene is related to the Wolf-Hirschhorn syndrome candidate-1 gene and encodes a protein with PWWP (proline-tryptophan-tryptophan-proline) domains. This protein methylates histone H3 at lysine residues 4 and 27, which represses gene transcription. Two alternatively spliced variants have been described.","This gene encodes a member of the neurotrophic tyrosine kinase receptor (NTKR) family. This kinase is a membrane-bound receptor that, upon neurotrophin binding, phosphorylates itself and members of the MAPK pathway. The presence of this kinase leads to cell differentiation and may play a role in specifying sensory neuron subtypes. Mutations in this gene have been associated with congenital insensitivity to pain, anhidrosis, self-mutilating behavior, cognitive disability and cancer. Alternate transcriptional splice variants of this gene have been found, but only three have been characterized to date.","This gene encodes a member of the neurotrophic tyrosine receptor kinase (NTRK) family. This kinase is a membrane-bound receptor that, upon neurotrophin binding, phosphorylates itself and members of the MAPK pathway. Signalling through this kinase leads to cell differentiation and may play a role in the development of proprioceptive neurons that sense body position. Mutations in this gene have been associated with medulloblastomas, secretory breast carcinomas and other cancers. Several transcript variants encoding different isoforms have been found for this gene.","Nuclear pore complexes (NPCs) regulate the transport of macromolecules between the nucleus and cytoplasm, and are composed of many polypeptide subunits, many of which belong to the nucleoporin family. This gene belongs to the nucleoporin gene family and encodes a 186 kDa precursor protein that undergoes autoproteolytic cleavage to generate a 98 kDa nucleoporin and 96 kDa nucleoporin. The 98 kDa nucleoporin contains a Gly-Leu-Phe-Gly (GLGF) repeat domain and participates in many cellular processes, including nuclear import, nuclear export, mitotic progression, and regulation of gene expression. The 96 kDa nucleoporin is a scaffold component of the NPC. Proteolytic cleavage is important for targeting of the proteins to the NPC. Translocations between this gene and many other partner genes have been observed in different leukemias. Rearrangements typically result in chimeras with the N-terminal GLGF domain of this gene to the C-terminus of the partner gene. Alternative splicing results in multiple transcript variants encoding different isoforms, at least two of which are proteolytically processed. Some variants lack the region that encodes the 96 kDa nucleoporin.","Predicted to be located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene belongs to the family of G-protein coupled receptors, that are preferentially activated by adenosine and uridine nucleotides. This gene is moderately expressed in undifferentiated HL60 cells, and is located on both chromosomes X and Y.","This gene encodes a poly(A) binding protein. The protein shuttles between the nucleus and cytoplasm and binds to the 3' poly(A) tail of eukaryotic messenger RNAs via RNA-recognition motifs. The binding of this protein to poly(A) promotes ribosome recruitment and translation initiation; it is also required for poly(A) shortening which is the first step in mRNA decay. The gene is part of a small gene family including three protein-coding genes and several pseudogenes.[provided by RefSeq, Aug 2010].","This gene encodes a protein that may function in tumor suppression. This protein binds to and colocalizes with the breast cancer 2 early onset protein (BRCA2) in nuclear foci and likely permits the stable intranuclear localization and accumulation of BRCA2.","Involved in negative regulation of activated T cell proliferation; negative regulation of interferon-gamma production; and negative regulation of interleukin-10 production. Predicted to be located in plasma membrane. Predicted to be active in external side of plasma membrane. Biomarker of pulmonary tuberculosis. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a cell surface tyrosine kinase receptor for members of the platelet-derived growth factor family. These growth factors are mitogens for cells of mesenchymal origin. The identity of the growth factor bound to a receptor monomer determines whether the functional receptor is a homodimer (PDGFB or PDGFD) or a heterodimer (PDGFA and PDGFB). This gene is essential for normal development of the cardiovascular system and aids in rearrangement of the actin cytoskeleton. This gene is flanked on chromosome 5 by the genes for granulocyte-macrophage colony-stimulating factor and macrophage-colony stimulating factor receptor; all three genes may be implicated in the 5-q syndrome. A translocation between chromosomes 5 and 12, that fuses this gene to that of the ETV6 gene, results in chronic myeloproliferative disorder with eosinophilia.","Phosphatidylinositol 3-kinase (PI3K) is a lipid kinase that phosphorylates phosphatidylinositol and similar compounds, creating second messengers important in growth signaling pathways. PI3K functions as a heterodimer of a regulatory and a catalytic subunit. The protein encoded by this gene is a regulatory component of PI3K. Three transcript variants, one protein coding and the other two non-protein coding, have been found for this gene.","Pleomorphic adenoma gene 1 encodes a zinc finger protein with 2 putative nuclear localization signals. PLAG1, which is developmentally regulated, has been shown to be consistently rearranged in pleomorphic adenomas of the salivary glands. PLAG1 is activated by the reciprocal chromosomal translocations involving 8q12 in a subset of salivary gland pleomorphic adenomas. Three transcript variants encoding two different isoforms have been found for this gene.","The protein encoded by this gene is a member of the tripartite motif (TRIM) family. The TRIM motif includes three zinc-binding domains, a RING, a B-box type 1 and a B-box type 2, and a coiled-coil region. This phosphoprotein localizes to nuclear bodies where it functions as a transcription factor and tumor suppressor. Its expression is cell-cycle related and it regulates the p53 response to oncogenic signals. The gene is often involved in the translocation with the retinoic acid receptor alpha gene associated with acute promyelocytic leukemia (APL). Extensive alternative splicing of this gene results in several variations of the protein's central and C-terminal regions; all variants encode the same N-terminus. Alternatively spliced transcript variants encoding different isoforms have been identified.","This gene encodes a protein belonging to the DNA mismatch repair mutL/hexB family. This protein is thought to be involved in the repair of DNA mismatches, and it can form heterodimers with MLH1, a known DNA mismatch repair protein. Mutations in this gene cause hereditary nonpolyposis colorectal cancer type 3 (HNPCC3) either alone or in combination with mutations in other genes involved in the HNPCC phenotype, which is also known as Lynch syndrome.","This gene encodes the 125-kDa catalytic subunit of DNA polymerase delta. DNA polymerase delta possesses both polymerase and 3' to 5' exonuclease activity and plays a critical role in DNA replication and repair. Alternatively spliced transcript variants have been observed for this gene, and a pseudogene of this gene is located on the long arm of chromosome 6.","This gene is a member of the telombin family and encodes a nuclear protein involved in telomere maintenance. Specifically, this protein functions as a member of a multi-protein complex that binds to the TTAGGG repeats of telomeres, regulating telomere length and protecting chromosome ends from illegitimate recombination, catastrophic chromosome instability, and abnormal chromosome segregation. Increased transcriptional expression of this gene is associated with stomach carcinogenesis and its progression. Alternatively spliced transcript variants have been described.","This gene encodes a protein that acts as a repressor of beta-interferon gene expression. The protein binds specifically to the PRDI (positive regulatory domain I element) of the beta-IFN gene promoter. Transcription of this gene increases upon virus induction. Two alternatively spliced transcript variants that encode different isoforms have been reported.","The reciprocal translocation t(1;3)(p36;q21) occurs in a subset of myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). This gene is located near the 1p36.3 breakpoint and has been shown to be specifically expressed in the t(1:3)(p36,q21)-positive MDS/AML. The protein encoded by this gene is a zinc finger transcription factor and contains an N-terminal PR domain. The translocation results in the overexpression of a truncated version of this protein that lacks the PR domain, which may play an important role in the pathogenesis of MDS and AML. Alternatively spliced transcript variants encoding distinct isoforms have been reported.","This tumor suppressor gene is a member of a nuclear histone/protein methyltransferase superfamily. It encodes a zinc finger protein that can bind to retinoblastoma protein, estrogen receptor, and the TPA-responsive element (MTE) of the heme-oxygenase-1 gene. Although the functions of this protein have not been fully characterized, it may (1) play a role in transcriptional regulation during neuronal differentiation and pathogenesis of retinoblastoma, (2) act as a transcriptional activator of the heme-oxygenase-1 gene, and (3) be a specific effector of estrogen action. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene belongs to the phosphatidylinositol 3,4,5-trisphosphate (PIP3)-dependent Rac exchanger (PREX) family, which are Dbl-type guanine-nucleotide exchange factors for Rac family small G proteins. Structural domains of this protein include the catalytic diffuse B-cell lymphoma homology and pleckstrin homology (DHPH) domain, two disheveled, EGL-10, and pleckstrin homology (DEP) domains, two PDZ domains, and a C-terminal inositol polyphosphate-4 phosphatase (IP4P) domain that is found in one of the isoforms. This protein facilitates the exchange of GDP for GTP on Rac1, allowing the GTP-bound Rac1 to activate downstream effectors. Studies also show that the pleckstrin homology domain of this protein interacts with the phosphatase and tensin homolog (PTEN) gene product to inhibit PTEN phosphatase activity, thus activating the phosphoinositide-3 kinase (PI3K) signaling pathway. Conversely, the PTEN gene product has also been shown to inhibit the GEF activity of this protein. This gene plays a role in insulin-signaling pathways, and either mutations or overexpression of this gene have been observed in some cancers.","Protein kinase C (PKC) is a family of serine- and threonine-specific protein kinases that can be activated by calcium and second messenger diacylglycerol. PKC family members phosphorylate a wide variety of protein targets and are known to be involved in diverse cellular signaling pathways. PKC family members also serve as major receptors for phorbol esters, a class of tumor promoters. Each member of the PKC family has a specific expression profile and is believed to play a distinct role in cells. The protein encoded by this gene is one of the PKC family members. This protein kinase has been reported to be involved in many different cellular functions, such as B cell activation, apoptosis induction, endothelial cell proliferation, and intestinal sugar absorption. Studies in mice also suggest that this kinase may also regulate neuronal functions and correlate fear-induced conflict behavior after stress. Alternatively spliced transcript variants encoding distinct isoforms have been reported.","The protein encoded by this gene is a cytoplasmic nonreceptor protein kinase which may function as an intracellular signal transducer in epithelial tissues. Overexpression of this gene in mammary epithelial cells leads to sensitization of the cells to epidermal growth factor and results in a partially transformed phenotype. Expression of this gene has been detected at low levels in some breast tumors but not in normal breast tissue. The encoded protein has been shown to undergo autophosphorylation. Alternative splicing results in multiple transcript variants.","Enables DNA-binding transcription factor binding activity. Involved in negative regulation of apoptotic process. Located in cytosol and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP is a large intracellular protein. It has a catalytic PTP domain at its C-terminus and two major structural domains: a region with five PDZ domains and a FERM domain that binds to plasma membrane and cytoskeletal elements. This PTP was found to interact with, and dephosphorylate, Fas receptor and IkappaBalpha through the PDZ domains. This suggests it has a role in Fas mediated programmed cell death. This PTP was also shown to interact with GTPase-activating protein, and thus may function as a regulator of Rho signaling pathways. Four alternatively spliced transcript variants, which encode distinct proteins, have been reported.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. N-terminal part of this PTP contains two tandem Src homolog (SH2) domains, which act as protein phospho-tyrosine binding domains, and mediate the interaction of this PTP with its substrates. This PTP is expressed primarily in hematopoietic cells, and functions as an important regulator of multiple signaling pathways in hematopoietic cells. This PTP has been shown to interact with, and dephosphorylate a wide spectrum of phospho-proteins involved in hematopoietic cell signaling. Multiple alternatively spliced variants of this gene, which encode distinct isoforms, have been reported.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP contains an extracellular domain, a single transmembrane segment and one intracytoplasmic catalytic domain, thus belongs to receptor type PTP. The extracellular region of this PTP is composed of multiple fibronectin type_III repeats, which was shown to interact with neuronal receptor and cell adhesion molecules, such as contactin and tenascin C. This protein was also found to interact with sodium channels, and thus may regulate sodium channels by altering tyrosine phosphorylation status. The functions of the interaction partners of this protein implicate the roles of this PTP in cell adhesion, neurite growth, and neuronal differentiation. Alternate transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP possesses an extracellular region, a single transmembrane region, and two tandem catalytic domains, and thus represents a receptor-type PTP. The extracellular region contains a meprin-A5 antigen-PTP mu (MAM) domain, an Ig-like domain and four fibronectin type III-like repeats. This PTP was shown to mediate homophilic intercellular interaction, possibly through the interaction with beta- and gamma-catenin at adherens junctions. Expression of this gene was found to be stimulated by TGF-beta 1, which may be important for the inhibition of keratinocyte proliferation.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP possesses an extracellular region, a single transmembrane region, and two tandem intracellular catalytic domains, and thus represents a receptor-type PTP. The extracellular region contains a meprin-A5 antigen-PTP (MAM) domain, Ig-like and fibronectin type III-like repeats. The protein domain structure and the expression pattern of the mouse counterpart of this PTP suggest its roles in both signal transduction and cellular adhesion in the central nervous system. Two alternatively spliced transcript variants of this gene, which encode distinct proteins, have been reported.","The protein encoded by this gene is an RNA-binding protein that regulates pre-mRNA splicing, export of mRNAs from the nucleus, protein translation, and mRNA stability. The encoded protein is involved in myelinization and oligodendrocyte differentiation and may play a role in schizophrenia. Multiple transcript variants encoding different isoforms have been found for this gene.","RAN is a small GTP-binding protein of the RAS superfamily that is associated with the nuclear membrane and is thought to control a variety of cellular functions through its interactions with other proteins. This gene encodes a very large RAN-binding protein that immunolocalizes to the nuclear pore complex. The protein is a giant scaffold and mosaic cyclophilin-related nucleoporin implicated in the Ran-GTPase cycle. The encoded protein directly interacts with the E2 enzyme UBC9 and strongly enhances SUMO1 transfer from UBC9 to the SUMO1 target SP100. These findings place sumoylation at the cytoplasmic filaments of the nuclear pore complex and suggest that, for some substrates, modification and nuclear import are linked events. This gene is partially duplicated in a gene cluster that lies in a hot spot for recombination on chromosome 2q.","The smg GDP dissociation stimulator (smgGDS) protein is a stimulatory GDP/GTP exchange protein with GTPase activity (Riess et al., 1993).","This gene represents a nuclear retinoic acid receptor. The encoded protein, retinoic acid receptor alpha, regulates transcription in a ligand-dependent manner. This gene has been implicated in regulation of development, differentiation, apoptosis, granulopoeisis, and transcription of clock genes. Translocations between this locus and several other loci have been associated with acute promyelocytic leukemia. Alternatively spliced transcript variants have been found for this locus.[provided by RefSeq, Sep 2010].","The protein encoded by this gene is located in the cytoplasm and is part of the GAP1 family of GTPase-activating proteins. The gene product stimulates the GTPase activity of normal RAS p21 but not its oncogenic counterpart. Acting as a suppressor of RAS function, the protein enhances the weak intrinsic GTPase activity of RAS proteins resulting in the inactive GDP-bound form of RAS, thereby allowing control of cellular proliferation and differentiation. Mutations leading to changes in the binding sites of either protein are associated with basal cell carcinomas. Mutations also have been associated with hereditary capillary malformations (CM) with or without arteriovenous malformations (AVM) and Parkes Weber syndrome. Alternative splicing results in two isoforms where the shorter isoform, lacking the N-terminal hydrophobic region but retaining the same activity, appears to be abundantly expressed in placental but not adult tissues.","The protein encoded by this gene is a DNA helicase that belongs to the RecQ helicase family. DNA helicases unwind double-stranded DNA into single-stranded DNAs and may modulate chromosome segregation. This gene is predominantly expressed in thymus and testis. Mutations in this gene are associated with Rothmund-Thomson, RAPADILINO and Baller-Gerold syndromes.","This gene is located in a cluster of Ran-binding protein related genes on chromosome 2 which arose through duplication in primates. The encoded protein contains an N-terminal TPR (tetratricopeptide repeat) domain, two Ran-binding domains, and a C-terminal GRIP domain (golgin-97, RanBP2alpha, Imh1p and p230/golgin-245) domain.","Enables G-protein beta-subunit binding activity and GTPase activator activity. Involved in positive regulation of GTPase activity. Located in cytosol. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of a subfamily of Ras-related GTPases. The encoded protein is involved in regulating p38 MAPK-dependent signaling cascades related to cellular stress. This protein also cooperates with nerve growth factor to promote neuronal development and regeneration. Alternate splicing results in multiple transcript variants.","The protein encoded by this gene belongs to the ROBO family, part of the immunoglobulin superfamily of proteins that are highly conserved from fly to human. The encoded protein is a transmembrane receptor for the slit homolog 2 protein and functions in axon guidance and cell migration. Mutations in this gene are associated with vesicoureteral reflux, characterized by the backward flow of urine from the bladder into the ureters or the kidney. Alternative splicing results in multiple transcript variants.","This proto-oncogene, highly-expressed in a variety of tumor cell lines, belongs to the sevenless subfamily of tyrosine kinase insulin receptor genes. The protein encoded by this gene is a type I integral membrane protein with tyrosine kinase activity. The protein may function as a growth or differentiation factor receptor.","This gene encodes a ribosomal protein that is a component of the 60S ribosome subunit. The related protein in chicken can bind to c-Jun and can repress c-Jun-mediated transcriptional activation. Some studies have detected an association between variation in this gene and autism spectrum disorders, though others do not detect this relationship. There are multiple pseudogenes of this gene dispersed throughout the genome. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the RSK (ribosomal S6 kinase) family of serine/threonine kinases. This kinase contains 2 non-identical kinase catalytic domains and phosphorylates various substrates, including members of the mitogen-activated kinase (MAPK) signalling pathway. The activity of this protein has been implicated in controlling cell growth and differentiation. Mutations in this gene have been associated with Coffin-Lowry syndrome (CLS).","This gene encodes a member of the R-Ras subfamily of Ras-like small GTPases. The encoded protein associates with the plasma membrane and may function as a signal transducer. This protein may play an important role in activating signal transduction pathways that control cell proliferation. Mutations in this gene are associated with the growth of certain tumors. Pseudogenes of this gene are found on chromosomes 1 and 2. Alternate splicing results in multiple transcript variants.","This gene encodes a member of the R-spondin family of proteins. These proteins are secreted ligands of leucine-rich repeat containing G protein-coupled receptors that enhance Wnt signaling through the inhibition of ubiquitin E3 ligases. A chromosomal translocation including this locus that results in the formation of a gene fusion has been identified in multiple human cancers. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the myeloid translocation gene family which interact with DNA-bound transcription factors and recruit a range of corepressors to facilitate transcriptional repression. The t(8;21)(q22;q22) translocation is one of the most frequent karyotypic abnormalities in acute myeloid leukemia. The translocation produces a chimeric gene made up of the 5'-region of the runt-related transcription factor 1 gene fused to the 3'-region of this gene. The chimeric protein is thought to associate with the nuclear corepressor/histone deacetylase complex to block hematopoietic differentiation. Alternative splicing results in multiple transcript variants.","Retinoid X receptors (RXRs) and retinoic acid receptors (RARs) are nuclear receptors that mediate the biological effects of retinoids by their involvement in retinoic acid-mediated gene activation. These receptors function as transcription factors by binding as homodimers or heterodimers to specific sequences in the promoters of target genes. The protein encoded by this gene is a member of the steroid and thyroid hormone receptor superfamily of transcriptional regulators. Alternative splicing of this gene results in multiple transcript variants.","This gene encodes a zinc finger transcription factor thought to play a role in the development of abducens motor neurons. Defects in this gene are a cause of Duane-radial ray syndrome (DRRS). Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a major catalytic subunit of succinate-ubiquinone oxidoreductase, a complex of the mitochondrial respiratory chain. The complex is composed of four nuclear-encoded subunits and is localized in the mitochondrial inner membrane. Mutations in this gene have been associated with a form of mitochondrial respiratory chain deficiency known as Leigh Syndrome. A pseudogene has been identified on chromosome 3q29. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","SET1B is a component of a histone methyltransferase complex that produces trimethylated histone H3 at Lys4 (Lee et al., 2007).","This gene encodes a histone methyltransferase which regulates histone methylation, gene silencing, and transcriptional repression. This gene has been identified as a target for treatment in Huntington Disease, given that gene silencing and transcription dysfunction likely play a role in the disease pathogenesis. Alternatively spliced transcript variants of this gene have been described.[provided by RefSeq, Jun 2011].","This gene encodes a serine/threonine protein kinase that plays an important role in cellular stress response. This kinase activates certain potassium, sodium, and chloride channels, suggesting an involvement in the regulation of processes such as cell survival, neuronal excitability, and renal sodium excretion. High levels of expression of this gene may contribute to conditions such as hypertension and diabetic nephropathy. Several alternatively spliced transcript variants encoding different isoforms have been noted for this gene.","This gene encodes a member of the SH2B adaptor family of proteins, which are involved in a range of signaling activities by growth factor and cytokine receptors. The encoded protein is a key negative regulator of cytokine signaling and plays a critical role in hematopoiesis. Mutations in this gene have been associated with susceptibility to celiac disease type 13 and susceptibility to insulin-dependent diabetes mellitus. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a transcriptional regulatory protein. It contains paired amphipathic helix (PAH) domains, which are important for protein-protein interactions and may mediate repression by the Mad-Max complex.","The protein encoded by this gene is a member of the signal-regulatory-protein (SIRP) family, and also belongs to the immunoglobulin superfamily. SIRP family members are receptor-type transmembrane glycoproteins known to be involved in the negative regulation of receptor tyrosine kinase-coupled signaling processes. This protein can be phosphorylated by tyrosine kinases. The phospho-tyrosine residues of this PTP have been shown to recruit SH2 domain containing tyrosine phosphatases (PTP), and serve as substrates of PTPs. This protein was found to participate in signal transduction mediated by various growth factor receptors. CD47 has been demonstrated to be a ligand for this receptor protein. This gene and its product share very high similarity with several other members of the SIRP family. These related genes are located in close proximity to each other on chromosome 20p13. Multiple alternatively spliced transcript variants have been determined for this gene.","The protein encoded by this gene is a homeobox protein that is similar to the Drosophila 'sine oculis' gene product. This gene is found in a cluster of related genes on chromosome 14 and is thought to be involved in limb development. Defects in this gene are a cause of autosomal dominant deafness type 23 (DFNA23) and branchiootic syndrome type 3 (BOS3).","The protein encoded by this gene is a pH-sensitive sodium-dependent phosphate transporter. Phosphate uptake is increased at lower pH. Defects in this gene are a cause of pulmonary alveolar microlithiasis. Three transcript variants encoding two different isoforms have been found for this gene.","The SMAD family of proteins are a group of intracellular signal transducer proteins similar to the gene products of the Drosophila gene 'mothers against decapentaplegic' (Mad) and the C. elegans gene Sma. The SMAD3 protein functions in the transforming growth factor-beta signaling pathway, and transmits signals from the cell surface to the nucleus, regulating gene activity and cell proliferation. This protein forms a complex with other SMAD proteins and binds DNA, functioning both as a transcription factor and tumor suppressor. Mutations in this gene are associated with aneurysms-osteoarthritis syndrome and Loeys-Dietz Syndrome 3.","The protein encoded by this gene is a member of the SWI/SNF family of proteins, whose members display helicase and ATPase activities and which are thought to regulate transcription of certain genes by altering the chromatin structure around those genes. The encoded protein is part of the large ATP-dependent chromatin remodeling complex SNF/SWI and has sequence similarity to the yeast Swp73 protein. Two transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a G protein-coupled receptor that interacts with the patched protein, a receptor for hedgehog proteins. The encoded protein tranduces signals to other proteins after activation by a hedgehog protein/patched protein complex.","This gene encodes a transcriptional co-activator that interacts with the acidic domain of Epstein-Barr virus nuclear antigen 2 (EBNA 2), a transcriptional activator that is required for B-lymphocyte transformation. Other transcription factors that interact with this protein are signal transducers and activators of transcription, STATs. This protein is also thought to be essential for normal cell growth. A similar protein in mammals and other organisms is a component of the RNA-induced silencing complex (RISC).","This gene encodes a member of the STAT-induced STAT inhibitor (SSI), also known as suppressor of cytokine signaling (SOCS), family. SSI family members are cytokine-inducible negative regulators of cytokine signaling. The expression of this gene can be induced by a subset of cytokines, including IL2, IL3 erythropoietin (EPO), CSF2/GM-CSF, and interferon (IFN)-gamma. The protein encoded by this gene functions downstream of cytokine receptors, and takes part in a negative feedback loop to attenuate cytokine signaling. Knockout studies in mice suggested the role of this gene as a modulator of IFN-gamma action, which is required for normal postnatal growth and survival.","This gene encodes a protein that is a guanine nucleotide exchange factor for RAS proteins, membrane proteins that bind guanine nucleotides and participate in signal transduction pathways. GTP binding activates and GTP hydrolysis inactivates RAS proteins. The product of this gene may regulate RAS proteins by facilitating the exchange of GTP for GDP. Mutations in this gene are associated with gingival fibromatosis 1 and Noonan syndrome type 4.","This gene encodes a member of the SOX (SRY-related HMG-box) family of transcription factors involved in the regulation of embryonic development and in the determination of the cell fate. The encoded protein may act as a transcriptional regulator after forming a protein complex with other proteins.","SRY-related HMG-box (SOX) genes encode a family of DNA-binding proteins containing a 79-amino acid HMG (high mobility group) domain that shares at least 50% sequence identity with the DNA-binding HMG box of the SRY protein (MIM 480000). SOX proteins are divided into 6 subgroups based on sequence similarity within and outside of the HMG domain. For additional background information on SOX genes, see SOX1 (MIM 602148).","This gene encodes a hormone inducible transcriptional repressor. Repression of transcription by this gene product can occur through interactions with other repressors, by the recruitment of proteins involved in histone deacetylation, or through sequestration of transcriptional activators. The product of this gene contains a carboxy-terminal domain that permits binding to other corepressor proteins. This domain also permits interaction with members of the NuRD complex, a nucleosome remodeling protein complex that contains deacetylase activity. In addition, this repressor contains several RNA recognition motifs that confer binding to a steroid receptor RNA coactivator; this binding can modulate the activity of both liganded and nonliganded steroid receptors.","The protein encoded by this gene is a member of the STAT protein family. In response to cytokines and growth factors, STAT family members are phosphorylated by the receptor associated kinases, and then form homo- or heterodimers that translocate to the cell nucleus where they act as transcription activators. This protein is activated through phosphorylation in response to various cytokines and growth factors including IFNs, EGF, IL5, IL6, HGF, LIF and BMP2. This protein mediates the expression of a variety of genes in response to cell stimuli, and thus plays a key role in many cellular processes such as cell growth and apoptosis. The small GTPase Rac1 has been shown to bind and regulate the activity of this protein. PIAS3 protein is a specific inhibitor of this protein. This gene also plays a role in regulating host response to viral and bacterial infections. Mutations in this gene are associated with infantile-onset multisystem autoimmune disease and hyper-immunoglobulin E syndrome.","The protein encoded by this gene is a member of the STAT family of transcription factors. In response to cytokines and growth factors, STAT family members are phosphorylated by the receptor associated kinases, and then form homo- or heterodimers that translocate to the cell nucleus where they act as transcription activators. This protein mediates the signal transduction triggered by various cell ligands, such as IL2, IL4, CSF1, and different growth hormones. It has been shown to be involved in diverse biological processes, such as TCR signaling, apoptosis, adult mammary gland development, and sexual dimorphism of liver gene expression. This gene was found to fuse to retinoic acid receptor-alpha (RARA) gene in a small subset of acute promyelocytic leukemias (APLL). The dysregulation of the signaling pathways mediated by this protein may be the cause of the APLL.","The protein encoded by this gene is a member of the STAT family of transcription factors. In response to cytokines and growth factors, STAT family members are phosphorylated by the receptor associated kinases, and then form homo- or heterodimers that translocate to the cell nucleus where they act as transcription activators. This protein plays a central role in exerting IL4 mediated biological responses. It is found to induce the expression of BCL2L1/BCL-X(L), which is responsible for the anti-apoptotic activity of IL4. Knockout studies in mice suggested the roles of this gene in differentiation of T helper 2 (Th2) cells, expression of cell surface markers, and class switch of immunoglobulins. Alternative splicing results in multiple transcript variants.[provided by RefSeq, May 2010].","The Hedgehog signaling pathway plays an important role in early human development. The pathway is a signaling cascade that plays a role in pattern formation and cellular proliferation during development. This gene encodes a negative regulator of the hedgehog signaling pathway. Defects in this gene are a cause of medulloblastoma. Alternative splicing results in multiple transcript variants.[provided by RefSeq, May 2010].","This zinc finger gene has been identified at the breakpoints of a recurrent chromosomal translocation reported in endometrial stromal sarcoma. Recombination of these breakpoints results in the fusion of this gene and JAZF1. The protein encoded by this gene contains a zinc finger domain in the C terminus of the coding region.","This gene encodes a member of the TET family of RNA-binding proteins. The encoded protein plays a role in RNA polymerase II gene transcription as a component of a distinct subset of multi-subunit transcription initiation factor TFIID complexes. Translocations involving this gene play a role in acute leukemia and extraskeletal myxoid chondrosarcoma, and mutations in this gene may play a role in amyotrophic lateral sclerosis. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","Enables several functions, including DNA-binding transcription factor activity; E-box binding activity; and histone deacetylase binding activity. Involved in several processes, including myeloid cell differentiation; positive regulation of cellular component organization; and positive regulation of erythrocyte differentiation. Located in chromatin and nucleoplasm. Part of transcription regulator complex. Implicated in acute lymphoblastic leukemia. [provided by Alliance of Genome Resources, Apr 2022]","This intronless gene encodes a helix-loop-helix protein. Translocations between this gene on chromosome 9 and the T-cell receptor beta-chain locus on chromosome 7 have been associated with activation of the T-cell acute lymphocytic leukemia 2 gene and T-cell acute lymphoblastic leukemia.","The protein encoded by this gene is a member of the basic helix-loop-helix (bHLH) E-protein family that recognizes the consensus binding site (E-box) CANNTG. This encoded protein is expressed in many tissues, among them skeletal muscle, thymus, B- and T-cells, and may participate in regulating lineage-specific gene expression through the formation of heterodimers with other bHLH E-proteins. Several alternatively spliced transcript variants of this gene have been described, but the full-length nature of some of these variants has not been determined.","Overexpression of the TCL1 gene in humans has been implicated in the development of mature T cell leukemia, in which chromosomal rearrangements bring the TCL1 gene in close proximity to the T-cell antigen receptor (TCR)-alpha (MIM 186880) or TCR-beta (MIM 186930) regulatory elements (summarized by Virgilio et al., 1998). In normal T cells TCL1 is expressed in CD4-/CD8- cells, but not in cells at later stages of differentiation. TCL1 functions as a coactivator of the cell survival kinase AKT (MIM 164730) (Laine et al., 2000 [PubMed 10983986]).","The protein encoded by this gene belongs to the Tec family of non-receptor protein-tyrosine kinases containing a pleckstrin homology domain. Tec family kinases are involved in the intracellular signaling mechanisms of cytokine receptors, lymphocyte surface antigens, heterotrimeric G-protein coupled receptors, and integrin molecules. They are also key players in the regulation of the immune functions. Tec kinase is an integral component of T cell signaling and has a distinct role in T cell activation. This gene may be associated with myelodysplastic syndrome.","Enables RNA adenylyltransferase activity. Involved in mRNA stabilization. Located in cytoplasm and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","Telomerase is a ribonucleoprotein polymerase that maintains telomere ends by addition of the telomere repeat TTAGGG. The enzyme consists of a protein component with reverse transcriptase activity, encoded by this gene, and an RNA component which serves as a template for the telomere repeat. Telomerase expression plays a role in cellular senescence, as it is normally repressed in postnatal somatic cells resulting in progressive shortening of telomeres. Deregulation of telomerase expression in somatic cells may be involved in oncogenesis. Studies in mouse suggest that telomerase also participates in chromosomal repair, since de novo synthesis of telomere repeats may occur at double-stranded breaks. Alternatively spliced variants encoding different isoforms of telomerase reverse transcriptase have been identified; the full-length sequence of some variants has not been determined. Alternative splicing at this locus is thought to be one mechanism of regulation of telomerase activity.","DNA methylation is an epigenetic mechanism that is important for controlling gene expression. The protein encoded by this gene is a demethylase that belongs to the TET (ten-eleven translocation) family. Members of the TET protein family play a role in the DNA methylation process and gene activation.","The protein encoded by this gene is a member of the three-amino acid loop extension (TALE) superclass of atypical homeodomains. TALE homeobox proteins are highly conserved transcription regulators. This particular homeodomain binds to a previously characterized retinoid X receptor responsive element from the cellular retinol-binding protein II promoter. In addition to its role in inhibiting 9-cis-retinoic acid-dependent RXR alpha transcription activation of the retinoic acid responsive element, the protein is an active transcriptional co-repressor of SMAD2 and may participate in the transmission of nuclear signals during development and in the adult. Mutations in this gene are associated with holoprosencephaly type 4, which is a structural anomaly of the brain. Alternative splicing has been observed at this locus and multiple splice variants encoding distinct isoforms are described.","Enables phosphoprotein binding activity; thyroid hormone receptor binding activity; and transcription coactivator activity. Involved in nuclear-transcribed mRNA catabolic process; positive regulation of circadian rhythm; and regulation of RNA metabolic process. Acts upstream of or within positive regulation of transcription by RNA polymerase II. Located in nuclear speck. Part of mediator complex. Colocalizes with exon-exon junction complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes an actin sequestering protein which plays a role in regulation of actin polymerization. The protein is also involved in cell proliferation, migration, and differentiation. This gene escapes X inactivation and has a homolog on chromosome Y.","This gene encodes an extracellular matrix protein with a spatially and temporally restricted tissue distribution. This protein is homohexameric with disulfide-linked subunits, and contains multiple EGF-like and fibronectin type-III domains. It is implicated in guidance of migrating neurons as well as axons during development, synaptic plasticity, and neuronal regeneration.","This gene encodes a member of the TNF (tumor necrosis factor) receptor superfamily. The encoded protein functions in signal transduction pathways that activate inflammatory and inhibitory T-cell immune response. It binds herpes simplex virus (HSV) viral envelope glycoprotein D (gD), mediating its entry into cells. Alternative splicing results in multiple transcript variants.","This gene encodes a member of the p53 family of transcription factors. The functional domains of p53 family proteins include an N-terminal transactivation domain, a central DNA-binding domain and an oligomerization domain. Alternative splicing of this gene and the use of alternative promoters results in multiple transcript variants encoding different isoforms that vary in their functional properties. These isoforms function during skin development and maintenance, adult stem/progenitor cell regulation, heart development and premature aging. Some isoforms have been found to protect the germline by eliminating oocytes or testicular germ cells that have suffered DNA damage. Mutations in this gene are associated with ectodermal dysplasia, and cleft lip/palate syndrome 3 (EEC3); split-hand/foot malformation 4 (SHFM4); ankyloblepharon-ectodermal defects-cleft lip/palate; ADULT syndrome (acro-dermato-ungual-lacrimal-tooth); limb-mammary syndrome; Rap-Hodgkin syndrome (RHS); and orofacial cleft 8.","The protein encoded by this gene is a member of the TNF receptor associated factor (TRAF) protein family. TRAF proteins associate with, and mediate the signal transduction from, members of the TNF receptor (TNFR) superfamily. This protein participates in the signal transduction of CD40, a TNFR family member important for the activation of the immune response. This protein is found to be a critical component of the lymphotoxin-beta receptor (LTbetaR) signaling complex, which induces NF-kappaB activation and cell death initiated by LTbeta ligation. Epstein-Barr virus encoded latent infection membrane protein-1 (LMP1) can interact with this and several other members of the TRAF family, which may be essential for the oncogenic effects of LMP1. The protein also plays a role in the regulation of antiviral response. Mutations in this are associated with Encephalopathy, acute, infection-induced, herpes-specific 5.","Tumor necrosis factor (TNF; see MIM 191160) receptor-associated factors, such as TRAF7, are signal transducers for members of the TNF receptor superfamily (see MIM 191190). TRAFs are composed of an N-terminal cysteine/histidine-rich region containing zinc RING and/or zinc finger motifs; a coiled-coil (leucine zipper) motif; and a homologous region that defines the TRAF family, the TRAF domain, which is involved in self-association and receptor binding.","The protein encoded by this gene mediates transcriptional control by interaction with the activation function 2 (AF2) region of several nuclear receptors, including the estrogen, retinoic acid, and vitamin D3 receptors. The protein localizes to nuclear bodies and is thought to associate with chromatin and heterochromatin-associated factors. The protein is a member of the tripartite motif (TRIM) family. The TRIM motif includes three zinc-binding domains - a RING, a B-box type 1 and a B-box type 2 - and a coiled-coil region. Two alternatively spliced transcript variants encoding different isoforms have been described for this gene.","The protein encoded by this gene is thought to be a transcriptional corepressor. However, molecules that interact with this protein have not yet been identified. The protein is a member of the tripartite motif family. This motif includes three zinc-binding domains, a RING, a B-box type 1 and a B-box type 2, and a coiled-coil region. Three alternatively spliced transcript variants for this gene have been described, however, the full-length nature of one variant has not been determined.","This gene encodes a large multidomain protein of the phosphoinositide 3-kinase-related kinases (PIKK) family. The encoded protein is a common component of many histone acetyltransferase (HAT) complexes and plays a role in transcription and DNA repair by recruiting HAT complexes to chromatin. Deregulation of this gene may play a role in several types of cancer including glioblastoma multiforme. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","The protein encoded by this gene is a membrane protein and a major controller of thyroid cell metabolism. The encoded protein is a receptor for thyrothropin and thyrostimulin, and its activity is mediated by adenylate cyclase. Defects in this gene are a cause of several types of hyperthyroidism. Three transcript variants encoding different isoforms have been found for this gene.","This gene encodes a progestin-induced protein, which belongs to the HECT (homology to E6-AP carboxyl terminus) family. The HECT family proteins function as E3 ubiquitin-protein ligases, targeting specific proteins for ubiquitin-mediated proteolysis. This gene is localized to chromosome 8q22 which is disrupted in a variety of cancers. This gene potentially has a role in regulation of cell proliferation or differentiation.","Enables thiol-dependent deubiquitinase. Involved in protein deubiquitination and regulation of vesicle-mediated transport. Located in plasma membrane and recycling endosome. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a protein that belongs to the ubiquitin-specific processing protease family of proteins. The encoded protein is thought to regulate the morphology of the endosome by ubiquitination of proteins on this organelle and is involved in cargo sorting and membrane trafficking at the early endosome stage. This protein is required for the cell to enter the S phase of the cell cycle and also functions as a positive regulator in the Hedgehog signaling pathway in development. Pseudogenes of this gene are present on chromosomes 2 and 6. Alternate splicing results in multiple transcript variants.","This gene is a member of the peptidase C19 family and encodes a protein that is similar to ubiquitin-specific proteases. Though this gene is located on the X chromosome, it escapes X-inactivation. Mutations in this gene have been associated with Turner syndrome. Alternate transcriptional splice variants, encoding different isoforms, have been characterized.","This gene is a member of the VAV gene family. The VAV proteins are guanine nucleotide exchange factors (GEFs) for Rho family GTPases that activate pathways leading to actin cytoskeletal rearrangements and transcriptional alterations. The encoded protein is important in hematopoiesis, playing a role in T-cell and B-cell development and activation. The encoded protein has been identified as the specific binding partner of Nef proteins from HIV-1. Coexpression and binding of these partners initiates profound morphological changes, cytoskeletal rearrangements and the JNK/SAPK signaling cascade, leading to increased levels of viral transcription and replication. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","The Wiskott-Aldrich syndrome (WAS) family of proteins share similar domain structure, and are involved in transduction of signals from receptors on the cell surface to the actin cytoskeleton.  The presence of a number of different motifs suggests that they are regulated by a number of different stimuli, and interact with multiple proteins.  Recent studies have demonstrated that these proteins, directly or indirectly, associate with the small GTPase, Cdc42, known to regulate formation of actin filaments, and the cytoskeletal organizing complex, Arp2/3. Wiskott-Aldrich syndrome is a rare, inherited, X-linked, recessive disease characterized by immune dysregulation and microthrombocytopenia, and is caused by mutations in the WAS gene. The WAS gene product is a cytoplasmic protein, expressed exclusively in hematopoietic cells, which show signalling and cytoskeletal abnormalities in WAS patients.  A transcript variant arising as a result of alternative promoter usage, and containing a different 5' UTR sequence, has been described, however, its full-length nature is not known.","The protein encoded by this gene is a cytoplasmic serine-threonine kinase that belongs to the protein kinase superfamily. The protein plays an important role in the regulation of electrolyte homeostasis, cell signaling survival, and proliferation. Alternative splicing results in multiple transcript variants.","Enables transcription coactivator activity. Involved in several processes, including hippo signaling; positive regulation of cell differentiation; and regulation of signal transduction. Located in cytosol and nuclear body. [provided by Alliance of Genome Resources, Apr 2022]","This gene is a member of the Krueppel C2H2-type zinc-finger protein family and encodes a zinc finger transcription factor that contains nine Kruppel-type zinc finger domains at the carboxyl terminus. This protein is located in the nucleus, is involved in cell cycle progression, and interacts with a histone deacetylase. Specific instances of aberrant gene rearrangement at this locus have been associated with acute promyelocytic leukemia (APL). Alternate transcriptional splice variants have been characterized.","This gene, which was initially designated as dendritic cell-derived BTB/POZ zinc finger (DPZF), belongs to a family of transcription factors with an N-terminal BTB/POZ domain and a C-terminal DNA-bindng zinc finger domain. The BTB/POZ domain is a hydrophobic region of approximately 120 aa which mediates association with other BTB/POZ domain-containing proteins. This gene acts as a transcriptional repressor and plays a role in many processes including neurogenesis, glucose homeostasis, and postnatal growth. Mutations in this gene have been associated with Primrose syndrome as well as the 3q13.31 microdeletion syndrome. Alternative splicing results in multiple transcript variants encoding distinct isoforms.","This gene encodes a zinc finger-containing transcription factor that acts as a key regulator of lineage commitment of immature T-cell precursors. It is necessary and sufficient for commitment of CD4 lineage, while its absence causes CD8 commitment. It also functions as a transcriptional repressor of type I collagen genes. Alternatively spliced transcript variants have been found for this gene.","This gene encodes a zinc finger transcription factor. The encoded protein likely plays a role in transcriptional repression of interleukin 2. Mutations in this gene have been associated with posterior polymorphous corneal dystrophy-3 and late-onset Fuchs endothelial corneal dystrophy. Alternatively spliced transcript variants encoding different isoforms have been described.[provided by RefSeq, Mar 2010].","This gene is a member of the TIS11 family of early response genes, which are induced by various agonists such as the phorbol ester TPA and the polypeptide mitogen EGF. This gene is well conserved across species and has a promoter that contains motifs seen in other early-response genes. The encoded protein contains a distinguishing putative zinc finger domain with a repeating cys-his motif. This putative nuclear transcription factor most likely functions in regulating the response to growth factors. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a zinc finger protein that may act as a transcription factor. The encoded protein may be part of a BHC histone deacetylase complex. Translocation of this gene with the fibroblast growth factor receptor-1 gene (FGFR1) results in a fusion gene, which may be a cause of stem cell leukemia lymphoma syndrome (SCLL). Several transcript variants encoding the same protein have been found for this gene.","This gene encodes a zinc finger protein containing a KRAB (Kruppel-associated box) domain found in transcriptional repressors. This gene may be methylated and silenced in cancer cells. This gene is located within a differentially methylated region (DMR) and shows allele-specific expression in placenta. Alternative splicing and the use of alternative promoters results in multiple transcript variants encoding the same protein.","Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Enables protein domain specific binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to act upstream of or within neuron fate commitment. Located in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a protein with a nuclear localization site and a C2H2 zinc finger domain. Mutations in this gene have been associated with seborrhea-like dermatitis with psoriasiform elements.","Enables frizzled binding activity and ubiquitin-protein transferase activity. Involved in cellular protein metabolic process and negative regulation of Wnt signaling pathway. Is integral component of plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes an essential splicing factor. The encoded protein associates with the U2 auxiliary factor heterodimer, which is required for the recognition of a functional 3' splice site in pre-mRNA splicing, and may play a role in network interactions during spliceosome assembly.","Mammalian apolipoprotein B mRNA undergoes site-specific C to U deamination, which is mediated by a multi-component enzyme complex containing a minimal core composed of APOBEC-1 and a complementation factor encoded by this gene. The gene product has three non-identical RNA recognition motifs and belongs to the hnRNP R family of RNA-binding proteins. It has been proposed that this complementation factor functions as an RNA-binding subunit and docks APOBEC-1 to deaminate the upstream cytidine. Studies suggest that the protein may also be involved in other RNA editing or RNA processing events. Several transcript variants encoding a few different isoforms have been found for this gene.","The protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intra-cellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White). This protein is a member of the MRP subfamily which is involved in multi-drug resistance. This family member plays a role in cellular detoxification as a pump for its substrate, organic anions. It may also function in prostaglandin-mediated cAMP signaling in ciliogenesis. Alternative splicing of this gene results in multiple transcript variants.","This gene encodes a member of the Abelson-interactor family of adaptor proteins. These proteins facilitate signal transduction as components of several multiprotein complexes, and regulate actin polymerization and cytoskeletal remodeling through interactions with Abelson tyrosine kinases. The encoded protein plays a role in macropinocytosis as a component of the WAVE2 complex, and also forms a complex with EPS8 and SOS1 that mediates signal transduction from Ras to Rac. This gene may play a role in the progression of several malignancies including melanoma, colon cancer and breast cancer, and a t(10;11) chromosomal translocation involving this gene and the MLL gene has been associated with acute myeloid leukemia. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene, and a pseudogene of this gene is located on the long arm of chromosome 14.","The protein encoded by this gene is an isozyme of the long-chain fatty-acid-coenzyme A ligase family. Although differing in substrate specificity, subcellular localization, and tissue distribution, all isozymes of this family convert free long-chain fatty acids into fatty acyl-CoA esters, and thereby play a key role in lipid biosynthesis and fatty acid degradation. This isozyme is highly expressed in brain, and preferentially utilizes myristate, arachidonate, and eicosapentaenoate as substrates. The amino acid sequence of this isozyme is 92% identical to that of rat homolog. Two transcript variants encoding the same protein have been found for this gene.","The protein encoded by this gene catalyzes the formation of acyl-CoA from fatty acids, ATP, and CoA, using magnesium as a cofactor. The encoded protein plays a major role in fatty acid metabolism in the brain. Translocations with the ETV6 gene are causes of myelodysplastic syndrome with basophilia, acute myelogenous leukemia with eosinophilia, and acute eosinophilic leukemia. Several transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Apr 2011].","Enables butyrate-CoA ligase activity. Predicted to be involved in acyl-CoA metabolic process and fatty acid biosynthetic process. Located in mitochondrion. Implicated in IgA glomerulonephritis. Biomarker of ulcerative colitis. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the ADAMTS (a disintegrin and metalloproteinase with thrombospondin motifs) protein family. Members of the family share several distinct protein modules, including a propeptide region, a metalloproteinase domain, a disintegrin-like domain, and a thrombospondin type 1 (TS) motif. Individual members of this family differ in the number of C-terminal TS motifs, and some have unique C-terminal domains. Members of the ADAMTS family have been implicated in the cleavage of proteoglycans, the control of organ shape during development, and the inhibition of angiogenesis. This gene is localized to chromosome 3p14.3-p14.2, an area known to be lost in hereditary renal tumors. Alternative splicing results in multiple transcript variants encoding different isoforms that may undergo similar proteolytic processing.","This gene encodes a member of the AF4/ lymphoid nuclear protein related to the Fragile X E syndrome (FRAXE) family of proteins, which have been implicated in human childhood lymphoblastic leukemia, fragile chromosome X intellectual disability, and ataxia. It is the prevalent mixed-lineage leukemia fusion gene associated with spontaneous acute lymphoblastic leukemia. Members of this family have three conserved domains: an N-terminal homology domain, an AF4/ lymphoid nuclear protein domain, and a C-terminal homology domain. The protein functions as a regulator of RNA polymerase II-mediated transcription through elongation and chromatin remodeling functions. Through RNA interference screens, this gene has been shown to promote the expression of CD133, a plasma membrane glycoprotein required for leukemia cell survival. Alternative splicing results in multiple transcript variants.","The A-kinase anchor proteins (AKAPs) are a group of structurally diverse proteins which have the common function of binding to the regulatory subunit of protein kinase A (PKA) and confining the holoenzyme to discrete locations within the cell. This gene encodes a member of the AKAP family. Alternate splicing of this gene results in at least two isoforms that localize to the centrosome and the Golgi apparatus, and interact with numerous signaling proteins from multiple signal transduction pathways. These signaling proteins include type II protein kinase A, serine/threonine kinase protein kinase N, protein phosphatase 1, protein phosphatase 2a, protein kinase C-epsilon and phosphodiesterase 4D3.","This gene is a putative oncogene encoding a protein belonging to a subfamily of serine/threonine kinases containing SH2-like (Src homology 2-like) domains, which is involved in signaling pathways. The gene serves as an oncogene in the tumorigenesis of cancer cells For example, its overexpression contributes to the malignant phenotype of a subset of human ductal pancreatic cancers. The encoded protein is a general protein kinase capable of phophorylating several known proteins, and has also been implicated in insulin signaling.","This gene encodes the most abundant protein in human blood. This protein functions in the regulation of blood plasma colloid osmotic pressure and acts as a carrier protein for a wide range of endogenous molecules including hormones, fatty acids, and metabolites, as well as exogenous drugs. Additionally, this protein exhibits an esterase-like activity with broad substrate specificity. The encoded preproprotein is proteolytically processed to generate the mature protein. A peptide derived from this protein, EPI-X4, is an endogenous inhibitor of the CXCR4 chemokine receptor.","This protein belongs to the aldehyde dehydrogenase family of proteins. Aldehyde dehydrogenase is the second enzyme of the major oxidative pathway of alcohol metabolism. Two major liver isoforms of aldehyde dehydrogenase, cytosolic and mitochondrial, can be distinguished by their electrophoretic mobilities, kinetic properties, and subcellular localizations. Most Caucasians have two major isozymes, while approximately 50% of East Asians have the cytosolic isozyme but not the mitochondrial isozyme. A remarkably higher frequency of acute alcohol intoxication among East Asians than among Caucasians could be related to the absence of a catalytically active form of the mitochondrial isozyme. The increased exposure to acetaldehyde in individuals with the catalytically inactive form may also confer greater susceptibility to many types of cancer. This gene encodes a mitochondrial isoform, which has a low Km for acetaldehydes, and is localized in mitochondrial matrix. Alternative splicing results in multiple transcript variants encoding distinct isoforms.[provided by RefSeq, Nov 2016].","This gene encodes an alpha kinase. Mice which were homozygous for disrupted copies of this gene exhibited coordination defects (PMID: 21208416). Multiple transcript variants encoding different isoforms have been found for this gene.","This locus encodes an ankryin repeat domain-containing protein. The encoded protein inhibits ligand-dependent activation of transcription. Mutations in this gene have been associated with KBG syndrome, which is characterized by macrodontia, distinctive craniofacial features, short stature, skeletal anomalies, global developmental delay, seizures and intellectual disability. Alternatively spliced transcript variants have been described. Related pseudogenes exist on chromosomes 2 and X.","This gene product is the main apolipoprotein of chylomicrons and low density lipoproteins (LDL), and is the ligand for the LDL receptor. It occurs in plasma as two main isoforms, apoB-48 and apoB-100: the former is synthesized exclusively in the gut and the latter in the liver. The intestinal and the hepatic forms of apoB are encoded by a single gene from a single, very long mRNA. The two isoforms share a common N-terminal sequence. The shorter apoB-48 protein is produced after RNA editing of the apoB-100 transcript at residue 2180 (CAA->UAA), resulting in the creation of a stop codon, and early translation termination. Mutations in this gene or its regulatory region cause hypobetalipoproteinemia, normotriglyceridemic hypobetalipoproteinemia, and hypercholesterolemia due to ligand-defective apoB, diseases affecting plasma cholesterol and apoB levels.","This gene is a member of the cytidine deaminase gene family. It is one of seven related genes or pseudogenes found in a cluster, thought to result from gene duplication, on chromosome 22. Members of the cluster encode proteins that are structurally and functionally related to the C to U RNA-editing cytidine deaminase APOBEC1. It is thought that the proteins may be RNA editing enzymes and have roles in growth or cell cycle control. A hybrid gene results from the deletion of approximately 29.5 kb of sequence between this gene, APOBEC3B, and the adjacent gene APOBEC3A. The breakpoints of the deletion are within the two genes, so the deletion allele is predicted to have the promoter and coding region of APOBEC3A, but the 3' UTR of APOBEC3B. Two transcript variants encoding different isoforms have been found for this gene.","Interaction of a cell with the extracellular matrix triggers integrin cell surface receptors to begin signaling cascades that regulate the organization of the actin-cytoskeleton. One of the proteins involved in these cascades is focal adhesion kinase. The protein encoded by this gene is a GTPase activating protein that binds to focal adhesion kinase and mediates the activity of the GTP binding proteins RhoA and Cdc42. Defects in this gene are a cause of juvenile myelomonocytic leukemia (JMML). Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the AT-rich interaction domain (ARID) family of DNA binding proteins. The encoded protein forms a histone H3K9Me2 demethylase complex with PHD finger protein 2 and regulates the transcription of target genes involved in adipogenesis and liver development. This gene also plays a role in cell growth and differentiation of B-lymphocyte progenitors, and single nucleotide polymorphisms in this gene are associated with acute lymphoblastic leukemia. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","This gene encodes an activating transcription factor, which belongs to the ATF subfamily and bZIP (basic-region leucine zipper) family. It influences cellular physiologic processes by regulating the expression of downstream target genes, which are related to growth, survival, and other cellular activities. This protein is phosphorylated at serine 63 in its kinase-inducible domain by serine/threonine kinases, cAMP-dependent protein kinase A, calmodulin-dependent protein kinase I/II, mitogen- and stress-activated protein kinase and cyclin-dependent kinase 3 (cdk-3). Its phosphorylation enhances its transactivation and transcriptional activities, and enhances cell transformation. Fusion of this gene and FUS on chromosome 16 or EWSR1 on chromosome 22 induced by translocation generates chimeric proteins in angiomatoid fibrous histiocytoma and clear cell sarcoma. This gene has a pseudogene on chromosome 6.","This gene encodes an E1-like activating enzyme that is essential for autophagy and cytoplasmic to vacuole transport. The encoded protein is also thought to modulate p53-dependent cell cycle pathways during prolonged metabolic stress. It has been associated with multiple functions, including axon membrane trafficking, axonal homeostasis, mitophagy, adipose differentiation, and hematopoietic stem cell maintenance. Alternative splicing results in multiple transcript variants.","Machado-Joseph disease, also known as spinocerebellar ataxia-3, is an autosomal dominant neurologic disorder. The protein encoded by this gene contains (CAG)n repeats in the coding region, and the expansion of these repeats from the normal 12-44 to 52-86 is one cause of Machado-Joseph disease. There is a negative correlation between the age of onset and CAG repeat numbers. Alternatively spliced transcript variants encoding different isoforms have been described for this gene.","The protein encoded by this gene belongs to the BCL2 protein family. BCL2 family members form hetero- or homodimers and act as anti- or pro-apoptotic regulators that are involved in a wide variety of cellular activities. This protein forms a heterodimer with BCL2, and functions as an apoptotic activator. The association and the ratio of BAX to BCL2 also determines survival or death of a cell following an apoptotic stimulus. This protein is reported to interact with, and increase the opening of, the mitochondrial voltage-dependent anion channel (VDAC), which leads to the loss in membrane potential and the release of cytochrome c. The expression of this gene is regulated by the tumor suppressor P53 and has been shown to be involved in P53-mediated apoptosis. Multiple alternatively spliced transcript variants, which encode different isoforms, have been reported for this gene.","The protein encoded by this gene belongs to the BCL-2 protein family. BCL-2 family members form hetero- or homodimers and act as anti- or pro-apoptotic regulators that are involved in a wide variety of cellular activities. The protein encoded by this gene contains a Bcl-2 homology domain 3 (BH3). It has been shown to interact with other members of the BCL-2 protein family and to act as an apoptotic activator. The expression of this gene can be induced by nerve growth factor (NGF), as well as by the forkhead transcription factor FKHR-L1, which suggests a role of this gene in neuronal and lymphocyte apoptosis. Transgenic studies of the mouse counterpart suggested that this gene functions as an essential initiator of apoptosis in thymocyte-negative selection. Several alternatively spliced transcript variants of this gene have been identified.","This gene encodes a member of a family of proteins containing a Bcl-2 homology domain 2 (BH2). The encoded protein is an anti-apoptotic factor that acts as an inhibitor of caspases 3 and 7 in the cytoplasm. In the nucleus, it binds to the p53 tumor suppressor protein, preventing its association with target genes. Overexpression of this gene has been detected in a number of different cancers. There is a pseudogene for this gene on chromosome 3. Alternative splicing results in multiple transcript variants.","This gene is a proto-oncogene candidate. It is identified by its translocation into the immunoglobulin alpha-locus in some cases of B-cell leukemia. The protein encoded by this gene contains seven ankyrin repeats, which are most closely related to those found in I kappa B proteins. This protein functions as a transcriptional co-activator that activates through its association with NF-kappa B homodimers. The expression of this gene can be induced by NF-kappa B, which forms a part of the autoregulatory loop that controls the nuclear residence of p50 NF-kappa B.","This gene is directly involved, with Myc and IgH, in a three-way gene translocation in a Burkitt lymphoma cell line. As a result of the gene translocation, the N-terminal region of the gene product is disrupted, which is thought to be related to the pathogenesis of a subset of high-grade B cell non-Hodgkin lymphoma. The N-terminal segment involved in the translocation includes the region that shares a strong sequence similarity with those of BCL7B and BCL7C. Two transcript variants encoding different isoforms have been found for this gene.","A reciprocal translocation between chromosomes 22 and 9 produces the Philadelphia chromosome, which is often found in patients with chronic myelogenous leukemia. The chromosome 22 breakpoint for this translocation is located within the BCR gene. The translocation produces a fusion protein which is encoded by sequence from both BCR and ABL, the gene at the chromosome 9 breakpoint. Although the BCR-ABL fusion protein has been extensively studied, the function of the normal BCR gene product is not clear. The unregulated tyrosine kinase activity of BCR-ABL1 contributes to the immortality of leukaemic cells. The BCR protein has serine/threonine kinase activity and is a GTPase-activating protein for p21rac and other kinases. Two transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jan 2020].","The Bloom syndrome is an autosomal recessive disorder characterized by growth deficiency, microcephaly and immunodeficiency among others. It is caused by homozygous or compound heterozygous mutation in the gene encoding DNA helicase RecQ protein on chromosome 15q26. This Bloom-associated helicase unwinds a variety of DNA substrates including Holliday junction, and is involved in several pathways contributing to the maintenance of genome stability. Identification of pathogenic Bloom variants is required for heterozygote testing in at-risk families.","This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate each subunit of the disulfide-linked homodimer, which plays a role in bone and cartilage development. Polymorphisms in this gene may be associated with osteoarthritis in human patients. This gene is differentially regulated in multiple human cancers. This gene encodes distinct protein isoforms that may be similarly proteolytically processed.","This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate each subunit of the disulfide-linked homodimer. This protein regulates a wide range of biological processes including iron homeostasis, fat and bone development, and ovulation. Differential expression of this gene may be associated with progression of breast and prostate cancer. Mutations in this gene may be associated with iron overload in human patients.","This gene was identified based on its homology to the gene encoding the RING3 protein, a serine/threonine kinase. The gene localizes to 9q34, a region which contains several major histocompatibility complex (MHC) genes. The function of the encoded protein is not known.","The protein encoded by this gene is a member of the RecQ DEAH helicase family and interacts with the BRCT repeats of breast cancer, type 1 (BRCA1). The bound complex is important in the normal double-strand break repair function of breast cancer, type 1 (BRCA1). This gene may be a target of germline cancer-inducing mutations.","This gene encodes a kinase involved in spindle checkpoint function. The protein has been localized to the kinetochore and plays a role in the inhibition of the anaphase-promoting complex/cyclosome (APC/C), delaying the onset of anaphase and ensuring proper chromosome segregation. Impaired spindle checkpoint function has been found in many forms of cancer.","Voltage-dependent calcium channels mediate the entry of calcium ions into excitable cells, and are also involved in a variety of calcium-dependent processes, including muscle contraction, hormone or neurotransmitter release, and gene expression. Calcium channels are multisubunit complexes composed of alpha-1, beta, alpha-2/delta, and gamma subunits. The channel activity is directed by the pore-forming alpha-1 subunit, whereas, the others act as auxiliary subunits regulating this activity. The distinctive properties of the calcium channel types are related primarily to the expression of a variety of alpha-1 isoforms, alpha-1A, B, C, D, E, and S. This gene encodes the alpha-1A subunit, which is predominantly expressed in neuronal tissue. Mutations in this gene are associated with 2 neurologic disorders, familial hemiplegic migraine and episodic ataxia 2. This gene also exhibits polymorphic variation due to (CAG)n-repeats. Multiple transcript variants encoding different isoforms have been found for this gene. In one set of transcript variants, the (CAG)n-repeats occur in the 3' UTR, and are not associated with any disease. But in another set of variants, an insertion extends the coding region to include the (CAG)n-repeats which encode a polyglutamine tract. Expansion of the (CAG)n-repeats from the normal 4-18 to 21-33 in the coding region is associated with spinocerebellar ataxia 6.","This gene encodes a member of the synaptic cell adhesion molecule 1 (SynCAM) family which belongs to the immunoglobulin (Ig) superfamily. The encoded protein has three Ig-like domains and a cytosolic protein 4.1 binding site near the C-terminus. Proteins belonging to the protein 4.1 family crosslink spectrin and interact with other cytoskeletal proteins. Multiple transcript variants encoding different isoforms have been found for this gene.","Calreticulin is a highly conserved chaperone protein which resides primarily in the endoplasmic reticulum, and is involved in a variety of cellular processes, among them, cell adhesion. Additionally, it functions in protein folding quality control and calcium homeostasis. Calreticulin is also found in the nucleus, suggesting that it may have a role in transcription regulation. Systemic lupus erythematosus is associated with increased autoantibody titers against calreticulin. Recurrent mutations in calreticulin have been linked to various neoplasms, including the myeloproliferative type.[provided by RefSeq, May 2020].","This protein encoded by this gene belongs to the apyrase family. It functions as a calcium-dependent nucleotidase with a preference for UDP. Mutations in this gene are associated with Desbuquois dysplasia with hand anomalies. Alternatively spliced transcript variants have been noted for this gene.[provided by RefSeq, Mar 2010].","This gene encodes a class 1 aminoacyl-tRNA synthetase, cysteinyl-tRNA synthetase. Each of the twenty aminoacyl-tRNA synthetases catalyzes the aminoacylation of a specific tRNA or tRNA isoaccepting family with the cognate amino acid. This gene is one of several located near the imprinted gene domain on chromosome 11p15.5, an important tumor-suppressor gene region. Alterations in this region have been associated with Beckwith-Wiedemann syndrome, Wilms tumor, rhabdomyosarcoma, adrenocortical carcinoma, and lung, ovarian and breast cancers. Alternative splicing of this gene results in multiple transcript variants.","The protein encoded by this gene is a cysteine-aspartic acid protease that plays a central role in the execution-phase of cell apoptosis. The encoded protein cleaves and inactivates poly(ADP-ribose) polymerase while it cleaves and activates sterol regulatory element binding proteins as well as caspases 6, 7, and 9. This protein itself is processed by caspases 8, 9, and 10. It is the predominant caspase involved in the cleavage of amyloid-beta 4A precursor protein, which is associated with neuronal death in Alzheimer's disease.","This gene encodes a member of the cysteine-aspartic acid protease (caspase) family. Sequential activation of caspases plays a central role in the execution-phase of cell apoptosis. Caspases exist as inactive proenzymes which undergo proteolytic processing at conserved aspartic residues to produce two subunits, large and small, that dimerize to form the active enzyme. This protein can undergo autoproteolytic processing and activation by the apoptosome, a protein complex of cytochrome c and the apoptotic peptidase activating factor 1; this step is thought to be one of the earliest in the caspase activation cascade. This protein is thought to play a central role in apoptosis and to be a tumor suppressor. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a zinc finger transcription factor. The encoded protein may function as a tumor suppressor, and single nucleotide polymorphisms in this gene are associated with blood pressure variation. Alternative splicing results in multiple transcript variants that encode different protein isoforms.","This gene encodes a member of the myeloid translocation gene family which interact with DNA-bound transcription factors and recruit a range of corepressors to facilitate transcriptional repression. The t(16;21)(q24;q22) translocation is one of the less common karyotypic abnormalities in acute myeloid leukemia. The translocation produces a chimeric gene made up of the 5'-region of the runt-related transcription factor 1 gene fused to the 3'-region of this gene. This gene is also a putative breast tumor suppressor. Alternative splicing results in transcript variants.","This gene encodes a member of the Cbl family of E3 ubiquitin ligases. Cbl proteins play important roles in cell signaling through the ubiquitination and subsequent downregulation of tyrosine kinases. Expression of this gene may be restricted to epithelial cells, and alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","Enables RNA polymerase II cis-regulatory region sequence-specific DNA binding activity; nuclear receptor coactivator activity; and transcription corepressor activity. Involved in positive regulation of cell migration and positive regulation of cell population proliferation. Acts upstream of or within positive regulation of apoptotic process. Located in nuclear envelope lumen. [provided by Alliance of Genome Resources, Apr 2022]","HEI10 is a member of the E3 ubiquitin ligase family and functions in progression of the cell cycle through G(2)/M.","The protein encoded by this gene is a member of the cyclin family of proteins. The encoded protein interacts with cyclin-dependent kinase 8 and induces the phophorylation of the carboxy-terminal domain of the large subunit of RNA polymerase II. The level of mRNAs for this gene peaks in the G1 phase of the cell cycle. Two transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene belongs to the highly conserved cyclin family, whose members are characterized by a dramatic periodicity in protein abundance through the cell cycle. Cyclins function as regulators of CDK kinases. Different cyclins exhibit distinct expression and degradation patterns which contribute to the temporal coordination of each mitotic event. This cyclin forms a complex with and functions as a regulatory subunit of CDK2, whose activity is required for cell cycle G1/S transition. This protein accumulates at the G1-S phase boundary and is degraded as cells progress through S phase. Overexpression of this gene has been observed in many tumors, which results in chromosome instability, and thus may contribute to tumorigenesis. This protein was found to associate with, and be involved in, the phosphorylation of NPAT protein (nuclear protein mapped to the ATM locus), which participates in cell-cycle regulated histone gene expression and plays a critical role in promoting cell-cycle progression in the absence of pRB.","This gene encodes a member of the highly conserved cyclin C subfamily. The encoded protein tightly associates with cyclin-dependent kinase 9, and is a major subunit of positive transcription elongation factor b (p-TEFb). In humans, there are multiple forms of positive transcription elongation factor b, which may include one of several different cyclins along with cyclin-dependent kinase 9. The complex containing the encoded cyclin and cyclin-dependent kinase 9 acts as a cofactor of human immunodeficiency virus type 1 (HIV-1) Tat protein, and is both necessary and sufficient for full activation of viral transcription. This cyclin and its kinase partner are also involved in triggering transcript elongation through phosphorylation of the carboxy-terminal domain of the largest RNA polymerase II subunit. Overexpression of this gene is implicated in tumor growth. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene belongs to the G-protein-coupled receptor family . It is a receptor for the CC chemokine - MIP-1, RANTES, TARC and MCP-1. Chemokines are a group of small polypeptide, structurally related molecules that regulate cell trafficking of various types of leukocytes. The chemokines also play fundamental roles in the development, homeostasis, and function of the immune system, and they have effects on cells of the central nervous system as well as on endothelial cells involved in angiogenesis or angiostasis.","The protein encoded by this gene is essential for T-cell proliferation and survival, cytokine production, and T-helper type-2 development. Several alternatively spliced transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Jul 2011].","This gene encodes a member of the immunoglobulin superfamily. The encoded protein is a ligand of the T lymphocyte CD2 protein, and functions in adhesion and activation of T lymphocytes. The protein is localized to the plasma membrane. Alternatively spliced transcript variants have been described.","The protein encoded by this gene is a cytokine that belongs to the tumor necrosis factor (TNF) ligand family. This cytokine is a ligand for TNFRSF27/CD27. It is a surface antigen on activated, but not on resting, T and B lymphocytes. It induces proliferation of costimulated T cells, enhances the generation of cytolytic T cells, and contributes to T cell activation. This cytokine is also reported to play a role in regulating B-cell activation, cytotoxic function of natural killer cells, and immunoglobulin sythesis.","The protein encoded by this gene associates with class II major histocompatibility complex (MHC) and is an important chaperone that regulates antigen presentation for immune response. It also serves as cell surface receptor for the cytokine macrophage migration inhibitory factor (MIF) which, when bound to the encoded protein, initiates survival pathways and cell proliferation. This protein also interacts with amyloid precursor protein (APP) and suppresses the production of amyloid beta (Abeta). Multiple alternatively spliced transcript variants encoding different isoforms have been identified.","The B lymphocyte antigen receptor is a multimeric complex that includes the antigen-specific component, surface immunoglobulin (Ig). Surface Ig non-covalently associates with two other proteins, Ig-alpha and Ig-beta, which are necessary for expression and function of the B-cell antigen receptor. This gene encodes the Ig-alpha protein of the B-cell antigen component. Alternatively spliced transcript variants encoding different isoforms have been described.","This gene is a member of the cadherin superfamily, genes encoding calcium-dependent, membrane-associated glycoproteins. The encoded protein is cadherin-like, consisting of an extracellular region, containing 7 cadherin domains, and a transmembrane region but lacking the conserved cytoplasmic domain. The protein is a component of the gastrointestinal tract and pancreatic ducts, acting as an intestinal proton-dependent peptide transporter in the first step in oral absorption of many medically important peptide-based drugs. The protein may also play a role in the morphological organization of liver and intestine. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a member of the CMGC family of serine/threonine protein kinases. This kinase is a catalytic subunit of the protein kinase complex that is important for cell cycle G1 phase progression and G1/S transition. The activity of this kinase first appears in mid-G1 phase, which is controlled by the regulatory subunits including D-type cyclins and members of INK4 family of CDK inhibitors. This kinase, as well as CDK4, has been shown to phosphorylate, and thus regulate the activity of, tumor suppressor protein Rb. Altered expression of this gene has been observed in multiple human cancers. A mutation in this gene resulting in reduced cell proliferation, and impaired cell motility and polarity, and has been identified in patients with primary microcephaly.","Involved in non-motile cilium assembly. Acts upstream of or within cilium assembly. Located in several cellular components, including cytosol; microtubule cytoskeleton; and non-motile cilium. Part of ciliary transition fiber. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the CHD family of proteins which are characterized by the presence of chromo (chromatin organization modifier) domains and SNF2-related helicase/ATPase domains. This protein is one of the components of a histone deacetylase complex referred to as the Mi-2/NuRD complex which participates in the remodeling of chromatin by deacetylating histones. Chromatin remodeling is essential for many processes including transcription. Autoantibodies against this protein are found in a subset of patients with dermatomyositis. Three alternatively spliced transcripts encoding different isoforms have been described.","This gene encodes a protein that contains several helicase family domains. Mutations in this gene have been found in some patients with the CHARGE syndrome. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the chromodomain-helicase-DNA binding protein family, which is characterized by a SNF2-like domain and two chromatin organization modifier domains. The encoded protein also contains brahma and kismet domains, which are common to the subfamily of chromodomain-helicase-DNA binding proteins to which this protein belongs. This gene has been shown to function in several processes that include transcriptional regulation, epigenetic remodeling, promotion of cell proliferation, and regulation of RNA synthesis. Allelic variants of this gene are associated with autism spectrum disorder. Alternative splicing results in multiple transcript variants.","This gene encodes an antagonist of bone morphogenetic protein 4. The encoded protein may play a role in topographic retinotectal projection and in the regulation of retinal angiogenesis in response to hypoxia. Alternatively spliced transcript variants encoding different isoforms have been described.","The protein encoded by this gene belongs to the sulfotransferase 2 family. It is localized to the golgi membrane, and catalyzes the transfer of sulfate to position 4 of the N-acetylgalactosamine (GalNAc) residue of chondroitin. Chondroitin sulfate constitutes the predominant proteoglycan present in cartilage, and is distributed on the surfaces of many cells and extracellular matrices. A chromosomal translocation involving this gene and IgH, t(12;14)(q23;q32), has been reported in a patient with B-cell chronic lymphocytic leukemia. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene links endocytic vesicles to microtubules. This gene is highly expressed in Reed-Sternberg cells of Hodgkin disease. Several transcript variants encoding different isoforms have been found for this gene.","Enables mRNA (nucleoside-2'-O-)-methyltransferase activity. Involved in 7-methylguanosine mRNA capping and cap2 mRNA methylation. Located in cytoplasm and nucleus. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a nucleic-acid binding protein with seven zinc-finger domains. The protein has a preference for binding single stranded DNA and RNA. The protein functions in cap-independent translation of ornithine decarboxylase mRNA, and may also function in sterol-mediated transcriptional regulation. A CCTG expansion from <30 repeats to 75-11000 repeats in the first intron of this gene results in myotonic dystrophy type 2. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a centrosomal protein required for the centrosome to function as a microtubule organizing center. The gene product is also associated with centrosome maturation. One version of stem cell myeloproliferative disorder is the result of a reciprocal translocation between chromosomes 8 and 9, with the breakpoint associated with fibroblast growth factor receptor 1 and centrosomal protein 1.","This gene encodes the pro-alpha1 chains of type I collagen whose triple helix comprises two alpha1 chains and one alpha2 chain. Type I is a fibril-forming collagen found in most connective tissues and is abundant in bone, cornea, dermis and tendon. Mutations in this gene are associated with osteogenesis imperfecta types I-IV, Ehlers-Danlos syndrome type VIIA, Ehlers-Danlos syndrome Classical type, Caffey Disease and idiopathic osteoporosis. Reciprocal translocations between chromosomes 17 and 22, where this gene and the gene for platelet-derived growth factor beta are located, are associated with a particular type of skin tumor called dermatofibrosarcoma protuberans, resulting from unregulated expression of the growth factor. Two transcripts, resulting from the use of alternate polyadenylation signals, have been identified for this gene. [provided by R. Dalgleish, Feb 2008].","This gene encodes an alpha chain for one of the low abundance fibrillar collagens. Fibrillar collagen molecules are trimers that can be composed of one or more types of alpha chains. Type V collagen is found in tissues containing type I collagen and appears to regulate the assembly of heterotypic fibers composed of both type I and type V collagen. This gene product is closely related to type XI collagen and it is possible that the collagen chains of types V and XI constitute a single collagen type with tissue-specific chain combinations. The encoded procollagen protein occurs commonly as the heterotrimer pro-alpha1(V)-pro-alpha1(V)-pro-alpha2(V). Mutations in this gene are associated with Ehlers-Danlos syndrome, types I and II. Alternative splicing of this gene results in multiple transcript variants.","This gene encodes the alpha-3 chain, one of the three alpha chains of type VI collagen, a beaded filament collagen found in most connective tissues. The alpha-3 chain of type VI collagen is much larger than the alpha-1 and -2 chains. This difference in size is largely due to an increase in the number of subdomains, similar to von Willebrand Factor type A domains, that are found in the amino terminal globular domain of all the alpha chains. These domains have been shown to bind extracellular matrix proteins, an interaction that explains the importance of this collagen in organizing matrix components. Mutations in the type VI collagen genes are associated with Bethlem myopathy, a rare autosomal dominant proximal myopathy with early childhood onset. Mutations in this gene are also a cause of Ullrich congenital muscular dystrophy, also referred to as Ullrich scleroatonic muscular dystrophy, an autosomal recessive congenital myopathy that is more severe than Bethlem myopathy. Multiple transcript variants have been identified, but the full-length nature of only some of these variants has been described.","This gene encodes a large protein that contains multiple von Willebrand factor domains and forms a component of the basal lamina of epithelial cells. This protein may regulate epithelial cell-fibronectin interactions. Variation in this gene may be implicated in skin diseases.","This gene encodes a transcription factor that is a member of the leucine zipper family of DNA binding proteins. This protein binds as a homodimer to the cAMP-responsive element, an octameric palindrome. The protein is phosphorylated by several protein kinases, and induces transcription of genes in response to hormonal stimulation of the cAMP pathway. Alternate splicing of this gene results in several transcript variants encoding different isoforms.","The protein encoded by this gene is normally found in the membrane of the endoplasmic reticulum (ER). However, upon stress to the ER, the encoded protein is cleaved and the released cytoplasmic transcription factor domain translocates to the nucleus. There it activates the transcription of target genes by binding to box-B elements.","This gene encodes a member of the oasis bZIP transcription factor family. Members of this family can dimerize but form homodimers only. The encoded protein is a transcriptional activator. Translocations between this gene on chromosome 7 and the gene fused in sarcoma on chromosome 16 can be found in some tumors. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the basic-leucine zipper family and the AMP-dependent transcription factor family. The encoded protein is localized to the endoplasmic reticulum and acts as a transcription factor activated by cyclic AMP stimulation. The encoded protein binds the cyclic AMP response element (CRE) and the box-B element and has been linked to acute inflammatory response, hepatocellular carcinoma, triglyceride metabolism, and hepcidin expression. Alternative splicing results in multiple transcript variants.","The crooked neck (crn) gene of Drosophila is essential for embryogenesis and is thought to be involved in cell cycle progression and pre-mRNA splicing. A protein encoded by this human locus has been found to localize to pre-mRNA splicing complexes in the nucleus and is necessary for pre-mRNA splicing. Alternatively spliced transcript variants have been described.","Enables RNA stem-loop binding activity. Involved in IRES-dependent viral translational initiation; nuclear-transcribed mRNA catabolic process, no-go decay; and stress granule assembly. Located in Golgi apparatus; cytosol; and plasma membrane. Part of CRD-mediated mRNA stability complex. [provided by Alliance of Genome Resources, Apr 2022]","Predicted to be involved in regulation of dendrite development. Predicted to be located in plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]","A human melanoma-associated chondroitin sulfate proteoglycan plays a role in stabilizing cell-substratum interactions during early events of melanoma cell spreading on endothelial basement membranes. CSPG4 represents an integral membrane chondroitin sulfate proteoglycan expressed by human malignant melanoma cells.","This gene encodes a member of the catenin family of proteins that play an important role in cell adhesion process by connecting cadherins located on the plasma membrane to the actin filaments inside the cell. The encoded mechanosensing protein contains three vinculin homology domains and undergoes conformational changes in response to cytoskeletal tension, resulting in the reconfiguration of cadherin-actin filament connections. Certain mutations in this gene cause butterfly-shaped pigment dystrophy.","Predicted to enable ubiquitin protein ligase binding activity and ubiquitin-protein transferase activity. Involved in SCF-dependent proteasomal ubiquitin-dependent protein catabolic process and protein ubiquitination. Located in plasma membrane. Part of Parkin-FBXW7-Cul1 ubiquitin ligase complex and SCF ubiquitin ligase complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a chromatin-associated protein that associates with other DNA-binding transcription factors to regulate gene expression and cell fate determination during development. The protein contains a Ski domain that is highly conserved from Drosophila to human. Expression of this gene is lost in some forms of metastatic cancer, and is correlated with poor prognosis. Multiple transcript variants encoding different isoforms have been found for this gene.","In mammals, the Y chromosome directs the development of the testes and plays an important role in spermatogenesis. A high percentage of infertile men have deletions that map to regions of the Y chromosome. The DAZ (deleted in azoospermia) gene cluster maps to the AZFc region of the Y chromosome and is deleted in many azoospermic and severely oligospermic men. It is thought that the DAZ gene cluster arose from the transposition, amplification, and pruning of the ancestral autosomal gene DAZL also involved in germ cell development and gametogenesis. This gene encodes a RNA-binding protein with two RNP motifs that was originally identified by its interaction with the infertility factors DAZ and DAZL. Two isoforms are encoded by transcript variants of this gene.","This gene encodes a member of the CCAAT/enhancer-binding protein (C/EBP) family of transcription factors. The protein functions as a dominant-negative inhibitor by forming heterodimers with other C/EBP members, such as C/EBP and LAP (liver activator protein), and preventing their DNA binding activity. The protein is implicated in adipogenesis and erythropoiesis, is activated by endoplasmic reticulum stress, and promotes apoptosis. Fusion of this gene and FUS on chromosome 16 or EWSR1 on chromosome 22 induced by translocation generates chimeric proteins in myxoid liposarcomas or Ewing sarcoma. Multiple alternatively spliced transcript variants encoding two isoforms with different length have been identified.","DEAD box proteins, characterized by the conserved motif Asp-Glu-Ala-Asp (DEAD), are putative RNA helicases. They are implicated in a number of cellular processes involving alteration of RNA secondary structure such as translation initiation, nuclear and mitochondrial splicing, and ribosome and spliceosome assembly. Based on their distribution patterns, some members of this family are believed to be involved in embryogenesis, spermatogenesis, and cellular growth and division. This gene encodes a DEAD box protein, and it may be involved in ribosome assembly. Fusion of this gene and the nucleoporin gene, NUP98, by inversion 11 (p15q22) chromosome translocation is found in the patients with de novo or therapy-related myeloid malignancies.","This gene encodes a member of the DEAD box family of RNA helicases that are involved in a variety of cellular processes as a result of its role as an adaptor molecule, promoting interactions with a large number of other factors. This protein is involved in pathways that include the alteration of RNA structures, plays a role as a coregulator of transcription, a regulator of splicing, and in the processing of small noncoding RNAs. Members of this family contain nine conserved motifs, including the conserved Asp-Glu-Ala-Asp (DEAD) motif, important to ATP binding and hydrolysis as well as RNA binding and unwinding activities. Dysregulation of this gene may play a role in cancer development. Alternative splicing results in multiple transcript variants.","This gene encodes a protein with one SAP domain. This protein binds to cruciform and superhelical DNA and induces positive supercoils into closed circular DNA, and is also involved in splice site selection during mRNA processing. Chromosomal aberrations involving this region, increased expression of this gene, and the presence of antibodies against this protein are all associated with various diseases. Two transcript variants encoding different isoforms have been found for this gene.","The product of this gene belongs to the diaphanous subfamily of the formin homology family of proteins. This gene may play a role in the development and normal function of the ovaries. Defects in this gene have been linked to premature ovarian failure 2. Alternatively spliced transcript variants encoding different isoforms have been identified.","Enables 3'-5'-exoribonuclease activity; endonuclease activity; and guanyl-nucleotide exchange factor activity. Involved in CUT catabolic process and rRNA catabolic process. Located in cytosol and nucleoplasm. Part of nuclear exosome (RNase complex). [provided by Alliance of Genome Resources, Apr 2022]","This gene spans a genomic range of greater than 2 Mb and encodes a large protein containing an N-terminal actin-binding domain and multiple spectrin repeats. The encoded protein forms a component of the dystrophin-glycoprotein complex (DGC), which bridges the inner cytoskeleton and the extracellular matrix. Deletions, duplications, and point mutations at this gene locus may cause Duchenne muscular dystrophy (DMD), Becker muscular dystrophy (BMD), or cardiomyopathy. Alternative promoter usage and alternative splicing result in numerous distinct transcript variants and protein isoforms for this gene.","Dynamins represent one of the subfamilies of GTP-binding proteins. These proteins share considerable sequence similarity over the N-terminal portion of the molecule, which contains the GTPase domain. Dynamins are associated with microtubules. They have been implicated in cell processes such as endocytosis and cell motility, and in alterations of the membrane that accompany certain activities such as bone resorption by osteoclasts. Dynamins bind many proteins that bind actin and other cytoskeletal proteins. Dynamins can also self-assemble, a process that stimulates GTPase activity. Five alternatively spliced transcripts encoding different proteins have been described. Additional alternatively spliced transcripts may exist, but their full-length nature has not been determined.","This gene encodes an enzyme that transfers methyl groups to cytosine nucleotides of genomic DNA. This protein is the major enzyme responsible for maintaining methylation patterns following DNA replication and shows a preference for hemi-methylated DNA. Methylation of DNA is an important component of mammalian epigenetic gene regulation. Aberrant methylation patterns are found in human tumors and associated with developmental abnormalities. Variation in this gene has been associated with cerebellar ataxia, deafness, and narcolepsy, and neuropathy, hereditary sensory, type IE. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a histone methyltransferase that methylates lysine-79 of histone H3. It is inactive against free core histones, but shows significant histone methyltransferase activity against nucleosomes.","This gene encodes a protein that anchors intermediate filaments to desmosomal plaques and forms an obligate component of functional desmosomes. Mutations in this gene are the cause of several cardiomyopathies and keratodermas, including skin fragility-woolly hair syndrome. Alternative splicing results in multiple transcript variants.","Studies in Drosophila have identified this gene as encoding a positive regulator of the Notch-signaling pathway. The human gene encodes a protein of unknown function; however, it may play a role in basic helix-loop-helix transcription factor activity.","This gene encodes a mitogen-activated protein kinase phosphatase that is a member of the dual specificity protein phosphatase subfamily. These phosphatases inactivate their target kinases by dephosphorylating both the phosphoserine/threonine and phosphotyrosine residues. The encoded protein specifically regulates the c-Jun amino-terminal kinase (JNK) and extracellular signal-regulated kinase (ERK) pathways.[provided by RefSeq, May 2010].","Predicted to enable guanyl-nucleotide exchange factor activity. Predicted to be involved in regulation of catalytic activity. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the Polycomb-group (PcG) family. PcG family members form multimeric protein complexes, which are involved in maintaining the transcriptional repressive state of genes over successive cell generations. This protein interacts with enhancer of zeste 2, the cytoplasmic tail of integrin beta7, immunodeficiency virus type 1 (HIV-1) MA protein, and histone deacetylase proteins. This protein mediates repression of gene activity through histone deacetylation, and may act as a specific regulator of integrin function. Two transcript variants encoding distinct isoforms have been identified for this gene.","This gene encodes an isoform of the alpha subunit of the elongation factor-1 complex, which is responsible for the enzymatic delivery of aminoacyl tRNAs to the ribosome. This isoform (alpha 1) is expressed in brain, placenta, lung, liver, kidney, and pancreas, and the other isoform (alpha 2) is expressed in brain, heart and skeletal muscle. This isoform is identified as an autoantigen in 66% of patients with Felty syndrome. This gene has been found to have multiple copies on many chromosomes, some of which, if not all, represent different pseudogenes.","This gene encodes a member of the GTP-binding translation elongation factor family. This protein is an essential factor for protein synthesis. It promotes the GTP-dependent translocation of the nascent protein chain from the A-site to the P-site of the ribosome. This protein is completely inactivated by EF-2 kinase phosporylation.","This gene encodes a GTPase which is a component of the spliceosome complex which processes precursor mRNAs to produce mature mRNAs. Mutations in this gene are associated with mandibulofacial dysostosis with microcephaly. Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a transcription factor with three tandem C2H2-type zinc fingers. Defects in this gene are associated with Charcot-Marie-Tooth disease type 1D (CMT1D), Charcot-Marie-Tooth disease type 4E (CMT4E), and with Dejerine-Sottas syndrome (DSS). Multiple transcript variants encoding two different isoforms have been found for this gene.","This gene encodes a transcriptional regulator that belongs to the EGR family of C2H2-type zinc-finger proteins. It is an immediate-early growth response gene which is induced by mitogenic stimulation. The protein encoded by this gene participates in the transcriptional regulation of genes in controling biological rhythm. It may also play a role in a wide variety of processes including muscle development, lymphocyte development, endothelial cell growth and migration, and neuronal development. Alternative splicing results in multiple transcript variants encoding distinct isoforms.[provided by RefSeq, Dec 2010].","Enables ATP hydrolysis activity. Involved in negative regulation of RNA-directed 5'-3' RNA polymerase activity. Located in perinuclear region of cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a protein that is one of the two components of elastic fibers. Elastic fibers comprise part of the extracellular matrix and confer elasticity to organs and tissues including the heart, skin, lungs, ligaments, and blood vessels. The encoded protein is rich in hydrophobic amino acids such as glycine and proline, which form mobile hydrophobic regions bounded by crosslinks between lysine residues. Degradation products of the encoded protein, known as elastin-derived peptides or elastokines, bind the elastin receptor complex and other receptors and stimulate migration and proliferation of monocytes and skin fibroblasts. Elastokines can also contribute to cancer progression. Deletions and mutations in this gene are associated with supravalvular aortic stenosis (SVAS) and autosomal dominant cutis laxa.","This gene encodes the protein elongin C, which is a subunit of the transcription factor B (SIII) complex. The SIII complex is composed of elongins A/A2, B and C. It activates elongation by RNA polymerase II by suppressing transient pausing of the polymerase at many sites within transcription units. Elongin A functions as the transcriptionally active component of the SIII complex, whereas elongins B and C are regulatory subunits. Elongin A2 is specifically expressed in the testis, and capable of forming a stable complex with elongins B and C. The von Hippel-Lindau tumor suppressor protein binds to elongins B and C, and thereby inhibits transcription elongation. Multiple alternatively spliced transcript variants encoding two distinct isoforms have been identified.","The protein encoded by this gene forms a complex with ERCC1 and is involved in the 5' incision made during nucleotide excision repair. This complex is a structure specific DNA repair endonuclease that interacts with EME1. Defects in this gene are a cause of xeroderma pigmentosum complementation group F (XP-F), or xeroderma pigmentosum VI (XP6).[provided by RefSeq, Mar 2009].","This gene encodes a single-strand specific DNA endonuclease that makes the 3' incision in DNA excision repair following UV-induced damage. The protein may also function in other cellular processes, including RNA polymerase II transcription, and transcription-coupled DNA repair. Mutations in this gene cause xeroderma pigmentosum complementation group G (XP-G), which is also referred to as xeroderma pigmentosum VII (XP7), a skin disorder characterized by hypersensitivity to UV light and increased susceptibility for skin cancer development following UV exposure. Some patients also develop Cockayne syndrome, which is characterized by severe growth defects, cognitive disability, and cachexia. Read-through transcription exists between this gene and the neighboring upstream BIVM (basic, immunoglobulin-like variable motif containing) gene.","The protein encoded by this gene is a nuclear receptor that is most closely related to the estrogen receptor. This protein acts as a site-specific transcription factor and interacts with members of the PGC-1 family of transcription cofactors to regulate the expression of most genes involved in cellular energy production as well as in the process of mitochondrial biogenesis. A processed pseudogene of ESRRA is located on chromosome 13q12.1.","This gene encodes an ethanolamine kinase, which functions in the first committed step of the phosphatidylethanolamine synthesis pathway. This cytosolic enzyme is specific for ethanolamine and exhibits negligible kinase activity on choline. Alternative splicing results in multiple transcript variants encoding distinct isoforms.","This gene encodes an endoplasmic reticulum-resident type II transmembrane glycosyltransferase involved in the chain elongation step of heparan sulfate biosynthesis. Mutations in this gene cause the type I form of multiple exostoses.","This gene encodes one of two glycosyltransferases involved in the chain elongation step of heparan sulfate biosynthesis. Mutations in this gene cause the type II form of multiple exostoses. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.","Involved in Golgi organization. Located in Golgi apparatus and plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a product belonging to a family of proteins with unknown function. The coding sequence of this family member includes several tandemly repeated regions.","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group C.","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group D2. This protein is monoubiquinated in response to DNA damage, resulting in its localization to nuclear foci with other proteins (BRCA1 AND BRCA2) involved in homology-directed DNA repair. Alternative splicing results in multiple transcript variants.","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group E.","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group F.","The Fanconi anemia complementation group (FANC) currently includes FANCA, FANCB, FANCC, FANCD1 (also called BRCA2), FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ (also called BRIP1), FANCL, FANCM and FANCN (also called PALB2). The previously defined group FANCH is the same as FANCA. Fanconi anemia is a genetically heterogeneous recessive disorder characterized by cytogenetic instability, hypersensitivity to DNA crosslinking agents, increased chromosomal breakage, and defective DNA repair. The members of the Fanconi anemia complementation group do not share sequence similarity; they are related by their assembly into a common nuclear protein complex. This gene encodes the protein for complementation group G.","This gene is the second identified human homolog of the Drosophila fat gene, which encodes a tumor suppressor essential for controlling cell proliferation during Drosophila development. The gene product is a member of the cadherin superfamily, a group of integral membrane proteins characterized by the presence of cadherin-type repeats. In addition to containing 34 tandem cadherin-type repeats, the gene product has two epidermal growth factor (EGF)-like repeats and one laminin G domain. This protein most likely functions as a cell adhesion molecule, controlling cell proliferation and playing an important role in cerebellum development.","The protein encoded by this gene is a component of connective tissue microfibrils and may be involved in elastic fiber assembly. Mutations in this gene cause congenital contractural arachnodactyly.","The protein encoded by this gene removes 5' overhanging flaps in DNA repair and processes the 5' ends of Okazaki fragments in lagging strand DNA synthesis. Direct physical interaction between this protein and AP endonuclease 1 during long-patch base excision repair provides coordinated loading of the proteins onto the substrate, thus passing the substrate from one enzyme to another. The protein is a member of the XPG/RAD2 endonuclease family and is one of ten proteins essential for cell-free DNA replication. DNA secondary structure can inhibit flap processing at certain trinucleotide repeats in a length-dependent manner by concealing the 5' end of the flap that is necessary for both binding and cleavage by the protein encoded by this gene. Therefore, secondary structure can deter the protective function of this protein, leading to site-specific trinucleotide expansions.","This gene encodes the human cellular counterpart of a feline sarcoma retrovirus protein with transforming capabilities. The gene product has tyrosine-specific protein kinase activity and that activity is required for maintenance of cellular transformation. Its chromosomal location has linked it to a specific translocation event identified in patients with acute promyelocytic leukemia but it is also involved in normal hematopoiesis as well as growth factor and cytokine receptor signaling. Alternative splicing results in multiple variants encoding different isoforms.[provided by RefSeq, Jan 2009].","This gene belongs to the ETS transcription factor family. ETS family members have a highly conserved 85-amino acid ETS domain that binds purine-rich DNA sequences. The alanine-rich C-terminus of this gene indicates that it may act as a transcription repressor. This gene is exclusively expressed in neurons of the central serotonin (5-HT) system, a system implicated in the pathogeny of such psychiatric diseases as depression, anxiety, and eating disorders. In some types of Ewing tumors, this gene is fused to the Ewing sarcoma (EWS) gene following chromosome translocations.","This gene encodes a protein that contains Dbl (DH) and pleckstrin (PH) homology domains and is similar to the Rho family of small GTP-binding proteins. The encoded protein specifically binds to the Rho family GTPase Cdc42Hs and can stimulate the GDP-GTP exchange of the isoprenylated form of Cdc42Hs. It also stimulates the mitogen activated protein kinase cascade leading to c-Jun kinase SAPK/JNK1 activation. Defects in this gene are the cause of the faciogenital dysplasia in Aarskog-Scott syndrome and a syndromatic form of X-linked cognitive disability.","Predicted to enable guanyl-nucleotide exchange factor activity and small GTPase binding activity. Predicted to be involved in several processes, including filopodium assembly; regulation of GTPase activity; and regulation of cell shape. Located in plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is an enzymatic component of the tricarboxylic acid (TCA) cycle, or Krebs cycle, and catalyzes the formation of L-malate from fumarate. It exists in both a cytosolic form and an N-terminal extended form, differing only in the translation start site used. The N-terminal extended form is targeted to the mitochondrion, where the removal of the extension generates the same form as in the cytoplasm. It is similar to some thermostable class II fumarases and functions as a homotetramer. Mutations in this gene can cause fumarase deficiency and lead to progressive encephalopathy.","Predicted to enable calcium ion binding activity. Predicted to be involved in protein folding. Predicted to be located in endoplasmic reticulum. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is an intermediate filament-associated protein that aggregates keratin intermediate filaments in mammalian epidermis. It is initially synthesized as a polyprotein precursor, profilaggrin (consisting of multiple filaggrin units of 324 aa each), which is localized in keratohyalin granules, and is subsequently proteolytically processed into individual functional filaggrin molecules. Mutations in this gene are associated with ichthyosis vulgaris.[provided by RefSeq, Dec 2009].","This gene encodes a transcription factor containing an ETS DNA-binding domain. The gene can undergo a t(11;22)(q24;q12) translocation with the Ewing sarcoma gene on chromosome 22, which results in a fusion gene that is present in the majority of Ewing sarcoma cases. An acute lymphoblastic leukemia-associated t(4;11)(q21;q23) translocation involving this gene has also been identified. Alternative splicing results in multiple transcript variants.","This gene is a member of the formin homology protein family. The encoded protein is thought to have essential roles in organization of the actin cytoskeleton and in cell polarity. This protein mediates the formation of an actin mesh that positions the spindle during oogenesis and also regulates the formation of actin filaments in the nucleus. This protein also forms a perinuclear actin/focal-adhesion system that regulates the shape and position of the nucleus during cell migration. Mutations in this gene have been associated with infertility and also with an autosomal recessive form of intellectual disability (MRT47). Alternatively spliced transcript variants have been identified.","Enables RNA binding activity. Predicted to act upstream of or within several processes, including negative regulation of osteoblast differentiation; substrate adhesion-dependent cell spreading; and type II pneumocyte differentiation. Predicted to be located in endoplasmic reticulum. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the O class of winged helix/forkhead transcription factor family. Proteins encoded by this class are regulated by factors involved in growth and differentiation indicating they play a role in these processes. A translocation involving this gene on chromosome X and the homolog of the Drosophila trithorax gene, encoding a DNA binding protein, located on chromosome 11 is associated with leukemia. Multiple transcript variants encoding different isoforms have been found for this gene.","FOXQ1 is a member of the FOX gene family, which is characterized by a conserved 110-amino acid DNA-binding motif called the forkhead or winged helix domain. FOX genes are involved in embryonic development, cell cycle regulation, tissue-specific gene expression, cell signaling, and tumorigenesis (Bieller et al., 2001).","This gene encodes a member of the forkhead box (FOX) family of transcription factors. FOX family members are monomeric, helix-turn-helix proteins with a core DNA-binding domain of approximately 110 aa. Many FOX transcription factors play roles in determining cell fates during early development. This forkhead box protein lacks the C-terminal basic region found in many other FOX family members. It is located within the 11q23.3 region which is commonly deleted in neuroblastomas.","Follistatin-like 3 is a secreted glycoprotein of the follistatin-module-protein family. It may have a role in leukemogenesis.","GABA is the major inhibitory neurotransmitter in the mammalian brain where it acts at GABA-A receptors, which are ligand-gated chloride channels. Chloride conductance of these channels can be modulated by agents such as benzodiazepines that bind to the GABA-A receptor. At least 16 distinct subunits of GABA-A receptors have been identified.","This gene encodes a protein which belongs to the GATA family of transcription factors. The protein plays an important role in erythroid development by regulating the switch of fetal hemoglobin to adult hemoglobin. Mutations in this gene have been associated with X-linked dyserythropoietic anemia and thrombocytopenia.","In the de novo synthesis of purine nucleotides, IMP is the branch point metabolite at which point the pathway diverges to the synthesis of either guanine or adenine nucleotides. In the guanine nucleotide pathway, there are 2 enzymes involved in converting IMP to GMP, namely IMP dehydrogenase (IMPD1), which catalyzes the oxidation of IMP to XMP, and GMP synthetase, which catalyzes the amination of XMP to GMP.","The protein encoded by this gene is an alpha subunit of guanine nucleotide binding proteins (G proteins). The encoded protein contains the guanine nucleotide binding site and is involved in the hormonal regulation of adenylate cyclase. Several transcript variants encoding different isoforms have been found for this gene.","Posttranslational glycosylphosphatidylinositol (GPI) anchor attachment serves as a general mechanism for linking proteins to the cell surface membrane. The protein encoded by this gene presumably functions in GPI anchoring at the GPI transfer step. The mRNA transcript is ubiquitously expressed in both fetal and adult tissues. The anchor attachment protein 1 contains an N-terminal signal sequence, 1 cAMP- and cGMP-dependent protein kinase phosphorylation site, 1 leucine zipper pattern, 2 potential N-glycosylation sites, and 8 putative transmembrane domains.","Cell surface heparan sulfate proteoglycans are composed of a membrane-associated protein core substituted with a variable number of heparan sulfate chains. Members of the glypican-related integral membrane proteoglycan family (GRIPS) contain a core protein anchored to the cytoplasmic membrane via a glycosyl phosphatidylinositol linkage. These proteins may play a role in the control of cell division and growth regulation. The protein encoded by this gene can bind to and inhibit the dipeptidyl peptidase activity of CD26, and it can induce apoptosis in certain cell types. Deletion mutations in this gene are associated with Simpson-Golabi-Behmel syndrome, also known as Simpson dysmorphia syndrome. Alternative splicing results in multiple transcript variants.","This gene encodes a protein involved in G protein-mitogen-activated protein kinase (MAPK) signaling cascades. When overexpressed in mammalian cells, this gene could potently suppress a RAS- and MAPK-mediated signal and interfere with JNK activity, suggesting that the function of this gene may be signal repression. The encoded protein is an integral subunit of the NCOR1-HDAC3 (nuclear receptor corepressor 1-histone deacetylase 3) complex, and it was shown that the complex inhibits JNK activation through this subunit and thus could potentially provide an alternative mechanism for hormone-mediated antagonism of AP1 (activator protein 1) function.","N-methyl-D-aspartate (NMDA) receptors are a class of ionotropic glutamate receptors. NMDA channel has been shown to be involved in long-term potentiation, an activity-dependent increase in the efficiency of synaptic transmission thought to underlie certain kinds of memory and learning. NMDA receptor channels are heteromers composed of the key receptor subunit NMDAR1 (GRIN1) and 1 or more of the 4 NMDAR2 subunits: NMDAR2A (GRIN2A), NMDAR2B (GRIN2B), NMDAR2C (GRIN2C), and NMDAR2D (GRIN2D).","The protein encoded by this gene is a serine-threonine kinase belonging to the glycogen synthase kinase subfamily. It is a negative regulator of glucose homeostasis and is involved in energy metabolism, inflammation, ER-stress, mitochondrial dysfunction, and apoptotic pathways. Defects in this gene have been associated with Parkinson disease and Alzheimer disease.","Histones are basic nuclear proteins responsible for nucleosome structure of the chromosomal fiber in eukaryotes. Two molecules of each of the four core histones (H2A, H2B, H3, and H4) form an octamer, around which approximately 146 bp of DNA is wrapped in repeating units, called nucleosomes. The linker histone, H1, interacts with linker DNA between nucleosomes and functions in the compaction of chromatin into higher order structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H1 family. Transcripts from this gene lack polyA tails but instead contain a palindromic termination element. This gene is found in the large histone gene cluster on chromosome 6.","Histones are basic nuclear proteins responsible for nucleosome structure of the chromosomal fiber in eukaryotes. Two molecules of each of the four core histones (H2A, H2B, H3, and H4) form an octamer, around which approximately 146 bp of DNA is wrapped in repeating units, called nucleosomes. The linker histone, H1, interacts with linker DNA between nucleosomes and functions in the compaction of chromatin into higher order structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H1 family. Transcripts from this gene lack polyA tails but instead contain a palindromic termination element. This gene is found in the large histone gene cluster on chromosome 6.","Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Two molecules of each of the four core histones (H2A, H2B, H3, and H4) form an octamer, around which approximately 146 bp of DNA is wrapped in repeating units, called nucleosomes. The linker histone, H1, interacts with linker DNA between nucleosomes and functions in the compaction of chromatin into higher order structures. This gene contains introns and its mRNA is polyadenylated, unlike most histone genes. The protein encoded by this gene is a replication-independent histone that is a member of the histone H3 family. Pseudogenes of this gene have been identified on the X chromosome, and on chromosomes 5, 13 and 17.","Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Nucleosomes consist of approximately 146 bp of DNA wrapped around a histone octamer composed of pairs of each of the four core histones (H2A, H2B, H3, and H4). The chromatin fiber is further compacted through the interaction of a linker histone, H1, with the DNA between the nucleosomes to form higher order chromatin structures. This gene contains introns and its mRNA is polyadenylated, unlike most histone genes. The protein encoded by this gene is a replication-independent histone that is a member of the histone H3 family.","Histones are basic nuclear proteins that are responsible for the nucleosome structure of the chromosomal fiber in eukaryotes. Two molecules of each of the four core histones (H2A, H2B, H3, and H4) form an octamer, around which approximately 146 bp of DNA is wrapped in repeating units, called nucleosomes. The linker histone, H1, interacts with linker DNA between nucleosomes and functions in the compaction of chromatin into higher order structures. This gene is intronless and encodes a replication-dependent histone that is a member of the histone H4 family. Transcripts from this gene lack polyA tails but instead contain a palindromic termination element. This gene is found in the histone microcluster on chromosome 6p21.33.","Predicted to enable GTP binding activity and purinergic nucleotide receptor activity. Predicted to be involved in G protein-coupled receptor signaling pathway. Located in cell junction. [provided by Alliance of Genome Resources, Apr 2022]","Histones play a critical role in transcriptional regulation, cell cycle progression, and developmental events. Histone acetylation/deacetylation alters chromosome structure and affects transcription factor access to DNA. The protein encoded by this gene belongs to class II of the histone deacetylase/acuc/apha family. It possesses histone deacetylase activity and represses transcription when tethered to a promoter. This protein does not bind DNA directly, but through transcription factors MEF2C and MEF2D. It seems to interact in a multiprotein complex with RbAp48 and HDAC3.","This gene encodes a nuclear protein belonging to the hairy and enhancer of split-related (HESR) family of basic helix-loop-helix (bHLH)-type transcriptional repressors. Expression of this gene is induced by the Notch and c-Jun signal transduction pathways. Two similar and redundant genes in mouse are required for embryonic cardiovascular development, and are also implicated in neurogenesis and somitogenesis. Alternative splicing results in multiple transcript variants.","HLA-C belongs to the HLA class I heavy chain paralogues. This class I molecule is a heterodimer consisting of a heavy chain and a light chain (beta-2 microglobulin). The heavy chain is anchored in the membrane. Class I molecules play a central role in the immune system by presenting peptides derived from endoplasmic reticulum lumen. They are expressed in nearly all cells. The heavy chain is approximately 45 kDa and its gene contains 8 exons. Exon one encodes the leader peptide, exons 2 and 3 encode the alpha1 and alpha2 domain, which both bind the peptide, exon 4 encodes the alpha3 domain, exon 5 encodes the transmembrane region, and exons 6 and 7 encode the cytoplasmic tail. Polymorphisms within exon 2 and exon 3 are responsible for the peptide binding specificity of each class one molecule. Typing for these polymorphisms is routinely done for bone marrow and kidney transplantation. About 6000 HLA-C alleles have been described. The HLA system plays an important role in the occurrence and outcome of infectious diseases, including those caused by the malaria parasite, the human immunodeficiency virus (HIV), and the severe acute respiratory syndrome coronavirus (SARS-CoV). The structural spike and the nucleocapsid proteins of the novel coronavirus SARS-CoV-2, which causes coronavirus disease 2019 (COVID-19), are reported to contain multiple Class I epitopes with predicted HLA restrictions. Individual HLA genetic variation may help explain different immune responses to a virus across a population.[provided by RefSeq, Aug 2020].","HLA-DQA1 belongs to the HLA class II alpha chain paralogues. The class II molecule is a heterodimer consisting of an alpha (DQA) and a beta chain (DQB), both anchored in the membrane. It plays a central role in the immune system by presenting peptides derived from extracellular proteins. Class II molecules are expressed in antigen presenting cells (APC: B Lymphocytes, dendritic cells, macrophages). The alpha chain is approximately 33-35 kDa. It is encoded by 5 exons; exon 1 encodes the leader peptide, exons 2 and 3 encode the two extracellular domains, and exon 4 encodes the transmembrane domain and the cytoplasmic tail. Within the DQ molecule both the alpha chain and the beta chain contain the polymorphisms specifying the peptide binding specificities, resulting in up to four different molecules. Typing for these polymorphisms is routinely done for bone marrow transplantation.","This gene encodes a member of the proline and acidic-rich (PAR) protein family, a subset of the bZIP transcription factors. The encoded protein forms homodimers or heterodimers with other PAR family members and binds sequence-specific promoter elements to activate transcription. Chromosomal translocations fusing portions of this gene with the E2A gene cause a subset of childhood B-lineage acute lymphoid leukemias. Alternatively spliced transcript variants have been described, but their biological validity has not been determined.","This gene encodes a chromatin-associated protein involved in the regulation of gene transcription, integration of retroviruses into chromosomes, and the metastatic progression of cancer cells. The encoded protein preferentially binds to the minor groove of AT-rich regions in double-stranded DNA. Multiple transcript variants encoding different isoforms have been found for this gene. Pseudogenes of this gene have been identified on multiple chromosomes.","This gene encodes a protein that belongs to the non-histone chromosomal high mobility group (HMG) protein family. HMG proteins function as architectural factors and are essential components of the enhancesome. This protein contains structural DNA-binding domains and may act as a transcriptional regulating factor. Identification of the deletion, amplification, and rearrangement of this gene that are associated with myxoid liposarcoma suggests a role in adipogenesis and mesenchymal differentiation. A gene knock out study of the mouse counterpart demonstrated that this gene is involved in diet-induced obesity. Alternate transcriptional splice variants, encoding different isoforms, have been characterized.","In vertebrates, the genes encoding the class of transcription factors called homeobox genes are found in clusters named A, B, C, and D on four separate chromosomes. Expression of these proteins is spatially and temporally regulated during embryonic development. This gene is part of the A cluster on chromosome 7 and encodes a DNA-binding transcription factor which may regulate gene expression, morphogenesis, and differentiation. This gene is involved in the regulation of uterine development and is required for female fertility. Mutations in this gene can cause radio-ulnar synostosis with amegakaryocytic thrombocytopenia.","In vertebrates, the genes encoding the class of transcription factors called homeobox genes are found in clusters named A, B, C, and D on four separate chromosomes. Expression of these proteins is spatially and temporally regulated during embryonic development. This gene is part of the A cluster on chromosome 7 and encodes a DNA-binding transcription factor which may regulate gene expression, morphogenesis, and differentiation. Expansion of a polyalanine tract in the encoded protein can cause hand-foot-uterus syndrome, also known as hand-foot-genital syndrome.","In vertebrates, the genes encoding the class of transcription factors called homeobox genes are found in clusters named A, B, C, and D on four separate chromosomes. Expression of these proteins is spatially and temporally regulated during embryonic development. This gene is part of the A cluster on chromosome 7 and encodes a DNA-binding transcription factor which may regulate gene expression, morphogenesis, and differentiation. This gene is highly similar to the abdominal-B (Abd-B) gene of Drosophila. A specific translocation event which causes a fusion between this gene and the NUP98 gene has been associated with myeloid leukemogenesis. Read-through transcription exists between this gene and the upstream homeobox A10 (HOXA10) gene.[provided by RefSeq, Mar 2011].","This gene belongs to the homeobox family of genes. The homeobox genes encode a highly conserved family of transcription factors that play an important role in morphogenesis in all multicellular organisms. Mammals possess four similar homeobox gene clusters, HOXA, HOXB, HOXC and HOXD, which are located on different chromosomes and consist of 9 to 11 genes arranged in tandem. This gene is one of several homeobox HOXC genes located in a cluster on chromosome 12. The product of this gene binds to a promoter element of the lactase-phlorizin hydrolase. It also may play a role in early intestinal development. An alternatively spliced variant encoding a shorter isoform has been described but its full-length nature has not been determined.","This gene belongs to the homeobox family of genes. The homeobox genes encode a highly conserved family of transcription factors that play an important role in morphogenesis in all multicellular organisms. Mammals possess four similar homeobox gene clusters, HOXA, HOXB, HOXC and HOXD, located on different chromosomes, consisting of 9 to 11 genes arranged in tandem. This gene is one of several homeobox HOXD genes located in a cluster on chromosome 2. Deletions that remove the entire HOXD gene cluster or the 5' end of this cluster have been associated with severe limb and genital abnormalities. The product of the mouse Hoxd11 gene plays a role in forelimb morphogenesis.","This gene belongs to the homeobox family of genes. The homeobox genes encode a highly conserved family of transcription factors that play an important role in morphogenesis in all multicellular organisms. Mammals possess four similar homeobox gene clusters, HOXA, HOXB, HOXC and HOXD, located on different chromosomes, consisting of 9 to 11 genes arranged in tandem. This gene is one of several homeobox HOXD genes located in a cluster on chromosome 2. Deletions that remove the entire HOXD gene cluster or the 5' end of this cluster have been associated with severe limb and genital abnormalities. Mutations in this particular gene cause synpolydactyly.","The protein encoded by this gene is an inducible molecular chaperone that functions as a homodimer. The encoded protein aids in the proper folding of specific target proteins by use of an ATPase activity that is modulated by co-chaperones. Two transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the heat shock protein 90 family; these proteins are involved in signal transduction, protein folding and degradation and morphological evolution. This gene encodes the constitutive form of the cytosolic 90 kDa heat-shock protein and is thought to play a role in gastric apoptosis and inflammation. Alternative splicing results in multiple transcript variants. Pseudogenes have been identified on multiple chromosomes.","This gene encodes the perlecan protein, which consists of a core protein to which three long chains of glycosaminoglycans (heparan sulfate or chondroitin sulfate) are attached. The perlecan protein is a large multidomain proteoglycan that binds to and cross-links many extracellular matrix components and cell-surface molecules. It has been shown that this protein interacts with laminin, prolargin, collagen type IV, FGFBP1, FBLN2, FGF7 and transthyretin, etc., and it plays essential roles in multiple biological activities. Perlecan is a key component of the vascular extracellular matrix, where it helps to maintain the endothelial barrier function. It is a potent inhibitor of smooth muscle cell proliferation and is thus thought to help maintain vascular homeostasis. It can also promote growth factor (e.g., FGF2) activity and thus stimulate endothelial growth and re-generation. It is a major component of basement membranes, where it is involved in the stabilization of other molecules as well as being involved with glomerular permeability to macromolecules and cell adhesion. Mutations in this gene cause Schwartz-Jampel syndrome type 1, Silverman-Handmaker type of dyssegmental dysplasia, and tardive dyskinesia. Alternative splicing of this gene results in multiple transcript variants.","This gene encodes a protein containing a C-terminal HECT (E6AP type E3 ubiquitin protein ligase) domain that functions as an E3 ubiquitin ligase. The encoded protein is required for the ubiquitination and subsequent degradation of the anti-apoptotic protein Mcl1 (myeloid cell leukemia sequence 1 (BCL2-related)). This protein also ubiquitinates the p53 tumor suppressor, core histones, and DNA polymerase beta. Mutations in this gene are associated with Turner type X-linked syndromic cognitive disability.","This gene encodes a protein that binds the 5' UTR of insulin-like growth factor 2 (IGF2) mRNA and regulates its translation. It plays an important role in metabolism and variation in this gene is associated with susceptibility to diabetes. Alternative splicing and promoter usage results in multiple transcript variants. Related pseudogenes are found on several chromosomes.","This gene encodes a member of the Ikaros family of zinc-finger proteins. Three members of this protein family (Ikaros, Aiolos and Helios) are hematopoietic-specific transcription factors involved in the regulation of lymphocyte development. This gene product is a transcription factor that is important in the regulation of B lymphocyte proliferation and differentiation. Both Ikaros and Aiolos can participate in chromatin remodeling. Regulation of gene expression in B lymphocytes by Aiolos is complex as it appears to require the sequential formation of Ikaros homodimers, Ikaros/Aiolos heterodimers, and Aiolos homodimers. Several alternative transcripts encoding different isoforms have been described, as well as some non-protein coding variants.","This gene encodes a tumor suppressor protein that can induce cell growth arrest and apoptosis. The encoded protein is a nuclear protein that physically interacts with the tumor suppressor protein TP53 and is a component of the p53 signaling pathway. Reduced expression and rearrangement of this gene have been detected in various cancers. Multiple alternatively spliced transcript variants encoding distinct isoforms have been reported.","This gene encodes the interleukin-1 receptor-associated kinase 1, one of two putative serine/threonine kinases that become associated with the interleukin-1 receptor (IL1R) upon stimulation. This gene is partially responsible for IL1-induced upregulation of the transcription factor NF-kappa B. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a transcriptional regulator and tumor suppressor, serving as an activator of genes involved in both innate and acquired immune responses. The encoded protein activates the transcription of genes involved in the body's response to viruses and bacteria, playing a role in cell proliferation, apoptosis, the immune response, and DNA damage response. This protein represses the transcription of several other genes. As a tumor suppressor, it both suppresses tumor cell growth and stimulates an immune response against tumor cells. Defects in this gene have been associated with gastric cancer, myelogenous leukemia, and lung cancer.","IRF2 encodes interferon regulatory factor 2, a member of the interferon regulatory transcription factor (IRF) family. IRF2 competitively inhibits the IRF1-mediated transcriptional activation of interferons alpha and beta, and presumably other genes that employ IRF1 for transcription activation. However, IRF2 also functions as a transcriptional activator of histone H4.","This gene encodes a member of the interferon regulatory transcription factor (IRF) family. Family members share a highly-conserved N-terminal helix-turn-helix DNA-binding domain and a less conserved C-terminal protein-binding domain. The encoded protein may be a transcriptional activator. Mutations in this gene can cause van der Woude syndrome and popliteal pterygium syndrome. Mutations in this gene are also associated with non-syndromic orofacial cleft type 6. Alternate splicing results in multiple transcript variants.[provided by RefSeq, May 2011].","Homeobox genes encode DNA-binding proteins, many of which are thought to be involved in early embryonic development. Homeobox genes encode a DNA-binding domain of 60 to 63 amino acids referred to as the homeodomain. This gene is a member of the RAXLX homeobox gene family.","This gene is the putative transforming gene of avian sarcoma virus 17. It encodes a protein which is highly similar to the viral protein, and which interacts directly with specific target DNA sequences to regulate gene expression. This gene is intronless and is mapped to 1p32-p31, a chromosomal region involved in both translocations and deletions in human malignancies.","This gene encodes a nuclear protein that is a subunit of two protein complexes involved with histone acetylation, the MLL1 complex and the NSL1 complex. The encoded protein has been implicated in a variety of cellular processes including enhancer regulation, cell proliferation, and mitosis. Mutations in this gene are associated with Koolen-de Vries Syndrome.","This gene encodes an integral membrane protein which belongs to one of seven subfamilies of inward-rectifier potassium channel proteins called potassium channel subfamily J. The encoded protein is a subunit of the potassium channel which is homotetrameric. It is controlled by G-proteins and has a greater tendency to allow potassium to flow into a cell rather than out of a cell. Naturally occurring mutations in this gene are associated with aldosterone-producing adenomas.","Predicted to enable chromatin DNA binding activity; histone H3-methyl-lysine-9 demethylase activity; and transcription coregulator activity. Predicted to be involved in histone H3-K9 demethylation and regulation of transcription by RNA polymerase II. Located in nucleoplasm. Biomarker of acute lymphoblastic leukemia; breast cancer; colorectal cancer; and lung non-small cell carcinoma. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a type II transmembrane glycoprotein that is the highly polymorphic Kell blood group antigen. The Kell glycoprotein links via a single disulfide bond to the XK membrane protein that carries the Kx antigen. The encoded protein contains sequence and structural similarity to members of the neprilysin (M13) family of zinc endopeptidases.","The protein encoded by this gene is a member of the kinesin family and functions as an anterograde motor protein that transports membranous organelles along axonal microtubules. Mutations at this locus have been associated with spastic paraplegia-30 and hereditary sensory neuropathy IIC. Alternatively spliced transcript variants encoding distinct isoforms have been described.","Enables identical protein binding activity; microtubule binding activity; and microtubule motor activity. Involved in several processes, including lysosome localization; natural killer cell mediated cytotoxicity; and positive regulation of protein localization to plasma membrane. Located in centriolar satellite; cytosol; and vesicle. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the kelch-like (KLHL) family of proteins, which is involved in B-lymphocyte antigen receptor signaling and germinal-center B-cell maturation. The encoded protein contains an N-terminal broad-complex, tramtrack and bric a brac (BTB) domain that facilitates protein binding and dimerization, a BTB and C-terminal kelch (BACK) domain, and six C-terminal kelch repeat domains. Naturally occurring mutations in this gene are associated with chronic lymphocytic leukemia. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a protein-lysine N-methyltransferase that can monomethylate Lys-20 of histone H4 to effect transcriptional repression of some genes. The encoded protein is required for cell proliferation and plays a role in chromatin condensation.","Predicted to enable structural molecule activity. Predicted to be located in intermediate filament. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of an evolutionarily conserved protein family implicated in RNA metabolism and translation. Members of this family are characterized by the presence of an La motif, which is often located adjacent to one or more RNA recognition motifs (RRM). Together, the two motifs constitute the functional region of the protein and enable its interaction with the RNA substrate. This protein family is divided into five sub-families: the genuine La proteins and four La-related protein (LARP) sub-families. The protein encoded by this gene belongs to LARP sub-family 4. It is a cytoplasmic protein that may play a stimulatory role in translation.","This gene is a member of the Src family of protein tyrosine kinases (PTKs). The encoded protein is a key signaling molecule in the selection and maturation of developing T-cells. It contains N-terminal sites for myristylation and palmitylation, a PTK domain, and SH2 and SH3 domains which are involved in mediating protein-protein interactions with phosphotyrosine-containing and proline-rich motifs, respectively. The protein localizes to the plasma membrane and pericentrosomal vesicles, and binds to cell surface receptors, including CD4 and CD8, and other signaling molecules. Multiple alternatively spliced variants encoding different isoforms have been described.","Enables LIM domain binding activity; RNA polymerase II-specific DNA-binding transcription factor binding activity; and enzyme binding activity. Involved in negative regulation of transcription, DNA-templated and positive regulation of transcription by RNA polymerase II. Located in chromatin. Part of beta-catenin-TCF complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a LEM domain-containing transmembrane protein of the inner nuclear membrane. The protein is involved in nuclear structure organization and plays a role in cell signaling and differentiation. Mutations in this gene result in Cataract 46, juvenile-onset. Multiple transcript variants have been found for this gene.","Enables identical protein binding activity. Predicted to be involved in late endosome to vacuole transport via multivesicular body sorting pathway and negative regulation of growth hormone receptor signaling pathway. Predicted to be integral component of membrane. Predicted to be active in endosome. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a protein that belongs to the type I cytokine receptor family. This protein combines with a high-affinity converter subunit, gp130, to form a receptor complex that mediates the action of the leukemia inhibitory factor, a polyfunctional cytokine that is involved in cellular differentiation, proliferation and survival in the adult and the embryo. Mutations in this gene cause Schwartz-Jampel syndrome type 2, a disease belonging to the group of the bent-bone dysplasias. A translocation that involves the promoter of this gene, t(5;8)(p13;q12) with the pleiomorphic adenoma gene 1, is associated with salivary gland pleiomorphic adenoma, a common type of benign epithelial tumor of the salivary gland. Multiple splice variants encoding two different isoforms have been found for this gene.","This locus encodes a transcriptional regulator that contains two cysteine-rich LIM domains but lacks a DNA-binding domain. LIM domains may play a role in protein interactions; thus the encoded protein may regulate transcription by competitively binding to specific DNA-binding transcription factors. Alterations at this locus have been associated with acute lymphoblastic T-cell leukemia. Chromosomal rearrangements have been observed between this locus and at least two loci, the delta subunit of the T-cell antigen receptor gene and the LIM domain binding 1 gene. Alternatively spliced transcript variants have been described.","LMO2 encodes a cysteine-rich, two LIM-domain protein that is required for yolk sac erythropoiesis. The LMO2 protein has a central and crucial role in hematopoietic development and is highly conserved. The LMO2 transcription start site is located approximately 25 kb downstream from the 11p13 T-cell translocation cluster (11p13 ttc), where a number T-cell acute lymphoblastic leukemia-specific translocations occur. Alternative splicing results in multiple transcript variants encoding different isoforms.[provided by RefSeq, Nov 2008].","The protein encoded by this gene is a serine proteinase that inhibits the activity of tissue-type plasminogen activator I. The encoded protein constitutes a substantial portion of lipoprotein(a) and is proteolytically cleaved, resulting in fragments that attach to atherosclerotic lesions and promote thrombogenesis. Elevated plasma levels of this protein are linked to atherosclerosis. Depending on the individual, the encoded protein contains 2-43 copies of kringle-type domains. The allele represented here contains 15 copies of the kringle-type repeats and corresponds to that found in the reference genome sequence.","This gene encodes a member of a subfamily of LIM domain proteins that are characterized by an N-terminal proline-rich region and three C-terminal LIM domains. The encoded protein localizes to the cell periphery in focal adhesions and may be involved in cell-cell adhesion and cell motility. This protein also shuttles through the nucleus and may function as a transcriptional co-activator. This gene is located at the junction of certain disease-related chromosomal translocations, which result in the expression of chimeric proteins that may promote tumor growth. Alternative splicing results in multiple transcript variants.","Sm-like proteins were identified in a variety of organisms based on sequence homology with the Sm protein family (see SNRPD2; 601061). Sm-like proteins contain the Sm sequence motif, which consists of 2 regions separated by a linker of variable length that folds as a loop. The Sm-like proteins are thought to form a stable heteromer present in tri-snRNP particles, which are important for pre-mRNA splicing.","This gene represents a basic helix-loop-helix transcription factor. The encoded protein may play roles in blood vessel maturation and hematopoeisis. A translocation between this locus and the T cell receptor beta locus (GeneID 6957) on chromosome 7 has been associated with acute lymphoblastic leukemia.","This gene encodes a tyrosine protein kinase, which maybe involved in the regulation of mast cell degranulation, and erythroid differentiation. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","This gene encodes a large protein containing numerous spectrin and leucine-rich repeat (LRR) domains. The encoded protein is a member of a family of proteins that form bridges between different cytoskeletal elements. This protein facilitates actin-microtubule interactions at the cell periphery and couples the microtubule network to cellular junctions. Alternative splicing results in multiple transcript variants, but the full-length nature of some of these variants has not been determined.","The protein encoded by this gene is a basic leucine zipper (bZIP) transcription factor that plays an important role in the regulation of lineage-specific hematopoiesis. The encoded nuclear protein represses ETS1-mediated transcription of erythroid-specific genes in myeloid cells. This gene contains no introns.","The protein encoded by this gene is a dual specificity protein kinase that belongs to the MAP kinase kinase family. This kinase is known to play a critical role in mitogen growth factor signal transduction. It phosphorylates and thus activates MAPK1/ERK2 and MAPK2/ERK3. The activation of this kinase itself is dependent on the Ser/Thr phosphorylation by MAP kinase kinase kinases. Mutations in this gene cause cardiofaciocutaneous syndrome (CFC syndrome), a disease characterized by heart defects, cognitive disability, and distinctive facial features similar to those found in Noonan syndrome. The inhibition or degradation of this kinase is also found to be involved in the pathogenesis of Yersinia and anthrax. A pseudogene, which is located on chromosome 7, has been identified for this gene.","The protein encoded by this gene is a dual specificity protein kinase that belongs to the MAP kinase kinase family. This kinase specifically activates MAPK8/JNK1 and MAPK9/JNK2, and this kinase itself is phosphorylated and activated by MAP kinase kinase kinases including MAP3K1/MEKK1, MAP3K2/MEKK2,MAP3K3/MEKK5, and MAP4K2/GCK. This kinase is involved in the signal transduction mediating the cell responses to proinflammatory cytokines, and environmental stresses. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a member of serine/threonine protein kinase family. This kinase contains a dual leucine-zipper motif, and has been shown to form dimers/oligomers through its leucine-zipper motif. This kinase can phosphorylate and activate MAPK8/JNK, MAP2K7/MKK7, which suggests a role in the JNK signaling pathway.","The central core of each mitogen-activated protein kinase (MAPK) pathway is a conserved cascade of 3 protein kinases: an activated MAPK kinase kinase (MAPKKK) phosphorylates and activates a specific MAPK kinase (MAPKK), which then activates a specific MAPK. While the ERK MAPKs are activated by mitogenic stimulation, the CSBP2 and JNK MAPKs are activated by environmental stresses such as osmotic shock, UV irradiation, wound stress, and inflammatory factors. This gene encodes a MAPKKK, the MEKK4 protein, also called MTK1. This protein contains a protein kinase catalytic domain at the C terminus. The N-terminal nonkinase domain may contain a regulatory domain. Expression of MEKK4 in mammalian cells activated the CSBP2 and JNK MAPK pathways, but not the ERK pathway. In vitro kinase studies indicated that recombinant MEKK4 can specifically phosphorylate and activate PRKMK6 and SERK1, MAPKKs that activate CSBP2 and JNK, respectively but cannot phosphorylate PRKMK1, an MAPKK that activates ERKs. MEKK4 is a major mediator of environmental stresses that activate the CSBP2 MAPK pathway, and a minor mediator of the JNK pathway. Several alternatively spliced transcripts encoding distinct isoforms have been described.","This gene encodes a member of the Par-1 family of serine/threonine protein kinases. The protein is an important regulator of cell polarity in epithelial and neuronal cells, and also controls the stability of microtubules through phosphorylation and inactivation of several microtubule-associating proteins. The protein localizes to cell membranes. Multiple transcript variants encoding different isoforms have been found for this gene.","Enables cadherin binding activity. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is one of the highly conserved mini-chromosome maintenance proteins (MCM) that are essential for the initiation of eukaryotic genome replication. The hexameric protein complex formed by the MCM proteins is a key component of the pre-replication complex (pre_RC) and may be involved in the formation of replication forks and in the recruitment of other DNA replication related proteins. The MCM complex consisting of this protein and MCM2, 4 and 6 proteins possesses DNA helicase activity, and may act as a DNA unwinding enzyme. Cyclin D1-dependent kinase, CDK4, is found to associate with this protein, and may regulate the binding of this protein with the tumorsuppressor protein RB1/RB. Alternatively spliced transcript variants encoding distinct isoforms have been reported.","Predicted to be involved in regulation of presynapse assembly; regulation of synaptic membrane adhesion; and spinal cord motor neuron differentiation. Predicted to act upstream of or within neuron migration and pattern specification process. Predicted to be located in extracellular region and plasma membrane. Predicted to be active in GABA-ergic synapse and glutamatergic synapse. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a nuclear protein that contains a p53 binding domain at the N-terminus and a RING finger domain at the C-terminus, and shows structural similarity to p53-binding protein MDM2. Both proteins bind the p53 tumor suppressor protein and inhibit its activity, and have been shown to be overexpressed in a variety of human cancers. However, unlike MDM2 which degrades p53, this protein inhibits p53 by binding its transcriptional activation domain. This protein also interacts with MDM2 protein via the RING finger domain, and inhibits the latter's degradation. So this protein can reverse MDM2-targeted degradation of p53, while maintaining suppression of p53 transactivation and apoptotic functions. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.","The activation of gene transcription is a multistep process that is triggered by factors that recognize transcriptional enhancer sites in DNA. These factors work with co-activators to direct transcriptional initiation by the RNA polymerase II apparatus. The protein encoded by this gene is a subunit of the CRSP (cofactor required for SP1 activation) complex, which, along with TFIID, is required for efficient activation by SP1. This protein is also a component of other multisubunit complexes e.g. thyroid hormone receptor-(TR-) associated proteins which interact with TR and facilitate TR function on DNA templates in conjunction with initiation factors and cofactors. This protein also acts as a metastasis suppressor. Several alternatively spliced transcript variants encoding different isoforms have been described for this gene.","The product of this gene is a member of the MADS/MEF2 family of DNA binding proteins. The protein is thought to regulate gene expression, including expression of the smooth muscle myosin heavy chain gene. This region undergoes considerable alternative splicing, with transcripts supporting two non-overlapping loci (GeneID 729991 and 100271849) as well as numerous read-through transcripts that span both loci (annotated as GeneID 4207). Several isoforms of this protein are expressed from either this locus or from some of the read-through transcripts annotated on GeneID 4207.","The protein encoded by this gene is an E3 ubiquitin protein ligase that mediates ubiquitination of proteins in the Notch signaling pathway. The encoded protein may be a suppressor of melanoma invasion.","The protein encoded by this gene is a transcription factor that contains both basic helix-loop-helix and leucine zipper structural features. The encoded protein regulates melanocyte development and is responsible for pigment cell-specific transcription of the melanogenesis enzyme genes. Heterozygous mutations in the this gene cause auditory-pigmentary syndromes, such as Waardenburg syndrome type 2 and Tietz syndrome.","This gene encodes an oncoprotein which is thought to play a role in the phenotypic determination of hemopoetic cells. Translocations between this gene and nucleophosmin have been associated with myelodysplastic syndrome and acute myeloid leukemia. Multiple transcript variants encoding different isoforms have been found for this gene.","Predicted to be involved in regulation of transcription, DNA-templated. Predicted to act upstream of or within negative regulation of protein kinase activity. Located in cytosol; fibrillar center; and nucleoplasm. Part of transcription elongation factor complex. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a transcription factor and has been identified as a partner gene involved in several chromosomal rearrangements resulting in various leukemias. Multiple transcript variants encoding different isoforms have been found for this gene.","Enables chromatin binding activity and lysine-acetylated histone binding activity. Involved in several processes, including hematopoietic stem cell differentiation; positive regulation of transcription, DNA-templated; and regulation of stem cell division. Acts upstream of or within negative regulation of canonical Wnt signaling pathway and positive regulation of Wnt signaling pathway, planar cell polarity pathway. Located in cytosol and nucleoplasm. Part of transcription elongation factor complex. [provided by Alliance of Genome Resources, Apr 2022]","Enables histone binding activity and nucleosome binding activity. Predicted to be involved in positive regulation of transcription by RNA polymerase II. Predicted to act upstream of or within several processes, including negative regulation of histone H3-K79 methylation; renal potassium excretion; and renal sodium excretion. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Meningioma 1 (MN1) contains two sets of CAG repeats. It is disrupted by a balanced translocation (4;22) in a meningioma, and its inactivation may contribute to meningioma 32 pathogenesis.","The protein encoded by this gene interacts with the transcription factor myocardin, a key regulator of smooth muscle cell differentiation. The encoded protein is predominantly nuclear and may help transduce signals from the cytoskeleton to the nucleus. This gene is involved in a specific translocation event that creates a fusion of this gene and the RNA-binding motif protein-15 gene. This translocation has been associated with acute megakaryocytic leukemia. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene forms a heterodimer with MSH2 to form MutS beta, part of the post-replicative DNA mismatch repair system. MutS beta initiates mismatch repair by binding to a mismatch and then forming a complex with MutL alpha heterodimer. This gene contains a polymorphic 9 bp tandem repeat sequence in the first exon. The repeat is present 6 times in the reference genome sequence and 3-7 repeats have been reported. Defects in this gene are a cause of susceptibility to endometrial cancer.","Moesin (for membrane-organizing extension spike protein) is a member of the ERM family which includes ezrin and radixin. ERM proteins appear to function as cross-linkers between plasma membranes and actin-based cytoskeletons. Moesin is localized to filopodia and other membranous protrusions that are important for cell-cell recognition and signaling and for cell movement.","This gene was identified by involvement in some t(X;14) translocations associated with mature T-cell proliferations. This region has a complex gene structure, with a common promoter and 5' exon spliced to two different sets of 3' exons that encode two different proteins. This gene represents the upstream 13 kDa protein that is a member of the TCL1 family. This protein may be involved in leukemogenesis.","This gene encodes a DNA glycosylase involved in oxidative DNA damage repair. The enzyme excises adenine bases from the DNA backbone at sites where adenine is inappropriately paired with guanine, cytosine, or 8-oxo-7,8-dihydroguanine, a major oxidatively damaged DNA lesion. The protein is localized to the nucleus and mitochondria. This gene product is thought to play a role in signaling apoptosis by the introduction of single-strand breaks following oxidative damage. Mutations in this gene result in heritable predisposition to colorectal cancer, termed MUTYH-associated polyposis (MAP). Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a protein with three HTH DNA-binding domains that functions as a transcription regulator. This protein plays an essential role in the regulation of hematopoiesis. This gene may be aberrently expressed or rearranged or undergo translocation in leukemias and lymphomas, and is considered to be an oncogene. Alternative splicing results in multiple transcript variants.","Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to act upstream of or within regulation of inner ear auditory receptor cell differentiation. Located in chromosome and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]",null,"The protein encoded by this gene is a smooth muscle myosin belonging to the myosin heavy chain family. The gene product is a subunit of a hexameric protein that consists of two heavy chain subunits and two pairs of non-identical light chain subunits. It functions as a major contractile protein, converting chemical energy into mechanical energy through the hydrolysis of ATP. A chromosomal rearrangement involving this gene is associated with acute myeloid leukemia of the M4Eo subtype. Mutations in this gene are associated with visceral myopathy, megacystis-microcolon-intestinal hypoperistalsis syndrome 2, and familial thoracic aortic aneurysm 4.","This gene is one of three myosin V heavy-chain genes, belonging to the myosin gene superfamily. Myosin V is a class of actin-based motor proteins involved in cytoplasmic vesicle transport and anchorage, spindle-pole alignment and mRNA translocation. The protein encoded by this gene is abundant in melanocytes and nerve cells. Mutations in this gene cause Griscelli syndrome type-1 (GS1) and neuroectodermal melanolysosomal disease, or Elejalde disease.","This gene encodes a nuclear protein that belongs to the basic helix-loop-helix family of transcription factors and the myogenic factors subfamily. It regulates muscle cell differentiation by inducing cell cycle arrest, a prerequisite for myogenic initiation. The protein is also involved in muscle regeneration. It activates its own transcription which may stabilize commitment to myogenesis.","This gene encodes a member of the family of NGFI-A binding (NAB) proteins, which function in the nucleus to repress transcription induced by some members of the EGR (early growth response) family of transactivators. NAB proteins can homo- or hetero-multimerize with other EGR or NAB proteins through a conserved N-terminal domain, and repress transcription through two partially redundant C-terminal domains. Transcriptional repression by the encoded protein is mediated in part by interactions with the nucleosome remodeling and deactylase (NuRD) complex. Alternatively spliced transcript variants have been described, but their biological validity has not been determined.","This gene encodes a protein that associates with basic transcription factor 3 (BTF3) to form the nascent polypeptide-associated complex (NAC). This complex binds to nascent proteins that lack a signal peptide motif as they emerge from the ribosome, blocking interaction with the signal recognition particle (SRP) and preventing mistranslocation to the endoplasmic reticulum. This protein is an IgE autoantigen in atopic dermatitis patients. Alternative splicing results in multiple transcript variants, but the full length nature of some of these variants, including those encoding very large proteins, has not been determined. There are multiple pseudogenes of this gene on different chromosomes.","Mutations in this gene are associated with Nijmegen breakage syndrome, an autosomal recessive chromosomal instability syndrome characterized by microcephaly, growth retardation, immunodeficiency, and cancer predisposition. The encoded protein is a member of the MRE11/RAD50 double-strand break repair complex which consists of 5 proteins. This gene product is thought to be involved in DNA double-strand break repair and DNA damage-induced checkpoint activation.","This gene encodes a member of the barr gene family and a regulatory subunit of the condensin complex. This complex is required for the conversion of interphase chromatin into condensed chromosomes. The protein encoded by this gene is associated with mitotic chromosomes, except during the early phase of chromosome condensation. During interphase, the protein has a distinct punctate nucleolar localization. Alternatively spliced transcript variants encoding different proteins have been described.","The protein encoded by this gene contains a nuclear localization signal. It plays a role in signal transduction, and may function in the maintenance of sarcomeres and in the assembly of myofibrils into sarcomeres. It also plays an important role in stress fiber formation This protein is involved in the formation and maintenance of dendritic spines, and modulates synaptic activity in neurons. The gene is involved in therapy-related leukemia by a chromosomal translocation t(3;11)(p21;q23) that involves this gene and the myeloid/lymphoid leukemia gene. Alternative splicing results in multiple transcript variants of this gene.","The protein encoded by this gene acts as a transcriptional coactivator for steroid and nuclear hormone receptors. It is a member of the p160/steroid receptor coactivator (SRC) family and like other family members has histone acetyltransferase activity and contains a nuclear localization signal, as well as bHLH and PAS domains. The product of this gene binds nuclear receptors directly and stimulates the transcriptional activities in a hormone-dependent fashion. Alternatively spliced transcript variants encoding different isoforms have been identified.","This gene encodes an androgen receptor coactivator. The encoded protein interacts with the androgen receptor in a ligand-dependent manner to enhance its transcriptional activity. Chromosomal translocations between this gene and the ret tyrosine kinase gene, also located on chromosome 10, have been associated with papillary thyroid carcinoma. Alternatively spliced transcript variants have been described. Pseudogenes are present on chromosomes 4, 5, 10, and 14.","This gene is a member of the N-myc downregulated gene family which belongs to the alpha/beta hydrolase superfamily. The protein encoded by this gene is a cytoplasmic protein involved in stress responses, hormone responses, cell growth, and differentiation. The encoded protein is necessary for p53-mediated caspase activation and apoptosis. Mutations in this gene are a cause of Charcot-Marie-Tooth disease type 4D, and expression of this gene may be a prognostic indicator for several types of cancer. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","This gene encodes a member of the NF-kappa-B inhibitor family, which contain multiple ankrin repeat domains. The encoded protein interacts with REL dimers to inhibit NF-kappa-B/REL complexes which are involved in inflammatory responses. The encoded protein moves between the cytoplasm and the nucleus via a nuclear localization signal and CRM1-mediated nuclear export. Mutations in this gene have been found in ectodermal dysplasia anhidrotic with T-cell immunodeficiency autosomal dominant disease.","This gene encodes one of the proteins important for centrosomal function. This protein is important for positioning and anchoring the microtubules minus-ends in epithelial cells. Localization of this protein to the centrosome requires three leucine zippers in the central coiled-coil domain. Multiple alternatively spliced transcript variants that encode different isoforms have been reported.","This gene encodes a membrane-anchored protein with a hydrophobic amino terminal domain and a cyclophilin-like PPIase domain. It is present on the surface of natural killer cells and facilitates their binding to targets. Its expression is regulated by IL2 activation of the cells.","Enables protein kinase activity. Contributes to GTPase activator activity. Involved in cellular response to amino acid starvation; negative regulation of TOR signaling; and negative regulation of kinase activity. Located in lysosomal membrane. Part of GATOR1 complex. Implicated in focal epilepsy. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the steroid-thyroid hormone-retinoid receptor superfamily. The encoded protein may act as a transcriptional activator. The protein can efficiently bind the NGFI-B Response Element (NBRE). Three different versions of extraskeletal myxoid chondrosarcomas (EMCs) are the result of reciprocal translocations between this gene and other genes. The translocation breakpoints are associated with Nuclear Receptor Subfamily 4, Group A, Member 3 (on chromosome 9) and either Ewing Sarcome Breakpoint Region 1 (on chromosome 22), RNA Polymerase II, TATA Box-Binding Protein-Associated Factor, 68-KD (on chromosome 17), or Transcription factor 12 (on chromosome 15). Multiple transcript variants encoding different isoforms have been found for this gene.","The mouse ortholog of this gene encodes a protein kinase required for JNK activation. The encoded protein may be involved in the induction of actin polymerization in late embryogenesis.[provided by RefSeq, Jun 2010].","This gene encodes a hydrolase that serves as an important role in cellular purine metabolism by acting primarily on inosine 5'-monophosphate and other purine nucleotides.","This gene encodes a member of the 5'-nucleotidase family of enzymes that catalyze the dephosphorylation of nucleoside 5'-monophosphates. The encoded protein is the type 1 isozyme of pyrimidine 5' nucleotidase and catalyzes the dephosphorylation of pyrimidine 5' monophosphates. Mutations in this gene are a cause of hemolytic anemia due to uridine 5-prime monophosphate hydrolase deficiency. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene, and pseudogenes of this gene are located on the long arm of chromosomes 3 and 4.","The protein encoded by this gene is a DNA N-glycosylase of the endonuclease III family. Like a similar protein in E. coli, the encoded protein has DNA glycosylase activity on DNA substrates containing oxidized pyrimidine residues and has apurinic/apyrimidinic lyase activity.","This gene encodes a member of the neurotrophic tyrosine receptor kinase (NTRK) family. This kinase is a membrane-bound receptor that, upon neurotrophin binding, phosphorylates itself and members of the MAPK pathway. Signalling through this kinase leads to cell differentiation. Mutations in this gene have been associated with obesity and mood disorders. Alternative splicing results in multiple transcript variants.","This gene encodes a large protein that forms a structural component of the nuclear matrix. The encoded protein interacts with microtubules and plays a role in the formation and organization of the mitotic spindle during cell division. Chromosomal translocation of this gene with the RARA (retinoic acid receptor, alpha) gene on chromosome 17 have been detected in patients with acute promyelocytic leukemia. Alternative splicing results in multiple transcript variants.","The nuclear envelope creates distinct nuclear and cytoplasmic compartments in eukaryotic cells. It consists of two concentric membranes perforated by nuclear pores, large protein complexes that form aqueous channels to regulate the flow of macromolecules between the nucleus and the cytoplasm. These complexes are composed of at least 100 different polypeptide subunits, many of which belong to the nucleoporin family. The nucleoporin protein encoded by this gene displays evolutionarily conserved interactions with other nucleoporins. This protein, which localizes to both sides of the nuclear pore complex at interphase, remains associated with the complex during mitosis and is targeted at early stages to the reforming nuclear envelope. This protein also localizes to kinetochores of mitotic cells.","The nuclear pore complex is a massive structure that extends across the nuclear envelope, forming a gateway that regulates the flow of macromolecules between the nucleus and the cytoplasm. Nucleoporins are the main components of the nuclear pore complex in eukaryotic cells. This gene is a member of the FG-repeat-containing nucleoporins. The protein encoded by this gene is localized to the cytoplasmic face of the nuclear pore complex where it is required for proper cell cycle progression and nucleocytoplasmic transport. The 3' portion of this gene forms a fusion gene with the DEK gene on chromosome 6 in a t(6,9) translocation associated with acute myeloid leukemia and myelodysplastic syndrome. Alternative splicing of this gene results in multiple transcript variants encoding different isoforms.","The nuclear pore complex is a massive structure that extends across the nuclear envelope, forming a gateway that regulates the flow of macromolecules between the nucleus and the cytoplasm. Nucleoporins are the main components of the nuclear pore complex in eukaryotic cells. This gene encodes a nucleoporin protein that localizes both to the basket of the pore and to the nuclear entry of the central gated channel of the pore. The encoded protein is a target of caspase cysteine proteases that play a central role in programmed cell death by apoptosis. Alternative splicing results in multiple transcript variants encoding different isoforms.",null,"This gene encodes a member of the AAA (ATPases associated with diverse cellular activities) superfamily. Multiple transcript variants encoding different isoforms have been found for this gene. Two encoded proteins, described as major and minor isoforms, have been localized to distinct regions of the nucleus. The largest encoded protein (major isoform) has been localized to the nucleolus and shown to participate in ribosome biosynthesis (PMID: 15469983, 16782053), while the minor isoform has been localized to the nucleoplasmin.","This gene is one member of a family of nuclear RNA export factor genes. Common domain features of this family are a noncanonical RNP-type RNA-binding domain (RBD), 4 leucine-rich repeats (LRRs), a nuclear transport factor 2 (NTF2)-like domain that allows heterodimerization with NTF2-related export protein-1 (NXT1), and a ubiquitin-associated domain that mediates interactions with nucleoporins. The LRRs and NTF2-like domains are required for export activity. Alternative splicing seems to be a common mechanism in this gene family. The encoded protein of this gene shuttles between the nucleus and the cytoplasm and binds in vivo to poly(A)+ RNA. It is the vertebrate homologue of the yeast protein Mex67p. The encoded protein overcomes the mRNA export block caused by the presence of saturating amounts of CTE (constitutive transport element) RNA of type D retroviruses. Alternative splicing results in multiple transcript variants.","This gene encodes a basic helix-loop-helix transcription factor which is expressed in oligodendroglial tumors of the brain. The protein is an essential regulator of ventral neuroectodermal progenitor cell fate. The gene is involved in a chromosomal translocation t(14;21)(q11.2;q22) associated with T-cell acute lymphoblastic leukemia. Its chromosomal location is within a region of chromosome 21 which has been suggested to play a role in learning deficits associated with Down syndrome.","This gene encodes poly(ADP-ribosyl)transferase-like 1 protein, which is capable of catalyzing a poly(ADP-ribosyl)ation reaction. This protein has a catalytic domain which is homologous to that of poly (ADP-ribosyl) transferase, but lacks an N-terminal DNA binding domain which activates the C-terminal catalytic domain of poly (ADP-ribosyl) transferase. Since this protein is not capable of binding DNA directly, its transferase activity may be activated by other factors such as protein-protein interaction mediated by the extensive carboxyl terminus.","The protein encoded by this gene contains an A-T hook DNA binding motif which usually binds to other DNA binding structures to play an important role in chromatin modeling and transcription regulation. Its Poz domain is thought to function as a site for protein-protein interaction and is required for transcriptional repression, and the zinc-fingers comprise the DNA binding domain. Since the encoded protein has typical features of a transcription factor, it is postulated to be a repressor of gene expression. In small round cell sarcoma, this gene is fused to EWS by a small inversion of 22q, then the hybrid is thought to be translocated (t(1;22)(p36.1;q12). The rearrangement of chromosome 22 involves intron 8 of EWS and exon 1 of this gene creating a chimeric sequence containing the transactivation domain of EWS fused to zinc finger domain of this protein. This is a distinct example of an intra-chromosomal rearrangement of chromosome 22. Four alternatively spliced transcript variants are described for this gene.","This gene is a member of the paired box (PAX) family of transcription factors. Members of the PAX family typically contain a paired box domain and a paired-type homeodomain. These genes play critical roles during fetal development. Mutations in paired box gene 3 are associated with Waardenburg syndrome, craniofacial-deafness-hand syndrome, and alveolar rhabdomyosarcoma. The translocation t(2;13)(q35;q14), which represents a fusion between PAX3 and the forkhead gene, is a frequent finding in alveolar rhabdomyosarcoma. Alternative splicing results in transcripts encoding isoforms with different C-termini.","This gene encodes a nuclear protein that belongs to the PBX homeobox family of transcriptional factors. Studies in mice suggest that this gene may be involved in the regulation of osteogenesis and required for skeletal patterning and programming. A chromosomal translocation, t(1;19) involving this gene and TCF3/E2A gene, is associated with pre-B-cell acute lymphoblastic leukemia. The resulting fusion protein, in which the DNA binding domain of E2A is replaced by the DNA binding domain of this protein, transforms cells by constitutively activating transcription of genes regulated by the PBX protein family. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a component of centriolar satellites, which are electron dense granules scattered around centrosomes. Inhibition studies show that this protein is essential for the correct localization of several centrosomal proteins, and for anchoring microtubules to the centrosome. Chromosomal aberrations involving this gene are associated with papillary thyroid carcinomas and a variety of hematological malignancies, including atypical chronic myeloid leukemia and T-cell lymphoma. Multiple transcript variants encoding different isoforms have been found for this gene.","Predicted to enable protein-L-isoaspartate (D-aspartate) O-methyltransferase activity. Predicted to be involved in protein methylation. Predicted to be located in membrane. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the protein family comprised of both platelet-derived growth factors (PDGF) and vascular endothelial growth factors (VEGF). The encoded preproprotein is proteolytically processed to generate platelet-derived growth factor subunit B, which can homodimerize, or alternatively, heterodimerize with the related platelet-derived growth factor subunit A. These proteins bind and activate PDGF receptor tyrosine kinases, which play a role in a wide range of developmental processes. Mutations in this gene are associated with meningioma. Reciprocal translocations between chromosomes 22 and 17, at sites where this gene and that for collagen type 1, alpha 1 are located, are associated with dermatofibrosarcoma protuberans, a rare skin tumor. Alternative splicing results in multiple transcript variants.","This gene encodes a protein that interacts with the conserved protein complex termed cohesin. The cohesin complex holds together sister chromatids and facilitates accurate chromosome segregation during mitosis and meiosis. This protein is also a negative regulator of cell proliferation and may be a tumor-suppressor gene.","The protein encoded by this gene contains six PDZ domains and shares sequence similarity with pro-interleukin-16 (pro-IL-16). Like pro-IL-16, the encoded protein localizes to the endoplasmic reticulum and is thought to be cleaved by a caspase to produce a secreted peptide containing two PDZ domains. In addition, this gene is upregulated in primary prostate tumors and may be involved in the early stages of prostate tumorigenesis.","In human, ZIM2 and PEG3 are treated as two distinct genes though they share multiple 5' exons and a common promoter and both genes are paternally expressed (PMID:15203203). Alternative splicing events connect their shared 5' exons either with the remaining 4 exons unique to ZIM2, or with the remaining 2 exons unique to PEG3. In contrast, in other mammals ZIM2 does not undergo imprinting and, in mouse, cow, and likely other mammals as well, the ZIM2 and PEG3 genes do not share exons. Human PEG3 protein belongs to the Kruppel C2H2-type zinc finger protein family. PEG3 may play a role in cell proliferation and p53-mediated apoptosis. PEG3 has also shown tumor suppressor activity and tumorigenesis in glioma and ovarian cells. Alternative splicing of this PEG3 gene results in multiple transcript variants encoding distinct isoforms.","This gene is a member of the Period family of genes and is expressed in a circadian pattern in the suprachiasmatic nucleus, the primary circadian pacemaker in the mammalian brain. Genes in this family encode components of the circadian rhythms of locomotor activity, metabolism, and behavior. This gene is upregulated by CLOCK/ARNTL heterodimers but then represses this upregulation in a feedback loop using PER/CRY heterodimers to interact with CLOCK/ARNTL. Polymorphisms in this gene may increase the risk of getting certain cancers. Alternative splicing has been observed in this gene; however, these variants have not been fully described.","This gene is a member of the Period family of genes and is expressed in a circadian pattern in the suprachiasmatic nucleus, the primary circadian pacemaker in the mammalian brain. Genes in this family encode components of the circadian rhythms of locomotor activity, metabolism, and behavior. This gene is upregulated by CLOCK/ARNTL heterodimers but then represses this upregulation in a feedback loop using PER/CRY heterodimers to interact with CLOCK/ARNTL. Polymorphisms in this gene have been linked to sleep disorders. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a member of the steroid receptor superfamily. The encoded protein mediates the physiological effects of progesterone, which plays a central role in reproductive events associated with the establishment and maintenance of pregnancy. This gene uses two distinct promotors and translation start sites in the first exon to produce several transcript variants, both protein coding and non-protein coding. Two of the isoforms (A and B) are identical except for an additional 165 amino acids found in the N-terminus of isoform B and mediate their own response genes and physiologic effects with little overlap.","The DNA-associated protein encoded by this gene is a member of the paired family of homeobox proteins localized to the nucleus. The protein functions as a transcription factor involved in the development of several major noradrenergic neuron populations and the determination of neurotransmitter phenotype. The gene product is linked to enhancement of second messenger-mediated activation of the dopamine beta-hydroylase, c-fos promoters and several enhancers, including cyclic amp-response element and serum-response element. Expansion of a 20 amino acid polyalanine tract in this protein by 5-13 aa has been associated with congenital central hypoventilation syndrome.","This gene encodes a phosphatidylinositol (PI) 4-kinase which catalyzes the first committed step in the biosynthesis of phosphatidylinositol 4,5-bisphosphate. The mammalian PI 4-kinases have been classified into two types, II and III, based on their molecular mass, and modulation by detergent and adenosine. The protein encoded by this gene is a type III enzyme that is not inhibited by adenosine. Alternative splicing results in multiple transcript variants encoding distinct isoforms.","This gene encodes a clathrin assembly protein, which recruits clathrin and adaptor protein complex 2 (AP2) to cell membranes at sites of coated-pit formation and clathrin-vesicle assembly. The protein may be required to determine the amount of membrane to be recycled, possibly by regulating the size of the clathrin cage. The protein is involved in AP2-dependent clathrin-mediated endocytosis at the neuromuscular junction. A chromosomal translocation t(10;11)(p13;q14) leading to the fusion of this gene and the MLLT10 gene is found in acute lymphoblastic leukemia, acute myeloid leukemia and malignant lymphomas. The polymorphisms of this gene are associated with the risk of Alzheimer disease. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene.","Phosphoinositide 3-kinases (PI3Ks) phosphorylate inositol lipids and are involved in the immune response. The protein encoded by this gene is a class I catalytic subunit of PI3K. Like other class I catalytic subunits (p110-alpha p110-beta, and p110-delta), the encoded protein binds a p85 regulatory subunit to form PI3K. This gene is located in a commonly deleted segment of chromosome 7 previously identified in myeloid leukemias. Several transcript variants encoding the same protein have been found for this gene.","The protein encoded by this gene catalyzes the formation of inositol 1,4,5-trisphosphate and diacylglycerol from phosphatidylinositol 4,5-bisphosphate. This reaction uses calcium as a cofactor and plays an important role in the intracellular transduction of many extracellular signals in the retina. Multiple transcript variants encoding different isoforms have been found for this gene.","Members of the B class of plexins, such as PLXNB2 are transmembrane receptors that participate in axon guidance and cell migration in response to semaphorins (Perrot et al. (2002)).","This gene encodes a member of the plexin family. Plexins are transmembrane receptors for semaphorins, a large family of proteins that regulate axon guidance, cell motility and migration, and the immune response. The encoded protein and its ligand regulate melanocyte adhesion, and viral semaphorins may modulate the immune response by binding to this receptor. The encoded protein may be a tumor suppressor protein for melanoma. Alternatively spliced transcript variants have been observed for this gene.","Mitochondrial DNA polymerase is heterotrimeric, consisting of a homodimer of accessory subunits plus a catalytic subunit. The protein encoded by this gene is the catalytic subunit of mitochondrial DNA polymerase. The encoded protein contains a polyglutamine tract near its N-terminus that may be polymorphic. Defects in this gene are a cause of progressive external ophthalmoplegia with mitochondrial DNA deletions 1 (PEOA1), sensory ataxic neuropathy dysarthria and ophthalmoparesis (SANDO), Alpers-Huttenlocher syndrome (AHS), and mitochondrial neurogastrointestinal encephalopathy syndrome (MNGIE). Two transcript variants encoding the same protein have been found for this gene.","This gene encodes a mitochondrial DNA-directed RNA polymerase. The gene product is responsible for mitochondrial gene expression as well as for providing RNA primers for initiation of replication of the mitochondrial genome. Although this polypeptide has the same function as the three nuclear DNA-directed RNA polymerases, it is more closely related to RNA polymerases of phage and mitochondrial polymerases of lower eukaryotes.","Enables transcription coactivator activity. Involved in positive regulation of transcription by RNA polymerase II. Part of RNA polymerase II transcription regulator complex. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a homeobox-containing transcription factor of the POU domain family. The encoded protein binds the octamer sequence 5'-ATTTGCAT-3', a common transcription factor binding site in immunoglobulin gene promoters. Several transcript variants encoding different isoforms have been found for this gene.","This gene encodes a transcription factor containing a POU homeodomain that plays a key role in embryonic development and stem cell pluripotency. Aberrant expression of this gene in adult tissues is associated with tumorigenesis. This gene can participate in a translocation with the Ewing's sarcoma gene on chromosome 21, which also leads to tumor formation. Alternative splicing, as well as usage of alternative AUG and non-AUG translation initiation codons, results in multiple isoforms. One of the AUG start codons is polymorphic in human populations. Related pseudogenes have been identified on chromosomes 1, 3, 8, 10, and 12.","This gene encodes a member of the peroxisome proliferator-activated receptor (PPAR) subfamily of nuclear receptors. PPARs form heterodimers with retinoid X receptors (RXRs) and these heterodimers regulate transcription of various genes. Three subtypes of PPARs are known: PPAR-alpha, PPAR-delta, and PPAR-gamma. The protein encoded by this gene is PPAR-gamma and is a regulator of adipocyte differentiation. Additionally, PPAR-gamma has been implicated in the pathology of numerous diseases including obesity, diabetes, atherosclerosis and cancer. Alternatively spliced transcript variants that encode different isoforms have been described.","Enables several functions, including ATPase binding activity; calmodulin binding activity; and calmodulin-dependent protein phosphatase activity. Involved in several processes, including calcineurin-NFAT signaling cascade; peptidyl-serine dephosphorylation; and response to calcium ion. Located in several cellular components, including cytosol; dendritic spine; and nucleoplasm. Part of calcineurin complex. Colocalizes with cytoplasmic side of plasma membrane. Implicated in developmental and epileptic encephalopathy 91. Biomarker of focal segmental glomerulosclerosis and schizophrenia. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a protein with structural similarities to complement component C9 that is important in immunity. This protein forms membrane pores that allow the release of granzymes and subsequent cytolysis of target cells. Whether pore formation occurs in the plasma membrane of target cells or in an endosomal membrane inside target cells is subject to debate. Mutations in this gene are associated with a variety of human disease including diabetes, multiple sclerosis, lymphomas, autoimmune lymphoproliferative syndrome (ALPS), aplastic anemia, and familial hemophagocytic lymphohistiocytosis type 2 (FHL2), a rare and lethal autosomal recessive disorder of early childhood.","This gene encodes one of the catalytic subunits of protein kinase A, which exists as a tetrameric holoenzyme with two regulatory subunits and two catalytic subunits, in its inactive form. cAMP causes the dissociation of the inactive holoenzyme into a dimer of regulatory subunits bound to four cAMP and two free monomeric catalytic subunits. Four different regulatory subunits and three catalytic subunits have been identified in humans. cAMP-dependent phosphorylation of proteins by protein kinase A is important to many cellular processes, including differentiation, proliferation, and apoptosis. Constitutive activation of this gene caused either by somatic mutations, or genomic duplications of regions that include this gene, have been associated with hyperplasias and adenomas of the adrenal cortex and are linked to corticotropin-independent Cushing's syndrome. Alternative splicing results in multiple transcript variants encoding different isoforms. Tissue-specific isoforms that differ at the N-terminus have been described, and these isoforms may differ in the post-translational modifications that occur at the N-terminus of some isoforms.","The protein encoded by this gene is a member of the protein kinase C family of serine- and threonine-specific protein kinases. The encoded protein is activated by diacylglycerol and is both a tumor suppressor and a positive regulator of cell cycle progression. Also, this protein can positively or negatively regulate apoptosis. Defects in this gene are a cause of autoimmune lymphoproliferative syndrome.","The protein encoded by this gene is a serine/threonine protein kinase involved in many cellular processes, including Golgi body membrane integrity and transport, cell migration and differentiation, MAPK8/JNK1 and Ras pathway signaling, MAPK1/3 (ERK1/2) pathway signaling, cell survival, and regulation of cell shape and adhesion.","The protein encoded by this gene belongs to the protein kinase D (PKD) family of serine/threonine protein kinases. This kinase can be activated by phorbol esters as well as by gastrin via the cholecystokinin B receptor (CCKBR) in gastric cancer cells. It can bind to diacylglycerol (DAG) in the trans-Golgi network (TGN) and may regulate basolateral membrane protein exit from TGN. Alternative splicing results in multiple transcript variants encoding different isoforms.","This gene encodes a WW-domain containing protein similar to yeast splicing factor PRP40. This protein has been shown to interact with Huntingtin and methyl CpG binding protein 2 (MeCP2). Alternative splicing results in different transcript variants.","The protein encoded by this gene belongs to the B-cell CLL/lymphoma 2 and adenovirus E1B 19 kDa interacting family, whose members play roles in many cellular processes including apotosis, cell transformation, and synaptic function. Several functions for this protein have been demonstrated including suppression of Ras homolog family member A activity, which results in reduced stress fiber formation and suppression of oncogenic cellular transformation. A high molecular weight isoform of this protein has also been shown to colocalize with Adaptor protein complex 2, beta-Adaptin and endodermal markers, suggesting an involvement in post-endocytic trafficking. In prostate cancer cells, this gene acts as a tumor suppressor and its expression is regulated by prostate cancer antigen 3, a non-protein coding gene on the opposite DNA strand in an intron of this gene. Prostate cancer antigen 3 regulates levels of this gene through formation of a double-stranded RNA that undergoes adenosine deaminase actin on RNA-dependent adenosine-to-inosine RNA editing. Alternative splicing results in multiple transcript variants.","This gene encodes a nucleolar protein that localizes to punctate subnuclear structures that occur close to splicing speckles, known as paraspeckles. These paraspeckles are composed of RNA-protein structures that include a non-coding RNA, NEAT1/Men epsilon/beta, and the Drosophila Behavior Human Splicing family of proteins, which include the product of this gene and the P54NRB/NONO and PSF/SFPQ proteins. Paraspeckles may function in the control of gene expression via an RNA nuclear retention mechanism. The protein encoded by this gene is found in paraspeckles in transcriptionally active cells, but it localizes to unique cap structures at the nucleolar periphery when RNA polymerase II transcription is inhibited, or during telophase. Alternative splicing of this gene results in multiple transcript variants. A related pseudogene, which is also located on chromosome 13, has been identified.","The protein encoded by this gene contains a characteristic motif of protein tyrosine phosphatases (PTPs). PTPs regulate activities of phosphoproteins through dephosphorylation. They are signaling molecules involved in the regulation of a wide variety of biological processes. The specific function of this protein has not yet been determined. Alternatively spliced transcript variants encoding distinct isoforms have been identified.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This PTP contains an N-terminal noncatalytic domain similar to that of band 4.1 superfamily cytoskeleton-associated proteins, which suggested the membrane or cytoskeleton localization of this protein. It appears to regulate lymphatic development in mammals, and a loss of function mutation has been found in a kindred with a lymphedema-choanal atresia.","The protein encoded by this gene is a member of the protein tyrosine phosphatase (PTP) family. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation. This protein contains a C-terminal PTP domain and an N-terminal domain homologous to the band 4.1 superfamily of cytoskeletal-associated proteins. P97, a cell cycle regulator involved in a variety of membrane related functions, has been shown to be a substrate of this PTP. This PTP was also found to interact with, and be regulated by adaptor protein 14-3-3 beta. Several alternatively spliced transcript variants encoding different isoforms have been found for this gene.","Enables chromatin binding activity and histone binding activity. Located in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is highly similar to the gene product of Schizosaccharomyces pombe rad17, a cell cycle checkpoint gene required for cell cycle arrest and DNA damage repair in response to DNA damage. This protein shares strong similarity with DNA replication factor C (RFC), and can form a complex with RFCs. This protein binds to chromatin prior to DNA damage and is phosphorylated by the checkpoint kinase ATR following damage. This protein recruits the RAD1-RAD9-HUS1 checkpoint protein complex onto chromatin after DNA damage, which may be required for its phosphorylation. The phosphorylation of this protein is required for the DNA-damage-induced cell cycle G2 arrest, and is thought to be a critical early event during checkpoint signaling in DNA-damaged cells. Multiple alternatively spliced transcript variants of this gene, which encode four distinct protein isoforms, have been reported. Two pseudogenes, located on chromosomes 7 and 13, have been identified.","The protein encoded by this gene is highly similar to Saccharomyces cerevisiae Rad50, a protein involved in DNA double-strand break repair. This protein forms a complex with MRE11 and NBS1. The protein complex binds to DNA and displays numerous enzymatic activities that are required for nonhomologous joining of DNA ends. This protein, cooperating with its partners, is important for DNA double-strand break repair, cell cycle checkpoint activation, telomere maintenance, and meiotic recombination. Knockout studies of the mouse homolog suggest this gene is essential for cell growth and viability. Mutations in this gene are the cause of Nijmegen breakage syndrome-like disorder.[provided by RefSeq, Apr 2010].","The protein encoded by this gene is a member of the RAD51 protein family. RAD51 family members are evolutionarily conserved proteins essential for DNA repair by homologous recombination. This protein has been shown to form a stable heterodimer with the family member RAD51C, which further interacts with the other family members, such as RAD51, XRCC2, and XRCC3. Overexpression of this gene was found to cause cell cycle G1 delay and cell apoptosis, which suggested a role of this protein in sensing DNA damage. Rearrangements between this locus and high mobility group AT-hook 2 (HMGA2, GeneID 8091) have been observed in uterine leiomyomata.","This gene encodes a member of the RAS-like small GTP-binding protein superfamily. Members of this family regulate multiple cellular processes including cell adhesion and growth and differentiation. This protein localizes to cellular membranes and has been shown to regulate integrin-mediated cell signaling. This protein also plays a role in regulating outside-in signaling in platelets. Alternate splicing results in multiple transcript variants. Pseudogenes of this gene are found on chromosomes 3, 5, 6 and 9.","The protein encoded by this gene is member of the GAP1 family of GTPase-activating proteins. The gene product stimulates the GTPase activity of normal RAS p21 but not its oncogenic counterpart. Acting as a suppressor of RAS function, the protein enhances the weak intrinsic GTPase activity of RAS proteins resulting in the inactive GDP-bound form of RAS, thereby allowing control of cellular proliferation and differentiation. Alternative splicing results in multiple transcript variants.","The Fox-1 family of RNA-binding proteins is evolutionarily conserved, and regulates tissue-specific alternative splicing in metazoa. Fox-1 recognizes a (U)GCAUG stretch in regulated exons or in flanking introns. The protein binds to the C-terminus of ataxin-2 and may contribute to the restricted pathology of spinocerebellar ataxia type 2 (SCA2). Ataxin-2 is the product of the SCA2 gene which causes familial neurodegenerative diseases. Fox-1 and ataxin-2 are both localized in the trans-Golgi network. Several alternatively spliced transcript variants encoding different isoforms have been found for this gene.","Members of the SPEN (Split-end) family of proteins, including RBM15, have repressor function in several signaling pathways and may bind to RNA through interaction with spliceosome components (Hiriart et al., 2005).","Enables mRNA 3'-UTR binding activity. Involved in DNA damage response, signal transduction by p53 class mediator; negative regulation of cell population proliferation; and regulation of RNA metabolic process. Located in cytosol and nucleus. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the U2AF65 family of proteins. The encoded protein is found in the nucleus, where it co-localizes with core spliceosomal proteins. It has been shown to play a role in both steroid hormone receptor-mediated transcription and alternative splicing, and it is also a transcriptional coregulator of the viral oncoprotein v-Rel. Multiple transcript variants have been observed for this gene. A related pseudogene has been identified on chromosome X.","This gene encodes a protein that belongs to the Rel homology domain/immunoglobulin-like fold, plexin, transcription factor (RHD/IPT) family. Members of this family regulate genes involved in apoptosis, inflammation, the immune response, and oncogenic processes. This proto-oncogene plays a role in the survival and proliferation of B lymphocytes. Mutation or amplification of this gene is associated with B-cell lymphomas, including Hodgkin's lymphoma. Single nucleotide polymorphisms in this gene are associated with susceptibility to ulcerative colitis and rheumatoid arthritis. Alternative splicing results in multiple transcript variants encoding different isoforms.","NF-kappa-B is a ubiquitous transcription factor involved in several biological processes. It is held in the cytoplasm in an inactive state by specific inhibitors. Upon degradation of the inhibitor, NF-kappa-B moves to the nucleus and activates transcription of specific genes. NF-kappa-B is composed of NFKB1 or NFKB2 bound to either REL, RELA, or RELB. The most abundant form of NF-kappa-B is NFKB1 complexed with the product of this gene, RELA. Four transcript variants encoding different isoforms have been found for this gene.","This gene was initially identified as a transcriptional repressor that represses neuronal genes in non-neuronal tissues. However, depending on the cellular context, this gene can act as either an oncogene or a tumor suppressor. The encoded protein is a member of the Kruppel-type zinc finger transcription factor family. It represses transcription by binding a DNA sequence element called the neuron-restrictive silencer element. The protein is also found in undifferentiated neuronal progenitor cells and it is thought that this repressor may act as a master negative regulator of neurogenesis. Alternatively spliced transcript variants have been described.","This gene encodes the large subunit of replication factor C, a five subunit DNA polymerase accessory protein, which is a DNA-dependent ATPase required for eukaryotic DNA replication and repair. The large subunit acts as an activator of DNA polymerases, binds to the 3' end of primers, and promotes coordinated synthesis of both strands. It may also have a role in telomere stability. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.","Enables MDM2/MDM4 family protein binding activity; p53 binding activity; and ubiquitin protein ligase activity. Involved in several processes, including DNA metabolic process; regulation of cell cycle phase transition; and response to ionizing radiation. Located in nucleoplasm and site of DNA damage. Colocalizes with site of double-strand break. Implicated in Fanconi anemia. [provided by Alliance of Genome Resources, Apr 2022]","This gene is a member of the small GTPase superfamily and encodes a lipid-anchored, cell membrane protein with five repeats of the RAS-related GTP-binding region. This protein is vital in regulation of growth and cell cycle progression due to its role in the insulin/TOR/S6K signaling pathway. The protein has GTPase activity and shuttles between a GDP-bound form and a GTP-bound form, and farnesylation of the protein is required for this activity. Three pseudogenes have been mapped, two on chromosome 10 and one on chromosome 22.","Predicted to enable GTP binding activity; GTPase activity; and protein kinase binding activity. Involved in several processes, including cellular response to hydrogen peroxide; cellular response to ionizing radiation; and regulation of cell migration. Located in cleavage furrow and endosome membrane. Biomarker of breast cancer. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a member of the Ras superfamily of guanosine triphosphate (GTP)-metabolizing enzymes. The encoded protein is expressed in hematopoietic cells, where it functions as a negative regulator of cell growth and survival. This gene may be hypermutated or misexpressed in leukemias and lymphomas. Chromosomal translocations in non-Hodgkin's lymphoma occur between this locus and B-cell CLL/lymphoma 6 (BCL6) on chromosome 3, leading to the production of fusion transcripts. Alternative splicing in the 5' untranslated region results in multiple transcript variants that encode the same protein.","This gene encodes a member of the rhophilin family of Ras-homologous (Rho)-GTPase binding proteins. The encoded protein binds both GTP- and GDP-bound RhoA and GTP-bound RhoB and may be involved in the organization of the actin cytoskeleton.","This gene encodes a member of the receptor-interacting protein (RIP) family of serine/threonine protein kinases. The encoded protein plays a role in inflammation and cell death in response to tissue damage, pathogen recognition, and as part of developmental regulation. RIPK1/RIPK3 kinase-mediated necrosis is referred to as necroptosis. Genetic disruption of this gene in mice results in death shortly after birth.","RMI2 is a component of the BLM (RECQL3; MIM 604610) complex, which plays a role in homologous recombination-dependent DNA repair and is essential for genome stability (Xu et al., 2008).","The protein encoded by this gene is a nuclear RING-domain containing E3 ubiquitin ligase. This protein interacts with the transforming growth factor (TGF) -beta/NODAL signaling pathway by promoting the ubiquitination and proteosomal degradation of negative regulators, like SMAD proteins, and thereby enhances TGF-beta target-gene transcription. As a modulator of the nodal signaling cascade, this gene plays a critical role in the induction of mesoderm during embryonic development. Alternative splicing of this gene results in multiple transcript variants encoding distinct isoforms.","This gene encodes a protein containing a C3HC4-type RING finger domain, which is a specialized type of Zn-finger that binds two atoms of zinc and is thought to be involved in mediating protein-protein interactions. The protein also contains an AAA domain, which is associated with ATPase activity. This gene is a susceptibility gene for Moyamoya disease, a vascular disorder of intracranial arteries. This gene is also a translocation partner in anaplastic large cell lymphoma and inflammatory myofibroblastic tumor cases, where a t(2;17)(p23;q25) translocation has been identified with the anaplastic lymphoma kinase (ALK) gene on chromosome 2, and a t(8;17)(q24;q25) translocation has been identified with the MYC gene on chromosome 8. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene contains a RING-H2 finger motif. Deletions and mutations in this gene were detected in esophageal squamous cell carcinoma (ESCC), suggesting that this protein may be a potential tumor suppressor. Studies of the mouse counterpart suggested a role of this protein in the transcription regulation that controls germinal differentiation. Multiple alternatively spliced transcript variants encoding the same protein are observed.","Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 40S subunit. The protein belongs to the S3AE family of ribosomal proteins. It is located in the cytoplasm. Disruption of the gene encoding rat ribosomal protein S3a, also named v-fos transformation effector protein, in v-fos-transformed rat cells results in reversion of the transformed phenotype. This gene is co-transcribed with the U73A and U73B small nucleolar RNA genes, which are located in its fourth and third introns, respectively. As is typical for genes encoding ribosomal proteins, there are multiple processed pseudogenes of this gene dispersed through the genome. Alternatively spliced transcript variants have been found for this gene.","This gene encodes a member of the GTR/RAG GTP-binding protein family. The encoded protein is a monomeric guanine nucleotide-binding protein which forms a heterodimer with RRAGA and RRAGB and is primarily localized to the cytoplasm. The encoded protein promotes intracellular localization of the mTOR complex. Alternative splicing results in multiple transcript variants.","Enables RNA polymerase I core promoter sequence-specific DNA binding activity. Predicted to be involved in transcription initiation from RNA polymerase I promoter. Predicted to act upstream of or within several processes, including DNA-templated transcription, initiation; negative regulation of intrinsic apoptotic signaling pathway by p53 class mediator; and regulation of transcription, DNA-templated. Located in nucleolus. [provided by Alliance of Genome Resources, Apr 2022]","This gene belongs to the R-spondin family. The encoded protein plays a role in the regulation of Wnt (wingless-type MMTV integration site family)/beta-catenin and Wnt/planar cell polarity (PCP) signaling pathways, which are involved in development, cell growth and disease pathogenesis. Genome-wide association studies suggest a correlation of this gene with bone mineral density and risk of fracture. This gene may be involved in tumor development.","The protein encoded by this gene is a member of the S100 family of proteins containing 2 EF-hand calcium-binding motifs. S100 proteins are localized in the cytoplasm and/or nucleus of a wide range of cells, and involved in the regulation of a number of cellular processes such as cell cycle progression and differentiation. S100 genes include at least 13 members which are located as a cluster on chromosome 1q21. This protein differs from the other S100 proteins of known structure in its lack of calcium binding ability in one EF-hand at the N-terminus. The protein is overexpressed in hyperproliferative skin diseases, exhibits antimicrobial activities against bacteria and induces immunomodulatory activities.","This gene may play a role in regulation of the innate immune response. The encoded protein is upregulated in response to viral infection and may be involved in mediation of tumor necrosis factor-alpha proinflammatory responses. Mutations in this gene have been associated with Aicardi-Goutieres syndrome.","This gene encodes a highly conserved protein that plays an essential role in ribosome biogenesis. The encoded protein interacts with elongation factor-like GTPase 1 to disassociate eukaryotic initiation factor 6 from the late cytoplasmic pre-60S ribosomal subunit allowing assembly of the 80S subunit. Mutations within this gene are associated with the autosomal recessive disorder Shwachman-Bodian-Diamond syndrome. This gene has a closely linked pseudogene that is distally located.","This gene likely encodes a member of the arginine/serine-rich splicing factor family. A similar protein in Rat appears to bind the large subunit of RNA polymerase II and provide a link between transcription and pre-mRNA splicing. Alternatively spliced transcript variants have been described.","The protein encoded by this gene was initially identified as a molecule linking syndecan-mediated signaling to the cytoskeleton. The syntenin protein contains tandemly repeated PDZ domains that bind the cytoplasmic, C-terminal domains of a variety of transmembrane proteins. This protein may also affect cytoskeletal-membrane organization, cell adhesion, protein trafficking, and the activation of transcription factors. The protein is primarily localized to membrane-associated adherens junctions and focal adhesions but is also found at the endoplasmic reticulum and nucleus. Alternative splicing results in multiple transcript variants encoding different isoforms. Related pseudogenes have been identified on multiple chromosomes.","This gene encodes a mitochondrial assembly factor needed for the flavination of a succinate dehydrogenase complex subunit (SDHA), which is required for activity of the succinate dehydrogenase complex. Mutations in this gene are associated with paraganglioma.","This tumor suppressor gene encodes the iron-sulfur protein subunit of the succinate dehydrogenase (SDH) enzyme complex which plays a critical role in mitochondria. The SDH enzyme complex is composed of four nuclear-encoded subunits. This enzyme complex converts succinate to fumarate which releases electrons as part of the citric acid cycle, and the enzyme complex additionally provides an attachment site for released electrons to be transferred to the oxidative phosphorylation pathway. The SDH enzyme complex plays a role in oxygen-related gene regulation through its conversion of succinate, which is an oxygen sensor that stabilizes the hypoxia-inducible factor 1 (HIF1) transcription factor. Sporadic and familial mutations in this gene result in paragangliomas, pheochromocytoma, and gastrointestinal stromal tumors, supporting a link between mitochondrial dysfunction and tumorigenesis. Mutations in this gene are also implicated in nuclear type 4 mitochondrial complex II deficiency.","This gene encodes one of four nuclear-encoded subunits that comprise succinate dehydrogenase, also known as mitochondrial complex II, a key enzyme complex of the tricarboxylic acid cycle and aerobic respiratory chains of mitochondria. The encoded protein is one of two integral membrane proteins that anchor other subunits of the complex, which form the catalytic core, to the inner mitochondrial membrane. There are several related pseudogenes for this gene on different chromosomes. Mutations in this gene have been associated with paragangliomas. Alternatively spliced transcript variants have been described.","This gene encodes a member of complex II of the respiratory chain, which is responsible for the oxidation of succinate. The encoded protein is one of two integral membrane proteins anchoring the complex to the matrix side of the mitochondrial inner membrane. Mutations in this gene are associated with the formation of tumors, including hereditary paraganglioma. Transmission of disease occurs almost exclusively through the paternal allele, suggesting that this locus may be maternally imprinted. There are pseudogenes for this gene on chromosomes 1, 2, 3, 7, and 18. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene inhibits acetylation of nucleosomes, especially histone H4, by histone acetylases (HAT). This inhibition is most likely accomplished by masking histone lysines from being acetylated, and the consequence is to silence HAT-dependent transcription. The encoded protein is part of a complex localized to the endoplasmic reticulum but is found in the nucleus and inhibits apoptosis following attack by cytotoxic T lymphocytes. This protein can also enhance DNA replication of the adenovirus genome. Several transcript variants encoding different isoforms have been found for this gene.","This gene encodes a nuclear pre-mRNA splicing factor. The encoded protein specifically recognizes the intron branch point sequence at the 3' splice site, together with the large subunit of U2 auxiliary factor (U2AF), and is required for the early stages of spliceosome assembly. It also plays a role in nuclear pre-mRNA retention and transcriptional repression. The encoded protein contains an N-terminal U2AF ligand motif, a central hnRNP K homology motif and quaking 2 region which bind a key branch-site adenosine within the branch point sequence, a zinc knuckles domain, and a C-terminal proline-rich domain. Alternative splicing results in multiple transcript variants.","Enables DNA binding activity; histone deacetylase binding activity; and protein homodimerization activity. Involved in several processes, including alternative mRNA splicing, via spliceosome; positive regulation of oxidative stress-induced intrinsic apoptotic signaling pathway; and regulation of transcription by RNA polymerase II. Acts upstream of or within double-strand break repair via homologous recombination. Located in chromatin; nuclear matrix; and paraspeckles. [provided by Alliance of Genome Resources, Apr 2022]","Secreted frizzled-related protein 4 (SFRP4) is a member of the SFRP family that contains a cysteine-rich domain homologous to the putative Wnt-binding site of Frizzled proteins. SFRPs act as soluble modulators of Wnt signaling. The expression of SFRP4 in ventricular myocardium correlates with apoptosis related gene expression.","This gene encodes a member of the endophilin family of Src homology 3 domain-containing proteins. The encoded protein is involved in endocytosis and may also play a role in the cell cycle. Overexpression of this gene may play a role in leukemogenesis, and the encoded protein has been implicated in acute myeloid leukemia as a fusion partner of the myeloid-lymphoid leukemia protein. Pseudogenes of this gene are located on the long arm of chromosomes 11 and 17. Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene.","Enables identical protein binding activity. Involved in positive regulation of neuron migration. Located in cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]","This gene is a member of the vertebrate gene family which encode proteins homologous to the Drosophila 'sine oculis' homeobox protein. The encoded protein is a transcription factor which, like other members of this gene family, may be involved in limb or eye development.","This gene encodes the nuclear protooncogene protein homolog of avian sarcoma viral (v-ski) oncogene. It functions as a repressor of TGF-beta signaling, and may play a role in neural tube development and muscle differentiation.","This gene encodes a member of the SWI/SNF family of proteins. The encoded protein is an ATPase which is expressed in diverse tissues and contributes to the chromatin remodeling complex that is involved in transcription. The protein may also play a role in DNA damage, growth inhibition and apoptosis of cancer cells. Alternative splicing results in multiple transcript variants.","The protein encoded by this gene is a member of the SWI/SNF family of proteins and is highly similar to the brahma protein of Drosophila. Members of this family have helicase and ATPase activities and are thought to regulate transcription of certain genes by altering the chromatin structure around those genes. The encoded protein is part of the large ATP-dependent chromatin remodeling complex SNF/SWI, which is required for transcriptional activation of genes normally repressed by chromatin. Alternatively spliced transcript variants encoding different isoforms have been found for this gene, which contains a trinucleotide repeat (CAG) length polymorphism.","The protein encoded by this gene is part of the large ATP-dependent chromatin remodeling complex SWI/SNF, which is required for transcriptional activation of genes normally repressed by chromatin. The encoded protein, either alone or when in the SWI/SNF complex, can bind to 4-way junction DNA, which is thought to mimic the topology of DNA as it enters or exits the nucleosome. The protein contains a DNA-binding HMG domain, but disruption of this domain does not abolish the DNA-binding or nucleosome-displacement activities of the SWI/SNF complex. Unlike most of the SWI/SNF complex proteins, this protein has no yeast counterpart.","This gene belongs to the SMC3 subfamily of SMC proteins. The encoded protein occurs in certain cell types as either an intracellular, nuclear protein or a secreted protein. The nuclear form, known as structural maintenance of chromosomes 3, is a component of the multimeric cohesin complex that holds together sister chromatids during mitosis, enabling proper chromosome segregation. Post-translational modification of the encoded protein by the addition of chondroitin sulfate chains gives rise to the secreted proteoglycan bamacan, an abundant basement membrane protein.","Predicted to enable phosphatidylinositol binding activity. [provided by Alliance of Genome Resources, Apr 2022]","This intronless gene encodes a member of the SRY-related HMG-box (SOX) family of transcription factors involved in the regulation of embryonic development and in the determination of cell fate. The product of this gene is required for stem-cell maintenance in the central nervous system, and also regulates gene expression in the stomach. Mutations in this gene have been associated with optic nerve hypoplasia and with syndromic microphthalmia, a severe form of structural eye malformation. This gene lies within an intron of another gene called SOX2 overlapping transcript (SOX2OT).","This gene encodes a member of the SP100 family of proteins, which are share common domains including an N-terminal homogeneously staining region domain followed by a SP100/autoimmune regulator/NucP41/P75/deformed epidermal autoregulatory factor domain, a plant homeobox zinc finger, and a bromodomain. The encoded protein is interferon-inducible and is expressed at high levels in the nuclei of leukocytes. Variants of this gene have been associated with multiple sclerosis, Crohn's disease, and chronic lymphocytic leukemia. Alternative splicing results in multiple variants.","This gene encodes a member of a family of molecular scaffold proteins that link the plasma membrane to the actin cytoskeleton and functions in the determination of cell shape, arrangement of transmembrane proteins, and organization of organelles. The encoded protein is primarily composed of 22 spectrin repeats which are involved in dimer formation. It forms a component of the erythrocyte plasma membrane. Mutations in this gene result in a variety of hereditary red blood cell disorders, including elliptocytosis-2, pyropoikilocytosis, and spherocytosis, type 3.","Spectrins are a family of filamentous cytoskeletal proteins that function as essential scaffold proteins that stabilize the plasma membrane and organize intracellular organelles. Spectrins are composed of alpha and beta dimers that associate to form tetramers linked in a head-to-head arrangement. This gene encodes an alpha spectrin that is specifically expressed in nonerythrocytic cells. The encoded protein has been implicated in other cellular functions including DNA repair and cell cycle regulation. Mutations in this gene are the cause of early infantile epileptic encephalopathy-5. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Sep 2010].","This gene is highly similar to the v-src gene of Rous sarcoma virus. This proto-oncogene may play a role in the regulation of embryonic development and cell growth. The protein encoded by this gene is a tyrosine-protein kinase whose activity can be inhibited by phosphorylation by c-SRC kinase. Mutations in this gene could be involved in the malignant progression of colon cancer. Two transcript variants encoding the same protein have been found for this gene.","Predicted to enable GTPase activator activity. Predicted to be involved in negative regulation of cell migration. Predicted to be located in cytosol. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]","The protein encoded by this gene is a member of the serine/arginine (SR)-rich family of pre-mRNA splicing factors, which constitute part of the spliceosome. Each of these factors contains an RNA recognition motif (RRM) for binding RNA and an RS domain for binding other proteins. The RS domain is rich in serine and arginine residues and facilitates interaction between different SR splicing factors. In addition to being critical for mRNA splicing, the SR proteins have also been shown to be involved in mRNA export from the nucleus and in translation. Two transcript variants, one protein-coding and the other non-coding, have been found for this gene.","The product of this gene belongs to the family of highly homologous synovial sarcoma X (SSX) breakpoint proteins. These proteins may function as transcriptional repressors. They are also capable of eliciting spontaneous humoral and cellular immune responses in cancer patients, and are potentially useful targets in cancer vaccine-based immunotherapy. This gene, and also the SSX2 and SSX4 family members, have been involved in t(X;18)(p11.2;q11.2) translocations that are characteristically found in all synovial sarcomas. This translocation results in the fusion of the synovial sarcoma translocation gene on chromosome 18 to one of the SSX genes on chromosome X. The encoded hybrid proteins are likely responsible for transforming activity. Alternative splicing of this gene results in multiple transcript variants. A related pseudogene has been identified on chromosome X.","The product of this gene belongs to the family of highly homologous synovial sarcoma X (SSX) breakpoint proteins. These proteins may function as transcriptional repressors. They are also capable of eliciting spontaneous humoral and cellular immune responses in cancer patients, and are potentially useful targets in cancer vaccine-based immunotherapy. This gene, and also the SSX1 and SSX4 family members, have been involved in t(X;18)(p11.2;q11.2) translocations that are characteristically found in all synovial sarcomas. This translocation results in the fusion of the synovial sarcoma translocation gene on chromosome 18 to one of the SSX genes on chromosome X. The encoded hybrid proteins are likely responsible for transforming activity. Alternative splicing of this gene results in multiple transcript variants. This gene also has an identical duplicate, GeneID: 727837, located about 45 kb downstream in the opposite orientation on chromosome X.","The product of this gene belongs to the family of highly homologous synovial sarcoma X (SSX) breakpoint proteins. These proteins may function as transcriptional repressors. They are also capable of eliciting spontaneously humoral and cellular immune responses in cancer patients, and are potentially useful targets in cancer vaccine-based immunotherapy. SSX1, SSX2 and SSX4 genes have been involved in the t(X;18) translocation characteristically found in all synovial sarcomas. This translocation results in the fusion of the synovial sarcoma translocation gene on chromosome 18 to one of the SSX genes on chromosome X. Chromosome Xp11 contains a segmental duplication resulting in two identical copies of synovial sarcoma, X breakpoint 4, SSX4 and SSX4B, in tail-to-tail orientation. This gene, SSX4, represents the more telomeric copy. Two transcript variants encoding distinct isoforms have been identified for this gene.","This gene is a member of the SCC3 family and is expressed in the nucleus. It encodes a component of cohesin, a multisubunit protein complex that provides sister chromatid cohesion along the length of a chromosome from DNA replication through prophase and prometaphase, after which it is dissociated in preparation for segregation during anaphase.","This gene encodes a cytoplasmic protein implicated in regulation of the mitotic spindle checkpoint, a regulatory pathway that monitors chromosome segregation during cell division to ensure the proper distribution of chromosomes to daughter cells. The protein is phosphorylated in mitosis and in response to activation of the spindle checkpoint, and disappears when cells transition to G1 phase. It interacts with a mitotic regulator, and its expression is required to efficiently activate the spindle checkpoint. It is proposed to regulate Cdc2 kinase activity during spindle checkpoint arrest. Chromosomal deletions that fuse this gene and the adjacent locus commonly occur in T cell leukemias, and are thought to arise through illegitimate V-(D)-J recombination events. Multiple transcript variants encoding different isoforms have been found for this gene.","Involved in negative regulation of cell cycle G1/S phase transition and negative regulation of cell division. Located in plasma membrane. [provided by Alliance of Genome Resources, Apr 2022]","This gene encodes a member of the family of non-receptor type Tyr protein kinases. This protein is widely expressed in hematopoietic cells and is involved in coupling activated immunoreceptors to downstream signaling events that mediate diverse cellular responses, including proliferation, differentiation, and phagocytosis. It is thought to be a modulator of epithelial cell growth and a potential tumour suppressor in human breast carcinomas. Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","Initiation of transcription by RNA polymerase II requires the activities of more than 70 polypeptides. The protein that coordinates these activities is the basal transcription factor TFIID, which binds to the core promoter to position the polymerase properly, serves as the scaffold for assembly of the remainder of the transcription complex, and acts as a channel for regulatory signals. TFIID is composed of the TATA-binding protein (TBP) and a group of evolutionarily conserved proteins known as TBP-associated factors or TAFs. TAFs may participate in basal transcription, serve as coactivators, function in promoter recognition or modify general transcription factors (GTFs) to facilitate complex assembly and transcription initiation. This gene encodes the largest subunit of TFIID. This subunit binds to core promoter sequences encompassing the transcription start site. It also binds to activators and other transcriptional regulators, and these interactions affect the rate of transcription initiation. This subunit contains two independent protein kinase domains at the N- and C-terminals, but also possesses acetyltransferase activity and can act as a ubiquitin-activating/conjugating enzyme. Mutations in this gene result in Dystonia 3, torsion, X-linked, a dystonia-parkinsonism disorder. Alternative splicing of this gene results in multiple transcript variants. This gene is part of a complex transcription unit (TAF1/DYT3), wherein some transcript variants share exons with TAF1 as well as additional downstream DYT3 exons.","This locus is intronless, and apparently arose in the primate lineage from retrotransposition of the transcript from the multi-exon TAF1 locus on the X chromosome. The gene is expressed in male germ cells, and the product has been shown to function interchangeably with the TAF1 product.","This gene encodes a member of the E protein (class I) family of helix-loop-helix transcription factors. E proteins activate transcription by binding to regulatory E-box sequences on target genes as heterodimers or homodimers, and are inhibited by heterodimerization with inhibitor of DNA-binding (class IV) helix-loop-helix proteins. E proteins play a critical role in lymphopoiesis, and the encoded protein is required for B and T lymphocyte development. Deletion of this gene or diminished activity of the encoded protein may play a role in lymphoid malignancies. This gene is also involved in several chromosomal translocations that are associated with lymphoid malignancies including pre-B-cell acute lymphoblastic leukemia (t(1;19), with PBX1), childhood leukemia (t(19;19), with TFPT) and acute leukemia (t(12;19), with ZNF384). Alternatively spliced transcript variants encoding multiple isoforms have been observed for this gene, and a pseudogene of this gene is located on the short arm of chromosome 9.","This gene encodes transcription factor 4, a basic helix-loop-helix transcription factor. The encoded protein recognizes an Ephrussi-box ('E-box') binding site ('CANNTG') - a motif first identified in immunoglobulin enhancers. This gene is broadly expressed, and may play an important role in nervous system development. Defects in this gene are a cause of Pitt-Hopkins syndrome. In addition, an intronic CTG repeat normally numbering 10-37 repeat units can expand to >50 repeat units and cause Fuchs endothelial corneal dystrophy. Multiple alternatively spliced transcript variants that encode different proteins have been described.","Antibodies against the protein encoded by this gene were found only in plasma from cancer patients. While it may be a target for immunotherapy, the function of this gene is unknown.","Transcription factors of the basic helix-loop-helix-zipper (bHLH-ZIP) family contain a basic domain, which is used for DNA binding, and HLH and ZIP domains, which are used for oligomerization. Transcription factor AP4 activates both viral and cellular genes by binding to the symmetrical DNA sequence CAGCTG (Mermod et al., 1988; Hu et al., 1990 [PubMed 2123466]).","This gene encodes a basic helix-loop-helix domain-containing transcription factor that binds MUE3-type E-box sequences in the promoter of genes. The encoded protein promotes the expression of genes downstream of transforming growth factor beta (TGF-beta) signaling. This gene may be involved in chromosomal translocations in renal cell carcinomas and other cancers, resulting in the production of fusion proteins. Translocation partners include PRCC (papillary renal cell carcinoma), NONO (non-POU domain containing, octamer-binding), and ASPSCR1 (alveolar soft part sarcoma chromosome region, candidate 1), among other genes. Alternative splicing results in multiple transcript variants.","Enables DNA-binding transcription factor activity; enzyme binding activity; and transcription cis-regulatory region binding activity. Involved in several processes, including cellular response to amino acid starvation; lysosome localization; and positive regulation of autophagy. Located in cytosol; lysosomal membrane; and nucleoplasm. [provided by Alliance of Genome Resources, Apr 2022]","There are several documented fusion oncoproteins encoded partially by this gene. This gene also participates in several oncogenic rearrangements resulting in anaplastic lymphoma and mixoid chondrosarcoma, and may play a role in the NF-kappaB pathway. Multiple transcript variants have been found for this gene.","This gene encodes a cell surface receptor necessary for cellular iron uptake by the process of receptor-mediated endocytosis. This receptor is required for erythropoiesis and neurologic development. Multiple alternatively spliced variants have been identified.","The protein encoded by this gene forms a heteromeric complex with type II TGF-beta receptors when bound to TGF-beta, transducing the TGF-beta signal from the cell surface to the cytoplasm. The encoded protein is a serine/threonine protein kinase. Mutations in this gene have been associated with Loeys-Dietz aortic aneurysm syndrome (LDAS). Multiple transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. They recognize pathogen-associated molecular patterns that are expressed on infectious agents, and mediate the production of cytokines necessary for the development of effective immunity. The various TLRs exhibit different patterns of expression. In silico studies have found a particularly strong binding of surface TLR4 with the spike protein of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of Coronavirus disease-2019 (COVID-19). This receptor has also been implicated in signal transduction events induced by lipopolysaccharide (LPS) found in most gram-negative bacteria. Mutations in this gene have been associated with differences in LPS responsiveness, and with susceptibility to age-related macular degeneration. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a nuclear transcription factor that belongs to the NK-linked or NK-like (NKL) subfamily of homeobox genes. The encoded protein is required for normal development of the spleen during embryogenesis. This protein is also involved in specification of neuronal cell fates. Ectopic expression of this gene due to chromosomal translocations is associated with certain T-cell acute lymphoblastic leukemias. Alternate splicing results in multiple transcript variants.","The protein encoded by this gene is an orphan homeobox protein that encodes a DNA-binding nuclear transcription factor. A translocation [t(5;14)(q35;q32)] involving this gene is associated with T-cell acute lymphoblastic leukemia (T-ALL) in children and young adults.","This gene encodes a transmembrane protein with four predicted transmembrane domains. The protein is associated with a subpopulation of vesicular organelles corresponding to early endosomal structures, with the Golgi, and with lysosomes, and may participate in protein trafficking between these structures. Mutations in this gene and several other genes cause pheochromocytomas. Alternatively spliced transcript variants encoding the same protein have been identified.","This gene encodes a protein that belongs to the serine protease family. The encoded protein contains a type II transmembrane domain, a receptor class A domain, a scavenger receptor cysteine-rich domain and a protease domain. Serine proteases are known to be involved in many physiological and pathological processes. This gene was demonstrated to be up-regulated by androgenic hormones in prostate cancer cells and down-regulated in androgen-independent prostate cancer tissue. The protease domain of this protein is thought to be cleaved and secreted into cell media after autocleavage. This protein also facilitates entry of viruses into host cells by proteolytically cleaving and activating viral envelope glycoproteins. Viruses found to use this protein for cell entry include Influenza virus and the human coronaviruses HCoV-229E, MERS-CoV, SARS-CoV and SARS-CoV-2 (COVID-19 virus). Alternatively spliced transcript variants encoding different isoforms have been found for this gene.","The protein encoded by this gene is a member of the TNF-receptor superfamily. This receptor is preferentially expressed in mature B lymphocytes, and may be important for B cell development and autoimmune response. This receptor has been shown to specifically bind to the tumor necrosis factor (ligand) superfamily, member 13b (TNFSF13B/TALL-1/BAFF), and to lead to NF-kappaB and MAPK8/JNK activation. This receptor also binds to various TRAF family members, and thus may transduce signals for cell survival and proliferation.","This gene encodes a DNA topoisomerase, an enzyme that controls and alters the topologic states of DNA during transcription. This enzyme catalyzes the transient breaking and rejoining of a single strand of DNA which allows the strands to pass through one another, thus altering the topology of DNA. This gene is localized to chromosome 20 and has pseudogenes which reside on chromosomes 1 and 22.","This gene encodes a DNA topoisomerase, an enzyme that controls and alters the topologic states of DNA during transcription. This nuclear enzyme is involved in processes such as chromosome condensation, chromatid separation, and the relief of torsional stress that occurs during DNA transcription and replication. It catalyzes the transient breaking and rejoining of two strands of duplex DNA which allows the strands to pass through one another, thus altering the topology of DNA. Two forms of this enzyme exist as likely products of a gene duplication event. The gene encoding this form, alpha, is localized to chromosome 17 and the beta gene is localized to chromosome 3. The gene encoding this enzyme functions as the target for several anticancer agents and a variety of mutations in this gene have been associated with the development of drug resistance. Reduced activity of this enzyme may also play a role in ataxia-telangiectasia.","This gene encodes a member of the tropomyosin family of actin-binding proteins. Tropomyosins are dimers of coiled-coil proteins that provide stability to actin filaments and regulate access of other actin-binding proteins. Mutations in this gene result in autosomal dominant nemaline myopathy and other muscle disorders. This locus is involved in translocations with other loci, including anaplastic lymphoma receptor tyrosine kinase (ALK) and neurotrophic tyrosine kinase receptor type 1 (NTRK1), which result in the formation of fusion proteins that act as oncogenes. There are numerous pseudogenes for this gene on different chromosomes. Alternative splicing results in multiple transcript variants.","This gene encodes a large coiled-coil protein that forms intranuclear filaments attached to the inner surface of nuclear pore complexes (NPCs). The protein directly interacts with several components of the NPC. It is required for the nuclear export of mRNAs and some proteins. Oncogenic fusions of the 5' end of this gene with several different kinase genes occur in some neoplasias.","This gene encodes a member of the tripartite motif (TRIM) family. The TRIM motif includes three zinc-binding domains, a RING, a B-box type 1 and a B-box type 2, and a coiled-coil region. This protein localizes to the nuclear matrix. It interacts with the enhancer of polycomb protein and represses gene transcription. It is also thought to be involved in the differentiation of male germ cells. Fusion of the N-terminus of this protein with the truncated C-terminus of the RET gene product has been shown to result in production of the ret transforming protein.","Predicted to enable ubiquitin protein ligase activity. Predicted to be involved in innate immune response; protein ubiquitination; and regulation of gene expression. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]","This gene was identified based on the interaction of its protein product with thyroid hormone receptor beta. This protein is associated with the Golgi apparatus. The N-terminal region of the protein binds Golgi membranes and the C-terminal region binds the minus ends of microtubules; thus, the protein is thought to play a role in assembly and maintenance of the Golgi ribbon structure around the centrosome. Mutations in this gene cause achondrogenesis type IA.[provided by RefSeq, Mar 2010].","This gene encodes a thioredoxin-binding protein that is a member of the alpha arrestin protein family. Thioredoxin is a thiol-oxidoreductase that is a major regulator of cellular redox signaling which protects cells from oxidative stress. This protein inhibits the antioxidative function of thioredoxin resulting in the accumulation of reactive oxygen species and cellular stress. This protein also functions as a regulator of cellular metabolism and of endoplasmic reticulum (ER) stress. This protein may also function as a tumor suppressor. Alternate splicing results in multiple transcript variants.","U2 auxiliary factor (U2AF), comprised of a large and a small subunit, is a non-snRNP protein required for the binding of U2 snRNP to the pre-mRNA branch site. This gene encodes the U2AF large subunit which contains a sequence-specific RNA-binding region with 3 RNA recognition motifs and an Arg/Ser-rich domain necessary for splicing. The large subunit binds to the polypyrimidine tract of introns early during spliceosome assembly. Multiple transcript variants have been detected for this gene, but the full-length natures of only two have been determined to date.","The modification of proteins with ubiquitin is an important cellular mechanism for targeting abnormal or short-lived proteins for degradation. Ubiquitination involves at least three classes of enzymes: ubiquitin-activating enzymes, ubiquitin-conjugating enzymes, and ubiquitin-protein ligases. This gene encodes a member of the E2 ubiquitin-conjugating enzyme family. This enzyme is required for post-replicative DNA damage repair, and may play a role in transcriptional regulation. Mutations in this gene are associated with cognitive disability. Alternative splicing results in multiple transcript variants.","This gene encodes a protein with a divergent C-terminal UBX domain. The homologous protein in the rat interacts with members of the Rnd subfamily of Rho GTPases at the cell periphery through its C-terminal region. It also interacts with several heterotrimeric G proteins through their G-alpha subunits and promotes Rho GTPase activation. It is proposed to serve a bidirectional role in the promotion and inhibition of Rho activity through upstream signaling pathways. The 3' coding sequence of this gene contains a polymoprhic region of 24 nt tandem repeats. Several transcripts containing between 1.5 and five repeat units have been reported. Multiple transcript variants encoding different isoforms have been found for this gene.","This gene encodes a homeobox transcription factor that is involved in somitogenesis and neurogenesis and is required for the maintenance and differentiation of specific elements of the axial skeleton. This gene also plays a role in controlling the development of connections of hypothalamic neurons to pituitary elements, allowing central neurons to reach the peripheral blood circulation and deliver hormones that control peripheral functions. The expression of this gene is associated with an increased frequency of acute myeloid leukemia.","The protein encoded by this gene is a protease that functions as a deubiquitinating enzyme. The encoded protein is thought to help regulate the spindle assembly checkpoint by preventing early anaphase onset. This protein specifically deubiquitinates CDC20, which stabilizes the anaphase promoting complex/cyclosome.","The protein encoded by this gene belongs to the peptidase C19 family, which includes ubiquitinyl hydrolases. This protein deubiquitinates target proteins such as p53 (a tumor suppressor protein) and WASH (essential for endosomal protein recycling), and regulates their activities by counteracting the opposing ubiquitin ligase activity of proteins such as HDM2 and TRIM27, involved in the respective process. Mutations in this gene have been implicated in a neurodevelopmental disorder.","The protein encoded by this gene functions to inhibit WNT proteins, which are extracellular signaling molecules that play a role in embryonic development. This protein contains a WNT inhibitory factor (WIF) domain and five epidermal growth factor (EGF)-like domains, and is thought to be involved in mesoderm segmentation. This gene functions as a tumor suppressor gene, and has been found to be epigenetically silenced in various cancers.","The WNT gene family consists of structurally related genes which encode secreted signaling proteins. These proteins have been implicated in oncogenesis and in several developmental processes, including regulation of cell fate and patterning during embryogenesis. This gene is a member of the WNT gene family. It is very conserved in evolution, and the protein encoded by this gene is known to be 98% identical to the mouse Wnt1 protein at the amino acid level. The studies in mouse indicate that the Wnt1 protein functions in the induction of the mesencephalon and cerebellum. This gene was originally considered as a candidate gene for Joubert syndrome, an autosomal recessive disorder with cerebellar hypoplasia as a leading feature. However,  further studies suggested that the gene mutations might not have a significant role in Joubert syndrome. This gene is clustered with another family member, WNT10B, in the chromosome 12q13 region.","This gene encodes a member of the RecQ subfamily of DNA helicase proteins. The encoded nuclear protein is important in the maintenance of genome stability and plays a role in DNA repair, replication, transcription and telomere maintenance. This protein contains a N-terminal 3' to 5' exonuclease domain, an ATP-dependent helicase domain and RQC (RecQ helicase conserved region) domain in its central region, and a C-terminal HRDC (helicase RNase D C-terminal) domain and nuclear localization signal. Defects in this gene are the cause of Werner syndrome, an autosomal recessive disorder characterized by accelerated aging and an elevated risk for certain cancers.","This gene encodes a zinc finger protein plays a central role in nucleotide excision repair (NER), a specialized type of DNA repair. NER is responsible for repair of UV radiation-induced photoproducts and DNA adducts induced by chemical carcinogens and chemotherapeutic drugs. The encoded protein interacts with DNA and several NER proteins, acting as a scaffold to assemble the NER incision complex at sites of DNA damage. Mutations in this gene cause Xeroderma pigmentosum complementation group A (XP-A), an autosomal recessive skin disorder featuring hypersensitivity to sunlight and increased risk for skin cancer.","The protein encoded by this gene is a key component of the XPC complex, which plays an important role in the early steps of global genome nucleotide excision repair (NER). The encoded protein is important for damage sensing and DNA binding, and shows a preference for single-stranded DNA. Mutations in this gene or some other NER components can result in Xeroderma pigmentosum, a rare autosomal recessive disorder characterized by increased sensitivity to sunlight with the development of carcinomas at an early age. Alternatively spliced transcript variants have been found for this gene.","This gene product belongs to the 14-3-3 family of proteins which mediate signal transduction by binding to phosphoserine-containing proteins. This highly conserved protein family is found in both plants and mammals, and this protein is 100% identical to the mouse ortholog. It interacts with CDC25 phosphatases, RAF1 and IRS1 proteins, suggesting its role in diverse biochemical activities related to signal transduction, such as cell division and regulation of insulin sensitivity. It has also been implicated in the pathogenesis of small cell lung cancer. Two transcript variants, one protein-coding and the other non-protein-coding, have been found for this gene.","Enables double-stranded telomeric DNA binding activity; identical protein binding activity; and transcription cis-regulatory region binding activity. Involved in positive regulation of transcription, DNA-templated and telomere maintenance via telomere lengthening. Located in chromosome, telomeric region. [provided by Alliance of Genome Resources, Apr 2022]","ZC3H12A is an MCP1 (CCL2; MIM 158105)-induced protein that acts as a transcriptional activator and causes cell death of cardiomyocytes, possibly via induction of genes associated with apoptosis.","This gene encodes a downstream effector of bone morphogenetic protein (BMP) signalling. This protein contains a zinc finger domain and functions as a transcriptional coactivator. Variation in this gene may be associated with X-linked cognitive disability. Alternative splicing results in multiple transcript variants.","This gene encodes a scaffold protein which serves as an assessory factor to the nuclear RNA exosome complex. The encoded protein forms a trimeric human nuclear exosome targeting (NEXT) complex, together with hMTR4 and the RNA-binding factor RBM7 which promotes the exosomal degradation of non-coding promoter-upstream transcripts, enhancer RNAs and 3'-extended products of histone- and small nuclear RNA transcription. This complex is also thought to recruit the exosome to degrade intronic RNAs via its interaction with both the exosome and the spliceosome. It contains both an N-terminal zinc-knuckle domain and a C-terminal proline-rich domain.","Pre-mRNA splicing is catalyzed by the spliceosome. U12-type spliceosome binds U12-type pre-mRNAs and recognizes the 5' splice site and branch-point sequence. U11 and U12 snRNPs are components of U12-type spliceosome and function as a molecular bridge connecting both ends of the intron. The protein encoded by this gene contains a RNA recognition motif. It was identified as one of the protein components of U11/U12 snRNPs. This protein and many other U11/U12 snRNP proteins are highly conserved in organisms known to contain U12-type introns. These proteins have been shown to be essential for cell viability, suggesting the key roles in U12-type splicing.","This gene is a member of the TIS11 family of early response genes. Family members are induced by various agonists such as the phorbol ester TPA and the polypeptide mitogen EGF. The encoded protein contains a distinguishing putative zinc finger domain with a repeating cys-his motif. This putative nuclear transcription factor most likely functions in regulating the response to growth factors.","Predicted to enable DNA-binding transcription repressor activity, RNA polymerase II-specific and RNA polymerase II transcription regulatory region sequence-specific DNA binding activity. Involved in negative regulation of transcription by RNA polymerase II. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Zinc finger proteins (ZNFs), such as ZNF208, bind DNA and, through this binding, regulate gene transcription. Most ZNFs contain conserved C2H2 motifs and are classified as Kruppel-type zinc fingers. A conserved protein motif, termed the Kruppel-associated box (KRAB) domain, mediates protein-protein interactions (Eichler et al., 1998). See ZNF91 (MIM 603971) for further information on ZNFs.","This gene encodes a zinc finger protein containing five C2H2-type zinc finger domains, one or two Kruppel-associated box A (KRAB A) domains, and a leucine-rich domain. The encoded protein has been suggested to be a transcriptional repressor. It localizes predominantly to the nucleolus. Alternatively spliced transcript variants encoding different isoforms exist. These variants utilize alternative polyadenylation signals.","Enables DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Involved in positive regulation of transcription by RNA polymerase II. Located in cytosol and nuclear speck. [provided by Alliance of Genome Resources, Apr 2022]","Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription, DNA-templated. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription, DNA-templated. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription, DNA-templated. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Predicted to enable DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Predicted to enable DNA-binding transcription activator activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription by RNA polymerase II. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Predicted to enable DNA-binding transcription factor activity, RNA polymerase II-specific and RNA polymerase II cis-regulatory region sequence-specific DNA binding activity. Predicted to be involved in regulation of transcription, DNA-templated. Predicted to be located in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Enables DNA-binding transcription factor activity. Involved in negative regulation of transcription, DNA-templated and negative regulation of transposon integration. Predicted to be active in nucleus. [provided by Alliance of Genome Resources, Apr 2022]","Predicted to enable ATP binding activity. Predicted to be active in cytoplasm. [provided by Alliance of Genome Resources, Apr 2022]","The ZXDB gene is one of a pair of duplicated zinc finger genes on chromosome Xp11.21 (Greig et al., 1993); see also ZXDA (MIM 300235)."]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>symbol<\/th>\n      <th>entrezgene<\/th>\n      <th>name<\/th>\n      <th>gene_biotype<\/th>\n      <th>intogen_driver<\/th>\n      <th>intogen_role<\/th>\n      <th>ncg_driver<\/th>\n      <th>ncg_phenotype<\/th>\n      <th>tcga_driver<\/th>\n      <th>driver_score<\/th>\n      <th>support_cancermine<\/th>\n      <th>ncbi_function_summary<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"buttons":["csv","excel"],"dom":"Bfrtip","columnDefs":[{"className":"dt-right","targets":[2,10]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"symbol","targets":1},{"name":"entrezgene","targets":2},{"name":"name","targets":3},{"name":"gene_biotype","targets":4},{"name":"intogen_driver","targets":5},{"name":"intogen_role","targets":6},{"name":"ncg_driver","targets":7},{"name":"ncg_phenotype","targets":8},{"name":"tcga_driver","targets":9},{"name":"driver_score","targets":10},{"name":"support_cancermine","targets":11},{"name":"ncbi_function_summary","targets":12}],"order":[],"autoWidth":false,"orderClasses":false,"responsive":true}},"evals":[],"jsHooks":[]}</script><p><br><br></p>
</div>
</div>
<div class="section level2">
<h2 id="get-cancer-predisposition-genes">Get cancer predisposition genes<a class="anchor" aria-label="anchor" href="#get-cancer-predisposition-genes"></a>
</h2>
<p>This show how to retrieve known cancer predisposition genes,
utilizing multiple sources, including Cancer Gene Census, Genomics
England PanelApp, TCGA’s pan-cancer study of germline variants, and
other/user-curated entries.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## load the data</span></span>
<span><span class="va">gene_predisposition</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/get_predisposition.html">get_predisposition</a></span><span class="op">(</span>cache_dir <span class="op">=</span> <span class="va">download_dir</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## Number of cancer predisposition genes</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">nrow</a></span><span class="op">(</span><span class="va">gene_predisposition</span><span class="op">$</span><span class="va">records</span> <span class="op">|&gt;</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span></span>
<span>  <span class="op">!</span><span class="fu">stringr</span><span class="fu">::</span><span class="fu"><a href="https://stringr.tidyverse.org/reference/str_detect.html" class="external-link">str_detect</a></span><span class="op">(</span><span class="va">cpg_source</span>, <span class="st">"^(ACMG_SF|CPIX_PGX_ONCOLOGY)$"</span><span class="op">)</span></span>
<span><span class="op">)</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 560</span></span>
<span></span>
<span><span class="co">## Get statistics regarding how reference sources on </span></span>
<span><span class="co">## cancer predisposition genes contribute</span></span>
<span><span class="co">##</span></span>
<span><span class="co">## CGC - Cancer Gene Census (germline)</span></span>
<span><span class="co">## PANEL_APP - N = 43 gene panels for inherited cancer conditions/</span></span>
<span><span class="co">##             cancer syndromes (Genomics England PanelApp)</span></span>
<span><span class="co">## CURATED_OTHER - curated/user-contributed genes</span></span>
<span><span class="co">## TCGA_PANCAN_2018 - TCGA's pancancer analysis of</span></span>
<span><span class="co">##                    germline variants in cancer</span></span>
<span><span class="co">##                    (Huang et al., Cell, 2019)</span></span>
<span><span class="fu">plyr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/plyr/man/count.html" class="external-link">count</a></span><span class="op">(</span><span class="va">gene_predisposition</span><span class="op">$</span><span class="va">records</span><span class="op">$</span><span class="va">cpg_source</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/desc.html" class="external-link">desc</a></span><span class="op">(</span><span class="va">freq</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">x</span> <span class="op">!=</span> <span class="st">"ACMG_SF"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">x</span> <span class="op">!=</span> <span class="st">"CPIC_PGX_ONCOLOGY"</span><span class="op">)</span></span>
<span><span class="co">#&gt;                                               x freq</span></span>
<span><span class="co">#&gt; 1                                     PANEL_APP  275</span></span>
<span><span class="co">#&gt; 2                                 CURATED_OTHER  117</span></span>
<span><span class="co">#&gt; 3          CANVAR_UK&amp;PANEL_APP&amp;TCGA_PANCAN_2018   68</span></span>
<span><span class="co">#&gt; 4                    PANEL_APP&amp;TCGA_PANCAN_2018   34</span></span>
<span><span class="co">#&gt; 5  ACMG_SF&amp;CANVAR_UK&amp;PANEL_APP&amp;TCGA_PANCAN_2018   24</span></span>
<span><span class="co">#&gt; 6                           CANVAR_UK&amp;PANEL_APP   15</span></span>
<span><span class="co">#&gt; 7                              TCGA_PANCAN_2018   12</span></span>
<span><span class="co">#&gt; 8            ACMG_SF&amp;PANEL_APP&amp;TCGA_PANCAN_2018    4</span></span>
<span><span class="co">#&gt; 9                             ACMG_SF&amp;PANEL_APP    2</span></span>
<span><span class="co">#&gt; 10                                    CANVAR_UK    2</span></span>
<span><span class="co">#&gt; 11                   CANVAR_UK&amp;TCGA_PANCAN_2018    2</span></span>
<span><span class="co">#&gt; 12           ACMG_SF&amp;CANVAR_UK&amp;TCGA_PANCAN_2018    1</span></span>
<span><span class="co">#&gt; 13                     ACMG_SF&amp;TCGA_PANCAN_2018    1</span></span>
<span></span>
<span></span>
<span><span class="co">## Cancer predisposition metadata</span></span>
<span><span class="va">gene_predisposition</span><span class="op">$</span><span class="va">metadata</span></span>
<span><span class="co">#&gt;                                        source</span></span>
<span><span class="co">#&gt; 1                   Genomics England PanelApp</span></span>
<span><span class="co">#&gt; 2                                        NCBI</span></span>
<span><span class="co">#&gt; 3                    Huang et al., Cell, 2018</span></span>
<span><span class="co">#&gt; 4        Maxwell et al., Am J Hum Genet, 2016</span></span>
<span><span class="co">#&gt; 5 Cancer predisposition genes - curated/other</span></span>
<span><span class="co">#&gt; 6                   ACMG - secondary findings</span></span>
<span><span class="co">#&gt; 7                                   CanVar-UK</span></span>
<span><span class="co">#&gt;                                                                               source_description</span></span>
<span><span class="co">#&gt; 1 Collection of &gt; 40 dedicated gene panels for various inherited cancer conditions and syndromes</span></span>
<span><span class="co">#&gt; 2          Basic gene identifiers (symbol, entrez ID, HGNC ID, name, synonyms, function summary)</span></span>
<span><span class="co">#&gt; 3                   Collection of cancer predisposition genes screened in TCGA's pancancer study</span></span>
<span><span class="co">#&gt; 4                                      Mechanisms of inheritance for cancer predisposition genes</span></span>
<span><span class="co">#&gt; 5                         Candidate cancer predisposition genes - contributed e.g. by CPSR users</span></span>
<span><span class="co">#&gt; 6            Genes recommended for reporting of incidental findings in clinical exome sequencing</span></span>
<span><span class="co">#&gt; 7                                                    Cancer predisposition gene variant database</span></span>
<span><span class="co">#&gt;                                  source_url</span></span>
<span><span class="co">#&gt; 1   https://panelapp.genomicsengland.co.uk/</span></span>
<span><span class="co">#&gt; 2        https://www.ncbi.nlm.nih.gov/gene/</span></span>
<span><span class="co">#&gt; 3 https://pubmed.ncbi.nlm.nih.gov/29625052/</span></span>
<span><span class="co">#&gt; 4 https://pubmed.ncbi.nlm.nih.gov/27153395/</span></span>
<span><span class="co">#&gt; 5                                      &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 6 https://pubmed.ncbi.nlm.nih.gov/40568962/</span></span>
<span><span class="co">#&gt; 7                     https://canvaruk.org/</span></span>
<span><span class="co">#&gt;                                   source_citation source_version</span></span>
<span><span class="co">#&gt; 1        Martin et al., Nat Genet, 2019; 31676867       v1 (API)</span></span>
<span><span class="co">#&gt; 2 Brown et al., Nucleic Acids Res, 2015; 25355515     2026-02-21</span></span>
<span><span class="co">#&gt; 3              Huang et al., Cell, 2018; 29625052           &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 4  Maxwell et al., Am J Hum Genet, 2016; 27153395           &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 5                                            &lt;NA&gt;       20260203</span></span>
<span><span class="co">#&gt; 6           Lee et al., Genet Med, 2025; 40568962           v3.3</span></span>
<span><span class="co">#&gt; 7                                            &lt;NA&gt;           v2.2</span></span>
<span><span class="co">#&gt;   source_abbreviation                             source_license</span></span>
<span><span class="co">#&gt; 1                gepa Commercial use requires separate agreement</span></span>
<span><span class="co">#&gt; 2                ncbi                   NCBI data usage policies</span></span>
<span><span class="co">#&gt; 3    tcga_pancan_2018                           Free/open access</span></span>
<span><span class="co">#&gt; 4         maxwell2016                           Free/open access</span></span>
<span><span class="co">#&gt; 5           cpg_other                           Free/open access</span></span>
<span><span class="co">#&gt; 6             acmg_sf                           Free/open access</span></span>
<span><span class="co">#&gt; 7           canvar_uk                           Free/open access</span></span>
<span><span class="co">#&gt;                                                                                 source_license_url</span></span>
<span><span class="co">#&gt; 1 https://panelapp.genomicsengland.co.uk/media/files/GEL_-_PanelApp_Terms_of_Use_December_2019.pdf</span></span>
<span><span class="co">#&gt; 2                                           https://www.ncbi.nlm.nih.gov/home/about/policies/#data</span></span>
<span><span class="co">#&gt; 3                                                                                             &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 4                                                                                             &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 5                                                                                             &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 6                                                  https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/</span></span>
<span><span class="co">#&gt; 7                                                                                             &lt;NA&gt;</span></span></code></pre></div>
<p><br><br></p>
</div>
<div class="section level2">
<h2 id="get-cancer-gene-panels">Get cancer gene panels<a class="anchor" aria-label="anchor" href="#get-cancer-gene-panels"></a>
</h2>
<p>This shows how to retrieve genes from cancer gene panels defined in
Genomics England PanelApp.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## load the data</span></span>
<span><span class="va">gene_panels</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/get_panels.html">get_panels</a></span><span class="op">(</span>cache_dir <span class="op">=</span> <span class="va">download_dir</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## panel data for genome build grch38</span></span>
<span><span class="va">panel_data</span> <span class="op">&lt;-</span> <span class="va">gene_panels</span><span class="op">$</span><span class="va">records</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">genome_build</span> <span class="op">==</span> <span class="st">"grch38"</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## show number of genes in each panel</span></span>
<span><span class="va">gene_freq</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="va">panel_data</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/group_by.html" class="external-link">group_by</a></span><span class="op">(</span><span class="va">gepa_panel_name</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/summarise.html" class="external-link">summarise</a></span><span class="op">(</span>n <span class="op">=</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/context.html" class="external-link">n</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu">desc</span><span class="op">(</span><span class="va">n</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/rename.html" class="external-link">rename</a></span><span class="op">(</span>panel_name <span class="op">=</span> <span class="va">gepa_panel_name</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="va">gene_freq</span></span>
<span><span class="co">#&gt;                                                                  panel_name   n</span></span>
<span><span class="co">#&gt; 1                          DNA Repair Genes pertinent cancer susceptibility 178</span></span>
<span><span class="co">#&gt; 2                                                   Childhood solid tumours 122</span></span>
<span><span class="co">#&gt; 3                         Haematological malignancies cancer susceptibility 108</span></span>
<span><span class="co">#&gt; 4                                 Adult solid tumours cancer susceptibility 105</span></span>
<span><span class="co">#&gt; 5                              Haematological malignancies for rare disease  89</span></span>
<span><span class="co">#&gt; 6                             Childhood solid tumours cancer susceptibility  85</span></span>
<span><span class="co">#&gt; 7                                      Adult solid tumours for rare disease  58</span></span>
<span><span class="co">#&gt; 8                                    Multiple monogenic benign skin tumours  46</span></span>
<span><span class="co">#&gt; 9                                                    Sarcoma susceptibility  44</span></span>
<span><span class="co">#&gt; 10                                            Sarcoma cancer susceptibility  33</span></span>
<span><span class="co">#&gt; 11                                                         GI tract tumours  31</span></span>
<span><span class="co">#&gt; 12                                                 Neurofibromatosis Type 1  30</span></span>
<span><span class="co">#&gt; 13                                   Inherited non-medullary thyroid cancer  29</span></span>
<span><span class="co">#&gt; 14                                                   Familial breast cancer  27</span></span>
<span><span class="co">#&gt; 15                         Inherited ovarian cancer (without breast cancer)  27</span></span>
<span><span class="co">#&gt; 16    Familial Tumours Syndromes of the central &amp; peripheral Nervous system  21</span></span>
<span><span class="co">#&gt; 17                            Inherited phaeochromocytoma and paraganglioma  20</span></span>
<span><span class="co">#&gt; 18 Inherited polyposis and early onset colorectal cancer - germline testing  19</span></span>
<span><span class="co">#&gt; 19                                                Familial rhabdomyosarcoma  18</span></span>
<span><span class="co">#&gt; 20                                                   Inherited renal cancer  18</span></span>
<span><span class="co">#&gt; 21                Inherited predisposition to acute myeloid leukaemia (AML)  16</span></span>
<span><span class="co">#&gt; 22                                               Multiple endocrine tumours  16</span></span>
<span><span class="co">#&gt; 23                                                 Familial prostate cancer  14</span></span>
<span><span class="co">#&gt; 24                        Colorectal cancer pertinent cancer susceptibility  13</span></span>
<span><span class="co">#&gt; 25                                         Genodermatoses with malignancies  13</span></span>
<span><span class="co">#&gt; 26                                              Inherited pancreatic cancer  13</span></span>
<span><span class="co">#&gt; 27                     Head and neck cancer pertinent cancer susceptibility  12</span></span>
<span><span class="co">#&gt; 28                    Neuroendocrine cancer pertinent cancer susceptibility  12</span></span>
<span><span class="co">#&gt; 29                             Renal cancer pertinent cancer susceptibility  10</span></span>
<span><span class="co">#&gt; 30                           Ovarian cancer pertinent cancer susceptibility   9</span></span>
<span><span class="co">#&gt; 31                                                        Familial melanoma   8</span></span>
<span><span class="co">#&gt; 32                             Brain cancer pertinent cancer susceptibility   7</span></span>
<span><span class="co">#&gt; 33                            Breast cancer pertinent cancer susceptibility   7</span></span>
<span><span class="co">#&gt; 34                                         Inherited predisposition to GIST   7</span></span>
<span><span class="co">#&gt; 35                                                       Parathyroid Cancer   7</span></span>
<span><span class="co">#&gt; 36                       Endometrial cancer pertinent cancer susceptibility   6</span></span>
<span><span class="co">#&gt; 37                                Inherited MMR deficiency (Lynch syndrome)   5</span></span>
<span><span class="co">#&gt; 38                          Prostate cancer pertinent cancer susceptibility   5</span></span>
<span><span class="co">#&gt; 39                           Thyroid cancer pertinent cancer susceptibility   5</span></span>
<span><span class="co">#&gt; 40                           Bladder cancer pertinent cancer susceptibility   4</span></span>
<span><span class="co">#&gt; 41            Upper gastrointestinal cancer pertinent cancer susceptibility   4</span></span>
<span><span class="co">#&gt; 42                                 Melanoma pertinent cancer susceptibility   3</span></span>
<span><span class="co">#&gt; 43                                                Familial rhabdoid tumours   2</span></span>
<span><span class="co">#&gt; 44         Inherited susceptibility to acute lymphoblastoid leukaemia (ALL)   2</span></span></code></pre></div>
<p><br><br></p>
</div>
<div class="section level2">
<h2 id="get-gene-aliases">Get gene aliases<a class="anchor" aria-label="anchor" href="#get-gene-aliases"></a>
</h2>
<p>This shows how to retrieve ambiguous and unambiguous gene aliases
(i.e. with respect to primary gene symbols).</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## load the data</span></span>
<span><span class="va">gene_alias</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/get_alias.html">get_alias</a></span><span class="op">(</span>cache_dir <span class="op">=</span> <span class="va">download_dir</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## number of gene synonyms that are ambiguous</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">nrow</a></span><span class="op">(</span><span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">gene_alias</span><span class="op">$</span><span class="va">records</span>, <span class="va">ambiguous</span> <span class="op">==</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 6744</span></span>
<span></span>
<span><span class="co">## show structure of alias records</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">gene_alias</span><span class="op">$</span><span class="va">records</span><span class="op">)</span></span>
<span><span class="co">#&gt;        alias      symbol entrezgene n_primary_map ambiguous source</span></span>
<span><span class="co">#&gt; 1   'C-K-RAS        KRAS       3845             1     FALSE   NCBI</span></span>
<span><span class="co">#&gt; 2     (FM-3)       NMUR1      10316             1     FALSE   NCBI</span></span>
<span><span class="co">#&gt; 3    (IV)-44 IGHVIV-44-1      28337             1     FALSE   NCBI</span></span>
<span><span class="co">#&gt; 4      (P)RR     ATP6AP2      10159             1     FALSE   NCBI</span></span>
<span><span class="co">#&gt; 5 (ppGpp)ase       HDDC3     374659             1     FALSE   NCBI</span></span>
<span><span class="co">#&gt; 6 A-116A10.1        NAE1       8883             1     FALSE   NCBI</span></span>
<span><span class="co">#&gt;   is_primary_symbol other_index</span></span>
<span><span class="co">#&gt; 1             FALSE        &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 2             FALSE        &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 3             FALSE        &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 4             FALSE        &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 5             FALSE        &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 6             FALSE        &lt;NA&gt;</span></span></code></pre></div>
<p><br><br></p>
</div>
<div class="section level2">
<h2 id="get-gencode-transcripts">Get GENCODE transcripts<a class="anchor" aria-label="anchor" href="#get-gencode-transcripts"></a>
</h2>
<p>This shows how to retrieve GENCODE transcripts for
<code>grch37</code> and <code>grch38</code>.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## load the data</span></span>
<span><span class="va">gene_gencode</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/get_gencode.html">get_gencode</a></span><span class="op">(</span>cache_dir <span class="op">=</span> <span class="va">download_dir</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## number of transcript records - grch37</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">nrow</a></span><span class="op">(</span><span class="va">gene_gencode</span><span class="op">$</span><span class="va">records</span><span class="op">$</span><span class="va">grch37</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 196520</span></span>
<span></span>
<span><span class="co">## number of transcript records - grch38</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">nrow</a></span><span class="op">(</span><span class="va">gene_gencode</span><span class="op">$</span><span class="va">records</span><span class="op">$</span><span class="va">grch38</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 507365</span></span>
<span></span>
<span><span class="co">## show colnames for transcript records</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">gene_gencode</span><span class="op">$</span><span class="va">records</span><span class="op">$</span><span class="va">grch38</span><span class="op">)</span></span>
<span><span class="co">#&gt;  [1] "chrom"                      "start"                     </span></span>
<span><span class="co">#&gt;  [3] "end"                        "transcript_start"          </span></span>
<span><span class="co">#&gt;  [5] "transcript_end"             "cds_start"                 </span></span>
<span><span class="co">#&gt;  [7] "strand"                     "ensembl_gene_id"           </span></span>
<span><span class="co">#&gt;  [9] "ensembl_gene_id_full"       "ensembl_transcript_id"     </span></span>
<span><span class="co">#&gt; [11] "ensembl_transcript_id_full" "ensembl_protein_id"        </span></span>
<span><span class="co">#&gt; [13] "symbol"                     "symbol_gencode"            </span></span>
<span><span class="co">#&gt; [15] "hgnc_id"                    "entrezgene"                </span></span>
<span><span class="co">#&gt; [17] "name"                       "gene_biotype"              </span></span>
<span><span class="co">#&gt; [19] "transcript_biotype"         "tag"                       </span></span>
<span><span class="co">#&gt; [21] "refseq_protein_id"          "refseq_transcript_id"      </span></span>
<span><span class="co">#&gt; [23] "mane_select"                "mane_plus_clinical"        </span></span>
<span><span class="co">#&gt; [25] "refseq_select"              "principal_isoform_flag"    </span></span>
<span><span class="co">#&gt; [27] "uniprot_acc"                "uniprot_id"                </span></span>
<span><span class="co">#&gt; [29] "ensembl_version"            "gencode_version"           </span></span>
<span><span class="co">#&gt; [31] "uniprot_version"</span></span>
<span></span>
<span><span class="co">## show metadata for underlying resources</span></span>
<span><span class="va">gene_gencode</span><span class="op">$</span><span class="va">metadata</span></span>
<span><span class="co">#&gt;            source</span></span>
<span><span class="co">#&gt; 1         GENCODE</span></span>
<span><span class="co">#&gt; 2 Ensembl Biomart</span></span>
<span><span class="co">#&gt; 3       UniprotKB</span></span>
<span><span class="co">#&gt; 4          APPRIS</span></span>
<span><span class="co">#&gt;                                                         source_description</span></span>
<span><span class="co">#&gt; 1                                                   Human gene transcripts</span></span>
<span><span class="co">#&gt; 2 API for retrieval of gene and transcript cross-references (MANE, RefSeq)</span></span>
<span><span class="co">#&gt; 3      UniProt identifiers and accessions with cross-references to Ensembl</span></span>
<span><span class="co">#&gt; 4                                 Prinicipal transcript isoform annotation</span></span>
<span><span class="co">#&gt;                                       source_url</span></span>
<span><span class="co">#&gt; 1                  https://www.gencodegenes.org/</span></span>
<span><span class="co">#&gt; 2       https://www.ensembl.org/biomart/martview</span></span>
<span><span class="co">#&gt; 3                        https://www.uniprot.org</span></span>
<span><span class="co">#&gt; 4 https://apprisws.bioinfo.cnio.es/landing_page/</span></span>
<span><span class="co">#&gt;                                         source_citation source_version</span></span>
<span><span class="co">#&gt; 1    Frankish et al., Nucleic Acids Res, 2021; 33270111             49</span></span>
<span><span class="co">#&gt; 2  Cunningham et al., Nucleic Acids Res, 2022; 34791404            115</span></span>
<span><span class="co">#&gt; 3 UniProt Consortium, Nucleic Acids Res, 2021; 33237286        2026_01</span></span>
<span><span class="co">#&gt; 4    Rodriguez et al, Nucleic Acids Res, 2022; 34755885     2026-02-21</span></span>
<span><span class="co">#&gt;   source_abbreviation        source_license</span></span>
<span><span class="co">#&gt; 1             gencode      Free/open access</span></span>
<span><span class="co">#&gt; 2             ensembl EMBL-EBI terms of use</span></span>
<span><span class="co">#&gt; 3             uniprot             CC BY 4.0</span></span>
<span><span class="co">#&gt; 4              appris      Free/open access</span></span>
<span><span class="co">#&gt;                             source_license_url</span></span>
<span><span class="co">#&gt; 1                                         &lt;NA&gt;</span></span>
<span><span class="co">#&gt; 2     https://www.ebi.ac.uk/about/terms-of-use</span></span>
<span><span class="co">#&gt; 3 https://creativecommons.org/licenses/by/4.0/</span></span>
<span><span class="co">#&gt; 4                                         &lt;NA&gt;</span></span></code></pre></div>
<p><br><br></p>
</div>
<div class="section level2">
<h2 id="get-dna-repair-genes">Get DNA repair genes<a class="anchor" aria-label="anchor" href="#get-dna-repair-genes"></a>
</h2>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## load the data</span></span>
<span><span class="va">gene_dna_repair</span> <span class="op">&lt;-</span> <span class="va">gene_basic</span><span class="op">$</span><span class="va">records</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/NA.html" class="external-link">is.na</a></span><span class="op">(</span><span class="va">woods_dnarepair_class</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span><span class="va">symbol</span>, <span class="va">woods_dnarepair_class</span>,</span>
<span>                <span class="va">woods_dnarepair_activity</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## count number of genes in each class</span></span>
<span><span class="va">dna_repair_class_freq</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span></span>
<span>  <span class="va">gene_dna_repair</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/group_by.html" class="external-link">group_by</a></span><span class="op">(</span><span class="va">woods_dnarepair_class</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/summarise.html" class="external-link">summarise</a></span><span class="op">(</span>n <span class="op">=</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/context.html" class="external-link">n</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu">desc</span><span class="op">(</span><span class="va">n</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/rename.html" class="external-link">rename</a></span><span class="op">(</span>dna_repair_class <span class="op">=</span> <span class="va">woods_dnarepair_class</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="va">dna_repair_class_freq</span></span>
<span><span class="co">#&gt;                                                                  dna_repair_class</span></span>
<span><span class="co">#&gt; 1                                                Nucleotide excision repair (NER)</span></span>
<span><span class="co">#&gt; 2                                                        Homologous recombination</span></span>
<span><span class="co">#&gt; 3                                                                  Fanconi anemia</span></span>
<span><span class="co">#&gt; 4                                            DNA polymerases (catalytic subunits)</span></span>
<span><span class="co">#&gt; 5                                       Other conserved DNA damage response genes</span></span>
<span><span class="co">#&gt; 6                                                      Base excision repair (BER)</span></span>
<span><span class="co">#&gt; 7                                                 Ubiquitination and modification</span></span>
<span><span class="co">#&gt; 8                                                  Mismatch excision repair (MMR)</span></span>
<span><span class="co">#&gt; 9              Other identified genes with known or suspected DNA repair function</span></span>
<span><span class="co">#&gt; 10                                               Editing and processing nucleases</span></span>
<span><span class="co">#&gt; 11                                                     Non-homologous end-joining</span></span>
<span><span class="co">#&gt; 12                                     Other BER and strand break joining factors</span></span>
<span><span class="co">#&gt; 13 Genes defective in diseases associated with sensitivity to DNA damaging agents</span></span>
<span><span class="co">#&gt; 14                                           Chromatin Structure and Modification</span></span>
<span><span class="co">#&gt; 15                                                      Direct reversal of damage</span></span>
<span><span class="co">#&gt; 16                                                 Modulation of nucleotide pools</span></span>
<span><span class="co">#&gt; 17                    Poly(ADP-ribose) polymerase (PARP) enzymes that bind to DNA</span></span>
<span><span class="co">#&gt; 18                                         Repair of DNA-topoisomerase crosslinks</span></span>
<span><span class="co">#&gt; 19                                                                        Unknown</span></span>
<span><span class="co">#&gt;     n</span></span>
<span><span class="co">#&gt; 1  28</span></span>
<span><span class="co">#&gt; 2  21</span></span>
<span><span class="co">#&gt; 3  17</span></span>
<span><span class="co">#&gt; 4  15</span></span>
<span><span class="co">#&gt; 5  15</span></span>
<span><span class="co">#&gt; 6  11</span></span>
<span><span class="co">#&gt; 7  11</span></span>
<span><span class="co">#&gt; 8  10</span></span>
<span><span class="co">#&gt; 9   9</span></span>
<span><span class="co">#&gt; 10  8</span></span>
<span><span class="co">#&gt; 11  7</span></span>
<span><span class="co">#&gt; 12  6</span></span>
<span><span class="co">#&gt; 13  5</span></span>
<span><span class="co">#&gt; 14  3</span></span>
<span><span class="co">#&gt; 15  3</span></span>
<span><span class="co">#&gt; 16  3</span></span>
<span><span class="co">#&gt; 17  3</span></span>
<span><span class="co">#&gt; 18  2</span></span>
<span><span class="co">#&gt; 19  1</span></span></code></pre></div>
<p><br><br></p>
</div>
<div class="section level2">
<h2 id="get-tso500-genes">Get TSO500 genes<a class="anchor" aria-label="anchor" href="#get-tso500-genes"></a>
</h2>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## load the data</span></span>
<span><span class="va">gene_tso500</span> <span class="op">&lt;-</span> <span class="va">gene_basic</span><span class="op">$</span><span class="va">records</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/NA.html" class="external-link">is.na</a></span><span class="op">(</span><span class="va">illumina_tso500</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## number of genes covered by TSO500 panel</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">nrow</a></span><span class="op">(</span><span class="va">gene_tso500</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 523</span></span>
<span></span>
<span><span class="co">## types of variant types covered</span></span>
<span><span class="va">illumina_tso500_variant_freq</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span></span>
<span>  <span class="va">gene_tso500</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/group_by.html" class="external-link">group_by</a></span><span class="op">(</span><span class="va">illumina_tso500</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/summarise.html" class="external-link">summarise</a></span><span class="op">(</span>n <span class="op">=</span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/context.html" class="external-link">n</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu">desc</span><span class="op">(</span><span class="va">n</span><span class="op">)</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="va">illumina_tso500_variant_freq</span></span>
<span><span class="co">#&gt;                 illumina_tso500   n</span></span>
<span><span class="co">#&gt; 1                     SNV_INDEL 433</span></span>
<span><span class="co">#&gt; 2            CNA_GAIN,SNV_INDEL  33</span></span>
<span><span class="co">#&gt; 3          RNA_FUSION,SNV_INDEL  31</span></span>
<span><span class="co">#&gt; 4 CNA_GAIN,RNA_FUSION,SNV_INDEL  22</span></span>
<span><span class="co">#&gt; 5 CNA_LOSS,RNA_FUSION,SNV_INDEL   2</span></span>
<span><span class="co">#&gt; 6            CNA_LOSS,SNV_INDEL   2</span></span></code></pre></div>
<p><br><br></p>
</div>
<div class="section level2">
<h2 id="session-info">Session Info<a class="anchor" aria-label="anchor" href="#session-info"></a>
</h2>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># set eval = FALSE if you don't want this info (useful for reproducibility) </span></span>
<span><span class="co"># to appear</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; R version 4.5.2 (2025-10-31)</span></span>
<span><span class="co">#&gt; Platform: x86_64-pc-linux-gnu</span></span>
<span><span class="co">#&gt; Running under: Ubuntu 24.04.3 LTS</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Matrix products: default</span></span>
<span><span class="co">#&gt; BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 </span></span>
<span><span class="co">#&gt; LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.26.so;  LAPACK version 3.12.0</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; locale:</span></span>
<span><span class="co">#&gt;  [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       </span></span>
<span><span class="co">#&gt;  [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   </span></span>
<span><span class="co">#&gt;  [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          </span></span>
<span><span class="co">#&gt; [10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; time zone: UTC</span></span>
<span><span class="co">#&gt; tzcode source: system (glibc)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attached base packages:</span></span>
<span><span class="co">#&gt; [1] stats     graphics  grDevices utils     datasets  methods   base     </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; other attached packages:</span></span>
<span><span class="co">#&gt; [1] geneOncoX_1.3.0</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; loaded via a namespace (and not attached):</span></span>
<span><span class="co">#&gt;  [1] jsonlite_2.0.0    dplyr_1.2.0       compiler_4.5.2    crayon_1.5.3     </span></span>
<span><span class="co">#&gt;  [5] Rcpp_1.1.1        tidyselect_1.2.1  stringr_1.6.0     jquerylib_0.1.4  </span></span>
<span><span class="co">#&gt;  [9] systemfonts_1.3.1 textshaping_1.0.4 yaml_2.3.12       fastmap_1.2.0    </span></span>
<span><span class="co">#&gt; [13] plyr_1.8.9        R6_2.6.1          generics_0.1.4    curl_7.0.0       </span></span>
<span><span class="co">#&gt; [17] knitr_1.51        htmlwidgets_1.6.4 tibble_3.3.1      desc_1.4.3       </span></span>
<span><span class="co">#&gt; [21] bslib_0.10.0      pillar_1.11.1     rlang_1.1.7       DT_0.34.0        </span></span>
<span><span class="co">#&gt; [25] stringi_1.8.7     cachem_1.1.0      lgr_0.5.2         xfun_0.56        </span></span>
<span><span class="co">#&gt; [29] fs_1.6.6          sass_0.4.10       otel_0.2.0        cli_3.6.5        </span></span>
<span><span class="co">#&gt; [33] withr_3.0.2       pkgdown_2.2.0     magrittr_2.0.4    crosstalk_1.2.2  </span></span>
<span><span class="co">#&gt; [37] digest_0.6.39     lifecycle_1.0.5   vctrs_0.7.1       evaluate_1.0.5   </span></span>
<span><span class="co">#&gt; [41] gargle_1.6.1      glue_1.8.0        ragg_1.5.0        googledrive_2.1.2</span></span>
<span><span class="co">#&gt; [45] httr_1.4.8        rmarkdown_2.30    purrr_1.2.1       tools_4.5.2      </span></span>
<span><span class="co">#&gt; [49] pkgconfig_2.0.3   htmltools_0.5.9</span></span></code></pre></div>
<p><br><br></p>
</div>
<div class="section level2">
<h2 class="unnumbered" id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0">
<div id="ref-Bailey2018-os" class="csl-entry">
Bailey, Matthew H, Collin Tokheim, Eduard Porta-Pardo, Sohini Sengupta,
Denis Bertrand, Amila Weerasinghe, Antonio Colaprico, et al. 2018.
<span>“Comprehensive Characterization of Cancer Driver Genes and
Mutations.”</span> <em>Cell</em> 173 (2): 371–385.e18. <a href="http://dx.doi.org/10.1016/j.cell.2018.02.060" class="external-link">http://dx.doi.org/10.1016/j.cell.2018.02.060</a>.
</div>
<div id="ref-Lever2019-xp" class="csl-entry">
Lever, Jake, Eric Y Zhao, Jasleen Grewal, Martin R Jones, and Steven J M
Jones. 2019. <span>“<span>CancerMine</span>: A Literature-Mined Resource
for Drivers, Oncogenes and Tumor Suppressors in Cancer.”</span> <em>Nat.
Methods</em> 16 (6): 505–7. <a href="http://dx.doi.org/10.1038/s41592-019-0422-y" class="external-link">http://dx.doi.org/10.1038/s41592-019-0422-y</a>.
</div>
<div id="ref-Liu2020-ga" class="csl-entry">
Liu, Xiaoming, Chang Li, Chengcheng Mou, Yibo Dong, and Yicheng Tu.
2020. <span>“<span class="nocase">dbNSFP</span> V4: A Comprehensive
Database of Transcript-Specific Functional Predictions and Annotations
for Human Nonsynonymous and Splice-Site <span>SNVs</span>.”</span>
<em>Genome Med.</em> 12 (1): 103. <a href="http://dx.doi.org/10.1186/s13073-020-00803-9" class="external-link">http://dx.doi.org/10.1186/s13073-020-00803-9</a>.
</div>
<div id="ref-Martin2019-nq" class="csl-entry">
Martin, Antonio Rueda, Eleanor Williams, Rebecca E Foulger, Sarah Leigh,
Louise C Daugherty, Olivia Niblock, Ivone U S Leong, et al. 2019.
<span>“<span>PanelApp</span> Crowdsources Expert Knowledge to Establish
Consensus Diagnostic Gene Panels.”</span> <em>Nat. Genet.</em> 51
(November): 1560–65. <a href="http://dx.doi.org/10.1038/s41588-019-0528-2" class="external-link">http://dx.doi.org/10.1038/s41588-019-0528-2</a>.
</div>
<div id="ref-Martinez-Jimenez2020-qx" class="csl-entry">
Martı́nez-Jiménez, Francisco, Ferran Muiños, Inés Sentı́s, Jordi Deu-Pons,
Iker Reyes-Salazar, Claudia Arnedo-Pac, Loris Mularoni, et al. 2020.
<span>“A Compendium of Mutational Cancer Driver Genes.”</span> <em>Nat.
Rev. Cancer</em> 20 (10): 555–72. <a href="https://www.nature.com/articles/s41568-020-0290-x" class="external-link">https://www.nature.com/articles/s41568-020-0290-x</a>.
</div>
<div id="ref-Repana2019-dd" class="csl-entry">
Repana, Dimitra, Joel Nulsen, Lisa Dressler, Michele Bortolomeazzi,
Santhilata Kuppili Venkata, Aikaterini Tourna, Anna Yakovleva, Tommaso
Palmieri, and Francesca D Ciccarelli. 2019. <span>“The Network of Cancer
Genes (<span>NCG)</span>: A Comprehensive Catalogue of Known and
Candidate Cancer Genes from Cancer Sequencing Screens.”</span>
<em>Genome Biol.</em> 20 (1): 1. <a href="http://dx.doi.org/10.1186/s13059-018-1612-0" class="external-link">http://dx.doi.org/10.1186/s13059-018-1612-0</a>.
</div>
<div id="ref-Sondka2018-wf" class="csl-entry">
Sondka, Zbyslaw, Sally Bamford, Charlotte G Cole, Sari A Ward, Ian
Dunham, and Simon A Forbes. 2018. <span>“The <span>COSMIC</span> Cancer
Gene Census: Describing Genetic Dysfunction Across All Human
Cancers.”</span> <em>Nat. Rev. Cancer</em> 18 (11): 696–705. <a href="http://dx.doi.org/10.1038/s41568-018-0060-1" class="external-link">http://dx.doi.org/10.1038/s41568-018-0060-1</a>.
</div>
<div id="ref-Wood2001-nk" class="csl-entry">
Wood, R D, M Mitchell, J Sgouros, and T Lindahl. 2001. <span>“Human
<span>DNA</span> Repair Genes.”</span> <em>Science</em> 291 (5507):
1284–89. <a href="http://dx.doi.org/10.1126/science.1056154" class="external-link">http://dx.doi.org/10.1126/science.1056154</a>.
</div>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by <a href="https://github.com/sigven" class="external-link">Sigve Nakken</a>.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.2.0.</p>
</div>

    </footer>
</div>





  </body>
</html>
